# abctools

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# abd

Version: 0.2-8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ABHgenotypeR

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# acc

Version: 1.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# accSDA

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rARPACK’ ‘sparseLDA’
      All declared Imports should be used.
    ```

# ACDm

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ACSNMineR

Version: 0.16.8.25

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# actogrammr

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# adapr

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘plotly’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# AdaptGauss

Version: 1.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# adaptiveGPCA

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# adegenet

Version: 2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        R       2.1Mb
        data    1.3Mb
        files   1.7Mb
    ```

# adklakedata

Version: 0.6.1

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘adklakedata-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: adk_lake_shapes
    > ### Title: Return path to Lake Polygons Shapefile
    > ### Aliases: adk_lake_shapes
    > 
    > ### ** Examples
    > 
    > library(sf)
    Error in library(sf) : there is no package called ‘sf’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# admixturegraph

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ADMMsigma

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# advclust

Version: 0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# agridat

Version: 1.13

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# agriTutorial

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ahnr

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# AID

Version: 2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# aimPlot

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# aire.zmvm

Version: 0.6.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 52 marked UTF-8 strings
    ```

# alakazam

Version: 0.2.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# alfred

Version: 0.1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# AlignStat

Version: 1.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# alphahull

Version: 2.1

## In both

*   checking whether package ‘alphahull’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/alphahull/new/alphahull.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘alphahull’ ...
** package ‘alphahull’ successfully unpacked and MD5 sums checked
** libs
gfortran   -fPIC  -g -O2  -c delaunaycircum.f -o delaunaycircum.o
gfortran   -fPIC  -g -O2  -c tripack.f -o tripack.o
/usr/local/clang4/bin/clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o alphahull.so delaunaycircum.o tripack.o -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/alphahull/new/alphahull.Rcheck/alphahull/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘alphahull’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/alphahull/new/alphahull.Rcheck/alphahull’

```
### CRAN

```
* installing *source* package ‘alphahull’ ...
** package ‘alphahull’ successfully unpacked and MD5 sums checked
** libs
gfortran   -fPIC  -g -O2  -c delaunaycircum.f -o delaunaycircum.o
gfortran   -fPIC  -g -O2  -c tripack.f -o tripack.o
/usr/local/clang4/bin/clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o alphahull.so delaunaycircum.o tripack.o -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/alphahull/old/alphahull.Rcheck/alphahull/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘alphahull’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/alphahull/old/alphahull.Rcheck/alphahull’

```
# ameco

Version: 0.2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.2Mb
      sub-directories of 1Mb or more:
        data  16.1Mb
    ```

# aMNLFA

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘gridExtra’
      All declared Imports should be used.
    ```

# amplican

Version: 1.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: metaplot_deletions
    > ### Title: MetaPlots deletions using ggplot2 and ggbio.
    > ### Aliases: metaplot_deletions
    > 
    > ### ** Examples
    > 
    > #example config
    > config <- read.csv(system.file("extdata", "results", "config_summary.csv",
    +                                package = "amplican"))
    > #example alignments results
    > alignments_file <- system.file("extdata", "results", "alignments",
    +                                "events_filtered_shifted_normalized.csv",
    +                                package = "amplican")
    > alignments <- read.csv(alignments_file)
    > metaplot_deletions(alignments[alignments$consensus, ],
    +                    config, "Group", "Betty")
    Error in rep(startX, each = length(xx)) : 
      attempt to replicate an object of type 'language'
    Calls: metaplot_deletions ... ggplot_deletions -> <Anonymous> -> <Anonymous> -> .local
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      2: ggbio::geom_arch(data = archRanges, ggplot2::aes(alpha = frequency, size = frequency, 
             height = frequency, x = start, xend = end, colour = seqnames))
      3: ggbio::geom_arch(data = archRanges, ggplot2::aes(alpha = frequency, size = frequency, 
             height = frequency, x = start, xend = end, colour = seqnames))
      4: .local(data, ...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 53 SKIPPED: 0 FAILED: 4
      1. Error: Mismatch plot is a construct of grobs. (@test_plot_helpers.R#37) 
      2. Error: Deletions plot is a construct of grobs. (@test_plot_helpers.R#48) 
      3. Error: Insertions plot is a construct of grobs. (@test_plot_helpers.R#59) 
      4. Error: Cuts plot is returning a plot. (@test_plot_helpers.R#70) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: data.table
    
    Attaching package: 'data.table'
    
    The following object is masked from 'package:IRanges':
    
        shift
    
    The following objects are masked from 'package:S4Vectors':
    
        first, second
    
    Quitting from lines 6-7 (example_amplicon_report.Rmd) 
    Quitting from lines 50-51 (example_amplicon_report.Rmd) 
    Error: processing vignette 'example_amplicon_report.Rmd' failed with diagnostics:
    attempt to replicate an object of type 'language'
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.4Mb
      sub-directories of 1Mb or more:
        doc  12.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘CrispRVariants’
    ```

# AmpliconDuo

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ampliQueso

Version: 1.18.0

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/Users/max/github/ggplot2/revdep/checks.noindex/ampliQueso/new/ampliQueso.Rcheck/ampliQueso’, resetting
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘rnaSeqMap’ ‘knitr’ ‘rgl’ ‘ggplot2’ ‘gplots’ ‘parallel’ ‘doParallel’
      ‘foreach’ ‘VariantAnnotation’ ‘genefilter’ ‘statmod’ ‘xtable’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘samr’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘VariantAnnotation’ ‘doParallel’ ‘foreach’ ‘genefilter’ ‘ggplot2’
      ‘gplots’ ‘knitr’ ‘parallel’ ‘rgl’ ‘statmod’ ‘xtable’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘rnaSeqMap:::newSeqReads’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    compareCoverages: no visible global function definition for
      ‘read.table’
    getCountTable: no visible global function definition for ‘read.table’
    runAQReport: no visible global function definition for ‘read.table’
    runAQReport: no visible global function definition for ‘rowttests’
    runAQReport: no visible global function definition for ‘p.adjust’
    runAQReport: no visible global function definition for ‘%do%’
    runAQReport: no visible global function definition for ‘foreach’
    runAQReport: no visible global function definition for ‘aggregate’
    runAQReport: no visible global function definition for ‘knit’
    Undefined global functions or variables:
      %do% %dopar% aggregate combn detectCores end foreach hasArg knit
      makeCluster p.adjust permp qual ranges read.table readVcf ref
      registerDoParallel rowttests seqnames start stopCluster str strand
      width
    Consider adding
      importFrom("methods", "hasArg")
      importFrom("stats", "aggregate", "end", "p.adjust", "start")
      importFrom("utils", "combn", "read.table", "str")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Attaching package: ‘VariantAnnotation’
    
    The following object is masked from ‘package:base’:
    
        tabulate
    
    Loading required package: genefilter
    
    Attaching package: ‘genefilter’
    
    The following objects are masked from ‘package:matrixStats’:
    
        rowSds, rowVars
    
    Loading required package: statmod
    Loading required package: xtable
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ampliQueso.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Anaquin

Version: 2.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DESeq2’
      All declared Imports should be used.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: plotConjoint.Rd:26-27: Dropping empty section \examples
    ```

# anchoredDistr

Version: 1.0.3

## Newly broken

*   checking whether package ‘anchoredDistr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘anchoredDistr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/anchoredDistr/new/anchoredDistr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# AnglerCreelSurveySimulation

Version: 1.0.2

## Newly broken

*   checking whether package ‘AnglerCreelSurveySimulation’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘AnglerCreelSurveySimulation’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/AnglerCreelSurveySimulation/new/AnglerCreelSurveySimulation.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ANOM

Version: 0.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: mvtnorm
    Loading required package: survival
    Loading required package: TH.data
    Loading required package: MASS
    
    Attaching package: 'TH.data'
    
    The following object is masked from 'package:MASS':
    
        geyser
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ANOM.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# anomalize

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘tidyquant’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        help   4.7Mb
    ```

# anomalyDetection

Version: 0.2.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘caret’
      All declared Imports should be used.
    ```

# antitrust

Version: 0.99.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'manual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# aop

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    ...
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    unsupported SEXPTYPE: 19
    > steatosis_aop_graph <- convert_aop_to_graph(steatosis_aop)
    Error in (function (cl, name, valueClass)  : 
      assignment of an object of class “NULL” is not valid for @‘nodes’ in an object of class “graphNEL”; is(value, "vector") is not TRUE
    Calls: convert_aop_to_graph ... new -> initialize -> initialize -> .local -> <Anonymous>
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
       
      1 Test Suite : 
      aop RUnit Tests - 1 test function, 1 error, 0 failures
      ERROR in test_aop_backdoor: Error in (function (cl, name, valueClass)  : 
        assignment of an object of class "NULL" is not valid for @'nodes' in an object of class "graphNEL"; is(value, "vector") is not TRUE
      
      Test files with failing tests
      
         test_functions.R 
           test_aop_backdoor 
      
      
      Error in BiocGenerics:::testPackage("aop", pattern = "^test_.*\\.R$") : 
        unit tests failed for package aop
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 48-50 (aop.Rmd) 
    Error: processing vignette 'aop.Rmd' failed with diagnostics:
    assignment of an object of class "NULL" is not valid for @'nodes' in an object of class "graphNEL"; is(value, "vector") is not TRUE
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# aoristic

Version: 0.6

## In both

*   checking whether package ‘aoristic’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/aoristic/new/aoristic.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘aoristic’ ...
** package ‘aoristic’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘spatstat’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘polyclip’
Error : package ‘spatstat’ could not be loaded
ERROR: lazy loading failed for package ‘aoristic’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/aoristic/new/aoristic.Rcheck/aoristic’

```
### CRAN

```
* installing *source* package ‘aoristic’ ...
** package ‘aoristic’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘spatstat’ in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
 there is no package called ‘polyclip’
Error : package ‘spatstat’ could not be loaded
ERROR: lazy loading failed for package ‘aoristic’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/aoristic/old/aoristic.Rcheck/aoristic’

```
# apsimr

Version: 1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'apsimr-basics.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘sensitivity’, ‘APSIMBatch’
    ```

# ArchaeoPhases

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘DT’
      All declared Imports should be used.
    ```

# archetypes

Version: 2.2-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘MASS’ ‘TSP’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    xyplot.stepArchetypes: no visible global function definition for ‘gray’
    xyplot.stepArchetypes: no visible global function definition for ‘plot’
    xyplot.stepArchetypes: no visible global function definition for
      ‘points’
    xyplot.stepArchetypes: no visible global function definition for
      ‘lines’
    xyplot.weightedArchetypes: no visible binding for global variable
      ‘gray’
    Undefined global functions or variables:
      TSP abline arrows axis box chull cmdscale combn dist ecdf ginv gray
      head history layout lines matlines matplot median mtext optim par
      plot points polygon rgb rnorm sd solve_TSP symbols tail text
    Consider adding
      importFrom("grDevices", "chull", "gray", "rgb")
      importFrom("graphics", "abline", "arrows", "axis", "box", "layout",
                 "lines", "matlines", "matplot", "mtext", "par", "plot",
                 "points", "polygon", "symbols", "text")
      importFrom("stats", "cmdscale", "dist", "ecdf", "median", "optim",
                 "rnorm", "sd")
      importFrom("utils", "combn", "head", "history", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      "axes" is not a graphical parameter
    Warning in plot.xy(xy.coords(x, y), type = type, ...) :
      "axes" is not a graphical parameter
    Warning in method(..., k = k[i]) : k=11: zs > maxKappa
    Warning in method(..., k = k[i]) : k=11: zs > maxKappa
    Warning in method(..., k = k[i]) : k=12: zs > maxKappa
    Warning in method(..., k = k[i]) : k=12: zs > maxKappa
    Warning in method(..., k = k[i]) : k=12: zs > maxKappa
    Warning in method(..., k = k[i]) : k=13: zs > maxKappa
    Warning in method(..., k = k[i]) : k=13: zs > maxKappa
    Warning in method(..., k = k[i]) : k=13: zs > maxKappa
    Warning in method(..., k = k[i]) : k=14: zs > maxKappa
    Warning in method(..., k = k[i]) : k=14: zs > maxKappa
    Warning in method(..., k = k[i]) : k=14: zs > maxKappa
    Warning in method(..., k = k[i]) : k=15: zs > maxKappa
    Warning in axis(1, at = a, ...) : "axes" is not a graphical parameter
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'archetypes.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# archivist

Version: 2.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘archivist.github’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘archivist.github’
    ```

# ARPobservation

Version: 1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Observation-algorithms.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    reported_observations: no visible global function definition for
      ‘aggregate’
    reported_observations: no visible binding for global variable ‘var’
    rgamma_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rgamma_eq: no visible global function definition for ‘runif’
    rgamma_mix_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rgamma_mix_eq: no visible global function definition for ‘runif’
    rweibull_eq : <anonymous>: no visible global function definition for
      ‘uniroot’
    rweibull_eq: no visible global function definition for ‘runif’
    smooth_cov: no visible global function definition for ‘dist’
    Undefined global functions or variables:
      aggregate dist integrate nobs pgamma qnorm quantile rbinom rexp
      rgamma rnorm runif rweibull uniroot var
    Consider adding
      importFrom("stats", "aggregate", "dist", "integrate", "nobs", "pgamma",
                 "qnorm", "quantile", "rbinom", "rexp", "rgamma", "rnorm",
                 "runif", "rweibull", "uniroot", "var")
    to your NAMESPACE file.
    ```

# ARTool

Version: 0.10.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ashr

Version: 2.2-7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘REBayes’ ‘Rmosek’
    ```

# ASICS

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# aslib

Version: 0.1

## In both

*   checking whether package ‘aslib’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/aslib/new/aslib.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘aslib’ ...
** package ‘aslib’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘aslib’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/aslib/new/aslib.Rcheck/aslib’

```
### CRAN

```
* installing *source* package ‘aslib’ ...
** package ‘aslib’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/aslib/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘aslib’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/aslib/old/aslib.Rcheck/aslib’

```
# asremlPlus

Version: 2.0-12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘asreml’
    ```

# ASSIGN

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# assignPOP

Version: 1.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# asVPC

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    VPC.graph : findQuantile: no visible global function definition for
      ‘quantile’
    VPC.graph : findQuantileCI : <anonymous>: no visible global function
      definition for ‘quantile’
    asVPC.binW: no visible global function definition for ‘median’
    asVPC.binW : <anonymous>: no visible global function definition for
      ‘quantile’
    asVPC.binW: no visible global function definition for ‘quantile’
    asVPC.distanceW: no visible global function definition for ‘median’
    asVPC.distanceW : <anonymous>: no visible global function definition
      for ‘quantile’
    asVPC.distanceW: no visible global function definition for ‘quantile’
    makeCOVbin: no visible global function definition for ‘quantile’
    makeCOVbin: no visible global function definition for ‘median’
    read_Simdata: no visible global function definition for ‘read.table’
    Undefined global functions or variables:
      median quantile read.table
    Consider adding
      importFrom("stats", "median", "quantile")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# atlantistools

Version: 0.4.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# autoimage

Version: 2.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      1: spatstat::owin at testthat/test-rotate.R:14
      2: getExportedValue(pkg, name)
      3: asNamespace(ns)
      4: getNamespace(ns)
      5: tryCatch(loadNamespace(name), error = function(e) stop(e))
      6: tryCatchList(expr, classes, parentenv, handlers)
      7: tryCatchOne(expr, names, parentenv, handlers[[1L]])
      8: value[[3L]](cond)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 126 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test-rotate.R#14) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘akima’
    ```

# automap

Version: 1.0-14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘gpclib’ ‘maptools’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘geom_point’
    posPredictionInterval: no visible global function definition for
      ‘median’
    posPredictionInterval: no visible global function definition for
      ‘qnorm’
    summary.autoKrige.cv: no visible global function definition for ‘cor’
    summary.autoKrige.cv: no visible global function definition for ‘sd’
    summary.autoKrige.cv: no visible global function definition for ‘IQR’
    Undefined global functions or variables:
      IQR aes_string as.formula chull coord_equal cor facet_wrap fortify
      geom_path geom_point ggplot grey is median modifyList qnorm quantile
      scale_color_gradient2 scale_size_continuous scale_x_continuous
      scale_y_continuous sd
    Consider adding
      importFrom("grDevices", "chull", "grey")
      importFrom("methods", "is")
      importFrom("stats", "IQR", "as.formula", "cor", "median", "qnorm",
                 "quantile", "sd")
      importFrom("utils", "modifyList")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# autoplotly

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BaalChIP

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 200.0Mb
      sub-directories of 1Mb or more:
        data   96.1Mb
        doc     1.6Mb
        test  101.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    applyBayes: no visible binding for global variable ‘SNP_id’
    plot.filt.barplot: no visible binding for global variable ‘cellname’
    plot.filt.barplot: no visible binding for global variable ‘value’
    plot.filt.barplot: no visible binding for global variable ‘variable’
    plot.filt.boxplot: no visible binding for global variable ‘variable’
    plot.filt.boxplot: no visible binding for global variable ‘value’
    plot.filt.boxplot: no visible binding for global variable ‘coltype’
    plot.filt.pie: no visible binding for global variable ‘variable’
    plot.filt.pie: no visible binding for global variable ‘value.mean’
    plot.simul: no visible binding for global variable ‘readslen’
    plot.simul: no visible binding for global variable ‘perc_right’
    plotadjustment: no visible binding for global variable ‘value’
    plotadjustment: no visible binding for global variable ‘variable’
    Undefined global functions or variables:
      SNP_id cellname coltype perc_right readslen value value.mean variable
    ```

# BACA

Version: 1.3

## In both

*   checking whether package ‘BACA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACA/new/BACA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BACA’ ...
** package ‘BACA’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACA/new/BACA.Rcheck/BACA’

```
### CRAN

```
* installing *source* package ‘BACA’ ...
** package ‘BACA’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACA/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACA/old/BACA.Rcheck/BACA’

```
# BacArena

Version: 1.7

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘glpkAPI’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# BACCT

Version: 1.0

## In both

*   checking whether package ‘BACCT’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/new/BACCT.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BACCT’ ...
** package ‘BACCT’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACCT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/new/BACCT.Rcheck/BACCT’

```
### CRAN

```
* installing *source* package ‘BACCT’ ...
** package ‘BACCT’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BACCT/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BACCT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BACCT/old/BACCT.Rcheck/BACCT’

```
# backShift

Version: 0.1.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘InvariantCausalPrediction’, ‘CompareCausalNetworks’
    ```

# bacon

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .hist: no visible binding for global variable ‘..density..’
    .qq: no visible binding for global variable ‘column’
    Undefined global functions or variables:
      ..density.. column
    ```

# ballr

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bamdit

Version: 3.1.0

## In both

*   checking whether package ‘bamdit’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/new/bamdit.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘bamdit’ ...
** package ‘bamdit’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘bamdit’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/new/bamdit.Rcheck/bamdit’

```
### CRAN

```
* installing *source* package ‘bamdit’ ...
** package ‘bamdit’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bamdit/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘bamdit’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bamdit/old/bamdit.Rcheck/bamdit’

```
# bang

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# base2grob

Version: 0.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# basecallQC

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BASiCS

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BatchGetSymbols

Version: 2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BatchMap

Version: 1.0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BAwiR

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bayesAB

Version: 1.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    [1] 0.04541
    
    --------------------------------------------
    
    Credible Interval on (A - B) / B for interval length(s) (0.9) : 
    
    $Probability
              5%          95% 
    -0.355943366 -0.006198834 
    
    --------------------------------------------
    
    Posterior Expected Loss for choosing B over A:
    
    $Probability
    [1] 0.2601664
    
    > plot(AB1)
    Error in geom_line() : could not find function "geom_line"
    Calls: plot ... do.call -> <Anonymous> -> plotDist_ -> <Anonymous> -> do.call
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
         -0.220855266742408, -0.131236325741687, 0.0403874116005306, 0.19461909788326, 0.0911183706022301, 
         0.322930395783452, 0.0474962113954877, 0.151881500325075, -0.162339934678337, 0.162953354526967, 
         0.0847354059225813, 0.168433185697805, -0.0156309707702392, 0.0145903259071101, 0.24784112614451, 
         0.0739139038546604, -0.115386241435621), name = "Mu", percentLift = 0)
      14: ggplot2::qplot(diff, data = diff, fill = under, binwidth = diff(range(inner))/250, 
             na.rm = TRUE)
      15: do.call(paste0("geom_", g), params, envir = caller_env)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 108 SKIPPED: 0 FAILED: 2
      1. Error: Success (@test-dists.R#23) 
      2. Error: Success (@test-generics.R#37) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 51-53 (introduction.Rmd) 
    Error: processing vignette 'introduction.Rmd' failed with diagnostics:
    could not find function "geom_line"
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bayesammi

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bayesbio

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bayesDP

Version: 1.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# BayesFM

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bayesplot

Version: 1.5.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      1/1 mismatches
      x[1]: "~rep_label"
      y[1]: "rep_label"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 937 SKIPPED: 18 FAILED: 6
      1. Failure: mcmc_intervals/areas with rhat (@test-mcmc-intervals.R#89) 
      2. Failure: mcmc_intervals/areas with rhat (@test-mcmc-intervals.R#98) 
      3. Failure: mcmc_intervals/areas with rhat (@test-mcmc-intervals.R#99) 
      4. Failure: mcmc_intervals/areas with rhat (@test-mcmc-intervals.R#105) 
      5. Failure: mcmc_intervals/areas with rhat (@test-mcmc-intervals.R#109) 
      6. Failure: ppc_dens,pp_hist,ppc_freqpoly,ppc_boxplot return ggplot objects (@test-ppc-distributions.R#33) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        R     1.8Mb
        doc   3.6Mb
    ```

# BayesRS

Version: 0.1.3

## In both

*   checking whether package ‘BayesRS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/new/BayesRS.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BayesRS’ ...
** package ‘BayesRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BayesRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/new/BayesRS.Rcheck/BayesRS’

```
### CRAN

```
* installing *source* package ‘BayesRS’ ...
** package ‘BayesRS’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BayesRS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BayesRS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BayesRS/old/BayesRS.Rcheck/BayesRS’

```
# baystability

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggfortify’ ‘ggplot2’ ‘matrixStats’ ‘reshape2’ ‘scales’
      All declared Imports should be used.
    ```

# BBEST

Version: 0.1-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bbmle

Version: 1.0.20

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(1.69696793084232,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.289372177019226,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(1.90983961894499,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.253207428692937,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(2.24718525432755,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.219940849413471,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(2.76406402172104,  :
      NaNs produced
    Warning in dbinom(x = c(0, 2, 1, 3, 4, 5, 0, 0, 0, 0, 1, 0, 0, 0, 0), prob = c(0.191138285903703,  :
      NaNs produced
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mle2.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BCEA

Version: 2.2-5

## In both

*   checking examples ... ERROR
    ```
    ...
    > ###   information
    > 
    > ### ** Examples
    > 
    > # See Baio G., Dawid A.P. (2011) for a detailed description of the 
    > # Bayesian model and economic problem
    > #
    > # Load the processed results of the MCMC simulation model
    > data(Vaccine)
    Warning: namespace ‘rjags’ is not available and has been replaced
    by .GlobalEnv when processing object ‘vaccine’
    > # 
    > # Runs the health economic evaluation using BCEA
    > m <- bcea(e,c,ref=2,interventions=treats)
    > #
    > # Computes the EVPPI for a bunch of parameters
    > inp <- CreateInputs(vaccine)
    Error in CreateInputs(vaccine) : 
      You need to install the package 'R2jags'. Please run in your R terminal:
     install.packages('R2jags')
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

# BCellMA

Version: 0.3.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Bclim

Version: 3.1.2

## In both

*   R CMD check timed out
    

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# bcmaps

Version: 0.17.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    Package suggested but not available for checking: ‘bcmaps.rdata’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# bcp

Version: 4.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bcrm

Version: 0.4.7

## In both

*   checking examples ... ERROR
    ```
    ...
    > ##D plot(Power.LN.bcrm.compare.sim,threep3=TRUE)
    > ## End(Not run)
    > 
    > ## A 2-parameter model, using priors as specified in Neuenschwander et al 2008.
    > ## Posterior mean used to choose the next dose
    > ## Standardised doses using reference dose, 250mg
    > sdose<-log(dose/250)
    > ## Bivariate lognormal prior for two parameters
    > mu<-c(2.15,0.52)
    > Sigma<-rbind(c(0.84^2,0.134),c(0.134,0.80^2))
    > ## Using rjags (requires JAGS to be installed)
    > TwoPLogistic.mean.bcrm<-bcrm(stop=list(nmax=18),data=data,sdose=sdose
    +   ,dose=dose,ff="logit2",prior.alpha=list(4,mu,Sigma),target.tox=target.tox
    +   ,constrain=FALSE,pointest="mean",method="rjags")
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bcrm/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BRugs’
    ```

# bde

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    rsample,BoundedDensity: no visible global function definition for
      ‘runif’
    setNormalizationConst,MacroBetaChen99Kernel: no visible global function
      definition for ‘integrate’
    setNormalizationConst,MacroBetaHirukawaTSKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstant,MacroBetaHirukawaJLNKernel: no visible global
      function definition for ‘integrate’
    setNormalizationConstants,MicroBetaChen99Kernel :
      density.kernelFunction: no visible global function definition for
      ‘dbeta’
    setNormalizationConstants,MicroBetaChen99Kernel : <anonymous>: no
      visible global function definition for ‘integrate’
    show,BoundedDensity: no visible global function definition for ‘str’
    show,KernelDensity: no visible global function definition for ‘str’
    Undefined global functions or variables:
      dbeta integrate runif str
    Consider adding
      importFrom("stats", "dbeta", "integrate", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# bdscale

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bdvis

Version: 0.2.20

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bea.R

Version: 1.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘chron’ ‘colorspace’ ‘gtable’ ‘htmltools’ ‘htmlwidgets’
      ‘httpuv’ ‘magrittr’ ‘munsell’ ‘plyr’ ‘scales’ ‘stringi’ ‘xtable’
      ‘yaml’
      All declared Imports should be used.
    ```

# beadarray

Version: 2.30.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        colnames, dirname, do.call, duplicated, eval, evalq, get,
        grep, grepl, intersect, is.unsorted, lapply, lengths,
        mapply, match, mget, order, paste, pmax, pmax.int, pmin,
        pmin.int, rank, rbind, rowMeans, rowSums, rownames,
        sapply, setdiff, sort, table, tapply, union, unique,
        unsplit, which, which.max, which.min
    
    Loading required package: Biobase
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages
        'citation("pkgname")'.
    
    Loading required package: ggplot2
    Welcome to beadarray version 2.30.0
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ImageProcessing.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘beadarrayExampleData’
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘Nozzle.R1’ ‘affy’ ‘ggbio’ ‘hwriter’ ‘lumi’ ‘vsn’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported objects imported by ':::' calls:
      ‘BeadDataPackR:::combineFiles’ ‘BeadDataPackR:::readHeader’
      ‘Biobase:::assayDataStorageMode’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘illuminaForeground_6x6’ ‘locsIndicesToGrid’ ‘obtainLocs’
      ‘simpleXMLparse’
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("roundLocsFileValues", ..., PACKAGE = "BeadDataPackR")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      geom_hline geom_point geom_tile geom_vline ggplot ggsave ggtitle
      hwrite hwriteImage is jpeg loess lowess lumiHumanIDMapping_dbconn
      lumiMouseIDMapping_dbconn lumiRatIDMapping_dbconn lumiT menu
      metaTemplate model.matrix new newCustomReport newFigure newParagraph
      newSection newTable normalize.invariantset normalize.qspline openPage
      opts p.adjust packageDescription pdf platformSigs plotIdeogram png
      predict qplot rainbow read.csv read.table rgb rsn runif
      scale_fill_discrete scale_fill_gradient setTxtProgressBar stat_binhex
      theme theme_blank theme_bw tracks txtProgressBar value value.1 vsn2
      write.csv write.table writeReport xlab ylab
    Consider adding
      importFrom("grDevices", "col2rgb", "dev.off", "jpeg", "pdf", "png",
                 "rainbow", "rgb")
      importFrom("methods", "as", "callNextMethod", "is", "new")
      importFrom("stats", "aggregate", "approx", "density", "loess",
                 "lowess", "model.matrix", "p.adjust", "predict", "runif")
      importFrom("utils", "data", "menu", "packageDescription", "read.csv",
                 "read.table", "setTxtProgressBar", "txtProgressBar",
                 "write.csv", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# beadplexr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BEARscc

Version: 1.0.0

## In both

*   R CMD check timed out
    

# belg

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# benchmark

Version: 0.3-6

## Newly broken

*   checking whether package ‘benchmark’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::sym’ by ‘relations::sym’ when loading ‘benchmark’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/benchmark/new/benchmark.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘multcomp’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘algorithms’
    stripchart.AlgorithmPerformance: no visible binding for global variable
      ‘value’
    stripchart.AlgorithmPerformance: no visible binding for global variable
      ‘samples’
    Undefined global functions or variables:
      abline addEdge agopen algorithms as.formula axis barplot box cancor
      characteristics col2rgb getDefaultAttrs gray is layout legend lines
      lm median mi.plugin mtext na.omit new par points predict rect
      relation_is_strict_weak_order reshape rgb samples segments terms
      theme_text value
    Consider adding
      importFrom("grDevices", "col2rgb", "gray", "rgb")
      importFrom("graphics", "abline", "axis", "barplot", "box", "layout",
                 "legend", "lines", "mtext", "par", "points", "rect",
                 "segments")
      importFrom("methods", "is", "new")
      importFrom("stats", "as.formula", "cancor", "lm", "median", "na.omit",
                 "predict", "reshape", "terms")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# benchmarkme

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# benchr

Version: 0.2.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 181 SKIPPED: 0 FAILED: 10
      1.  Failure: Boxplot ggplot without log (@test-plot.R#39) 
      2.  Failure: Boxplot ggplot without log (@test-plot.R#40) 
      3.  Failure: Boxplot ggplot with log and horizontal (@test-plot.R#53) 
      4.  Failure: Boxplot ggplot with log and horizontal (@test-plot.R#54) 
      5.  Failure: Boxplot ggplot with violin geom (@test-plot.R#67) 
      6.  Failure: Boxplot ggplot with violin geom (@test-plot.R#68) 
      7.  Failure: Scatter plot ggplot with log (@test-plot.R#83) 
      8.  Failure: Scatter plot ggplot with log (@test-plot.R#84) 
      9.  Failure: Scatter plot ggplot without log (@test-plot.R#99) 
      10. Failure: Scatter plot ggplot without log (@test-plot.R#100) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# benthos

Version: 1.3-5

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:dplyr':
    
        vars
    
    Quitting from lines 476-485 (benthos.Rmd) 
    Error: processing vignette 'benthos.Rmd' failed with diagnostics:
    Free scales are only supported with `coord_cartesian()` and `coord_flip()`
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BETS

Version: 0.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bibliometrix

Version: 1.9.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bife

Version: 0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bigKRLS

Version: 3.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BIGL

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# billboarder

Version: 0.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +   bb_y_grid(show = TRUE)
    > 
    > 
    > 
    > # Hack stacked barcharts (to color bar)
    > stars_wide <- data.frame(
    +   author = c("dreamRs", "davidgohel", "davidgohel", "dreamRs",
    +              "datastorm-open", "datastorm-open", "AntoineGuillot2"),
    +   package = c("billboarder", "ggiraph", "officer",
    +               "shinyWidgets", "visNetwork", "rAmCharts", 
    +               "D3partitionR"),
    +   stars = c(36, 194, 72, 61, 183, 25, 18)
    + )
    > 
    > billboarder() %>%
    +   bb_barchart(data = stars_wide, 
    +               mapping = bbaes(package, stars, group = author),
    +               stacked = TRUE)
    Error in unique.default(x_un) : unique() applies only to vectors
    Calls: %>% ... bb_barchart -> bbmapping -> unique -> unique.default
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 66-71 (billboarder-intro.Rmd) 
    Error: processing vignette 'billboarder-intro.Rmd' failed with diagnostics:
    unique() applies only to vectors
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BinarybalancedCut

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# binom

Version: 1.1-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘lattice’ ‘polynom’ ‘tcltk’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘tkdestroy’
    var.probit: no visible global function definition for ‘qnorm’
    var.probit: no visible global function definition for ‘dnorm’
    Undefined global functions or variables:
      abline aes_string aggregate approx axis binomial box col.whitebg
      current.panel.limits dbeta dbinom dnorm facet_wrap find geom_polygon
      ggplot heat.colors integral legend lines llines lpoints lpolygon
      ltext optim panel.abline panel.levelplot panel.xyplot pbeta plot
      pnorm polynomial predict qbeta qnorm rbinom reorder spline tclObj
      tclVar theme_bw tkbutton tkcheckbutton tkdestroy tkframe tklabel
      tkpack tkradiobutton tkscale tktoplevel tkwm.title trellis.par.get
      trellis.par.set uniroot xlab xlim xyplot ylab ylim
    Consider adding
      importFrom("grDevices", "heat.colors")
      importFrom("graphics", "abline", "axis", "box", "legend", "lines",
                 "plot")
      importFrom("stats", "aggregate", "approx", "binomial", "dbeta",
                 "dbinom", "dnorm", "optim", "pbeta", "pnorm", "predict",
                 "qbeta", "qnorm", "rbinom", "reorder", "spline", "uniroot")
      importFrom("utils", "find")
    to your NAMESPACE file.
    ```

# bioassayR

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# biobroom

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘DESeq2’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Missing or unexported object: ‘dplyr::tbl_dt’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for ‘colData’
    tidy.deSet: no visible global function definition for ‘exprs<-’
    tidy.deSet: no visible binding for global variable ‘value’
    tidy.deSet: no visible binding for global variable ‘gene’
    tidy.deSet: no visible global function definition for ‘pData’
    tidy.qvalue: no visible binding for global variable ‘smoothed’
    tidy.qvalue: no visible binding for global variable ‘pi0’
    tidy.qvalue: no visible binding for global variable ‘lambda’
    tidy_matrix: no visible binding for global variable ‘value’
    tidy_matrix: no visible binding for global variable ‘gene’
    Undefined global functions or variables:
      . DGEList calcNormFactors colData counts design end estimate
      estimateSizeFactors exprs<- fData<- gene gr is lambda model.matrix
      p.adjust pData pData<- pi0 protein rowRanges sample.id seqnames
      setNames smoothed start tbl_dt term value voom voomWithQualityWeights
    Consider adding
      importFrom("methods", "is")
      importFrom("stats", "end", "model.matrix", "p.adjust", "setNames",
                 "start")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# BiocOncoTK

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data   4.0Mb
        doc    1.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘seqlengths’
    checkCache_patel: no visible global function definition for
      ‘BiocFileCache’
    chrbounds_basic: no visible global function definition for ‘seqlengths’
    rainfall: no visible global function definition for ‘genome’
    rainfall: no visible global function definition for ‘ggplot’
    rainfall: no visible global function definition for ‘aes’
    rainfall: no visible global function definition for ‘geom_point’
    rainfall: no visible global function definition for ‘theme’
    rainfall: no visible global function definition for ‘element_blank’
    rainfall: no visible global function definition for ‘ylab’
    rainfall: no visible global function definition for ‘xlab’
    rainfall: no visible global function definition for ‘geom_vline’
    rainfall: no visible global function definition for
      ‘scale_x_continuous’
    rainfall: no visible global function definition for ‘ggtitle’
    rainfall: no visible global function definition for ‘geom_text’
    Undefined global functions or variables:
      BiocFileCache aes element_blank genome geom_point geom_text
      geom_vline ggplot ggtitle scale_x_continuous seqlengths theme xlab
      ylab
    ```

# biogas

Version: 1.10.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'biogas_quick_start.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# biogram

Version: 1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Biograph

Version: 2.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Bioi

Version: 0.2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bioinactivation

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# biomod2

Version: 3.3-7

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    	> Projecting GuloGulo_AllData_RUN1_RF ...
    	> Projecting GuloGulo_AllData_RUN2_RF ...
    -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-= Done -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
    >                           
    > 
    > # 4. Creating the ensemble projections
    > BIOMOD_EnsembleForecasting( projection.output = myBiomodProjection,
    +                             EM.output = myBiomodEM)
    
    -=-=-=-=-=-=-=-=-=-=-=-= Do Ensemble Models Projections -=-=-=-=-=-=-=-=-=-=-=-=
    
    
    	> Projecting GuloGulo_EMmeanByTSS_RF_mergedRun_mergedData ...Error in calc(formal_predictions, function(x) { : 
      unused arguments (function(x) {
        m <- mean(x)
        if (on_0_1000) m <- round(m)
        return(m)
    }, filename = filename, overwrite = TRUE)
    Calls: BIOMOD_EnsembleForecasting ... .local -> .predict.EMmean_biomod2_model.RasterStack
    Execution halted
    ```

*   checking whether package ‘biomod2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'calc(data, function(x) {': unused argument (function(x) {
      Note: possible error in 'calc(BinaryTransformation(formal_predictions, ': unused arguments (function(x) {
      Note: possible error in 'calc(formal_predictions, ': unused argument (mean) 
      Note: possible error in 'calc(formal_predictions, ': unused argument (sd) 
      Note: possible error in 'calc(formal_predictions, ': unused arguments (cv, filename = filename, overwrite = TRUE, na.rm = TRUE, aszero = TRUE) 
      Note: possible error in 'calc(formal_predictions, ': unused arguments (function(x) {
      Note: possible error in 'calc(newdata, function(x) {': unused argument (function(x) {
      Warning: replacing previous import ‘raster::calc’ by ‘ggplot2::calc’ when loading ‘biomod2’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/biomod2/new/biomod2.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      as.numeric(predict.maxent(get_formal_model(object), x.no.na)[, "1"])
      proj.out[-attr(x.no.na, "na.action")] <- proj.not.na }
      return(proj.out) })
    .predict.MAXENT.Tsuruoka_biomod2_model.RasterStack: possible error in
      return(proj.out): unused argument (function(x) { proj.out <- rep(NA,
      nrow(x)) x.no.na <- na.omit(x) if (nrow(x.no.na)) { proj.not.na <-
      as.numeric(predict.maxent(get_formal_model(object), x.no.na)[, "1"])
      proj.out[-attr(x.no.na, "na.action")] <- proj.not.na }
      return(proj.out) })
    .predict.MAXENT.Tsuruoka_biomod2_model.RasterStack: possible error in
      }): unused argument (function(x) { proj.out <- rep(NA, nrow(x))
      x.no.na <- na.omit(x) if (nrow(x.no.na)) { proj.not.na <-
      as.numeric(predict.maxent(get_formal_model(object), x.no.na)[, "1"])
      proj.out[-attr(x.no.na, "na.action")] <- proj.not.na }
      return(proj.out) })
    BinaryTransformation,RasterStack: possible error in calc(data,
      function(x) {: unused argument (function(x) { x >= threshold })
    BinaryTransformation,RasterStack: possible error in x >= threshold:
      unused argument (function(x) { x >= threshold })
    BinaryTransformation,RasterStack: possible error in }): unused argument
      (function(x) { x >= threshold })
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ecospat’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    BugReports field is not a suitable URL but contains an email address
      which will be used as from R 3.4.0
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported object: ‘gam::step.gam’
    ```

# BioNetStat

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        extdata   1.9Mb
        shiny     1.6Mb
    ```

# bioOED

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BioPET

Version: 0.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘BioPET-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_enrichment_summaries
    > ### Title: Plotting Prognostic Enrichment Estimates
    > ### Aliases: plot_enrichment_summaries
    > 
    > ### ** Examples
    > 
    > 
    > data(dcaData)
    > # one marker
    > analysis.single.marker <- enrichment_analysis(Cancer ~ Marker1,
    + data=dcaData,
    + reduction.under.treatment=0.3,
    + cost.screening=100, cost.keeping=1000)
    > plot_enrichment_summaries(analysis.single.marker)
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bioplots

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

# bioset

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BIS

Version: 0.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        intersect, setdiff, setequal, union
    
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:dplyr':
    
        vars
    
    
    Attaching package: 'zoo'
    
    The following objects are masked from 'package:base':
    
        as.Date, as.Date.numeric
    
    Quitting from lines 61-79 (BIS.Rmd) 
    Error: processing vignette 'BIS.Rmd' failed with diagnostics:
    Evaluation error: yearmon variable can only have one format.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bisect

Version: 0.9.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BlandAltmanLeh

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# blandr

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘knitr’
      All declared Imports should be used.
    ```

# blastula

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘purrr’ ‘rlang’ ‘tibble’
      All declared Imports should be used.
    ```

# blkbox

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘bigrf’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘glmnet’ ‘gtools’ ‘knitr’ ‘nnet’ ‘parallel’ ‘rJava’ ‘reshape’
      ‘rmarkdown’ ‘shinyjs’
      All declared Imports should be used.
    Missing or unexported object: ‘xgboost::predict’
    ```

# blockseg

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BloodCancerMultiOmics2017

Version: 1.0.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:purrr':
    
        discard
    
    The following object is masked from 'package:readr':
    
        col_factor
    
    
    Attaching package: 'xtable'
    
    The following object is masked from 'package:ggdendro':
    
        label
    
    Quitting from lines 61-62 (./src/part01.Rmd) 
    Quitting from lines 18-19 (./src/part01.Rmd) 
    Error: processing vignette 'BloodCancerMultiOmics2017.Rmd' failed with diagnostics:
    "plot.tag" is not a valid theme element name.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 115.1Mb
      sub-directories of 1Mb or more:
        data     80.0Mb
        doc      26.2Mb
        extdata   8.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘vsn’
    ```

# blorr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘descriptr’
      All declared Imports should be used.
    ```

# blscrapeR

Version: 3.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bmlm

Version: 1.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# bmmix

Version: 0.1-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    bmmix : LL.y: no visible global function definition for ‘dmultinom’
    bmmix : LL.x : <anonymous>: no visible global function definition for
      ‘dmultinom’
    bmmix : LPrior.alpha: no visible global function definition for ‘dexp’
    bmmix : alpha.move: no visible global function definition for ‘rnorm’
    bmmix : alpha.move: no visible global function definition for ‘runif’
    bmmix : phi.move: no visible global function definition for ‘rnorm’
    bmmix : phi.move: no visible global function definition for ‘runif’
    bmmix: no visible global function definition for ‘read.table’
    Undefined global functions or variables:
      dexp dmultinom read.table rnorm runif
    Consider adding
      importFrom("stats", "dexp", "dmultinom", "rnorm", "runif")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# BNSP

Version: 2.0.2

## In both

*   checking whether package ‘BNSP’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/new/BNSP.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BNSP’ ...
** package ‘BNSP’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c BNSP_init.c -o BNSP_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c OneResLtnt.c -o OneResLtnt.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c mvrmGAM.c -o mvrmGAM.o
OneResLtnt.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
mvrmGAM.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [mvrmGAM.o] Error 1
make: *** Waiting for unfinished jobs....
1 error generated.
make: *** [OneResLtnt.o] Error 1
ERROR: compilation failed for package ‘BNSP’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/new/BNSP.Rcheck/BNSP’

```
### CRAN

```
* installing *source* package ‘BNSP’ ...
** package ‘BNSP’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c BNSP_init.c -o BNSP_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c OneResLtnt.c -o OneResLtnt.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c mvrmGAM.c -o mvrmGAM.o
mvrmGAM.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
#include <gsl/gsl_matrix.h>
OneResLtnt.c:26:10: fatal error: 'gsl/gsl_matrix.h' file not found
         ^~~~~~~~~~~~~~~~~~
#include <gsl/gsl_matrix.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [mvrmGAM.o] Error 1
make: *** Waiting for unfinished jobs....
1 error generated.
make: *** [OneResLtnt.o] Error 1
ERROR: compilation failed for package ‘BNSP’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BNSP/old/BNSP.Rcheck/BNSP’

```
# bodenmiller

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        data   8.7Mb
    ```

# bomrang

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bootnet

Version: 1.0.1

## Newly broken

*   checking whether package ‘bootnet’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘bootnet’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bootnet/new/bootnet.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bossMaps

Version: 0.1.0

## Newly broken

*   checking whether package ‘bossMaps’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘bossMaps’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bossMaps/new/bossMaps.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rgdal’ ‘tidyr’
      All declared Imports should be used.
    ```

# BPEC

Version: 1.2.1

## In both

*   checking whether package ‘BPEC’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BPEC/new/BPEC.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘BPEC’ ...
** package ‘BPEC’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c bpecfunction.c -o bpecfunction.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c hashingfunctions.c -o hashingfunctions.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c loopfunctions.c -o loopfunctions.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c matrices.c -o matrices.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c randomnumbers.c -o randomnumbers.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c registerDynamicSymbol.c -o registerDynamicSymbol.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c treelikelifunctions.c -o treelikelifunctions.o
/usr/local/clang4/bin/clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o BPEC.so bpecfunction.o hashingfunctions.o loopfunctions.o matrices.o randomnumbers.o registerDynamicSymbol.o treelikelifunctions.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/BPEC/new/BPEC.Rcheck/BPEC/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BPEC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BPEC/new/BPEC.Rcheck/BPEC’

```
### CRAN

```
* installing *source* package ‘BPEC’ ...
** package ‘BPEC’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c bpecfunction.c -o bpecfunction.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c hashingfunctions.c -o hashingfunctions.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c loopfunctions.c -o loopfunctions.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c matrices.c -o matrices.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c randomnumbers.c -o randomnumbers.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c registerDynamicSymbol.c -o registerDynamicSymbol.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c treelikelifunctions.c -o treelikelifunctions.o
/usr/local/clang4/bin/clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o BPEC.so bpecfunction.o hashingfunctions.o loopfunctions.o matrices.o randomnumbers.o registerDynamicSymbol.o treelikelifunctions.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/BPEC/old/BPEC.Rcheck/BPEC/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BPEC/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BPEC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BPEC/old/BPEC.Rcheck/BPEC’

```
# BPRMeth

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   1.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiocStyle’
      All declared Imports should be used.
    ```

# BradleyTerryScalable

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# braidReports

Version: 0.5.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BrailleR

Version: 0.26.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘installr’
    ```

# brainGraph

Version: 2.0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘RGtk2’ ‘cairoDevice’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# branchpointer

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: GenomicRanges
    Loading required package: Biostrings
    Loading required package: XVector
    
    Attaching package: 'Biostrings'
    
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: rtracklayer
    Loading required package: caret
    Loading required package: lattice
    Loading required package: ggplot2
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    filtering for SNPs in branchpoint windows
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'branchpointer.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        R         4.2Mb
        extdata   1.1Mb
    ```

# brazilmaps

Version: 0.1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# breakDown

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# breakpoint

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# breathtestcore

Version: 0.4.1

## Newly broken

*   checking whether package ‘breathtestcore’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘breathtestcore’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/breathtestcore/new/breathtestcore.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘breathteststan’
    ```

# breathteststan

Version: 0.4.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +   inner_join(d$record, by = "patient_id") %>%
    +   select(patient_id, m_in = m.y, m_out = m.x,
    +          beta_in = beta.y, beta_out = beta.x,
    +          k_in = k.y, k_out = k.x)
    # A tibble: 3 x 7
      patient_id  m_in m_out beta_in beta_out    k_in   k_out
      <chr>      <dbl> <dbl>   <dbl>    <dbl>   <dbl>   <dbl>
    1 rec_01        34  36.1    2.19     1.99 0.0140  0.0123 
    2 rec_02        42  44.7    2.30     2.10 0.0108  0.00965
    3 rec_03        32  38.1    2.23     2.09 0.00795 0.00657
    > # For a detailed analysis of the fit, use the shinystan library
    > # The following plots are somewhat degenerate because
    > # of the few iterations in stan_fit
    > suppressPackageStartupMessages(library(rstan))
    > stan_plot(fit$stan_fit, pars = c("beta[1]","beta[2]","beta[3]"))
    ci_level: 0.8 (80% intervals)
    outer_level: 0.95 (95% intervals)
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > # Only one test per file to avoid hanging 32-bit compile
      > #test_check("breathteststan", filter = "stan_fit")
      > Sys.unsetenv("R_TESTS") # https://github.com/r-lib/testthat/issues/603
      > test_check("breathteststan")
      [1] "liquid_normal" "solid_normal"  "solid_patient"
      [1] "liquid_normal" "solid_normal"  "solid_patient"
      ── 1. Failure: Data that cannot be fitted with nls_list/nlme work with stan_fit 
      sigma(fit) is not strictly more than 0.9. Difference: -0.00781
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 28 SKIPPED: 7 FAILED: 1
      1. Failure: Data that cannot be fitted with nls_list/nlme work with stan_fit (@test_stan_fit_2.R#28) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BreedingSchemeLanguage

Version: 0.9.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bridgedist

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bridger2

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# brms

Version: 2.2.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             name <- expr_text(expr)
             name <- gsub("\n.*$", "...", name)
             name
         } else {
             abort("`expr` must quote a symbol, scalar, or call")
         })
      12: abort("`expr` must quote a symbol, scalar, or call")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1091 SKIPPED: 1 FAILED: 2
      1. Error: all S3 methods have reasonable ouputs (@tests.brmsfit-methods.R#276) 
      2. Error: plotting functions don't throw unexpected errors (@tests.plots.R#52) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'brms_multilevel.ltx' failed with diagnostics:
    Running 'texi2dvi' on 'brms_multilevel.ltx' failed.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        R     5.1Mb
        doc   2.4Mb
    ```

# broman

Version: 0.67-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# broom

Version: 0.4.4

## In both

*   R CMD check timed out
    

# brotli

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# brt

Version: 1.3.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘brt-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: dtavg
    > ### Title: Average of the Student t Distribution
    > ### Aliases: dtavg ptavg
    > ### Keywords: distribution
    > 
    > ### ** Examples
    > 
    > x=seq(from=-10, to=10, length.out=100)
    > ggplot2::qplot(x, ptavg(x, df=3, hi=3), geom='line')
    Error in geom_line() : could not find function "geom_line"
    Calls: <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bsam

Version: 1.1.2

## In both

*   checking whether package ‘bsam’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/new/bsam.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘bsam’ ...
** package ‘bsam’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘bsam’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/new/bsam.Rcheck/bsam’

```
### CRAN

```
* installing *source* package ‘bsam’ ...
** package ‘bsam’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/bsam/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘bsam’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/bsam/old/bsam.Rcheck/bsam’

```
# bsamGP

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BSDA

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BSL

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# bssm

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# btergm

Version: 1.9.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# BTSPAS

Version: 2014.0901

## In both

*   checking whether package ‘BTSPAS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/new/BTSPAS.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘BRugs’
    ```

## Installation

### Devel

```
* installing *source* package ‘BTSPAS’ ...
** package ‘BTSPAS’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BTSPAS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/new/BTSPAS.Rcheck/BTSPAS’

```
### CRAN

```
* installing *source* package ‘BTSPAS’ ...
** package ‘BTSPAS’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/BTSPAS/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘BTSPAS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/BTSPAS/old/BTSPAS.Rcheck/BTSPAS’

```
# BubbleTree

Version: 2.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 29.2Mb
      sub-directories of 1Mb or more:
        data  23.4Mb
        doc    5.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    annoByOverlap,Annotate: no visible binding for global variable
      'queryHits'
    Undefined global functions or variables:
      queryHits
    ```

# bulletr

Version: 0.1

## Newly broken

*   checking whether package ‘bulletr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘bulletr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/bulletr/new/bulletr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# burnr

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# C443

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CAFE

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    facetPlot: no visible global function definition for 'dev.off'
    facetPlot: no visible global function definition for 'na.omit'
    fisher.method: no visible global function definition for 'pchisq'
    fisherExact: no visible global function definition for 'fisher.test'
    makelevels: no visible global function definition for 'na.omit'
    rawPlot: no visible global function definition for 'png'
    rawPlot: no visible global function definition for 'dev.off'
    rawPlot: no visible global function definition for 'data'
    slidPlot: no visible global function definition for 'png'
    slidPlot: no visible global function definition for 'dev.off'
    slidPlot: no visible global function definition for 'data'
    Undefined global functions or variables:
      chisq.test data dev.off download.file fisher.test na.omit p.adjust
      pchisq png read.table setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("stats", "chisq.test", "fisher.test", "na.omit", "p.adjust",
                 "pchisq")
      importFrom("utils", "data", "download.file", "read.table",
                 "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        geom_bar, geom_rect, geom_segment, ggsave, stat_bin, stat_identity,
        xlim
    
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    Writing plot to Chr17Mul
    
    
    Writing plot to Chr17slidMul
    
    
    Writing plot to Chr17discMul
    
    
    Writing plot to FacetSlidMul
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CAFE-manual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# caffsim

Version: 0.2.2

## Newly broken

*   checking whether package ‘caffsim’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘caffsim’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/caffsim/new/caffsim.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘markdown’
      All declared Imports should be used.
    ```

# CAGEr

Version: 1.22.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   3.1Mb
    ```

# CAISEr

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CALF

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CaliCo

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.5Mb
      sub-directories of 1Mb or more:
        doc   5.7Mb
    ```

# cancensus

Version: 0.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# CAnD

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    CAnD : pairedTtest: no visible global function definition for ‘t.test’
    barPlotAncest: no visible binding for global variable ‘id’
    barPlotAncest: no visible binding for global variable ‘value’
    barPlotAncest: no visible binding for global variable ‘variable’
    calc_combP: no visible binding for global variable ‘var’
    calc_combP: no visible global function definition for ‘pchisq’
    plotPvals: no visible binding for global variable ‘numPs’
    plotPvals: no visible binding for global variable ‘pval’
    Undefined global functions or variables:
      id numPs pchisq pval t.test value var variable
    Consider adding
      importFrom("stats", "pchisq", "t.test", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CAnD.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# capm

Version: 0.11.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# caret

Version: 6.0-79

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        R        2.0Mb
        data     1.5Mb
        models   2.4Mb
    ```

# caretEnsemble

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# casebase

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CAST

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CATALYST

Version: 1.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotDiffHeatmap,matrix-SummarizedExperiment: no visible binding for
      global variable ‘cluster_id’
    plotDiffHeatmap,matrix-SummarizedExperiment: no visible binding for
      global variable ‘sample_id’
    Undefined global functions or variables:
      cluster_id sample_id
    ```

# cate

Version: 1.0.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cate-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# catenary

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyverse’
      All declared Imports should be used.
    ```

# CausalImpact

Version: 1.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ConstructModel: warning in AddDynamicRegression(ss, formula, data =
      data, sigma.mean.prior = sigma.mean.prior): partial argument match of
      'sigma.mean.prior' to 'sigma.mean.prior.DEPRECATED'
    ```

# Causata

Version: 4.2-0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Loading required package: Matrix
    Loading required package: foreach
    Loaded glmnet 2.0-16
    
    Type 'citation("pROC")' for a citation.
    
    Attaching package: 'pROC'
    
    The following object is masked from 'package:glmnet':
    
        auc
    
    The following objects are masked from 'package:stats':
    
        cov, smooth, var
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Causata-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    PredictivePowerCv: no visible global function definition for ‘sd’
    ReadCausataCsv: no visible global function definition for ‘read.csv’
    ToPmml.GlmnetModelDefinition: no visible global function definition for
      ‘coef’
    ToPmml.GlmnetModelDefinition: no visible global function definition for
      ‘terms.formula’
    ValidateModel: no visible global function definition for ‘predict’
    predict.GlmnetModelDefinition: no visible global function definition
      for ‘model.matrix’
    predict.GlmnetModelDefinition: no visible binding for global variable
      ‘contrasts’
    predict.GlmnetModelDefinition: no visible global function definition
      for ‘predict’
    Undefined global functions or variables:
      coef contrasts dbGetQuery median model.matrix na.omit predict
      quantile read.csv sd terms.formula
    Consider adding
      importFrom("stats", "coef", "contrasts", "median", "model.matrix",
                 "na.omit", "predict", "quantile", "sd", "terms.formula")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# cbar

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ccfa

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cdom

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cellHTS2

Version: 2.44.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Unknown package ‘cellHTS’ in Rd xrefs
    ```

*   checking running R code from vignettes ...
    ```
    ...
    > pushViewport(viewport(layout = grid.layout(3, 2)))
    
    > dv <- Data(xn)[, , 1]
    
    > myPlots(dv, main = "untransformed", plotCol = 1)
    
      When sourcing ‘cellhts2.R’:
    Error: could not find function "geom_qq"
    Execution halted
    when running code in ‘cellhts2Complete.Rnw’
      ...
    +     nrow(res)), la .... [TRUNCATED] 
    
    > library(KEGG.db)
    
      When sourcing ‘cellhts2Complete.R’:
    Error: there is no package called ‘KEGG.db’
    recover called non-interactively; frames dumped, use debugger() to view
    > proc.time()
       user  system elapsed 
     30.851   1.842  33.961 
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘RColorBrewer’ ‘Biobase’ ‘genefilter’ ‘splots’ ‘vsn’ ‘hwriter’
      ‘locfit’ ‘grid’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        KcViab   2.0Mb
        R        2.0Mb
        doc      1.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘genefilter’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘Biobase:::.showAnnotatedDataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘GO.db’ ‘KEGG.db’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages
        'citation("pkgname")'.
    
    Loading required package: genefilter
    Loading required package: splots
    Loading required package: vsn
    Loading required package: hwriter
    Loading required package: locfit
    locfit 1.5-9.1 	 2013-03-22
    Loading required package: grid
    Warning: The R script which produced this cellHTS2 report has not been provided via the 'mainScriptFile' argument.
    We recommend storing this script for future reference along with the report.
    Warning: The R script which produced this cellHTS2 report has not been provided via the 'mainScriptFile' argument.
    We recommend storing this script for future reference along with the report.
    
    Error: processing vignette 'cellhts2.Rnw' failed with diagnostics:
     chunk 36 (label = transfplots) 
    Error in geom_qq() : could not find function "geom_qq"
    Execution halted
    ```

# cellity

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CellNOptR

Version: 1.26.0

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
      Warning: working directory was changed to ‘/private/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T/RtmpEZh50C’, resetting
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘RBGL’ ‘graph’ ‘hash’ ‘RCurl’ ‘Rgraphviz’ ‘XML’ ‘ggplot2’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘Fig1.png’, ‘Fig5.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: RCurl
    Loading required package: bitops
    Loading required package: Rgraphviz
    Loading required package: grid
    Loading required package: XML
    
    Attaching package: ‘XML’
    
    The following object is masked from ‘package:graph’:
    
        addNode
    
    The following object is masked from ‘package:tools’:
    
        toHTML
    
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CellNOptR-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cellWise

Version: 2.0.8

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: perry
    Loading required package: parallel
    Loading required package: robustbase
    Quitting from lines 125-209 (DDC_examples.Rmd) 
    Error: processing vignette 'DDC_examples.Rmd' failed with diagnostics:
    unused argument (range = limits)
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CEMiTool

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.9Mb
      sub-directories of 1Mb or more:
        data      3.1Mb
        doc       2.3Mb
        extdata   2.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_ora,CEMiTool : <anonymous>: no visible global function definition
      for ‘head’
    plot_qq,CEMiTool: no visible binding for global variable ‘data’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘hclust’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dist’
    plot_sample_tree,CEMiTool: no visible global function definition for
      ‘dev.off’
    save_plots,CEMiTool : <anonymous>: no visible global function
      definition for ‘dev.off’
    save_plots,CEMiTool: no visible global function definition for
      ‘dev.off’
    Undefined global functions or variables:
      ..eq.label.. ..rr.label.. Mean Variance as.dist cem data dev.off dist
      hclust head tail var
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "as.dist", "dist", "hclust", "var")
      importFrom("utils", "data", "head", "tail")
    to your NAMESPACE file.
    ```

# census

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Census2016

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

# CGPfunctions

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘BSDA’, ‘janitor’
    ```

# ChainLadder

Version: 0.2.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: car
    Loading required package: carData
    Loading required package: lmtest
    Loading required package: zoo
    
    Attaching package: ‘zoo’
    
    The following objects are masked from ‘package:base’:
    
        as.Date, as.Date.numeric
    
    
    Please cite the 'systemfit' package as:
    Arne Henningsen and Jeff D. Hamann (2007). systemfit: A Package for Estimating Systems of Simultaneous Equations in R. Journal of Statistical Software 23(4), 1-40. http://www.jstatsoft.org/v23/i04/.
    
    If you have questions, suggestions, or comments regarding the 'systemfit' package, please use a forum or 'tracker' at systemfit's R-Forge site:
    https://r-forge.r-project.org/projects/systemfit/
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ChainLadder.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ChannelAttributionApp

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

# ChaosGame

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# checkmate

Version: 1.8.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# childsds

Version: 0.6.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gamlss.dist’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 20 marked UTF-8 strings
    ```

# ChIPexoQual

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ChIPQC

Version: 1.16.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ChIPQC-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ChIPQCreport-methods
    > ### Title: Generate a summary QC report
    > ### Aliases: ChIPQCreport-methods ChIPQCreport
    > ###   ChIPQCreport,ChIPQCexperiment-method ChIPQCreport,ChIPQCsample-method
    > ###   ChIPQCreport,list-method
    > ### Keywords: methods
    > 
    > ### ** Examples
    > 
    > data(example_QCexperiment)
    > ChIPQCreport(exampleExp,facetBy=c("Tissue","Factor"))
    Saving 7 x 7 in image
    Error: `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::tabulate2’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("rle_sum_any", ..., PACKAGE = "chipseq")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    makeRegiPlot: no visible binding for global variable ‘Sample’
    makeRegiPlot: no visible binding for global variable ‘GenomicIntervals’
    makeRegiPlot: no visible binding for global variable ‘log2_Enrichment’
    makeSSDPlot: no visible binding for global variable ‘Sample’
    makeSSDPlot: no visible binding for global variable ‘SSD’
    makeSSDPlot: no visible global function definition for ‘geom_point’
    sampleQC: no visible global function definition for ‘seqlevels<-’
    plotCC,ChIPQCexperiment: no visible binding for global variable
      ‘Sample’
    plotCC,list: no visible binding for global variable ‘Sample’
    plotPeakProfile,ChIPQCexperiment: no visible binding for global
      variable ‘Sample’
    plotPeakProfile,list: no visible binding for global variable ‘Sample’
    Undefined global functions or variables:
      CC_Score CountsInPeaks Depth Distance FRIBL FRIP GenomicIntervals
      Reads SSD Sample Shift_Size Signal TxDb.Hsapiens.UCSC.hg38.knownGene
      geom_point log10_bp log2_Enrichment seqlengths seqlengths<-
      seqlevels<-
    Consider adding
      importFrom("stats", "SSD")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'ChIPQC'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        anyMissing, rowMedians
    
    Loading required package: BiocParallel
    
    Attaching package: ‘DelayedArray’
    
    The following objects are masked from ‘package:matrixStats’:
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following objects are masked from ‘package:base’:
    
        aperm, apply
    
    
    
    Error: processing vignette 'ChIPQC.Rnw' failed with diagnostics:
     chunk 12 (label = figCoverageHist) 
    Error : `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

# ChIPSeqSpike

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    ...
    Processing H3K79me2_0
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    Processing H3K79me2_50
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    Processing H3K79me2_100
    	 Reading experiment bigWig file.
    	 Apply scaling factor
    	 Output bigWig file
    
    
    		 ### Step 6. Extract values ###
    Retrieving values for profiles and heatmaps.
    Extraction at pf
    Error in getPlotSetArray(tracks = files, features = gff_vec, refgenome = genome_version,  : 
      No genomes installed!
    Calls: spikePipe ... .retrieveAverageValuesList -> lapply -> lapply -> FUN -> getPlotSetArray
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max, which.min
    
    Loading required package: S4Vectors
    
    Attaching package: 'S4Vectors'
    
    The following object is masked from 'package:base':
    
        expand.grid
    
    Loading required package: IRanges
    Loading required package: GenomeInfoDb
    Quitting from lines 72-98 (ChIPSeqSpike.Rmd) 
    Error: processing vignette 'ChIPSeqSpike.Rmd' failed with diagnostics:
    No genomes installed!
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ChocoLattes

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘knitr’ ‘tools’
      All declared Imports should be used.
    ```

# cholera

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc   3.7Mb
    ```

# choroplethr

Version: 3.6.1

## In both

*   checking whether package ‘choroplethr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/new/choroplethr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘choroplethr’ ...
** package ‘choroplethr’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘choroplethr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/new/choroplethr.Rcheck/choroplethr’

```
### CRAN

```
* installing *source* package ‘choroplethr’ ...
** package ‘choroplethr’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘choroplethr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/choroplethr/old/choroplethr.Rcheck/choroplethr’

```
# choroplethrAdmin1

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 18.0Mb
      sub-directories of 1Mb or more:
        data  17.9Mb
    ```

# choroplethrMaps

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# chorrrds

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        data   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘httr’ ‘jsonlite’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 8008 marked UTF-8 strings
    ```

# chromstaR

Version: 1.6.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Warning in .Seqinfo.mergexy(x, y) :
      Each of the 2 combined objects has sequence levels not in the other:
      - in 'x': chrM
      - in 'y': chrMT
      Make sure to always combine/compare objects based on the same reference
      genome (use suppressWarnings() to suppress this warning).
    Warning in .Seqinfo.mergexy(x, y) :
      Each of the 2 combined objects has sequence levels not in the other:
      - in 'x': chrM
      - in 'y': chrMT
      Make sure to always combine/compare objects based on the same reference
      genome (use suppressWarnings() to suppress this warning).
    Warning in .Seqinfo.mergexy(x, y) :
      Each of the 2 combined objects has sequence levels not in the other:
      - in 'x': chrM
      - in 'y': chrMT
      Make sure to always combine/compare objects based on the same reference
      genome (use suppressWarnings() to suppress this warning).
    Error in f(...) : unused argument (range = limits)
    Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous>
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Exporting browser files
    =======================
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmplyYsMg/SHR-example/BROWSERFILES/combined_mode-combinatorial_binsize1000_stepsize500_combinations_SHR.bed.gz ... 0.11s
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmplyYsMg/SHR-example/BROWSERFILES/combined_mode-combinatorial_binsize1000_stepsize500_peaks_H3K27me3-SHR-rep1.bed.gz ... 0.04s
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmplyYsMg/SHR-example/BROWSERFILES/combined_mode-combinatorial_binsize1000_stepsize500_peaks_H3K27me3-SHR-rep2.bed.gz ... 0.04s
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmplyYsMg/SHR-example/BROWSERFILES/combined_mode-combinatorial_binsize1000_stepsize500_peaks_H3K4me3-SHR-rep1.bed.gz ... 0.03s
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmplyYsMg/SHR-example/BROWSERFILES/combined_mode-combinatorial_binsize1000_stepsize500_peaks_H3K4me3-SHR-rep2.bed.gz ... 0.03s
    ==> Total time spent: 43s <==
    Enrichment inside of annotations ... 2.15s
    Enrichment 15000bp before annotations 0.11s
    Enrichment 15000bp after annotations 0.15s
    Enrichment 10000bp before annotations 0.14s
    Enrichment 10000bp before annotations 0.11s
    Overlaps with annotation ... 0.09s
    Getting surrounding indices ... 0.04s
    Getting read counts 0.01s
    Making the plot ... 0.17s
    Quitting from lines 420-426 (chromstaR.Rnw) 
    Error: processing vignette 'chromstaR.Rnw' failed with diagnostics:
    unused argument (range = limits)
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking installed package size ... NOTE
    ```
      installed size is  7.2Mb
      sub-directories of 1Mb or more:
        data   4.5Mb
        doc    1.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    Chromstar: multiple local function definitions for ‘parallel.helper’
      with different formal arguments
    combineMultivariates: no visible global function definition for
      ‘endoapply’
    variableWidthBins: no visible global function definition for
      ‘endoapply’
    Undefined global functions or variables:
      endoapply
    ```

# chromVAR

Version: 1.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘chromVAR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: chromVAR_theme
    > ### Title: chromVAR_theme
    > ### Aliases: chromVAR_theme
    > 
    > ### ** Examples
    > 
    > p <- ggplot2::qplot(1:3,1:3) + chromVAR_theme(18)
    Error in geom_point() : could not find function "geom_point"
    Calls: <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# chron

Version: 2.3-52

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CINNA

Version: 1.1.41

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'igraph'
    
    The following objects are masked from 'package:stats':
    
        decompose, spectrum
    
    The following object is masked from 'package:base':
    
        union
    
    Loading required namespace: circlize
    Quitting from lines 315-318 (CINNA.Rmd) 
    Error: processing vignette 'CINNA.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~subgraph_cent".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘circlize’
      All declared Imports should be used.
    ```

# circglmbayes

Version: 1.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# civis

Version: 1.4.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      7: all.equal.numeric(c(m$metrics$metrics$y_yhat_plot$values), p$data[[var_name]])
      8: attr.all.equal(target, current, tolerance = tolerance, scale = scale, ...)
      9: mode(current)
      10: p$data[[var_name]]
      11: `[[.data.frame`(p$data, var_name)
      12: (function(x, i, exact) if (is.matrix(i)) as.matrix(x)[[i]] else .subset2(x, i, exact = exact))(x, 
             ..., exact = exact)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 919 SKIPPED: 0 FAILED: 2
      1. Error: decile plot for classification is produced (@test_civis_ml_plot.R#22) 
      2. Error: y_yhat plot for reg is produced (@test_civis_ml_plot.R#53) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cjoint

Version: 2.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ck37r

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > Boston = Boston[row_subset, ]
    > X = subset(Boston, select = -chas)
    > 
    > cvsl = CV.SuperLearner(Boston$chas, X[, 1:2], family = binomial(),
    +                       cvControl = list(V = 2, stratifyCV = TRUE),
    +                       SL.library = c("SL.mean", "SL.glm"))
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    Warning: glm.fit: fitted probabilities numerically 0 or 1 occurred
    > cvsl_plot_roc(cvsl)
    Error in geom_line() : could not find function "geom_line"
    Calls: cvsl_plot_roc -> print -> <Anonymous> -> do.call
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             geom = "line", main = title) + ggplot2::theme_bw() + ggplot2::annotate("text", 
             x = 0.63, y = 0.15, label = txt, size = 6) + ggplot2::annotate("segment", x = 0, 
             xend = 1, y = 0, yend = 1))
      3: ggplot2::qplot(1 - methods::slot(perf1, "x.values")[[1]], methods::slot(perf1, "y.values")[[1]], 
             xlab = "1 - Specificity (false positives)", ylab = "Sensitivity (true positives)", 
             geom = "line", main = title)
      4: do.call(paste0("geom_", g), params, envir = caller_env)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 14 SKIPPED: 0 FAILED: 2
      1. Error: (unknown) (@test-sl_plot_roc.R#13) 
      2. Error: (unknown) (@test_cvsl_plot_roc.R#16) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# classifierplots

Version: 1.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# classify

Version: 1.3

## In both

*   checking whether package ‘classify’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘classify’ ...
** package ‘classify’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c exp.cpp -o exp.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c gpcm.cpp -o gpcm.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c rcpp_w_lord.cpp -o rcpp_w_lord.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o classify.so exp.o gpcm.o rcpp_w_lord.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/classify/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘classify’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/new/classify.Rcheck/classify’

```
### CRAN

```
* installing *source* package ‘classify’ ...
** package ‘classify’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c exp.cpp -o exp.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c gpcm.cpp -o gpcm.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/classify/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c rcpp_w_lord.cpp -o rcpp_w_lord.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o classify.so exp.o gpcm.o rcpp_w_lord.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/classify/old/classify.Rcheck/classify/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/classify/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘classify’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/classify/old/classify.Rcheck/classify’

```
# ClassifyR

Version: 2.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +     # Leave-one-out cross-validation.
    +     resubstituteParams <- ResubstituteParams(nFeatures = 2:10,
    +                                          performanceType = "balanced error",
    +                                          better = "lower")
    +     classified <-
    +     runTests(measurements, classes, "Melanoma", "Different Means",
    +              validation = "leaveOut", leave = 1,
    +              params = list(SelectParams(limmaSelection, "Moderated t Statistic",
    +                                         resubstituteParams = resubstituteParams),
    +                            TrainParams(DLDAtrainInterface),
    +                            PredictParams(DLDApredictInterface,
    +                                          getClasses = function(result) result[["class"]])))
    +     class(classified)
    +   }
    Loading required package: sparsediscrim
    Warning in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) :
      port 11605 cannot be opened
    Error in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) : 
      cannot open the connection
    Calls: runTests ... .local -> .bpfork -> .bpforkConnect -> socketConnection
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following objects are masked from 'package:base':
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append, as.data.frame, basename, cbind, colMeans, colSums,
        colnames, dirname, do.call, duplicated, eval, evalq, get, grep, grepl, intersect, is.unsorted, lapply, lengths,
        mapply, match, mget, order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind, rowMeans, rowSums, rownames,
        sapply, setdiff, sort, table, tapply, union, unique, unsplit, which, which.max, which.min
    
    
    Attaching package: 'S4Vectors'
    
    The following object is masked from 'package:base':
    
        expand.grid
    
    Loading required package: MultiAssayExperiment
    Loading required package: BiocParallel
    Quitting from lines 159-163 (ClassifyR.Rmd) 
    Error: processing vignette 'ClassifyR.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘measurement’
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘..density..’
    plotFeatureClasses,DataFrame : <anonymous>: no visible binding for
      global variable ‘legends grouping’
    samplesMetricMap,list: no visible binding for global variable ‘Class’
    samplesMetricMap,list: no visible binding for global variable ‘name’
    samplesMetricMap,list: no visible binding for global variable ‘type’
    samplesMetricMap,list: no visible binding for global variable ‘Metric’
    selectionPlot,list: no visible binding for global variable ‘Freq’
    Undefined global functions or variables:
      ..density.. Class Freq Metric legends grouping measurement name type
    Consider adding
      importFrom("base", "grouping")
    to your NAMESPACE file.
    ```

# classyfire

Version: 0.1-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .boxRadial: no visible global function definition for ‘predict’
    .radialSVM: no visible global function definition for ‘predict’
    cfPredict: no visible global function definition for ‘predict’
    getPerm5Num: no visible global function definition for ‘median’
    getPerm5Num: no visible global function definition for ‘quantile’
    ggClassPred: no visible global function definition for ‘ftable’
    ggEnsHist: no visible global function definition for ‘sd’
    ggPermHist: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      ftable median predict quantile sd
    Consider adding
      importFrom("stats", "ftable", "median", "predict", "quantile", "sd")
    to your NAMESPACE file.
    ```

# cleanEHR

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clhs

Version: 0.6-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘scales’ ‘sp’
      All declared Imports should be used.
    ```

# clickstream

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clifro

Version: 3.1-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# climbeR

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ClimClass

Version: 2.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# climwin

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clinDR

Version: 1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CLLmethylation

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clusterfly

Version: 0.4

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘rggobi’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# clusternomics

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clusterProfiler

Version: 3.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘topGO’
    ```

# ClusterR

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ClusterSignificance

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# clustMD

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clustRcompaR

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clustrd

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# clustree

Version: 0.1.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘clustree-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: clustree
    > ### Title: Plot a clustering tree
    > ### Aliases: clustree clustree.matrix clustree.data.frame
    > ###   clustree.SingleCellExperiment clustree.seurat
    > 
    > ### ** Examples
    > 
    > data(iris_clusts)
    > clustree(iris_clusts, prefix = "K")
    Error in guide_train.edge_colourbar(guide, scale, output) : 
      unused argument (output)
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> build_guides -> guides_train -> guide_train
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 77-78 (clustree.Rmd) 
    Error: processing vignette 'clustree.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cmaesr

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CNEr

Version: 1.16.0

## In both

*   R CMD check timed out
    

*   checking compiled code ... WARNING
    ```
    ...
        Objects: ‘ucsc/axt.o’, ‘ucsc/errabort.o’, ‘ucsc/obscure.o’,
          ‘ucsc/verbose.o’, ‘ucsc/os.o’
      Found ‘___stdoutp’, possibly from ‘stdout’ (C)
        Objects: ‘ucsc/common.o’, ‘ucsc/errabort.o’, ‘ucsc/verbose.o’,
          ‘ucsc/os.o’
      Found ‘_abort’, possibly from ‘abort’ (C)
        Object: ‘ucsc/errabort.o’
      Found ‘_exit’, possibly from ‘exit’ (C)
        Objects: ‘ucsc/errabort.o’, ‘ucsc/pipeline.o’
      Found ‘_printf’, possibly from ‘printf’ (C)
        Objects: ‘ceScan.o’, ‘ucsc/pipeline.o’
      Found ‘_puts’, possibly from ‘printf’ (C), ‘puts’ (C)
        Object: ‘ucsc/pipeline.o’
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘ucsc/obscure.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 19.4Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata  15.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘S4Vectors:::make_zero_col_DataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

# CNPBayes

Version: 1.10.0

## In both

*   checking Rd \usage sections ... WARNING
    ```
    Undocumented arguments in documentation object 'marginal_lik'
      ‘value’
    
    Functions with \usage entries need to have the appropriate \alias
    entries, and all their arguments documented.
    The \usage entries must correspond to syntactically valid R code.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Trying to automatically install missing LaTeX packages...
    tlmgr install dvips
    TeX Live 2017 is frozen forever and will no
    longer be updated.  This happens in preparation for a new release.
    
    If you're interested in helping to pretest the new release (when
    pretests are available), please read http://tug.org/texlive/pretest.html.
    Otherwise, just wait, and the new release will be ready in due time.
    tlmgr: package repository ftp://tug.org/historic/systems/texlive/2017/tlnet-final (not verified: gpg unavailable)
    [1/2, ??:??/??:??] install: dvips.x86_64-darwin [146k]
    [2/2, 00:02/00:02] install: dvips [34k]
    running mktexlsr ...
    done running mktexlsr.
    tlmgr: package log updated: /Users/max/Library/TinyTeX/texmf-var/web2c/tlmgr.log
    tlmgr path add
    add_link_dir_dir: /usr/local/bin/allcm exists; not making symlink.
    add_link_dir_dir: /usr/local/bin/texhash exists; not making symlink.
    Quitting from lines 51-61 (FindCNPs.Rmd) 
    Error: processing vignette 'FindCNPs.Rmd' failed with diagnostics:
    unable to find required package 'VanillaICE'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        doc    3.4Mb
        libs   1.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    copyNumber,SingleBatchCopyNumber: no visible binding for global
      variable ‘theta.star’
    Undefined global functions or variables:
      theta.star
    ```

# CNVPanelizer

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max, which.min
    
    Loading required package: S4Vectors
    
    Attaching package: 'S4Vectors'
    
    The following object is masked from 'package:base':
    
        expand.grid
    
    Loading required package: IRanges
    Loading required package: GenomeInfoDb
    Calculating Background for Sample_1
    Calculating Background for Sample_2
    Calculating Background for Sample_3
    Calculating Background for Sample_4
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CNVPanelizer.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘utils:::format.object_size’
      See the note in ?`:::` about the use of this operator.
    ```

# CNVrd2

Version: 1.18.0

## In both

*   checking whether package ‘CNVrd2’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/new/CNVrd2.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CNVrd2’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘CNVrd2’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/new/CNVrd2.Rcheck/CNVrd2’

```
### CRAN

```
* installing *source* package ‘CNVrd2’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CNVrd2/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘CNVrd2’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CNVrd2/old/CNVrd2.Rcheck/CNVrd2’

```
# coalitions

Version: 0.6.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cobalt

Version: 3.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cocoreg

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# codebook

Version: 0.5.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘pander’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 65 marked UTF-8 strings
    ```

# codingMatrices

Version: 0.3.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'codingMatrices.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# codyn

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# coefplot

Version: 1.2.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cofeatureR

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CoGAPS

Version: 3.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    cellMatchR: no visible binding for global variable ‘cluster.method’
    patternMarkers: no visible binding for global variable ‘nP’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘Samples’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘value’
    patternMatcher : <anonymous>: no visible binding for global variable
      ‘BySet’
    runFinalPhase: no visible binding for global variable ‘i’
    runFinalPhase: no visible binding for global variable ‘sampleS’
    runInitialPhase: no visible binding for global variable ‘i’
    runInitialPhase: no visible binding for global variable ‘sampleS’
    scCoGapsFromCheckpoint: no visible binding for global variable ‘i’
    scCoGapsFromCheckpoint: no visible binding for global variable
      ‘sampleS’
    sc_runFinalPhase: no visible binding for global variable ‘i’
    sc_runFinalPhase: no visible binding for global variable ‘sampleS’
    sc_runInitialPhase: no visible binding for global variable ‘i’
    sc_runInitialPhase: no visible binding for global variable ‘sampleS’
    Undefined global functions or variables:
      BySet Samples cluster.method i nP sampleS value
    ```

*   checking compiled code ... NOTE
    ```
    File ‘CoGAPS/libs/CoGAPS.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘test-runner.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘test-runner.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning in gapsRun(SimpSim.D, SimpSim.S, nFactor = 3, nEquil = nIter, nSample = nIter) :
      gapsRun is deprecated with v3.0, use CoGAPS
    Warning in v2CoGAPS(result, ...) :
      GStoGenes is deprecated with v3.0, see CoGAPS documentation
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CoGAPSUsersManual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cogena

Version: 1.14.0

## In both

*   checking whether package ‘cogena’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘cogena’
      Warning: replacing previous import ‘class::somgrid’ by ‘kohonen::somgrid’ when loading ‘cogena’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/cogena/new/cogena.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   3.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    corInCluster,cogena: no visible global function definition for ‘cor’
    heatmapCluster,cogena: no visible global function definition for
      ‘topo.colors’
    heatmapCluster,cogena: no visible global function definition for
      ‘rainbow’
    heatmapCluster,cogena: no visible global function definition for ‘par’
    heatmapCluster,cogena: no visible global function definition for
      ‘legend’
    Undefined global functions or variables:
      abline as.dist axis cor data density dist hist image layout legend
      lines median mtext order.dendrogram p.adjust par phyper plot.new
      rainbow rect reorder sd text title topo.colors
    Consider adding
      importFrom("grDevices", "rainbow", "topo.colors")
      importFrom("graphics", "abline", "axis", "hist", "image", "layout",
                 "legend", "lines", "mtext", "par", "plot.new", "rect",
                 "text", "title")
      importFrom("stats", "as.dist", "cor", "density", "dist", "median",
                 "order.dendrogram", "p.adjust", "phyper", "reorder", "sd")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# coinmarketcapr

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CollapsABEL

Version: 0.10.11

## In both

*   checking whether package ‘CollapsABEL’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CollapsABEL/new/CollapsABEL.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CollapsABEL’ ...
** package ‘CollapsABEL’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘CollapsABEL’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CollapsABEL/new/CollapsABEL.Rcheck/CollapsABEL’

```
### CRAN

```
* installing *source* package ‘CollapsABEL’ ...
** package ‘CollapsABEL’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CollapsABEL/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘CollapsABEL’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CollapsABEL/old/CollapsABEL.Rcheck/CollapsABEL’

```
# CollapseLevels

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# coloc

Version: 3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘flashClust’ ‘speedglm’
      All declared Imports should be used.
    ```

# colorednoise

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# colormap

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# colorpatch

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘grid’ ‘gridExtra’
      All declared Imports should be used.
    ```

# colorplaner

Version: 0.1.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘colorplaner-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: color_projections
    > ### Title: Color Space Projections
    > ### Aliases: YUV_projection color_projections interpolate_projection
    > ###   red_blue_projection
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > ggplot(mtcars, aes(x = wt, y = mpg, color = disp, colour2 = hp)) +
    +   geom_point() +
    +   scale_color_colorplane(color_projection = interpolate_projection,
    +                          zero_color = "darkorange2",
    +                          horizontal_color = "mediumspringgreen",
    +                          vertical_color = "magenta")
    Error in guide_train.colorplane(guide, scale, output) : 
      unused argument (output)
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> build_guides -> guides_train -> guide_train
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3 SKIPPED: 0 FAILED: 20
      1. Error: Basic plotting occurs without error (@test_basics.R#16) 
      2. Error: Transformed colorplane scale plots without error (@test_basics.R#37) 
      3. Error: OOB and NA data handled without error (@test_basics.R#50) 
      4. Error: Arguments can be passed to projection functions (@test_basics.R#71) 
      5. Error: Faceted plots (@test_complex_plots.R#18) 
      6. Error: Grouped plots (@test_complex_plots.R#29) 
      7. Error: Multiple data layer plots (@test_complex_plots.R#39) 
      8. Error: Colorplane only on some layers without warning (@test_complex_plots.R#52) 
      9. Error: Title options (@test_guide_options.R#20) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 35-54 (colorplaner.Rmd) 
    Error: processing vignette 'colorplaner.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# colourpicker

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CommT

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    CommT.anova: no visible global function definition for ‘aov’
    CommT.legendpos: no visible global function definition for ‘quantile’
    CommT.plotcolors: no visible global function definition for ‘hcl’
    Undefined global functions or variables:
      aov hcl quantile
    Consider adding
      importFrom("grDevices", "hcl")
      importFrom("stats", "aov", "quantile")
    to your NAMESPACE file.
    ```

# compcodeR

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: sm
    Package 'sm', version 2.2-5.5: type help(sm) for summary information
    
    This is package 'modeest' written by P. PONCET.
    For a complete list of functions, use 'library(help = "modeest")' or 'help.start()'.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'compcodeR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking: ‘rpanel’ ‘DSS’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘rpanel’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘packageVersion’
    voom.limma.createRmd: no visible global function definition for
      ‘packageVersion’
    voom.ttest.createRmd: no visible global function definition for
      ‘packageVersion’
    vst.limma.createRmd: no visible global function definition for
      ‘packageVersion’
    vst.ttest.createRmd: no visible global function definition for
      ‘packageVersion’
    show,compData: no visible global function definition for ‘head’
    Undefined global functions or variables:
      as.dist axis cor hclust head heat.colors legend lines loess median
      na.omit packageVersion par predict rexp rnbinom rpois runif sd title
    Consider adding
      importFrom("grDevices", "heat.colors")
      importFrom("graphics", "axis", "legend", "lines", "par", "title")
      importFrom("stats", "as.dist", "cor", "hclust", "loess", "median",
                 "na.omit", "predict", "rexp", "rnbinom", "rpois", "runif",
                 "sd")
      importFrom("utils", "head", "packageVersion")
    to your NAMESPACE file.
    ```

# CompGO

Version: 1.16.0

## In both

*   checking whether package ‘CompGO’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CompGO/new/CompGO.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CompGO’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘RDAVIDWebService’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘RDAVIDWebService’ could not be loaded
ERROR: lazy loading failed for package ‘CompGO’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CompGO/new/CompGO.Rcheck/CompGO’

```
### CRAN

```
* installing *source* package ‘CompGO’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘RDAVIDWebService’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/CompGO/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘RDAVIDWebService’ could not be loaded
ERROR: lazy loading failed for package ‘CompGO’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CompGO/old/CompGO.Rcheck/CompGO’

```
# complmrob

Version: 0.6.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# comtradr

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# CONFESS

Version: 1.8.0

## In both

*   checking whether package ‘CONFESS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘stats::BIC’ by ‘flowMerge::BIC’ when loading ‘CONFESS’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CONFESS/new/CONFESS.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    Warning: replacing previous import 'stats::BIC' by 'flowMerge::BIC' when loading 'CONFESS'
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `vignette_files/figure-html/unnamed-chunk-10-2.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `vignette_files/figure-html/unnamed-chunk-11-2.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    
    Setting B = 1 performs run-based cross-validation only (keeps run(s) 1,2,3,4)
    
    -----------------------------------------------------------
    | Running the CV analysis for reference run 5 and CV 1... |
    -----------------------------------------------------------
    
    Setting B = 1 performs run-based cross-validation only (keeps run(s) 1,2,3,4)
    
    -----------------------------------------------------------
    | Running the CV analysis for reference run 5 and CV 1... |
    -----------------------------------------------------------
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette_tex.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# confidence

Version: 1.1-2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: tcltk
    
    For more information on the usage of the Confidence tool, type:
    vignette("confidence")
    Examples of input files can be found in:
    '/Users/max/github/ggplot2/revdep/checks.noindex/confidence/new/confidence.Rcheck/confidence/extdata'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'confidence.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ConfoundedMeta

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# congressbr

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# Conigrave

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# consensusOV

Version: 1.2.0

## In both

*   R CMD check timed out
    

# consensusSeekeR

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# constellation

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# contiBAIT

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ideogramPlot.func: no visible binding for global variable ‘w’
    ideogramPlot.func: no visible binding for global variable ‘w2’
    makeChrTable: no visible global function definition for ‘queryHits’
    makeChrTable: no visible global function definition for ‘subjectHits’
    orderContigsGreedy: no visible global function definition for
      ‘makeCluster’
    orderContigsGreedy: no visible global function definition for
      ‘clusterCall’
    orderContigsGreedy: no visible global function definition for
      ‘stopCluster’
    strandSeqFreqTable: no visible global function definition for
      ‘queryHits’
    strandSeqFreqTable: no visible global function definition for
      ‘subjectHits’
    ideogramPlot,StrandReadMatrix-StrandReadMatrix-ChrTable: no visible
      binding for global variable ‘w’
    ideogramPlot,StrandReadMatrix-StrandReadMatrix-ChrTable: no visible
      binding for global variable ‘w2’
    Undefined global functions or variables:
      clusterCall makeCluster queryHits stopCluster subjectHits w w2
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    -> Generating plotting data [20/35]
    -> Generating plotting data [21/35]
    -> Generating plotting data [22/35]
    -> Generating plotting data [23/35]
    -> Generating plotting data [24/35]
    -> Generating plotting data [25/35]
    -> Generating plotting data [26/35]
    -> Generating plotting data [27/35]
    -> Generating plotting data [28/35]
    -> Generating plotting data [29/35]
    -> Generating plotting data [30/35]
    -> Generating plotting data [31/35]
    -> Generating plotting data [32/35]
    -> Generating plotting data [33/35]
    -> Generating plotting data [34/35]
    -> Generating plotting data [35/35]
    Loading required package: shape
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'contiBAIT.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# contoureR

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cooccur

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CoordinateCleaner

Version: 1.0-7

## In both

*   checking whether package ‘CoordinateCleaner’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/new/CoordinateCleaner.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘CoordinateCleaner’ ...
** package ‘CoordinateCleaner’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘CoordinateCleaner’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/new/CoordinateCleaner.Rcheck/CoordinateCleaner’

```
### CRAN

```
* installing *source* package ‘CoordinateCleaner’ ...
** package ‘CoordinateCleaner’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘CoordinateCleaner’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/CoordinateCleaner/old/CoordinateCleaner.Rcheck/CoordinateCleaner’

```
# CopulaDTA

Version: 1.0.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 292-293 (CopulaDTA_Vignette.Rmd) 
    Error: processing vignette 'CopulaDTA_Vignette.Rmd' failed with diagnostics:
    missing value where TRUE/FALSE needed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# corkscrew

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# corrr

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# coseq

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cosinor

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    cosinor_analyzer : <anonymous>: no visible global function definition
      for ‘as.formula’
    ggplot.cosinor.lm: no visible global function definition for ‘predict’
    predict.cosinor.lm: no visible global function definition for ‘fitted’
    predict.cosinor.lm: no visible global function definition for ‘predict’
    simulate_cosinor: no visible global function definition for ‘runif’
    simulate_cosinor: no visible global function definition for ‘rbinom’
    simulate_cosinor: no visible global function definition for ‘rnorm’
    summary.cosinor.lm: no visible global function definition for ‘vcov’
    summary.cosinor.lm: no visible global function definition for ‘qnorm’
    summary.cosinor.lm: no visible global function definition for ‘pnorm’
    test_cosinor: no visible global function definition for ‘pchisq’
    test_cosinor: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      as.formula fitted lm na.omit pchisq pnorm predict qnorm rbinom rnorm
      runif terms vcov vitamind
    Consider adding
      importFrom("stats", "as.formula", "fitted", "lm", "na.omit", "pchisq",
                 "pnorm", "predict", "qnorm", "rbinom", "rnorm", "runif",
                 "terms", "vcov")
    to your NAMESPACE file.
    ```

# cosinor2

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
           25        26        27        28        29        30 
    1.6250000 1.3815789 1.2894737 1.3333333 1.1710526 0.8088235 
    
    $cosinors[[17]]$fitted.values
     [1] 0.9751600 1.3201375 1.6020512 1.6086144 1.3348849 0.9869865 0.8268939
     [8] 0.9751600 1.3201375 1.6020512 1.6086144 1.3348849 0.9869865 0.8268939
    
    $cosinors[[17]]$residuals
              17           18           19           20           21           22 
     0.372062185 -0.099549245 -0.129828980  0.030274467  0.408536153  0.061624562 
              23           24           25           26           27           28 
    -0.368560591 -0.113317932  0.304862520 -0.220472255 -0.319140737 -0.001551566 
              29           30 
     0.184066083 -0.018070395 
    
    
    
    $plots
    $plots[[1]]
    Error: Column `y` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CosmoPhotoz

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    CosmoPhotoZestimator: no visible global function definition for
      ‘predict’
    computeCombPCA: no visible global function definition for ‘prcomp’
    computeDiagPhotoZ: no visible global function definition for ‘sd’
    computeDiagPhotoZ: no visible global function definition for ‘median’
    computeDiagPhotoZ: no visible global function definition for ‘mad’
    glmPredictPhotoZ: no visible global function definition for ‘predict’
    glmTrainPhotoZ: no visible global function definition for ‘glm’
    glmTrainPhotoZ: no visible global function definition for ‘Gamma’
    glmTrainPhotoZ: no visible global function definition for
      ‘inverse.gaussian’
    plotDiagPhotoZ: no visible global function definition for ‘median’
    plotDiagPhotoZ: no visible global function definition for ‘mad’
    Undefined global functions or variables:
      Gamma glm inverse.gaussian mad median prcomp predict sd
    Consider adding
      importFrom("stats", "Gamma", "glm", "inverse.gaussian", "mad",
                 "median", "prcomp", "predict", "sd")
    to your NAMESPACE file.
    ```

# CountClust

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        data   8.1Mb
    ```

# countyfloods

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# countytimezones

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: stringr
    Loading required package: XML
    
    Attaching package: 'XML'
    
    The following object is masked from 'package:tools':
    
        toHTML
    
    
    Attaching package: 'acs'
    
    The following object is masked from 'package:base':
    
        apply
    
    Quitting from lines 111-123 (countytimezones.Rmd) 
    Error: processing vignette 'countytimezones.Rmd' failed with diagnostics:
    package or namespace load failed for 'choroplethr' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
     there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# countyweather

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# covafillr

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cowbell

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘misc3d’
      All declared Imports should be used.
    ```

# cowplot

Version: 0.9.2

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Component 1: Attributes: < Component 2: target is character, current is list >
      Component 1: Attributes: < Component 3: Modes: numeric, character >
      Component 1: Attributes: < Component 3: target is numeric, current is character >
      
      ── 2. Failure: complex alignments, h, v, hv (@test_align_plots.R#42)  ──────────
      grid::convertUnit(...) not equal to grid::convertUnit(...).
      Mean relative difference: 0.6174489
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 18 SKIPPED: 0 FAILED: 2
      1. Failure: complex alignments, h, v, hv (@test_align_plots.R#39) 
      2. Failure: complex alignments, h, v, hv (@test_align_plots.R#42) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘cowplot-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: draw_image
    > ### Title: Draw an image
    > ### Aliases: draw_image
    > 
    > ### ** Examples
    > 
    > # Use image as plot background
    > p <- ggplot(iris, aes(x = Sepal.Length, fill = Species)) + geom_density(alpha = 0.7)
    > ggdraw() +
    +   draw_image("http://jeroen.github.io/images/tiger.svg") +
    +   draw_plot(p + theme(legend.box.background = element_rect(color = "white")))
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: cowplot
    Loading required package: ggplot2
    
    Attaching package: 'cowplot'
    
    The following object is masked from 'package:ggplot2':
    
        ggsave
    
    Loading required package: grid
    Loading required package: viridisLite
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cplm

Version: 0.7-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: coda
    Loading required package: Matrix
    Loading required package: splines
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cplm.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cpr

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cpvSNP

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GSEABase’ which was already attached by Depends.
      Please remove these calls from your code.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    createArrayData: no visible global function definition for
      ‘elementMetadata<-’
    geneToSNPList: no visible global function definition for ‘findOverlaps’
    glossiMarginal: no visible global function definition for ‘pgamma’
    plotPvals: no visible binding for global variable ‘df’
    plotPvals: no visible binding for global variable ‘pval’
    simulate_chisq: no visible global function definition for ‘rnorm’
    vegasMarginal: no visible global function definition for ‘rbinom’
    vegasMarginal: no visible global function definition for ‘qchisq’
    vegasPrep: no visible global function definition for ‘elementMetadata’
    show,VEGASResult: no visible global function definition for ‘var’
    Undefined global functions or variables:
      GRanges IRanges Rle col2rgb density df elementMetadata
      elementMetadata<- findOverlaps legend lines pgamma plot points
      polygon pval qchisq rbinom rgb rnorm var
    Consider adding
      importFrom("grDevices", "col2rgb", "rgb")
      importFrom("graphics", "legend", "lines", "plot", "points", "polygon")
      importFrom("stats", "density", "df", "pgamma", "qchisq", "rbinom",
                 "rnorm", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: XML
    
    Attaching package: ‘XML’
    
    The following object is masked from ‘package:tools’:
    
        toHTML
    
    Loading required package: graph
    
    Attaching package: ‘graph’
    
    The following object is masked from ‘package:XML’:
    
        addNode
    
    making GRanges object
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cpvSNP.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# cr17

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cranly

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc   4.7Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4 marked Latin-1 strings
      Note: found 25 marked UTF-8 strings
    ```

# crawl

Version: 2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# cRegulome

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘AnnotationDbi’ ‘clusterProfiler’ ‘org.Hs.eg.db’
      All declared Imports should be used.
    ```

# cricketr

Version: 0.0.14

## Newly broken

*   checking whether package ‘cricketr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘cricketr’
      Warning: replacing previous import ‘ggplot2::autolayer’ by ‘forecast::autolayer’ when loading ‘cricketr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/cricketr/new/cricketr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CrispRVariants

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# crmPack

Version: 0.2.7

## In both

*   checking whether package ‘crmPack’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/new/crmPack.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘crmPack’ ...
** package ‘crmPack’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘crmPack’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/new/crmPack.Rcheck/crmPack’

```
### CRAN

```
* installing *source* package ‘crmPack’ ...
** package ‘crmPack’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/crmPack/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘crmPack’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/crmPack/old/crmPack.Rcheck/crmPack’

```
# Crossover

Version: 0.1-17

## In both

*   checking whether package ‘Crossover’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Crossover/new/Crossover.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘Crossover’ ...
** package ‘Crossover’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/RcppArmadillo/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Crossover_init.c -o Crossover_init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/RcppArmadillo/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c searchCOD.cpp -o searchCOD.o
searchCOD.cpp:122:11: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
          pinv(trans(X) * cor * X);
          ^~~~
1 warning generated.
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o Crossover.so Crossover_init.o searchCOD.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/Crossover/new/Crossover.Rcheck/Crossover/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘Crossover’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Crossover/new/Crossover.Rcheck/Crossover’

```
### CRAN

```
* installing *source* package ‘Crossover’ ...
** package ‘Crossover’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/RcppArmadillo/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Crossover_init.c -o Crossover_init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/Crossover/RcppArmadillo/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c searchCOD.cpp -o searchCOD.o
searchCOD.cpp:122:11: warning: ignoring return value of function declared with 'warn_unused_result' attribute [-Wunused-result]
          pinv(trans(X) * cor * X);
          ^~~~
1 warning generated.
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o Crossover.so Crossover_init.o searchCOD.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/Crossover/old/Crossover.Rcheck/Crossover/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Crossover/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘Crossover’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Crossover/old/Crossover.Rcheck/Crossover’

```
# CrossScreening

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in wilcox.test.default(d.lead, alternative = "greater") :
      cannot compute exact p-value with ties
    Warning in wilcox.test.default(d.lead, alternative = "greater") :
      cannot compute exact p-value with zeroes
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'CrossScreening-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# crosstalk

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CRTgeeDR

Version: 2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# csabounds

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘qte’
      All declared Imports should be used.
    ```

# csp

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.2Mb
      sub-directories of 1Mb or more:
        data  11.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 59670 marked Latin-1 strings
      Note: found 16 marked UTF-8 strings
    ```

# CTDquerier

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.6Mb
      sub-directories of 1Mb or more:
        data      4.1Mb
        doc       6.1Mb
        extdata   4.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 161 marked UTF-8 strings
    ```

# ctsGE

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cummeRbund

Version: 2.22.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘RSQLite’ ‘ggplot2’ ‘reshape2’ ‘fastcluster’
      ‘rtracklayer’ ‘Gviz’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.4Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        doc       1.6Mb
        extdata   5.5Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘BiocGenerics’ ‘plyr’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      'NMFN' 'cluster' 'rjson' 'stringr'
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: 'S4Vectors'
      All declared Imports should be used.
    Packages in Depends field not imported from:
      'Gviz' 'RSQLite' 'fastcluster' 'ggplot2' 'reshape2' 'rtracklayer'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      geom_point geom_rect geom_rug geom_segment geom_smooth geom_text
      geom_tile geom_vline ggplot guides hasAxis<- hclust ids import labs
      log2_fold_change make.db.names makeTxDb mean_cl_boot melt nnmf
      obsnames order.dendrogram p.adjust p_value pam plot plotIdeogram
      plotTracks position_dodge prcomp quant_status ranges read.delim
      read.table rowInd sample_1 sample_2 sample_name scale_color_gradient
      scale_color_hue scale_color_manual scale_colour_manual
      scale_fill_continuous scale_fill_gradient scale_fill_gradient2
      scale_fill_hue scale_x_continuous scale_x_discrete scale_x_log10
      scale_y_continuous scale_y_discrete scale_y_log10 sd seqnames
      significant start stat_density stat_smooth stat_sum stat_summary
      stdev str_split_fixed strand theme theme_bw toJSON tracking_id tracks
      unit v1 v2 value values variable varnames write.table x xlab xlim y
      ylab
    Consider adding
      importFrom("graphics", "plot")
      importFrom("stats", "as.dendrogram", "as.dist", "as.formula",
                 "cmdscale", "dist", "end", "hclust", "order.dendrogram",
                 "p.adjust", "prcomp", "sd", "start")
      importFrom("utils", "read.delim", "read.table", "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    	Replicate FPKMs
    	Counts
    Getting TSS information:
    	FPKM
    	Differential Expression Data
    	Annotation Data
    	Replicate FPKMs
    	Counts
    Getting promoter information:
    	distData
    Warning: Closing open result set, pending rows
    Getting splicing information:
    	distData
    Getting relCDS information:
    	distData
    Warning: Closing open result set, pending rows
    Warning: Closing open result set, pending rows
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'cummeRbund-example-workflow.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# curatedBreastData

Version: 2.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 283.1Mb
      sub-directories of 1Mb or more:
        data  282.7Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Analyzing dataset 2 or dataset named study_2034_GPL96_all
    
    Note: this function assumes your missing values
      are proper NAs, not "null",etc.
    
    It's best to impute NA values before running this function
    otherwise it may set averages to NA if there is 1 NA present.
    This function just removes any genes whose key is NA.
    
    You may get a warning here because key (usually gene) names are
      duplicated so it can't use them as row names. 
      That's OK, because we are immediately collapsing them into one feature.
    
    
    Starting with  286patients.
    found no multiple samples from the same patient(s)
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'curatedBreastData-manual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# curatedMetagenomicData

Version: 1.10.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    see ?curatedMetagenomicData and browseVignettes('curatedMetagenomicData') for documentation
    downloading 0 resources
    loading from cache 
        '/Users/max//.ExperimentHub/391'
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `curatedMetagenomicData_files/figure-html/unnamed-chunk-18-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `curatedMetagenomicData_files/figure-html/unnamed-chunk-21-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Warning in vegdist(c(12.95146, 3.07787, 12.5427, 0.00815, 0, 4.77261, 7.03481,  :
      you have empty rows: their dissimilarities may be meaningless in method "bray"
    Warning: Transformation introduced infinite values in discrete y-axis
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `curatedMetagenomicData_files/figure-html/unnamed-chunk-32-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Warning in estimate_richness(physeq, split = TRUE, measures = measures) :
      The data you have provided does not have
    any singletons. This is highly suspicious. Results of richness
    estimates (for example) are probably unreliable, or wrong, if you have already
    trimmed low-abundance taxa from the data.
    
    We recommended that you find the un-trimmed data and retry.
    Quitting from lines 330-332 (curatedMetagenomicData.Rmd) 
    Error: processing vignette 'curatedMetagenomicData.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        doc    1.4Mb
        help   2.7Mb
    ```

# cutoffR

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Cut: no visible global function definition for ‘cor’
    MissSimulation: no visible global function definition for ‘rbinom’
    cutoff: no visible global function definition for ‘cor’
    Undefined global functions or variables:
      cor rbinom
    Consider adding
      importFrom("stats", "cor", "rbinom")
    to your NAMESPACE file.
    ```

# cutpointr

Version: 0.7.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘cutpointr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_sensitivity_specificity
    > ### Title: Sensitivity and specificity plot from a cutpointr object
    > ### Aliases: plot_sensitivity_specificity
    > 
    > ### ** Examples
    > 
    > library(cutpointr)
    > 
    > ## Optimal cutpoint for dsi
    > data(suicide)
    > opt_cut <- cutpointr(suicide, dsi, suicide)
    Assuming the positive class is yes
    Assuming the positive class has higher x values
    > plot_sensitivity_specificity(opt_cut)
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to linetype
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# CVE

Version: 1.6.0

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                                    old_size new_size compress
      WGCNAmelanoma_extension.RData    2.4Mb    2.1Mb       xz
      crcCase.RData                    1.1Mb    675Kb       xz
      melanomaCase.RData               654Kb    472Kb       xz
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    1.3Mb
    ```

# cvequality

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cvxclustr

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    create_clustering_problem: no visible global function definition for
      ‘rnorm’
    Undefined global functions or variables:
      rnorm
    Consider adding
      importFrom("stats", "rnorm")
    to your NAMESPACE file.
    ```

*   checking line endings in Makefiles ... NOTE
    ```
    Found the following Makefile(s) without a final LF:
      src/Makevars
    Some ‘make’ programs ignore lines not ending in LF.
    ```

# Cyclops

Version: 1.3.2

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Sparseness = 77.35 %
      Sorting covariates by covariateId and rowId
      ── 1. Failure: Data errors and casting in COO-constructor (@test-xyConstructor.R
      `loadNewSeqlCyclopsDataMultipleX(dataPtr, c(1, 1), c(1, 1), NULL)` threw an error with unexpected message.
      Expected match: "1 - 1"
      Actual message: "c++ exception (unknown reason)"
      
      Sparseness = 99.09 %
      Sorting covariates by covariateId and rowId
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 201 SKIPPED: 17 FAILED: 1
      1. Failure: Data errors and casting in COO-constructor (@test-xyConstructor.R#212) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cystiSim

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# cytofkit

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'cytofkit_example.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        doc       4.3Mb
        extdata   3.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    cytof_clusterPlot: no visible global function definition for
      ‘geom_text’
    cytof_progressionPlot: no visible global function definition for ‘aes’
    cytof_progressionPlot: no visible binding for global variable
      ‘Pseudotime’
    cytof_progressionPlot: no visible binding for global variable ‘cluster’
    cytofkitShinyAPP : <anonymous> : C_ScatterPlotInput: no visible global
      function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous>: no visible global function definition
      for ‘heatMap’
    cytofkitShinyAPP : <anonymous> : M_markerExpressionPlotInput: no
      visible global function definition for ‘scatterPlot’
    cytofkitShinyAPP : <anonymous> : M_stackDensityPlotInput: no visible
      global function definition for ‘stackDenistyPlot’
    cytofkitShinyAPP : <anonymous> : P_markerPlotInput: no visible global
      function definition for ‘cytof_expressionTrends’
    Undefined global functions or variables:
      Pseudotime aes cluster cytof_expressionTrends geom_text heatMap
      scatterPlot stackDenistyPlot
    ```

# cytometree

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dada2

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘ShortRead:::.set_omp_threads’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call(ShortRead:::.set_omp_threads, ...)
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotQualityProfile: no visible binding for global variable ‘Score’
    plotQualityProfile: no visible binding for global variable ‘Count’
    plotQualityProfile: no visible binding for global variable ‘Mean’
    plotQualityProfile: no visible binding for global variable ‘Q25’
    plotQualityProfile: no visible binding for global variable ‘Q50’
    plotQualityProfile: no visible binding for global variable ‘Q75’
    plotQualityProfile: no visible binding for global variable ‘Cum’
    samdf_to_qiime2: no visible global function definition for
      ‘write.table’
    seqtab_to_mothur: no visible global function definition for
      ‘write.table’
    seqtab_to_mothur: no visible binding for global variable ‘four’
    seqtab_to_qiime: no visible global function definition for
      ‘write.table’
    Undefined global functions or variables:
      := Count Cum Cycle Estimated Input Mean Nominal Observed Q25 Q50 Q75
      Qual Score abundance accept allMismatch als1 als2 biocLite four indel
      mismatch n0F n0R prefer seqF seqR write.table
    Consider adding
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# dae

Version: 3.0-12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DALEX

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dalmatian

Version: 0.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 36-71 (weights-1-simulate.Rmd) 
    Error: processing vignette 'weights-1-simulate.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/dalmatian/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DaMiRseq

Version: 1.4.1

## In both

*   checking whether package ‘DaMiRseq’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DaMiRseq/new/DaMiRseq.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘DaMiRseq’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘DaMiRseq’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/DaMiRseq/new/DaMiRseq.Rcheck/DaMiRseq’

```
### CRAN

```
* installing *source* package ‘DaMiRseq’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/DaMiRseq/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘DaMiRseq’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/DaMiRseq/old/DaMiRseq.Rcheck/DaMiRseq’

```
# dams

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# DAPAR

Version: 1.12.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘DAPARdata’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# darksky

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dartR

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DBI’ ‘Demerelate’ ‘misc3d’ ‘plotly’ ‘quadprog’ ‘rgl’
      All declared Imports should be used.
    ```

# DataExplorer

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dataMaid

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# datarobot

Version: 2.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘curl’
      All declared Imports should be used.
    ```

# datasauRus

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DataVisualizations

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DBHC

Version: 0.0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘seqHMM’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# dbplot

Version: 0.3.0

## Newly broken

*   checking whether package ‘dbplot’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘dbplot’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dbplot/new/dbplot.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DCD

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DChIPRep

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc       1.7Mb
        extdata   2.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# dcmr

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘combn’
    SampleParameterEstimates: no visible global function definition for
      ‘rnorm’
    plot,attribute.class-missing: no visible binding for global variable
      ‘attr.number’
    plot,attribute.class-missing: no visible binding for global variable
      ‘mean.attr’
    plot,attribute.profile.class-missing: no visible binding for global
      variable ‘attr.profile.number’
    plot,attribute.profile.class-missing: no visible binding for global
      variable ‘mean.attr.profile’
    summary,attribute.class: no visible binding for global variable ‘value’
    summary,attribute.profile.class: no visible binding for global variable
      ‘max.class’
    Undefined global functions or variables:
      attr.number attr.profile.number combn max.class mean.attr
      mean.attr.profile rmultinom rnorm runif value
    Consider adding
      importFrom("stats", "rmultinom", "rnorm", "runif")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# ddpcr

Version: 1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ddPCRclust

Version: 1.0.0

## In both

*   checking whether package ‘ddPCRclust’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: next used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ddPCRclust/new/ddPCRclust.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    exportPlots: no visible binding for global variable ‘Cluster’
    Undefined global functions or variables:
      Cluster
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning: replacing previous import ‘flowCore::plot’ by ‘graphics::plot’ when loading ‘flowDensity’
    double positive detection failed.
    Switching the middle two double populations.
    
    Starting the flow Peaks analysis...
    
        Task A: compute kmeans...
    
            ...finished summarization at 0.045 sec
    
        Task B: find peaks...
    finished at 0.063 sec
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ddPCRclust.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# debrowser

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    scatterZoom: no visible binding for global variable ‘size.hover’
    scatterZoom: no visible binding for global variable ‘key’
    startDEBrowser: no visible binding for global variable
      ‘.startdebrowser.called’
    volcanoPlot: no visible global function definition for ‘:=’
    volcanoPlot: no visible binding for global variable ‘size’
    volcanoPlot: no visible binding for global variable ‘size.hover’
    volcanoPlot: no visible binding for global variable ‘fillOpacity’
    volcanoPlot: no visible binding for global variable ‘fillOpacity.hover’
    volcanoPlot: no visible binding for global variable ‘fill.brush’
    volcanoPlot: no visible binding for global variable ‘opacity’
    volcanoPlot: no visible binding for global variable ‘key’
    volcanoZoom: no visible global function definition for ‘:=’
    volcanoZoom: no visible binding for global variable ‘size’
    volcanoZoom: no visible binding for global variable ‘size.hover’
    volcanoZoom: no visible binding for global variable ‘key’
    Undefined global functions or variables:
      .startdebrowser.called := NUL align baseline biocLite conds debrowser
      demodata div fill fill.brush fillOpacity fillOpacity.hover fit
      fontSize imageOutput key opacity renderImage samples searched size
      size.hover stroke
    ```

# DeconRNASeq

Version: 1.22.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.8Mb
      sub-directories of 1Mb or more:
        data   6.5Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘grid’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘grid’ ‘pcaMethods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    condplot: no visible global function definition for ‘plot’
    condplot: no visible global function definition for ‘rainbow’
    condplot: no visible global function definition for ‘lines’
    condplot: no visible global function definition for ‘axis’
    condplot: no visible global function definition for ‘title’
    decon.bootstrap: no visible global function definition for ‘t.test’
    multiplot: no visible global function definition for ‘grid.newpage’
    multiplot: no visible global function definition for ‘pushViewport’
    multiplot: no visible global function definition for ‘viewport’
    multiplot: no visible global function definition for ‘grid.layout’
    multiplot : vplayout: no visible global function definition for
      ‘viewport’
    Undefined global functions or variables:
      R2cum aes axis geom_abline geom_point ggplot grid.layout grid.newpage
      labs lines pca plot prep pushViewport rainbow t.test title viewport
      xlab ylab
    Consider adding
      importFrom("grDevices", "rainbow")
      importFrom("graphics", "axis", "lines", "plot", "title")
      importFrom("stats", "t.test")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    
    Attaching package: ‘pcaMethods’
    
    The following object is masked from ‘package:stats’:
    
        loadings
    
    Loading required package: ggplot2
    
    Attaching package: ‘ggplot2’
    
    The following object is masked from ‘package:limSolve’:
    
        resolution
    
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'DeconRNASeq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# decontam

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plot_frequency: no visible binding for global variable ‘DNA_conc’
    plot_frequency: no visible binding for global variable
      ‘taxon_abundance’
    plot_frequency: no visible binding for global variable ‘Type’
    plot_frequency: no visible binding for global variable ‘contam’
    plot_frequency: no visible binding for global variable ‘non.contam’
    Undefined global functions or variables:
      DNA_conc Type contam non.contam taxon_abundance
    ```

# deconvolveR

Version: 1.0-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Deducer

Version: 0.7-9

## In both

*   checking whether package ‘Deducer’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Deducer/new/Deducer.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘Deducer’ ...
** package ‘Deducer’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘Deducer’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Deducer/new/Deducer.Rcheck/Deducer’

```
### CRAN

```
* installing *source* package ‘Deducer’ ...
** package ‘Deducer’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Deducer/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘Deducer’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Deducer/old/Deducer.Rcheck/Deducer’

```
# DeepBlueR

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking Rd files ... NOTE
    ```
    prepare_Rd: deepblue_enrich_regions_fast.Rd:35-38: Dropping empty section \examples
    ```

# deGradInfer

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DEGreport

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘knitr’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘enrichGO’
    .run_cluster_profiler: no visible global function definition for
      ‘simplify’
    .table_w_fc: no visible binding for global variable ‘comp’
    .table_w_fc: no visible binding for global variable ‘log2FoldChange’
    degCheckFactors: no visible binding for global variable ‘ratios’
    degCorCov: no visible binding for global variable ‘compare’
    degMV: no visible binding for global variable ‘min_median’
    degMV: no visible binding for global variable ‘max_sd’
    degPatterns: no visible global function definition for ‘rowMedians’
    degPatterns: no visible binding for global variable ‘genes’
    degPlotWide : <anonymous>: no visible binding for global variable
      ‘count’
    significants,TopTags: no visible binding for global variable ‘FDR’
    significants,TopTags: no visible binding for global variable ‘logFC’
    significants,list : <anonymous>: no visible binding for global variable
      ‘gene’
    Undefined global functions or variables:
      .x FDR base_mean comp compare count counts covar enrichGO gene genes
      keys log2FoldChange log2fc logFC max_sd min_median ratios rowMedians
      simplify
    ```

# deltaGseg

Version: 1.20.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Note: nlevels has been renamed to nlevelsWT
    
    Loading required package: tseries
    Loading required package: pvclust
    Loading required package: fBasics
    Loading required package: timeDate
    Loading required package: timeSeries
    
    Attaching package: 'timeSeries'
    
    The following object is masked from 'package:zoo':
    
        time<-
    
    Loading required package: grid
    Loading required package: reshape
    Loading required package: scales
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'deltaGseg.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘ggplot2’ ‘changepoint’ ‘wavethresh’ ‘tseries’ ‘pvclust’ ‘fBasics’
      ‘grid’ ‘reshape’ ‘scales’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    diagnosticPlots,SegSeriesTrajectories: no visible binding for global
      variable ‘residuals’
    diagnosticPlots,SegSeriesTrajectories: no visible global function
      definition for ‘acf’
    diagnosticPlots,SegSeriesTrajectories: no visible global function
      definition for ‘qnorm’
    diagnosticPlots,SegSeriesTrajectories: no visible binding for global
      variable ‘lag’
    Undefined global functions or variables:
      abline acf as.dist cor cutree dist hclust identify lag layout lines
      locator median mtext na.omit par points qnorm quantile read.table
      residuals setTxtProgressBar text txtProgressBar
    Consider adding
      importFrom("graphics", "abline", "identify", "layout", "lines",
                 "locator", "mtext", "par", "points", "text")
      importFrom("stats", "acf", "as.dist", "cor", "cutree", "dist",
                 "hclust", "lag", "median", "na.omit", "qnorm", "quantile",
                 "residuals")
      importFrom("utils", "read.table", "setTxtProgressBar",
                 "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘diagplots.png’, ‘simclust.png’, ‘simtraj.png’, ‘simtrajtr.png’,
      ‘simtrajtr2.png’, ‘traj1.png’, ‘traj1break.png’, ‘traj1ss0.png’,
      ‘traj1ss1.png’, ‘traj1tr.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

# demi

Version: 1.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'demi.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    demi.wilcox.test.fast: no visible global function definition for
      ‘wilcox.test’
    demiequal: no visible global function definition for ‘wilcox.test’
    wprob: no visible global function definition for ‘combn’
    diffexp,DEMIDiff: no visible global function definition for ‘median’
    diffexp,DEMIDiff: no visible global function definition for ‘p.adjust’
    loadAnnotation,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadBlat,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadCytoband,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    loadPathway,DEMIExperiment-environment: no visible global function
      definition for ‘data’
    Undefined global functions or variables:
      combn data dhyper median p.adjust t.test wilcox.test write.table
    Consider adding
      importFrom("stats", "dhyper", "median", "p.adjust", "t.test",
                 "wilcox.test")
      importFrom("utils", "combn", "data", "write.table")
    to your NAMESPACE file.
    ```

# dendextend

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘dendroextras’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘dendroextras’, ‘moduleColor’, ‘distory’, ‘phangorn’
    ```

# DendroSync

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    The error most likely occurred in:
    
    > ### Name: sync.plot
    > ### Title: Plot within- and between-group synchrony
    > ### Aliases: sync.plot
    > 
    > ### ** Examples
    > 
    > ## Plot homoscedastic narrow evaluation (mNE) and unstructured model (mUN)
    >  # synchronies for conifersIP data:
    >  data(conifersIP)
    >      
    >  ##Fit the homoscedastic set of varcov models (mBE, mNE, mCS, mUN)
    >  # using geographic grouping criteria (ie. Region)
    >  ModHm <- dendro.varcov(TRW ~ Code, varTime = "Year", varGroup = "Region", 
    +                         data = conifersIP, homoscedastic = TRUE)
    [1] "Please wait. I am fitting the models now :)"
    >  
    >  sync.plot(sync(ModHm, modname = "mNE"))
    Error: Columns `ymin`, `ymax` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dendroTools

Version: 0.0.7

## In both

*   checking whether package ‘dendroTools’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dendroTools/new/dendroTools.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘dendroTools’ ...
** package ‘dendroTools’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘dendroTools’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dendroTools/new/dendroTools.Rcheck/dendroTools’

```
### CRAN

```
* installing *source* package ‘dendroTools’ ...
** package ‘dendroTools’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/dendroTools/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘dendroTools’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dendroTools/old/dendroTools.Rcheck/dendroTools’

```
# DengueRT

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# densityClust

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DEP

Version: 1.2.0

## Newly broken

*   checking whether package ‘DEP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘DEP’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DEP/new/DEP.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   1.4Mb
        doc    3.1Mb
    ```

# DepthProc

Version: 2.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# derfinder

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in citation("BiocStyle") :
      no date field in DESCRIPTION file of package 'BiocStyle'
    Quitting from lines 63-131 (derfinder-quickstart.Rmd) 
    Error: processing vignette 'derfinder-quickstart.Rmd' failed with diagnostics:
    package 'knitrBootstrap' not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.3Mb
      sub-directories of 1Mb or more:
        doc   5.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘GenomeInfoDb:::.guessSpeciesStyle’
      ‘GenomeInfoDb:::.supportedSeqnameMappings’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.smootherFstats’
    ```

# derfinderPlot

Version: 1.14.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > genes <- annotateTranscripts(txdb = TxDb.Hsapiens.UCSC.hg19.knownGene, 
    +     annotationPackage = 'org.Hs.eg.db')
    Getting TSS and TSE.
    Getting CSS and CSE.
    Getting exons.
    Annotating genes.
    > annotation <- matchGenes(x = genomeRegions$regions, subject = genes)
    > 
    > ## Make the plot
    > plotCluster(idx=1, regions=genomeRegions$regions, annotation=annotation, 
    +     coverageInfo=genomeDataRaw$coverage, groupInfo=genomeInfo$pop, 
    +     txdb=TxDb.Hsapiens.UCSC.hg19.knownGene)
    Warning in getIdeoGR(data) :
      geom(ideogram) need valid seqlengths information for accurate mapping,
                     now use reduced information as ideogram... 
    Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
    Error in rep(startY, each = length(yy)) : 
      attempt to replicate an object of type 'language'
    Calls: plotCluster ... layout_karyogram -> layout_karyogram -> .local -> geom_arch_flip2
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      5: print(e1)
      6: print(e1)
      7: .local(x, ...)
      8: layout_karyogram(obj, cytobands = cytoband, geom = NULL)
      9: layout_karyogram(obj, cytobands = cytoband, geom = NULL)
      10: .local(data, ...)
      11: geom_arch_flip2(df.tri.p2, aes(x = x, y = y, xend = xend, yend = yend, height = height), 
             color = "black", size = 0.5)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 7 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_adv-plotCluster.R#38) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in citation("BiocStyle") :
      no date field in DESCRIPTION file of package 'BiocStyle'
    Warning in citation("scales") :
      no date field in DESCRIPTION file of package 'scales'
    Warning in citation("scales") :
      could not determine year for 'scales' from package DESCRIPTION file
    Warning in citation("biovizBase") :
      no date field in DESCRIPTION file of package 'biovizBase'
    Warning in citation("TxDb.Hsapiens.UCSC.hg19.knownGene") :
      no date field in DESCRIPTION file of package 'TxDb.Hsapiens.UCSC.hg19.knownGene'
    Writing 24 Bibtex entries ... OK
    Results written to file 'derfinderPlotRef.bib'
    Quitting from lines 233-235 (derfinderPlot.Rmd) 
    Error: processing vignette 'derfinderPlot.Rmd' failed with diagnostics:
    subscript contains invalid names
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'RefManageR'
      All declared Imports should be used.
    ```

# DescribeDisplay

Version: 0.2.5

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘DescribeDisplay-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ggplot.scatmat
    > ### Title: Create a nice plots in a scatter plot matrix Create a nice
    > ###   looking plots in a matrix.  The 1d sections along the diagonal have a
    > ###   smooth density while the values are compared to eachother within the
    > ###   matrix.
    > ### Aliases: ggplot.scatmat
    > ### Keywords: hplot
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > print(ggplot(dd_example("scatmat")))
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~tars1", "~col", "~pch".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~tars1", "~col", "~pch".  Please rename the columns or make a new column.variables: "shape" have non standard format: "~tars1", "~col", "~pch".  Please rename the columns or make a new column.
    Calls: print ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             GGally::ggally_densityDiag(data, mapping, ...) + scale_fill_identity()
         }, discrete = "barDiag", na = "naDiag"), sectionAes = structure(list(x = ~tars1, 
             y = ~`NA`, colour = ~col, shape = ~pch), class = "uneval"))
      8: plot_fn(types, sectionAes)
      9: make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name), mapping = sectionAes)
      10: stop(paste("variables: ", paste(shQuote(nonCallNames, type = "cmd"), sep = ", "), 
             " have non standard format: ", paste(shQuote(unlist(mapping[nonCallVals]), type = "cmd"), 
                 collapse = ", "), ".  Please rename the columns or make a new column.", sep = ""))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 0 SKIPPED: 0 FAILED: 1
      1. Error: all basic examples (@test-examples.R#10) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# descriptr

Version: 0.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DESeq2

Version: 1.20.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# destiny

Version: 2.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        doc   4.3Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘SingleCellExperiment’
    A package should be listed in only one of these fields.
    'LinkingTo' for ‘grDevices’ is unused as it has no 'include' directory
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.DPT':
      ‘plot.DPT’
    
    S3 methods shown with full name in documentation object 'plot.DiffusionMap':
      ‘plot.DiffusionMap’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' imports not declared from:
      ‘gridExtra’ ‘viridis’
    'library' or 'require' calls not declared from:
      ‘IRdisplay’ ‘IRkernel’ ‘base64enc’ ‘forcats’ ‘readxl’ ‘repr’
      ‘tidyverse’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'DPT.ipynbmeta' failed with diagnostics:
    Either IPython 3+ or Jupyter has to be installed, but neither could be called.
    Execution halted
    ```

# DEsubs

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# detectRUNS

Version: 0.9.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# detzrcr

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# dfexplore

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    simulate_dataframe : generate_dumb_answer: no visible global function
      definition for ‘rbinom’
    simulate_dataframe : generate_dumb_answer: no visible global function
      definition for ‘rnorm’
    simulate_dataframe : generate_initial: no visible global function
      definition for ‘rbinom’
    simulate_dataframe: no visible binding for global variable ‘randu’
    simulate_dataframe: no visible global function definition for ‘rbinom’
    simulate_dataframe: no visible global function definition for ‘rnorm’
    simulate_dataframe: no visible global function definition for ‘rpois’
    simulate_dataframe : add_NAs: no visible global function definition for
      ‘rbinom’
    Undefined global functions or variables:
      randu rbinom rnorm rpois
    Consider adding
      importFrom("datasets", "randu")
      importFrom("stats", "rbinom", "rnorm", "rpois")
    to your NAMESPACE file.
    ```

# DFIT

Version: 1.0-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dfmeta

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Dforest

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dfped

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dfpk

Version: 3.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        libs   5.3Mb
    ```

# DGCA

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dggridR

Version: 2.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DGM

Version: 1.7.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dgo

Version: 0.2.14

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             if (is.character(x)) {
                 x <- rlang::parse_expr(x)
             }
             new_aesthetic(x, env = caller_env)
         })
      17: FUN(X[[i]], ...)
      18: rlang::parse_expr(x)
      19: abort("More than one expression parsed")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 375 SKIPPED: 1 FAILED: 1
      1. Error: argument validation works (@test-plot.r#33) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DGVM3D

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   4.1Mb
        doc    1.3Mb
    ```

# diagis

Version: 0.1.3-1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > #optimal case
    > set.seed(42)
    > s_opt <- sqrt(2)
    > x_opt <- rnorm(1000, sd = s_opt)
    > w_opt <- p(x_opt) / q(x_opt, s_opt) 
    > weighted_mean(x_opt, w_opt)
    [1] -0.01223051
    > weighted_var(x_opt, w_opt)
    [1] 0.9954861
    > s_inf <- 0.25
    > x_inf <- rnorm(1000, sd = s_inf)
    > w_inf <- p(x_inf) / q(x_inf, s_inf)
    > weighted_mean(x_inf, w_inf) #!!
    [1] 0.1281881
    > weighted_var(x_inf, w_inf) #!!
    [1] 0.2531165
    > # diagnostic plots
    > weight_plot(w_inf)
    Error: Column `y` must be a 1d atomic vector or a list
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 73-74 (diagis.Rmd) 
    Error: processing vignette 'diagis.Rmd' failed with diagnostics:
    Column `y` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DiallelAnalysisR

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# diceR

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# did

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dief

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dielectric

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    L2eV: no visible binding for global variable ‘constants’
    L2w: no visible binding for global variable ‘constants’
    dielectric: no visible global function definition for ‘new’
    dielectric2plot: no visible global function definition for ‘reshape’
    eV2L: no visible binding for global variable ‘constants’
    t2eV: no visible binding for global variable ‘constants’
    Undefined global functions or variables:
      constants new reshape
    Consider adding
      importFrom("methods", "new")
      importFrom("stats", "reshape")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# DiffBind

Version: 2.8.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘XLConnect’
    ```

*   checking R code for possible problems ... NOTE
    ```
    pv.DBAplotVolcano: no visible binding for global variable ‘Fold’
    pv.DBAplotVolcano: no visible binding for global variable ‘Legend’
    Undefined global functions or variables:
      Fold Legend
    ```

# diffcyt

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    calcCounts: no visible binding for global variable ‘cluster_id’
    calcCounts: no visible binding for global variable ‘sample_id’
    calcMedians: no visible binding for global variable ‘cluster_id’
    calcMedians: no visible binding for global variable ‘sample_id’
    calcMedians: no visible binding for global variable ‘value’
    calcMediansByClusterMarker: no visible binding for global variable
      ‘cluster_id’
    calcMediansByClusterMarker: no visible binding for global variable
      ‘marker’
    calcMediansByClusterMarker: no visible binding for global variable
      ‘value’
    calcMediansBySampleMarker: no visible binding for global variable
      ‘sample_id’
    calcMediansBySampleMarker: no visible binding for global variable
      ‘marker’
    calcMediansBySampleMarker: no visible binding for global variable
      ‘value’
    Undefined global functions or variables:
      cluster_id marker sample_id value
    ```

# diffeR

Version: 0.0-4

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘diffeR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: MAD
    > ### Title: Mean Absolute Deviation (MAD)
    > ### Aliases: MAD
    > ### Keywords: spatial
    > 
    > ### ** Examples
    > 
    > old.par <- par(no.readonly = TRUE)
    > grid1 <- raster(system.file("external/GRID1_INT.rst", package="diffeR"))
    > grid2 <- raster(system.file("external/GRID2_INT.rst", package="diffeR"))
    > strata <- raster(system.file("external/strata_int.rst", package="diffeR"))
    > MAD(grid1, grid2, strata, eval="original")
    Error in calc(strata, fun) : unused argument (fun)
    Calls: MAD
    Execution halted
    ```

*   checking whether package ‘diffeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'calc(strata, fun)': unused argument (fun) 
      Note: possible error in 'calc(strataM2, fun1)': unused argument (fun1) 
      Note: possible error in 'calc(tmp1, fun2)': unused argument (fun2) 
      Note: possible error in 'calc(strataM, sum)': unused argument (sum) 
      Warning: replacing previous import ‘raster::calc’ by ‘ggplot2::calc’ when loading ‘diffeR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/diffeR/new/diffeR.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking R code for possible problems ... NOTE
    ```
    MAD: possible error in calc(strata, fun): unused argument (fun)
    MAD: possible error in calc(strataM2, fun1): unused argument (fun1)
    MAD: possible error in calc(tmp1, fun2): unused argument (fun2)
    MAD: possible error in calc(strataM, sum): unused argument (sum)
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# diffloop

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# diffuStats

Version: 0.104.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Using supplied kernel matrix...
    Using supplied kernel matrix...
    X1: permuting scores...
    Permuting...
    X1: computing heatRank...
    All done
    Using supplied kernel matrix...
    X1: permuting scores...
    Permuting...
    X1: computing heatRank...
    All done
    Using supplied kernel matrix...
    All done
    Warning in wilcox.test.default(x = perf_mat[, met1], y = perf_mat[, met2],  :
      cannot compute exact p-value with zeroes
    Warning in wilcox.test.default(x = perf_mat[, met1], y = perf_mat[, met2],  :
      cannot compute exact confidence interval with zeroes
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'diffuStats.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# difNLR

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dimRed

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DirectEffects

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# directlabels

Version: 2017.03.31

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: direct.label
    > ### Title: Direct labels for color decoding
    > ### Aliases: direct.label
    > 
    > ### ** Examples
    > 
    > if(require(ggplot2)){
    +   ## Add direct labels to a ggplot2 scatterplot, making sure that each
    +   ## label is close to its point cloud, and doesn't overlap points or
    +   ## other labels.
    +   scatter <- qplot(jitter(hwy),jitter(cty),data=mpg,colour=class,
    +                    main="Fuel efficiency depends on car size")
    +   print(direct.label(scatter))
    + }
    Loading required package: ggplot2
    Error in parse(text = x) : <text>:1:2: unexpected ';'
    1: ~;
         ^
    Calls: print ... lapply -> FUN -> <Anonymous> -> parse_exprs -> parse
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/ggplot.R’ failed.
    Last 13 lines of output:
      +       stopifnot(L$colour == result$colour)
      +       ## label it and check for different legends.
      +       dl <- direct.label(L$plot)
      +       after <- legends2hide(dl)
      +       stopifnot(is.null(after))
      +     }
      +   }
      + }
      Loading required package: ggplot2
      [1] "qplot"
      Error in parse(text = x) : <text>:1:2: unexpected ';'
      1: ~;
           ^
      Calls: direct.label ... lapply -> FUN -> <Anonymous> -> parse_exprs -> parse
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dirichletprocess

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dirichletprocess.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# disclapmix

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# diveRsity

Version: 1.9.90

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'diveRsity.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# DiversityOccupancy

Version: 1.0.6

## In both

*   checking whether package ‘DiversityOccupancy’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/DiversityOccupancy/new/DiversityOccupancy.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘DiversityOccupancy’ ...
** package ‘DiversityOccupancy’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘DiversityOccupancy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/DiversityOccupancy/new/DiversityOccupancy.Rcheck/DiversityOccupancy’

```
### CRAN

```
* installing *source* package ‘DiversityOccupancy’ ...
** package ‘DiversityOccupancy’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/DiversityOccupancy/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘DiversityOccupancy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/DiversityOccupancy/old/DiversityOccupancy.Rcheck/DiversityOccupancy’

```
# DLMtool

Version: 5.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dlookr

Version: 0.3.0

## Newly broken

*   checking whether package ‘dlookr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘dlookr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dlookr/new/dlookr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘randomForest’
      All declared Imports should be used.
    ```

# dlstats

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dMod

Version: 0.4

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ## Do many fits from random positions and store them into parlist
    > out <- as.parlist(lapply(1:50, function(i) {
    +   trust(normL2(data, g*x), pars + rnorm(length(pars), 0, 1), rinit = 1, rmax = 10)
    + }))
    > 
    > summary(out)
    Results of the best and worst fit
          index      value converged iterations         a
    best     25   3.591833      TRUE          9 0.9891171
    worst    17 454.623420      TRUE         16 2.3633414
    
    Statistics of fit outcome
    Fits aborted:       0
    Fits not converged: 0
    Fits converged:     50
    Fits total:         50 [50]> 
    > ## Reduce parlist to parframe
    > parframe <- as.parframe(out)
    > plotValues(parframe)
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        profile
    
    jakstat.c:7:15: warning: unused variable 'cons' [-Wunused-variable]
    static double cons[0];
                  ^
    jakstat.c:8:15: warning: unused variable 'range' [-Wunused-variable]
    static double range[2];
                  ^
    2 warnings generated.
    jakstat_s.c:7:15: warning: unused variable 'cons' [-Wunused-variable]
    static double cons[0];
                  ^
    jakstat_s.c:8:15: warning: unused variable 'range' [-Wunused-variable]
    static double range[2];
                  ^
    2 warnings generated.
    Quitting from lines 211-217 (dMod.Rmd) 
    Error: processing vignette 'dMod.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dmrseq

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > 
    > # get annotation information for hg19
    > annoTrack <- getAnnot('hg19')
    updating metadata: retrieving 1 resource
    snapshotDate(): 2018-04-23
    Building CpG islands...
    downloading 0 resources
    loading from cache 
        ‘/Users/max//.AnnotationHub/5086’
    Building CpG shores...
    Building CpG shelves...
    Building inter-CpG-islands...
    Download of CpG annotation successful!
    Download of Gene annotation failed. 4 attempts left
    Download of Gene annotation failed. 3 attempts left
    Download of Gene annotation failed. 2 attempts left
    Download of Gene annotation failed. 1 attempts left
    Download of Gene annotation failed. 0 attempts left
    Error in getAnnot("hg19") : Annotation could not be retrieved.
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Assuming file type is cov
    Genome hg18 will be built by lifting over hg19 annotations from annotatr
    snapshotDate(): 2018-04-23
    Building CpG islands...
    downloading 0 resources
    loading from cache 
        '/Users/max//.AnnotationHub/5086'
    Building CpG shores...
    Building CpG shelves...
    Building inter-CpG-islands...
    Download of CpG annotation successful!
    Download of Gene annotation failed. 4 attempts left
    Download of Gene annotation failed. 3 attempts left
    Download of Gene annotation failed. 2 attempts left
    Download of Gene annotation failed. 1 attempts left
    Download of Gene annotation failed. 0 attempts left
    Quitting from lines 538-543 (dmrseq.Rmd) 
    Error: processing vignette 'dmrseq.Rmd' failed with diagnostics:
    Annotation could not be retrieved.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.6Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
        doc    1.3Mb
    ```

# doBy

Version: 4.6-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'linest-lsmeans.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# docxtools

Version: 0.1.1

## Newly broken

*   checking whether package ‘docxtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘docxtools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/docxtools/new/docxtools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dodgr

Version: 0.0.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Doscheda

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data             2.1Mb
        doc              1.6Mb
        shiny-examples   2.0Mb
    ```

# DOSE

Version: 3.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    2.7Mb
    ```

# DoTC

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dotwhisker

Version: 0.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dpcR

Version: 0.4

## In both

*   checking whether package ‘dpcR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dpcR/new/dpcR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘dpcR’ ...
** package ‘dpcR’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘dpcR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dpcR/new/dpcR.Rcheck/dpcR’

```
### CRAN

```
* installing *source* package ‘dpcR’ ...
** package ‘dpcR’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘dpcR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dpcR/old/dpcR.Rcheck/dpcR’

```
# dplyr

Version: 0.7.4

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      3: eval_bare(get_expr(quo), get_env(quo))
      4: summarise(gdf, out = 1:2)
      5: summarise.tbl_df(gdf, out = 1:2)
      6: summarise_impl(.data, dots)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2670 SKIPPED: 6 FAILED: 3
      1. Error: mutate() supports unquoted values (@test-mutate.r#721) 
      2. Failure: summarise() supports unquoted values (@test-summarise.r#980) 
      3. Error: summarise() supports unquoted values (@test-summarise.r#985) 
      
      Error: testthat unit tests failed
      In addition: Warning message:
      call dbDisconnect() when finished working with a connection 
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4 marked UTF-8 strings
    ```

# dr4pl

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# drake

Version: 5.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# drawProteins

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DrBats

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# driftR

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘readr’
      All declared Imports should be used.
    ```

# DRIMSeq

Version: 1.8.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    3 FBgn0259735   0.9383577  2 6.255157e-01 7.744480e-01
    4 FBgn0032785   3.4718633  2 1.762359e-01 3.143951e-01
    5 FBgn0040297   6.0769622  4 1.934739e-01 3.143951e-01
    6 FBgn0032979   1.7139224  1 1.904773e-01 3.143951e-01
    > 
    > ## Plot feature proportions for a top DTU gene
    > res <- results(d)
    > res <- res[order(res$pvalue, decreasing = FALSE), ]
    > 
    > top_gene_id <- res$gene_id[1]
    > 
    > plotProportions(d, gene_id = top_gene_id, group_variable = "group")
    > 
    > plotProportions(d, gene_id = top_gene_id, group_variable = "group", 
    +   plot_type = "lineplot")
    > 
    > plotProportions(d, gene_id = top_gene_id, group_variable = "group", 
    +   plot_type = "ribbonplot")
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Error: Columns `ymin`, `ymax` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Quitting from lines 369-379 (DRIMSeq.Rnw) 
    Error: processing vignette 'DRIMSeq.Rnw' failed with diagnostics:
    Columns `ymin`, `ymax` must be 1d atomic vectors or lists
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# drLumi

Version: 0.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'drLumi.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# drugCombo

Version: 1.0.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
                 x <- rlang::parse_expr(x)
             }
             new_aesthetic(x, env = caller_env)
         })
      16: FUN(X[[i]], ...)
      17: rlang::parse_expr(x)
      18: abort("More than one expression parsed")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 66 SKIPPED: 0 FAILED: 2
      1. Error: plots work for tauSurface (@test-tauSurface.R#145) 
      2. Error: plots work for tauSurface without CI (@test-tauSurface.R#167) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Using constraints: b = 1
    Using constraints: b = 1
    Using constraints: b = 1
    Using constraints: b = 1
    Using constraints: b = 1, h2 = 3.494
    Running default monotherapy fitting. Note that if constraints are needed, run mono fitting manually first
    Using constraints: b = 1
    Using constraints: b = 1
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 408-410 (userGuide.Rmd) 
    Error: processing vignette 'userGuide.Rmd' failed with diagnostics:
    More than one expression parsed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dSimer

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dslabs

Version: 0.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dslice

Version: 1.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    economics               US economic time series
    economics_long          US economic time series
    faithfuld               2d density estimate of Old Faithful data
    luv_colours             'colors()' in Luv space
    midwest                 Midwest demographics
    mpg                     Fuel economy data from 1999 and 2008 for 38
                            popular models of car
    msleep                  An updated and expanded version of the mammals
                            sleep dataset
    presidential            Terms of 11 presidents from Eisenhower to Obama
    seals                   Vector field of seal movements
    txhousing               Housing sales in TX
    
    
    Use ‘data(package = .packages(all.available = TRUE))’
    to list the data sets in all *available* packages.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dslice.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# dsm

Version: 2.2.15

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DstarM

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DTR

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DTRlearn

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dtwclust

Version: 5.3.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: TSclust
    Loading required package: wmtsa
    Loading required package: pdc
    Loading required package: cluster
    Loading required package: TSdist
    Loaded TSdist v3.4. See ?TSdist for help, citation("TSdist") for use in publication.
    
    Warning in tsclust(CharTraj[1L:20L], type = "h", k = 4L, distance = "dtw_basic",  :
      Distance matrix is not symmetric, and hierarchical clustering assumes it is (it ignores the upper triangular).
    Loading required package: doParallel
    Loading required package: foreach
    Loading required package: iterators
    Loading required package: clue
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dtwclust.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# dtwSat

Version: 0.2.3

## Newly broken

*   checking whether package ‘dtwSat’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘dtwSat’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dtwSat/new/dtwSat.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Attaching package: 'caret'
    
    The following object is masked from 'package:survival':
    
        cluster
    
    This is BibTeX, Version 0.99d (TeX Live 2017)
    The top-level auxiliary file: applying_twdtw.aux
    The style file: jss.bst
    Illegal, another \bibstyle command---line 155 of file applying_twdtw.aux
     : \bibstyle
     :          {jss}
    I'm skipping whatever remains of this command
    Database file #1: references.bib.bib
    Warning--entry type for "Dutrieux:2014" isn't style-file defined
    --line 584 of file references.bib.bib
    (There was 1 error message)
    Error: processing vignette 'applying_twdtw.Rmd' failed with diagnostics:
    Failed to build the bibliography via bibtex
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DVHmetrics

Version: 0.3.7

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in getVolume(pm$valRefNum, type = "absolute", unitRef = pm$unitRef,  :
      max dose is less than requested dose
    Warning in getVolume(pm$valRefNum, type = "absolute", unitRef = pm$unitRef,  :
      max dose is less than requested dose
    Warning in getVolume(pm$valRefNum, type = "absolute", unitRef = pm$unitRef,  :
      max dose is less than requested dose
    Warning in getVolume(pm$valRefNum, type = "absolute", unitRef = pm$unitRef,  :
      max dose is less than requested dose
    Warning in getEQD2.default(D = D$dvh[, "dose"], fd = fd, ab = ab) :
      'D' must be > 0
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'DVHmetrics.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dynfrail

Version: 0.5.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# DynNom

Version: 4.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dynOmics

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dynr

Version: 0.1.12-5

## In both

*   checking whether package ‘dynr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/new/dynr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘dynr’ ...
** package ‘dynr’ successfully unpacked and MD5 sums checked
checking for gsl-config... no
configure: error: gsl-config not found, is GSL installed?
ERROR: configuration failed for package ‘dynr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/new/dynr.Rcheck/dynr’

```
### CRAN

```
* installing *source* package ‘dynr’ ...
** package ‘dynr’ successfully unpacked and MD5 sums checked
checking for gsl-config... no
configure: error: gsl-config not found, is GSL installed?
ERROR: configuration failed for package ‘dynr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/dynr/old/dynr.Rcheck/dynr’

```
# dynsim

Version: 1.2.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Lagging invest by 1 time units.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dynsim-overview.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# dynsurv

Version: 0.3-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# earlywarnings

Version: 1.0.59

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    surrogates_RShiny: no visible global function definition for ‘par’
    surrogates_RShiny: no visible global function definition for ‘hist’
    surrogates_RShiny: no visible global function definition for ‘abline’
    surrogates_RShiny: no visible global function definition for ‘points’
    surrogates_RShiny: no visible global function definition for ‘title’
    surrogates_ews: no visible global function definition for ‘dev.new’
    surrogates_ews: no visible global function definition for ‘par’
    surrogates_ews: no visible global function definition for ‘hist’
    surrogates_ews: no visible global function definition for ‘abline’
    surrogates_ews: no visible global function definition for ‘points’
    surrogates_ews: no visible global function definition for ‘mtext’
    Undefined global functions or variables:
      abline aes contour dev.new geom_tile grid hist labs layout legend
      lines mtext par plot points rainbow scale_fill_gradient stat_contour
      text title topo.colors xlab ylab
    Consider adding
      importFrom("grDevices", "dev.new", "rainbow", "topo.colors")
      importFrom("graphics", "abline", "contour", "grid", "hist", "layout",
                 "legend", "lines", "mtext", "par", "plot", "points", "text",
                 "title")
    to your NAMESPACE file.
    ```

# earthtones

Version: 0.1.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 26-28 (earthtones.rmd) 
    Error: processing vignette 'earthtones.rmd' failed with diagnostics:
    cannot open URL 'http://maps.googleapis.com/maps/api/staticmap?center=36.094994,-111.837962&zoom=12&size=640x640&scale=2&maptype=satellite&language=en-EN&sensor=false'
    Execution halted
    ```

## Newly fixed

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      2: quasi_label(enquo(object), label)
      3: eval_bare(get_expr(quo), get_env(quo))
      4: get_earthtones(method = "kmeans")
      5: ggmap::get_map(location = c(longitude, latitude), maptype = "satellite", zoom = zoom, 
             ...)
      6: get_googlemap(center = location, zoom = zoom, maptype = maptype, scale = scale, messaging = messaging, 
             urlonly = urlonly, force = force, filename = filename, color = color, language = language)
      7: download.file(url, destfile = tmp, quiet = !messaging, mode = "wb")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 4 SKIPPED: 0 FAILED: 1
      1. Error: returns normal (@test-earth.R#6) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EasyHTMLReport

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘reshape2’ ‘scales’ ‘xtable’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘base64enc’ ‘knitr’ ‘markdown’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .file_attachment: no visible global function definition for
      ‘base64encode’
    .plot_attachment: no visible global function definition for ‘dev.off’
    .update_list: no visible global function definition for ‘modifyList’
    easyHtmlReport: no visible global function definition for ‘knit’
    easyHtmlReport: no visible global function definition for
      ‘markdownToHTML’
    mime_part.data.frame: no visible global function definition for
      ‘write.table’
    mime_part.ggplot: no visible binding for global variable ‘pdf’
    mime_part.matrix: no visible global function definition for
      ‘write.table’
    mime_part.trellis: no visible binding for global variable ‘pdf’
    simpleHtmlReport : <anonymous>: no visible global function definition
      for ‘write.table’
    Undefined global functions or variables:
      base64encode dev.off knit markdownToHTML modifyList pdf write.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("utils", "modifyList", "write.table")
    to your NAMESPACE file.
    ```

# EbayesThresh

Version: 1.4-12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ecb

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ecd

Version: 0.9.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# echor

Version: 0.1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ecoengine

Version: 1.11.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# EcoGenetics

Version: 1.2.1-4

## Newly broken

*   checking whether package ‘EcoGenetics’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘EcoGenetics’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/EcoGenetics/new/EcoGenetics.Rcheck/00install.out’ for details.
    ```

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      Warning message:
      replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘EcoGenetics’ 
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ecolottery

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EcoNetGen

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ecotox

Version: 1.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ecr

Version: 2.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EDA

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eda4treeR

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dae’ ‘dplyr’
      All declared Imports should be used.
    ```

# edarf

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# edeaR

Version: 0.8.0

## Newly broken

*   checking whether package ‘edeaR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘edeaR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/edeaR/new/edeaR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# edfReader

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# edgar

Version: 1.0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# edgarWebR

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# edge

Version: 2.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    '/Users/max/github/ggplot2/revdep/checks.noindex/edge/new/edge.Rcheck/edge/doc':
    
    edge: edge Package (source, pdf)
    
    Warning: code chunks must not depend on the uncached chunk "gibson_import_data"
    Warning: code chunks must not depend on the uncached chunk "gibson_fit_models"
    Warning: code chunks must not depend on the uncached chunk "kidney_import_data"
    Warning: code chunks must not depend on the uncached chunk "kidney_fit_models"
    Warning: code chunks must not depend on the uncached chunk "kidney_extract"
    Warning: code chunks must not depend on the uncached chunk "endotoxin_import_data"
    Warning: code chunks must not depend on the uncached chunk "endotoxin_build_models"
    Warning: code chunks must not depend on the uncached chunk "endotoxin_fit_models"
    `geom_smooth()` using method = 'loess' and formula 'y ~ x'
    Warning: code chunks must not depend on the uncached chunk "endotoxin_sig"
    Warning: code chunks must not depend on the uncached chunk "endotoxin_extract"
    Warning: code chunks must not depend on the uncached chunk "sva"
    Warning: code chunks must not depend on the uncached chunk "kidneyModel"
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'edge.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    fitFDist: no visible global function definition for ‘median’
    fitFDist: no visible global function definition for ‘lm.fit’
    fitFDist: no visible global function definition for ‘predict’
    fit_wmodels: no visible global function definition for ‘model.matrix’
    fit_wmodels: no visible global function definition for ‘lm.wfit’
    null: no visible global function definition for ‘model.matrix’
    apply_sva,deSet: no visible global function definition for ‘as.formula’
    apply_sva,deSet: no visible global function definition for ‘terms’
    fit_models,deSet: no visible global function definition for
      ‘model.matrix’
    fullModel<-,deSet: no visible global function definition for
      ‘model.matrix’
    lrt,deSet-deFit: no visible global function definition for ‘pf’
    nullModel<-,deSet: no visible global function definition for
      ‘model.matrix’
    Undefined global functions or variables:
      as.formula lm.fit lm.wfit median model.matrix pf predict terms
    Consider adding
      importFrom("stats", "as.formula", "lm.fit", "lm.wfit", "median",
                 "model.matrix", "pf", "predict", "terms")
    to your NAMESPACE file.
    ```

# edstan

Version: 1.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eechidna

Version: 1.1

## Newly broken

*   checking whether package ‘eechidna’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘eechidna’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/eechidna/new/eechidna.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        discard
    
    
    Attaching package: 'gridExtra'
    
    The following object is masked from 'package:dplyr':
    
        combine
    
    
    Attaching package: 'GGally'
    
    The following object is masked from 'package:dplyr':
    
        nasa
    
    Quitting from lines 277-294 (exploring-election-data.Rmd) 
    Error: processing vignette 'exploring-election-data.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~Prop_Labour".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
        doc    1.2Mb
    ```

# eegc

Version: 1.5.0

## In both

*   checking whether package ‘eegc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/eegc/new/eegc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘eegc’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘plot’ is not exported by 'namespace:clusterProfiler'
ERROR: lazy loading failed for package ‘eegc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/eegc/new/eegc.Rcheck/eegc’

```
### CRAN

```
* installing *source* package ‘eegc’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘plot’ is not exported by 'namespace:clusterProfiler'
ERROR: lazy loading failed for package ‘eegc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/eegc/old/eegc.Rcheck/eegc’

```
# EEM

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eemR

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eeptools

Version: 1.1.1

## Newly broken

*   checking whether package ‘eeptools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::syms’ by ‘memisc::syms’ when loading ‘eeptools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/eeptools/new/eeptools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eesim

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EFDR

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .gdf : find_loss: no visible global function definition for ‘rnorm’
    .p.values : <anonymous>: no visible global function definition for
      ‘pnorm’
    .relist.dwt: no visible global function definition for ‘relist’
    .relist.dwt: no visible global function definition for ‘as’
    .std.wav.coeff : <anonymous>: no visible global function definition for
      ‘mad’
    regrid: no visible global function definition for ‘predict’
    regrid: no visible global function definition for ‘var’
    regrid: no visible global function definition for ‘medpolish’
    Undefined global functions or variables:
      as mad medpolish pnorm predict relist rnorm var
    Consider adding
      importFrom("methods", "as")
      importFrom("stats", "mad", "medpolish", "pnorm", "predict", "rnorm",
                 "var")
      importFrom("utils", "relist")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# effectFusion

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EffectLiteR

Version: 0.4-2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘EffectLiteR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: conditionalEffectsPlot
    > ### Title: Plot conditional effects
    > ### Aliases: conditionalEffectsPlot
    > 
    > ### ** Examples
    > 
    > m1 <- effectLite(y="dv", x="x", k="k1", z="z1", control="control", data=example01)
    > conditionalEffectsPlot(m1, zsel="z1", gxsel="g1", colour="k1")
    Error in geom_point() : could not find function "geom_point"
    Calls: conditionalEffectsPlot -> <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# effectR

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ega

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# egcm

Version: 1.0.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# egg

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EGSEA

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking R code for possible problems ... NOTE
    ```
    buildGeneSetDBIdx: no visible binding for global variable ‘GOTERM’
    buildMSigDBIdx: no visible binding for global variable ‘msigdb’
    combinePvalues : <anonymous>: no visible global function definition for
      ‘votep’
    generateSummaryPlots: no visible binding for global variable ‘x.data’
    generateSummaryPlots: no visible binding for global variable ‘y.data’
    generateSummaryPlots: no visible binding for global variable ‘gsSize’
    generateSummaryPlots: no visible binding for global variable ‘id’
    generateSummaryPlots: no visible binding for global variable ‘sig’
    loadKeggData: no visible binding for global variable ‘kegg.pathways’
    Undefined global functions or variables:
      GOTERM gsSize id kegg.pathways msigdb sig votep x.data y.data
    ```

# eiCompare

Version: 2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ELMER

Version: 2.4.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 36.4Mb
      sub-directories of 1Mb or more:
        doc  36.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': 'TCGAbiolinks:::TCGAVisualize_volcano'
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      'TCGAbiolinks:::colDataPrepare' 'TCGAbiolinks:::get.GRCh.bioMart'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for 'fisher.test'
    heatmapGene: no visible global function definition for 'precede'
    heatmapGene: no visible global function definition for
      'subsetByOverlaps'
    heatmapGene: no visible global function definition for 'resize'
    motif.enrichment.plot: no visible binding for global variable 'y'
    motif.enrichment.plot: no visible binding for global variable 'x'
    motif.enrichment.plot: no visible binding for global variable 'z'
    motif.enrichment.plot: no visible binding for global variable 'upperOR'
    motif.enrichment.plot: no visible binding for global variable 'lowerOR'
    motif.enrichment.plot: no visible binding for global variable 'motif'
    motif.enrichment.plot: no visible binding for global variable 'OR'
    scatter: no visible binding for global variable 'value'
    scatter: no visible global function definition for 'cor.test'
    Undefined global functions or variables:
      Gene GeneID Hugo_Symbol OR Probe Target cor.test fisher.test label
      lowerOR motif precede pvalue resize subsetByOverlaps upperOR value x
      y z
    Consider adding
      importFrom("stats", "cor.test", "fisher.test")
    to your NAMESPACE file.
    ```

# EMAtools

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# emdi

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘graphics’ ‘maptools’ ‘rgeos’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 24 marked Latin-1 strings
    ```

# EMDomics

Version: 2.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Sadhika Malladi <contact@sadhikamalladi.com> [aut, cre]
      Daniel Schmolze <emd@schmolze.com> [aut, cre]
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    calculate_cvm : <anonymous>: no visible global function definition for
      ‘median’
    calculate_cvm_gene: no visible global function definition for ‘combn’
    calculate_emd: no visible global function definition for ‘combn’
    calculate_emd : <anonymous>: no visible global function definition for
      ‘median’
    calculate_emd_gene: no visible global function definition for ‘combn’
    calculate_ks: no visible global function definition for ‘combn’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘p.adjust’
    calculate_ks : <anonymous>: no visible global function definition for
      ‘median’
    calculate_ks_gene: no visible global function definition for ‘combn’
    calculate_ks_gene: no visible global function definition for ‘ks.test’
    Undefined global functions or variables:
      combn hist ks.test median p.adjust
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "ks.test", "median", "p.adjust")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# emil

Version: 2.2.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# emmeans

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘CARBayes’ ‘coxme’ ‘gee’ ‘glmmADMB’ ‘MCMCglmm’ ‘MCMCpack’
    ```

# EMMIXgene

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# emojifont

Version: 0.5.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ggtree v1.12.0  For help: https://guangchuangyu.github.io/software/ggtree
    
    If you use ggtree in published research, please cite:
    Guangchuang Yu, David Smith, Huachen Zhu, Yi Guan, Tommy Tsan-Yuk Lam. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 2017, 8(1):28-36, doi:10.1111/2041-210X.12628
    Quitting from lines 46-54 (phylomoji.Rmd) 
    Error: processing vignette 'phylomoji.Rmd' failed with diagnostics:
    <text>:1:25: unexpected ')'
    1: suppressMessages(emoji(~)
                                ^
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        emoji_fonts   6.4Mb
    ```

# EmpiricalCalibration

Version: 1.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# emuR

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# enetLTS

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > xout[y==0,][1:m,] <- xout[1:m,] + 10;          # class 0
    > yout <- y                                      # wrong classification for vertical outliers
    > 
    > ## Don't show: 
    > set.seed(86)
    > n <- 5; p <- 15                            
    > beta <- rep(0,p); beta[1:6] <- 1              
    > sigma <- 0.5                        
    > x <- matrix(rnorm(n*p, sigma),nrow=n)   
    > e <- rnorm(n,0,1)                             # error terms
    > eps <- 0.1                                    # contamination level
    > m <- ceiling(eps*n)                           # observations to be contaminated
    > eout <- e; eout[1:m] <- eout[1:m] + 10        # vertical outliers
    > yout <- c(x %*% beta + sigma * eout)          # response
    > xout <- x; xout[1:m,] <- xout[1:m,] + 10      # bad leverage points
    > fit2 <- enetLTS(xout,yout,alphas=0.5,lambdas=0.05,plot=FALSE)
    Warning: Option grouped=FALSE enforced in cv.glmnet, since < 3 observations per fold
    > plot(fit2,method="resid",vers="raw")
    Press <Enter> to continue...
    Error: Column `y` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# enpls

Version: 5.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# enrichplot

Version: 1.0.0

## In both

*   R CMD check timed out
    

# enrichwith

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘betareg’
    ```

# EnsCat

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# envalysis

Version: 0.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EnvStats

Version: 2.3.0

## In both

*   checking whether package ‘EnvStats’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/EnvStats/new/EnvStats.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.3Mb
      sub-directories of 1Mb or more:
        R      3.0Mb
        doc    3.3Mb
        help   4.3Mb
    ```

# EpiCurve

Version: 2.1-1

## Newly broken

*   checking whether package ‘EpiCurve’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘EpiCurve’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/EpiCurve/new/EpiCurve.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EpiDynamics

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EpiModel

Version: 1.6.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# episensr

Version: 0.9.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘llogistic’ ‘logitnorm’
      All declared Imports should be used.
    ```

# episheet

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# erccdashboard

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    dynRangePlot: no visible binding for global variable ‘value’
    dynRangePlot: no visible binding for global variable ‘Rep’
    saveERCCPlots: no visible global function definition for ‘par’
    testDECount: no visible binding for global variable ‘Atrend’
    testDECount: no visible binding for global variable ‘Dispersion’
    testDECount: no visible binding for global variable ‘Ratio’
    Undefined global functions or variables:
      Atrend Dispersion Ratio Rep par value
    Consider adding
      importFrom("graphics", "par")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: gridExtra
    Warning: Ignoring unknown aesthetics: y
    Warning: Transformation introduced infinite values in continuous y-axis
    Warning: Ignoring unknown aesthetics: y
    Warning: Transformation introduced infinite values in continuous y-axis
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'erccdashboard.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# erma

Version: 0.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 200.6Mb
      sub-directories of 1Mb or more:
        bed_tabix  161.3Mb
        data        37.1Mb
        doc          1.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    geneTxRange: no visible global function definition for ‘select’
    genemodelOLD: no visible binding for global variable ‘exonsBy’
    stateProfile: no visible binding for global variable ‘states_25’
    stateProfile: no visible binding for global variable ‘short_celltype’
    stateProfile: no visible binding for global variable ‘name’
    statesByRange: no visible binding for global variable ‘mod’
    statesByRange: no visible binding for global variable ‘upstream’
    statesByRange: no visible binding for global variable ‘downstream’
    statesByRange : <anonymous>: no visible binding for global variable
      ‘tss’
    statesByRange: no visible binding for global variable ‘states_25’
    Undefined global functions or variables:
      downstream exonsBy mod name select short_celltype states_25 tss
      upstream
    ```

# ERSA

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘modelr’
      All declared Imports should be used.
    ```

# esATAC

Version: 1.2.1

## In both

*   checking whether package ‘esATAC’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/esATAC/new/esATAC.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

## Installation

### Devel

```
* installing *source* package ‘esATAC’ ...
** libs
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c BedLine.cpp -o BedLine.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c BedUtils.cpp -o BedUtils.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c ChrDivi.cpp -o ChrDivi.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c CutCountPre.cpp -o CutCountPre.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c CutSiteCount.cpp -o CutSiteCount.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c LibComplexQC.cpp -o LibComplexQC.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c SortBed.cpp -o SortBed.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c rcpp_wrapper.cpp -o rcpp_wrapper.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c renamer.cpp -o renamer.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c sam2bed.cc -o sam2bed.o
sam2bed.cc:111:16: warning: unused variable 'xspm' [-Wunused-variable]
    regmatch_t xspm[1];
               ^
sam2bed.cc:280:16: warning: unused variable 'xspm' [-Wunused-variable]
    regmatch_t xspm[1];
               ^
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o esATAC.so BedLine.o BedUtils.o ChrDivi.o CutCountPre.o CutSiteCount.o LibComplexQC.o RcppExports.o SortBed.o rcpp_wrapper.o renamer.o sam2bed.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/esATAC/new/esATAC.Rcheck/esATAC/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘esATAC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/esATAC/new/esATAC.Rcheck/esATAC’

```
### CRAN

```
* installing *source* package ‘esATAC’ ...
** libs
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c BedLine.cpp -o BedLine.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c BedUtils.cpp -o BedUtils.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c ChrDivi.cpp -o ChrDivi.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c CutCountPre.cpp -o CutCountPre.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c CutSiteCount.cpp -o CutSiteCount.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c LibComplexQC.cpp -o LibComplexQC.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c SortBed.cpp -o SortBed.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c rcpp_wrapper.cpp -o rcpp_wrapper.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c renamer.cpp -o renamer.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -DNDEBUG -DPLF_SYS_LINUX  -DR_EVN_FLAG -I"/Users/max/github/ggplot2/revdep/library.noindex/esATAC/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2 -c sam2bed.cc -o sam2bed.o
sam2bed.cc:111:16: warning: unused variable 'xspm' [-Wunused-variable]
    regmatch_t xspm[1];
               ^
sam2bed.cc:280:16: warning: unused variable 'xspm' [-Wunused-variable]
    regmatch_t xspm[1];
               ^
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o esATAC.so BedLine.o BedUtils.o ChrDivi.o CutCountPre.o CutSiteCount.o LibComplexQC.o RcppExports.o SortBed.o rcpp_wrapper.o renamer.o sam2bed.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/esATAC/old/esATAC.Rcheck/esATAC/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/esATAC/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘esATAC’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/esATAC/old/esATAC.Rcheck/esATAC’

```
# esetVis

Version: 1.6.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        expand.grid
    
    Loading required package: org.Hs.eg.db
    
    
    'select()' returned 1:many mapping between keys and columns
    Loading required package: MASS
    
    Attaching package: 'MASS'
    
    The following object is masked from 'package:AnnotationDbi':
    
        select
    
    Warning: Removed 1 rows containing missing values (geom_hex).
    Warning: Removed 5 rows containing missing values (geom_point).
    Quitting from lines 260-269 (esetVis-vignette.Rmd) 
    Error: processing vignette 'esetVis-vignette.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   4.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    getMethodsInputObjectEsetVis: no visible binding for global variable
      ‘rowData’
    getMethodsInputObjectEsetVis: no visible binding for global variable
      ‘colData’
    getMethodsInputObjectEsetVis: no visible binding for global variable
      ‘assay’
    getMethodsInputObjectEsetVis : <anonymous>: no visible global function
      definition for ‘colData’
    getMethodsInputObjectEsetVis : <anonymous>: no visible global function
      definition for ‘rowData’
    ggvisPlotEset: no visible global function definition for ‘:=’
    ggvisPlotEset: no visible binding for global variable ‘fill’
    ggvisPlotEset: no visible binding for global variable ‘shape’
    ggvisPlotEset: no visible binding for global variable ‘size’
    Undefined global functions or variables:
      := assay colData fill rowData shape size
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘a4Base’
    ```

# ESGtoolkit

Version: 0.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following object is masked from 'package:base':
    
        as.list
    
    Loading required package: timeDate
    Loading required package: timeSeries
    Loading required package: fBasics
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/<example_SVJD_2" -> "figure/_example_SVJD_2"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/<example_SVJD_4" -> "figure/_example_SVJD_4"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/<example_SVJD_cvS0" -> "figure/_example_SVJD_cvS0"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/<example_SVJD_martingale_2" -> "figure/_example_SVJD_martingale_2"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/<example_pricing" -> "figure/_example_pricing"
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ESG2toolkit_Intro_062014.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    esgplotts: no visible global function definition for ‘deltat’
    esgplotts: no visible global function definition for ‘xlab’
    esgplotts: no visible global function definition for ‘ylab’
    esgplotts: no visible global function definition for ‘theme’
    simdiff: no visible global function definition for ‘ts’
    simshocks: no visible global function definition for ‘ts’
    simshocks: no visible global function definition for ‘qnorm’
    simshocks : <anonymous>: no visible global function definition for ‘ts’
    Undefined global functions or variables:
      abline aes colorRampPalette coord_flip cor cor.test deltat
      element_blank end geom_density geom_point is.ts lines matplot par
      plot points polygon pt qnorm qt quantile scale_color_manual
      scale_fill_manual sd start t.test theme time ts tsp window xlab ylab
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "lines", "matplot", "par", "plot",
                 "points", "polygon")
      importFrom("stats", "cor", "cor.test", "deltat", "end", "is.ts", "pt",
                 "qnorm", "qt", "quantile", "sd", "start", "t.test", "time",
                 "ts", "tsp", "window")
    to your NAMESPACE file.
    ```

# esmisc

Version: 0.0.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘esmisc-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: theme_edi
    > ### Title: Custom ggplot2 theme
    > ### Aliases: theme_edi
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > p <- ggplot(mtcars) + 
    +  geom_point(aes(x = wt, y = mpg, 
    + colour=factor(gear))) + facet_wrap(~am)
    > p
    > p + theme_edi()
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

*   checking whether package ‘esmisc’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘esmisc’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/esmisc/new/esmisc.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ESTER

Version: 0.2.0

## Newly broken

*   checking whether package ‘ESTER’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ESTER’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ESTER/new/ESTER.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# etl

Version: 0.3.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# etm

Version: 0.6-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.clos.etm: no visible global function definition for ‘plot’
    plot.clos.etm: no visible global function definition for ‘axis’
    plot.clos.etm: no visible global function definition for ‘box’
    plot.clos.etm: no visible global function definition for ‘lines’
    plot.clos.etm: no visible global function definition for ‘close.screen’
    plot.etm: no visible global function definition for ‘plot’
    plot.etm: no visible global function definition for ‘lines’
    plot.etmCIF: no visible global function definition for ‘plot’
    plot.etmCIF: no visible global function definition for ‘lines’
    plot.etmCIF: no visible global function definition for ‘segments’
    print.summary.etm: no visible global function definition for ‘quantile’
    print.summary.etmCIF: no visible global function definition for
      ‘quantile’
    Undefined global functions or variables:
      axis box close.screen lines model.extract par plot qnorm quantile
      screen segments split.screen terms
    Consider adding
      importFrom("graphics", "axis", "box", "close.screen", "lines", "par",
                 "plot", "screen", "segments", "split.screen")
      importFrom("stats", "model.extract", "qnorm", "quantile", "terms")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: etm
    Loading required package: survival
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'etmCIF_tutorial.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# europepmc

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eurostat

Version: 3.1.5

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      trying URL 'http://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?sort=1&file=data%2Ftsdtr210.tsv.gz'
      downloaded 17 KB
      
      ── 1. Error: (unknown) (@test-all.R#112)  ──────────────────────────────────────
      tsdtr210 does not exist or is not readable
      1: get_eurostat("tsdtr210", type = "label", keepFlags = T, cache = FALSE) at testthat/test-all.R:112
      2: get_eurostat_raw(id)
      3: stop(id, " does not exist or is not readable")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 0 SKIPPED: 14 FAILED: 1
      1. Error: (unknown) (@test-all.R#112) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        vars
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    trying URL 'http://ec.europa.eu/eurostat/estat-navtree-portlet-prod/BulkDownloadListing?sort=1&file=data%2Ften00081.tsv.gz'
    Content type 'application/octet-stream;charset=UTF-8' length 13491 bytes (13 KB)
    ==================================================
    downloaded 13 KB
    
    Table ten00081 cached at /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpWbZ5Pl/eurostat/ten00081_date_code_TF.rds
    Quitting from lines 291-309 (eurostat_tutorial.Rmd) 
    Error: processing vignette 'eurostat_tutorial.Rmd' failed with diagnostics:
    package or namespace load failed for 'tmap' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
     there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# evaluate

Version: 0.10.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# evaluator

Version: 0.2.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘evaluator-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: generate_event_outcomes_plot
    > ### Title: Display the distribution of threat events contained vs. realized
    > ###   across all domains
    > ### Aliases: generate_event_outcomes_plot
    > 
    > ### ** Examples
    > 
    > data(simulation_results)
    > data(domains)
    > dat <- calculate_weak_domains(simulation_results, domains)
    > generate_event_outcomes_plot(dat)
    Error in if (label.size == 0) NA else row$colour : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... labelGrob -> gTree -> gpar -> validGP -> numnotnull
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             lineheight = row$lineheight), rect.gp = gpar(col = if (label.size == 0) NA else row$colour, 
             fill = alpha(row$fill, row$alpha), lwd = label.size * .pt))
      50: gTree(label = label, x = x, y = y, just = just, padding = padding, r = r, name = name, 
             text.gp = text.gp, rect.gp = rect.gp, vp = vp, cl = "labelgrob")
      51: gpar(col = if (label.size == 0) NA else row$colour, fill = alpha(row$fill, row$alpha), 
             lwd = label.size * .pt)
      52: validGP(list(...))
      53: numnotnull("fontsize")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 84 SKIPPED: 3 FAILED: 1
      1. Error: Analyze report renders (@test-reports.R#33) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking whether package ‘evaluator’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘evaluator’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/evaluator/new/evaluator.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EValue

Version: 1.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# EventStudy

Version: 0.34

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘curl’ ‘openxlsx’ ‘stringr’
      All declared Imports should be used.
    ```

# evolqg

Version: 0.2-5

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: LModularity
    > 
    > ### ** Examples
    > 
    > # A modular matrix:
    > modules = matrix(c(rep(c(1, 0, 0), each = 5),
    + rep(c(0, 1, 0), each = 5),
    + rep(c(0, 0, 1), each = 5)), 15)
    > cor.hypot = CreateHypotMatrix(modules)[[4]]
    > hypot.mask = matrix(as.logical(cor.hypot), 15, 15)
    > mod.cor = matrix(NA, 15, 15)
    > mod.cor[ hypot.mask] = runif(length(mod.cor[ hypot.mask]), 0.8, 0.9) # within-modules
    > mod.cor[!hypot.mask] = runif(length(mod.cor[!hypot.mask]), 0.3, 0.4) # between-modules
    > diag(mod.cor) = 1
    > mod.cor = (mod.cor + t(mod.cor))/2 # correlation matrices should be symmetric
    > 
    > LModularity(mod.cor)
    Error in method(g, ...) : 
      At optimal_modularity.c:85 : GLPK is not available, Unimplemented function call
    Calls: LModularity -> method
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Loading required package: plyr
      ── 1. Error: LModularity resturn resonable results (@test.LModularity.r#12)  ───
      At optimal_modularity.c:85 : GLPK is not available, Unimplemented function call
      1: expect_equal(modules, LModularity(mod.cor)[[2]]) at testthat/test.LModularity.r:12
      2: quasi_label(enquo(expected), expected.label)
      3: eval_bare(get_expr(quo), get_env(quo))
      4: LModularity(mod.cor)
      5: method(g, ...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 20953 SKIPPED: 1 FAILED: 1
      1. Error: LModularity resturn resonable results (@test.LModularity.r#12) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# EvolutionaryGames

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Evomorph

Version: 0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# evoper

Version: 0.4.0

## In both

*   checking whether package ‘evoper’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/evoper/new/evoper.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘evoper’ ...
** package ‘evoper’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘evoper’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/evoper/new/evoper.Rcheck/evoper’

```
### CRAN

```
* installing *source* package ‘evoper’ ...
** package ‘evoper’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/evoper/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘evoper’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/evoper/old/evoper.Rcheck/evoper’

```
# ewoc

Version: 0.2.0

## In both

*   checking whether package ‘ewoc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/new/ewoc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ewoc’ ...
** package ‘ewoc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘ewoc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/new/ewoc.Rcheck/ewoc’

```
### CRAN

```
* installing *source* package ‘ewoc’ ...
** package ‘ewoc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ewoc/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘ewoc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ewoc/old/ewoc.Rcheck/ewoc’

```
# explor

Version: 0.3.3

## Newly broken

*   checking whether package ‘explor’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘explor’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/explor/new/explor.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# exploreR

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# expss

Version: 0.8.6

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Use 'expss_output_rnotebook()' to display tables inside R Notebooks.
     To return to the console output, use 'expss_output_default()'.
    
    Quitting from lines 57-64 (labels-support.Rmd) 
    Error: processing vignette 'labels-support.Rmd' failed with diagnostics:
    `data` must be a data frame, or other object coercible by `fortify()`, not an S3 object with class quosures
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# exreport

Version: 0.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# extracat

Version: 1.7-4

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Name: facetshade
    > ### Title: facetshade
    > ### Aliases: facetshade
    > 
    > ### ** Examples
    > 
    > 
    > # produces a modified data.frame mdata and returns:
    > # ggplot(data = mdata, mapping, ... ) + facet_grid(f)
    > 
    > require(scales)
    Loading required package: scales
    > require(ggplot2)
    Loading required package: ggplot2
    > 
    > # facetshade object:
    > fs1 <- facetshade( data = olives, aes(x = palmitoleic, y = oleic),
    + 	f = .~Region )
    Error in `[.data.frame`(data, xn) : undefined columns selected
    Calls: facetshade -> [ -> [.data.frame
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eyelinker

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# eyetrackingR

Version: 0.1.7

## Newly broken

*   checking whether package ‘eyetrackingR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘eyetrackingR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/eyetrackingR/new/eyetrackingR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ez

Version: 4.4-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ezsim

Version: 0.5.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    BugReports field is not a suitable URL but contains an email address
      which will be used as from R 3.4.0
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.ezsim: no visible global function definition for ‘tail’
    plot.ezsim: no visible global function definition for ‘plot’
    plot.ezsim: no visible binding for global variable ‘dnorm’
    plot.ezsim: no visible binding for global variable ‘pdf’
    plot.ezsim: no visible global function definition for ‘dev.off’
    plot.ezsim : <anonymous>: no visible global function definition for
      ‘dev.new’
    plot.summary.ezsim: no visible global function definition for ‘head’
    plot.summary.ezsim: no visible global function definition for ‘tail’
    plot.summary.ezsim: no visible binding for global variable ‘pdf’
    plot.summary.ezsim: no visible global function definition for ‘dev.off’
    plot.summary.ezsim : <anonymous>: no visible global function definition
      for ‘dev.new’
    Undefined global functions or variables:
      as.formula dev.new dev.off dnorm head pdf plot quantile runif tail
    Consider adding
      importFrom("grDevices", "dev.new", "dev.off", "pdf")
      importFrom("graphics", "plot")
      importFrom("stats", "as.formula", "dnorm", "quantile", "runif")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Warning: The labeller API has been updated. Labellers taking `variable`and `value` arguments are now deprecated. See labellers documentation.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# facopy

Version: 1.13.0

## In both

*   checking whether package ‘facopy’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/new/facopy.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘cgdsr’ ‘coin’ ‘ggplot2’ ‘gridExtra’ ‘facopy.annot’ ‘grid’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

## Installation

### Devel

```
* installing *source* package ‘facopy’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘setting.graph.attributes’ is not exported by 'namespace:DOSE'
ERROR: lazy loading failed for package ‘facopy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/new/facopy.Rcheck/facopy’

```
### CRAN

```
* installing *source* package ‘facopy’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : object ‘setting.graph.attributes’ is not exported by 'namespace:DOSE'
ERROR: lazy loading failed for package ‘facopy’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/facopy/old/facopy.Rcheck/facopy’

```
# FactoClass

Version: 1.2.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# factoextra

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘NbClust’
    ```

# factorMerger

Version: 0.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘forcats’ ‘formula.tools’
      All declared Imports should be used.
    ```

# FAOSTAT

Version: 2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FAOSTAT.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Aggregation : foo: no visible global function definition for
      ‘weighted.mean’
    Aggregation: no visible global function definition for ‘txtProgressBar’
    Aggregation: no visible global function definition for
      ‘setTxtProgressBar’
    constructSYB: no visible global function definition for ‘str’
    fillCountryCode: no visible global function definition for ‘na.omit’
    getFAO: no visible global function definition for ‘read.csv’
    getWDImetaData: no visible global function definition for ‘write.csv’
    getWDItoSYB: no visible global function definition for ‘na.omit’
    lsgr: no visible global function definition for ‘na.omit’
    lsgr: no visible global function definition for ‘coef’
    lsgr: no visible global function definition for ‘lm’
    Undefined global functions or variables:
      coef lm na.omit read.csv setTxtProgressBar str txtProgressBar
      weighted.mean write.csv
    Consider adding
      importFrom("stats", "coef", "lm", "na.omit", "weighted.mean")
      importFrom("utils", "read.csv", "setTxtProgressBar", "str",
                 "txtProgressBar", "write.csv")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 179 marked UTF-8 strings
    ```

# fastqcr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fastR2

Version: 0.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +   group_by(airport, airline) %>%
    +   mutate(total = sum(count), percent = count/total * 100) %>% 
    +   filter(result == "Delayed") 
    > AirlineArrival3 <- 
    +   AirlineArrival %>% 
    +   group_by(airline, result) %>% 
    +   summarise(count = n()) %>%
    +   group_by(airline) %>%
    +   mutate(total = sum(count), percent = count/total * 100) %>% 
    +   filter(result == "Delayed") 
    >   gf_line(percent ~ airport, color = ~ airline, group = ~ airline, 
    +           data = AirlineArrival2) %>%
    +     gf_point(percent ~ airport, color = ~ airline, size = ~total, 
    +              data = AirlineArrival2) %>%
    +     gf_hline(yintercept = ~ percent, color = ~airline, 
    +              data = AirlineArrival3, linetype = "dashed") %>%
    +     gf_labs(y = "percent delayed") 
    Don't know how to automatically pick scale for object of type formula. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type formula. Defaulting to continuous.
    Error: Aesthetics must be either length 1 or the same as the data (10): group, colour, x, y
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘magrittr’
      All declared Imports should be used.
    ```

# fbroc

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fCCAC

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    fccac: no visible binding for global variable ‘variables’
    Undefined global functions or variables:
      variables
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        mapply, match, mget, order, paste, pmax,
        pmax.int, pmin, pmin.int, rank, rbind, rowMeans,
        rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max,
        which.min
    
    
    Attaching package: ‘S4Vectors’
    
    The following object is masked from ‘package:base’:
    
        expand.grid
    
    Loading required package: IRanges
    Loading required package: GenomicRanges
    Loading required package: GenomeInfoDb
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fCCAC.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# fcm

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fdANOVA

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fdANOVA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# FDboost

Version: 0.3-1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Help for individual functions is also obtained by adding the
    suffix '.spam' to the function name, e.g. 'help( chol.spam)'.
    
    Attaching package: 'spam'
    
    The following object is masked from 'package:Matrix':
    
        det
    
    The following objects are masked from 'package:base':
    
        backsolve, forwardsolve
    
    Loading required package: maps
    See www.image.ucar.edu/~nychka/Fields for
     a vignette and other supplements. 
    Loading required package: mapdata
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FLAM_canada.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fdq

Version: 0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Fgmutils’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# fence

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fergm

Version: 0.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Name: coef_posterior_density
    > ### Title: Plots the posterior density for FERGM model terms.
    > ### Aliases: coef_posterior_density
    > ### Keywords: FERGM interpret summary
    > 
    > ### ** Examples
    > 
    > # load example data
    > data("ergm.fit")
    > data("fergm.fit")
    > data("mesa")
    > 
    > # You can also look at the density of particular variables
    >   # using the following built in rstan functions:
    > plot(fergm.fit$stan.fit, par = "beta")
    ci_level: 0.8 (80% intervals)
    outer_level: 0.95 (95% intervals)
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fermicatsR

Version: 1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fetchR

Version: 2.1-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FField

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘gridExtra’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    FFieldPtRepDemo: no visible global function definition for ‘ggplot’
    FFieldPtRepDemo: no visible binding for global variable ‘mtcars’
    FFieldPtRepDemo: no visible global function definition for ‘aes’
    FFieldPtRepDemo: no visible binding for global variable ‘mpg’
    FFieldPtRepDemo: no visible global function definition for ‘geom_point’
    FFieldPtRepDemo: no visible global function definition for ‘geom_text’
    FFieldPtRepDemo: no visible global function definition for ‘ggtitle’
    FFieldPtRepDemo: no visible global function definition for
      ‘geom_segment’
    FFieldPtRepDemo: no visible global function definition for
      ‘grid.arrange’
    Undefined global functions or variables:
      aes geom_point geom_segment geom_text ggplot ggtitle grid.arrange mpg
      mtcars
    Consider adding
      importFrom("datasets", "mtcars")
    to your NAMESPACE file.
    ```

# Fgmutils

Version: 0.9.4

## In both

*   checking whether package ‘Fgmutils’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/new/Fgmutils.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘Fgmutils’ ...
** package ‘Fgmutils’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘Fgmutils’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/new/Fgmutils.Rcheck/Fgmutils’

```
### CRAN

```
* installing *source* package ‘Fgmutils’ ...
** package ‘Fgmutils’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/Fgmutils/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘Fgmutils’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/Fgmutils/old/Fgmutils.Rcheck/Fgmutils’

```
# fgsea

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fiftystater

Version: 1.0.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 83-87 (fiftystater.Rmd) 
    Error: processing vignette 'fiftystater.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FinCal

Version: 0.6.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FindIt

Version: 1.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FindMyFriends

Version: 1.10.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Functions or methods with usage in documentation object 'pgVirtual-class' but not in code:
      as
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘gtable:::insert.unit’ ‘gtable:::z_arrange_gtables’
      See the note in ?`:::` about the use of this operator.
    ```

# findviews

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fingerPro

Version: 1.0

## In both

*   checking whether package ‘fingerPro’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [RcppExports.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/new/fingerPro.Rcheck/fingerPro’

```
### CRAN

```
* installing *source* package ‘fingerPro’ ...
** package ‘fingerPro’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c fingerprinting.cpp -o fingerprinting.o
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [RcppExports.o] Error 1
make: *** Waiting for unfinished jobs....
In file included from fingerprinting.cpp:5:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSL.h:25:
/Users/max/github/ggplot2/revdep/library.noindex/fingerPro/RcppGSL/include/RcppGSLForward.h:26:10: fatal error: 'gsl/gsl_vector.h' file not found
#include <gsl/gsl_vector.h> 
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [fingerprinting.o] Error 1
ERROR: compilation failed for package ‘fingerPro’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/fingerPro/old/fingerPro.Rcheck/fingerPro’

```
# fingertipsR

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# fishmove

Version: 0.3-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    fishmove : coefs: no visible global function definition for ‘coef’
    fishmove : coefs: no visible global function definition for ‘pf’
    fishmove: no visible global function definition for ‘lm’
    fishmove: no visible global function definition for ‘predict.lm’
    fishmove.estimate: no visible global function definition for ‘quantile’
    fishmove.estimate : ddoublenorm: no visible global function definition
      for ‘dnorm’
    fishmove.query: no visible global function definition for ‘hasArg’
    fishmove.query : f: no visible global function definition for ‘pnorm’
    fishmove.query: no visible global function definition for ‘uniroot’
    pdk: no visible global function definition for ‘hasArg’
    pdk : eq: no visible global function definition for ‘dnorm’
    Undefined global functions or variables:
      coef dnorm hasArg lm pf pnorm predict.lm quantile uniroot
    Consider adding
      importFrom("methods", "hasArg")
      importFrom("stats", "coef", "dnorm", "lm", "pf", "pnorm", "predict.lm",
                 "quantile", "uniroot")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# fitbitScraper

Version: 0.1.8

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 22-24 (fitbitScraper-examples.Rmd) 
    Error: processing vignette 'fitbitScraper-examples.Rmd' failed with diagnostics:
    login failed
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fitdistrplus

Version: 1.0-9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      NaNs produced
    Warning in dgamma(c(5.18363400559485e-11, 0.00101165181456365, 8.55450755319305e-22,  :
      NaNs produced
    Warning in dgamma(c(5.18363400559485e-11, 0.00101165181456365, 8.55450755319305e-22,  :
      NaNs produced
    Warning in dgamma(c(5.18363400559485e-11, 0.00101165181456365, 8.55450755319305e-22,  :
      NaNs produced
    Loading required package: knitr
    Loading required package: rgenoud
    ##  rgenoud (Version 5.8-2.0, Build Date: 2018-04-03)
    ##  See http://sekhon.berkeley.edu/rgenoud for additional documentation.
    ##  Please cite software as:
    ##   Walter Mebane, Jr. and Jasjeet S. Sekhon. 2011.
    ##   ``Genetic Optimization Using Derivatives: The rgenoud package for R.''
    ##   Journal of Statistical Software, 42(11): 1-26. 
    ##
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paper2JSS.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# fitur

Version: 0.5.25

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# flacco

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# flexrsurv

Version: 1.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FLightR

Version: 0.4.6

## Newly fixed

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
         estimating directions
         estimating mean directions and kappas
         estimating kappas
         estimating mean dists
         estimating probs of migration
         estimating median dists
         creating output DONE!
      DONE!
      bird likely did not move, exiting without result
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 29 SKIPPED: 0 FAILED: 1
      1. Error: map_flightr_ggmap_works (@test_data_result_summary_and_plotting.R#36) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    ```

# flippant

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘multipanelfigure’
    ```

# flowAI

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘knitr’
    A package should be listed in only one of these fields.
    ```

# flowCHIC

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        extdata   5.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘as.dist’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘par’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘plot’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘hclust’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘points’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘text’
    plot_nmds,data.frame-data.frame: no visible global function definition
      for ‘legend’
    Undefined global functions or variables:
      as.dist count dev.off hclust legend par plot png points text
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "legend", "par", "plot", "points", "text")
      importFrom("stats", "as.dist", "hclust")
    to your NAMESPACE file.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘flowViz’, ‘flowCyBar’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning in metaMDS(mat.dist, autotransform = FALSE) :
      stress is (nearly) zero: you may have insufficient data
    Warning in metaMDS(mat.dist, autotransform = FALSE) :
      stress is (nearly) zero: you may have insufficient data
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'flowCHICmanual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# flowDiv

Version: 2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# flowr

Version: 0.9.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# flowTime

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fontMPlus

Version: 0.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 56-67 (Using_fontMPlus.Rmd) 
    Error: processing vignette 'Using_fontMPlus.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# forcats

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forecast

Version: 8.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    [103] 1149.00291 2406.42135 2700.01698 2977.40311 1471.74813  412.10540
    [109]  260.63972  621.74204 1005.94427 1865.28886 2894.83953 3236.72817
    
    $testset
     [1]   1   4   8  31  34  35  39  41  50  51  56  62  68  83  84  93  96  98 105
    [20] 108 109 112 113
    
    > 
    > library(ggplot2)
    
    Attaching package: ‘ggplot2’
    
    The following object is masked from ‘package:forecast’:
    
        autolayer
    
    > autoplot(lynx, series="Data") +
    +   autolayer(modelcv$testfit, series="Fits") +
    +   autolayer(modelcv$residuals, series="Residuals")
    Error: Objects of type ts not supported by autolayer.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘seasonal’, ‘forecTheta’
    ```

# ForecastComb

Version: 1.1

## Newly broken

*   checking whether package ‘ForecastComb’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘ForecastComb’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ForecastComb/new/ForecastComb.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ForecastCombinations’
    ```

# ForecastFramework

Version: 0.9.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Warning in private$.dnames[[1]][rows] = rownames(data) :
      number of items to replace is not a multiple of replacement length
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ClassDiagram.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forecastHybrid

Version: 2.2.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# foreSIGHT

Version: 0.9.2

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ###Example 1
    > # Saved from a previous example
    > # tank_simpleScale_plot<-performanceSpaces(data=tank_simple_scenarios,
    > #                                          plotTag = "Heat",
    > #                                          plotArgs=plotArgs,
    > #                                          systemModel = tankWrapper,
    > #                                          systemArgs = systemArgs)
    > 
    > #Create plotting arguments
    > plotArgs<-list(title="Scenario neutral space with projections overlaid",
    +                ylim=c(0.7,1.3),
    +                xlim=c(-2,2))
    > 
    > #Plot performance space with projections overlaid
    > tank_overlay_plot=plotLayers(plot=tank_simpleScale_plot$plotEdit,
    +                              plotArgs=plotArgs,
    +                              climdata=climdata,
    +                              climArgs=NULL)
    Error in FUN(X[[i]], ...) : object 'df1' not found
    Calls: plotLayers ... ggplot_build.ggplot -> by_layer -> f -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 185-213 (Vignette.Rmd) 
    Error: processing vignette 'Vignette.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forestinventory

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forestmodel

Version: 0.5.0

## Newly broken

*   checking whether package ‘forestmodel’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘forestmodel’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/forestmodel/new/forestmodel.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forestr

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# forwards

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FourCSeq

Version: 1.14.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Name: plotDifferences
    > ### Title: Plot differences
    > ### Aliases: plotDifferences
    > 
    > ### ** Examples
    > 
    > data(fcf, package="FourCSeq")
    > 
    > fcf <- getDifferences(fcf, referenceCondition="WE_68h")
    [1] "ap"
    found already estimated dispersions, replacing these
    gene-wise dispersion estimates
    mean-dispersion relationship
    final dispersion estimates
    > 
    > plotDifferences(fcf)
    [1] "ap"
    Error in min(c(x, xmin), na.rm = TRUE) : 
      invalid 'type' (list) of argument
    Calls: plotDifferences ... <Anonymous> -> getLimits -> getLimits -> .local -> .getLimits
    Execution halted
    ```

## In both

*   checking whether package ‘FourCSeq’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/FourCSeq/new/FourCSeq.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:base':
    
        aperm, apply
    
    Loading required package: splines
    Loading required package: LSD
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    gene-wise dispersion estimates
    mean-dispersion relationship
    final dispersion estimates
    Loading required package: GenomicFeatures
    Loading required package: AnnotationDbi
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Quitting from lines 534-538 (FourCSeq.Rnw) 
    Error: processing vignette 'FourCSeq.Rnw' failed with diagnostics:
    invalid 'type' (list) of argument
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘DESeq2’ ‘GenomicRanges’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      'ggplot2' 'methods'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      'keepSeqlevels'
    updateObject,FourC: no visible global function definition for
      'callNextMethod'
    Undefined global functions or variables:
      DataFrame IRanges Rle Seqinfo SimpleList abline aes axis blues9
      callNextMethod change count differentialInteraction element_blank fit
      fitDown fitUp formula geom_path geom_point ggplot keepSeqlevels labs
      legend mad median metadata mid mtext new p.adjust par peak plot pnorm
      points predict rel relevel scale_fill_gradient2 scale_y_continuous
      seqlengths seqlevels seqlevels<- subjectHits subsetByOverlaps theme
      theme_bw theme_set write.table
    Consider adding
      importFrom("grDevices", "blues9")
      importFrom("graphics", "abline", "axis", "legend", "mtext", "par",
                 "plot", "points")
      importFrom("methods", "callNextMethod", "new")
      importFrom("stats", "formula", "mad", "median", "p.adjust", "pnorm",
                 "predict", "relevel")
      importFrom("utils", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# fourierin

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fpp2

Version: 2.3

## Newly broken

*   checking whether package ‘fpp2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘fpp2’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/fpp2/new/fpp2.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fractional

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# frailtyEM

Version: 0.8.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: survival
    Loading required package: gridExtra
    Warning: Removed 2 rows containing missing values (geom_path).
    Warning in emfrail(Surv(time, status) ~ age + sex + cluster(id), data = kidney,  :
      Hessian virtually 0; frailty variance might be at the edge of the parameter space.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'frailtyEM_manual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# frailtySurv

Version: 1.3.2

## In both

*   checking whether package ‘frailtySurv’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      hcubature.c:923:12: warning: comparison of unsigned enum expression < 0 is always false [-Wtautological-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/frailtySurv/new/frailtySurv.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FREddyPro

Version: 1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > 
    > ## Load the data
    > data(fluxes)
    > 
    > ## Clean fluxes
    > fluxes=cleanFluxes(fluxes,sdCor=TRUE,sdTimes=3,timesList=3,distCor=TRUE,
    +                    thresholdList=list(H=c(-100,1000),LE=c(-100,1000)))	
    > 
    > ## Remove some line as an example
    > fluxes=fluxes[-c(832,833,834,835,840,953),]
    > 
    > ## Create timestamp from date and time
    > fluxes=createTimestamp(fluxes)
    > 
    > ## Now fill the gap timestmaps with NA's
    > fluxes=fillTimestamp(fluxes,flux=TRUE)
    [1] Discontinuity in 3 instances
    [1] 2014-06-14 03:30:00 UTC 2014-06-14 06:00:00 UTC 2014-06-16 14:30:00 UTC
    Error in FUN(X[[i]], ...) : subscript out of bounds
    Calls: fillTimestamp ... shift_time -> [[ -> [[.POSIXlt -> .POSIXlt -> lapply
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# freqparcoord

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FreqProf

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# frequencyConnectedness

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# freqweights

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FRK

Version: 0.2.1

## Newly broken

*   checking whether package ‘FRK’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘FRK’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/FRK/new/FRK.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    
    Attaching package: 'dplyr'
    
    The following object is masked from 'package:ggplot2':
    
        vars
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Note: show_basis assumes spherical distance functions when plotting
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FRK_intro.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘dggrids’ ‘INLA’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.1Mb
      sub-directories of 1Mb or more:
        data   5.4Mb
        doc    1.8Mb
    ```

# frontiles

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    alphafrontier.2d: no visible global function definition for ‘par’
    alphafrontier.2d: no visible global function definition for ‘polygon’
    alphafrontier.2d: no visible global function definition for ‘lines’
    alphafrontier.2d: no visible global function definition for ‘points’
    alphafrontier.3d: no visible global function definition for ‘layout’
    alphafrontier.3d: no visible global function definition for ‘barplot’
    alphafrontier.3d: no visible global function definition for ‘axis’
    alphafrontier.3d: no visible global function definition for ‘lines’
    alphafrontier.3d: no visible global function definition for ‘na.omit’
    ordermfrontier.2d: no visible global function definition for ‘par’
    ordermfrontier.2d: no visible global function definition for ‘polygon’
    ordermfrontier.2d: no visible global function definition for ‘lines’
    ordermfrontier.2d: no visible global function definition for ‘points’
    ordermscore.boot: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      axis barplot layout legend lines mtext na.omit par points polygon sd
    Consider adding
      importFrom("graphics", "axis", "barplot", "layout", "legend", "lines",
                 "mtext", "par", "points", "polygon")
      importFrom("stats", "na.omit", "sd")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: classInt
    Loading required package: spData
    To access larger datasets in this package, install the
    spDataLarge package with: `install.packages('spDataLarge',
    repos='https://nowosad.github.io/drat/', type='source'))`
    Loading required package: colorspace
    Loading required package: rgl
    Loading required package: sp
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'frontiles1.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# FSelectorRcpp

Version: 0.1.8

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             else stop(msg, call. = FALSE, domain = NA)
         })
      3: tryCatchList(expr, classes, parentenv, handlers)
      4: tryCatchOne(expr, names, parentenv, handlers[[1L]])
      5: value[[3L]](cond)
      6: stop(msg, call. = FALSE, domain = NA)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 22 SKIPPED: 0 FAILED: 3
      1. Error: (unknown) (@test-cutoff.R#2) 
      2. Error: (unknown) (@test-discretize.R#2) 
      3. Error: (unknown) (@test-information_gain.R#2) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RTCGA.rnaseq’
    ```

# fSRM

Version: 0.6.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# FunciSNP

Version: 1.24.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    bedColors: no visible global function definition for 'col2rgb'
    bedColors: no visible global function definition for 'png'
    bedColors: no visible global function definition for 'box'
    bedColors: no visible global function definition for 'par'
    bedColors: no visible global function definition for 'axis'
    bedColors: no visible global function definition for 'dev.off'
    Undefined global functions or variables:
      R.squared TSS.human.GRCh37 abline as.dendrogram axis box col2rgb
      colorRampPalette data dev.off dist distance.from.tag fisher.test
      hclust lincRNA order.dendrogram p.adjust par pdf plot png r.2
      read.delim read.table refseqgenes sig text timestamp value variable
      write.table
    Consider adding
      importFrom("grDevices", "col2rgb", "colorRampPalette", "dev.off",
                 "pdf", "png")
      importFrom("graphics", "abline", "axis", "box", "par", "plot", "text")
      importFrom("stats", "as.dendrogram", "dist", "fisher.test", "hclust",
                 "order.dendrogram", "p.adjust")
      importFrom("utils", "data", "read.delim", "read.table", "timestamp",
                 "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning: Removed 1427 rows containing non-finite values (stat_bin).
    Saving 7 x 7 in image
    Warning: Removed 1427 rows containing non-finite values (stat_bin).
    
    
    See FunciSNP/plots/ folder in /Users/max/github/ggplot2/revdep/checks.noindex/FunciSNP/new/FunciSNP.Rcheck/vign_test/FunciSNP/vignettes for all plots.
    
    
    ####
    Bed file "FunciSNP_results_rsq.0.22.bed" created successfully.
    (See folder: "/Users/max/github/ggplot2/revdep/checks.noindex/FunciSNP/new/FunciSNP.Rcheck/vign_test/FunciSNP/vignettes")
    
    To view results, submit bed file as a
    custom track in UCSC Genome Browser (genome.ucsc.edu), 
    
    Now have fun with your new YAFSNPss!!
    ####
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'FunciSNP_vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# funData

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# funModeling

Version: 1.6.7

## Newly broken

*   checking whether package ‘funModeling’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘funModeling’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/funModeling/new/funModeling.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# funnelR

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# funrar

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# furrowSeg

Version: 1.8.0

## In both

*   checking whether package ‘furrowSeg’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘EBImage::abind’ by ‘abind::abind’ when loading ‘furrowSeg’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/furrowSeg/new/furrowSeg.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'exampleFurrowSegmentation.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 349.5Mb
      sub-directories of 1Mb or more:
        data  348.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    identifyTimeMinArea: no visible global function definition for ‘points’
    identifyTimeMinArea: no visible global function definition for ‘abline’
    plotFeatureEvolution: no visible global function definition for ‘par’
    plotFeatureEvolution: no visible binding for global variable ‘median’
    plotFeatureEvolution: no visible binding for global variable ‘quantile’
    plotFeatureEvolution: no visible global function definition for ‘plot’
    plotFeatureEvolution: no visible global function definition for
      ‘polygon’
    plotFeatureEvolution: no visible global function definition for ‘rgb’
    plotFeatureEvolution: no visible global function definition for ‘axis’
    plotFeatureEvolution: no visible global function definition for ‘mtext’
    plotFeatureEvolution: no visible global function definition for ‘title’
    Undefined global functions or variables:
      abline axis median mtext par plot points polygon predict quantile rgb
      title
    Consider adding
      importFrom("grDevices", "rgb")
      importFrom("graphics", "abline", "axis", "mtext", "par", "plot",
                 "points", "polygon", "title")
      importFrom("stats", "median", "predict", "quantile")
    to your NAMESPACE file.
    ```

# fuser

Version: 1.0.0

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'rBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# futureheatwaves

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fuzzyforest

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# fuzzyjoin

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# G2Sd

Version: 2.1.5

## In both

*   checking whether package ‘G2Sd’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/G2Sd/new/G2Sd.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘G2Sd’ ...
** package ‘G2Sd’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘G2Sd’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/G2Sd/new/G2Sd.Rcheck/G2Sd’

```
### CRAN

```
* installing *source* package ‘G2Sd’ ...
** package ‘G2Sd’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/G2Sd/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘G2Sd’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/G2Sd/old/G2Sd.Rcheck/G2Sd’

```
# GADMTools

Version: 2.1-1

## Newly broken

*   checking whether package ‘GADMTools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘GADMTools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GADMTools/new/GADMTools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scales’
      All declared Imports should be used.
    ```

# gaiah

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘maptools’ ‘rgeos’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# gapfill

Version: 0.9.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘doMPI’
    ```

# gapmap

Version: 0.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gapminder

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GARS

Version: 1.0.0

## In both

*   checking whether package ‘GARS’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GARS/new/GARS.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘GARS’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘GARS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GARS/new/GARS.Rcheck/GARS’

```
### CRAN

```
* installing *source* package ‘GARS’ ...
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/GARS/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘GARS’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/GARS/old/GARS.Rcheck/GARS’

```
# gastempt

Version: 0.4.01

## Newly broken

*   checking whether package ‘gastempt’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘gastempt’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gastempt/new/gastempt.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        libs   6.6Mb
    ```

# GCalignR

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gcatest

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'gcatest.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    gcat: no visible global function definition for ‘pchisq’
    gcat.stat: no visible global function definition for ‘complete.cases’
    gcatest: no visible global function definition for ‘pchisq’
    Undefined global functions or variables:
      complete.cases pchisq
    Consider adding
      importFrom("stats", "complete.cases", "pchisq")
    to your NAMESPACE file.
    ```

# gcerisk

Version: 18.02.22

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gcookbook

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gCrisprTools

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.2Mb
      sub-directories of 1Mb or more:
        data   2.3Mb
        doc    1.1Mb
    ```

# GD

Version: 1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GDAdata

Version: 0.93

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GDCRNATools

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > metaMatrix <- data.frame(sample_type=rep('PrimaryTumor',6),
    +                         sample=samples,
    +                         days_to_death=seq(100,600,100),
    +                         days_to_last_follow_up=rep(NA,6))
    > rnaExpr <- matrix(c(2.7,7.0,4.9,6.9,4.6,2.5,
    +                     0.5,2.5,5.7,6.5,4.9,3.8,
    +                     2.1,2.9,5.9,5.7,4.5,3.5,
    +                     2.7,5.9,4.5,5.8,5.2,3.0,
    +                     2.5,2.2,5.3,4.4,4.4,2.9,
    +                     2.4,3.8,6.2,3.8,3.8,4.2),6,6)
    > rownames(rnaExpr) <- genes
    > colnames(rnaExpr) <- samples
    > gdcKMPlot(gene='ENSG00000000938', rna.expr=rnaExpr, 
    +     metadata=metaMatrix, sep='median')
    Warning in gdcKMPlot(gene = "ENSG00000000938", rna.expr = rnaExpr, metadata = metaMatrix,  :
      Incompatible methods ("+.ggsurv", "+.gg") for "+"
    Error in ggsurvplot(fit, data = survDa, pval = FALSE, pval.coord = c(2200,  : 
      non-numeric argument to binary operator
    Calls: gdcKMPlot
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'GDCRNATools.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        R      3.2Mb
        data   1.1Mb
        doc    2.1Mb
    ```

*   checking top-level files ... NOTE
    ```
    File
      LICENSE
    is not mentioned in the DESCRIPTION file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    gdcRNADownload: no visible global function definition for ‘write.table’
    gdcRNAMerge : <anonymous>: no visible global function definition for
      ‘read.table’
    gdcRNAMerge: no visible global function definition for ‘read.table’
    hyperTestFun: no visible global function definition for ‘phyper’
    kmTestFun: no visible global function definition for ‘pchisq’
    kmTestFun: no visible global function definition for ‘qnorm’
    manifestDownloadFun: no visible global function definition for
      ‘read.table’
    mirCorTestFun: no visible global function definition for ‘cor.test’
    multiRegFun: no visible global function definition for ‘cor.test’
    Undefined global functions or variables:
      Category Counts FDR Regulation Terms URLencode aggregate cor.test
      download.file foldEnrichment model.matrix p.adjust pchisq phyper
      qnorm read.table unzip write.table
    Consider adding
      importFrom("stats", "aggregate", "cor.test", "model.matrix",
                 "p.adjust", "pchisq", "phyper", "qnorm")
      importFrom("utils", "URLencode", "download.file", "read.table",
                 "unzip", "write.table")
    to your NAMESPACE file.
    ```

# GDINA

Version: 2.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GEM

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# genBaRcode

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# genBart

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘NMF’ ‘RColorBrewer’ ‘VennDiagram’ ‘clValid’ ‘data.table’ ‘grid’
      ‘gtools’ ‘pca3d’ ‘rmarkdown’ ‘scales’ ‘shinydashboard’ ‘shinyjs’
      ‘statmod’ ‘stringr’
      All declared Imports should be used.
    ```

# gender

Version: 0.5.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘genderdata’
    ```

# GENEAsphere

Version: 1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GeneralizedUmatrix

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# geneSLOPE

Version: 0.37.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# genlogis

Version: 1.0.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘genlogis-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: genlog_simu
    > ### Title: Simulating the Generalized logistic distribution
    > ### Aliases: genlog_simu
    > ### Keywords: genlogis
    > 
    > ### ** Examples
    > 
    >  
    > genlog_simu(real.par = c(0.3, 0.9, 1.5, 0.0), init.par = c(0.9, 0.3, 0.2, 0.0), 
    +             sample.size = 100, k = 50, threads = 2, seed = 200) 
    Error in { : task 1 failed - "[tcl] invalid command name "toplevel".
    "
    Calls: genlog_simu -> %dopar% -> <Anonymous>
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GenoGAM

Version: 1.8.0

## In both

*   checking whether package ‘GenoGAM’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GenoGAM/new/GenoGAM.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following objects are masked from 'package:matrixStats':
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following object is masked from 'package:Biostrings':
    
        type
    
    The following objects are masked from 'package:base':
    
        aperm, apply
    
    converting counts to integer mode
    Warning: Removed 4 rows containing missing values (geom_point).
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GenoGAM.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        doc       1.2Mb
        extdata   1.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot_base: no visible global function definition for ‘lines’
    plot_base: no visible global function definition for ‘abline’
    plot_base: no visible global function definition for ‘mtext’
    writeToBroadPeaks: no visible global function definition for
      ‘write.table’
    writeToNarrowPeaks: no visible global function definition for
      ‘write.table’
    xsd : <anonymous>: no visible binding for global variable ‘position’
    xsd: no visible binding for global variable ‘position’
    Undefined global functions or variables:
      abline as.formula axis coefficients dev.off dnbinom estimate fdr fit
      gene hist id lines mtext optim p.adjust par plot png pnorm position
      pval pvalue region runif write.table zscore
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "abline", "axis", "hist", "lines", "mtext",
                 "par", "plot")
      importFrom("stats", "as.formula", "coefficients", "dnbinom", "optim",
                 "p.adjust", "pnorm", "runif")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.GenoGAM':
      ‘plot.GenoGAM’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# genogeographer

Version: 0.1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# genomation

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc       3.5Mb
        extdata   1.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘RUnit’
      All declared Imports should be used.
    Unexported object imported by a ':::' call: ‘BiocGenerics:::testPackage’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ScoreMatrixBin,RleList-GRangesList: no visible binding for global
      variable ‘id’
    ScoreMatrixBin,RleList-GRangesList: no visible global function
      definition for ‘:=’
    Undefined global functions or variables:
      := id
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘cache’
    Please remove from your package.
    ```

# genomeplot

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GenomicDataCommons

Version: 1.4.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: magrittr
    
    Attaching package: 'GenomicDataCommons'
    
    The following object is masked from 'package:stats':
    
        filter
    
    Quitting from lines 125-127 (overview.Rmd) 
    Error: processing vignette 'overview.Rmd' failed with diagnostics:
    failed to rename downloaded file:
    
      from: '/Users/max/Library/Caches/GenomicDataCommons/310e2052-8e80-473e-875b-78b402853a66/.partial_download'
      to: '/Users/max/Library/Caches/GenomicDataCommons/310e2052-8e80-473e-875b-78b402853a66/ad9a29de-7c46-4caa-a802-b18053b4e32d.htseq.counts.gz'
      reason:
        cannot rename file
        '/Users/max/Library/Caches/GenomicDataCommons/310e2052-8e80-473e-875b-78b402853a66/.partial_download'
        to
        '/Users/max/Library/Caches/GenomicDataCommons/310e2052-8e80-473e-875b-78b402853a66/ad9a29de-7c46-4caa-a802-b18053b4e32d.htseq.counts.gz',
        reason 'No such file or directory'
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    default_fields.character: no visible binding for global variable
      ‘defaults’
    Undefined global functions or variables:
      defaults
    ```

# GenomicInteractions

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        doc       1.8Mb
        extdata   7.9Mb
    ```

# genotypeeval

Version: 1.12.0

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘makeHistogram’ ‘pcaPlot’ ‘tsnePlot’
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported object: ‘GenomicRanges::subset’
    ```

# GenVisR

Version: 1.12.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 14.0Mb
      sub-directories of 1Mb or more:
        doc      11.5Mb
        extdata   1.0Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘reshape2’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    setTierTwo,data.table : a: no visible binding for global variable ‘tmp’
    toLolliplot,GMS: no visible binding for global variable ‘missingINdex’
    toLolliplot,VEP: no visible binding for global variable ‘missingINdex’
    Undefined global functions or variables:
      missingINdex tmp
    ```

# geofacet

Version: 0.1.9

## In both

*   checking whether package ‘geofacet’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/new/geofacet.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘geofacet’ ...
** package ‘geofacet’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘geofacet’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/new/geofacet.Rcheck/geofacet’

```
### CRAN

```
* installing *source* package ‘geofacet’ ...
** package ‘geofacet’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘geofacet’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/geofacet/old/geofacet.Rcheck/geofacet’

```
# geoknife

Version: 1.5.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GeomComb

Version: 1.0

## Newly broken

*   checking whether package ‘GeomComb’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘GeomComb’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/GeomComb/new/GeomComb.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# geomerge

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# geomnet

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# geotoolsR

Version: 1.0

## Newly broken

*   checking whether package ‘geotoolsR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘geotoolsR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/geotoolsR/new/geotoolsR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# geozoning

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘utils’
      All declared Imports should be used.
    ```

# GERGM

Version: 0.11.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GerminaR

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘shinydashboard’
      All declared Imports should be used.
    ```

# germinationmetrics

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gespeR

Version: 1.12.0

## Newly broken

*   checking whether package ‘gespeR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘gespeR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gespeR/new/gespeR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'c,Phenotypes-method':
    \S4method{c}{Phenotypes}
      Code: function(x, ...)
      Docs: function(x, ..., recursive = FALSE)
      Argument names in docs not in code:
        recursive
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'gespeR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .gespeR.cv: no visible global function definition for ‘coef’
    .select.model: no visible global function definition for ‘predict’
    concordance: no visible global function definition for ‘cor’
    lasso.rand: no visible global function definition for ‘runif’
    plot.gespeR: no visible global function definition for ‘hist’
    stability.selection: no visible global function definition for ‘lm’
    Phenotypes,character: no visible global function definition for
      ‘read.delim’
    Undefined global functions or variables:
      coef cor hist lm predict read.delim runif
    Consider adding
      importFrom("graphics", "hist")
      importFrom("stats", "coef", "cor", "lm", "predict", "runif")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# getCRUCLdata

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GetDFPData

Version: 0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GetHFData

Version: 1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GetITRData

Version: 0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GetLattesData

Version: 0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# getmstatistic

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# getTBinR

Version: 0.5.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘getTBinR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: map_tb_burden
    > ### Title: Map TB Burden
    > ### Aliases: map_tb_burden
    > 
    > ### ** Examples
    > 
    > 
    > ## Map raw incidence rates
    > map_tb_burden()
    Loading data from: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmpv9GQiD/TB_burden.rds
    Loading data from: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmpv9GQiD/TB_data_dict.rds
    1 results found for your variable search for e_inc_100k
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             layerData = layerData, originalData = originalData, source = source, ...)
      8: by_layer(function(l, d) l$compute_aesthetics(d, plot))
      9: f(l = layers[[i]], d = data[[i]])
      10: l$compute_aesthetics(d, plot)
      11: f(..., self = self)
      12: scales_add_defaults(plot$scales, data, aesthetics, plot$plot_env)
      13: lapply(aesthetics[new_aesthetics], rlang::eval_tidy, data = data)
      14: FUN(X[[i]], ...)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 72 SKIPPED: 7 FAILED: 1
      1. Error: map_tb_burden produces an interactive plot (@test-map_tb_burden.R#56) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    ✖ dplyr::vars()   masks ggplot2::vars()
    Loading required package: viridisLite
    
    Attaching package: 'scales'
    
    The following object is masked from 'package:viridis':
    
        viridis_pal
    
    The following object is masked from 'package:purrr':
    
        discard
    
    The following object is masked from 'package:readr':
    
        col_factor
    
    Quitting from lines 58-67 (case_fatality_rate.Rmd) 
    Error: processing vignette 'case_fatality_rate.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GetTDData

Version: 1.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gfcanalysis

Version: 1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gfer

Version: 0.1.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# ggallin

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggalluvial

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GGally

Version: 1.3.2

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    The error most likely occurred in:
    
    > ### Name: add_and_overwrite_aes
    > ### Title: Add new aes
    > ### Aliases: add_and_overwrite_aes
    > ### Keywords: internal
    > 
    > ### ** Examples
    > 
    > data(diamonds, package="ggplot2")
    > diamonds.samp <- diamonds[sample(1:dim(diamonds)[1], 1000), ]
    > pm <- ggpairs(diamonds.samp, columns = 5:7,
    +   mapping = ggplot2::aes(color = color),
    +   upper = list(continuous = "cor", mapping = ggplot2::aes_string(color = "clarity")),
    +   lower = list(continuous = "cor", mapping = ggplot2::aes_string(color = "cut")),
    +   title = "Diamonds Sample"
    + )
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~depth", "~color".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~depth", "~color".  Please rename the columns or make a new column.
    Calls: ggpairs ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 347 SKIPPED: 1 FAILED: 74
      1. Error: crosstalk works with ggduo and ggpairs (@test-crosstalk.R#12) 
      2. Error: denstrip (@test-gg-plots.R#13) 
      3. Error: density (@test-gg-plots.R#26) 
      4. Error: cor (@test-gg-plots.R#36) 
      5. Error: diagAxis (@test-gg-plots.R#102) 
      6. Error: dates (@test-gg-plots.R#135) 
      7. Failure: rescale (@test-gg-plots.R#161) 
      8. Error: generally works (@test-ggfacet.R#33) 
      9. Error: legend (@test-gglegend.R#73) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggalt

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plotly’
      All declared Imports should be used.
    ```

# ggbeeswarm

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'usageExamples.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ggbuildr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggChernoff

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggconf

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggcorrplot

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggcyto

Version: 1.8.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggcyto-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: as.ggplot
    > ### Title: It fortifies the data, fills some default settings and returns a
    > ###   regular ggplot object.
    > ### Aliases: as.ggplot
    > 
    > ### ** Examples
    > 
    > data(GvHD)
    > fs <- GvHD[1:3]
    > #construct the `ggcyto` object (inherits from `ggplot` class)
    > p <- ggcyto(fs, aes(x = `FSC-H`)) + geom_histogram() 
    Error in `[.data.table`(dims.tbl, , name) : 
      j (the 2nd argument inside [...]) is a single symbol but column name 'name' is not found. Perhaps you intended DT[, ..name]. This difference to data.frame is deliberate and explained in FAQ 1.1.
    Calls: ggcyto -> ggcyto.flowSet -> [ -> [.data.table
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: flowCore
    Loading required package: ncdfFlow
    Loading required package: RcppArmadillo
    Loading required package: BH
    Loading required package: flowWorkspace
    loading R object...
    loading tree object...
    Done
    Quitting from lines 59-60 (Top_features_of_ggcyto.Rmd) 
    Error: processing vignette 'Top_features_of_ggcyto.Rmd' failed with diagnostics:
    j (the 2nd argument inside [...]) is a single symbol but column name 'name' is not found. Perhaps you intended DT[, ..name]. This difference to data.frame is deliberate and explained in FAQ 1.1.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scales’
      All declared Imports should be used.
    Unexported objects imported by ':::' calls:
      ‘flowWorkspace:::.mergeGates’ ‘flowWorkspace:::compact’
      ‘flowWorkspace:::fix_y_axis’ ‘flowWorkspace:::isNegated’
      ‘ggplot2:::+.gg’ ‘ggplot2:::add_group’ ‘ggplot2:::check_aesthetics’
      ‘ggplot2:::ggplot.data.frame’ ‘ggplot2:::is.waive’
      ‘ggplot2:::is_calculated_aes’ ‘ggplot2:::make_scale’
      ‘ggplot2:::plot_clone’ ‘ggplot2:::print.ggplot’
      ‘ggplot2:::scales_add_defaults’ ‘ggplot2:::update_theme’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    fortify_fs.GatingSetList: no visible global function definition for
      ‘getS3method’
    getFlowFrame.GatingSetList: no visible global function definition for
      ‘getS3method’
    getFlowFrame.ncdfFlowList: no visible global function definition for
      ‘getS3method’
    ggcyto.GatingSetList: no visible global function definition for
      ‘getS3method’
    ggcyto.flowSet: no visible binding for global variable ‘name’
    ggcyto.flowSet: no visible binding for global variable ‘axis’
    ggcyto.ncdfFlowList: no visible global function definition for
      ‘getS3method’
    ggcyto_arrange: no visible binding for global variable ‘name’
    Undefined global functions or variables:
      approx axis density desc dist getS3method gray modifyList name
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "axis")
      importFrom("stats", "approx", "density", "dist")
      importFrom("utils", "getS3method", "modifyList")
    to your NAMESPACE file.
    ```

# ggdag

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > dag <- dagify(m ~ a + b, x ~ a, y ~ b)
    > 
    > control_for(dag, var = "m")
    # A tibble: 8 x 10
      name      x     y direction to     xend  yend circular collider_line adjusted
      <chr> <dbl> <dbl> <fct>     <chr> <dbl> <dbl> <lgl>    <lgl>         <fct>   
    1 a      31.6  23.1 ->        m      31.6  24.6 FALSE    FALSE         unadjus…
    2 a      31.6  23.1 ->        x      31.6  21.8 FALSE    FALSE         unadjus…
    3 b      31.6  26.0 ->        m      31.6  24.6 FALSE    FALSE         unadjus…
    4 b      31.6  26.0 ->        y      31.5  27.3 FALSE    FALSE         unadjus…
    5 m      31.6  24.6 <NA>      <NA>   NA    NA   FALSE    FALSE         adjusted
    6 x      31.6  21.8 <NA>      <NA>   NA    NA   FALSE    FALSE         unadjus…
    7 y      31.5  27.3 <NA>      <NA>   NA    NA   FALSE    FALSE         unadjus…
    8 a      31.6  23.1 <->       b      31.6  26.0 NA       TRUE          unadjus…
    > ggdag_adjust(dag, var = "m")
    Error in arrow.fill %||% trans$colour : 
      argument "arrow.fill" is missing, with no default
    Calls: <Anonymous> ... <Anonymous> -> .fun -> do.call -> <Anonymous> -> f -> %||%
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    
    Attaching package: 'ggdag'
    
    The following object is masked from 'package:ggplot2':
    
        expand_scale
    
    The following object is masked from 'package:stats':
    
        filter
    
    Quitting from lines 70-72 (bias-structures.Rmd) 
    Error: processing vignette 'bias-structures.Rmd' failed with diagnostics:
    argument "arrow.fill" is missing, with no default
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggforce’ ‘plyr’
      All declared Imports should be used.
    ```

# ggdendro

Version: 0.1-20

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggdmc

Version: 0.1.3.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GGEBiplots

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGEBiplotGUI’ ‘gge’ ‘utils’
      All declared Imports should be used.
    ```

# ggedit

Version: 0.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggedit-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: layersList
    > ### Title: layersList
    > ### Aliases: layersList
    > 
    > ### ** Examples
    > 
    > p=ggplot(iris,aes(x =Sepal.Length,y=Sepal.Width))
    > p=p+geom_point(aes(colour=Species))+geom_line()
    > p
    > p.list=layersList(p)
    Error in `[.data.frame`(p$data, , as.character(x)) : 
      undefined columns selected
    Calls: layersList ... fetch_aes_ggplotBuild -> class_layer -> lapply -> FUN -> [ -> [.data.frame
    Execution halted
    ```

*   checking whether package ‘ggedit’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggedit’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggedit/new/ggedit.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggeffects

Version: 0.3.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ggeffect(fit, terms = "c12hour")
    # A tibble: 62 x 5
           x predicted conf.low conf.high group
     * <dbl>     <dbl>    <dbl>     <dbl> <fct>
     1     4      74.4     72.3      76.5 1    
     2     5      74.2     72.1      76.3 1    
     3     6      73.9     71.9      76.0 1    
     4     7      73.7     71.6      75.7 1    
     5     8      73.4     71.4      75.4 1    
     6     9      73.2     71.2      75.2 1    
     7    10      72.9     70.9      74.9 1    
     8    11      72.7     70.7      74.6 1    
     9    12      72.4     70.5      74.3 1    
    10    14      71.9     70.0      73.8 1    
    # ... with 52 more rows
    > 
    > mydf <- ggeffect(fit, terms = c("c12hour", "c161sex"))
    > plot(mydf)
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to linetype
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: hp. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Following variables had many unique values and were prettified: c12hour. Use `pretty = FALSE` to get smoother plots with all values, however, at the cost of increased memory usage.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 191-209 (marginaleffects.Rmd) 
    Error: processing vignette 'marginaleffects.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to linetype
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ordinal’
    ```

# ggenealogy

Version: 0.3.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggenealogy-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotAncDes
    > ### Title: Returns the image object to show the ancestors and descendants
    > ###   of a variety
    > ### Aliases: plotAncDes
    > 
    > ### ** Examples
    > 
    > data(sbGeneal)
    > plotAncDes("Tokyo", sbGeneal, vColor = "red")
    Error in geom_text() : could not find function "geom_text"
    Calls: plotAncDes -> <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tibble’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2356 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      geneal: the full genealogy (in data frame format)
    
    vertexinfo: (default NULL) either names of columns in the genealogy
              which should be added to the database as vertex information
              or a data frame with information for all vertices such that
              the first column contains vertex names.
    
    edgeweights: (default 1) name of a column which contains edge weights
    
    isDirected: (default FALSE) should the graph be a directed graph?
    
    _S_e_e _A_l_s_o:
    
         <URL: https://www.r-project.org> for iGraph information
    
    
    
    Error: processing vignette 'ggenealogy.Rnw' failed with diagnostics:
     chunk 23 (label = plotAncDes1) 
    Error in geom_text() : could not find function "geom_text"
    Execution halted
    ```

# ggExtra

Version: 0.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# ggFacetSample

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    >   testdata <- data.frame(
    +     rowcat = paste0("1", sample(LETTERS[1:10], sampn, replace = TRUE)),
    +     rowcat2 = paste0("2", sample(LETTERS[11:15], sampn, replace = TRUE)),
    +     colcat = paste0("2", sample(letters[1:10], sampn, replace = TRUE)),
    +     x = runif(sampn),
    +     y = runif(sampn)
    +   )
    > 
    >   p <- ggplot2::ggplot(testdata) + ggplot2::aes(x=x, y=y, color=rowcat2) +
    +     ggplot2::geom_point()
    > 
    >   # plot the works (not actually recommended)
    >   ## Not run: p + ggplot2::facet_grid(rowcat ~ colcat)
    > 
    >   # plot a sample of the row-facets:
    >   p + facet_sample(rowcat ~ colcat, sample_n = c(3, NA))
    Error in UseMethod("as.quoted") : 
      no applicable method for 'as.quoted' applied to an object of class "quosures"
    Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous>
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plyr’
      All declared Imports should be used.
    ```

# ggfan

Version: 0.1.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 59-62 (geom_fan.Rmd) 
    Error: processing vignette 'geom_fan.Rmd' failed with diagnostics:
    replacement has 0 rows, data has 1
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colorspace’ ‘grid’ ‘rstan’
      All declared Imports should be used.
    ```

# ggfittext

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggforce

Version: 0.1.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Link points with paths
    > ### Aliases: geom_link geom_link0 geom_link2 stat_link stat_link2
    > 
    > ### ** Examples
    > 
    > # Lets make some data
    > lines <- data.frame(
    +   x = c(5, 12, 15, 9, 6),
    +   y = c(17, 20, 4, 15, 5),
    +   xend = c(19, 17, 2, 9, 5),
    +   yend = c(10, 18, 7, 12, 1),
    +   width = c(1, 10, 6, 2, 3),
    +   colour = letters[1:5]
    + )
    > 
    > ggplot() + geom_link(aes(x = x, y = y, xend = xend, yend = yend,
    +                          colour = colour, alpha = ..index..,
    +                          size = ..index..),
    +                      data = lines)
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'geom_link':
    geom_link0
      Code: function(mapping = NULL, data = NULL, stat = "identity",
                     position = "identity", ..., arrow = NULL, arrow.fill =
                     NULL, lineend = "butt", linejoin = "round", na.rm =
                     FALSE, show.legend = NA, inherit.aes = TRUE)
      Docs: function(mapping = NULL, data = NULL, stat = "identity",
                     position = "identity", ..., arrow = NULL, lineend =
                     "butt", na.rm = FALSE, show.legend = NA, inherit.aes =
                     TRUE)
      Argument names in code not in docs:
        arrow.fill linejoin
      Mismatches in argument names (first 3):
        Position: 7 Code: arrow.fill Docs: lineend
        Position: 8 Code: lineend Docs: na.rm
        Position: 9 Code: linejoin Docs: show.legend
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 185-198 (Visual_Guide.Rmd) 
    Error: processing vignette 'Visual_Guide.Rmd' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggformula

Version: 0.6.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggformula-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: gf_ash
    > ### Title: Average Shifted Histograms
    > ### Aliases: gf_ash stat_ash geom_ash
    > 
    > ### ** Examples
    > 
    > gf_ash(~Sepal.Length, color = ~ Species, data = iris)
    Don't know how to automatically pick scale for object of type formula. Defaulting to continuous.
    Error: Aesthetics must be either length 1 or the same as the data (150): colour, x
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 170-172 (ggformula-blog.Rmd) 
    Error: processing vignette 'ggformula-blog.Rmd' failed with diagnostics:
    Aesthetics must be either length 1 or the same as the data (9583): fill, x, y
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggfortify

Version: 0.4.4

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      1.  Failure: Check geom_factory works (@test-plotlib.R#152) 
      2.  Failure: Check geom_factory works (@test-plotlib.R#160) 
      3.  Failure: Check geom_factory works (@test-plotlib.R#169) 
      4.  Failure: Check geom_factory works (@test-plotlib.R#178) 
      5.  Failure: Check geom_factory works (@test-plotlib.R#203) 
      6.  Failure: autoplot.lm works with factors (@test-stats-lm.R#519) 
      7.  Failure: autoplot.lm works with factors (@test-stats-lm.R#524) 
      8.  Failure: autoplot.lm works with factors (@test-stats-lm.R#531) 
      9.  Failure: autoplot.lm works with characters (@test-stats-lm.R#539) 
      10. Failure: autoplot.lm works with characters (@test-stats-lm.R#544) 
      
      Error: testthat unit tests failed
      In addition: Warning message:
      replacing previous import 'dplyr::vars' by 'ggplot2::vars' when loading 'ggfortify' 
      Execution halted
    ```

*   checking whether package ‘ggfortify’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggfortify’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggfortify/new/ggfortify.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gggenes

Version: 0.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘gggenes-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: make_alignment_dummies
    > ### Title: Prepare dummy data to visually align a single gene across
    > ###   faceted molecules
    > ### Aliases: make_alignment_dummies
    > 
    > ### ** Examples
    > 
    > 
    > dummies <- make_alignment_dummies(example_genes, ggplot2::aes(xmin = start,
    +   xmax = end, y = molecule, id = gene), on = "genE")
    Error in `[.data.frame`(data, as.character(unlist(mapping))) : 
      undefined columns selected
    Calls: make_alignment_dummies -> [ -> [.data.frame
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(gggenes)
      > 
      > test_check("gggenes")
      ── 1. Failure: make_alignment_dummies works without errors (@test-make_alignment
      `{ ... }` threw an error.
      Message: undefined columns selected
      Class:   simpleError/error/condition
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2 SKIPPED: 0 FAILED: 1
      1. Failure: make_alignment_dummies works without errors (@test-make_alignment_dummies.R#4) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 59-72 (introduction-to-gggenes.Rmd) 
    Error: processing vignette 'introduction-to-gggenes.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggghost

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggguitar

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘lazyeval’ ‘readr’
      All declared Imports should be used.
    ```

# gghalfnorm

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gghighlight

Version: 0.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘gghighlight-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: gghighlight
    > ### Title: Highlight Data With Predicate
    > ### Aliases: gghighlight gghighlight_line gghighlight_point
    > 
    > ### ** Examples
    > 
    > d <- data.frame(
    +   idx = c( 1, 1, 1, 2, 2, 2, 3, 3, 3),
    +   value = c( 1, 2, 3,10,11,12, 9,10,11),
    +   category = rep(c("a","b","c"), 3),
    +   stringsAsFactors = FALSE
    + )
    > 
    > gghighlight_line(d, aes(idx, value, colour = category), max(value) > 10)
    Error in .subset2(x, i) : no such index at level 1
    Calls: gghighlight_line -> gghighlight -> [[ -> [[.tbl_df
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             max_highlight = max_highlight, unhighlighted_colour = unhighlighted_colour, geom_func = ggplot2::geom_line, 
             use_group_by = use_group_by, ..., environment = environment)
      11: data_predicated[[as.character(group_key)]]
      12: `[[.tbl_df`(data_predicated, as.character(group_key))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 27 SKIPPED: 0 FAILED: 5
      1. Failure: gghighlight_point() without label_key works (@test-gghighlight_point.R#38) 
      2. Error: gghighligt_line() with usual arguments works (@test-gghighligt_line.R#12) 
      3. Error: gghighligt_line() without direct labeling works (@test-gghighligt_line.R#35) 
      4. Error: gghighligt_line() works with facets (@test-gghighligt_line.R#76) 
      5. Error: gghighligt_line() works with numerical predicate (@test-gghighligt_line.R#107) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggimage

Version: 0.1.5

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘ggimage-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: geom_image
    > ### Title: geom_image
    > ### Aliases: geom_image
    > 
    > ### ** Examples
    > 
    > library("ggplot2")
    > library("ggimage")
    > set.seed(2017-02-21)
    > d <- data.frame(x = rnorm(10),
    +                 y = rnorm(10),
    +                 image = sample(c("https://www.r-project.org/logo/Rlogo.png",
    +                                 "https://jeroenooms.github.io/images/frink.png"),
    +                               size=10, replace = TRUE)
    +                )
    > ggplot(d, aes(x, y)) + geom_image(aes(image=image))
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridGraphics’ ‘rvcheck’ ‘scales’
      All declared Imports should be used.
    ```

# ggiraph

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggjoy

Version: 0.4.0

## Newly broken

*   checking whether package ‘ggjoy’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::scale_discrete_manual’ by ‘ggridges::scale_discrete_manual’ when loading ‘ggjoy’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggjoy/new/ggjoy.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gglogo

Version: 0.1.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'gglogo'
    
    The following object is masked from 'package:ggplot2':
    
        fortify
    
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Garamond"
    Quitting from lines 46-52 (gglogo-alphabet.Rmd) 
    Error: processing vignette 'gglogo-alphabet.Rmd' failed with diagnostics:
    replacement has 1 row, data has 0
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# ggloop

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gglorenz

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggmap

Version: 2.6.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘ggmap-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: theme_nothing
    > ### Title: Make a blank ggplot2 theme.
    > ### Aliases: theme_nothing
    > 
    > ### ** Examples
    > 
    > 
    > 
    > # no legend example
    > n <- 50
    > df <- expand.grid(x = 1:n,y = 1:n)[sample(n^2,.5*n^2),]
    > p <- qplot(x, y, data = df, geom = 'tile')
    > p
    > p + theme_nothing()
    Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
    Error in if (theme$panel.ontop) { : argument is of length zero
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggmcmc

Version: 1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call.graphics(C_text, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "Source Sans Pro"
    Quitting from lines 142-143 (using_ggmcmc.Rmd) 
    Error: processing vignette 'using_ggmcmc.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggmosaic

Version: 0.1.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggmosaic-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: geom_mosaic
    > ### Title: Mosaic plots.
    > ### Aliases: geom_mosaic stat_mosaic
    > 
    > ### ** Examples
    > 
    > 
    > data(Titanic)
    > titanic <- as.data.frame(Titanic)
    > titanic$Survived <- factor(titanic$Survived, levels=c("Yes", "No"))
    > 
    > 
    > ggplot(data=titanic) +
    +   geom_mosaic(aes(weight=Freq, x=product(Class), fill=Survived))
    Error in is.finite(x) : default method not implemented for type 'list'
    Calls: <Anonymous> ... lapply -> FUN -> <Anonymous> -> f -> lapply -> FUN -> f
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        vars
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    
    Attaching package: 'gridExtra'
    
    The following object is masked from 'package:dplyr':
    
        combine
    
    Quitting from lines 171-175 (ggmosaic.Rmd) 
    Error: processing vignette 'ggmosaic.Rmd' failed with diagnostics:
    default method not implemented for type 'list'
    Execution halted
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'scale_type.product':
      ‘scale_type.product’
    
    S3 methods shown with full name in documentation object 'scale_type.productlist':
      ‘scale_type.productlist’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘NHANES’ ‘gridExtra’
      All declared Imports should be used.
    ```

# ggmuller

Version: 0.5.1

## Newly broken

*   checking whether package ‘ggmuller’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggmuller’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggmuller/new/ggmuller.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggnetwork

Version: 0.5.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
                        Skye Bender-deMoll, University of Washington
     For citation information, type citation("network").
     Type help("network-package") to get started.
    
    Loading required package: sna
    Loading required package: statnet.common
    
    Attaching package: ‘statnet.common’
    
    The following object is masked from ‘package:base’:
    
        order
    
    sna: Tools for Social Network Analysis
    Version 2.4 created on 2016-07-23.
    copyright (c) 2005, Carter T. Butts, University of California-Irvine
     For citation information, type citation("sna").
     Type help(package="sna") to get started.
    
    Error: Columns `x`, `y`, `xend`, `yend` must be 1d atomic vectors or lists
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
     For citation information, type citation("network").
     Type help("network-package") to get started.
    
    Loading required package: statnet.common
    
    Attaching package: 'statnet.common'
    
    The following object is masked from 'package:base':
    
        order
    
    sna: Tools for Social Network Analysis
    Version 2.4 created on 2016-07-23.
    copyright (c) 2005, Carter T. Butts, University of California-Irvine
     For citation information, type citation("sna").
     Type help(package="sna") to get started.
    
    Quitting from lines 130-133 (ggnetwork.Rmd) 
    Error: processing vignette 'ggnetwork.Rmd' failed with diagnostics:
    Columns `x`, `y`, `xend`, `yend` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggparallel

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggplotAssist

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gcookbook’ ‘ggthemes’ ‘moonBook’ ‘tidyverse’
      All declared Imports should be used.
    ```

# ggplotgui

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggplotify

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggpmisc

Version: 0.2.17

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘devtools’ ‘grid’ ‘gridExtra’
      All declared Imports should be used.
    ```

# ggpol

Version: 0.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +                  gender = c("M","F"), levels = c("M", "F"))
    > 
    > # Get the count per age and sex
    > df$count <- 1
    > df <- aggregate(count ~ gender + age, data = df, length)
    > 
    > # For the horizontally shared axis, if we want to mirror the axes,
    > # we have to multiply the first panel by -1, and use coord_flip().
    > df_h <- df 
    > df_h$count = ifelse(df_h$gender == "F", df_h$count * -1, df_h$count)
    > 
    > p <- ggplot(df_h, aes(x = factor(age), y = count, fill = gender)) + 
    +   geom_bar(stat = "identity") +
    +   facet_share(~gender, dir = "h", scales = "free", reverse_num = TRUE) + 
    +   coord_flip() +
    +   labs(x = "Age", y = "Count") + 
    +   theme(legend.position = "bottom")
    > 
    > p
    Error: `quo` must be a quosure
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 146-161 (ggpol_guide.Rmd) 
    Error: processing vignette 'ggpol_guide.Rmd' failed with diagnostics:
    `quo` must be a quosure
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# ggpolypath

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggpubr

Version: 0.1.6

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > # Some key genes of interest to be highlighted
    > key.gns <- c("MYC", "PRDM1", "CD69", "IRF4", "CASP3", "BCL2L1", "MYB",  "BACH2", "BIM1",  "PTEN",
    +             "KRAS", "FOXP1", "IGF1R", "KLF4", "CDK6", "CCND2", "IGF1", "TNFAIP3", "SMAD3", "SMAD7",
    +             "BMPR2", "RB1", "IGF2R", "ARNT")
    > # Density distribution
    > ggdensity(gene_citation, x = "citation_index", y = "..count..",
    +   xlab = "Number of citation",
    +   ylab = "Number of genes",
    +   fill = "lightgray", color = "black",
    +   label = "gene", label.select = key.gns, repel = TRUE,
    +   font.label = list(color= "citation_index"),
    +   xticks.by = 20, # Break x ticks by 20
    +   gradient.cols = c("blue", "red"),
    +   legend = "bottom",
    +   legend.title = ""                                     # Hide legend title
    +   )
    Error in `[.data.frame`(.data, , .mapping["x"]) : 
      undefined columns selected
    Calls: ggdensity ... <Anonymous> -> .f -> ggpar -> .set_ticksby -> [ -> [.data.frame
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggpval

Version: 0.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggpval-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: add_pval
    > ### Title: Add p-values to ggplot objects.
    > ### Aliases: add_pval
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > library(ggpval)
    > data("PlantGrowth")
    > plt <- ggplot(PlantGrowth, aes(group, weight)) +
    +   geom_boxplot()
    > add_pval(plt, pairs = list(c(1, 3)), test='wilcox.test')
    Error in if (grepl(")", str)) { : argument is of length zero
    Calls: add_pval ... [.data.table -> eval -> eval -> get -> get_in_parenthesis
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-add_pval.R’ failed.
    Last 13 lines of output:
      > dt[, G := factor(G, levels = c("G2", "G1"))]
      > 
      > plt <- ggplot(dt, aes(variable, value)) +
      +   geom_boxplot() +
      +   geom_jitter()
      > 
      > plt_facet <- ggplot(dt, aes(variable, value)) +
      +   geom_boxplot() +
      +   geom_jitter() +
      +   facet_wrap(~G)
      > 
      > add_pval(plt, pairs = list(c(1, 2)))
      Error in if (grepl(")", str)) { : argument is of length zero
      Calls: add_pval ... [.data.table -> eval -> eval -> get -> get_in_parenthesis
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 61-66 (ggpval.Rmd) 
    Error: processing vignette 'ggpval.Rmd' failed with diagnostics:
    argument is of length zero
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggQC

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggQQunif

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# ggquickeda

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘Hmisc’ ‘colourpicker’ ‘dplyr’ ‘ggrepel’ ‘grDevices’ ‘gridExtra’
      ‘lazyeval’ ‘markdown’ ‘plotly’ ‘quantreg’ ‘rlang’ ‘shinyjs’ ‘tidyr’
      All declared Imports should be used.
    ```

# ggquiver

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggRandomForests

Version: 2.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘randomForest’
      All declared Imports should be used.
    ```

# ggraph

Version: 1.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    The following objects are masked from ‘package:stats’:
    
        decompose, spectrum
    
    The following object is masked from ‘package:base’:
    
        union
    
    > gr <- graph_from_data_frame(highschool)
    > V(gr)$popularity <- as.character(cut(degree(gr, mode = 'in'), breaks = 3,
    +                                      labels = c('low', 'medium', 'high')))
    > ggraph(gr) +
    +     geom_edge_link() +
    +     geom_node_point() +
    +     facet_graph(year~popularity)
    Using `nicely` as default layout
    Error in UseMethod("as.quoted") : 
      no applicable method for 'as.quoted' applied to an object of class "quosures"
    Calls: <Anonymous> ... ggplot_build.ggplot -> <Anonymous> -> f -> <Anonymous> -> f -> as.quoted
    Execution halted
    ```

*   checking S3 generic/method consistency ... WARNING
    ```
    guide_train:
      function(guide, scale, aesthetic)
    guide_train.edge_colourbar:
      function(guide, scale)
    
    guide_train:
      function(guide, scale, aesthetic)
    guide_train.edge_direction:
      function(guide, scale)
    
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    
    Attaching package: 'igraph'
    
    The following objects are masked from 'package:stats':
    
        decompose, spectrum
    
    The following object is masked from 'package:base':
    
        union
    
    Quitting from lines 210-213 (Edges.Rmd) 
    Error: processing vignette 'Edges.Rmd' failed with diagnostics:
    unused argument (output)
    Execution halted
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'guide_train.edge_colourbar':
      ‘guide_train.edge_colourbar’ ‘guide_train.edge_direction’
      ‘guide_merge.edge_direction’ ‘guide_geom.edge_direction’
      ‘guide_gengrob.edge_direction’
    
    S3 methods shown with full name in documentation object 'scale_type.geometry':
      ‘scale_type.geometry’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggraptR

Version: 1.0

## Newly broken

*   checking whether package ‘ggraptR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggraptR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggraptR/new/ggraptR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RSelenium’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DBI’ ‘GGally’ ‘RColorBrewer’ ‘Rcpp’ ‘assertthat’ ‘backports’
      ‘colorspace’ ‘colourpicker’ ‘evaluate’ ‘futile.options’ ‘gdtools’
      ‘gtable’ ‘htmltools’ ‘htmlwidgets’ ‘httpuv’ ‘labeling’ ‘lambda.r’
      ‘lazyeval’ ‘magrittr’ ‘miniUI’ ‘munsell’ ‘plyr’ ‘reshape’ ‘rprojroot’
      ‘scales’ ‘stringi’ ‘stringr’ ‘svglite’ ‘tibble’ ‘xtable’ ‘yaml’
      All declared Imports should be used.
    ```

# ggrepel

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggridges

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6242 marked UTF-8 strings
    ```

# ggROC

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘ggplot2’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ggroc: no visible global function definition for ‘ggplot’
    ggroc: no visible global function definition for ‘aes’
    ggroc: no visible global function definition for ‘geom_point’
    ggroc: no visible global function definition for ‘geom_line’
    ggroc: no visible global function definition for ‘theme’
    ggroc: no visible global function definition for ‘element_text’
    ggroc: no visible global function definition for ‘labs’
    ggroc: no visible global function definition for ‘ggsave’
    Undefined global functions or variables:
      aes element_text geom_line geom_point ggplot ggsave labs theme
    ```

# ggsci

Version: 2.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggseas

Version: 0.5.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggseas-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ggsdc
    > ### Title: Visualise seasonal decomposition
    > ### Aliases: ggsdc
    > 
    > ### ** Examples
    > 
    > # sample time series data in data frame
    > ap_df <- tsdf(AirPassengers)
    > 
    > ggsdc(ap_df, aes(x = x, y = y), method = "decompose") +
    +    geom_line()
    Error in `[.data.frame`(data, , xvar) : undefined columns selected
    Calls: ggsdc ... ggsdc_helper -> [ -> [.data.frame -> order -> [ -> [.data.frame
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             start = start, facet.titles = facet.titles)
      3: data[order(data[, xvar]), ]
      4: `[.data.frame`(data, order(data[, xvar]), )
      5: order(data[, xvar])
      6: data[, xvar]
      7: `[.data.frame`(data, , xvar)
      8: stop("undefined columns selected")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 6 SKIPPED: 0 FAILED: 2
      1. Error: ggsdc decompose produces a plot with four facets (@test-ggsdc.R#4) 
      2. Error: ggsdc stl produces a plot with four facets (@test-ggsdc.R#13) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 125-126 (ggsdc.Rmd) 
    Error: processing vignette 'ggsdc.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggseqlogo

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggsignif

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggsn

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggspatial

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggspectra

Version: 0.2.4-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggstance

Version: 0.3

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 12 SKIPPED: 0 FAILED: 16
      1. Error: geom_linerangeh() flips (@test-geoms.R#7) 
      2. Error: geom_pointangeh() flips (@test-geoms.R#13) 
      3. Error: geom_crossbarh() flips (@test-geoms.R#31) 
      4. Error: geom_errorbarh() flips (@test-geoms.R#37) 
      5. Error: geom_barh() flips (@test-geoms.R#45) 
      6. Error: geom_histogramh() flips (@test-geoms.R#57) 
      7. Error: geom_violinh() flips (@test-geoms.R#71) 
      8. Failure: geom_boxploth() flips (@test-geoms.R#81) 
      9. Error: geom_boxploth() flips (@test-geoms.R#81) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lazyeval’
      All declared Imports should be used.
    ```

# ggstatsplot

Version: 0.0.2

## Newly broken

*   checking whether package ‘ggstatsplot’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggstatsplot’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ggstatsplot/new/ggstatsplot.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggswissmaps

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggtern

Version: 2.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: annotate
    > ### Title: Create an annotation layer (ggtern version).
    > ### Aliases: annotate
    > 
    > ### ** Examples
    > 
    > ggtern() + 
    + annotate(geom  = 'text',
    +               x     = c(0.5,1/3,0.0),
    +               y     = c(0.5,1/3,0.0),
    +               z     = c(0.0,1/3,1.0),
    +               angle = c(0,30,60),
    +               vjust = c(1.5,0.5,-0.5),
    +               label = paste("Point",c("A","B","C")),
    +               color = c("green","red",'blue')) +
    +   theme_dark() + 
    +   theme_nomask()
    Error in f(..., self = self) : unused argument (plot$coordinates)
    Calls: <Anonymous> -> print.ggplot -> ggplot_build -> <Anonymous>
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘chemometrics’
    ```

# ggThemeAssist

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ggthemes

Version: 3.5.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ggthemes-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: scale_shape_stata
    > ### Title: Stata shape scale
    > ### Aliases: scale_shape_stata
    > 
    > ### ** Examples
    > 
    > library("ggplot2")
    > 
    > p <- ggplot(mtcars) +
    +      geom_point(aes(x = wt, y = mpg, shape = factor(gear))) +
    +      facet_wrap(~am)
    > p + theme_stata() + scale_shape_stata()
    Error in if (theme$panel.ontop) { : argument is of length zero
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 108-110 (./children/examples.Rmd) 
    Quitting from lines 36-40 (./children/examples.Rmd) 
    Error: processing vignette 'ggthemes.Rmd' failed with diagnostics:
    argument is of length zero
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GGtools

Version: 5.16.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘MatrixEQTL’ ‘gwascat’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 74.5Mb
      sub-directories of 1Mb or more:
        R       2.0Mb
        data   27.0Mb
        doc     1.6Mb
        parts   2.0Mb
        pup     2.0Mb
        rdas   10.3Mb
        vcf    28.8Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to 'Homo.sapiens' which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      'Homo.sapiens' 'parallel'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    topKfeats: no visible binding for global variable 'i2'
    tscan2df: no visible global function definition for '%dopar%'
    tscan2df: no visible global function definition for 'foreach'
    tscan2df: no visible binding for global variable 'i'
    tscan2gr: no visible global function definition for '%dopar%'
    tscan2gr: no visible global function definition for 'foreach'
    tscan2gr: no visible binding for global variable 'i'
    waldtests : <anonymous>: no visible global function definition for
      'wald.test'
    plot,gwSnpScreenResult-character: no visible global function definition
      for 'snpcount'
    plot,gwSnpScreenResult-character: no visible global function definition
      for 'snpsBySeqname'
    Undefined global functions or variables:
      %dopar% .N FDR Homo.sapiens Matrix_eQTL_engine SlicedData
      as.data.table assay assays bindcadd bindmaf chi.squared colData curp
      detectCores excl export.gff3 firstHalf firstThird foreach forestplot
      gwastagger gwrngs hg19.si.df hmm878 i i1 i2 lastThird maf mafs
      mclapply midThird modelLINEAR npc overlapsAny pl radiusUsed ranges<-
      rowRanges runOneSplit select setkey setkeyv setnames snp snpcount
      snpsBySeqname target tileGenome value wald.test x
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘MatrixEQTL’, ‘gwascat’
    ```

# ggtree

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    If you use ggtree in published research, please cite:
    Guangchuang Yu, David Smith, Huachen Zhu, Yi Guan, Tommy Tsan-Yuk Lam. ggtree: an R package for visualization and annotation of phylogenetic trees with their covariates and other associated data. Methods in Ecology and Evolution 2017, 8(1):28-36, doi:10.1111/2041-210X.12628
    
    Attaching package: 'ggtree'
    
    The following object is masked from 'package:cowplot':
    
        ggsave
    
    The following object is masked from 'package:ape':
    
        rotate
    
    Average angle change [1] 0.205749672759501
    Average angle change [2] 0.0594793517814446
    Average angle change [3] 0.0265103772447353
    Quitting from lines 107-110 (treeAnnotation.Rmd) 
    Error: processing vignette 'treeAnnotation.Rmd' failed with diagnostics:
    argument "arrow.fill" is missing, with no default
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.2Mb
      sub-directories of 1Mb or more:
        doc        4.9Mb
        examples   3.7Mb
    ```

# ghibli

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# gitter

Version: 1.1.1

## In both

*   checking S3 generic/method consistency ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    xdpyinfo:  unable to open display "".
    xdpyinfo:  unable to open display "".
    xdpyinfo:  unable to open display "".
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    ```

*   checking foreign function calls ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    ```

*   checking Rd \usage sections ... NOTE
    ```
    xdpyinfo:  unable to open display "".
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# GJRM

Version: 0.1-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# glmmTMB

Version: 0.2.1.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 15.0Mb
      sub-directories of 1Mb or more:
        libs  13.9Mb
    ```

# glue

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gMOIP

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GOexpress

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   4.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    GO_analyse: no visible binding for global variable ‘microarray2dataset’
    GO_analyse: no visible binding for global variable ‘prefix2dataset’
    mart_from_ensembl: no visible binding for global variable
      ‘prefix2dataset’
    Undefined global functions or variables:
      microarray2dataset prefix2dataset
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘VennDiagram’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      There are other near singularities as well. 1783.4
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      pseudoinverse used at 1.77
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      neighborhood radius 22.23
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      reciprocal condition number  1.3784e-16
    Warning in simpleLoess(y, x, w, span, degree = degree, parametric = parametric,  :
      There are other near singularities as well. 1783.4
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      pseudoinverse used at 1.77
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      neighborhood radius 22.23
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      reciprocal condition number  1.3784e-16
    Warning in predLoess(object$y, object$x, newx = if (is.null(newdata)) object$x else if (is.data.frame(newdata)) as.matrix(model.frame(delete.response(terms(object)),  :
      There are other near singularities as well. 1783.4
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GOexpress-UsersGuide.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# gofMC

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gogamer

Version: 0.4.3

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(gogamer)
      > 
      > test_check("gogamer")
      Error: segfault from C stack overflow
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 36 SKIPPED: 1 FAILED: 0
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GoogleGenomics

Version: 2.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘GoogleGenomics-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: getReads
    > ### Title: Get reads from Google Genomics.
    > ### Aliases: getReads
    > 
    > ### ** Examples
    > 
    > # Authenticated on package load from the env variable GOOGLE_API_KEY.
    > reads <- getReads()
    Error in getSearchPage("reads", body, fields, pageToken) : 
      You are not authenticated; see ?GoogleGenomics::authenticate.
    Calls: getReads -> getReadsPage -> getSearchPage
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# googlesheets

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GOplot

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GOsummaries

Version: 2.13.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ##D data(tissue_example)
    > ##D 
    > ##D pcr = prcomp(t(tissue_example$exp))
    > ##D gs_pca = gosummaries(pcr, annotation = tissue_example$annot)
    > ##D 
    > ##D plot(gs_pca, classes = "Tissue", components = 1:3, fontsize = 8)
    > ## End(Not run)
    > 
    > # Read metabolomic data
    > data(metabolomic_example)
    > 
    > pca = prcomp(t(metabolomic_example$data))
    > 
    > # Turn off GO enricment, since it does not work on metabolites
    > gs = gosummaries(pca, annotation = metabolomic_example$annot, 
    +                  show_gene = TRUE, gconvert_target = NULL)
    > plot(gs, class = "Tissue", components = 1:3, fontsize = 8)
    Error in geom_histogram(colour = "grey70") : 
      could not find function "geom_histogram"
    Calls: plot ... plot_panel -> plot_function -> <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    panel_boxplot: no visible binding for global variable ‘x’
    panel_boxplot: no visible binding for global variable ‘y’
    panel_boxplot: no visible binding for global variable ‘Class’
    panel_dummy: no visible binding for global variable ‘y’
    panel_dummy: no visible binding for global variable ‘x’
    plotWordcloud: no visible global function definition for
      ‘colorRampPalette’
    plotWordcloud: no visible global function definition for ‘runif’
    plot_motor: no visible global function definition for ‘dev.off’
    print.gosummaries: no visible global function definition for ‘head’
    pspearman: no visible global function definition for ‘pt’
    spearman_mds: no visible global function definition for ‘cor’
    Undefined global functions or variables:
      Class bmp boxplot.stats colorRampPalette cor dev.off head jpeg
      packageVersion pdf png pt runif tiff x y
    Consider adding
      importFrom("grDevices", "bmp", "boxplot.stats", "colorRampPalette",
                 "dev.off", "jpeg", "pdf", "png", "tiff")
      importFrom("stats", "cor", "pt", "runif")
      importFrom("utils", "head", "packageVersion")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘BiocStyle’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    
    Error: processing vignette 'GOsummaries-basics.Rnw' failed with diagnostics:
     chunk 1 (label = style-Sweave) 
    Error in loadNamespace(name) : there is no package called ‘BiocStyle’
    Execution halted
    ```

# GOTHiC

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'package_vignettes.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    ':::' call which should be '::': ‘S4Vectors:::orderIntegerPairs’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .getRestrictionSitesFromBSgenome: no visible global function definition
      for ‘seqlengths’
    .importHicup: no visible global function definition for ‘read.table’
    .onlyPairing: no visible global function definition for ‘read.table’
    .onlyPairing: no visible global function definition for ‘ScanBamParam’
    GOTHiC: no visible binding for global variable
      ‘BSgenome.Hsapiens.UCSC.hg19’
    GOTHiC: no visible binding for global variable ‘filtered’
    GOTHiC: no visible binding for global variable ‘interactingLoci’
    mapReadsToRestrictionSites: no visible binding for global variable
      ‘resGR’
    Undefined global functions or variables:
      BSgenome.Hsapiens.UCSC.hg19 IntervalTree ScanBamParam V1 binom.test
      biocLite chr1 chr2 dev.off filtered frequencies int1 int2
      interactingLoci isCircular locus1 locus2 p.adjust pdf pvalue
      read.table resGR seqlengths x11
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "x11")
      importFrom("stats", "binom.test", "p.adjust")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# gpmap

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘foreach’ ‘ggplot2’ ‘isotone’ ‘plyr’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot3_dec: no visible global function definition for ‘foreach’
    plot3_dec: no visible global function definition for ‘ggplot’
    plot3_dec: no visible global function definition for ‘aes_string’
    plot3_dec: no visible global function definition for ‘geom_line’
    plot3_dec: no visible global function definition for ‘facet_grid’
    plot3_dec: no visible global function definition for ‘as.formula’
    plot3_dec: no visible global function definition for ‘labs’
    plot3_orig: no visible global function definition for ‘%do%’
    plot3_orig: no visible global function definition for ‘foreach’
    plot3_orig: no visible global function definition for ‘ggplot’
    plot3_orig: no visible global function definition for ‘aes_string’
    plot3_orig: no visible global function definition for ‘geom_line’
    plot3_orig: no visible global function definition for ‘facet_wrap’
    plot3_orig: no visible global function definition for ‘as.formula’
    plot3_orig: no visible global function definition for ‘labs’
    Undefined global functions or variables:
      %do% %dopar% aaply activeSet aes_string as.formula facet_grid
      facet_wrap foreach geom_line geom_point ggplot labs laply var
    Consider adding
      importFrom("stats", "as.formula", "var")
    to your NAMESPACE file.
    ```

# gQTLstats

Version: 1.12.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 65.7Mb
      sub-directories of 1Mb or more:
        data        11.0Mb
        doc          1.1Mb
        registries  18.8Mb
        vcf         33.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘permScore_3’
    storeToMaxAssocBySNP: no visible global function definition for ‘nth’
    storeToMaxAssocBySNP: no visible binding for global variable ‘MAF’
    storeToMaxAssocBySNP: no visible binding for global variable ‘probeid’
    storeToMaxAssocBySNP: no visible binding for global variable ‘mindist’
    tqbrowser: no visible global function definition for ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘experiments’
    tqbrowser : server: no visible global function definition for
      ‘TabixFile’
    tqbrowser : server: no visible binding for global variable ‘assoc’
    tqbrowser : server: no visible binding for global variable ‘stateid’
    tqbrowser : server: no visible binding for global variable ‘state’
    transTable: no visible binding for global variable ‘i’
    tsByRank_sing: no visible binding for global variable ‘i’
    tsByRank_sing : <anonymous>: no visible binding for global variable ‘i’
    boxswarm,SnpToGeneQTL: no visible binding for global variable ‘g1’
    Undefined global functions or variables:
      DNAStringSetList MAF TabixFile TxDb assoc calls ch chisq criterion ex
      exonsBy experiments g1 geom_boxplot gt i id maf mindist ml10fdr nth
      permScore_1 permScore_2 permScore_3 probeid snp state stateid value x
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘MultiAssayExperiment’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 8 marked Latin-1 strings
      Note: found 12 marked UTF-8 strings
    ```

# granovaGG

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# grapherator

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘reshape2’
      All declared Imports should be used.
    ```

# GraphPCA

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# grattan

Version: 1.5.3.7

## Newly fixed

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      4: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
      5: eval(quote(`_fseq`(`_lhs`)), env, env)
      6: eval(quote(`_fseq`(`_lhs`)), env, env)
      7: `_fseq`(`_lhs`)
      8: freduce(value, `_function_list`)
      9: function_list[[i]](value)
      10: setnames(., c(index.col, time.col), c("Index", "Time"))
      11: stop("Items of 'old' not found in column names: ", paste(old[is.na(i)], collapse = ","))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 224 SKIPPED: 57 FAILED: 1
      1. Error: ABS connection (@test_cpi.R#136) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘taxstats’
    ```

# Greg

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘rmeta’
    ```

# GRENITS

Version: 1.32.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘Rcpp’ ‘RcppArmadillo’ ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘scale_fill_gradient’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘theme’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘element_blank’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘element_text’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘ggtitle’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘labs’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘scale_x_discrete’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘scale_y_discrete’
    .plotDistribParents.LargeMat: no visible global function definition for
      ‘facet_wrap’
    Undefined global functions or variables:
      GeneNames Var1 Var2 aes element_blank element_text facet_wrap
      geom_tile ggplot ggtitle labs scale_fill_gradient scale_x_discrete
      scale_y_discrete theme value variable
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: ggplot2
    Created folder /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpUlQp3e/Example_LinearNet
    Started MCMC chain 1  ============= 
    MCMC chain 1 finished!
    Started MCMC chain 2  ============= 
    MCMC chain 2 finished!
    Analysis finished. Output plots can be found in folder: "/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpUlQp3e/Example_LinearNet"
    network: Classes for Relational Data
    Version 1.13.0.1 created on 2015-08-31.
    copyright (c) 2005, Carter T. Butts, University of California-Irvine
                        Mark S. Handcock, University of California -- Los Angeles
                        David R. Hunter, Penn State University
                        Martina Morris, University of Washington
                        Skye Bender-deMoll, University of Washington
     For citation information, type citation("network").
     Type help("network-package") to get started.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'GRENITS_package.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# greport

Version: 0.7-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    accrualReport: multiple local function definitions for ‘gg’ with
      different formal arguments
    ```

# gridDebug

Version: 0.5-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gridExtra

Version: 2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gridsampler

Version: 0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘BiasedUrn’
      All declared Imports should be used.
    ```

# GRmetrics

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .GRcalculate: no visible binding for global variable
      ‘treatment_duration’
    .GRlogisticFit: no visible binding for global variable ‘experiment’
    GRdrawDRC: no visible binding for global variable ‘GRvalue’
    Undefined global functions or variables:
      GRvalue experiment treatment_duration
    ```

# GROAN

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# groupdata2

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# growcurves

Version: 0.2.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# growfunctions

Version: 0.14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# growthcurver

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gsDesign

Version: 3.0-1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: gsDesign
    Loading required package: xtable
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'gsSurvTemplate.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotgsCP: no visible global function definition for ‘opts’
    plotgsPower: no visible global function definition for ‘opts’
    qplotit: no visible global function definition for ‘opts’
    Undefined global functions or variables:
      opts
    ```

# GSE

Version: 4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rrcov’
      All declared Imports should be used.
    ```

# gsloid

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GSODR

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# GSparO

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gsynth

Version: 1.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gtrendsR

Version: 1.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# GUIgems

Version: 0.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rpanel’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# Guitar

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Rsamtools’ ‘GenomicFeatures’ ‘rtracklayer’ ‘GenomicAlignments’
      ‘GenomicRanges’ ‘ggplot2’ ‘grid’ ‘IRanges’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        extdata   4.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    .makeFigure_fill: no visible global function definition for ‘pdf’
    .makeFigure_fill: no visible global function definition for ‘dev.off’
    .makeFigure_nofill: no visible global function definition for ‘pdf’
    .makeFigure_nofill: no visible global function definition for ‘dev.off’
    BED12toGRangesList: no visible global function definition for
      ‘read.table’
    BED12toGRangesList: no visible global function definition for ‘head’
    narrowPeaktoGRanges: no visible global function definition for
      ‘read.table’
    Undefined global functions or variables:
      dev.off head pdf read.table
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("utils", "head", "read.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      the condition has length > 1 and only the first element will be used
    Warning in if (suppressWarnings(is.na(GuitarCoordsFromTxDb))) { :
      the condition has length > 1 and only the first element will be used
    Warning in if (suppressWarnings(is.na(GuitarCoordsFromTxDb))) { :
      the condition has length > 1 and only the first element will be used
    Warning in if (suppressWarnings(is.na(GuitarCoordsFromTxDb))) { :
      the condition has length > 1 and only the first element will be used
    Scale for 'x' is already present. Adding another scale
    for 'x', which will replace the existing scale.
    Scale for 'x' is already present. Adding another scale
    for 'x', which will replace the existing scale.
    Warning in if (suppressWarnings(is.na(GuitarCoordsFromTxDb))) { :
      the condition has length > 1 and only the first element will be used
    Scale for 'x' is already present. Adding another scale
    for 'x', which will replace the existing scale.
    Scale for 'x' is already present. Adding another scale
    for 'x', which will replace the existing scale.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Guitar-Overview.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# gunsales

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gutenbergr

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 13617 marked UTF-8 strings
    ```

# gwdegree

Version: 0.1.1

## Newly broken

*   checking whether package ‘gwdegree’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘gwdegree’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/gwdegree/new/gwdegree.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# gWQS

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# h2o

Version: 3.18.0.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 145.8Mb
      sub-directories of 1Mb or more:
        java  144.1Mb
    ```

# harrietr

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hBayesDM

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rstantools’
      All declared Imports should be used.
    ```

# HBP

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HDCytoData

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hdm

Version: 0.2.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in grep(cn[I.c], colnames(X)) :
      argument 'pattern' has length > 1 and only the first element will be used
    Warning in grep(cn[I.c], colnames(X)) :
      argument 'pattern' has length > 1 and only the first element will be used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'hdm_introduction.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hdme

Version: 0.1.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rglpk’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# hdnom

Version: 4.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hdrcde

Version: 3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# healthcareai

Version: 2.0.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      nrow(ggplot_build(output)$layout$panel_layout) not equal to length(unique(test_df$var1)).
      target is NULL, current is numeric
      
      ── 3. Failure: control_chart has correct number of panels (@test-control_chart.R
      nrow(ggplot_build(output)$layout$panel_layout) not equal to length(unique(test_df$var1)) * length(unique(test_df$var2)).
      target is NULL, current is numeric
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 726 SKIPPED: 7 FAILED: 3
      1. Failure: control_chart has correct number of panels (@test-control_chart.R#47) 
      2. Failure: control_chart has correct number of panels (@test-control_chart.R#49) 
      3. Failure: control_chart has correct number of panels (@test-control_chart.R#52) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking whether package ‘healthcareai’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘healthcareai’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/healthcareai/new/healthcareai.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# heatmaply

Version: 0.14.1

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      12: plotly_build(d, registerFrames = FALSE)
      13: plotly_build.plotly(d, registerFrames = FALSE)
      14: verify_attr_spec(p)
      15: verify_attr(thisTrace, validAttrs)
      16: retain(proposed[[attr]], unique)
      17: structure(f(x), class = oldClass(x))
      18: f(x)
      19: unique.default(x)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 267 SKIPPED: 1 FAILED: 1
      1. Error: cellnote (@test_heatmaply_misc.R#67) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘d3heatmap’
    ```

# heemod

Version: 0.9.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hei

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/Analysis.Rmd’
    (Is a VignetteBuilder field missing?)
    ```

# hesim

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# heuristica

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hexSticker

Version: 0.4.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hhi

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hiAnnotator

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    makeChunks: no visible global function definition for ‘breakInChunks’
    makeChunks: no visible global function definition for ‘detectCores’
    makeChunks : <anonymous>: no visible global function definition for
      ‘keepSeqlevels’
    makeChunks : <anonymous>: no visible global function definition for
      ‘seqlevelsInUse’
    makeGRanges: no visible global function definition for ‘IRanges’
    makeGRanges: no visible global function definition for ‘seqlengths’
    makeGRanges: no visible global function definition for ‘seqlevels<-’
    makeGRanges: no visible global function definition for ‘sortSeqlevels’
    makeGRanges: no visible global function definition for ‘seqlevelsInUse’
    makeGRanges: no visible global function definition for ‘seqlengths<-’
    makeGRanges: no visible global function definition for ‘seqlevels’
    Undefined global functions or variables:
      IRanges breakInChunks countQueryHits detectCores dist featureName
      keepSeqlevels mid n overlapsAny qStrand queryHits seqlengths
      seqlengths<- seqlevels seqlevels<- seqlevelsInUse sortSeqlevels
      subjectHits
    Consider adding
      importFrom("stats", "dist")
    to your NAMESPACE file.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# HIBAG

Version: 1.16.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +     snp.sel = match(snpid, HapMap_CEU_Geno$snp.id),
    +     samp.sel = match(hlatab$training$value$sample.id,
    +     HapMap_CEU_Geno$sample.id))
    > test.geno <- hlaGenoSubset(HapMap_CEU_Geno,
    +     samp.sel=match(hlatab$validation$value$sample.id,
    +     HapMap_CEU_Geno$sample.id))
    > 
    > 
    > #############################################################################
    > 
    > set.seed(100)
    > 
    > # train a HIBAG model in parallel with 2 cores
    > # please use "nclassifier=100" when you use HIBAG for real data
    > model <- hlaParallelAttrBagging(2, hlatab$training, train.geno, nclassifier=4)
    Warning in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  :
      port 11099 cannot be opened
    Error in socketConnection("localhost", port = port, server = TRUE, blocking = TRUE,  : 
      cannot open the connection
    Calls: hlaParallelAttrBagging ... makePSOCKcluster -> newPSOCKnode -> socketConnection
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HiCcompare

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        data   6.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sim.other.methods: no visible binding for global variable ‘IF2’
    sim.other.methods: no visible binding for global variable ‘adj.M’
    sim.other.methods: no visible binding for global variable ‘M’
    sim_matrix: no visible binding for global variable ‘bias.slope’
    total_sum: no visible binding for global variable ‘IF2’
    total_sum: no visible binding for global variable ‘M’
    total_sum: no visible binding for global variable ‘IF1’
    total_sum: no visible binding for global variable ‘chr1’
    volcano: no visible binding for global variable ‘A’
    volcano: no visible binding for global variable ‘adj.IF1’
    volcano: no visible binding for global variable ‘adj.IF2’
    volcano: no visible binding for global variable ‘p.value’
    volcano: no visible binding for global variable ‘D’
    Undefined global functions or variables:
      A D IF IF1 IF2 M Z adj.IF1 adj.IF2 adj.M axis bias.slope bp
      centromere_locations chr1 chr2 count fold.change i j p.adj p.value
      pnorm region1 region2 start1 start2
    Consider adding
      importFrom("graphics", "axis")
      importFrom("stats", "D", "pnorm")
    to your NAMESPACE file.
    ```

# hierarchicalDS

Version: 2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    rrw: no visible global function definition for ‘rnorm’
    simulate_data: no visible global function definition for ‘pnorm’
    simulate_data: no visible global function definition for ‘rpois’
    switch_pdf: no visible global function definition for ‘dpois’
    switch_pdf: no visible global function definition for ‘dnorm’
    switch_sample: no visible global function definition for ‘rpois’
    switch_sample: no visible global function definition for ‘rnorm’
    switch_sample: no visible global function definition for ‘runif’
    switch_sample_prior: no visible global function definition for ‘rgamma’
    switch_sample_prior: no visible global function definition for ‘rnorm’
    table.mcmc: no visible global function definition for ‘write.csv’
    Undefined global functions or variables:
      abline dnorm dpois legend median model.matrix optimize par plot pnorm
      points rgamma rnorm rpois runif var write.csv
    Consider adding
      importFrom("graphics", "abline", "legend", "par", "plot", "points")
      importFrom("stats", "dnorm", "dpois", "median", "model.matrix",
                 "optimize", "pnorm", "rgamma", "rnorm", "rpois", "runif",
                 "var")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# hierarchicalSets

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘ggplot2:::build_guides’ ‘ggplot2:::guide_axis’
      ‘ggplot2:::scales_list’
      See the note in ?`:::` about the use of this operator.
    ```

# HighDimOut

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Func.ABOD: no visible global function definition for ‘combn’
    Func.ABOD: no visible global function definition for ‘var’
    Func.SOD: no visible global function definition for ‘var’
    Func.trans : erf: no visible global function definition for ‘pnorm’
    Func.trans: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      combn pnorm sd var
    Consider adding
      importFrom("stats", "pnorm", "sd", "var")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

# hIRT

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HistData

Version: 0.8-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘Guerry’, ‘alr3’, ‘agridat’
    ```

# HistDAWass

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# historydata

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HLMdiag

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Hmisc

Version: 4.1-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HMP

Version: 1.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘genalg’
    ```

# HMP16SData

Version: 1.0.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        '/Users/max//.ExperimentHub/1117'
    snapshotDate(): 2018-04-27
    see ?HMP16SData and browseVignettes('HMP16SData') for documentation
    downloading 0 resources
    loading from cache 
        '/Users/max//.ExperimentHub/1117'
    snapshotDate(): 2018-04-27
    see ?HMP16SData and browseVignettes('HMP16SData') for documentation
    downloading 0 resources
    loading from cache 
        '/Users/max//.ExperimentHub/1118'
    snapshotDate(): 2018-04-27
    see ?HMP16SData and browseVignettes('HMP16SData') for documentation
    downloading 0 resources
    loading from cache 
        '/Users/max//.ExperimentHub/1118'
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 227-235 (HMP16SData.Rmd) 
    Error: processing vignette 'HMP16SData.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 18.5Mb
      sub-directories of 1Mb or more:
        extdata  17.4Mb
    ```

# HoRM

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# horserule

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘Rdpack’
      All declared Imports should be used.
    ```

# hrbrthemes

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HRM

Version: 0.9.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘RGtk2’ ‘cairoDevice’ ‘RGtk2Extras’
    ```

# HTSSIP

Version: 1.3.2

## Newly broken

*   checking whether package ‘HTSSIP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘HTSSIP’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/HTSSIP/new/HTSSIP.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   4.4Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘igraph’
      All declared Imports should be used.
    ```

# httk

Version: 1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 15.1Mb
      sub-directories of 1Mb or more:
        data  13.1Mb
        doc    1.3Mb
    ```

# hurricaneexposure

Version: 0.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘hurricaneexposure-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: default_map
    > ### Title: Create a default map with eastern US states
    > ### Aliases: default_map
    > 
    > ### ** Examples
    > 
    > default_map()
    Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
    Theme element panel.border missing
    Error in if (theme$panel.ontop) { : argument is of length zero
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘hurricaneexposuredata’
    ```

# hutils

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# huxtable

Version: 3.0.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      6: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message)
      7: withVisible(code)
      8: eval_bare(get_expr(quo), get_env(quo))
      9: quick_pdf(m, dfr, ht, file = tf)
      10: tools::texi2pdf(latex_file, clean = TRUE)
      11: texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet, texi2dvi = texi2dvi, 
             texinputs = texinputs, index = index)
      12: stop(msg, domain = NA)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 354 SKIPPED: 52 FAILED: 1
      1. Error: Quick output functions work (@test-output.R#135) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# HWxtest

Version: 1.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hybridModels

Version: 0.2.15

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# hydrolinks

Version: 0.10.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# hydroscoper

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3031 marked UTF-8 strings
    ```

# IAPWS95

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘ggplot2’ ‘pander’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# IAT

Version: 0.3

## Newly broken

*   checking whether package ‘IAT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘IAT’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IAT/new/IAT.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ibmdbR

Version: 1.49.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ibmdbR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iBMQ

Version: 1.20.0

## In both

*   checking whether package ‘iBMQ’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/new/iBMQ.Rcheck/00install.out’ for details.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

## Installation

### Devel

```
* installing *source* package ‘iBMQ’ ...
checking for gcc... /usr/local/clang4/bin/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /usr/local/clang4/bin/clang accepts -g... yes
checking for /usr/local/clang4/bin/clang option to accept ISO C89... none needed
checking how to run the C preprocessor... /usr/local/clang4/bin/clang -E
checking for pkg-config... no
checking for GSL... checking for gsl-config... no
checking for GSL - version >= 1.2... no
*** The gsl-config script installed by GSL could not be found
*** If GSL was installed in PREFIX, make sure PREFIX/bin is in
*** your path, or set the GSL_CONFIG environment variable to the
*** full path to gsl-config.
configure: error: Cannot find Gnu Scientific Library >=1.6
ERROR: configuration failed for package ‘iBMQ’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/new/iBMQ.Rcheck/iBMQ’

```
### CRAN

```
* installing *source* package ‘iBMQ’ ...
checking for gcc... /usr/local/clang4/bin/clang
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether /usr/local/clang4/bin/clang accepts -g... yes
checking for /usr/local/clang4/bin/clang option to accept ISO C89... none needed
checking how to run the C preprocessor... /usr/local/clang4/bin/clang -E
checking for pkg-config... no
checking for GSL... checking for gsl-config... no
checking for GSL - version >= 1.2... no
*** The gsl-config script installed by GSL could not be found
*** If GSL was installed in PREFIX, make sure PREFIX/bin is in
*** your path, or set the GSL_CONFIG environment variable to the
*** full path to gsl-config.
configure: error: Cannot find Gnu Scientific Library >=1.6
ERROR: configuration failed for package ‘iBMQ’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/iBMQ/old/iBMQ.Rcheck/iBMQ’

```
# icarus

Version: 0.3.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Package Rglpk needed for this function to work. Please install it.
      1: calibration(data = data_ex2, marginMatrix = margins, colWeights = "poids", description = FALSE, 
             bounds = "min", method = "min") at testthat/test_calibration_small_dataset.R:127
      2: minBoundsCalib(Xs = matrixCal, d = weights, total = formattedMargins, q = rep(1, 
             nrow(matrixCal)), maxIter = maxIter, description = description, precisionBounds = precisionBounds, 
             forceSimplex = forceSimplex, forceBisection = forceBisection)
      3: solveMinBoundsCalib(Xs, d, total, q, maxIter, calibTolerance, description)
      4: stop("Package Rglpk needed for this function to work. Please install it.", call. = FALSE)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 59 SKIPPED: 0 FAILED: 1
      1. Error: Calibration on minimum bounds checks out (@test_calibration_small_dataset.R#127) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Rglpk’
    ```

# iCNV

Version: 1.0.0

## In both

*   checking whether package ‘iCNV’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iCNV/new/iCNV.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iCOBRA

Version: 1.8.0

## Newly broken

*   checking whether package ‘iCOBRA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘iCOBRA’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/iCOBRA/new/iCOBRA.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# icr

Version: 0.5.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ICtest

Version: 0.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: ggplot.ictest
    > ### Title: Scatterplot Matrix for a ictest Object using ggplot2
    > ### Aliases: ggplot.ictest
    > ### Keywords: hplot
    > 
    > ### ** Examples
    > 
    > # The data
    > X <- iris[, 1:4]
    > 
    > # The aesthetics variables
    > mapvar <- data.frame(iris[, 5])
    > colnames(mapvar) <- "species"
    > 
    > TestCov <- PCAasymp(X, k = 2)
    > ggplot(TestCov)
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~PC.1".  Please rename the columns or make a new column.
    Calls: ggplot ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: JADE
    Loading required package: ICS
    Loading required package: mvtnorm
    Loading required package: ggplot2
    Quitting from lines 122-127 (PCA.Rmd) 
    Error: processing vignette 'PCA.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~PC.1".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘fICA’
    ```

# IDE

Version: 0.1.0

## Newly broken

*   checking whether package ‘IDE’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘IDE’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IDE/new/IDE.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IDE_intro.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ideal

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ideal: no visible binding for '<<-' assignment to ‘ideal_env’
    ideal : <anonymous>: no visible binding for global variable ‘airway’
    ideal : <anonymous>: no visible binding for global variable ‘ideal_env’
    Undefined global functions or variables:
      airway ideal_env
    ```

# idealstan

Version: 0.2.7

## Newly broken

*   checking whether package ‘idealstan’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘idealstan’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/idealstan/new/idealstan.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in .local(object, ...) :
      some chains had errors; consider specifying chains = 1 to debug
    Warning in .local(object, ...) :
      some chains had errors; consider specifying chains = 1 to debug
    Quitting from lines 94-95 (Package_Introduction.Rmd) 
    Error: processing vignette 'Package_Introduction.Rmd' failed with diagnostics:
    Column `Rhats` must be a 1d atomic vector or a list
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.0Mb
      sub-directories of 1Mb or more:
        data   3.5Mb
        libs   3.6Mb
    ```

# idm

Version: 1.8.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ifaTools

Version: 0.14

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pritikin-schmidt.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IGP

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IHW

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   5.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ihw.default: no visible global function definition for ‘p.adjust’
    ihw_convex: no visible global function definition for ‘gurobi’
    ihw_internal: no visible global function definition for ‘p.adjust’
    ihw_milp: no visible global function definition for ‘str’
    ihw_milp: no visible global function definition for ‘gurobi’
    plot_decisionboundary: no visible binding for global variable ‘stratum’
    plot_decisionboundary: no visible binding for global variable
      ‘covariate’
    plot_decisionboundary: no visible binding for global variable ‘pvalue’
    plot_decisionboundary: no visible binding for global variable ‘fold’
    thresholds_ihwResult: no visible global function definition for
      ‘na.exclude’
    thresholds,ihwResult: no visible global function definition for
      ‘na.exclude’
    Undefined global functions or variables:
      covariate fold gurobi mcols mcols<- metadata metadata<- na.exclude
      p.adjust pvalue runif str stratum
    Consider adding
      importFrom("stats", "na.exclude", "p.adjust", "runif")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# IHWpaper

Version: 1.7.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 29-65 (BH-explanation.Rmd) 
    Error: processing vignette 'BH-explanation.Rmd' failed with diagnostics:
    Palette not found.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 23.9Mb
      sub-directories of 1Mb or more:
        doc      13.1Mb
        extdata   9.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    scott_fdrreg: no visible global function definition for ‘FDRreg’
    scott_fdrreg: no visible global function definition for ‘getFDR’
    sim_fun_eval: no visible binding for global variable ‘fdr_method’
    sim_fun_eval: no visible binding for global variable ‘fdr_pars’
    sim_fun_eval: no visible binding for global variable ‘FDP’
    sim_fun_eval: no visible binding for global variable ‘rj_ratio’
    sim_fun_eval: no visible binding for global variable ‘FPR’
    sim_fun_eval: no visible binding for global variable ‘FWER’
    Undefined global functions or variables:
      FDP FDRreg FPR FWER fdr_method fdr_pars getFDR rj_ratio
    ```

# iJRF

Version: 1.1-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# imageData

Version: 0.1-41

## In both

*   checking whether package ‘imageData’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/imageData/new/imageData.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘imageData’ ...
** package ‘imageData’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘imageData’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/imageData/new/imageData.Rcheck/imageData’

```
### CRAN

```
* installing *source* package ‘imageData’ ...
** package ‘imageData’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/imageData/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘imageData’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/imageData/old/imageData.Rcheck/imageData’

```
# imager

Version: 0.40.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘spatstat’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.6Mb
      sub-directories of 1Mb or more:
        data      1.4Mb
        doc       4.9Mb
        include   2.8Mb
        libs      3.9Mb
    ```

# imaginator

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IMAS

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    Warning in doTryCatch(return(expr), name, parentenv, handler) :
      length of NULL cannot be changed
    'select()' returned 1:1 mapping between keys and columns
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IMAS.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# imbalance

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Imetagene

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iml

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘lime’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘lime’
    ```

# ImmuneSpaceR

Version: 1.8.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 4 SKIPPED: 0 FAILED: 16
      1. Failure: all studies can loaded with empty string (@test-CreateConnection.R#33) 
      2. Failure: Existing study can be loaded (@test-CreateConnection.R#44) 
      3. Failure: ImmuneSignatures study can be loaded (@test-CreateConnection.R#49) 
      4. Failure: ImmuneSignatures study can be loaded (@test-CreateConnection.R#50) 
      5. Error: (unknown) (@test-dataIntegration.R#11) 
      6. Error: (unknown) (@test-getDataset.R#9) 
      7. Error: (unknown) (@test-getGEAnalysis.R#11) 
      8. Error: (unknown) (@test-getGEFiles.R#10) 
      9. Error: (unknown) (@test-getGEInputs.R#10) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    A .netrc file is required to connect to ImmuneSpace. For more information on how to create one, refer to the Configuration section of the introduction vignette.
    Quitting from lines 63-66 (Intro_to_ImmuneSpaceR.Rmd) 
    Error: processing vignette 'Intro_to_ImmuneSpaceR.Rmd' failed with diagnostics:
    SDY269 is not a valid study. 
     Use `verbose = TRUE` to see list of valid studies.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc   6.8Mb
    ```

# IMP

Version: 1.1

## Newly broken

*   checking whether package ‘IMP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘IMP’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IMP/new/IMP.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ImpulseDE2

Version: 1.4.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'ImpulseDE2_Tutorial.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotGenes: no visible global function definition for ‘error’
    plotGenes: no visible binding for global variable ‘normCounts’
    plotGenes: no visible binding for global variable ‘Condition’
    plotGenes: no visible binding for global variable ‘Batch’
    plotGenes: no visible binding for global variable ‘value’
    plotGenes: no visible binding for global variable ‘BatchFit’
    Undefined global functions or variables:
      Batch BatchFit Condition error normCounts value
    ```

# imputeTestbench

Version: 3.0.1

## Newly broken

*   checking whether package ‘imputeTestbench’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘imputeTestbench’
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘imputeTestbench’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/imputeTestbench/new/imputeTestbench.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IncDTW

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# incidence

Version: 1.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +  ## plot data and model (recommended)
    +  plot(i.7, fit=f)
    +  plot(i.7[1:25], fit=f)
    + 
    + 
    + ## EXAMPLE WITH 2 PHASES
    +  ## specifying the peak manually
    +  f2 <- fit(i.7, split=as.Date("2014-10-15"))
    +  f2
    +  plot(i.7, fit=f2)
    + 
    + ## finding the best 'peak' date
    + f3 <- fit_optim_split(i.7)
    + f3
    + plot(i.7, fit=f3$fit)
    + }
    Loading required package: outbreaks
    Error in as.Date.default(x.major_source, origin = "1970-01-01") : 
      do not know how to convert 'x.major_source' to class “Date”
    Calls: plot -> plot.incidence -> as.Date -> as.Date.default
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 80-86 (conversions.Rmd) 
    Error: processing vignette 'conversions.Rmd' failed with diagnostics:
    do not know how to convert 'x.major_source' to class "Date"
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# incR

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘rgeos’
      All declared Imports should be used.
    ```

# inctools

Version: 1.0.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IncucyteDRC

Version: 0.5.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iNEXT

Version: 2.0.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iNextPD

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# infer

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Information

Version: 0.0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# InformationValue

Version: 1.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    BugReports field should be the URL of a single webpage
    ```

# inlabru

Version: 2.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘INLA’
    ```

# InSilicoVA

Version: 1.1.5

## In both

*   checking whether package ‘InSilicoVA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/InSilicoVA/new/InSilicoVA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘InSilicoVA’ ...
** package ‘InSilicoVA’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘InSilicoVA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/InSilicoVA/new/InSilicoVA.Rcheck/InSilicoVA’

```
### CRAN

```
* installing *source* package ‘InSilicoVA’ ...
** package ‘InSilicoVA’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/InSilicoVA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘InSilicoVA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/InSilicoVA/old/InSilicoVA.Rcheck/InSilicoVA’

```
# InTAD

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IntClust

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘prodlim’
      All declared Imports should be used.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    12
    13
    1
    2
    3
    4
    5
    6
    7
    8
    9
    10
    11
    12
    13
    14
    15
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IntClustVignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# IntegratedJM

Version: 1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IntegratedMRF

Version: 1.1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# interactiveDisplay

Version: 1.18.0

## In both

*   checking dependencies in R code ... WARNING
    ```
    '::' or ':::' import not declared from: ‘.interactiveDisplayBase’
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'interactiveDisplay.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘rstudio’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for global variable ‘Var1’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘Var2’
    display,RangedSummarizedExperiment : <anonymous>: no visible binding
      for global variable ‘value’
    Undefined global functions or variables:
      AnnotationTrack GO.db GRanges GRangesList GenomeAxisTrack IRanges
      IdeogramTrack MRcounts Var1 Var2 as.dendrogram assays cim
      colorRampPalette coord_flip cutree dendrapply dev.off dist
      elementMetadata experimentData exprs fData hclust installed.packages
      is.leaf layout_circle legend libSize mcols pData plot plotFeature
      plotOrd plotTracks png rainbow ranges rowRanges seqlengths
      seqlengths<- seqlevels<- seqnames ucscGenomes value
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png",
                 "rainbow")
      importFrom("graphics", "legend", "plot")
      importFrom("stats", "as.dendrogram", "cutree", "dendrapply", "dist",
                 "hclust", "is.leaf")
      importFrom("utils", "installed.packages")
    to your NAMESPACE file.
    ```

# interflex

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# interplot

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# intsvy

Version: 2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IONiseR

Version: 2.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc       3.7Mb
        extdata   1.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘start_time’
    readFast5Summary.mc: no visible binding for global variable ‘duration’
    readFast5Summary.mc: no visible binding for global variable
      ‘num_events’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘baseCalledTemplate’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘baseCalledComplement’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘component’
    [,Fast5Summary-ANY-ANY-ANY: no visible binding for global variable
      ‘idx’
    show,Fast5Summary: no visible binding for global variable ‘full_2D’
    show,Fast5Summary: no visible binding for global variable ‘pass’
    Undefined global functions or variables:
      := AAAAA TTTTT accumulation baseCalledComplement baseCalledTemplate
      bases_called category channel circleFun component duration error freq
      full_2D group hour idx matrixCol matrixRow meanZValue mean_value
      median_signal minute mux name nbases new_reads num_events oddEven
      pass pentamer rbindlist readIDs seq_length start_time time_bin
      time_group x y zvalue
    ```

# ipft

Version: 0.7.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iprior

Version: 0.7.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# IPtoCountry

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ipumsr

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# IrisSpatialFeatures

Version: 1.3.0

## In both

*   checking whether package ‘IrisSpatialFeatures’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IrisSpatialFeatures/new/IrisSpatialFeatures.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘IrisSpatialFeatures’ ...
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c interaction.cpp -o interaction.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c masking.cpp -o masking.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o IrisSpatialFeatures.so RcppExports.o interaction.o masking.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/IrisSpatialFeatures/new/IrisSpatialFeatures.Rcheck/IrisSpatialFeatures/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘IrisSpatialFeatures’
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘IrisSpatialFeatures’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/IrisSpatialFeatures/new/IrisSpatialFeatures.Rcheck/IrisSpatialFeatures’

```
### CRAN

```
* installing *source* package ‘IrisSpatialFeatures’ ...
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c interaction.cpp -o interaction.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/IrisSpatialFeatures/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c masking.cpp -o masking.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o IrisSpatialFeatures.so RcppExports.o interaction.o masking.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/IrisSpatialFeatures/old/IrisSpatialFeatures.Rcheck/IrisSpatialFeatures/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘IrisSpatialFeatures’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/IrisSpatialFeatures/old/IrisSpatialFeatures.Rcheck/IrisSpatialFeatures’

```
# iSEE

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘ExperimentHub’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘S4Vectors:::selectSome’
      See the note in ?`:::` about the use of this operator.
    ```

# isobar

Version: 1.26.0

## In both

*   checking Rd cross-references ... WARNING
    ```
    Missing link or links in documentation object 'IBSpectra-class.Rd':
      ‘[MSnbase]{MSnbase}’
    
    See section 'Cross-references' in the 'Writing R Extensions' manual.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   1.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘.as.matrix’ ‘.as.vect’ ‘.convertPeptideModif’
      ‘.proteinGroupAsConciseDataFrame’ ‘.read.idfile’ ‘.sum.bool’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,IBSpectra-MSnSet: no visible binding for global variable ‘o’
    coerce,MSnSet-IBSpectra: no visible global function definition for
      ‘qual’
    df,Tlsd: no visible global function definition for ‘param’
    estimateRatio,IBSpectra-ANY-missing-missing-character-missing: no
      visible binding for global variable ‘i’
    estimateRatio,IBSpectra-ANY-missing-missing-missing-character: no
      visible binding for global variable ‘i’
    estimateRatioNumeric,numeric-numeric-NoiseModel: no visible binding for
      global variable ‘center.var’
    location,Tlsd: no visible global function definition for ‘param’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘pch’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘noise.model.col’
    plotRatio,IBSpectra-character-character-character: no visible binding
      for global variable ‘pch.p’
    scale,Tlsd: no visible global function definition for ‘param’
    Undefined global functions or variables:
      center.var d g i mz name noise.model.col o param pch pch.p peptide
      qual ratio type
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Welcome to isobar (v 1.26.0)
       'openVignette("isobar")' and '?isobar' provide help on usage.
    
    
    Attaching package: ‘isobar’
    
    The following object is masked from ‘package:BiocGenerics’:
    
        normalize
    
    The following object is masked from ‘package:base’:
    
        paste0
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'isobar-devel.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# IsoformSwitchAnalyzeR

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc   3.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    switchPlotTranscript: no visible global function definition for
      'queryHits'
    switchPlotTranscript: no visible binding for global variable 'y'
    switchPlotTranscript: no visible binding for global variable 'yend'
    switchPlotTranscript: no visible binding for global variable 'x'
    switchPlotTranscript: no visible binding for global variable 'ymin'
    switchPlotTranscript: no visible binding for global variable 'xmin'
    switchPlotTranscript: no visible binding for global variable 'ymax'
    switchPlotTranscript: no visible binding for global variable 'xmax'
    switchPlotTranscript: no visible binding for global variable 'Domain'
    Undefined global functions or variables:
      AStype CI_down CI_hi CI_low CI_up Comparison Condition Domain IF Rle
      Significant feature2 fisher.test geneFraction gene_expression
      grid.draw idNr isoFraction isoform_feature nrGenesWithConsequences
      nrIsoWithConsequences plotViewport popViewport propCiHi propCiLo
      propOfRelevantEvents propUp propUpCiHi propUpCiLo queryHits sigEval
      sigLevel sigLevelPos significance splicingResult switchConsequence
      value variable x xmax xmin y yend ymax ymin
    Consider adding
      importFrom("stats", "fisher.test")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘pheatmap’
    ```

# isomiRs

Version: 1.8.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: isoPLSDAplot
    > ### Title: Plot components from isoPLSDA analysis (pairs plot)
    > ### Aliases: isoPLSDAplot
    > 
    > ### ** Examples
    > 
    > data(mirData)
    > # Only miRNAs with > 10 reads in all samples.
    > ids <- isoCounts(mirData, minc=10, mins=6)
    > ids <- isoNorm(ids, formula=~group)
    converting counts to integer mode
    -- note: fitType='parametric', but the dispersion trend was not well captured by the
       function: y = a/x + b, and a local regression fit was automatically substituted.
       specify fitType='local' or 'mean' to avoid this message next time.
    > pls.ids <- isoPLSDA(ids, "group", nperm = 2)
    > isoPLSDAplot(pls.ids)
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~t1", "~condition".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~t1", "~condition".  Please rename the columns or make a new column.
    Calls: isoPLSDAplot ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      11: make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name), mapping = sectionAes)
      12: stop(paste("variables: ", paste(shQuote(nonCallNames, type = "cmd"), sep = ", "), 
             " have non standard format: ", paste(shQuote(unlist(mapping[nonCallVals]), type = "cmd"), 
                 collapse = ", "), ".  Please rename the columns or make a new column.", sep = ""))
      
      Dimmension of cor matrix: 20 20 
      Dimmension of cor matrix: 3 2 
      Dimmension of cor matrix: 3 2 
      Dimmension of cor matrix: 0 0 
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 27 SKIPPED: 0 FAILED: 1
      1. Error: psl-da (@test_basic.R#24) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following objects are masked from 'package:matrixStats':
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following objects are masked from 'package:base':
    
        aperm, apply
    
    converting counts to integer mode
    converting counts to integer mode
    converting counts to integer mode
    converting counts to integer mode
    converting counts to integer mode
    -- note: fitType='parametric', but the dispersion trend was not well captured by the
       function: y = a/x + b, and a local regression fit was automatically substituted.
       specify fitType='local' or 'mean' to avoid this message next time.
    Quitting from lines 245-249 (isomiRs.Rmd) 
    Error: processing vignette 'isomiRs.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~t1", "~condition".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~t1", "~condition".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.7Mb
      sub-directories of 1Mb or more:
        data   7.1Mb
        doc    1.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘tidyr’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mrna_de’
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mirna_se’
    .predict_correlate_mirna_targets: no visible binding for global
      variable ‘mrna_se’
    .run_enricher: no visible binding for global variable ‘sel_genes’
    .run_enricher: no visible global function definition for ‘enrichGO’
    .run_enricher: no visible binding for global variable ‘Count’
    .viz_mirna_gene_enrichment: no visible binding for global variable ‘X1’
    isoNetwork: no visible binding for global variable ‘Description’
    isoNetwork: no visible binding for global variable ‘geneID’
    isoPlotNet: no visible binding for global variable ‘ngene’
    Undefined global functions or variables:
      Count Description X1 enrichGO geneID mirna_se mrna_de mrna_se ngene
      sel_genes
    ```

# itan

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# iteremoval

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ITGM

Version: 0.6

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Fgmutils’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# ITNr

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘animation’ ‘comtradr’ ‘ndtv’ ‘statnet’ ‘xergm’
      All declared Imports should be used.
    ```

# IVAS

Version: 2.0.0

## In both

*   checking whether package ‘IVAS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: next used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/IVAS/new/IVAS.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        expand.grid
    
    Loading required package: IRanges
    Loading required package: GenomeInfoDb
    Loading required package: GenomicRanges
    Loading required package: AnnotationDbi
    Loading required package: Biobase
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Loading required package: ggplot2
    Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ggfortify’
    'select()' returned 1:1 mapping between keys and columns
    'select()' returned 1:1 mapping between keys and columns
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'IVAS.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ivmodel

Version: 1.7.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ivygapSE

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'ivygapSE.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.6Mb
      sub-directories of 1Mb or more:
        data  13.9Mb
        doc    2.4Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    exprByType: no visible binding for global variable ‘ivySE’
    ivyGlimpse: no visible binding for global variable ‘ivySE’
    ivyGlimpse : server: no visible binding for global variable ‘ivySE’
    Undefined global functions or variables:
      ivySE
    ```

# iWISA

Version: 1.0-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# jcext

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# jcolors

Version: 0.0.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘jcolors-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: theme_dark_bg
    > ### Title: minimal theme for dark backgrounds
    > ### Aliases: theme_dark_bg theme_light_bg
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > 
    > p <- ggplot(mtcars) + geom_point(aes(x = wt, y = mpg,
    +          colour = factor(gear))) + facet_grid(vs~am)
    > p + theme_dark_bg()
    Error in if (theme$panel.ontop) { : argument is of length zero
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# jmv

Version: 0.8.6.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BayesFactor’ ‘GGally’ ‘GPArotation’ ‘MASS’ ‘PMCMR’ ‘R6’ ‘ROCR’
      ‘afex’ ‘car’ ‘emmeans’ ‘ggridges’ ‘lavaan’ ‘multcomp’ ‘mvnormtest’
      ‘nnet’ ‘psych’ ‘vcd’ ‘vcdExtra’
      All declared Imports should be used.
    ```

# jmvcore

Version: 0.8.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# joineRmeta

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘nlme’
      All declared Imports should be used.
    ```

# joineRML

Version: 0.4.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'technical.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# jtools

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘jtools-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: cat_plot
    > ### Title: Plot interaction effects between categorical predictors.
    > ### Aliases: cat_plot
    > 
    > ### ** Examples
    > 
    > 
    > library(ggplot2)
    > fit <- lm(price ~ cut * color, data = diamonds)
    > cat_plot(fit, pred = color, modx = cut, interval = TRUE)
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to linetype
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 359 SKIPPED: 0 FAILED: 27
      1. Error: plot.distributions works (@test-export-summs.R#272) 
      2. Error: cat_plot handles simple plot (bar) (@test-misc.R#398) 
      3. Error: cat_plot handles intervals (bar) (@test-misc.R#403) 
      4. Error: cat_plot handles plotted points (bar) (@test-misc.R#409) 
      5. Error: cat_plot handles simple plot (line) (@test-misc.R#414) 
      6. Error: cat_plot handles intervals (line) (@test-misc.R#420) 
      7. Error: cat_plot handles plotted points (line) (@test-misc.R#426) 
      8. Error: cat_plot handles point.shape (line) (@test-misc.R#432) 
      9. Error: cat_plot handles simple plot (point) (@test-misc.R#444) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 67-68 (categorical.Rmd) 
    Error: processing vignette 'categorical.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to linetype
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘effects’, ‘wec’, ‘rockchalk’, ‘pequod’, ‘piecewiseSEM’
    ```

# JWileymisc

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# kamila

Version: 0.1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘mclust’
      All declared Imports should be used.
    ```

# kdetrees

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    as.matrix.multiPhylo : tip2tip: no visible global function definition
      for ‘cophenetic’
    bw.nn : <anonymous>: no visible global function definition for
      ‘quantile’
    dist.diss: no visible binding for global variable ‘median’
    dist.diss: no visible global function definition for ‘dist’
    hist.kdetrees: no visible binding for global variable ‘density’
    kdetrees : cutoff: no visible global function definition for ‘quantile’
    kdetrees.complete: no visible global function definition for ‘plot’
    kdetrees.complete: no visible global function definition for ‘hist’
    kdetrees.complete: no visible global function definition for
      ‘write.csv’
    plot.kdetrees: no visible binding for global variable ‘density’
    Undefined global functions or variables:
      cophenetic density dist hist median plot quantile write.csv
    Consider adding
      importFrom("graphics", "hist", "plot")
      importFrom("stats", "cophenetic", "density", "dist", "median",
                 "quantile")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'kdetrees.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# kdtools

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# keras

Version: 2.1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# kerasformula

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# kfigr

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# kgschart

Version: 1.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# kissDE

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Trying to fit models on data...
    This can be a time-consuming step, so do not hesitate to have 
        a look at the very well-written vignette !
    Computing pvalues...
    Computing size of the effect and last cutoffs...
    Pre-processing the data...
    Trying to fit models on data...
    This can be a time-consuming step, so do not hesitate to have 
        a look at the very well-written vignette !
    Computing pvalues...
    Computing size of the effect and last cutoffs...
    Pre-processing the data...
    Trying to fit models on data...
    This can be a time-consuming step, so do not hesitate to have 
        a look at the very well-written vignette !
    Computing pvalues...
    Computing size of the effect and last cutoffs...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'kissDE.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# kmed

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# knitrBootstrap

Version: 1.0.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 51-55 (cars.Rmd) 
    Error: processing vignette 'cars.Rmd' failed with diagnostics:
    Aesthetics must be either length 1 or the same as the data (32): x, y, method, formula, colour
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# konfound

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# KraljicMatrix

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘tibble’
      All declared Imports should be used.
    ```

# KSD

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# l1kdeconv

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Lahman

Version: 6.0-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        data   7.4Mb
    ```

# LambertW

Version: 0.6.4

## In both

*   checking whether package ‘LambertW’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      W_gamma_Cpp.cpp:14:39: warning: explicitly assigning value of variable of type 'int' to itself [-Wself-assign]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/LambertW/new/LambertW.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LANDD

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    getGO: no visible global function definition for ‘new’
    getGO: no visible global function definition for ‘getGeneric’
    graph.kd: no visible global function definition for ‘as’
    Undefined global functions or variables:
      as getGeneric new
    Consider adding
      importFrom("methods", "as", "getGeneric", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# landscapetools

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lans2r

Version: 1.0.5

## Newly broken

*   checking whether package ‘lans2r’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘lans2r’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/lans2r/new/lans2r.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# largeVis

Version: 0.2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        libs           1.6Mb
        testdata       1.8Mb
        vignettedata   2.0Mb
    ```

# latex2exp

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lavaSearch2

Version: 1.3.3

## Newly broken

*   checking whether package ‘lavaSearch2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘lava::vars’ when loading ‘lavaSearch2’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/lavaSearch2/new/lavaSearch2.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LBSPR

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lda

Version: 1.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ldatuning

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LDheatmap

Version: 0.99-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LDheatmap.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ldhmm

Version: 0.4.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LEAP

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LEAP_Vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# learningCurve

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘scales’ ‘tibble’
      All declared Imports should be used.
    ```

# learnstats

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    qqsim: no visible global function definition for ‘mtext’
    qqsim: no visible global function definition for ‘jpeg’
    qqsim: no visible global function definition for ‘dev.off’
    timeseressim: no visible global function definition for ‘rnorm’
    timeseressim: no visible global function definition for ‘par’
    timeseressim: no visible global function definition for ‘plot’
    timeseressim: no visible global function definition for ‘mtext’
    timeseressim: no visible global function definition for ‘jpeg’
    timeseressim: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      axis dchisq dev.off dnorm dt hist jpeg legend mtext par pchisq pf
      plot pnorm polygon pt qchisq qf qqline qqnorm qt rchisq rnorm rt
      runif segments title
    Consider adding
      importFrom("grDevices", "dev.off", "jpeg")
      importFrom("graphics", "axis", "hist", "legend", "mtext", "par",
                 "plot", "polygon", "segments", "title")
      importFrom("stats", "dchisq", "dnorm", "dt", "pchisq", "pf", "pnorm",
                 "pt", "qchisq", "qf", "qqline", "qqnorm", "qt", "rchisq",
                 "rnorm", "rt", "runif")
    to your NAMESPACE file.
    ```

# LedPred

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components which are templates and need '+ file LICENSE':
      MIT
    ```

*   checking R code for possible problems ... NOTE
    ```
    .plotCostGamma: no visible global function definition for ‘title’
    .plotCostGamma: no visible global function definition for ‘points’
    mapFeaturesToCRMs: no visible global function definition for
      ‘write.table’
    mapFeaturesToCRMs: no visible global function definition for
      ‘read.table’
    mapFeaturesToCRMs: no visible global function definition for
      ‘download.file’
    scoreData: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      download.file points read.table title write.table
    Consider adding
      importFrom("graphics", "points", "title")
      importFrom("utils", "download.file", "read.table", "write.table")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: e1071
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'LedPred.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# lemon

Version: 0.3.3

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 60-61 (facet-rep-labels.Rmd) 
    Error: processing vignette 'facet-rep-labels.Rmd' failed with diagnostics:
    Free scales are only supported with `coord_cartesian()` and `coord_flip()`
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LexisPlotR

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lfa

Version: 1.10.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'lfa':
    lfa
      Code: function(X, d, adjustments = NULL, override = FALSE, safety =
                     FALSE, ploidy = 2)
      Docs: function(X, d, adjustments = NULL, override = FALSE, safety =
                     FALSE)
      Argument names in code not in docs:
        ploidy
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'lfa.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    compute.nulls: no visible global function definition for ‘rbinom’
    lfa: no visible global function definition for ‘complete.cases’
    lfa: no visible global function definition for ‘residuals’
    lfa: no visible global function definition for ‘lm’
    lreg: no visible global function definition for ‘glm’
    read.bed: no visible global function definition for ‘read.table’
    trunc.svd: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      complete.cases glm lm rbinom read.table residuals rnorm
    Consider adding
      importFrom("stats", "complete.cases", "glm", "lm", "rbinom",
                 "residuals", "rnorm")
      importFrom("utils", "read.table")
    to your NAMESPACE file.
    ```

# lg

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lifelogr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# likeLTD

Version: 6.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    Loading required package: likeLTD
    Loading required package: DEoptim
    Loading required package: parallel
    
    DEoptim package
    Differential Evolution algorithm in R
    Authors: D. Ardia, K. Mullen, B. Peterson and J. Ulrich
    
    Loading required package: ggplot2
    Loading required package: gtools
    Loading required package: rtf
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'usage.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# likert

Version: 1.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: xtable
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'likert-xtable.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# lime

Version: 0.4.0

## Newly fixed

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      1: h2o.importFile(path) at testthat/test-h2o.R:12
      2: h2o.importFolder(path, pattern = "", destination_frame = destination_frame, parse, 
             header, sep, col.names, col.types, na.strings = na.strings, decrypt_tool = decrypt_tool)
      3: .h2o.__remoteSend(.h2o.__IMPORT, path = path, pattern = pattern)
      4: .h2o.doSafeREST(h2oRestApiVersion = h2oRestApiVersion, urlSuffix = page, parms = .params, 
             method = method)
      5: stop(msg)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 28 SKIPPED: 0 FAILED: 1
      1. Error: H2OBinomialClassification: lime explanation only produces one entry per case and feature (@test-h2o.R#12) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘lime-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_superpixels
    > ### Title: Test super pixel segmentation
    > ### Aliases: plot_superpixels
    > 
    > ### ** Examples
    > 
    > image <- system.file('extdata', 'produce.png', package = 'lime')
    > 
    > # plot with default settings
    > plot_superpixels(image)
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

*   checking S3 generic/method consistency ... WARNING
    ```
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# LINC

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        doc   3.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    justlinc,matrix: no visible global function definition for ‘plot’
    Undefined global functions or variables:
      plot
    Consider adding
      importFrom("graphics", "plot")
    to your NAMESPACE file.
    ```

*   checking R/sysdata.rda ... NOTE
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                  old_size new_size compress
      sysdata.rda    449Kb    307Kb       xz
    ```

# lindia

Version: 0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LineagePulse

Version: 1.0.0

## In both

*   checking whether package ‘LineagePulse’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'evalLogLikGeneMM(vecCounts = vecCounts, ': unused arguments (matDispParam = matDispParam, matDropParam = matDropParam, matWeights = matWeights) 
      Note: possible error in 'evalLogLikGeneMM(vecCounts = vecCounts, ': unused arguments (matDispParam = do.call(cbind, lapply(seq_len(ncol(matWeights)), function(m) vecDispParam)), matDropParam = do.call(cbind, lapply(seq_len(ncol(matWeights)), function(m) vecPiParam)), matWeights = matWeights) 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/LineagePulse/new/LineagePulse.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'LineagePulse_Tutorial.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      function(m) vecPiParam)), matWeights = matWeights)
    evalLogLikMatrix : <anonymous>: possible error in scaNCells =
      length(vecCounts)): unused arguments (matDispParam = do.call(cbind,
      lapply(seq_len(ncol(matWeights)), function(m) vecDispParam)),
      matDropParam = do.call(cbind, lapply(seq_len(ncol(matWeights)),
      function(m) vecPiParam)), matWeights = matWeights)
    plotCellDensity: no visible binding for global variable ‘continuous’
    plotGene: no visible binding for global variable ‘x’
    plotGene: no visible binding for global variable ‘dropout_posterior’
    plotGene: no visible binding for global variable ‘groups’
    plotGene: no visible binding for global variable ‘dfAnnot’
    plotGene: no visible binding for global variable ‘mean_count’
    plotGene: no visible binding for global variable ‘quantile_25’
    plotGene: no visible binding for global variable ‘quantile_75’
    plotGene: no visible binding for global variable ‘model’
    plotGene: no visible binding for global variable ‘continuous’
    plotGene: no visible binding for global variable ‘trajectory_contour’
    plotGene: no visible binding for global variable ‘ncells’
    Undefined global functions or variables:
      continuous dfAnnot dropout_posterior groups lsDispModel mean_count
      model ncells quantile_25 quantile_75 trajectory_contour x
    ```

# linear.tools

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Linnorm

Version: 2.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Linnorm.HClust: no visible binding for global variable ‘x’
    Linnorm.HClust: no visible binding for global variable ‘y’
    Linnorm.HClust: no visible binding for global variable ‘xend’
    Linnorm.HClust: no visible binding for global variable ‘yend’
    Linnorm.HClust: no visible binding for global variable ‘cluster’
    Linnorm.HClust: no visible binding for global variable ‘X1’
    Linnorm.HClust: no visible binding for global variable ‘X2’
    Linnorm.HVar: no visible binding for global variable ‘SD’
    Linnorm.HVar: no visible binding for global variable ‘group’
    Undefined global functions or variables:
      SD X1 X2 cluster group x xend y yend
    ```

# listdtr

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# live

Version: 1.5.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# llama

Version: 0.9.1

## In both

*   checking whether package ‘llama’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/llama/new/llama.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘llama’ ...
** package ‘llama’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘llama’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/llama/new/llama.Rcheck/llama’

```
### CRAN

```
* installing *source* package ‘llama’ ...
** package ‘llama’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/llama/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘llama’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/llama/old/llama.Rcheck/llama’

```
# lme4

Version: 1.1-17

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        R          1.5Mb
        doc        2.4Mb
        testdata   1.6Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
      Please specify either 'title' or 'pagetitle' in the metadata.
      Falling back to 'lmerperf.utf8'
    Loading required package: Matrix
    Computing profile confidence intervals ...
    Computing bootstrap confidence intervals ...
    Loading required package: mgcv
    Loading required package: nlme
    
    Attaching package: 'nlme'
    
    The following object is masked from 'package:lme4':
    
        lmList
    
    This is mgcv 1.8-23. For overview type 'help("mgcv-package")'.
    This is gamm4 0.2-5
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'lmer.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# lmerTest

Version: 3.0-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lmms

Version: 1.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LocFDRPois

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    AnalyticalOptim: no visible global function definition for ‘optim’
    LLConstructor : LL: no visible global function definition for ‘dpois’
    MixtureDensity: no visible global function definition for ‘glm’
    MixtureDensity : f_hat: no visible global function definition for
      ‘predict’
    NullDensity : f0: no visible global function definition for ‘dpois’
    Undefined global functions or variables:
      dpois glm optim predict
    Consider adding
      importFrom("stats", "dpois", "glm", "optim", "predict")
    to your NAMESPACE file.
    ```

# logihist

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# logisticPCA

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LOGIT

Version: 1.3

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Display confusion or classification matrix following logistic
    > ###   regression using glm with binomial family.
    > ### Aliases: confusion_stat
    > ### Keywords: models
    > 
    > ### ** Examples
    > 
    > library(MASS)
    > library(LOGIT)
    > data(R84)
    > R84$cage <- R84$age - mean(R84$age)
    > R84$cdoc <- R84$docvis - mean(R84$docvis)
    > mylogit <- glm(outwork ~ cdoc + female + kids + cage + factor(edlevel),
    + family=binomial, data=R84)
    > mu <- predict(mylogit, type="response")
    > cutpoint<-ROCtest(mylogit, fold=10, type="Sensitivity")$cut
    > mu[mu>=cutpoint]<-1
    > mu[mu<cutpoint]<-0
    > confusion_stat(mu, R84$outwork)
    Error: `data` and `reference` should be factors with the same levels.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# logitnorm

Version: 0.8.35

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Logolas

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lolR

Version: 2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# longCatEDA

Version: 0.31

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# longRPart2

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lpdensity

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lsbclust

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lsl

Version: 0.5.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lslx

Version: 0.6.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggplot2’ ‘lavaan’
      All declared Imports should be used.
    ```

# lspartition

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lspline

Version: 1.0-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ltbayes

Version: 0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# lvplot

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# LymphoSeq

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   5.5Mb
    ```

# m2b

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘e1071’ ‘utils’
      All declared Imports should be used.
    ```

# M3C

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'M3Cvignette.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.6Mb
      sub-directories of 1Mb or more:
        doc   7.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    M3Cdendcomputations: no visible global function definition for
      ‘dev.off’
    M3Creal: no visible global function definition for ‘colorRampPalette’
    M3Creal: no visible global function definition for ‘png’
    M3Creal: no visible global function definition for ‘dev.off’
    clustersim: no visible binding for global variable ‘X3’
    clustersim: no visible binding for global variable ‘PC1’
    clustersim: no visible binding for global variable ‘PC2’
    clustersim: no visible global function definition for ‘png’
    clustersim: no visible global function definition for ‘dev.off’
    myPal: no visible global function definition for ‘rgb’
    pca: no visible binding for global variable ‘PC1’
    pca: no visible binding for global variable ‘PC2’
    pca: no visible global function definition for ‘png’
    pca: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      K PAC_SCORE PC1 PC2 P_SCORE RCSI X3 colorRampPalette consensusindex
      dev.off i k png rgb
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png", "rgb")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' imports not declared from:
      ‘ggsci’ ‘gplots’
    'library' or 'require' calls not declared from:
      ‘ggsci’ ‘gplots’
    ```

# MAc

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    
    Packages which this enhances but not available for checking:
      ‘compute.es’ ‘irr’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘R2wd’ ‘ggplot2’ ‘metafor’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    wd.mareg: no visible global function definition for ‘wdGet’
    wd.mareg: no visible global function definition for ‘wdNewDoc’
    wd.mareg: no visible global function definition for ‘wdHeading’
    wd.mareg: no visible global function definition for ‘wdTable’
    wd.omni: no visible global function definition for ‘wdGet’
    wd.omni: no visible global function definition for ‘wdNewDoc’
    wd.omni: no visible global function definition for ‘wdHeading’
    wd.omni: no visible global function definition for ‘wdTable’
    Undefined global functions or variables:
      TukeyHSD aes aggregate anova aov contrasts facet_wrap g geom_boxplot
      geom_errorbarh geom_jitter geom_point geom_smooth geom_vline ggplot
      id l.ci95 lm model.extract model.matrix model.response na.omit opts
      pchisq pf pnorm printCoefmat qf qt r rma scale_y_continuous se.z
      se.z.tau stat_abline stat_summary u.ci95 var var.g wdGet wdHeading
      wdNewDoc wdTable wi wi.tau xlab xlim ylab ylim z
    Consider adding
      importFrom("stats", "TukeyHSD", "aggregate", "anova", "aov",
                 "contrasts", "lm", "model.extract", "model.matrix",
                 "model.response", "na.omit", "pchisq", "pf", "pnorm",
                 "printCoefmat", "qf", "qt", "var")
    to your NAMESPACE file.
    ```

# macleish

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MACPET

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MAd

Version: 0.8-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    
    Packages which this enhances but not available for checking:
      ‘compute.es’ ‘irr’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘R2wd’ ‘ggplot2’ ‘metafor’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    omni: no visible global function definition for ‘pnorm’
    omni: no visible global function definition for ‘pchisq’
    p.ancova_to_d1: no visible global function definition for ‘qt’
    p.ancova_to_d2: no visible global function definition for ‘qt’
    p_to_d1: no visible global function definition for ‘qt’
    p_to_d2: no visible global function definition for ‘qt’
    print.summary.mareg: no visible global function definition for
      ‘printCoefmat’
    robustSE: no visible global function definition for ‘residuals’
    robustSE: no visible global function definition for ‘pt’
    robustSE: no visible global function definition for ‘qt’
    Undefined global functions or variables:
      TukeyHSD aggregate anova aov contrasts model.extract model.matrix
      model.response na.omit pchisq pf pnorm printCoefmat pt qf qt
      residuals var
    Consider adding
      importFrom("stats", "TukeyHSD", "aggregate", "anova", "aov",
                 "contrasts", "model.extract", "model.matrix",
                 "model.response", "na.omit", "pchisq", "pf", "pnorm",
                 "printCoefmat", "pt", "qf", "qt", "residuals", "var")
    to your NAMESPACE file.
    ```

# maddison

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mafs

Version: 0.0.3

## Newly broken

*   checking whether package ‘mafs’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘mafs’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mafs/new/mafs.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘cmprsk’ ‘colorspace’ ‘etm’ ‘fracdiff’ ‘gtable’ ‘munsell’
      ‘numDeriv’ ‘plyr’ ‘quadprog’ ‘scales’ ‘timeDate’ ‘tseries’ ‘zoo’
      All declared Imports should be used.
    ```

# maftools

Version: 1.6.07

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.9Mb
      sub-directories of 1Mb or more:
        doc       6.0Mb
        extdata   5.1Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      aa.length adjPval amp anno.df assembly_version bg boxStat cf
      chromosome chromosome_end chromosome_start ci.low ci.up clusters
      cohort con con.class consequence_type conv count count2 cytoband
      distance domain_lenght downstream dp endDist ens_id fdr fisher_pvalue
      fract fract_muts_in_clusters fraction fraction_APOBEC_mutations fs
      g1_muts g1_title g1_tot g2_muts g2_title g2_tot gene gene1 gene2
      gene_affected gg.fp hgnc_symbol i.End_Position i.Start_Position
      icgc_sample_id id idx lab labPos labThis label loc log10OR
      log10OR_high log10OR_low minDiff mut.load.p mutated_from_allele
      mutated_to_allele mutations muts_in_clusters nGenes nMut nMuts
      nSamples nVars n_A n_C n_C>G_and_C>T n_G n_T n_mutated_Feature
      n_mutated_Feature1 n_mutated_Feature2 n_mutated_group1
      n_mutated_group2 n_mutations nonApobec non_APOBEC_mutations numMat or
      pValue p_value pair peakID pfam poissonFdr pos2 posRounded protein.ID
      pval qvalues reference_genome_allele refseq.ID sampleId
      sequencing_strategy significant site startDist statFontSize survLower
      survProb survUp tCw tCw_to_A tCw_to_G tCw_to_G+tCw_to_T tCw_to_T tFdr
      t_alt_count t_ref_count t_vaf tcw th tot total trinucleotide uid
      updown upstream value variable variantId verification_platform
      verification_status wGa wGa_to_A wGa_to_C wGa_to_T wga ybig ysmall
      ystart
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘pheatmap’
    'library' or 'require' call not declared from: ‘pheatmap’
    ```

# MAGeCKFlute

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ScatterView: no visible binding for global variable ‘group’
    SquareView: no visible binding for global variable ‘Control’
    SquareView: no visible binding for global variable ‘Treatment’
    SquareView: no visible binding for global variable ‘group’
    SquareView: no visible binding for global variable ‘Gene’
    ViolinView: no visible binding for global variable ‘variable’
    ViolinView: no visible binding for global variable ‘value’
    enrich.DAVID: no visible global function definition for ‘qvalue’
    enrich.GSE: no visible global function definition for ‘gseDO’
    enrich.GSE: no visible global function definition for ‘gseNCG’
    enrich.ORT: no visible global function definition for ‘enrichDO’
    enrich.ORT: no visible global function definition for ‘enrichNCG’
    getOrg: no visible binding for global variable ‘bods’
    Undefined global functions or variables:
      ..density.. Control Count Diff Gene NES Name PC1 PC2 Rank Treatment
      Zuber_Essential bods enrichDO enrichNCG group gseDO gseNCG keggConv
      logP par parseKGML2Graph2 plot.new qvalue r rn.list shape value
      variable viewport x
    Consider adding
      importFrom("graphics", "par", "plot.new")
    to your NAMESPACE file.
    ```

# magick

Version: 1.8

## In both

*   checking whether package ‘magick’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/magick/new/magick.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘rsvg’ ‘tesseract’
    ```

## Installation

### Devel

```
* installing *source* package ‘magick’ ...
** package ‘magick’ successfully unpacked and MD5 sums checked
/usr/local/bin/Magick++-config: line 44: /usr/local/opt/pkg-config/bin/pkg-config: No such file or directory
/usr/local/bin/Magick++-config: line 56: /usr/local/opt/pkg-config/bin/pkg-config: No such file or directory
Using PKG_CFLAGS=-I/usr/local/opt/imagemagick@6/include/ImageMagick-6 -DMAGICKCORE_QUANTUM_DEPTH=16 -DMAGICKCORE_HDRI_ENABLE=0
Using PKG_LIBS=-L/usr/local/opt/imagemagick@6/lib -lMagick++-6.Q16
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because Magick++ was not found. Try installing:
 - deb: 'libmagick++-dev' (Debian, Ubuntu)
 - rpm: 'ImageMagick-c++-devel' (Fedora, CentOS, RHEL)
 - csw: 'imagemagick_dev' (Solaris)
On MacOS it is recommended to use install ImageMagick-6 from homebrew
with extra support for fontconfig and rsvg rendering:
   brew reinstall imagemagick@6 --with-fontconfig --with-librsvg
For Ubuntu Trusty (14.04) or Xenial (16.04) use our PPA:
   sudo add-apt-repository -y ppa:opencpu/imagemagick
   sudo apt-get update
   sudo apt-get install -y libmagick++-dev
If Magick++ is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a Magick++.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘magick’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/magick/new/magick.Rcheck/magick’

```
### CRAN

```
* installing *source* package ‘magick’ ...
** package ‘magick’ successfully unpacked and MD5 sums checked
/usr/local/bin/Magick++-config: line 44: /usr/local/opt/pkg-config/bin/pkg-config: No such file or directory
/usr/local/bin/Magick++-config: line 56: /usr/local/opt/pkg-config/bin/pkg-config: No such file or directory
Using PKG_CFLAGS=-I/usr/local/opt/imagemagick@6/include/ImageMagick-6 -DMAGICKCORE_QUANTUM_DEPTH=16 -DMAGICKCORE_HDRI_ENABLE=0
Using PKG_LIBS=-L/usr/local/opt/imagemagick@6/lib -lMagick++-6.Q16
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because Magick++ was not found. Try installing:
 - deb: 'libmagick++-dev' (Debian, Ubuntu)
 - rpm: 'ImageMagick-c++-devel' (Fedora, CentOS, RHEL)
 - csw: 'imagemagick_dev' (Solaris)
On MacOS it is recommended to use install ImageMagick-6 from homebrew
with extra support for fontconfig and rsvg rendering:
   brew reinstall imagemagick@6 --with-fontconfig --with-librsvg
For Ubuntu Trusty (14.04) or Xenial (16.04) use our PPA:
   sudo add-apt-repository -y ppa:opencpu/imagemagick
   sudo apt-get update
   sudo apt-get install -y libmagick++-dev
If Magick++ is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a Magick++.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘magick’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/magick/old/magick.Rcheck/magick’

```
# malariaAtlas

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grid’
      All declared Imports should be used.
    ```

# manhattanly

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘qqman’
    ```

# mapfuser

Version: 0.1.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘LPmerge’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# mapr

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 164 marked UTF-8 strings
    ```

# margins

Version: 0.3.20

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    m <- lm(mpg ~ wt + am + factor(cyl), data = mtcars)
    margins(m)
    summary(margins(m))
    Chunk margins3a:
      Average marginal effects 
    
    m <- lm(mpg ~ wt * I(wt^2) * hp * I(hp^2), data = mtcars)
    margins(m)
    Chunk margins3a:
      lm(formula = mpg ~ wt * I(wt^2) * hp * I(hp^2), data = mtcars) 
    
    m <- lm(mpg ~ wt * I(wt^2) * hp * I(hp^2), data = mtcars)
    margins(m)
    
    Number of messages:
    message 
          6 
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TechnicalDetails.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘AER’ ‘betareg’ ‘ordinal’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘betareg’, ‘AER’
    ```

# marked

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Starting optimization for 7 parameters...
    Computing hessian...
    Model: PROBITCJS
    
    Processing data...
    
    Creating design data...
    
    Fitting model
    
    Computing initial parameter estimates
    
    probitCJS MCMC beginning...
    p model = ~time + sex
    phi model = ~Flood
    MCMC complete, processing output...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'markedVignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# marmap

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MARSS

Version: 3.10.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'EMDerivation.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MAST

Version: 1.6.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning: code chunks must not depend on the uncached chunk "data"
    Assuming data assay in position 1, with name et is log-transformed.
    Quitting from lines 93-106 (MAITAnalysis.Rmd) 
    Error: processing vignette 'MAITAnalysis.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~PC1", "~condition".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~PC1", "~condition".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.7Mb
      sub-directories of 1Mb or more:
        R      1.0Mb
        data   3.7Mb
        doc    3.7Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    assay_idx: no visible global function definition for ‘assayNames’
    collectResiduals: no visible global function definition for
      ‘assayNames’
    collectResiduals: no visible global function definition for ‘assay<-’
    collectResiduals: no visible global function definition for
      ‘assayNames<-’
    primerAverage: no visible global function definition for ‘assay<-’
    primerAverage: no visible global function definition for ‘rowData<-’
    assay<-,SingleCellAssay-missing: no visible global function definition
      for ‘assay<-’
    assayNames<-,SingleCellAssay-character: no visible global function
      definition for ‘assayNames’
    assayNames<-,SingleCellAssay-character: no visible global function
      definition for ‘assays<-’
    exprs<-,SingleCellAssay-ANY: no visible global function definition for
      ‘assay<-’
    Undefined global functions or variables:
      assay<- assayNames assayNames<- assays<- rowData<-
    ```

# matchingMarkets

Version: 0.3-5

## In both

*   checking whether package ‘matchingMarkets’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/matchingMarkets/new/matchingMarkets.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘matchingMarkets’ ...
** package ‘matchingMarkets’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c Options.cc -o Options.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c System.cc -o System.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c kprmatcher.cc -o kprmatcher.o
In file included from System.cc:24:
In file included from ../inst/include/System.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from Options.cc:20:
In file included from ../inst/include/Sort.h:24:
In file included from ../inst/include/Vec.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from Options.cc:21:
../inst/include/Options.h:147:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:143:19: note: initialize the variable 'end' to silence this warning
        char*  end;
                  ^
                   = nullptr
../inst/include/Options.h:207:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:204:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
../inst/include/Options.h:275:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:271:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
In file included from kprmatcher.cc:41:
In file included from ../inst/include/kprmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from kprmatcher.cc:41:
In file included from ../inst/include/kprmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from System.cc:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from Options.cc:24:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c params.cc -o params.o
5 warnings generated.
In file included from params.cc:32:
In file included from ../inst/include/Options.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c problem.cc -o problem.o
In file included from params.cc:32:
../inst/include/Options.h:147:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:143:19: note: initialize the variable 'end' to silence this warning
        char*  end;
                  ^
                   = nullptr
../inst/include/Options.h:207:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:204:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
../inst/include/Options.h:275:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:271:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
4 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c rcp.cc -o rcp.o
2 warnings generated.
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c rpmatcher.cc -o rpmatcher.o
In file included from problem.cc:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from problem.cc:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from rcp.cc:33:
In file included from ../inst/include/rcp.h:41:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from rcp.cc:33:
In file included from ../inst/include/rcp.h:41:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c runMatch.cpp -o runMatch.o
In file included from rpmatcher.cc:46:
In file included from ../inst/include/rpmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from rpmatcher.cc:46:
In file included from ../inst/include/rpmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Mat1.cpp -o stabit2Mat1.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Sel1.cpp -o stabit2Sel1.o
In file included from runMatch.cpp:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from runMatch.cpp:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Sel2.cpp -o stabit2Sel2.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabitSel2.cpp -o stabitSel2.o
In file included from stabit2Mat1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Mat1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabit2Sel1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Sel1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabit2Sel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Sel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabitSel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabitSel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
2 warnings generated.
2 warnings generated.
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o matchingMarkets.so Options.o RcppExports.o System.o kprmatcher.o params.o problem.o rcp.o rpmatcher.o runMatch.o stabit2Mat1.o stabit2Sel1.o stabit2Sel2.o stabitSel2.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -fopenmp -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/matchingMarkets/new/matchingMarkets.Rcheck/matchingMarkets/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘matchingMarkets’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/matchingMarkets/new/matchingMarkets.Rcheck/matchingMarkets’

```
### CRAN

```
* installing *source* package ‘matchingMarkets’ ...
** package ‘matchingMarkets’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c Options.cc -o Options.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c System.cc -o System.o
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c kprmatcher.cc -o kprmatcher.o
In file included from System.cc:24:
In file included from ../inst/include/System.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from Options.cc:20:
In file included from ../inst/include/Sort.h:24:
In file included from ../inst/include/Vec.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from Options.cc:21:
../inst/include/Options.h:147:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:143:19: note: initialize the variable 'end' to silence this warning
        char*  end;
                  ^
                   = nullptr
../inst/include/Options.h:207:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:204:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
../inst/include/Options.h:275:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:271:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
In file included from kprmatcher.cc:41:
In file included from ../inst/include/kprmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from kprmatcher.cc:41:
In file included from ../inst/include/kprmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from System.cc:25:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from Options.cc:24:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c params.cc -o params.o
5 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c problem.cc -o problem.o
In file included from params.cc:32:
In file included from ../inst/include/Options.h:28:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from params.cc:32:
../inst/include/Options.h:147:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:143:19: note: initialize the variable 'end' to silence this warning
        char*  end;
                  ^
                   = nullptr
../inst/include/Options.h:207:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:204:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
../inst/include/Options.h:275:13: warning: variable 'end' is uninitialized when used here [-Wuninitialized]
        if (end == NULL) 
            ^~~
../inst/include/Options.h:271:20: note: initialize the variable 'end' to silence this warning
        char*   end;
                   ^
                    = nullptr
4 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c rcp.cc -o rcp.o
2 warnings generated.
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from RcppExports.cpp:4:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c rpmatcher.cc -o rpmatcher.o
In file included from problem.cc:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from problem.cc:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from rcp.cc:33:
In file included from ../inst/include/rcp.h:41:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from rcp.cc:33:
In file included from ../inst/include/rcp.h:41:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c runMatch.cpp -o runMatch.o
In file included from rpmatcher.cc:46:
In file included from ../inst/include/rpmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from rpmatcher.cc:46:
In file included from ../inst/include/rpmatcher.h:36:
In file included from ../inst/include/damatcher.h:36:
In file included from ../inst/include/problem.h:43:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Mat1.cpp -o stabit2Mat1.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Sel1.cpp -o stabit2Sel1.o
In file included from runMatch.cpp:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from runMatch.cpp:1:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabit2Sel2.cpp -o stabit2Sel2.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppProgress/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include" -I/usr/local/include  -fopenmp -fPIC  -Wall -g -O2 -c stabitSel2.cpp -o stabitSel2.o
In file included from stabit2Mat1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Mat1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabit2Sel1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Sel1.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabit2Sel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabit2Sel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
In file included from stabitSel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:10: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#include <inttypes.h>                   // needed with g++-4.7 to declare intptr_t
         ^~~~~~~~~~~~
         <IntTypes.h>
In file included from stabitSel2.cpp:6:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/RcppArmadillo/include/RcppArmadillo.h:34:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp.h:74:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/sugar/sugar.h:28:
In file included from /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/Rcpp/include/Rcpp/hash/hash.h:25:
../inst/include/inttypes.h:34:13: warning: non-portable path to file '<IntTypes.h>'; specified path differs in case from file name on disk [-Wnonportable-include-path]
#   include <inttypes.h>
            ^~~~~~~~~~~~
            <IntTypes.h>
2 warnings generated.
2 warnings generated.
2 warnings generated.
2 warnings generated.
/usr/local/clang4/bin/clang++ -std=gnu++11 -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o matchingMarkets.so Options.o RcppExports.o System.o kprmatcher.o params.o problem.o rcp.o rpmatcher.o runMatch.o stabit2Mat1.o stabit2Sel1.o stabit2Sel2.o stabitSel2.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -fopenmp -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/matchingMarkets/old/matchingMarkets.Rcheck/matchingMarkets/libs
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/matchingMarkets/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘matchingMarkets’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/matchingMarkets/old/matchingMarkets.Rcheck/matchingMarkets’

```
# matrixStats

Version: 0.53.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mau

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MAVIS

Version: 1.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MaxentVariableSelection

Version: 1.0-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mbgraphic

Version: 1.0.0

## In both

*   checking whether package ‘mbgraphic’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mbgraphic/new/mbgraphic.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘mbgraphic’ ...
** package ‘mbgraphic’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c cmasum.cpp -o cmasum.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mbgraphic_init.c -o mbgraphic_init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c variableflip.cpp -o variableflip.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o mbgraphic.so RcppExports.o cmasum.o mbgraphic_init.o variableflip.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/mbgraphic/new/mbgraphic.Rcheck/mbgraphic/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘mbgraphic’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mbgraphic/new/mbgraphic.Rcheck/mbgraphic’

```
### CRAN

```
* installing *source* package ‘mbgraphic’ ...
** package ‘mbgraphic’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c cmasum.cpp -o cmasum.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mbgraphic_init.c -o mbgraphic_init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c variableflip.cpp -o variableflip.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o mbgraphic.so RcppExports.o cmasum.o mbgraphic_init.o variableflip.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/mbgraphic/old/mbgraphic.Rcheck/mbgraphic/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/mbgraphic/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘mbgraphic’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mbgraphic/old/mbgraphic.Rcheck/mbgraphic’

```
# MCbiclust

Version: 1.4.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Iteration	Correlation Score
    100 		 0.33875
    Iteration	Correlation Score
    100 		 0.35392
    > 
    > for(i in 1:3){
    +     CCLE.cor.vec[[i]] <-  CVEval(gem.part = CCLE.mito,
    +                                  gem.all = CCLE_small,
    +                                  seed = CCLE.seed[[i]],
    +                                 splits = 10)}
    > 
    > 
    > 
    > CCLE.cor.df <- (as.data.frame(CCLE.cor.vec))
    > 
    > CVPlot(cv.df = CCLE.cor.df, geneset.loc = mito.loc,
    +        geneset.name = "Mitochondrial",alpha1 = 0.5)
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~CV1".  Please rename the columns or make a new column.
    Calls: CVPlot ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    
    Attaching package: 'MASS'
    
    The following object is masked from 'package:dplyr':
    
        select
    
    Warning: Column `CCLE.name` joining factor and character vector, coercing into character vector
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `MCbiclust_vignette_files/figure-html/unnamed-chunk-38-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Quitting from lines 519-528 (MCbiclust_vignette.Rmd) 
    Error: processing vignette 'MCbiclust_vignette.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~CV1".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        data   3.1Mb
        doc    5.9Mb
    ```

# MCMC.OTU

Version: 1.0.10

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'cBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MCMC.qpcr

Version: 1.2.3

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'cBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mcMST

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘parallelMap’ ‘reshape2’
      All declared Imports should be used.
    ```

# Mcomp

Version: 2.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mcprofile

Version: 0.2-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mdp

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mdpeer

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘glmnet’
      All declared Imports should be used.
    ```

# mdsr

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        data   5.5Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2694 marked UTF-8 strings
    ```

# meaRtools

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        extdata   3.9Mb
    ```

# medicalrisk

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# medmod

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘ggplot2’ ‘lavaan’
      All declared Imports should be used.
    ```

# mekko

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# meltt

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mem

Version: 2.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# memapp

Version: 2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘RColorBrewer’ ‘RODBC’ ‘dplyr’ ‘formattable’ ‘ggplot2’
      ‘ggthemes’ ‘magrittr’ ‘mem’ ‘openxlsx’ ‘plotly’ ‘readxl’ ‘shinyBS’
      ‘shinydashboard’ ‘shinyjs’ ‘shinythemes’ ‘stringi’ ‘stringr’ ‘tidyr’
      All declared Imports should be used.
    ```

# meme

Version: 0.2.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
    OMP: Hint This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://openmp.llvm.org/
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        doc   4.9Mb
    ```

# memery

Version: 0.4.2

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(memery)
      Loading required package: showtext
      Loading required package: sysfonts
      Loading required package: showtextdb
      > 
      > test_check("memery")
      OMP: Error #15: Initializing libomp.dylib, but found libomp.dylib already initialized.
      OMP: Hint: This means that multiple copies of the OpenMP runtime have been linked into the program. That is dangerous, since it can degrade performance or cause incorrect results. The best thing to do is to ensure that only a single OpenMP runtime is linked into the process, e.g. by avoiding static linking of the OpenMP runtime in any library. As an unsafe, unsupported, undocumented workaround you can set the environment variable KMP_DUPLICATE_LIB_OK=TRUE to allow the program to continue to execute, but that may cause crashes or silently produce incorrect results. For more information, please see http://www.intel.com/software/products/support/.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘colourpicker’ ‘shinyBS’ ‘shinycssloaders’
      All declared Imports should be used.
    ```

# MendelianRandomization

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MergeGUI

Version: 0.2-1

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘cairoDevice’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# merTools

Version: 0.3.0

## Newly broken

*   checking whether package ‘merTools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘merTools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/merTools/new/merTools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# messina

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max, which.min
    
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'messina.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘Biobase’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘quantile’
    messinaSurvKMplot : <anonymous>: no visible binding for global variable
      ‘sd’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘median’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘quantile’
    messinaSurvKMplotSingleGroup: no visible binding for global variable
      ‘sd’
    messinaSurvObjPlot: no visible global function definition for
      ‘quantile’
    messinaSurvObjectiveFunc: no visible global function definition for
      ‘coef’
    messinaSurvPlot: no visible global function definition for ‘str’
    Undefined global functions or variables:
      approx coef median quantile reorder sd str
    Consider adding
      importFrom("stats", "approx", "coef", "median", "quantile", "reorder",
                 "sd")
      importFrom("utils", "str")
    to your NAMESPACE file.
    ```

# metabomxtr

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:base’:
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Metabomxtr_Vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# metacart

Version: 1.0-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# metacoder

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘ggrepel’ ‘reshape’
      All declared Imports should be used.
    ```

# MetaComp

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MetaCyto

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    collectData: no visible binding for global variable ‘parameter_name’
    collectData: no visible binding for global variable ‘value’
    panelSummary: no visible binding for global variable ‘antibodies’
    panelSummary: no visible binding for global variable ‘value’
    plotGA: no visible binding for global variable ‘lower’
    plotGA: no visible binding for global variable ‘upper’
    searchCluster : <anonymous>: no visible binding for global variable
      ‘triS’
    Undefined global functions or variables:
      antibodies lower parameter_name triS upper value
    ```

# metaforest

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# metagen

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    pivotalStream: no visible global function definition for ‘rchisq’
    pivotalStream : <anonymous>: no visible global function definition for
      ‘rchisq’
    pivotalStream: no visible global function definition for ‘rnorm’
    rBinomGauss: no visible global function definition for ‘rnorm’
    rBinomGauss: no visible global function definition for ‘rbinom’
    rD: no visible global function definition for ‘rchisq’
    rY: no visible global function definition for ‘rnorm’
    render: no visible global function definition for ‘pdf’
    render: no visible global function definition for ‘dev.off’
    renderSVG: no visible global function definition for ‘svg’
    renderSVG: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      data dev.off pdf qchisq qnorm qt quantile rbinom rchisq rnorm svg
      uniroot var
    Consider adding
      importFrom("grDevices", "dev.off", "pdf", "svg")
      importFrom("stats", "qchisq", "qnorm", "qt", "quantile", "rbinom",
                 "rchisq", "rnorm", "uniroot", "var")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# metagene

Version: 2.12.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 10.1Mb
      sub-directories of 1Mb or more:
        doc       3.2Mb
        extdata   5.4Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘similaRpeak’
    A package should be listed in only one of these fields.
    ```

*   checking R code for possible problems ... NOTE
    ```
    avoid_gaps_update: no visible binding for global variable 'value'
    avoid_gaps_update: no visible binding for global variable 'bam'
    avoid_gaps_update: no visible binding for global variable 'tab'
    avoid_gaps_update: no visible binding for global variable 'nuc'
    permutation_test: no visible global function definition for '.'
    permutation_test: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'bin'
    plot_metagene: no visible binding for global variable 'value'
    plot_metagene: no visible binding for global variable 'qinf'
    plot_metagene: no visible binding for global variable 'qsup'
    plot_metagene: no visible binding for global variable 'group'
    plot_metagene: no visible binding for global variable 'nuc'
    plot_metagene: no visible binding for global variable 'design'
    plot_metagene: no visible binding for global variable 'nuctot'
    Undefined global functions or variables:
      . bam bin design group nuc nuctot qinf qsup tab value
    ```

# metaheur

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MetaIntegrator

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MetaLonDA

Version: 1.0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MetamapsDB

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘shiny’
      All declared Imports should be used.
    ```

# metamisc

Version: 0.1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# metaMix

Version: 0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rmpi’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# metaplotr

Version: 0.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# metaviz

Version: 0.2.0

## Newly broken

*   checking whether package ‘metaviz’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘metaviz’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/metaviz/new/metaviz.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# meteogRam

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    crosssection: no visible global function definition for
      ‘colorRampPalette’
    crosssection: no visible global function definition for
      ‘filled.contour’
    crosssection: no visible global function definition for ‘axis’
    crosssection: no visible global function definition for ‘par’
    crosssection: no visible global function definition for ‘contour’
    temperatures: no visible binding for global variable ‘time’
    Undefined global functions or variables:
      axis colorRampPalette contour filled.contour par time
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "axis", "contour", "filled.contour", "par")
      importFrom("stats", "time")
    to your NAMESPACE file.
    ```

# methimpute

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      category covered
    Upstream and downstream counts for context CHG
      category missing
      category covered
    Upstream and downstream counts for context CHH
      category missing
      category covered
    Inside counts for context CG
      category missing
      category covered
    Inside counts for context CHG
      category missing
      category covered
    Inside counts for context CHH
      category missing
      category covered
    Writing to file /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmpno1njW/file15297df70510 ... 0.17s
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methimpute.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Methplot

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    getdata: no visible global function definition for ‘read.csv’
    plotdata: no visible global function definition for ‘head’
    Undefined global functions or variables:
      head read.csv
    Consider adding
      importFrom("utils", "head", "read.csv")
    to your NAMESPACE file.
    ```

# MethylAid

Version: 1.14.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘MethylAidData’
    ```

# methylInheritance

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MethylMix

Version: 2.10.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    FOXD1 :  2  components are best.
    ME1 :  2  components are best.
    MGMT :  2  components are best.
    OAS1 :  2  components are best.
    SOX10 :  2  components are best.
    TRAF6 :  2  components are best.
    ZNF217 :  2  components are best.
    > 
    > # Plot the most famous methylated gene for glioblastoma
    > MethylMix_PlotModel("MGMT", MethylMixResults, METcancer)
    $MixtureModelPlot
    
    $CorrelationPlot
    NULL
    
    > 
    > # Plot MGMT also with its normal methylation variation
    > MethylMix_PlotModel("MGMT", MethylMixResults, METcancer, METnormal = METnormal)
    $MixtureModelPlot
    Error: Aesthetics must be either length 1 or the same as the data (251): x, y, xend, yend
    Execution halted
    ```

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'ComBat_NoFiles':
    ComBat_NoFiles
      Code: function(dat, saminfo, type = "txt", write = F, covariates =
                     "all", par.prior = F, filter = F, skip = 0,
                     prior.plots = T)
      Docs: function(dat, saminfo, type = "txt", write = F, covariates =
                     "all", par.prior = T, filter = F, skip = 0,
                     prior.plots = T)
      Mismatches in argument default values:
        Name: 'par.prior' Code: F Docs: T
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘digest’
      All declared Imports should be used.
    ```

# methylumi

Version: 2.26.0

## In both

*   checking for missing documentation entries ... WARNING
    ```
    Undocumented code objects:
      ‘IDATsToMatrices’ ‘IDATtoMatrix’ ‘tcgaPipeline’
    Undocumented S4 methods:
      generic '[' and siglist 'MethyLumiM,ANY,ANY,ANY'
      generic '[' and siglist 'MethyLumiSet,ANY,ANY,ANY'
    All user-level objects in a package (including S4 classes and methods)
    should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking contents of ‘data’ directory ... WARNING
    ```
    Files not of a type allowed in a ‘data’ directory:
      ‘5318317007_A_Grn.idat’ ‘5318317007_A_Red.idat’
      ‘5318317007_B_Grn.idat’ ‘5318317007_B_Red.idat’
      ‘5318317007_C_Grn.idat’ ‘5318317007_C_Red.idat’
    Please use e.g. ‘inst/extdata’ for non-R data files
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methylumi.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘Biobase’ ‘scales’ ‘reshape2’ ‘ggplot2’ ‘matrixStats’
      ‘FDb.InfiniumMethylation.hg19’ ‘minfi’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.1Mb
      sub-directories of 1Mb or more:
        R         2.0Mb
        data      6.8Mb
        extdata   1.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘Biobase’ ‘minfi’ ‘lattice’ ‘matrixStats’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘minfi’
      ‘reshape2’ ‘scales’
      Please remove these calls from your code.
    'library' or 'require' calls in package code:
      ‘Biostrings’ ‘MASS’ ‘lumi’ ‘parallel’ ‘rtracklayer’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘FDb.InfiniumMethylation.hg19’ ‘ggplot2’ ‘matrixStats’ ‘methods’
      ‘reshape2’ ‘scales’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' call which should be '::': ‘lumi:::produceMethylationGEOSubmissionFile’
      See the note in ?`:::` about the use of this operator.
    Unexported objects imported by ':::' calls:
      ‘Biobase:::unsafeSetSlot’ ‘genefilter:::.findCentralMap’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      IlluminaHumanMethylation450kCOLORCHANNEL MethylSet RGChannelSet
      abline addColorChannelInfo aes allele as.dist axis box capture.output
      channel.probes colMedians colQuantiles colSds colorRampPalette
      coord_flip data dnorm dropouts drops ecdf facet_grid features
      gamma.integral gamma.mle gamma.mode geom_histogram ggplot hclust
      hm27.controls hm27.ordering hm450.controls hm450.ordering huber index
      intensity lines log_trans mclapply melt mu offset opts p.adjust
      packageDescription packageVersion par plot.density pnorm points
      position_identity read.csv read.delim read.table rect rowMins
      scale_colour_manual scale_fill_manual scale_shape_manual
      scale_x_continuous scale_y_continuous scale_y_discrete
      subsetCommonProbes text theme_bw title value variable weighted.mean
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "abline", "axis", "box", "lines", "par",
                 "points", "rect", "text", "title")
      importFrom("stats", "as.dist", "dnorm", "ecdf", "hclust", "offset",
                 "p.adjust", "pnorm", "weighted.mean")
      importFrom("utils", "capture.output", "data", "packageDescription",
                 "packageVersion", "read.csv", "read.delim", "read.table")
    to your NAMESPACE file.
    ```

*   checking Rd files ... NOTE
    ```
    prepare_Rd: estimateM.Rd:34-36: Dropping empty section \seealso
    prepare_Rd: estimateM.Rd:37-39: Dropping empty section \examples
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘TxDb.Hsapiens.UCSC.hg19.knownGene’
    ```

# methyvim

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# metricsgraphics

Version: 0.9.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mfa

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mfGARCH

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mgc

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MGLM

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mglR

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mhtboot

Version: 1.3.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vignette1.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# microbenchmark

Version: 1.4-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# microbiome

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# micromap

Version: 1.9.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in Polygon(crds_s) : less than 4 coordinates in polygon
    Warning in Polygon(crds_s) : less than 4 coordinates in polygon
    Warning in Polygon(crds_s) : less than 4 coordinates in polygon
    Warning in Polygon(crds_s) : less than 4 coordinates in polygon
    Warning in Polygon(crds_s) : less than 4 coordinates in polygon
    rgeos version: 0.3-26, (SVN revision 560)
     GEOS runtime version: 3.6.1-CAPI-1.10.1 r0 
     Linking to sp version: 1.2-7 
     Polygon checking: TRUE 
    
    Saving 7 x 7 in image
    Saving 7 x 7 in image
    Coordinate system already present. Adding new coordinate system, which will replace the existing one.
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Warning: Ignoring unknown aesthetics: x
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Introduction_Guide.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# microplot

Version: 1.0-38

## In both

*   checking whether package ‘microplot’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/microplot/new/microplot.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘microplot’ ...
** package ‘microplot’ successfully unpacked and MD5 sums checked
** R
** data
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘microplot’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/microplot/new/microplot.Rcheck/microplot’

```
### CRAN

```
* installing *source* package ‘microplot’ ...
** package ‘microplot’ successfully unpacked and MD5 sums checked
** R
** data
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/microplot/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘microplot’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/microplot/old/microplot.Rcheck/microplot’

```
# microsamplingDesign

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MIGSA

Version: 1.4.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
      
          Filter, Find, Map, Position, Reduce, anyDuplicated, append,
          as.data.frame, basename, cbind, colMeans, colSums, colnames,
          dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
          intersect, is.unsorted, lapply, lengths, mapply, match, mget,
          order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
          rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
          union, unique, unsplit, which, which.max, which.min
      
      
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        toTable
    
    
    
    
    Attaching package: ‘MIGSA’
    
    The following object is masked from ‘package:mGSZ’:
    
        geneSetsList
    
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MIGSA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# milr

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MIMOSA

Version: 1.18.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: data.table
    
    Attaching package: 'data.table'
    
    The following object is masked from 'package:reshape':
    
        melt
    
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Warning in .local(formula, data, ...) :
      Formula does not contain the RefTreat variable. It will be added automatically. Set RT=FALSE to disable this.
    Failed to fit subset 2 with EM. Trying mcmc
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MIMOSA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Undefined global functions or variables:
      AnnotatedDataFrame ExpressionSet PTID Proportion Proportion_REF
      RefTreat aes as.formula callNextMethod contour coord_trans dbeta dexp
      ecdf facet_grid facet_wrap fisher.test geom_boxplot geom_jitter
      geom_line ggtitle gray.colors huber image is.formula ldply legend
      lines mclapply model.frame na.omit optim p.adjust pData<- par pbeta
      plot points position_jitter qbeta quantile rbeta rbinom read.table
      rmultinom rnorm runif scale_color_brewer scale_fill_brewer show terms
      title var
    Consider adding
      importFrom("grDevices", "gray.colors")
      importFrom("graphics", "contour", "image", "legend", "lines", "par",
                 "plot", "points", "title")
      importFrom("methods", "callNextMethod", "show")
      importFrom("stats", "as.formula", "dbeta", "dexp", "ecdf",
                 "fisher.test", "model.frame", "na.omit", "optim",
                 "p.adjust", "pbeta", "qbeta", "quantile", "rbeta", "rbinom",
                 "rmultinom", "rnorm", "runif", "terms", "var")
      importFrom("utils", "read.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# MineICA

Version: 1.20.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘Biobase’ ‘plyr’ ‘ggplot2’ ‘scales’ ‘foreach’ ‘xtable’
      ‘biomaRt’ ‘gtools’ ‘GOstats’ ‘cluster’ ‘marray’ ‘mclust’
      ‘RColorBrewer’ ‘colorspace’ ‘igraph’ ‘Rgraphviz’ ‘graph’ ‘annotate’
      ‘Hmisc’ ‘fastICA’ ‘JADE’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘biomaRt’ ‘GOstats’ ‘cluster’ ‘mclust’ ‘igraph’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GOstats’ which was already attached by Depends.
      Please remove these calls from your code.
    Namespace in Imports field not imported from: ‘lumiHumanAll.db’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘GOstats’ ‘Hmisc’ ‘JADE’ ‘RColorBrewer’ ‘Rgraphviz’ ‘annotate’
      ‘biomaRt’ ‘cluster’ ‘colorspace’ ‘fastICA’ ‘foreach’ ‘ggplot2’
      ‘graph’ ‘gtools’ ‘igraph’ ‘marray’ ‘mclust’ ‘methods’ ‘plyr’ ‘scales’
      ‘xtable’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ':::' calls which should be '::':
      ‘Biobase:::annotation’ ‘Biobase:::validMsg’ ‘fpc:::pamk’
      ‘lumi:::getChipInfo’ ‘mclust:::adjustedRandIndex’
      See the note in ?`:::` about the use of this operator.
    Unexported object imported by a ':::' call: ‘Biobase:::isValidVersion’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      reorder scale_colour_gradientn scale_colour_manual scale_fill_manual
      scale_linetype_manual scale_shape_manual scale_x_continuous
      scale_x_discrete scale_y_continuous shapiro.test sigCategories
      terrain_hcl theme theme_bw title tkplot.fit.to.screen unit useMart
      validObject vcount viewport wilcox.test write.table xlab xtable
    Consider adding
      importFrom("grDevices", "cm.colors", "dev.off", "graphics.off",
                 "heat.colors", "pdf")
      importFrom("graphics", "abline", "axis", "frame", "hist", "layout",
                 "legend", "mtext", "par", "plot", "plot.new", "points",
                 "title")
      importFrom("methods", "callNextMethod", "new", "validObject")
      importFrom("stats", "aggregate", "as.dendrogram", "as.dist",
                 "as.hclust", "chisq.test", "cor", "cor.test", "cutree",
                 "dist", "hclust", "kmeans", "kruskal.test", "lm", "median",
                 "na.omit", "order.dendrogram", "p.adjust", "quantile",
                 "reorder", "shapiro.test", "wilcox.test")
      importFrom("utils", "capture.output", "combn", "read.table",
                 "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Warning: `axis.ticks.margin` is deprecated. Please set `margin` property  of `axis.text` instead
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MineICA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# MIRA

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MiRAnorm

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘cluster’ ‘dendextend’ ‘npmv’ ‘utils’
      All declared Imports should be used.
    ```

# mirIntegrator

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    integrate_mir : isGraphNEL: no visible global function definition for
      ‘is’
    pathways2pdf: no visible global function definition for ‘pdf’
    pathways2pdf: no visible global function definition for ‘par’
    pathways2pdf: no visible global function definition for ‘graphics.off’
    plotLines: no visible binding for global variable ‘x’
    plotLines: no visible binding for global variable ‘y’
    plotLines: no visible binding for global variable ‘pathways_set’
    plotPathway2Colors: no visible global function definition for ‘legend’
    plot_change: no visible global function definition for ‘complete.cases’
    Undefined global functions or variables:
      complete.cases graphics.off is legend par pathways_set pdf x y
    Consider adding
      importFrom("grDevices", "graphics.off", "pdf")
      importFrom("graphics", "legend", "par")
      importFrom("methods", "is")
      importFrom("stats", "complete.cases")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Loading required package: ROntoTools
    Loading required package: boot
    Loading required package: KEGGREST
    Loading required package: KEGGgraph
    
    Attaching package: ‘KEGGgraph’
    
    The following object is masked from ‘package:graphics’:
    
        plot
    
    Loading required package: Rgraphviz
    Loading required package: grid
    There are  881 miRNAs meassured and 15 of them were included in the analysis.
    There are  19611 mRNAs meassured and 0 of them were included in the analysis.
    The over-representaion p-value is not defined for cut-off free analysis and will not be computed!
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mirIntegrator.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Mirsynergy

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:stats’:
    
        decompose, spectrum
    
    The following object is masked from ‘package:base’:
    
        union
    
    Loading required package: ggplot2
    Discard row/column of all zeros in W
    Loading required package: Matrix
    Loading required package: foreach
    Loaded glmnet 2.0-16
    
    Discard row/column of all zeros in W
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Mirsynergy.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# miscset

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MissingDataGUI

Version: 0.2-5

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘cairoDevice’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# missRows

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: MultiAssayExperiment
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'missRows.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mistat

Version: 1.0-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# mistral

Version: 2.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MIXFIM

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mixKernel

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mixOmics

Version: 6.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   4.0Mb
    ```

# mixpack

Version: 0.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mixR

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MixSIAR

Version: 3.1.10

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘MixSIAR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: mixsiar_gui
    > ### Title: Run the GUI version of MixSIAR
    > ### Aliases: mixsiar_gui
    > 
    > ### ** Examples
    > 
    > mixsiar_gui()
    Error in mixsiar_gui() : 
      *** gWidgetsRGtk2 package not able to be loaded. ***
            If 'library('gWidgetsRGtk2')' does not work, MixSIAR GUI will not run.
            On Windows/Linux, try 'install.packages('gWidgetsRGtk2')'.
            On Mac, close R, download and install GTK+ from:
            http://r.research.att.com/#other. Then install latest X11 application
            (xQuartz) from http://xquartz.macosforge.org/landing/.
    
            If installing GTK+ continues to be problematic, consider using the
            script version of MixSIAR. See manual for help and examples.
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 25 SKIPPED: 0 FAILED: 9
      1. Error: Wolves ex works (@test_ex1_wolves.R#33) 
      2. Error: Geese ex works (@test_ex2_geese.R#27) 
      3. Error: Lake ex works (@test_ex3_lake.R#27) 
      4. Error: Palmyra ex works (@test_ex4_palmyra.R#27) 
      5. Error: Killer whale ex works (@test_ex5_killerwhale.R#28) 
      6. Error: Snail ex works (@test_ex6_snail.R#27) 
      7. Error: Storm-petrel ex works (@test_ex7_stormpetrel.R#27) 
      8. Error: Cladocera ex works (@test_ex8_cladocera.R#31) 
      9. Error: Isopod ex works (@test_ex9_isopod.R#27) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘gWidgetsRGtk2’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rjags’
      All declared Imports should be used.
    ```

# mizer

Version: 1.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(mizer)
      > 
      > test_check("mizer")
      ── 1. Failure: retune_abundance reproduces scaling model (@test-wrapper_function
      max(abs(initial_n - pr@initial_n)) is not strictly less than 2e-11. Difference: 2.47e-11
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 508 SKIPPED: 1 FAILED: 1
      1. Failure: retune_abundance reproduces scaling model (@test-wrapper_functions.R#17) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Note: 	No ks column in species data frame so using ks = h * 0.2.
    	Note: No sel_func column in species data frame. Setting selectivity to be 'knife_edge' for all species.
    Note: 	No knife_edge_size column in species data frame. Setting knife edge selectivity equal to w_mat.
    Note: 	No h column in species data frame so using f0 and k_vb to calculate it.
    Note: 	No gamma column in species data frame so using f0, h, beta, sigma, lambda and kappa to calculate it.
    Note: 	No z0 column in species data frame so using z0 = z0pre * w_inf ^ z0exp.
    Note: 	No ks column in species data frame so using ks = h * 0.2.
    Note: 	No h column in species data frame so using f0 and k_vb to calculate it.
    Note: 	No gamma column in species data frame so using f0, h, beta, sigma, lambda and kappa to calculate it.
    Note: 	No z0 column in species data frame so using z0 = z0pre * w_inf ^ z0exp.
    Note: 	No ks column in species data frame so using ks = h * 0.2.
    Note: 	No gamma column in species data frame so using f0, h, beta, sigma, lambda and kappa to calculate it.
    Note: 	No gamma column in species data frame so using f0, h, beta, sigma, lambda and kappa to calculate it.
    Note: 	No gamma column in species data frame so using f0, h, beta, sigma, lambda and kappa to calculate it.
    Note: Negative abundance values in background resource overwritten with zeros
    Note: 	No ks column in new species data frame. Setting ks = 0.2*h.
    Note: 	No ks column in new species data frame. Setting ks = 0.2*h.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'mizer_vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘deSolve’ ‘progress’
      All declared Imports should be used.
    ```

# MKmisc

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘genefilter’, ‘limma’
    ```

# mlbgameday

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘doParallel’ ‘iterators’ ‘parallel’
      All declared Imports should be used.
    ```

# mleap

Version: 0.1.1

## In both

*   checking whether package ‘mleap’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mleap/new/mleap.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘mleap’ ...
** package ‘mleap’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
Error: package or namespace load failed for ‘mleap’:
 .onLoad failed in loadNamespace() for 'mleap', details:
  call: NULL
  error: .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so
  Reason: image not found
Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mleap/new/mleap.Rcheck/mleap’

```
### CRAN

```
* installing *source* package ‘mleap’ ...
** package ‘mleap’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
** building package indices
** testing if installed package can be loaded
Error: package or namespace load failed for ‘mleap’:
 .onLoad failed in loadNamespace() for 'mleap', details:
  call: NULL
  error: .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/mleap/rJava/libs/rJava.so
  Reason: image not found
Error: loading failed
Execution halted
ERROR: loading failed
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mleap/old/mleap.Rcheck/mleap’

```
# mlmc

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mlogit

Version: 0.3-0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: Formula
    Loading required package: maxLik
    Loading required package: miscTools
    
    Please cite the 'maxLik' package as:
    Henningsen, Arne and Toomet, Ott (2011). maxLik: A package for maximum likelihood estimation in R. Computational Statistics 26(3), 443-458. DOI 10.1007/s00180-010-0217-1.
    
    If you have questions, suggestions, or comments regarding the 'maxLik' package, please use a forum or 'tracker' at maxLik's R-Forge site:
    https://r-forge.r-project.org/projects/maxlik/
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Exercises.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mlr

Version: 2.12.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/run-base.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3332 SKIPPED: 0 FAILED: 10
      1.  Error: clustering performance (@test_base_clustering.R#15) 
      2.  Error: clustering performance with missing clusters (@test_base_clustering.R#27) 
      3.  Error: clustering resample (@test_base_clustering.R#41) 
      4.  Error: clustering benchmark (@test_base_clustering.R#52) 
      5.  Error: clustering tune (@test_base_clustering.R#65) 
      6.  Error: plotFilterValues (@test_base_generateFilterValuesData.R#68) 
      7.  Error: getFeatureImportance (@test_base_getFeatureImportance.R#33) 
      8.  Error: getOOBPreds (@test_base_getOOBPreds.R#14) 
      9.  Error: hyperpars (@test_base_hyperpars.R#12) 
      10. Error: check measure calculations (@test_base_measures.R#206) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.3Mb
    ```

# mlrMBO

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mlxR

Version: 3.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MMDiff2

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      pdfcrop not installed or not in PATH
    No Samples specified, plotting all samples
    No Contrast specified, plotting all samples in one plot
    No normalization factors applied
    checking availability of package BSgenome.Mmusculus.UCSC.mm9
    No Samples specified, plotting all samples
    No Contrast specified, plotting all samples in one plot
    No normalization factors applied
    checking availability of package BSgenome.Mmusculus.UCSC.mm9
    checking parameters...
    estimating sigma...
    pre-computing Kernel matrix...
    computing 3 pair-wise distances...
    computing distances for WT.AB2 vs Null.AB2
    computing distances for WT.AB2 vs Resc.AB2
    computing distances for Null.AB2 vs Resc.AB2
    preparing Output...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'MMDiff2.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    DBAmmd: no visible global function definition for ‘new’
    getReads: no visible global function definition for ‘IRangesList’
    Undefined global functions or variables:
      IRangesList new
    Consider adding
      importFrom("methods", "new")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# mmpf

Version: 0.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mnreadR

Version: 1.2.0

## Newly broken

*   checking whether package ‘mnreadR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘mnreadR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mnreadR/new/mnreadR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Mobilize

Version: 2.16-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘theme_text’
    mapmobility: no visible global function definition for ‘lines’
    responseplot: no visible global function definition for ‘na.omit’
    scatterplot: no visible global function definition for ‘na.omit’
    scatterplot: no visible global function definition for ‘opts’
    scatterplot: no visible global function definition for ‘theme_text’
    sharedtimeplot: no visible global function definition for ‘na.omit’
    timeplot: no visible global function definition for ‘na.omit’
    timeplot: no visible global function definition for ‘opts’
    timeplot.character: no visible global function definition for ‘runif’
    timeplot.character: no visible global function definition for ‘rnorm’
    timeplot.character: no visible binding for global variable ‘text’
    userplot: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      head lines na.omit opts quantile rnorm runif text theme_blank
      theme_text
    Consider adding
      importFrom("graphics", "lines", "text")
      importFrom("stats", "na.omit", "quantile", "rnorm", "runif")
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# modcmfitr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# model4you

Version: 0.9-1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: lm_plot
    > 
    > ### ** Examples
    > 
    > ## example taken from ?lm
    > ctl <- c(4.17,5.58,5.18,6.11,4.50,4.61,5.17,4.53,5.33,5.14)
    > trt <- c(4.81,4.17,4.41,3.59,5.87,3.83,6.03,4.89,4.32,4.69)
    > group <- gl(2, 10, 20, labels = c("Ctl","Trt"))
    > weight <- c(ctl, trt)
    > data <- data.frame(weight, group)
    > lm.D9 <- lm(weight ~ group, data = data)
    > lm_plot(lm.D9)
    > 
    > ## example taken from ?glm (modified version)
    > data(anorexia, package = "MASS")
    > anorexia$treatment <- anorexia$Treat != "Cont"
    > anorex.1 <- glm(Postwt ~ treatment + offset(Prewt),
    +                 family = gaussian, data = anorexia)
    > lm_plot(anorex.1)
    Error: More than one expression parsed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# modeldb

Version: 0.1.0

## Newly broken

*   checking whether package ‘modeldb’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘modeldb’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/modeldb/new/modeldb.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# modelr

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# moderndive

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MOEADr

Version: 1.1.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 243-278 (Comparison_Usage.Rmd) 
    Error: processing vignette 'Comparison_Usage.Rmd' failed with diagnostics:
    Discrete value supplied to continuous scale
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# momentuHMM

Version: 1.4.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in parse(text = code, keep.source = FALSE) : 
      <text>:2:35: unexpected ')'
    1: DM$angle = list(mean = ~state2(angleStrength(d, strength = w)), 
    2:                 concentration= ~1))
                                         ^
    Warning in highr::hilight(x, format, prompt = options$prompt, markup = opts$markup) :
      the syntax of the source code is invalid; the fallback mode is used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'momentuHMM.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.0Mb
      sub-directories of 1Mb or more:
        data   2.1Mb
        doc    1.7Mb
    ```

# Momocs

Version: 1.2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# monocle

Version: 2.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'monocle-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘biocViews’
      All declared Imports should be used.
    Missing or unexported object: ‘scater::newSCESet’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    buildBranchCellDataSet: no visible global function definition for ‘nei’
    count_leaf_descendents: no visible global function definition for ‘nei’
    cth_classifier_cds: no visible global function definition for ‘nei’
    cth_classifier_cell: no visible global function definition for ‘nei’
    diff_test_helper: no visible binding for global variable ‘Size_Factor’
    exportCDS: no visible binding for global variable ‘use_for_ordering’
    extract_good_ordering: no visible global function definition for ‘nei’
    fit_model_helper: no visible binding for global variable ‘Size_Factor’
    get_next_node_id: no visible binding for '<<-' assignment to
      ‘next_node’
    get_next_node_id: no visible binding for global variable ‘next_node’
    make_canonical: no visible global function definition for ‘nei’
    measure_diameter_path: no visible global function definition for ‘nei’
    orderCells: no visible binding for '<<-' assignment to ‘next_node’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘pseudocount’
    plot_multiple_branches_pseudotime: no visible binding for global
      variable ‘Branch’
    project2MST: no visible global function definition for ‘nei’
    Undefined global functions or variables:
      Branch Size_Factor nei next_node pseudocount use_for_ordering
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# MonteCarlo

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# moonBook

Version: 0.1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# morse

Version: 3.0.0

## In both

*   checking whether package ‘morse’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/new/morse.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘morse’ ...
** package ‘morse’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘morse’
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘morse’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/new/morse.Rcheck/morse’

```
### CRAN

```
* installing *source* package ‘morse’ ...
** package ‘morse’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/morse/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘morse’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/morse/old/morse.Rcheck/morse’

```
# mortAAR

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MortalityTables

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘pracma’
    ```

# mosaic

Version: 1.1.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > rdata <- rmultinom(999, sum(x), prob = rep(.25, 4))
    > statTally(x, rdata, fun = max, binwidth = 1)  # unusual test statistic
    Null distribution appears to be asymmetric. (p = 0.00199)
    
    Test statistic applied to sample data =  18
    
    Quantiles of test statistic applied to random data:
    50% 90% 95% 99% 
     17  19  20  22 
    
    
    
    Of the 1000 samples (1 original + 999 random),
    
    	152 ( 15.2 % ) had test stats = 18
    
    	330 ( 33 % ) had test stats >= 18
    Error in is.finite(x) : 
      default method not implemented for type 'language'
    Calls: <Anonymous> ... lapply -> FUN -> <Anonymous> -> f -> lapply -> FUN -> f
    Execution halted
    ```

*   checking whether package ‘mosaic’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘mosaic’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mosaic/new/mosaic.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Don't know how to automatically pick scale for object of type formula. Defaulting to continuous.
    Quitting from lines 87-88 (ggformula-lattice.Rmd) 
    Error: processing vignette 'ggformula-lattice.Rmd' failed with diagnostics:
    default method not implemented for type 'language'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘manipulate’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R     2.6Mb
        doc   2.4Mb
    ```

# mosaicData

Version: 0.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 7 marked UTF-8 strings
    ```

# mosaicModel

Version: 0.3.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 66-68 (Basics.Rmd) 
    Error: processing vignette 'Basics.Rmd' failed with diagnostics:
    Aesthetics must be either length 1 or the same as the data (32): colour, x, y
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘caret’ ‘ggformula’ ‘knitr’ ‘testthat’ ‘tidyverse’
      All declared Imports should be used.
    ```

# mousetrap

Version: 3.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘mousetrap-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: bezier
    > ### Title: Create Bezier-curves using the Bernstein approximation.
    > ### Aliases: bezier
    > 
    > ### ** Examples
    > 
    > # Generate range of Bezier-curves
    > bezier_curves <- bezier(w=seq(0,10,.1))
    > 
    > # Plot curves
    > mt_plot(bezier_curves)
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to linetype
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# moveHMM

Version: 1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in CI(m) :
      Some of the parameter estimates seem to lie close to the boundaries of their parameter space.
      The associated CIs are probably unreliable (or might not be computable).
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Note: Some angles are equal to pi, and the corresponding pseudo-residuals are not included
    Warning: 'mode(width)' and 'mode(height)' differ between new and previous
    	 ==> NOT changing 'width' & 'height'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'moveHMM-guide.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# moveVis

Version: 0.9.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mplot

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MplusAutomation

Version: 0.7-2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mpoly

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mrfDepth

Version: 1.0.6

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘mrfDepth-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: fom
    > ### Title: Draws the Functional Outlier Map (FOM)
    > ### Aliases: fom
    > ### Keywords: Graphical
    > 
    > ### ** Examples
    > 
    > data(octane)
    > 
    > # To construct the FOM, one first need to calculate 
    > # the functional outlyingness.
    > # Note that the option diagnostic in fOutl must be 
    > # set to TRUE. If not calling fom will result in an 
    > # error
    > Result <- fOutl(octane, alpha = 0, type = "fAO", diagnostic = TRUE);
    > fom(Result)
    Error: Columns `x`, `y` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mrMLM

Version: 3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mrMLM.GUI

Version: 3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MRMR

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MSCMT

Version: 1.3.3

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rglpk’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# MSG

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# msgbsR

Version: 1.4.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > # Obtain the length of chromosome 20 in rn6
    > library(BSgenome.Rnorvegicus.UCSC.rn6)
    Loading required package: BSgenome
    Loading required package: Biostrings
    Loading required package: XVector
    
    Attaching package: 'Biostrings'
    
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: rtracklayer
    > chr20 <- seqlengths(BSgenome.Rnorvegicus.UCSC.rn6)["chr20"]
    > plotCircos(cutSites = cuts, seqlengths = chr20,
    +            cutSite.colour = "red", seqlengths.colour = "blue")
    Warning in if (!lbs %in% c(colnames(mold(obj[1, ])), "start", "end", "seqnames",  :
      the condition has length > 1 and only the first element will be used
    Error in .local(data, ...) : label must be one of column names
    Calls: plotCircos ... do.call -> do.call -> <Anonymous> -> <Anonymous> -> .local
    Execution halted
    ```

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      
      Note: significantly better compression could be obtained
            by using R CMD build --resave-data
                   old_size new_size compress
      ratdata.rda     318Kb    127Kb       xz
      ratdata2.rda    287Kb    116Kb       xz
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.1Mb
      sub-directories of 1Mb or more:
        extdata  15.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'utils'
      All declared Imports should be used.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: Biostrings
    Loading required package: XVector
    
    Attaching package: 'Biostrings'
    
    The following object is masked from 'package:DelayedArray':
    
        type
    
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: rtracklayer
    Warning in if (!lbs %in% c(colnames(mold(obj[1, ])), "start", "end", "seqnames",  :
      the condition has length > 1 and only the first element will be used
    
    Error: processing vignette 'msgbsR_Vignette.Rnw' failed with diagnostics:
     chunk 11 (label = circos plot) 
    Error in .local(data, ...) : label must be one of column names
    Execution halted
    ```

# MSstatsQC

Version: 1.2.0

## Newly broken

*   checking whether package ‘MSstatsQC’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘MSstatsQC’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/MSstatsQC/new/MSstatsQC.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

# mtconnectR

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mthapower

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mudata2

Version: 1.0.2

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 930 SKIPPED: 0 FAILED: 12
      1. Error: mudata_prepare_column and mudata_parse_column are opposites (@test_mudata.io.R#244) 
      2. Error: mudata_prepare_tbl works as intended (@test_mudata.io.R#288) 
      3. Error: mudata_prepare_tbl and mudata_parse_tbl are opposites (@test_mudata.io.R#322) 
      4. Failure: as_* functions produce the expected output type (@test_types.R#210) 
      5. Failure: wkt parsing returns an sf::sfc (@test_types.R#252) 
      6. Failure: wkt parsing works when there are parsing errors/NA values (@test_types.R#259) 
      7. Error: wkt parsing works when there are parsing errors/NA values (@test_types.R#261) 
      8. Failure: wkt parsing works with zero-length input (@test_types.R#285) 
      9. Failure: objects generate the correct type strings (@test_types.R#306) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# mudfold

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# multdyn

Version: 1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MultiBD

Version: 0.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SIRtrans.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MultiDataSet

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘Biobase:::assayDataEnvLock’ ‘Biobase:::assayDataStorageMode’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    downloadGEO : <anonymous>: no visible global function definition for
      ‘methylationSet’
    lambdaClayton: no visible global function definition for ‘qnorm’
    lambdaClayton: no visible global function definition for ‘qchisq’
    qq_plot: no visible global function definition for ‘qbeta’
    Undefined global functions or variables:
      methylationSet qbeta qchisq qnorm
    Consider adding
      importFrom("stats", "qbeta", "qchisq", "qnorm")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' calls not declared from:
      ‘gplots’ ‘lattice’
    ```

# multiDimBio

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# multilevelPSA

Version: 1.2.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MultiMeta

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    multi_meta: no visible global function definition for ‘write.table’
    multi_meta: no visible global function definition for ‘read.table’
    multi_meta: no visible global function definition for ‘pchisq’
    qqplotter: no visible global function definition for ‘read.table’
    qqplotter: no visible global function definition for ‘qchisq’
    qqplotter: no visible global function definition for ‘ppoints’
    qqplotter: no visible global function definition for ‘median’
    qqplotter: no visible global function definition for ‘pchisq’
    qqplotter: no visible global function definition for ‘pdf’
    qqplotter: no visible global function definition for ‘plot’
    qqplotter: no visible global function definition for ‘abline’
    qqplotter: no visible global function definition for ‘dev.off’
    Undefined global functions or variables:
      abline axis colorRampPalette cov2cor dev.off median pchisq pdf plot
      points ppoints qchisq read.table write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf")
      importFrom("graphics", "abline", "axis", "plot", "points")
      importFrom("stats", "cov2cor", "median", "pchisq", "ppoints", "qchisq")
      importFrom("utils", "read.table", "write.table")
    to your NAMESPACE file.
    ```

# multipanelfigure

Version: 1.0.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rsvg’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# multiROC

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘boot’ ‘ggplot2’
      All declared Imports should be used.
    ```

# munsell

Version: 0.4.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘munsell-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: complement_slice
    > ### Title: A vertical slice through the Munsell space
    > ### Aliases: complement_slice
    > 
    > ### ** Examples
    > 
    > complement_slice("5PB")
    Warning: Removed 41 rows containing missing values (geom_text).
    Error: Free scales are only supported with `coord_cartesian()` and `coord_flip()`
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# musica

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MutationalPatterns

Version: 1.6.1

## In both

*   R CMD check timed out
    

# mutSignatures

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mvdalab

Version: 1.4

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘mvdalab-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot.plusminus
    > ### Title: 2D Graph of the PCA scores associated with a plusminusFit
    > ### Aliases: plot.plusminus
    > 
    > ### ** Examples
    > 
    > ###  PLUS-Minus CLASSIFIER WITH validation = 'none', i.e. no CV ###
    > data(plusMinusDat)
    > mod1 <- plusminusFit(Y ~., data = plusMinusDat, validation = "none", n_cores = 2)
    > plot(mod1, ncomp = 2, comps = c(1, 2))
    Error: Columns `colour`, `shape` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# mvtboost

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# MWASTools

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    JBA_binning: no visible binding for global variable ‘median’
    JBA_corDistribution: no visible binding for global variable ‘r.coeff’
    JBA_mergeClusters: no visible binding for global variable ‘median’
    JBA_plotBins: no visible binding for global variable ‘limits’
    JBA_plotBins: no visible binding for global variable ‘value’
    JBA_plotBins: no visible binding for global variable ‘intensity’
    JBA_plotBins: no visible binding for global variable ‘mean_cor’
    JBA_plotBins: no visible binding for global variable ‘clean_mean’
    MWAS_heatmap: no visible binding for '<<-' assignment to ‘hr’
    MWAS_heatmap: no visible binding for global variable ‘hr’
    MWAS_scatterplotMS: no visible binding for global variable ‘logpval’
    QC_CV_scatterplot: no visible binding for global variable ‘abs.CV’
    Undefined global functions or variables:
      abs.CV clean_mean hr intensity limits logpval mean_cor median r.coeff
      value
    Consider adding
      importFrom("stats", "median")
    to your NAMESPACE file.
    ```

# mwaved

Version: 1.1.5

## In both

*   checking whether package ‘mwaved’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/new/mwaved.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘mwaved’ ...
** package ‘mwaved’ successfully unpacked and MD5 sums checked
checking for Rscript... yes
configure: Checking if FFTW3 library is available using pkg-config
checking for pkg-config... no
checking for FFTW... no
FFTW library not found, please install fftw-3-3-3 or greater
configure: creating ./config.status
config.status: creating src/Makevars
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mwaved.cpp -o mwaved.o
mwaved.cpp:1:10: fatal error: 'fftw3.h' file not found
#include <fftw3.h>
         ^~~~~~~~~
1 error generated.
make: *** [mwaved.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘mwaved’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/new/mwaved.Rcheck/mwaved’

```
### CRAN

```
* installing *source* package ‘mwaved’ ...
** package ‘mwaved’ successfully unpacked and MD5 sums checked
checking for Rscript... yes
configure: Checking if FFTW3 library is available using pkg-config
checking for pkg-config... no
checking for FFTW... no
FFTW library not found, please install fftw-3-3-3 or greater
configure: creating ./config.status
config.status: creating src/Makevars
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/mwaved/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c mwaved.cpp -o mwaved.o
mwaved.cpp:1:10: fatal error: 'fftw3.h' file not found
#include <fftw3.h>
         ^~~~~~~~~
1 error generated.
make: *** [mwaved.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘mwaved’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/mwaved/old/mwaved.Rcheck/mwaved’

```
# myTAI

Version: 0.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘biomartr’
      All declared Imports should be used.
    ```

# naniar

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrrlyr’
      All declared Imports should be used.
    ```

# natserv

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NeatMap

Version: 0.3.6.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    data.reduction: no visible global function definition for ‘prcomp’
    data.reduction: no visible global function definition for ‘hclust’
    data.reduction: no visible global function definition for ‘as.dist’
    data.reduction: no visible global function definition for ‘cor’
    data.reduction: no visible global function definition for ‘dist’
    heatmap1: no visible global function definition for ‘sd’
    lineplot: no visible global function definition for ‘sd’
    normalize: no visible global function definition for ‘sd’
    profileplot3d : ColorFunction: no visible global function definition
      for ‘rgb’
    profileplot3d : ColorFunction: no visible global function definition
      for ‘colorRamp’
    profileplot3d: no visible global function definition for ‘sd’
    profileplot3d: no visible global function definition for ‘gray’
    Undefined global functions or variables:
      as.dist colorRamp cor dist gray hclust prcomp rgb sd
    Consider adding
      importFrom("grDevices", "colorRamp", "gray", "rgb")
      importFrom("stats", "as.dist", "cor", "dist", "hclust", "prcomp", "sd")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘NeatMap/libs/NeatMap.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘nMDS_R.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘nMDS_R.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# neatmaps

Version: 1.0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# netgen

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nethet

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'nethet.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    t2cov.lr: no visible global function definition for ‘var’
    t2cov.lr: no visible global function definition for ‘pchisq’
    t2diagcov.lr: no visible global function definition for ‘var’
    t2diagcov.lr: no visible global function definition for ‘pchisq’
    test.sd: no visible global function definition for ‘var’
    test.sd: no visible global function definition for ‘pnorm’
    twosample_single_regr: no visible global function definition for ‘coef’
    twosample_single_regr: no visible global function definition for ‘lm’
    Undefined global functions or variables:
      abline boxplot coef cor cov.wt dev.off dnorm grey hist kmeans legend
      lines lm median optimize p.adjust par pchisq pdf pnorm quantile rbeta
      rnorm segments shapiro.test var write.csv
    Consider adding
      importFrom("grDevices", "dev.off", "grey", "pdf")
      importFrom("graphics", "abline", "boxplot", "hist", "legend", "lines",
                 "par", "segments")
      importFrom("stats", "coef", "cor", "cov.wt", "dnorm", "kmeans", "lm",
                 "median", "optimize", "p.adjust", "pchisq", "pnorm",
                 "quantile", "rbeta", "rnorm", "shapiro.test", "var")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# netprioR

Version: 1.6.0

## Newly broken

*   checking whether package ‘netprioR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘netprioR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/netprioR/new/netprioR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# netrankr

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# netresponse

Version: 1.40.0

## In both

*   checking files in ‘vignettes’ ... WARNING
    ```
    Files in the 'vignettes' directory but no files in 'inst/doc':
      ‘NetResponse.Rmd’, ‘NetResponse.md’, ‘TODO/TODO.Rmd’,
        ‘fig/NetResponse2-1.png’, ‘fig/NetResponse2b-1.png’,
        ‘fig/NetResponse3-1.png’, ‘fig/NetResponse4-1.png’,
        ‘fig/NetResponse5-1.png’, ‘fig/NetResponse7-1.png’,
        ‘fig/vdp-1.png’, ‘main.R’, ‘netresponse.bib’, ‘netresponse.pdf’
    Package has no Sweave vignette sources and no VignetteBuilder field.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘netresponse/libs/netresponse.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘netresponse.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Object: ‘netresponse.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# netSmooth

Version: 1.0.0

## In both

*   checking whether package ‘netSmooth’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: namespace ‘bigmemory’ is not available and has been replaced
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/netSmooth/new/netSmooth.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   3.6Mb
        doc    2.3Mb
    ```

# NetworkChange

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘abind’
      All declared Imports should be used.
    ```

# NetworkExtinction

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NetworkInference

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# networkreporting

Version: 0.1.1

## Newly broken

*   checking whether package ‘networkreporting’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘networkreporting’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/networkreporting/new/networkreporting.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Loading required package: functional
    Loading required package: stringr
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:dplyr':
    
        vars
    
    Quitting from lines 221-222 (network_scaleup.Rmd) 
    Error: processing vignette 'network_scaleup.Rmd' failed with diagnostics:
    Column `x` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NetworkRiskMeasures

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# networktools

Version: 1.1.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘networktools-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot.expectedInf
    > ### Title: Plot "expectedInf" objects
    > ### Aliases: plot.expectedInf
    > 
    > ### ** Examples
    > 
    > ## Don't show: 
    > myNetwork <- cor(depression[,1:5])
    > ## End(Don't show)
    > out1 <- expectedInf(myNetwork)
    > plot(out1$step1)
    Error: Column `x` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘devtools’
      All declared Imports should be used.
    ```

# NeuralNetTools

Version: 1.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# neurobase

Version: 1.26.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# neuropsychology

Version: 0.5.0

## Newly broken

*   checking whether package ‘neuropsychology’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘neuropsychology’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/neuropsychology/new/neuropsychology.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘htmlTable’ ‘lme4’ ‘stringi’
      All declared Imports should be used.
    ```

# neutralitytestr

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NFP

Version: 0.99.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        parLapplyLB, parRapply, parSapply, parSapplyLB
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    
    The following objects are masked from 'package:base':
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Loading required package: graphite
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NFP.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘NFPdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.5Mb
      sub-directories of 1Mb or more:
        data   8.0Mb
    ```

# nhstplot

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nima

Version: 0.4.5

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 1. Failure: attrnames finds the names of the attributes of a data frame (@tes
      attrnames(y) not equal to c("names", "row.names", "class").
      2/3 mismatches
      x[2]: "class"
      y[2]: "row.names"
      
      x[3]: "row.names"
      y[3]: "class"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 21 SKIPPED: 0 FAILED: 1
      1. Failure: attrnames finds the names of the attributes of a data frame (@test-attribute_names.R#15) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘survival’
      All declared Imports should be used.
    ```

# nimble

Version: 0.6-10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 16.5Mb
      sub-directories of 1Mb or more:
        CppCode        1.1Mb
        R              6.1Mb
        classic-bugs   1.5Mb
        include        5.6Mb
        tests          1.3Mb
    ```

# NIRStat

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nlmixr

Version: 0.9.0-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dparser’ ‘inline’
      All declared Imports should be used.
    ```

# nls.multstart

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nlstimedist

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NlsyLinks

Version: 2.0.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Faq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.7Mb
      sub-directories of 1Mb or more:
        data      4.4Mb
        reports   2.2Mb
    ```

# nLTT

Version: 1.4

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 60-65 (get_nltt_values.Rmd) 
    Error: processing vignette 'get_nltt_values.Rmd' failed with diagnostics:
    could not find function "geom_step"
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘deSolve’
      All declared Imports should be used.
    ```

# nmathresh

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NMF

Version: 0.21.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in citation(x) : package 'doMPI' not found
    Converted 9 of 10 package citations to BibTeX
    Writing 11 Bibtex entries ... OK
    Results written to file 'Rpackages.bib'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NMF-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# NNTbiomarker

Version: 0.29.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# noaastormevents

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘hurricaneexposuredata’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘XML’ ‘choroplethr’ ‘choroplethrMaps’ ‘data.table’
      ‘forcats’ ‘hurricaneexposure’ ‘plyr’
      All declared Imports should be used.
    ```

# NOAAWeather

Version: 0.1.0

## Newly broken

*   checking whether package ‘NOAAWeather’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘NOAAWeather’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/NOAAWeather/new/NOAAWeather.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data   5.1Mb
    ```

# nonmem2R

Version: 0.1.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nord

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NormalizeMets

Version: 0.25

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        doc   8.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘DiffCorr’
    ```

# NORRRM

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    CIPW: no visible global function definition for ‘data’
    CIPW: no visible binding for global variable ‘AtomWeight’
    CIPW: no visible binding for global variable ‘MinWeight’
    CIPW: no visible binding for global variable ‘OxiWeight’
    CIPW.trace: no visible global function definition for ‘data’
    CIPW.trace: no visible binding for global variable ‘AtomWeight’
    CIPW.trace: no visible binding for global variable ‘MinWeight’
    CIPW.trace: no visible binding for global variable ‘OxiWeight’
    Undefined global functions or variables:
      AtomWeight MinWeight OxiWeight data
    Consider adding
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# nparACT

Version: 0.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# NPflow

Version: 0.13.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# npregfast

Version: 1.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nprobust

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nucleR

Version: 2.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# nullabor

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    reg_dist: no visible global function definition for ‘lm’
    resid_boot: no visible global function definition for ‘resid’
    resid_pboot: no visible global function definition for ‘rnorm’
    resid_rotate: no visible global function definition for ‘rnorm’
    resid_rotate: no visible global function definition for ‘update’
    resid_rotate: no visible global function definition for ‘resid’
    resid_sigma: no visible global function definition for ‘rnorm’
    rorschach: no visible global function definition for ‘rbinom’
    rss: no visible global function definition for ‘resid’
    sep_dist: no visible global function definition for ‘dist’
    sep_dist: no visible global function definition for ‘cutree’
    sep_dist: no visible global function definition for ‘hclust’
    uni_dist: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      coef cutree dist filter hclust lm predict quantile rbinom resid rnorm
      sd update
    Consider adding
      importFrom("stats", "coef", "cutree", "dist", "filter", "hclust", "lm",
                 "predict", "quantile", "rbinom", "resid", "rnorm", "sd",
                 "update")
    to your NAMESPACE file.
    ```

# nzelect

Version: 0.4.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 64-82 (README.Rmd) 
    Error: processing vignette 'README.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~`proportion Labour`", "~voting_type".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~`proportion Labour`", "~voting_type".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.0Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6409 marked UTF-8 strings
    ```

# oaPlots

Version: 0.0.25

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# oaxaca

Version: 0.1.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'oaxaca.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# obAnalytics

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# observer

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘ensurer’, ‘validate’
    ```

# oddsratio

Version: 1.0.2

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    +                      pred = "x2", values = c(0.099, 0.198))
    > 
    > # insert first odds ratios to plot
    > plot_object <- insert_or(plot_object, or_object1, or_yloc = 3,
    +                          values_xloc = 0.04, line_size = 0.5,
    +                          line_type = "dotdash", text_size = 6,
    +                          values_yloc = 0.5, arrow_col = "red")
    > 
    > # calculate second odds ratio
    > or_object2 <- or_gam(data = data_gam, model = fit_gam, pred = "x2",
    +                      values = c(0.4, 0.6))
    > 
    > # add or_object2 into plot
    > insert_or(plot_object, or_object2, or_yloc = 2.1, values_yloc = 2,
    +           line_col = "green4", text_col = "black",
    +           rect_col = "green4", rect_alpha = 0.2,
    +           line_alpha = 1, line_type = "dashed",
    +           arrow_xloc_r = 0.01, arrow_xloc_l = -0.01,
    +           arrow_length = 0.01, rect = TRUE)
    Error: Aesthetics must be either length 1 or the same as the data (1): xmin, xmax, ymin, ymax, fill, alpha
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      length(ggplot_build(plot_object)$layout$panel_ranges[[1]]$y.range) not equal to 2.
      1/1 mismatches
      [1] 0 - 2 == -2
      
      Warning: No confident interval calculation possible
              for 'glmmPQL' models
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 5 SKIPPED: 0 FAILED: 3
      1. Failure: check bevhaviour of ggplot_build (changed in ggplot2 v2.2) (@test-insert-or.R#19) 
      2. Failure: check bevhaviour of ggplot_build (changed in ggplot2 v2.2) (@test-insert-or.R#20) 
      3. Failure: check bevhaviour of ggplot_build (changed in ggplot2 v2.2) (@test-insert-or.R#21) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 124-133 (function_tutorial.Rmd) 
    Error: processing vignette 'function_tutorial.Rmd' failed with diagnostics:
    Aesthetics must be either length 1 or the same as the data (1): xmin, xmax, ymin, ymax, fill, alpha
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OECD

Version: 0.2.2

## Newly fixed

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 68-70 (oecd_vignette_main.Rmd) 
    Error: processing vignette 'oecd_vignette_main.Rmd' failed with diagnostics:
    HTTP request failed with status: 429 Too Many Requests
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# officer

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# olsrr

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# omicRexposome

Version: 1.2.0

## In both

*   checking whether package ‘omicRexposome’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: namespace ‘omicRexposome’ is not available and has been replaced
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/omicRexposome/new/omicRexposome.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OmicsPLS

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# oncomix

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OncoSimulR

Version: 2.10.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
       Ending warning-mutPropGrowth tests Fri May 11 17:13:15 2018 
        Took  2.08 
      
      
       Starting wide2long tests Fri May 11 17:13:15 2018 
      
       Ending wide2long tests Fri May 11 17:13:15 2018 
        Took  0.55 
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3213 SKIPPED: 10 FAILED: 1
      1. Failure: We crash as we should (@test.mutPropGrowth.R#482) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        doc   5.4Mb
    ```

# onemap

Version: 2.1.1

## In both

*   checking whether package ‘onemap’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: loading Rplot failed
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/onemap/new/onemap.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# onewaytests

Version: 1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OpasnetUtils

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# openEBGM

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# openPrimeR

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.5Mb
      sub-directories of 1Mb or more:
        R         2.1Mb
        extdata  10.2Mb
    ```

# OpenStreetMap

Version: 0.3.3

## In both

*   checking whether package ‘OpenStreetMap’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/OpenStreetMap/new/OpenStreetMap.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘OpenStreetMap’ ...
** package ‘OpenStreetMap’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘OpenStreetMap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/OpenStreetMap/new/OpenStreetMap.Rcheck/OpenStreetMap’

```
### CRAN

```
* installing *source* package ‘OpenStreetMap’ ...
** package ‘OpenStreetMap’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/OpenStreetMap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘OpenStreetMap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/OpenStreetMap/old/OpenStreetMap.Rcheck/OpenStreetMap’

```
# openVA

Version: 1.0.3

## In both

*   checking whether package ‘openVA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/openVA/new/openVA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘openVA’ ...
** package ‘openVA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘openVA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/openVA/new/openVA.Rcheck/openVA’

```
### CRAN

```
* installing *source* package ‘openVA’ ...
** package ‘openVA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/openVA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘openVA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/openVA/old/openVA.Rcheck/openVA’

```
# OptimClassifier

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# optimParallel

Version: 0.7-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# optiRum

Version: 0.37.3

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      > test_check("optiRum")
      Loading required package: optiRum
      ── 1. Failure: giniChart correctly produce a chart, numeric outcome (@test-giniC
      `x` inherits from `ggplot_built` not `list`.
      
      ── 2. Failure: giniChart correctly produce a chart, factor outcome (@test-giniCh
      `x` inherits from `ggplot_built` not `list`.
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 117 SKIPPED: 14 FAILED: 2
      1. Failure: giniChart correctly produce a chart, numeric outcome (@test-giniChart.R#7) 
      2. Failure: giniChart correctly produce a chart, factor outcome (@test-giniChart.R#14) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# optiSel

Version: 2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OPWeight

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# orderedLasso

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plot.orderedLasso.cv: no visible global function definition for
      ‘abline’
    plot.timeLagLasso.cv: no visible global function definition for ‘par’
    plot.timeLagLasso.cv: no visible global function definition for ‘plot’
    plot.timeLagLasso.cv: no visible global function definition for ‘axis’
    plot.timeLagLasso.cv: no visible global function definition for ‘mtext’
    plot.timeLagLasso.cv: no visible global function definition for
      ‘points’
    plot.timeLagLasso.cv: no visible global function definition for
      ‘abline’
    timeLagLasso: no visible binding for global variable ‘sd’
    timeLagLasso.cv: no visible binding for global variable ‘sd’
    timeLagLasso.cv: no visible binding for global variable ‘var’
    timeLagLasso.path: no visible binding for global variable ‘sd’
    Undefined global functions or variables:
      abline approx axis mtext par plot points sd segments var
    Consider adding
      importFrom("graphics", "abline", "axis", "mtext", "par", "plot",
                 "points", "segments")
      importFrom("stats", "approx", "sd", "var")
    to your NAMESPACE file.
    ```

# ordinalClust

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ordinalForest

Version: 2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# ORFik

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Organism.dplyr

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘AnnotationDbi:::smartKeys’ ‘GenomicFeatures:::.exons_with_3utr’
      ‘GenomicFeatures:::.exons_with_5utr’
      ‘GenomicFeatures:::get_TxDb_seqinfo0’
      ‘S4Vectors:::extract_data_frame_rows’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .toGRanges: no visible binding for global variable ‘.’
    intronsByTranscript,src_organism: no visible binding for global
      variable ‘.’
    orgPackageName,src_organism: no visible binding for global variable
      ‘name’
    orgPackageName,src_organism: no visible binding for global variable
      ‘organism’
    orgPackageName,src_organism: no visible binding for global variable
      ‘OrgDb’
    Undefined global functions or variables:
      . OrgDb name organism
    ```

# orgR

Version: 0.9.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OriGen

Version: 1.4.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘OriGen/libs/OriGen.so’:
      Found ‘__gfortran_st_close’, possibly from ‘close’ (Fortran)
        Object: ‘OriGen.o’
      Found ‘__gfortran_st_open’, possibly from ‘open’ (Fortran)
        Object: ‘OriGen.o’
      Found ‘__gfortran_st_read’, possibly from ‘read’ (Fortran)
        Object: ‘OriGen.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# osmplotr

Version: 0.3.0

## In both

*   checking whether package ‘osmplotr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/osmplotr/new/osmplotr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘osmplotr’ ...
** package ‘osmplotr’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘osmplotr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/osmplotr/new/osmplotr.Rcheck/osmplotr’

```
### CRAN

```
* installing *source* package ‘osmplotr’ ...
** package ‘osmplotr’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘osmplotr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/osmplotr/old/osmplotr.Rcheck/osmplotr’

```
# otvPlots

Version: 0.2.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(otvPlots)
      > 
      > test_check("otvPlots")
      ── 1. Failure: expected plot elements are returned (@test_PlotRatesOverTime.R#19
      as.character(p$facet$params$facets[[1]]) not identical to "job".
      Lengths differ: 2 is not 1
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 204 SKIPPED: 0 FAILED: 1
      1. Failure: expected plot elements are returned (@test_PlotRatesOverTime.R#19) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘quantreg’
      All declared Imports should be used.
    ```

# outbreaker2

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘adegenet’
    ```

# OutbreakTools

Version: 0.1-16

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    835          417 2000-01-10 gene2    417
    836          418 2000-01-10 gene2    418
    > x@trees
    1 phylogenetic trees
    > 
    > ## Plotting the dynamic contact network ##
    > par(mfrow=c(2,2))
    > plot(get.contacts(x),main="Contact network - days 0-3",displaylabels=TRUE)
    > plot(get.contacts(x, from=0, to=1.1), main="Contact network - days 0-1",
    +      displaylabels=TRUE)
    > plot(get.contacts(x, from=2, to=2.1), main="Contact network - day 2",
    +      displaylabels=TRUE)
    > plot(get.contacts(x, from=3, to=3.1), main="Contact network - day 3",
    +      displaylabels=TRUE)
    > 
    > ## Mapping the outbreak (by sex) ##
    > plot(x,'geo',location=c('lon','lat'),zoom=13,colorBy='Sex')
    Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=13&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in .local(x, ...) :
      The following sequence IDs are not in the dataset: 311, 222
    Warning in .local(x, ...) :
      The following sequence IDs are not in the dataset: 311, 222
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figs/OutbreakTools-out.width==".8\\textwidth"" -> "figs/OutbreakTools-out.width___.8\\textwidth_"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figs/OutbreakTools-out_width____8//textwidth_")
    Warning: The shape palette can deal with a maximum of 6 discrete values because
    more than 6 becomes difficult to discriminate; you have 11. Consider
    specifying shapes manually if you must have them.
    Warning: Removed 58 rows containing missing values (geom_point).
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=14&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 1022-1023 (OutbreakTools.Rnw) 
    Error: processing vignette 'OutbreakTools.Rnw' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# OutliersO3

Version: 0.5.4

## Newly broken

*   checking whether package ‘OutliersO3’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘OutliersO3’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/OutliersO3/new/OutliersO3.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# overlapping

Version: 1.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# P2C2M

Version: 0.7.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘genealogicalSorting’ ‘phybase’ ‘Rmpi’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ms.exec: no visible global function definition for ‘tail’
    p2c2m.readstarb: no visible global function definition for ‘head’
    p2c2m.readstarb: no visible binding for global variable ‘head’
    rmOutlrs: no visible global function definition for ‘boxplot.stats’
    rmext: no visible global function definition for ‘tail’
    rowMedians: no visible binding for global variable ‘median’
    stats.acrGenes: no visible binding for global variable ‘sd’
    stats.perGene: no visible binding for global variable ‘sd’
    statshelpers.cv: no visible binding for global variable ‘sd’
    statshelpers.qntls: no visible binding for global variable ‘quantile’
    Undefined global functions or variables:
      boxplot.stats head median quantile sd tail
    Consider adding
      importFrom("grDevices", "boxplot.stats")
      importFrom("stats", "median", "quantile", "sd")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 490 marked UTF-8 strings
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: P2C2M
    Loading required package: ggplot2
    Loading required package: grid
    Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
    Warning: Removed 575 rows containing non-finite values
    (stat_density).
    Warning: Removed 2 rows containing missing values (geom_vline).
    Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
    Warning: Removed 575 rows containing non-finite values
    (stat_density).
    Warning: Removed 2 rows containing missing values (geom_vline).
    Warning: Removed 370 rows containing missing values (geom_point).
    Warning: Removed 370 rows containing missing values (geom_point).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'P2C2M_Vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pa

Version: 1.2-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Package in Depends field not imported from: ‘grid’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .formula.make: no visible global function definition for ‘formula’
    regress: no visible global function definition for ‘lm’
    regress: no visible global function definition for ‘model.matrix’
    Undefined global functions or variables:
      formula lm model.matrix
    Consider adding
      importFrom("stats", "formula", "lm", "model.matrix")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: grid
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pa.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PAC

Version: 1.0.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# packcircles

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pacotest

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# padr

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pafdR

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pagenum

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PairedData

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# paleofire

Version: 1.2.2

## Newly broken

*   checking whether package ‘paleofire’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘paleofire’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/paleofire/new/paleofire.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: GCD
    Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘paleofire’
    This is paleofire v1.2.2 | GCD v4.0.2
    trying URL 'http://blarquez.com/public/data/data_cageo.zip'
    Content type 'application/zip' length 14857 bytes (14 KB)
    ==================================================
    downloaded 14 KB
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paleofire-paper.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# palettetown

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pals

Version: 1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pAnalysis

Version: 2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pandaR

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.3Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    dFunction: no visible global function definition for ‘cor’
    importPandaMatlab: no visible global function definition for
      ‘read.delim’
    panda: no visible global function definition for ‘cor’
    panda: no visible global function definition for ‘aggregate’
    plot.panda: no visible global function definition for ‘hist’
    plotCommunityDetection: no visible global function definition for
      ‘title’
    plotZbyTF: no visible global function definition for ‘aggregate’
    prepResult: no visible global function definition for ‘pnorm’
    Undefined global functions or variables:
      aggregate cor hist pnorm read.delim title
    Consider adding
      importFrom("graphics", "hist", "title")
      importFrom("stats", "aggregate", "cor", "pnorm")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.panda':
      ‘plot.panda’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# pander

Version: 0.6.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# panelvar

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# panelView

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# parallelDist

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'parallelDist.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# ParallelTree

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ParamHelpers

Version: 1.10

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/run-all.R’ failed.
    Last 13 lines of output:
         -2.53545942614891, 0.422975436239536, -2.47230070010972, -1.46055918222656), .algo = structure(c(1L, 
         1L, 1L, 1L, 1L, 1L, 1L, 1L, 1L, 1L, 1L, 2L, 2L, 2L, 2L, 2L, 2L, 2L, 3L, 3L, 3L, 3L
         ), .Label = c("algo1", "algo2", "algo3"), class = "factor"), .repl = c(1L, 1L, 1L, 
         1L, 1L, 2L, 2L, 2L, 2L, 2L, 3L, 1L, 1L, 2L, 2L, 2L, 2L, 2L, 1L, 1L, 1L, 1L)), row.names = c("5", 
         "7", "12", "15", "19", "2", "51", "6", "151", "20", "8", "22", "52", "21", "3", "11", 
         "17", "18", "1", "71", "81", "181"), class = "data.frame"), groups = .algo, maximise = c(y1 = FALSE, 
         y2 = FALSE))
      5: stop("formula missing")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1036 SKIPPED: 1 FAILED: 1
      1. Error: plotEAF works (@test_plotEAF.R#27) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# paramtest

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# parlitools

Version: 0.2.1

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# parsemsf

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dbplyr’
      All declared Imports should be used.
    ```

# parSim

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# partialAR

Version: 1.0.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# partialCI

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PASenseWear

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# passport

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PASWR2

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PathoStat

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pathVar

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotOneSample: no visible binding for global variable ‘Number_of_genes’
    plotTwoSamplesCont: no visible binding for global variable ‘PwayName’
    plotTwoSamplesCont: no visible binding for global variable
      ‘NumOfGenesFromDataSetInPathway’
    plotTwoSamplesCont: no visible binding for global variable ‘value’
    plotTwoSamplesCont: no visible binding for global variable
      ‘..density..’
    plotTwoSamplesCont: no visible binding for global variable ‘group’
    plotTwoSamplesDisc: no visible binding for global variable ‘Cluster’
    plotTwoSamplesDisc: no visible binding for global variable
      ‘Number_of_genes’
    sigOneSample: no visible binding for global variable ‘APval’
    sigOneSample: no visible binding for global variable ‘PwayName’
    sigTwoSamplesCont: no visible binding for global variable ‘APval’
    sigTwoSamplesCont: no visible binding for global variable ‘PwayName’
    sigTwoSamplesDisc: no visible binding for global variable ‘APval’
    sigTwoSamplesDisc: no visible binding for global variable ‘PwayName’
    Undefined global functions or variables:
      ..density.. APval Cluster NumOfGenesFromDataSetInPathway
      Number_of_genes PercOfGenesInPway PwayName avg cv group medAbsDev
      standDev value
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Loading required package: gridExtra
    Using X1, X2 as id variables
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pathVar.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# patPRO

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Checking should be performed on sources prepared by ‘R CMD build’.
    ```

# patternplot

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# pauwels2014

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking R code for possible problems ... NOTE
    ```
    add_noise: no visible global function definition for ‘rnorm’
    estimate_risk_dream6 : <anonymous>: no visible global function
      definition for ‘var’
    generate_sample: no visible global function definition for ‘rnorm’
    generate_sample: no visible global function definition for ‘runif’
    sample_function: no visible global function definition for ‘runif’
    sample_function : <anonymous>: no visible global function definition
      for ‘var’
    sample_function_multi_mod_weight: no visible global function definition
      for ‘runif’
    sample_function_multi_mod_weight : <anonymous>: no visible global
      function definition for ‘var’
    sample_function_single_mod: no visible global function definition for
      ‘runif’
    Undefined global functions or variables:
      rnorm runif var
    Consider adding
      importFrom("stats", "rnorm", "runif", "var")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: deSolve
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pauwels2014.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PAWL

Version: 0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘PAWL’ for: ‘show’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘foreach’ ‘ggplot2’ ‘reshape’
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘foreach’ ‘ggplot2’ ‘methods’ ‘mvtnorm’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    smcparameters.constructor: no visible global function definition for
      ‘new’
    systematic_resampling: no visible global function definition for
      ‘runif’
    target.constructor: no visible global function definition for ‘new’
    tuningparameters.constructor: no visible global function definition for
      ‘new’
    show,smcparameters: no visible global function definition for ‘tail’
    Undefined global functions or variables:
      %do% abline aes aes_string cov element_text facet_wrap foreach
      geom_density2d geom_line geom_point geom_step geom_vline ggplot hist
      labs mahalanobis melt new quantile rgamma rmultinom rmvnorm rnorm
      runif stat_bin2d tail theme xlab ylab ylim
    Consider adding
      importFrom("graphics", "abline", "hist")
      importFrom("methods", "new")
      importFrom("stats", "cov", "mahalanobis", "quantile", "rgamma",
                 "rmultinom", "rnorm", "runif")
      importFrom("utils", "tail")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# Pbase

Version: 0.20.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        data      1.7Mb
        extdata   2.2Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
      Johannes Rainer <Johannes.Rainer@eurac.edu> [aut, cre]
      Sebastian Gibb <mail@sebastiangibb.de> [aut, cre]
    ```

# pbcmc

Version: 1.8.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
          order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
          rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
          union, unique, unsplit, which, which.max, which.min
      
      Welcome to Bioconductor
      
          Vignettes contain introductory material; view with
          'browseVignettes()'. To cite Bioconductor, see
          'citation("Biobase")', and for packages 'citation("pkgname")'.
      
      No methods found in package 'BiocGenerics' for request: 'unlist' when loading 'pbcmc'
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Keeping only annotated EntrezGene.ID genes
    Keeping only PAM50 available genes.
    Getting PAM50 subtypes...
    50/50 probes are used for clustering
    no standardization of the gene expressions
    Obtaining 10000 permutations for 5 subjects...
    Obtaining 10000 permutations for 5 subjects... done.
    PAM50 Permutation Test results!!! 
    
    Permutations: 10000
    pcutoff < 0.01
    corCutoff > 0.1
    
    Global results:
    Assigned 
           5 
    Particular results:
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pbcmc-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PBImisc

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PCADSC

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Matrix’ ‘pander’
      All declared Imports should be used.
    ```

# pcaExplorer

Version: 2.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        doc   6.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    pcaExplorer: no visible binding for '<<-' assignment to
      ‘pcaexplorer_env’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘airway’
    pcaExplorer : <anonymous>: no visible binding for global variable
      ‘pcaexplorer_env’
    Undefined global functions or variables:
      airway pcaexplorer_env
    ```

# pcaMethods

Version: 1.72.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    svdImpute: no visible global function definition for ‘prcomp’
    svdPca: no visible global function definition for ‘prcomp’
    plot,pcaRes: no visible global function definition for ‘gray’
    plot,pcaRes: no visible global function definition for ‘barplot’
    plot,pcaRes: no visible global function definition for ‘legend’
    slplot,pcaRes: no visible global function definition for ‘par’
    slplot,pcaRes: no visible global function definition for ‘layout’
    slplot,pcaRes: no visible global function definition for ‘abline’
    slplot,pcaRes: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline barplot cor cov gray layout legend lines median na.omit pairs
      par points prcomp qf rnorm runif setTxtProgressBar text
      txtProgressBar
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "abline", "barplot", "layout", "legend",
                 "lines", "pairs", "par", "points", "text")
      importFrom("stats", "cor", "cov", "median", "na.omit", "prcomp", "qf",
                 "rnorm", "runif")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    
    Attaching package: ‘pcaMethods’
    
    The following object is masked from ‘package:stats’:
    
        loadings
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'missingValues.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pcaPA

Version: 2.0.2

## In both

*   checking whether package ‘pcaPA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/new/pcaPA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘pcaPA’ ...
** package ‘pcaPA’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG `gsl-config --cflags`  -I/usr/local/include   -fPIC  -Wall -g -O2  -c Cpolychoric.cpp -o Cpolychoric.o
/bin/sh: gsl-config: command not found
Cpolychoric.cpp:36:10: fatal error: 'gsl/gsl_math.h' file not found
#include <gsl/gsl_math.h>
         ^~~~~~~~~~~~~~~~
1 error generated.
make: *** [Cpolychoric.o] Error 1
ERROR: compilation failed for package ‘pcaPA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/new/pcaPA.Rcheck/pcaPA’

```
### CRAN

```
* installing *source* package ‘pcaPA’ ...
** package ‘pcaPA’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG `gsl-config --cflags`  -I/usr/local/include   -fPIC  -Wall -g -O2  -c Cpolychoric.cpp -o Cpolychoric.o
/bin/sh: gsl-config: command not found
Cpolychoric.cpp:36:10: fatal error: 'gsl/gsl_math.h' file not found
#include <gsl/gsl_math.h>
         ^~~~~~~~~~~~~~~~
1 error generated.
make: *** [Cpolychoric.o] Error 1
ERROR: compilation failed for package ‘pcaPA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/pcaPA/old/pcaPA.Rcheck/pcaPA’

```
# pcr

Version: 1.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Joining, by = c("group", "gene")
    Joining, by = c("group", "gene")
    Joining, by = "gene"
    Joining, by = c("group", "gene")
    Joining, by = c("group", "gene")
    Joining, by = "gene"
    Joining, by = c("group", "gene")
    Joining, by = c("group", "gene")
    Parsed with column specification:
    cols(
      ref = col_double(),
      target = col_double()
    )
    Joining, by = c("group", "gene")
    Joining, by = c("group", "gene", "calibrated")
    Joining, by = c("group", "gene")
    ! LaTeX Error: \caption outside float.
    
    Error: processing vignette 'qpcr_analysis.Rmd' failed with diagnostics:
    Failed to compile qpcr_analysis.tex. See qpcr_analysis.log for more info.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pcrcoal

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pcrsim

Version: 1.0.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘strvalidator’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# PDN

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘glmnet’
      All declared Imports should be used.
    ```

# pdp

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ICEbox’
    ```

# PDQutils

Version: 0.1.6

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    large z may result in inaccurate quantiles
    Loading required package: polynom
    Loading required package: reshape2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PDQutils.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pdSpecEst

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PeakError

Version: 2017.06.19

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PeakSegDP

Version: 2017.08.15

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PeakSegJoint

Version: 2017.08.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PeakSegOptimal

Version: 2017.07.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PedCNV

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘Rcpp’ ‘RcppArmadillo’ ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    Inde: no visible global function definition for ‘var’
    STE: no visible global function definition for ‘pchisq’
    STIM: no visible global function definition for ‘cor’
    STIM: no visible global function definition for ‘pchisq’
    plot.clust: no visible global function definition for ‘princomp’
    plot.clust: no visible global function definition for ‘loadings’
    plot.clust: no visible global function definition for ‘qplot’
    plot.clust: no visible global function definition for ‘geom_histogram’
    plot.clust: no visible global function definition for ‘aes_string’
    plot.clust: no visible global function definition for ‘barplot’
    plot.clust: no visible global function definition for ‘title’
    plot.clust: no visible global function definition for ‘mtext’
    plot.clust: no visible global function definition for ‘text’
    Undefined global functions or variables:
      aes_string barplot cor cov geom_histogram kmeans lm loadings mtext
      pchisq prcomp princomp qplot rbinom text title var
    Consider adding
      importFrom("graphics", "barplot", "mtext", "text", "title")
      importFrom("stats", "cor", "cov", "kmeans", "lm", "loadings", "pchisq",
                 "prcomp", "princomp", "rbinom", "var")
    to your NAMESPACE file.
    ```

# pems.utils

Version: 0.2.17.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# penaltyLearning

Version: 2017.12.08

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             })
             do.call(rbind, dt.list)
         })
      7: values(fs)
      8: values.list(fs)
      9: value(tmp, ...)
      10: value.Future(tmp, ...)
      11: resignalCondition(future)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 58 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test-demo8.R#8) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pepStat

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        expand.grid
    
    No methods found in package 'GenomicRanges' for request: 'mcols<-' when loading 'pepStat'
    No methods found in package 'GenomicRanges' for request: 'mcols' when loading 'pepStat'
    Loading required package: Gviz
    Loading required package: GenomicRanges
    Loading required package: GenomeInfoDb
    Loading required package: grid
    
    Attaching package: 'Gviz'
    
    The following object is masked from 'package:pepStat':
    
        position
    
    Some peptides have no match in the GRanges object rownames and are removed from the peptideSet!
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pepStat.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘pepStat’ for: ‘end’, ‘start’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    coerce,peptideSet-ExpressionSet: no visible global function definition
      for ‘annotation’
    end,peptideSet: no visible global function definition for ‘end’
    position,peptideSet: no visible global function definition for ‘start’
    position,peptideSet: no visible global function definition for ‘end’
    start,peptideSet: no visible global function definition for ‘start’
    write.pSet,peptideSet: no visible global function definition for
      ‘start’
    write.pSet,peptideSet: no visible global function definition for ‘end’
    write.pSet,peptideSet: no visible global function definition for
      ‘write.csv’
    Undefined global functions or variables:
      annotation dev.flush dev.hold dev.interactive devAskNewPage end
      lm.fit lm.wfit mcols mcols<- median read.csv sd start write.csv
    Consider adding
      importFrom("grDevices", "dev.flush", "dev.hold", "dev.interactive",
                 "devAskNewPage")
      importFrom("stats", "end", "lm.fit", "lm.wfit", "median", "sd",
                 "start")
      importFrom("utils", "read.csv", "write.csv")
    to your NAMESPACE file.
    ```

# peptider

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pequod

Version: 0.0-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# perccalc

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# performanceEstimation

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# permubiome

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# perry

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    hasMethod: no visible global function definition for ‘getS3method’
    Undefined global functions or variables:
      getS3method
    Consider adding
      importFrom("utils", "getS3method")
    to your NAMESPACE file.
    ```

# personalized

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# perspectev

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# perturbatr

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# perturbR

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# petro.One

Version: 0.1.3

## In both

*   checking whether package ‘petro.One’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/petro.One/new/petro.One.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘petro.One’ ...
** package ‘petro.One’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘petro.One’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/petro.One/new/petro.One.Rcheck/petro.One’

```
### CRAN

```
* installing *source* package ‘petro.One’ ...
** package ‘petro.One’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/petro.One/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘petro.One’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/petro.One/old/petro.One.Rcheck/petro.One’

```
# PGA

Version: 1.10.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning: Ignoring unknown aesthetics: position
    Warning: Ignoring unknown aesthetics: position
    Output variation table and variant protein sequence caused bySNVs...  done
    Output novel junction peptides...  done
    Output novel transripts... Import genomic features from the file as a GRanges object ... OK
    Prepare the 'metadata' data frame ... OK
    Make the TxDb object ... OK
    ====== Processing reverse strand ======
    ====== Processing forward strand ======
    ====== Processing unknown strand ======
     done
    Cat all fasta ...  done
    Warning: Ignoring unknown aesthetics: position
    Warning: Ignoring unknown aesthetics: position
    INDEL(DB) didn't exist!
    Warning: Ignoring unknown aesthetics: position
    Warning: Ignoring unknown aesthetics: position
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PGA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        extdata   1.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘biomaRt:::martBM’ ‘biomaRt:::martDataset’ ‘biomaRt:::martHost’
      ‘customProDB:::makeTranscriptDbFromBiomart_archive’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    reportSNV: no visible binding for global variable ‘isUnique’
    reportSNV: no visible binding for global variable ‘Index’
    reportSNV: no visible binding for global variable ‘aaref’
    reportSNV: no visible binding for global variable ‘aavar’
    reportSNV: no visible binding for global variable ‘genename’
    reportSNV: no visible binding for global variable ‘proname’
    reportSNV: no visible binding for global variable ‘.SD’
    reportSNV: no visible binding for global variable ‘ID’
    reportSNV: no visible binding for global variable ‘Change’
    reportSNV: no visible binding for global variable ‘aapos’
    reportSNV: no visible binding for global variable ‘abc’
    reportSNV: no visible binding for global variable ‘xyz’
    Undefined global functions or variables:
      . .I .N .SD CUFF_ID Change Class Evalue Frame Freq ID Index Mass
      MutNum Query Qvalue Strand Substring Type aapos aaref aavar abc
      alleleCount alleles charge chr chrom cumlen delta_da delta_ppm evalue
      gene_name genename genome<- id isSAP isUnique junType jun_type label
      miss mods mrnaAcc mz name output pep peptide pincoding position
      pro_name proname prot protAcc protein rbindlist readAAStringSet
      readDNAStringSet refbase rsid seqlengths seqlevels seqlevels<- subseq
      transcript tx_name txid txname varbase writeXStringSet x xyz y
    ```

# PGPC

Version: 1.8.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘EBImage’ ‘imageHTS’ ‘SearchTrees’ ‘limma’ ‘RColorBrewer’ ‘gplots’
      ‘splots’ ‘ggplot2’ ‘geneplotter’ ‘ChemmineR’ ‘reshape2’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 251.9Mb
      sub-directories of 1Mb or more:
        data     172.8Mb
        doc        7.9Mb
        extdata   70.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘ChemmineR’ ‘RColorBrewer’ ‘geneplotter’ ‘ggplot2’ ‘gplots’ ‘plyr’
      ‘reshape2’ ‘splots’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    Unexported object imported by a ':::' call: ‘imageHTS:::writeThumbnail’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    summarizeWellsExtended: no visible binding for global variable ‘sd’
    summarizeWellsExtended: no visible binding for global variable
      ‘quantile’
    summarizeWellsExtended: no visible global function definition for
      ‘write.table’
    Undefined global functions or variables:
      abline aes as.dist colorRampPalette concdrug1.micro_M. cor data
      dev.off element_blank element_text experiment geom_bar geom_errorbar
      geom_jitter geom_line geom_point geom_vline ggplot gpar grid.raster
      grid.text hclust heatmap.2 identifier inhibition labs lm mad
      medpolish p.adjust plot png points popViewport position_dodge
      position_jitter pushViewport pvalue quantile scale_colour_manual
      scale_fill_manual scale_x_log10 scale_y_continuous sd sem theme
      theme_bw unit value variable viewport write.table xlab ylab
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "png")
      importFrom("graphics", "abline", "plot", "points")
      importFrom("stats", "as.dist", "cor", "hclust", "lm", "mad",
                 "medpolish", "p.adjust", "quantile", "sd")
      importFrom("utils", "data", "write.table")
    to your NAMESPACE file.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 28 marked Latin-1 strings
    ```

# PGRdup

Version: 0.2.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘RecordLinkage’
    ```

# phantasus

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 18.0Mb
      sub-directories of 1Mb or more:
        doc   4.6Mb
        www  12.8Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'GEOquery:::getDirListing' 'opencpu:::rookhandler'
      'opencpu:::tmp_root' 'opencpu:::win_or_mac'
      See the note in ?`:::` about the use of this operator.
    ```

# phase1RMD

Version: 1.0.8

## In both

*   checking whether package ‘phase1RMD’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/new/phase1RMD.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘phase1RMD’ ...
** package ‘phase1RMD’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘phase1RMD’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/new/phase1RMD.Rcheck/phase1RMD’

```
### CRAN

```
* installing *source* package ‘phase1RMD’ ...
** package ‘phase1RMD’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/phase1RMD/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘phase1RMD’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/phase1RMD/old/phase1RMD.Rcheck/phase1RMD’

```
# PhaseType

Version: 0.1.3

## In both

*   checking whether package ‘PhaseType’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      LJMA_arms.c:297:11: warning: comparison of array 'env->p' equal to a null pointer is always false [-Wtautological-pointer-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PhaseType/new/PhaseType.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Packages in Depends field not imported from:
      ‘ggplot2’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plot.phtMCMC: no visible global function definition for ‘ggplot’
    plot.phtMCMC: no visible global function definition for ‘melt’
    plot.phtMCMC: no visible global function definition for ‘geom_line’
    plot.phtMCMC: no visible global function definition for ‘aes_string’
    plot.phtMCMC: no visible global function definition for ‘geom_smooth’
    plot.phtMCMC: no visible global function definition for ‘facet_wrap’
    plot.phtMCMC: no visible global function definition for ‘ggtitle’
    plot.phtMCMC: no visible global function definition for ‘xlab’
    plot.phtMCMC: no visible global function definition for ‘ylab’
    plot.phtMCMC: no visible global function definition for ‘geom_density’
    Undefined global functions or variables:
      aes_string facet_wrap geom_density geom_line geom_smooth ggplot
      ggtitle melt xlab ylab
    ```

*   checking compiled code ... NOTE
    ```
    File ‘PhaseType/libs/PhaseType.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Object: ‘LJMA_arms.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# phenopath

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# phenoTest

Version: 1.28.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘org.Ce.eg.db’ ‘org.Mm.eg.db’ ‘org.Rn.eg.db’ ‘org.Dm.eg.db’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    Packages listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘annotate’ ‘GSEABase’
    A package should be listed in only one of these fields.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘phenoTest’ for: ‘show’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘ggplot2’ which was already attached by Depends.
      Please remove these calls from your code.
    Namespace in Imports field not imported from: ‘annotate’
      All declared Imports should be used.
    Packages in Depends field not imported from:
      ‘BMA’ ‘Heatplus’ ‘annotate’ ‘ggplot2’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘sortDragHtmlTable’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      gray hclust heatmap_plus image integrate kruskal.test layout legend
      lines lm median model.matrix new p.adjust pData par pc1 pc2 pchisq
      pdf png pnorm polygon prcomp predict qplot quantile rect rgb sd text
      theme wilcox.test write.csv y
    Consider adding
      importFrom("grDevices", "densCols", "dev.off", "gray", "pdf", "png",
                 "rgb")
      importFrom("graphics", "abline", "axTicks", "axis", "barplot",
                 "boxplot", "dotchart", "image", "layout", "legend", "lines",
                 "par", "polygon", "rect", "text")
      importFrom("methods", "new")
      importFrom("stats", "BIC", "anova", "approx", "approxfun",
                 "as.dendrogram", "as.dist", "binom.test", "chisq.test",
                 "coef", "coefficients", "complete.cases", "confint", "cor",
                 "cov", "cutree", "density", "dist", "hclust", "integrate",
                 "kruskal.test", "lm", "median", "model.matrix", "p.adjust",
                 "pchisq", "pnorm", "prcomp", "predict", "quantile", "sd",
                 "wilcox.test")
      importFrom("utils", "write.csv")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'plot.gseaData':
      ‘plot.gseaData’
    
    S3 methods shown with full name in documentation object 'plot.gseaSignatures':
      ‘plot.gseaSignaturesSign’
    
    S3 methods shown with full name in documentation object 'summary.gseaData':
      ‘summary.gseaData’
    
    S3 methods shown with full name in documentation object 'summary.gseaSignificance':
      ‘summary.gseaSignificanceSign’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
     
    //------------please use function changes() to see new changes-------------//
    
    //------------please report any bug to xinwang2hms@gmail.com---------------//
    
    /////////////////////////////////////////////////////////////////////////////
    Loading required package: graph
    
    Attaching package: ‘graph’
    
    The following object is masked from ‘package:XML’:
    
        addNode
    
    Warning in checkGsetSimmetry(x, gsets, test, B) :
      The provided score is strongly not simmetric or is not centered on 0. You should set paremeter 'center' to TRUE.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'phenoTest.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PhenotypeSimulator

Version: 0.2.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘PhenotypeSimulator-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: readStandardGenotypes
    > ### Title: Read genotypes from file.
    > ### Aliases: readStandardGenotypes
    > 
    > ### ** Examples
    > 
    > # Genome format
    > filename_genome  <- system.file("extdata/genotypes/genome/",
    + "genotypes_genome.txt",
    + package = "PhenotypeSimulator") 
    > data_genome <- readStandardGenotypes(N=100, filename_genome, format ="genome")
    Warning in data.table::fread(filename, skip = "Samples:", data.table = FALSE,  :
      Detected 1 column names but the data has 2 columns (i.e. invalid file). Added 1 extra default column name for the first column which is guessed to be row names or an index. Use setnames() afterwards if this guess is not correct, or fix the file write command that created the file to create a valid file.
    Error in strsplit(data$V2, split = "") : non-character argument
    Calls: readStandardGenotypes -> matrix -> unlist -> strsplit
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# PhenStat

Version: 2.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    ----------------------------------------------------------------------------
    Classification Tag
    ----------------------------------------------------------------------------
    With phenotype threshold value 0.01 - significant in males, females and in combined dataset
    
    ----------------------------------------------------------------------------
    Count Matrices
    ----------------------------------------------------------------------------
    
    
    
    
    
    
    
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PhenStat.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# philr

Version: 1.6.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 142-145 (philr-intro.Rmd) 
    Error: processing vignette 'philr-intro.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    name.balance: no visible global function definition for ‘as’
    vote.annotation: no visible global function definition for ‘is’
    Undefined global functions or variables:
      as is
    Consider adding
      importFrom("methods", "as", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘compositions’
    ```

# phonics

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# photobiologyFilters

Version: 0.4.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# photobiologyInOut

Version: 0.4.14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# photobiologyLamps

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘photobiologyLEDs’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked Latin-1 string
    ```

# photobiologyLEDs

Version: 0.4.3-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# photobiologyPlants

Version: 0.4.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Warning in O_plot(spct = x, w.band = w.band, range = range, pc.out = pc.out,  :
      Only 'total' transmittance can be meaningfully plotted in a combined plot
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'dataplotsPlants.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# photobiologySensors

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# photobiologySun

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Phxnlme

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PhyInformR

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gplots’ ‘phytools’
      All declared Imports should be used.
    ```

# phylopath

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# phyloseq

Version: 1.24.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    $rank:  2
    
    eigen values: 0.004811 0.00125
    
      vector length mode    content           
    1 $dw    58     numeric category weights  
    2 $lw    3      numeric collection weights
    3 $eig   2      numeric eigen values      
    
      data.frame nrow ncol content                                       
    1 $dls       58   2    coordinates of the categories                 
    2 $li        3    2    coordinates of the collections                
    3 $c1        57   2    scores of the principal axes of the categories
    > plot_ordination(esophagus, eso.dpcoa, "samples")
    No available covariate data to map on the points for this plot `type`
    > plot_ordination(esophagus, eso.dpcoa, "species")
    No available covariate data to map on the points for this plot `type`
    > plot_ordination(esophagus, eso.dpcoa, "biplot")
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to shape
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat-phyloseq.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 521 SKIPPED: 0 FAILED: 12
      1. Failure: all 4 plot_ordination type options result in valid ggplot2 object (@test-plot.R#51) 
      2. Failure: all 4 plot_ordination type options result in valid ggplot2 object (@test-plot.R#52) 
      3. Error: all 4 plot_ordination type options result in valid ggplot2 object (@test-plot.R#53) 
      4. Error: plot_ordination: When variables are present or not, color SampleType (@test-plot.R#83) 
      5. Failure: plot_ordination: When variables are present or not, shape SamplyType (@test-plot.R#105) 
      6. Error: plot_ordination: When variables are present or not, shape SamplyType (@test-plot.R#106) 
      7. Error: plot_ordination: When variables are present or not, label SamplyType (@test-plot.R#121) 
      8. Error: plot_ordination: Continuous variables still mapped, uses added dummy variable (@test-plot.R#138) 
      9. Error: plot_ordination: Some additional formats and warnings. (@test-plot.R#165) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 181-183 (phyloseq-FAQ.Rmd) 
    Error: processing vignette 'phyloseq-FAQ.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.1Mb
      sub-directories of 1Mb or more:
        doc   6.3Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    merge_phyloseq_pair,sample_data-sample_data : <anonymous>: no visible
      binding for global variable ‘X0’
    merge_samples,sample_data: no visible global function definition for
      ‘aggregate’
    plot_phyloseq,phyloseq: no visible binding for global variable
      ‘esophagus’
    Undefined global functions or variables:
      #OTU ID .SD := Abundance Classification Consensus Lineage J OTU
      OTULabel SE.sim Sample V1 V2 X0 aggregate as.dist as.formula
      as.hclust axis capture.output combn complete.cases count cutree
      dcast.data.table download.file eigenvalue esophagus gap h.adj.index
      head i k label queryID queryString read read.table relevel se tail
      untar unzip value vmax vmin write.table x xdodge xend xfartiplab
      xleft xright y yend
    Consider adding
      importFrom("graphics", "axis")
      importFrom("stats", "aggregate", "as.dist", "as.formula", "as.hclust",
                 "complete.cases", "cutree", "relevel")
      importFrom("utils", "capture.output", "combn", "download.file", "head",
                 "read.table", "tail", "untar", "unzip", "write.table")
    to your NAMESPACE file.
    ```

# phyloseqGraphTest

Version: 0.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘phyloseqGraphTest-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_permutations
    > ### Title: Plots the permutation distribution
    > ### Aliases: plot_permutations
    > 
    > ### ** Examples
    > 
    > library(phyloseq)
    > data(enterotype)
    > gt = graph_perm_test(enterotype, sampletype = "SeqTech")
    > plot_permutations(gt)
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Error: Aesthetics must be either length 1 or the same as the data (1): x, y, xend, yend, bins
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 175-176 (gt_vignette.Rmd) 
    Error: processing vignette 'gt_vignette.Rmd' failed with diagnostics:
    Columns `xend`, `yend`, `x`, `y` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘intergraph’
      All declared Imports should be used.
    ```

# phylosim

Version: 3.0.2

## In both

*   R CMD check timed out
    

# physiology

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Pi

Version: 1.8.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘XGR’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# pid

Version: 0.36

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pifpaf

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# pinbasic

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pipe.design

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pirate

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pitchRx

Version: 1.8.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘ggsubplot’
    ```

# pivottabler

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.4Mb
      sub-directories of 1Mb or more:
        data   2.0Mb
        doc    2.4Mb
    ```

# pkggraph

Version: 0.2.2

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 34-58 (vignette_pkggraph.Rmd) 
    Error: processing vignette 'vignette_pkggraph.Rmd' failed with diagnostics:
    Columns `xend`, `yend`, `x`, `y` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PKgraph

Version: 1.7

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available:
      ‘RGtk2’ ‘gWidgetsRGtk2’ ‘cairoDevice’ ‘rggobi’
    
    Depends: includes the non-default packages:
      ‘RGtk2’ ‘gWidgetsRGtk2’ ‘cairoDevice’ ‘lattice’ ‘rggobi’ ‘ggplot2’
      ‘proto’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# PKNCA

Version: 0.8.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PKPDmisc

Version: 2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘purrr’
      All declared Imports should be used.
    ```

# PKreport

Version: 1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PKreport.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    generate.plot: no visible global function definition for ‘dev.off’
    writeHTML: no visible global function definition for
      ‘packageDescription’
    writeLst: no visible global function definition for
      ‘packageDescription’
    writeLst.tab: no visible global function definition for
      ‘packageDescription’
    writeTable: no visible global function definition for
      ‘packageDescription’
    writeTable: no visible global function definition for ‘quantile’
    initialize,nonmem: no visible global function definition for
      ‘read.table’
    Undefined global functions or variables:
      bmp browseURL dev.cur dev.new dev.off formula heatmap jpeg
      packageDescription png quantile read.table tiff
    Consider adding
      importFrom("grDevices", "bmp", "dev.cur", "dev.new", "dev.off", "jpeg",
                 "png", "tiff")
      importFrom("stats", "formula", "heatmap", "quantile")
      importFrom("utils", "browseURL", "packageDescription", "read.table")
    to your NAMESPACE file.
    ```

# planar

Version: 1.6

## In both

*   checking whether package ‘planar’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      hcubature.c:917:15: warning: comparison of unsigned enum expression < 0 is always false [-Wtautological-compare]
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/planar/new/planar.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PlasmaMutationDetector

Version: 1.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Plasmidprofiler

Version: 0.1.6

## Newly broken

*   checking whether package ‘Plasmidprofiler’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘Plasmidprofiler’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/Plasmidprofiler/new/Plasmidprofiler.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# platetools

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Plots multiple b-scored normalised platemaps
    > ### Aliases: b_grid
    > 
    > ### ** Examples
    > 
    > df01 <- data.frame(well = num_to_well(1:96),
    +   vals = rnorm(96),
    +   plate = 1)
    > 
    > df02 <- data.frame(well = num_to_well(1:96),
    +   vals = rnorm(96),
    +   plate = 2)
    > 
    > df <- rbind(df01, df02)
    > 
    > b_grid(data = df$vals,
    +     well = df$well,
    +     plate_id = df$plate,
    +     plate = 96)
    Error: Free scales are only supported with `coord_cartesian()` and `coord_flip()`
    Execution halted
    ```

*   checking whether package ‘platetools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘platetools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/platetools/new/platetools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# playwith

Version: 0.9-54

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘cairoDevice’ ‘gWidgetsRGtk2’ ‘RGtk2’
    
    Package suggested but not available for checking: ‘latticist’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# plethy

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘csv.to.table’ ‘find.break.ranges.integer’ ‘fix.time’ ‘multi.grep’
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    mvtsplot.data.frame: no visible global function definition for ‘bxp’
    mvtsplot.data.frame : <anonymous>: no visible binding for global
      variable ‘median’
    mvtsplot.data.frame: no visible global function definition for ‘lines’
    mvtsplot.data.frame: no visible global function definition for ‘Axis’
    mvtsplot.data.frame: no visible global function definition for ‘legend’
    retrieveMatrix,BuxcoDB: no visible global function definition for
      ‘terms’
    tsplot,BuxcoDB: no visible binding for global variable ‘Days’
    tsplot,BuxcoDB: no visible binding for global variable ‘Value’
    tsplot,BuxcoDB: no visible binding for global variable ‘Sample_Name’
    Undefined global functions or variables:
      Axis Days Sample_Name Value abline bxp colors layout legend lines
      median mtext packageDescription par plot rnorm strwidth terms
    Consider adding
      importFrom("grDevices", "colors")
      importFrom("graphics", "Axis", "abline", "bxp", "layout", "legend",
                 "lines", "mtext", "par", "plot", "strwidth")
      importFrom("stats", "median", "rnorm", "terms")
      importFrom("utils", "packageDescription")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Loading required package: S4Vectors
    Loading required package: stats4
    
    Attaching package: ‘S4Vectors’
    
    The following object is masked from ‘package:base’:
    
        expand.grid
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plethy.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# plotluck

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘hexbin’ ‘quantreg’
      All declared Imports should be used.
    ```

# plotly

Version: 4.7.1

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1124 SKIPPED: 25 FAILED: 25
      1. Error: SharedData produces key/set in ggpairs (@test-animate-highlight.R#28) 
      2. Failure: Translates both dates and datetimes (with dynamic ticks) correctly (@test-ggplot-lines.R#85) 
      3. Failure: Translates both dates and datetimes (with dynamic ticks) correctly (@test-ggplot-lines.R#86) 
      4. Failure: geom_linerange() without a y aesthetic translates to a path (@test-ggplot-lines.R#111) 
      5. Failure: polygons with different hovertext must be different traces  (@test-ggplot-polygons.R#25) 
      6. Failure: polygons with different hovertext must be different traces  (@test-ggplot-polygons.R#26) 
      7. Failure: polygons with different hovertext must be different traces  (@test-ggplot-polygons.R#31) 
      8. Failure: borders become one trace with NA (@test-ggplot-polygons.R#147) 
      9. Failure: borders become one trace with NA (@test-ggplot-polygons.R#149) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘sf’ ‘RSelenium’
    ```

# plotMElm

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PlotPrjNetworks

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# plotROC

Version: 2.2.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 167-171 (examples.Rmd) 
    Error: processing vignette 'examples.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# plotrr

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘stats’
      All declared Imports should be used.
    ```

# plspm

Version: 0.4.9

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:plspm':
    
        alpha
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plspm_introduction.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pltesim

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# plyranges

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pmc

Version: 1.0.2

## In both

*   R CMD check timed out
    

# pmxTools

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘GGally’ ‘PKNCA’ ‘grid’ ‘magrittr’ ‘plyr’ ‘xpose’
      All declared Imports should be used.
    ```

# pogos

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'pogos.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    compoundsByCell: no visible binding for global variable ‘cell_lines_v1’
    compoundsByCell: no visible binding for global variable ‘compounds_v1’
    compoundsByCell: no visible binding for global variable ‘datasets_v1’
    lkc: no visible binding for global variable ‘cell_lines_v1’
    lkc: no visible binding for global variable ‘compounds_v1’
    lkc: no visible binding for global variable ‘datasets_v1’
    plot,DRProfSet-missing: no visible binding for global variable ‘dose’
    plot,DRProfSet-missing: no visible binding for global variable
      ‘response’
    plot,DRProfSet-missing: no visible binding for global variable ‘drug’
    plot,DRTraceSet-missing: no visible binding for global variable ‘dose’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘response’
    plot,DRTraceSet-missing: no visible binding for global variable
      ‘cell_line’
    Undefined global functions or variables:
      cell_line cell_lines_v1 compounds_v1 datasets_v1 dose drug response
    ```

# pointRes

Version: 1.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# politeness

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pollstR

Version: 2.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# polypoly

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pomp

Version: 1.16

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pompom

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pooling

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PopED

Version: 0.3.2

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 125-126 (intro-poped.Rmd) 
    Error: processing vignette 'intro-poped.Rmd' failed with diagnostics:
    Discrete value supplied to continuous scale
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Missing or unexported object: ‘GA::gaMonitor2’
    ```

# popEpi

Version: 0.4.4

## In both

*   checking examples ... ERROR
    ```
    ...
    > dt <- makeWeightsDT(ag, print = quote(gender), adjust = as, 
    +                     values = vs, weights = ws, enclos = TF)
    > ## or NULL: uses calling frame by default.
    > dt <- makeWeightsDT(ag, print = quote(gender), adjust = as, 
    +                     values = vs, weights = ws,
    +                     enclos = NULL)
    > ## passing parent.fram(1) is the same thing (as below),
    > ## but won't pass in testing these examples somehow (but work in real life)
    > # dt <- makeWeightsDT(ag, print = quote(gender), adjust = as, 
    > #                     values = vs, weights = ws,
    > #                     enclos = NULL)                  
    > 
    > #### formula usage
    > form <- Surv(fot, factor(from0to1))~gender
    > dt <- makeWeightsDT(ag, formula = form, Surv.response = TRUE,
    +                     adjust = as, values = vs, weights = ws,
    +                     enclos = NULL)
    Error in as.data.table.array(xi, keep.rownames = keep.rownames) : 
      as.data.table.array method should be only called for arrays with 3+ dimensions, for 2 dimensions matrix method should be used
    Calls: makeWeightsDT ... Surv2DT -> data.table -> as.data.table -> as.data.table.array
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      4: as.data.table(xi, keep.rownames = keep.rownames)
      5: as.data.table.array(xi, keep.rownames = keep.rownames)
      6: stop("as.data.table.array method should be only called for arrays with 3+ dimensions, for 2 dimensions matrix method should be used")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 215 SKIPPED: 41 FAILED: 6
      1. Error: removing consecutively bad surv.ints is logical w/ & w/out adjusting (@test_survtab_bad_surv_ints.R#16) 
      2. Error: survtab_ag messages & results due to non-consecutively bad surv.ints are OK (@test_survtab_bad_surv_ints.R#69) 
      3. Error: surv.obs about the same as Kaplan-Meier & CIFs close to Aalen-Johansen (@test_survtab_observed.R#14) 
      4. Error: survtab works with more complicated estimation (@test_survtab_observed.R#112) 
      5. Error: relative survivals about the same as relsurv's (@test_survtab_relative.R#57) 
      6. Error: evalPopFormula & usePopFormula output is stable (@test_utils.R#251) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 38-67 (survtab_examples.Rmd) 
    Error: processing vignette 'survtab_examples.Rmd' failed with diagnostics:
    as.data.table.array method should be only called for arrays with 3+ dimensions, for 2 dimensions matrix method should be used
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PopGenReport

Version: 3.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PopGenReportIntroduction.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘ecodist’
    ```

# poppr

Version: 2.7.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in replayPlot(x) : "border" is not a graphical parameter
    Warning in replayPlot(x) : "fill" is not a graphical parameter
    Warning in infinite_vals_replacement(D, warning) :
      Infinite values detected.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'algo.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        R     2.1Mb
        doc   2.4Mb
    ```

# popReconstruct

Version: 1.0-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    estMod.rinvGamma.mar29: no visible global function definition for
      ‘rgamma’
    life.expectancy.stationary: no visible global function definition for
      ‘head’
    life.expectancy.stationary: no visible global function definition for
      ‘tail’
    log.lhood.mar29: no visible global function definition for ‘dnorm’
    log.post.mar29: no visible global function definition for ‘dnorm’
    popRecon.sampler: no visible global function definition for ‘rnorm’
    popRecon.sampler: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      dnorm head rgamma rnorm runif tail
    Consider adding
      importFrom("stats", "dnorm", "rgamma", "rnorm", "runif")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:stats’:
    
        nobs
    
    The following object is masked from ‘package:utils’:
    
        object.size
    
    The following object is masked from ‘package:base’:
    
        startsWith
    
    Calculating life expectancy at birth ...
    ... done
    Calculating net number of migrants ...
    ... done
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'burkina-faso-females.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# popsom

Version: 4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PortfolioEffectHFT

Version: 1.8

## In both

*   checking whether package ‘PortfolioEffectHFT’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/PortfolioEffectHFT/new/PortfolioEffectHFT.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘PortfolioEffectHFT’ ...
** package ‘PortfolioEffectHFT’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘PortfolioEffectHFT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/PortfolioEffectHFT/new/PortfolioEffectHFT.Rcheck/PortfolioEffectHFT’

```
### CRAN

```
* installing *source* package ‘PortfolioEffectHFT’ ...
** package ‘PortfolioEffectHFT’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/PortfolioEffectHFT/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘PortfolioEffectHFT’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/PortfolioEffectHFT/old/PortfolioEffectHFT.Rcheck/PortfolioEffectHFT’

```
# powerbydesign

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PowerExplorer

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# powerlmm

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ppcSpatial

Version: 0.2.0

## In both

*   checking whether package ‘ppcSpatial’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/new/ppcSpatial.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ppcSpatial’ ...
** package ‘ppcSpatial’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ppcSpatial’
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘ppcSpatial’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/new/ppcSpatial.Rcheck/ppcSpatial’

```
### CRAN

```
* installing *source* package ‘ppcSpatial’ ...
** package ‘ppcSpatial’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘ppcSpatial’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ppcSpatial/old/ppcSpatial.Rcheck/ppcSpatial’

```
# PPforest

Version: 0.1.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        combine
    
    Loading required package: PPtreeViz
    Loading required package: ggplot2
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:dplyr':
    
        vars
    
    Loading required package: partykit
    Loading required package: grid
    Loading required package: libcoin
    Loading required package: mvtnorm
    Loading required package: rpart
    Quitting from lines 118-129 (PPforest-vignette.Rmd) 
    Error: processing vignette 'PPforest-vignette.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~FL", "~Type".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~FL", "~Type".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PPInfer

Version: 1.6.0

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘biomaRt’ ‘fgsea’ ‘kernlab’ ‘ggplot2’ ‘igraph’ ‘STRINGdb’
      ‘yeastExpData’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ORA: no visible global function definition for ‘setTxtProgressBar’
    ORA: no visible global function definition for ‘p.adjust’
    ORA.dotplot: no visible global function definition for ‘p.adjust’
    enrich.net: no visible global function definition for ‘stack’
    enrich.net: no visible global function definition for ‘adjustcolor’
    enrich.net : <anonymous>: no visible global function definition for
      ‘adjustcolor’
    enrich.net: no visible binding for global variable ‘legend’
    net.infer: no visible global function definition for ‘na.omit’
    net.infer.ST: no visible global function definition for ‘na.omit’
    ppi.infer.human: no visible global function definition for ‘na.omit’
    ppi.infer.mouse: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      adjustcolor fisher.test legend na.omit p.adjust setTxtProgressBar
      stack txtProgressBar
    Consider adding
      importFrom("grDevices", "adjustcolor")
      importFrom("graphics", "legend")
      importFrom("stats", "fisher.test", "na.omit", "p.adjust")
      importFrom("utils", "setTxtProgressBar", "stack", "txtProgressBar")
    to your NAMESPACE file.
    ```

# PPtreeViz

Version: 2.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# pqsfinder

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking if this is a source package ... NOTE
    ```
    Found the following apparent object files/libraries:
      src/boost_regex/c_regex_traits.o src/boost_regex/cpp_regex_traits.o
      src/boost_regex/cregex.o src/boost_regex/fileiter.o
      src/boost_regex/icu.o src/boost_regex/instances.o
      src/boost_regex/posix_api.o src/boost_regex/regex.o
      src/boost_regex/regex_debug.o src/boost_regex/regex_raw_buffer.o
      src/boost_regex/regex_traits_defaults.o
      src/boost_regex/static_mutex.o src/boost_regex/usinstances.o
      src/boost_regex/w32_regex_traits.o src/boost_regex/wc_regex_traits.o
      src/boost_regex/wide_posix_api.o src/boost_regex/winstances.o
    Object files/libraries should not be included in a source package.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.8Mb
      sub-directories of 1Mb or more:
        lib  11.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘BH’ is only usable in R >= 3.0.2
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# prcbench

Version: 0.7.3

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘prcbench-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: ToolAUCCalculator
    > ### Title: R6 class of the AUCCalculator tool
    > ### Aliases: ToolAUCCalculator
    > ### Keywords: datasets
    > 
    > ### ** Examples
    > 
    > ## Initialization
    > toolauccalc <- ToolAUCCalculator$new()
    Error: .onLoad failed in loadNamespace() for 'rJava', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so
      Reason: image not found
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 404 SKIPPED: 0 FAILED: 42
      1. Error: run_benchmark (@test_main_benchmark.R#32) 
      2. Error: run_benchmark: testset (@test_main_benchmark.R#58) 
      3. Error: run_benchmark: toolset (@test_main_benchmark.R#84) 
      4. Error: run_benchmark: times (@test_main_benchmark.R#131) 
      5. Error: run_benchmark: unit (@test_main_benchmark.R#166) 
      6. Error: run_evalcurve (@test_main_evalcurves.R#6) 
      7. Error: run_evalcurve: testset (@test_main_evalcurves.R#19) 
      8. Error: run_evalcurve: toolset (@test_main_evalcurves.R#32) 
      9. Error: run_evalcurve testscores (@test_main_evalcurves.R#57) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 65-70 (introduction.Rmd) 
    Error: processing vignette 'introduction.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rJava', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/prcbench/rJava/libs/rJava.so
      Reason: image not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# prcr

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# precintcon

Version: 2.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# precrec

Version: 0.9.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# predatory

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# predictionInterval

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# predictmeans

Version: 0.99

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘predictmeans-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: covariatemeans
    > ### Title: Predicted Means of a Linear Model with Covariate Variable(s)
    > ### Aliases: covariatemeans
    > 
    > ### ** Examples
    > 
    >   library(predictmeans)
    >   data(Oats, package="nlme")
    >   fm <- lme(yield ~ nitro*Variety, random=~1|Block/Variety, data=Oats)
    > # library(lme4)
    > # fm <- lmer(yield ~ nitro*Variety+(1|Block/Variety), data=Oats)
    >   covariatemeans(fm, "Variety", covariate="nitro")
    dev.new(): using pdf(file="Rplots2.pdf")
    Error: Column `y` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    terms.gls: no visible global function definition for ‘model.frame’
    Undefined global functions or variables:
      abline anova as.formula axis box coef delete.response dev.new edit
      family fitted formula ftable identify image legend lines lm.influence
      loess.smooth model.frame model.matrix na.omit na.pass p.adjust par
      plot points ppoints predict pt ptukey qnorm qqline qqnorm qt resid
      residuals slot terms text title update vcov xtabs
    Consider adding
      importFrom("grDevices", "dev.new")
      importFrom("graphics", "abline", "axis", "box", "identify", "image",
                 "legend", "lines", "par", "plot", "points", "text", "title")
      importFrom("methods", "slot")
      importFrom("stats", "anova", "as.formula", "coef", "delete.response",
                 "family", "fitted", "formula", "ftable", "lm.influence",
                 "loess.smooth", "model.frame", "model.matrix", "na.omit",
                 "na.pass", "p.adjust", "ppoints", "predict", "pt", "ptukey",
                 "qnorm", "qqline", "qqnorm", "qt", "resid", "residuals",
                 "terms", "update", "vcov", "xtabs")
      importFrom("utils", "edit")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# PredPsych

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# preference

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PReMiuM

Version: 3.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# prepplot

Version: 0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# preprocomb

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# preprosim

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘e1071’
      All declared Imports should be used.
    ```

# preproviz

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# preText

Version: 0.6.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# prevR

Version: 3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sparr’
    ```

# pRF

Version: 1.2

## Newly broken

*   checking whether package ‘pRF’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘pRF’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/pRF/new/pRF.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pricesensitivitymeter

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# primerTree

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# prism

Version: 0.0.7

## Newly broken

*   checking whether package ‘prism’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘prism’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/prism/new/prism.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PRISMA

Version: 0.2-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: Matrix
    Loading required package: gplots
    
    Attaching package: ‘gplots’
    
    The following object is masked from ‘package:stats’:
    
        lowess
    
    Loading required package: ggplot2
       [[ suppressing 24 column names ‘line56’, ‘line32’, ‘line6’ ... ]]
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PRISMA.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# Prize

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: ggplot2
    Loading required package: stringr
    0 Prioritization_of_DE_genes is processed.
    1 Tumor_expression is processed.
    2 Normal_expression is processed.
    3 Frequency is processed.
    4 Epitopes is processed.
    4.1 Number_of_epitopes is processed.
    4.2 Size_of_epitopes is processed.
    Loading required package: reshape2
    0 Prioritization_of_DE_genes is processed.
    1 Tumor_expression is processed.
    2 Normal_expression is processed.
    3 Frequency is processed.
    4 Epitopes is processed.
    4.1 Number_of_epitopes is processed.
    4.2 Size_of_epitopes is processed.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Prize.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# pROC

Version: 1.12.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# processmapR

Version: 0.3.0

## Newly broken

*   checking whether package ‘processmapR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘processmapR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/processmapR/new/processmapR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# processmonitR

Version: 0.1.0

## Newly broken

*   checking whether package ‘processmonitR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘processmonitR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/processmonitR/new/processmonitR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# productplots

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# profileR

Version: 0.3-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# profr

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘ggplot2’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    ggplot.profr: no visible binding for global variable ‘start’
    ggplot.profr: no visible binding for global variable ‘end’
    ggplot.profr: no visible global function definition for ‘geom_text’
    ggplot.profr: no visible binding for global variable ‘time’
    ggplot.profr: no visible global function definition for
      ‘scale_y_continuous’
    ggplot.profr: no visible global function definition for
      ‘scale_x_continuous’
    plot.profr: no visible global function definition for ‘plot’
    plot.profr: no visible global function definition for ‘rect’
    plot.profr: no visible binding for global variable ‘time’
    plot.profr: no visible global function definition for ‘text’
    profr: no visible global function definition for ‘Rprof’
    Undefined global functions or variables:
      Rprof aes end geom_rect geom_text ggplot hist plot rect
      scale_x_continuous scale_y_continuous start text time
    Consider adding
      importFrom("graphics", "hist", "plot", "rect", "text")
      importFrom("stats", "end", "start", "time")
      importFrom("utils", "Rprof")
    to your NAMESPACE file.
    ```

# profvis

Version: 0.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ProgGUIinR

Version: 0.0-4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘qtbase’ ‘RGtk2’ ‘cairoDevice’ ‘gWidgetsRGtk2’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘apropos’
    showGtkWidgetInfo : getMethodForObject: no visible global function
      definition for ‘help’
    showGtkWidgetInfo: no visible global function definition for
      ‘gSignalConnect’
    Undefined global functions or variables:
      PangoStyle addSpring apropos gSignalConnect gbutton gframe ggroup
      gnotebook gpanedgroup gstatusbar gtable gtext gtkAction
      gtkCellRendererText gtkEntry gtkEntryCompletionNew gtkFrame gtkHBox
      gtkHBoxNew gtkHPaned gtkLabel gtkListStore gtkNotebook
      gtkScrolledWindow gtkSeparatorToolItem gtkStatusbar gtkTextBuffer
      gtkTextView gtkToolButton gtkToolbar gtkTreeView gtkTreeViewColumn
      gtkTypeGetSignals gtkVBox gtkVBoxNew gtkWindow gwindow help
      rGtkDataFrame svalue svalue<- tcl tclvalue tkbind tkconfigure
      tkdelete tkget tkgrid tkgrid.columnconfigure tkgrid.rowconfigure
      tkinsert tkmenu tkpack tksee tkset tktag.add tktag.configure
      tktag.remove tktext tktoplevel tkwm.title tkxview ttkbutton ttkframe
      ttklabel ttkpanedwindow ttkscrollbar ttktreeview visible<-
    Consider adding
      importFrom("utils", "apropos", "help")
    to your NAMESPACE file.
    ```

# ProjectionBasedClustering

Version: 1.0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# projections

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘distcrete’ ‘incidence’
      All declared Imports should be used.
    ```

# ProjectTemplate

Version: 0.8.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# projmanr

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# projpred

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# pRoloc

Version: 1.20.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Class '"ClustDist"'
    > ### Aliases: ClustDist class:ClustDist ClustDist-class
    > ###   plot,ClustDist,MSnSet-method show,ClustDist-method
    > ### Keywords: classes
    > 
    > ### ** Examples
    > 
    >   showClass("ClustDist")
    Class "ClustDist" [package "pRoloc"]
    
    Slots:
                                                                            
    Name:           k       dist       term         id       nrow    clustsz
    Class:    numeric       list  character  character    numeric       list
                                
    Name:  components       fcol
    Class:     vector  character
    >   
    >   library('pRolocdata')
    Error in library("pRolocdata") : there is no package called ‘pRolocdata’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
      The following object is masked from 'package:mzR':
      
          nChrom
      
      Loading required package: cluster
      
      This is pRoloc version 1.20.0 
        Visit https://lgatto.github.io/pRoloc/ to get started.
      
      Warning message:
      replacing previous import 'BiocGenerics::var' by 'stats::var' when loading 'MLInterfaces' 
      > library("pRolocdata")
      Error in library("pRolocdata") : there is no package called 'pRolocdata'
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following object is masked from 'package:tools':
    
        toHTML
    
    
    Attaching package: 'annotate'
    
    The following object is masked from 'package:mzR':
    
        nChrom
    
    Loading required package: cluster
    Warning: replacing previous import 'BiocGenerics::var' by 'stats::var' when loading 'MLInterfaces'
    
    This is pRoloc version 1.20.0 
      Visit https://lgatto.github.io/pRoloc/ to get started.
    
    Quitting from lines 87-93 (pRoloc-goannotations.Rmd) 
    Error: processing vignette 'pRoloc-goannotations.Rmd' failed with diagnostics:
    there is no package called 'pRolocdata'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pRolocdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.9Mb
      sub-directories of 1Mb or more:
        doc  10.6Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘MLInterfaces:::.macroF1’ ‘MLInterfaces:::.precision’
      ‘MLInterfaces:::.recall’ ‘MLInterfaces:::es2df’
      ‘caret:::predict.plsda’
      See the note in ?`:::` about the use of this operator.
    There are ::: calls to the package's namespace in its code. A package
      almost never needs to use ::: for its own objects:
      ‘opt’
    ```

*   checking R code for possible problems ... NOTE
    ```
    Found the following possibly unsafe calls:
    File ‘pRoloc/R/annotation.R’:
      unlockBinding("params", .pRolocEnv)
    ```

# pRolocGUI

Version: 1.14.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘pRolocGUI-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: pRolocVis
    > ### Title: Interactive visualisation of spatial proteomics data
    > ### Aliases: pRolocVis pRolocVis_aggregate pRolocVis_classify
    > ###   pRolocVis_compare pRolocVis_pca
    > 
    > ### ** Examples
    > 
    > library("pRoloc")
    > library("pRolocdata")
    Error in library("pRolocdata") : there is no package called ‘pRolocdata’
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    
    Attaching package: 'annotate'
    
    The following object is masked from 'package:mzR':
    
        nChrom
    
    Loading required package: cluster
    Warning: replacing previous import 'BiocGenerics::var' by 'stats::var' when loading 'MLInterfaces'
    
    This is pRoloc version 1.20.0 
      Visit https://lgatto.github.io/pRoloc/ to get started.
    
    
    This is pRolocGUI version 1.14.0
    
    Quitting from lines 77-79 (pRolocGUI.Rmd) 
    Error: processing vignette 'pRolocGUI.Rmd' failed with diagnostics:
    there is no package called 'pRolocdata'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘pRolocdata’
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Authors@R field gives more than one person with maintainer role:
      Lisa Breckels <lms79@cam.ac.uk> [aut, cre]
      Laurent Gatto <lg390@cam.ac.uk> [aut, cre]
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘pRoloc:::remap’
      See the note in ?`:::` about the use of this operator.
    ```

# prophet

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# proportion

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘binom’, ‘PropCIs’, ‘BlakerCI’, ‘prevalence’
    ```

# propr

Version: 3.5.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Alert: This function ignores index.
    Alert: This function ignores index.
    Alert: This function ignores index.
    Green: Pair positively proportional across all samples.
    Quitting from lines 217-229 (c_indexing.Rmd) 
    Error: processing vignette 'c_indexing.Rmd' failed with diagnostics:
    <text>:1:2: unexpected ';'
    1: ~;
         ^
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# proteomics

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    addIonSatistics: no visible binding for global variable ‘median’
    adjustOne: no visible binding for global variable ‘median’
    attachModifications: no visible global function definition for
      ‘na.omit’
    avrgLoadingCalculation: no visible binding for global variable ‘median’
    channelResponses : estimate: no visible binding for global variable
      ‘var’
    responseStatisics : func: no visible global function definition for
      ‘aggregate’
    testing: no visible global function definition for ‘aov’
    testingOneshot: no visible global function definition for ‘anova’
    testingTukey: no visible global function definition for ‘TukeyHSD’
    Undefined global functions or variables:
      TukeyHSD aggregate anova aov median na.omit var
    Consider adding
      importFrom("stats", "TukeyHSD", "aggregate", "anova", "aov", "median",
                 "na.omit", "var")
    to your NAMESPACE file.
    ```

# proteoQC

Version: 1.16.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'proteoQC.Rmd' failed with diagnostics:
    there is no package called ‘prettydoc’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.4Mb
      sub-directories of 1Mb or more:
        doc       2.5Mb
        extdata   3.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotMS2boxplot: no visible binding for global variable ‘techRep’
    plotMS2boxplot: no visible binding for global variable ‘fraction’
    plotMS2boxplot: no visible binding for global variable ‘MS2QC’
    plotSampleIDResultErrorBar: no visible binding for global variable
      ‘fraction’
    plotSampleIDResultErrorBar: no visible binding for global variable
      ‘val’
    plotSampleIDResultErrorBar: no visible binding for global variable ‘se’
    plotSampleVenn: no visible global function definition for ‘grid.draw’
    plotTechRepVenn : <anonymous>: no visible global function definition
      for ‘grid.draw’
    qcHist: no visible binding for global variable ‘error’
    qcHist: no visible binding for global variable ‘techRep’
    qcHist: no visible binding for global variable ‘bioRep’
    qcHist2: no visible binding for global variable ‘error’
    qcHist2: no visible binding for global variable ‘fractile’
    Undefined global functions or variables:
      ..count.. Intensity MS1QC MS2QC TMT10 TMT6 Tag V1 V2 V3 V4 V5 bioRep
      curenv delta error exprs fractile fraction grid.draw iTRAQ4 iTRAQ8
      label peplength peptide_summary precursorCharge quantify ratio
      readMgfData se techRep val x y
    ```

# PRROC

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PRROC.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# PSCBS

Version: 0.63.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        int [1:3] 6 7 9
       Left indices of neighboring segments:
        int 6
      Cluster #4 of 5...done
      Cluster #5 of 5...
       Segments in cluster:
        int 8
       Left indices of neighboring segments:
        int(0) 
      Cluster #5 of 5...done
      Left indices of segments to be merged:
       int 6
     Merging neighboring segments within each cluster...done
     Merging segments...
     Merging segments...done
     Updating segment means...
     Updating segment means...done
    Prune segments by hierarchical clustering...done
    Error: processing vignette 'CBS.tex.rsp' failed with diagnostics:
    Running 'texi2dvi' on 'CBS.tex' failed.
    Execution halted
    ```

# pscore

Version: 0.1-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    mahalanobisComposite: no visible global function definition for
      ‘princomp’
    prepareComposite: no visible global function definition for ‘cov’
    winsorizor : f: no visible global function definition for ‘quantile’
    Undefined global functions or variables:
      cov princomp quantile
    Consider adding
      importFrom("stats", "cov", "princomp", "quantile")
    to your NAMESPACE file.
    ```

# psd

Version: 1.0-1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/Interpolate results" -> "figure/Interpolate_results"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/Summarize regression statistics" -> "figure/Summarize_regression_statistics"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/AR spectrum" -> "figure/AR_spectrum"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/Compute spectral properties" -> "figure/Compute_spectral_properties"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/Get adaptive history" -> "figure/Get_adaptive_history"
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      replaced special characters in figure filename "figure/and manipulate it a bit" -> "figure/and_manipulate_it_a_bit"
    /Library/Frameworks/R.framework/Resources/bin/Rscript -e "if (getRversion() < '3.0.0') knitr::knit2pdf('fftw.Rnw') else tools::texi2pdf('fftw.tex')"
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fftw.tex' failed.
    Calls: <Anonymous> -> texi2dvi
    Execution halted
    make: *** [fftw.pdf] Error 1
    Error in buildVignettes(dir = "/Users/max/github/ggplot2/revdep/checks.noindex/psd/new/psd.Rcheck/vign_test/psd") : 
      running 'make' failed
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    psdcore.default: no visible global function definition for ‘ts’
    pspectrum.ts: no visible global function definition for ‘frequency’
    spec_confint.default: no visible global function definition for
      ‘pchisq’
    spec_confint.default: no visible global function definition for
      ‘qchisq’
    splineGrad.default: no visible global function definition for ‘par’
    splineGrad.default: no visible global function definition for ‘plot’
    splineGrad.default: no visible global function definition for ‘lines’
    Undefined global functions or variables:
      abline acf as.ts frequency head is.ts layout legend lines lm
      loess.control mtext par pchisq plot qchisq residuals spec.pgram start
      tail title ts var
    Consider adding
      importFrom("graphics", "abline", "layout", "legend", "lines", "mtext",
                 "par", "plot", "title")
      importFrom("stats", "acf", "as.ts", "frequency", "is.ts", "lm",
                 "loess.control", "pchisq", "qchisq", "residuals",
                 "spec.pgram", "start", "ts", "var")
      importFrom("utils", "head", "tail")
    to your NAMESPACE file.
    ```

# psda

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘raster’
      All declared Imports should be used.
    ```

# pseval

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# psichomics

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.5Mb
      sub-directories of 1Mb or more:
        doc   5.6Mb
    ```

*   checking compiled code ... NOTE
    ```
    File ‘psichomics/libs/psichomics.so’:
      Found ‘___stdoutp’, possibly from ‘stdout’ (C)
        Object: ‘psiFastCalc.o’
      Found ‘_printf’, possibly from ‘printf’ (C)
        Object: ‘psiFastCalc.o’
      Found ‘_putchar’, possibly from ‘putchar’ (C)
        Object: ‘psiFastCalc.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# PSLM2015

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   5.2Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 26 marked Latin-1 strings
    ```

# PSTR

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# psychmeta

Version: 1.0.2

## Newly broken

*   checking whether package ‘psychmeta’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘psychmeta’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/psychmeta/new/psychmeta.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# psygenet2r

Version: 1.12.0

## In both

*   checking whether package ‘psygenet2r’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'psygenetGene(x, databse = "ALL")': unused argument (databse = "ALL") 
      Note: possible error in 'psygenetGene(x, databse = "ALL", ': unused argument (databse = "ALL") 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/psygenet2r/new/psygenet2r.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'case_study.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.7Mb
      sub-directories of 1Mb or more:
        doc   5.2Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘GO.db’
      All declared Imports should be used.
    package 'methods' is used but not declared
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘Disease1’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘Disease2’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘JaccardIndex’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘value’
    plot,JaccardIndexPsy-ANY: no visible binding for global variable
      ‘variable’
    Undefined global functions or variables:
      Category Disease1 Disease2 JaccardIndex Var1 c0.Number_of_Abstracts
      c0.Score c1.Gene_Symbol c2.DiseaseName c2.PsychiatricDisorder
      category combn database diseases gene new perc phyper pie read.csv
      read.delim universe value variable
    Consider adding
      importFrom("graphics", "pie")
      importFrom("methods", "new")
      importFrom("stats", "phyper")
      importFrom("utils", "combn", "read.csv", "read.delim")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# psyosphere

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# PureCN

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        type
    
    The following object is masked from 'package:base':
    
        strsplit
    
    
    Attaching package: 'VariantAnnotation'
    
    The following object is masked from 'package:base':
    
        tabulate
    
    Warning: Removed 1 rows containing missing values (geom_path).
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'PureCN.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.3Mb
      sub-directories of 1Mb or more:
        data      1.1Mb
        doc       2.6Mb
        extdata   3.0Mb
    ```

# pwr

Version: 1.2-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# QCAtools

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# qcmetrics

Version: 1.18.0

## In both

*   checking examples ... ERROR
    ```
    ...
    $lab
    [1] "Big lab"
    
    $uni
    [1] "Cabin University"
    
    > show(qcm)
    Object of class "QcMetrics"
     containing 2 QC metrics.
     and 3 metadata variables.
    > 
    > destdir <- tempdir()
    > (report <- file.path(destdir, "testQCReport"))
    [1] "/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpOguSrE/testQCReport"
    > 
    > ## pdf report
    > qcReport(qcm, reportname = report)
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on '/var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpOguSrE/testQCReport.tex' failed.
    Calls: qcReport ... qcReport -> .local -> reporting_pdf -> <Anonymous> -> texi2dvi
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 51-72 (qcmetrics.Rnw) 
    Error: processing vignette 'qcmetrics.Rnw' failed with diagnostics:
    there is no package called 'RforProteomics'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RforProteomics’
    ```

*   checking R code for possible problems ... NOTE
    ```
    n15qc: no visible global function definition for ‘fvarLabels’
    n15qc: no visible global function definition for ‘fData’
    n15qc : <anonymous>: no visible global function definition for ‘ggplot’
    n15qc : <anonymous>: no visible global function definition for ‘aes’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_jitter’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_boxplot’
    n15qc : <anonymous>: no visible global function definition for
      ‘geom_hline’
    n15qc : <anonymous>: no visible global function definition for ‘labs’
    n15qc: no visible global function definition for ‘fData<-’
    n15qc: no visible global function definition for ‘combineFeatures’
    n15qc: no visible global function definition for ‘exprs’
    n15qc: no visible global function definition for ‘fileNames’
    n15qc: no visible global function definition for ‘experimentData’
    Undefined global functions or variables:
      aes combineFeatures experimentData exprs fData fData<- fileNames
      fvarLabels geom_boxplot geom_hline geom_jitter ggplot labs
    ```

# qcQpcr

Version: 1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# QCSimulator

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# qdap

Version: 2.2.9

## In both

*   checking whether package ‘qdap’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/qdap/new/qdap.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘qdap’ ...
** package ‘qdap’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘qdap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/qdap/new/qdap.Rcheck/qdap’

```
### CRAN

```
* installing *source* package ‘qdap’ ...
** package ‘qdap’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/qdap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘qdap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/qdap/old/qdap.Rcheck/qdap’

```
# qgraph

Version: 1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# qicharts

Version: 0.5.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# qicharts2

Version: 0.4.0

## Newly broken

*   checking whether package ‘qicharts2’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘qicharts2’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/qicharts2/new/qicharts2.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 372-379 (qicharts2.Rmd) 
    Error: processing vignette 'qicharts2.Rmd' failed with diagnostics:
    missing value where TRUE/FALSE needed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# qqplotr

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘knitr’
      All declared Imports should be used.
    ```

# qte

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# quadrupen

Version: 0.2-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# QualInt

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    powerval: no visible global function definition for ‘qnorm’
    powerval: no visible global function definition for ‘pnorm’
    print.expplot: no visible global function definition for ‘gray’
    print.expplot: no visible binding for global variable ‘coef’
    print.outplot: no visible global function definition for ‘gray’
    print.outplot: no visible binding for global variable ‘coef’
    qualint: no visible global function definition for ‘qnorm’
    qualint: no visible global function definition for ‘plot’
    qualval: no visible global function definition for ‘qnorm’
    qualval: no visible global function definition for ‘plot’
    surout: no visible global function definition for ‘tail’
    surout: no visible global function definition for ‘coef’
    Undefined global functions or variables:
      coef dbinom gray pchisq plot pnorm qnorm rnorm sd tail
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "plot")
      importFrom("stats", "coef", "dbinom", "pchisq", "pnorm", "qnorm",
                 "rnorm", "sd")
      importFrom("utils", "tail")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: survival
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'qualint.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# qualvar

Version: 0.2.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:GGally':
    
        nasa
    
    The following object is masked from 'package:ggplot2':
    
        vars
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Quitting from lines 111-120 (wilcox1973.Rmd) 
    Error: processing vignette 'wilcox1973.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~MDA".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# quanteda

Version: 1.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘quanteda-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: textplot_keyness
    > ### Title: Plot word keyness
    > ### Aliases: textplot_keyness
    > ### Keywords: textplot
    > 
    > ### ** Examples
    > 
    > # compare Trump speeches to other Presidents by chi^2
    > dem_dfm <- data_corpus_inaugural %>%
    +      corpus_subset(Year > 1980) %>%
    +      dfm(groups = "President", remove = stopwords("english"), remove_punct = TRUE)
    > dem_key <- textstat_keyness(dem_dfm, target = "Trump")
    > textplot_keyness(dem_key, margin = 0.2, n = 10)
    Error in if (label.size == 0) NA else row$colour : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... labelGrob -> gTree -> gpar -> validGP -> numnotnull
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      as.character(unique(ggplot2::ggplot_build(p)$layout$panel_layout$keyword)) not equal to c("people", "american").
      Lengths differ: 0 is not 2
      
      
        |                                                                            
        |                                                                      |   0%
        |                                                                            
        |======================================================================| 100%
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 2132 SKIPPED: 21 FAILED: 2
      1. Failure: test plot.kwic facet order parameter (@test-textplot.R#73) 
      2. Failure: test plot.kwic keeps order of keywords passed (@test-textplot.R#83) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 71 marked UTF-8 strings
    ```

# QuantNorm

Version: 1.0.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > #QuantNorm correction
    > corrected.distance.matrix <- QuantNorm(ENCODE,batches,method='row/column', cor_method='pearson',
    +                                        logdat=FALSE,standardize = TRUE, tol=1e-4)
    It may take a long time to standardize the data. 
    Algorithm converged after 9 iterations. 
    > 
    > #Constructing connection matrix
    > 
    > mat <- connection.matrix(mat=corrected.distance.matrix,label=colnames(corrected.distance.matrix))
    > 
    > #Creating network object and plot
    > ENCODE.net=network(mat, directed=FALSE)
    > ENCODE.net %v% "Species" <- c(rep('Human',13),rep('Mouse',13))
    > p0 <- ggnet2(ENCODE.net,label=TRUE,color = 'Species', palette = "Set2",
    +              size = 3, vjust = -0.6,mode = "kamadakawai",label.size = 3,
    +              color.legend = 'Species')+theme(legend.position = 'bottom')
    Loading required package: scales
    > plot(p0)
    Error: Columns `x`, `y`, `xend`, `yend` must be 1d atomic vectors or lists
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# quantro

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Attaching package: 'Biostrings'
    
    The following object is masked from 'package:DelayedArray':
    
        type
    
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: bumphunter
    Loading required package: foreach
    Loading required package: iterators
    Loading required package: locfit
    locfit 1.5-9.1 	 2013-03-22
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'quantro-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# QuantumClone

Version: 1.0.0.6

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > cat("Cluster data: will try to cluster between 3 and 4 clones, with 1 maximum search each time,
    +       and will use priors from preclustering (e.g. k-medoids on A and AB sites)")
    Cluster data: will try to cluster between 3 and 4 clones, with 1 maximum search each time,
          and will use priors from preclustering (e.g. k-medoids on A and AB sites)> print("The genotype is provided in the list frame, and
    +           there is no associated data from FREEC to get genotype from.")
    [1] "The genotype is provided in the list frame, and\n          there is no associated data from FREEC to get genotype from."
    > print("The computation will run on a single CPU.")
    [1] "The computation will run on a single CPU."
    > Clustering_output<-One_step_clustering(SNV_list = Mutations,
    + FREEC_list = NULL,contamination = c(0,0),nclone_range = 3:4,
    + clone_priors = NULL,prior_weight = NULL ,
    + Initializations = 1,preclustering = "FLASH", simulated = TRUE,
    + save_plot = TRUE,ncores=1,output_directory="Example")
    FREEC_list is empty. Checking that there is a genotype column in all samples...
    Genotype is provided.
    epsilon set to: 0.00678978815860945
    Checking all possibilities for Example_1
    Error in geom_point() : could not find function "geom_point"
    Calls: One_step_clustering ... plot_cell_from_Return_out -> <Anonymous> -> do.call
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      Iteration: 192 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 193 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 194 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 195 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 196 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 197 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 198 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 199 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      Iteration: 200 bestvalit: 142.476167 bestmemit:    0.995714    0.260384
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 21 SKIPPED: 0 FAILED: 1
      1. Error: plot_with_margins_densities returns a dridExtra object (@test-plots.R#15) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    [WARNING] This document format requires a nonempty <title> element.
      Please specify either 'title' or 'pagetitle' in the metadata.
      Falling back to 'Release.utf8'
    Quitting from lines 134-135 (Use_case.Rmd) 
    Error: processing vignette 'Use_case.Rmd' failed with diagnostics:
    could not find function "geom_point"
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# queuecomputer

Version: 0.8.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# quickPlot

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘fastshp’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    ```

# quickpsy

Version: 0.1.4

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘quickpsy-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotcurves
    > ### Title: Plot the curves
    > ### Aliases: plotcurves
    > 
    > ### ** Examples
    > 
    > library(MPDiR) # contains the Vernier data
    > fit <- quickpsy(Vernier, Phaseshift, NumUpward, N,
    +                 grouping = .(Direction, WaveForm, TempFreq), B = 5)
    > plotcurves(fit)
    Warning: Ignoring unknown aesthetics: x
    Error: Aesthetics must be either length 1 or the same as the data (8): ymin, ymax, colour, x
    Execution halted
    ```

*   checking whether package ‘quickpsy’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘quickpsy’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/quickpsy/new/quickpsy.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# quickReg

Version: 1.5.0

## Newly broken

*   checking whether package ‘quickReg’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘quickReg’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/quickReg/new/quickReg.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘psych’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PredictABEL’
    ```

# quokar

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MCMCpack’ ‘gridExtra’ ‘knitr’
      All declared Imports should be used.
    ```

# qvalue

Version: 2.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    summary.qvalue          Display q-value object
    write.qvalue            Write results to file
    
    Further information is available in the
    following vignettes in directory
    '/Users/max/github/ggplot2/revdep/checks.noindex/qvalue/new/qvalue.Rcheck/qvalue/doc':
    
    qvalue: qvalue Package (source, pdf)
    
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Warning: 'mode(width)' differs between new and previous
    	 ==> NOT changing 'width'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'qvalue.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    lfdr: no visible global function definition for ‘qnorm’
    lfdr: no visible global function definition for ‘density’
    lfdr: no visible global function definition for ‘smooth.spline’
    lfdr: no visible global function definition for ‘predict’
    lfdr: no visible global function definition for ‘dnorm’
    pi0est: no visible global function definition for ‘smooth.spline’
    pi0est: no visible global function definition for ‘predict’
    pi0est: no visible global function definition for ‘quantile’
    plot.qvalue: no visible global function definition for ‘quantile’
    write.qvalue: no visible global function definition for ‘write.table’
    Undefined global functions or variables:
      density dnorm predict qnorm quantile smooth.spline write.table
    Consider adding
      importFrom("stats", "density", "dnorm", "predict", "qnorm", "quantile",
                 "smooth.spline")
      importFrom("utils", "write.table")
    to your NAMESPACE file.
    ```

# qwraps2

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# R2admb

Version: 0.7.16

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: stats4
    
    Attaching package: 'bbmle'
    
    The following object is masked from 'package:R2admb':
    
        stdEr
    
    Warning: Removed 36 rows containing missing values (geom_path).
    Warning: Removed 109 rows containing missing values (geom_point).
    Loading required package: Matrix
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'R2admb.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# r2glmm

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘data.table’ ‘dplyr’ ‘lmerTest’
      All declared Imports should be used.
    ```

# R3CPET

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Loading required package: Rcpp
    
    Attaching package: 'R3CPET'
    
    The following object is masked from 'package:igraph':
    
        sizes
    
    304 interacting DNA regions loaded
    a total of  81414 binding sites for 6 TF  were loaded
    loading HPRD network, with 9616 nodes
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    plotting networks
    plots will be also available on file AllGraphs.pdf
    Warning in if (!lbs %in% c(colnames(mold(obj[1, ])), "start", "end", "seqnames",  :
      the condition has length > 1 and only the first element will be used
    Quitting from lines 424-425 (R3CPET.Rnw) 
    Error: processing vignette 'R3CPET.Rnw' failed with diagnostics:
    label must be one of column names
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.0Mb
      sub-directories of 1Mb or more:
        data      3.0Mb
        example   1.0Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘methods’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: 'BiocGenerics'
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .formatDAVIDResult: no visible global function definition for
      'formatList'
    .formatDAVIDResult: no visible global function definition for
      'formatGene2Gene'
    .formatDAVIDResult: no visible global function definition for
      'formatAnnotationReport'
    .get.NetworksGenes: no visible global function definition for
      'annotatePeakInBatch'
    EnsemblToHGNC: no visible global function definition for 'useMart'
    EnsemblToHGNC: no visible global function definition for 'useDataset'
    EnsemblToHGNC: no visible global function definition for 'getBM'
    EntrezToHGNC: no visible global function definition for 'useMart'
    EntrezToHGNC: no visible global function definition for 'useDataset'
    EntrezToHGNC: no visible global function definition for 'getBM'
    createServer,ChiapetExperimentData-NetworkCollection-ChromMaintainers:
      no visible global function definition for 'runApp'
    Undefined global functions or variables:
      TxDb.Hsapiens.UCSC.hg19.knownGene annotatePeakInBatch
      formatAnnotationReport formatGene2Gene formatGeneReport
      formatGeneReportFull formatList getBM org.Hs.eg.db org.Hs.egUCSCKG
      runApp select toTable useDataset useMart
    ```

# R3port

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# R6

Version: 2.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# radiant.basics

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# radiant.data

Version: 0.8.1

## Newly broken

*   checking whether package ‘radiant.data’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘radiant.data’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/radiant.data/new/radiant.data.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# radiant.model

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2 marked UTF-8 strings
    ```

# radiant.multivariate

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# rags2ridges

Version: 2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ragtop

Version: 0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# raincpc

Version: 0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    cpc_get_rawdata: no visible global function definition for
      ‘download.file’
    Undefined global functions or variables:
      download.file
    Consider adding
      importFrom("utils", "download.file")
    to your NAMESPACE file.
    ```

# RAM

Version: 1.2.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# randomForestExplainer

Version: 0.9

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘randomForestExplainer-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_importance_ggpairs
    > ### Title: Plot importance measures with ggpairs
    > ### Aliases: plot_importance_ggpairs
    > 
    > ### ** Examples
    > 
    > forest <- randomForest::randomForest(Species ~ ., data = iris, localImp = TRUE, ntree = 200)
    > frame <- measure_importance(forest, measures = c("mean_min_depth", "times_a_root"))
    > plot_importance_ggpairs(frame, measures = c("mean_min_depth", "times_a_root"))
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~mean_min_depth".  Please rename the columns or make a new column.
    Calls: plot_importance_ggpairs ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking whether package ‘randomForestExplainer’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘randomForestExplainer’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/randomForestExplainer/new/randomForestExplainer.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    randomForest 4.6-14
    Type rfNews() to see new features/changes/bug fixes.
    Warning: replacing previous import 'dplyr::vars' by 'ggplot2::vars' when loading 'randomForestExplainer'
    Warning: Using alpha for a discrete variable is not advised.
    Quitting from lines 149-151 (randomForestExplainer.Rmd) 
    Error: processing vignette 'randomForestExplainer.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~mean_min_depth".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘dtplyr’
      All declared Imports should be used.
    ```

# randomizeR

Version: 1.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Warning in rq.fit.br(wx, wy, tau = tau, ...) :
      Solution may be nonunique
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'article.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# randomUniformForest

Version: 1.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rnorm’
    update.unsupervised: no visible global function definition for
      ‘predict’
    weightedVoteModel: no visible global function definition for ‘lm’
    which.is.duplicate: no visible global function definition for ‘na.omit’
    Undefined global functions or variables:
      abline aggregate barplot cor dev.new dev.off dist graphics.off grid
      head heat.colors kmeans legend lm median memory.limit model.frame
      model.matrix model.response mosaicplot na.omit par pbinom persp plot
      points predict qnorm quantile rnorm runif sd title
    Consider adding
      importFrom("grDevices", "dev.new", "dev.off", "graphics.off",
                 "heat.colors")
      importFrom("graphics", "abline", "barplot", "grid", "legend",
                 "mosaicplot", "par", "persp", "plot", "points", "title")
      importFrom("stats", "aggregate", "cor", "dist", "kmeans", "lm",
                 "median", "model.frame", "model.matrix", "model.response",
                 "na.omit", "pbinom", "predict", "qnorm", "quantile",
                 "rnorm", "runif", "sd")
      importFrom("utils", "head", "memory.limit")
    to your NAMESPACE file.
    ```

# rangeMapper

Version: 0.3-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# raptr

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘gurobi’ ‘rgurobi’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   3.6Mb
        doc    1.4Mb
    ```

# Rariant

Version: 1.16.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    >   region = GRanges("chr17", IRanges(7572050, width = 100))
    > 
    >   control_bam = system.file("extdata", "platinum", "control.bam", package =
    +     "Rariant", mustWork = TRUE)
    >   test1_bam = system.file("extdata", "platinum", "test.bam", package =
    +     "Rariant", mustWork = TRUE)
    >   test2_bam = system.file("extdata", "platinum", "test2.bam", package =
    +     "Rariant", mustWork = TRUE)
    >   mix_bam = system.file("extdata", "platinum", "mix.bam", package =
    +     "Rariant", mustWork = TRUE)
    > 
    >   bam_files = c(control_bam, test1_bam, test2_bam, mix_bam)
    > 
    >   library(BSgenome.Hsapiens.UCSC.hg19)
    >   ref = BSgenome.Hsapiens.UCSC.hg19
    > 
    >   p = tracks(lapply(bam_files, tallyPlot, region, ref, minQual = 25))
    Error in min(c(x, xmin), na.rm = TRUE) : 
      invalid 'type' (list) of argument
    Calls: tracks ... scale_by_xlim -> getLimits -> getLimits -> .local -> .getLimits
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Attaching package: 'VariantAnnotation'
    
    The following object is masked from 'package:base':
    
        tabulate
    
    Need specific help about ggbio? try mailing 
     the maintainer or visit http://tengfei.github.com/ggbio/
    
    Attaching package: 'ggbio'
    
    The following objects are masked from 'package:ggplot2':
    
        geom_bar, geom_rect, geom_segment, ggsave, stat_bin,
        stat_identity, xlim
    
    Quitting from lines 613-616 (Rariant-vignette.Rhtml) 
    Error: processing vignette 'Rariant-vignette.Rhtml' failed with diagnostics:
    invalid 'type' (list) of argument
    Execution halted
    ```

## Newly fixed

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'rbind_all' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.8Mb
      sub-directories of 1Mb or more:
        doc       2.3Mb
        extdata   5.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    tallyBamRegion: no visible global function definition for 'PileupParam'
    tallyBamRegion: no visible global function definition for
      'ScanBamParam'
    tallyBamRegion: no visible global function definition for 'pileup'
    Undefined global functions or variables:
      PileupParam ScanBamParam pileup
    ```

*   checking installed files from ‘inst/doc’ ... NOTE
    ```
    The following files should probably not be installed:
      ‘rariant-inspect-ci.png’, ‘rariant-inspect-shift.png’
    
    Consider the use of a .Rinstignore file: see ‘Writing R Extensions’,
    or move the vignette sources from ‘inst/doc’ to ‘vignettes’.
    ```

# rasterVis

Version: 0.44

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RATest

Version: 0.1.2

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: RATest
    Loading required package: ggplot2
    Loading required package: gridExtra
    Warning in eval(family$initialize) :
      non-integer #successes in a binomial glm!
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figure/panel_a")
    Warning in eval(family$initialize) :
      non-integer #successes in a binomial glm!
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figure/panel_b")
    Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
      dots in figure paths replaced with _ ("figure/hist_cdf")
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RDperm.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RatingScaleReduction

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rattle

Version: 5.1.0

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘RGtk2’ ‘cairoDevice’
    
    Packages suggested but not available for checking:
      ‘gWidgetsRGtk2’ ‘playwith’ ‘rggobi’ ‘RGtk2Extras’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# rbefdata

Version: 0.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    bef.portal.get.attachments_for: no visible global function definition
      for ‘flush.console’
    bef.portal.get.categories_for: no visible binding for global variable
      ‘id’
    bef.portal.get.categories_for: no visible global function definition
      for ‘read.csv’
    bef.portal.get.dataset: no visible global function definition for
      ‘read.csv’
    bef.portal.get.dataset_by: no visible global function definition for
      ‘read.csv’
    bef.portal.get.datasets.for_keyword : <anonymous>: no visible global
      function definition for ‘read.csv’
    bef.portal.get.datasets.for_proposal: no visible global function
      definition for ‘read.csv’
    upload_file: no visible global function definition for ‘write.csv’
    Undefined global functions or variables:
      browseURL flush.console id read.csv write.csv
    Consider adding
      importFrom("utils", "browseURL", "flush.console", "read.csv",
                 "write.csv")
    to your NAMESPACE file.
    ```

# RBesT

Version: 1.3-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lme4’
      All declared Imports should be used.
    ```

# rbison

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RBMRB

Version: 2.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rbokeh

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# rcartocolor

Version: 0.0.22

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

# rCGH

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    	Current version: 1.10.0
    	This version may contain important changes.
    	Use news(Version == '1.10.0', package = 'rCGH').
    	********************************************************
    
    SNP probes will be used.
    Reading information...
    Reading values...
    Adding presettings...
    Genome build: hg19
    
    Instance of class rCGH-cytoScan
    
    Dataset with 15848 probes and 8 columns.
    Array information:
    
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rCGH.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        extdata   1.2Mb
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    The following directory looks like a leftover from 'knitr':
      ‘figure’
    Please remove from your package.
    ```

# rchess

Version: 0.1

## Newly broken

*   checking whether package ‘rchess’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘rchess’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rchess/new/rchess.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rclimateca

Version: 1.0.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
        |======================================================================| 100%
      
        |                                                                            
        |                                                                      |   0%
        |                                                                            
        |===================================                                   |  50%
        |                                                                            
        |======================================================================| 100%
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 187 SKIPPED: 0 FAILED: 2
      1. Failure: column types for ec_climate_data() are correct (@test-climate_data.R#123) 
      2. Failure: get mudata function for climate data works (@test-climate_data.R#463) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 24 marked UTF-8 strings
    ```

# RcmdrPlugin.FuzzyClust

Version: 1.1

## In both

*   R CMD check timed out
    

# RcmdrPlugin.KMggplot2

Version: 0.2-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RcmdrPlugin.MA

Version: 0.0-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RCMIP5

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rcongresso

Version: 0.3.2

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘rcongresso-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: fetch_despesas_deputado
    > ### Title: Fetches expenditures from deputy using its id
    > ### Aliases: fetch_despesas_deputado
    > 
    > ### ** Examples
    > 
    > gastos_abel_mesquita <- fetch_despesas_deputado(dep_id = 178957)
    Error in .congresso_api(.$path, query) : Bad Request (HTTP 400).
    Calls: fetch_despesas_deputado ... overscope_eval_next -> .congresso_api -> <Anonymous>
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      15: .congresso_api(.$path)
      16: httr::stop_for_status(resp)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 18 SKIPPED: 0 FAILED: 7
      1. Failure: GET deputado inexistente (@test_deputados.R#3) 
      2. Error: (unknown) (@test_deputados.R#10) 
      3. Failure: Dimensoes do dataframe (@test_proposicoes.R#44) 
      4. Failure: Atributos do dataframe (@test_proposicoes.R#50) 
      5. Failure: Campos do dataframe (@test_proposicoes.R#56) 
      6. Error: (unknown) (@test_proposicoes.R#68) 
      7. Error: (unknown) (@test_votacoes.R#9) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Quitting from lines 45-50 (introducao-rcongresso.Rmd) 
    Error: processing vignette 'introducao-rcongresso.Rmd' failed with diagnostics:
    Not Found (HTTP 404).
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RDAVIDWebService

Version: 1.18.0

## In both

*   checking whether package ‘RDAVIDWebService’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RDAVIDWebService/new/RDAVIDWebService.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘RDAVIDWebService’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘RDAVIDWebService’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RDAVIDWebService/new/RDAVIDWebService.Rcheck/RDAVIDWebService’

```
### CRAN

```
* installing *source* package ‘RDAVIDWebService’ ...
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RDAVIDWebService/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘RDAVIDWebService’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RDAVIDWebService/old/RDAVIDWebService.Rcheck/RDAVIDWebService’

```
# rddensity

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rddtools

Version: 0.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rdefra

Version: 0.3.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rdi

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rdiversity

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RDML

Version: 0.9-9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rdpla

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RDS

Version: 0.8-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Rdtq

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# re2r

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# ReactomePA

Version: 1.24.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# realestateDK

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# recipes

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# recoup

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    recoup : <anonymous>: no visible global function definition for
      ‘runValue’
    recoupCorrelation: no visible binding for global variable ‘Index’
    recoupCorrelation: no visible binding for global variable ‘Coverage’
    recoupCorrelation: no visible binding for global variable ‘Condition’
    recoupCorrelation: no visible binding for global variable ‘Design’
    recoupHeatmap : <anonymous>: no visible global function definition for
      ‘grid.text’
    recoupProfile: no visible binding for global variable ‘Signal’
    recoupProfile: no visible binding for global variable ‘Condition’
    recoupProfile: no visible binding for global variable ‘Design’
    reduceExons : <anonymous>: no visible global function definition for
      ‘DataFrame’
    splitVector: no visible global function definition for ‘Rle’
    Undefined global functions or variables:
      Condition Coverage DataFrame Design IRanges Index Rle ScanBamParam
      Seqinfo Signal alphabetFrequency bamWhich<- biocLite dbConnect
      dbDisconnect dbDriver dbGetQuery dbWriteTable f flankedSexon gene
      get.transcript.utr.attributes getBSgenome grid.text indexBam
      installed.genomes mclapply mcmapply runValue seqlevels seqlevels<-
      sexon sortBam subjectHits warnwrap
    ```

# reda

Version: 0.4.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rEDM

Version: 0.6.9

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning in ccm(cbind(thrips_block$Thrips_imaginis, surr_Rain[, i]), E = 8,  :
      Note: CCM results are typically interpreted in the opposite direction of causation. Please see 'Detecting causality in complex ecosystems' (Sugihara et al. 2012) for more details.
    Warning in model$run() :
      Found overlap between lib and pred. Enabling cross-validation with exclusion radius = 0.
    Warning in model$run() :
      lib size request was larger than maximum available; corrected
    Warning in ccm(cbind(thrips_block$Thrips_imaginis, surr_maxT[, i]), E = 8,  :
      Note: CCM results are typically interpreted in the opposite direction of causation. Please see 'Detecting causality in complex ecosystems' (Sugihara et al. 2012) for more details.
    Warning in model$run() :
      Found overlap between lib and pred. Enabling cross-validation with exclusion radius = 0.
    Warning in model$run() :
      lib size request was larger than maximum available; corrected
    Warning in ccm(cbind(thrips_block$Thrips_imaginis, surr_Rain[, i]), E = 8,  :
      Note: CCM results are typically interpreted in the opposite direction of causation. Please see 'Detecting causality in complex ecosystems' (Sugihara et al. 2012) for more details.
    Warning in model$run() :
      Found overlap between lib and pred. Enabling cross-validation with exclusion radius = 0.
    Warning in model$run() :
      lib size request was larger than maximum available; corrected
    Error: processing vignette 'rEDM-algorithms.ltx' failed with diagnostics:
    Running 'texi2dvi' on 'rEDM-algorithms.ltx' failed.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# refuge

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# refund

Version: 0.1-16

## In both

*   checking whether package ‘refund’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/refund/new/refund.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    ...
      Code: function(x, P = NULL, data = NULL, label = "", se1.mult = 1,
                     se2.mult = 2, partial.resids = FALSE, rug = TRUE, se =
                     TRUE, scale = -1, n = 100, n2 = 40, n3 = 3, pers =
                     FALSE, theta = 30, phi = 30, jit = FALSE, xlab = NULL,
                     ylab = NULL, main = NULL, ylim = NULL, xlim = NULL,
                     too.far = 0.1, shade = FALSE, shade.col = "gray80",
                     shift = 0, trans = I, by.resids = FALSE, scheme = 0,
                     ...)
      Docs: function(x, P = NULL, data = NULL, label = "", se1.mult = 1,
                     se2.mult = 2, partial.resids = FALSE, rug = TRUE, se =
                     TRUE, scale = -1, n = 100, n2 = 40, pers = FALSE,
                     theta = 30, phi = 30, jit = FALSE, xlab = NULL, ylab =
                     NULL, main = NULL, ylim = NULL, xlim = NULL, too.far =
                     0.1, shade = FALSE, shade.col = "gray80", shift = 0,
                     trans = I, by.resids = FALSE, scheme = 0, ...)
      Argument names in code not in docs:
        n3
      Mismatches in argument names (first 3):
        Position: 13 Code: n3 Docs: pers
        Position: 14 Code: pers Docs: theta
        Position: 15 Code: theta Docs: phi
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# refund.shiny

Version: 0.3.0

## Newly broken

*   checking whether package ‘refund.shiny’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘refund.shiny’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/refund.shiny/new/refund.shiny.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# reghelper

Version: 0.3.3

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 281 SKIPPED: 0 FAILED: 18
      1. Failure: lm with 2 continuous int. works (@test_graph_model.R#20) 
      2. Failure: lm with 2 continuous int. works (@test_graph_model.R#21) 
      3. Failure: lm with 2 continuous int. works (@test_graph_model.R#22) 
      4. Failure: lm with continuous x 2-level categorical int. works (@test_graph_model.R#46) 
      5. Failure: lm with continuous x 2-level categorical int. works (@test_graph_model.R#47) 
      6. Failure: lm with continuous x 2-level categorical int. works (@test_graph_model.R#48) 
      7. Failure: lm with continuous x 3-level categorical int. works (@test_graph_model.R#72) 
      8. Failure: lm with continuous x 3-level categorical int. works (@test_graph_model.R#73) 
      9. Failure: lm with continuous x 3-level categorical int. works (@test_graph_model.R#74) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# regionReport

Version: 1.14.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    label: MAstyle (with options) 
    List of 2
     $ fig.width : num 10
     $ fig.height: num 10
    
       inline R code fragments
    
    label: genomeOverview1 (with options) 
    List of 6
     $ message   : logi FALSE
     $ fig.width : num 7
     $ fig.height: num 9
     $ dpi       : num 300
     $ eval      : symbol nullExist
     $ echo      : symbol nullExist
    
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Quitting from lines 365-366 (basicExploration.Rmd) 
    Error: subscript contains invalid names
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    [WARNING] This document format requires a nonempty <title> element.
      Please specify either 'title' or 'pagetitle' in the metadata.
      Falling back to 'bumphunterExampleOutput.utf8'
    [WARNING] This document format requires a nonempty <title> element.
      Please specify either 'title' or 'pagetitle' in the metadata.
      Falling back to 'bumphunterExampleOutput.utf8'
    Warning in citation("BiocStyle") :
      no date field in DESCRIPTION file of package 'BiocStyle'
    Warning in citation("biovizBase") :
      no date field in DESCRIPTION file of package 'biovizBase'
    Warning in citation("TxDb.Hsapiens.UCSC.hg19.knownGene") :
      no date field in DESCRIPTION file of package 'TxDb.Hsapiens.UCSC.hg19.knownGene'
    Warning in citation("DEFormats") :
      no date field in DESCRIPTION file of package 'DEFormats'
    Writing 34 Bibtex entries ... OK
    Results written to file 'regionReportRef.bib'
    Quitting from lines 230-235 (basicExploration.Rmd) 
    Error: processing vignette 'regionReport.Rmd' failed with diagnostics:
    subscript contains invalid names
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘DESeq2:::pvalueAdjustment’
      See the note in ?`:::` about the use of this operator.
    ```

# RegularizedSCA

Version: 0.5.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ReinforcementLearning

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# reinsureR

Version: 0.1.0

## Newly broken

*   checking whether package ‘reinsureR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘reinsureR’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/reinsureR/new/reinsureR.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RelimpPCR

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rem

Version: 1.2.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# repijson

Version: 0.1.0

## In both

*   checking examples ... ERROR
    ```
    ...
    > toyll$date.of.admission <- as.POSIXct(toyll$date.of.admission)
    > toyll$date.of.discharge <- as.POSIXct(toyll$date.of.discharge)
    > toyll$contact1.date <- as.POSIXct(toyll$contact1.date)
    > toyll$contact2.date <- as.POSIXct(toyll$contact2.date)
    > toyll$contact3.date <- as.POSIXct(toyll$contact3.date)
    > 
    > ind.fields <- c(names(toyll)[1:5], "hospital", "fever", "sleepy")
    > x <- as.ejObject(toyll,
    +                  recordAttributes=ind.fields,
    +                  eventDefinitions=list(
    +                  define_ejEvent(name="admission", date="date.of.admission"),
    +                  define_ejEvent(name="discharge", date="date.of.discharge"),
    +                  define_ejEvent(name="contact1", date="contact1.date", attributes="contact1.id"),
    +                  define_ejEvent(name="contact2", date="contact2.date", attributes="contact2.id"),
    +                  define_ejEvent(name="contact3", date="contact3.date", attributes="contact3.id")
    +                  ))
    > print(x)
    Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
      there is no package called ‘sf’
    Calls: print ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 68-80 (repijson.Rmd) 
    Error: processing vignette 'repijson.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# replicationInterval

Version: 2.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# replyr

Version: 0.9.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘magick’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘rquery’
    ```

# ReporteRs

Version: 0.8.10

## In both

*   checking whether package ‘ReporteRs’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ReporteRs/new/ReporteRs.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘ReporteRs’ ...
** package ‘ReporteRs’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘ReporteRsjars’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘ReporteRsjars’ could not be loaded
ERROR: lazy loading failed for package ‘ReporteRs’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ReporteRs/new/ReporteRs.Rcheck/ReporteRs’

```
### CRAN

```
* installing *source* package ‘ReporteRs’ ...
** package ‘ReporteRs’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘ReporteRsjars’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/ReporteRs/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘ReporteRsjars’ could not be loaded
ERROR: lazy loading failed for package ‘ReporteRs’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/ReporteRs/old/ReporteRs.Rcheck/ReporteRs’

```
# ReportingTools

Version: 2.20.0

## In both

*   R CMD check timed out
    

*   checking R code for possible problems ... NOTE
    ```
    ...
      'fData'
    .marrayLM.to.html: no visible global function definition for
      'featureNames'
    .marrayLM.to.html: no visible global function definition for 'fData'
    check.ids: no visible binding for global variable 'org.Hs.eg.db'
    check.ids: no visible global function definition for 'keys'
    custHeaderPanel : <anonymous>: no visible binding for global variable
      'tags'
    custHeaderPanel : <anonymous>: no visible global function definition
      for 'HTML'
    custHeaderPanel: no visible global function definition for 'tagList'
    custHeaderPanel: no visible global function definition for 'tag'
    custHeaderPanel: no visible global function definition for 'div'
    custHeaderPanel: no visible global function definition for 'h1'
    publish,trellis-HTMLReport: no visible binding for global variable
      'htmlRep'
    toReportDF,DESeqDataSet: no visible global function definition for
      'mcols'
    Undefined global functions or variables:
      HTML columns description div exprs fData featureNames h1 htmlRep keys
      keytype mcols org.Hs.eg.db tag tagList tags
    ```

# reproducer

Version: 0.1.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rerddap

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package which this enhances but not available for checking: ‘taxize’
    ```

# reReg

Version: 1.1-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# reval

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# revdbayes

Version: 1.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rex

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rexposome

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.0Mb
      sub-directories of 1Mb or more:
        data      2.8Mb
        doc       5.1Mb
        extdata   2.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    imputeLOD: multiple local function definitions for ‘faux’ with
      different formal arguments
    plotHistogram,ExposomeSet: no visible binding for global variable
      ‘..density..’
    Undefined global functions or variables:
      ..density..
    ```

# rfigshare

Version: 0.3.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    fs_author_ids : <anonymous>: no visible global function definition for
      ‘select.list’
    fs_download : <anonymous>: no visible global function definition for
      ‘download.file’
    Undefined global functions or variables:
      download.file select.list
    Consider adding
      importFrom("utils", "download.file", "select.list")
    to your NAMESPACE file.
    ```

# rfisheries

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rfm

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RFmarkerDetector

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rfordummies

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RforProteomics

Version: 1.18.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following object is masked from 'package:tools':
    
        toHTML
    
    
    Attaching package: 'annotate'
    
    The following object is masked from 'package:mzR':
    
        nChrom
    
    Loading required package: cluster
    Warning: replacing previous import 'BiocGenerics::var' by 'stats::var' when loading 'MLInterfaces'
    
    This is pRoloc version 1.20.0 
      Visit https://lgatto.github.io/pRoloc/ to get started.
    
    Quitting from lines 29-46 (RProtVis.Rmd) 
    Error: processing vignette 'RProtVis.Rmd' failed with diagnostics:
    there is no package called 'pRolocdata'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘synapterdata’ ‘pRolocdata’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 11.7Mb
      sub-directories of 1Mb or more:
        doc  10.8Mb
    ```

# rfPermute

Version: 2.1.5

## In both

*   checking whether package ‘rfPermute’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rfPermute/new/rfPermute.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rfPermute’ ...
** package ‘rfPermute’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘rfPermute’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rfPermute/new/rfPermute.Rcheck/rfPermute’

```
### CRAN

```
* installing *source* package ‘rfPermute’ ...
** package ‘rfPermute’ successfully unpacked and MD5 sums checked
** R
** data
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘rfPermute’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rfPermute/old/rfPermute.Rcheck/rfPermute’

```
# rgbif

Version: 0.9.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RGCCA

Version: 2.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RGraphics

Version: 2.0-14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘gWidgetsRGtk2’ ‘playwith’ ‘pmg’ ‘rggobi’ ‘RGraphics’ ‘SVGAnnotation’
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.9Mb
      sub-directories of 1Mb or more:
        extra   9.4Mb
    ```

# ri2

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RiboProfiling

Version: 1.10.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > data(ctrlGAlignments)
    > aln <- ctrlGAlignments
    > #transform the GAlignments object into a GRanges object (faster processing)
    > alnGRanges <- readsToStartOrEnd(aln, what="start")
    > #make a txdb object containing the annotations for the specified species.
    > #In this case hg19.
    > txdb <- TxDb.Hsapiens.UCSC.hg19.knownGene::TxDb.Hsapiens.UCSC.hg19.knownGene
    > #Please make sure that seqnames of txdb correspond to
    > #the seqnames of the alignment files ("chr" particle)
    > #if not rename the txdb seqlevels
    > #renameSeqlevels(txdb, sub("chr", "",seqlevels(txdb)))
    > #get the flanking region around the promoter of the best expressed CDSs
    > oneBinRanges <- aroundPromoter(txdb, alnGRanges)
    > #the read start coverage around the TSS as a percentage for all match sizes.
    > covSummarized <- readStartCov(alnGRanges, oneBinRanges, matchSize="all",
    + c(-20,20), "aroundTSS", charPerc="perc")
    > trackPlotTSS <- plotSummarizedCov(covSummarized)
    Error in min(c(x, xmin), na.rm = TRUE) : 
      invalid 'type' (list) of argument
    Calls: plotSummarizedCov ... <Anonymous> -> getLimits -> getLimits -> .local -> .getLimits
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             }
             else {
                 res <- NULL
             }
         })
      21: FUN(X[[i]], ...)
      22: data.frame(xmin = min(c(x, xmin), na.rm = TRUE), xmax = max(c(x, xmax, xend), na.rm = TRUE), 
             ymin = min(c(y, ymin), na.rm = TRUE), ymax = max(c(y, ymax, yend), na.rm = TRUE))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 6 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test-riboSeqFromBAM.R#7) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   R CMD check timed out
    

*   checking whether package ‘RiboProfiling’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘BiocGenerics::Position’ by ‘ggplot2::Position’ when loading ‘RiboProfiling’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RiboProfiling/new/RiboProfiling.Rcheck/00install.out’ for details.
    ```

*   checking R code for possible problems ... NOTE
    ```
    applyShiftFeature: no visible global function definition for 'is'
    aroundPromoter: no visible global function definition for 'is'
    codonInfo: no visible global function definition for 'is'
    countShiftReads: no visible global function definition for 'is'
    histMatchLength: no visible global function definition for 'is'
    orfRelativePos: no visible global function definition for 'is'
    plotSummarizedCov: no visible global function definition for 'is'
    readStartCov: no visible global function definition for 'is'
    readsToStartOrEnd: no visible global function definition for 'is'
    riboSeqFromBAM: no visible global function definition for 'is'
    Undefined global functions or variables:
      is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# Rilostat

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RImagePalette

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rinat

Version: 0.1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# riskRegression

Version: 2018.04.21

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RITAN

Version: 1.3.0

## Newly broken

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

## Newly fixed

*   R CMD check timed out
    

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘RITAN-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot.term_enrichment
    > ### Title: plot.term_enrichment
    > ### Aliases: plot.term_enrichment
    > 
    > ### ** Examples
    > 
    > require(RITANdata)
    Loading required package: RITANdata
    > e <- term_enrichment(vac1.day0vs31.de.genes, term_sources = 'GO_slim_generic')
    Warning in scan(file = file, what = what, sep = sep, quote = quote, dec = dec,  :
      URL 'ftp://ftp.ebi.ac.uk/pub/databases/genenames/new/tsv/locus_groups/protein-coding_gene.txt': status was 'Failure when receiving data from the peer'
    Error in scan(file = file, what = what, sep = sep, quote = quote, dec = dec,  : 
      cannot read from connection
    Calls: term_enrichment ... load_all_protein_coding_symbols -> read.table -> scan
    Execution halted
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘sqldf’
    A package should be listed in only one of these fields.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BgeeDB’ ‘dynamicTreeCut’ ‘gsubfn’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    load_geneset_symbols: no visible binding for global variable
      ‘geneset_list’
    load_geneset_symbols: no visible binding for global variable ‘f’
    load_geneset_symbols: no visible binding for global variable
      ‘active_genesets’
    network_overlap : <anonymous>: no visible binding for global variable
      ‘network_list’
    network_overlap: no visible binding for global variable ‘network_list’
    network_overlap : select_edges: no visible binding for global variable
      ‘network_list’
    plot.term_enrichment_by_subset: no visible binding for global variable
      ‘Var2’
    plot.term_enrichment_by_subset: no visible binding for global variable
      ‘Var1’
    show_active_genesets_hist: no visible binding for global variable
      ‘active_genesets’
    term_enrichment : process_source: no visible binding for global
      variable ‘active_genesets’
    Undefined global functions or variables:
      Var1 Var2 active_genesets all_net all_symbols f geneset_list
      network_list ss2
    ```

# RIVER

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rivr

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RJafroc

Version: 1.0.1

## In both

*   checking whether package ‘RJafroc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RJafroc/new/RJafroc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘RJafroc’ ...
** package ‘RJafroc’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BMFuncs.cpp -o BMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c CBMFuncs.cpp -o CBMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c CorCBMFuncs.cpp -o CorCBMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c MyFOM.cpp -o MyFOM.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RsmFuncs.cpp -o RsmFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c SmAucInt.cpp -o SmAucInt.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Transforms.cpp -o Transforms.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o RJafroc.so BMFuncs.o CBMFuncs.o CorCBMFuncs.o MyFOM.o RcppExports.o RsmFuncs.o SmAucInt.o Transforms.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/RJafroc/new/RJafroc.Rcheck/RJafroc/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘RJafroc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RJafroc/new/RJafroc.Rcheck/RJafroc’

```
### CRAN

```
* installing *source* package ‘RJafroc’ ...
** package ‘RJafroc’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BMFuncs.cpp -o BMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c CBMFuncs.cpp -o CBMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c CorCBMFuncs.cpp -o CorCBMFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c MyFOM.cpp -o MyFOM.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RsmFuncs.cpp -o RsmFuncs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c SmAucInt.cpp -o SmAucInt.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Transforms.cpp -o Transforms.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o RJafroc.so BMFuncs.o CBMFuncs.o CorCBMFuncs.o MyFOM.o RcppExports.o RsmFuncs.o SmAucInt.o Transforms.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/RJafroc/old/RJafroc.Rcheck/RJafroc/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RJafroc/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘RJafroc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RJafroc/old/RJafroc.Rcheck/RJafroc’

```
# rKIN

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rlfsm

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RmarineHeatWaves

Version: 0.16.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rmcfs

Version: 1.2.10

## In both

*   checking whether package ‘rmcfs’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rmcfs/new/rmcfs.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rmcfs’ ...
** package ‘rmcfs’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘rmcfs’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rmcfs/new/rmcfs.Rcheck/rmcfs’

```
### CRAN

```
* installing *source* package ‘rmcfs’ ...
** package ‘rmcfs’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/rmcfs/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘rmcfs’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rmcfs/old/rmcfs.Rcheck/rmcfs’

```
# rmcorr

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RMCriteria

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# rmetasim

Version: 3.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rms

Version: 5.1-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘multiwayvcov’, ‘treatSens’
    ```

# rmweather

Version: 0.1.1

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      > library(testthat)
      > library(rmweather)
      > 
      > test_check("rmweather")
      ── 1. Failure: Test training function (@test_02_rmw_functions.R#53)  ───────────
      model$r.squared not equal to 0.4184627.
      1/1 mismatches
      [1] 0.399 - 0.418 == -0.0196
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 32 SKIPPED: 0 FAILED: 1
      1. Failure: Test training function (@test_02_rmw_functions.R#53) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘testthat’
      All declared Imports should be used.
    ```

# RNAdecay

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 98.8Mb
      sub-directories of 1Mb or more:
        data   2.7Mb
        libs  95.8Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    decay_plot: no visible binding for global variable ‘text’
    decay_plot: no visible binding for global variable ‘x’
    decay_plot: no visible binding for global variable ‘y’
    decay_plot: no visible binding for global variable ‘t.decay’
    decay_plot: no visible binding for global variable ‘value’
    decay_plot: no visible binding for global variable ‘treatment’
    Undefined global functions or variables:
      t.decay text treatment value x y
    Consider adding
      importFrom("graphics", "text")
    to your NAMESPACE file.
    ```

# RNAseqNet

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RnBeads

Version: 1.12.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘BiocGenerics’ ‘S4Vectors’ ‘GenomicRanges’ ‘MASS’ ‘cluster’ ‘ff’
      ‘fields’ ‘ggplot2’ ‘gplots’ ‘gridExtra’ ‘limma’ ‘matrixStats’
      ‘illuminaio’ ‘methylumi’ ‘plyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.6Mb
      sub-directories of 1Mb or more:
        R     3.1Mb
        doc   3.2Mb
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘RnBeads’ for: ‘samples’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      'Gviz:::.getBMFeatureMap' 'doParallel:::.options'
      'grDevices:::.smoothScatterCalcDensity'
      'minfi:::.default.450k.annotation' 'minfi:::.extractFromRGSet450k'
      'minfi:::.normalizeFunnorm450k'
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      Density Deviance Difference DoISVA Error EstDimRMT GenomeAxisTrack ID
      IdeogramTrack IlluminaHumanMethylation450kmanifest
      IlluminaHumanMethylationEPICmanifest Index Intensity Measure
      PairsBootRefFreeEwasModel Probe RGChannelSet Rd2HTML RefFreeEwasModel
      SNP Sample Slide Target Term UcscTrack Value addSex as.profileCGH
      assayDataElement assayDataElementNames barcode bv chrom combinedRank
      combinedRank.var comma covgMedian covgPercLow covgPercUp cv.glmnet
      daglad diffmeth diffmeth.p.adj.fdr diffmeth.p.val
      dinucleotideFrequency eps expectedCounts featureData featureData<-
      featureNames featureNames<- foreach geneCounts genome<- getCN
      getDoParWorkers getGreen getManifest getMeth getRed getSex getUnmeth
      getVarCov glmnet grid.draw grid.newpage group group1 group2 i
      impute.knn intensities is.subsegmentation k letterFrequency lme
      loadRegionDB log10FDR log10P mapToGenome mean.diff mean.quot.log2
      melt mergeRegionDBs muted n.sites ng.2 num.sites numSites
      numeric.names oddsRatios p.vals.t.na.adj pData percent_format
      phenoData phenoData<- plotTracks preprocessSWAN pvalues qvalue
      refText reg.type region.size registerDoParallel relative.coord report
      runLOLA samples seqlengths seqlevels<- sigCategories sites2ignore
      size solve.QP stopCluster sva target tsne type types universeCounts
      useMart v var.diff varFit varLabels x xmlValue y yint
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RnBeads.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# RndTexExams

Version: 1.4.1

## In both

*   checking examples ... ERROR
    ```
    ...
    > # Build pdfs
    > result.out <- rte.build.rdn.test(list.in = list.out,
    +                                  f.out = f.out,
    +                                  n.test = n.test,
    +                                  n.question = n.question,
    +                                  latex.dir.out = latex.dir.out,
    +                                  pdf.dir.out = pdf.dir.out)
    
    rte: Checking for error in inputs... Done
    rte: pdflatex flavor: texlive
    rte: Type of OS: Darwin
    rte: Latex compile function: custom
    rte: Type of exam template: examdesign
    rte: Number of mchoice questions: 4
    rte: Building Test #1...This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017) (preloaded format=pdflatex)
     restricted \write18 enabled.
    entering extended mode
    Error in rte.compile.latex(f.temp.tex, pdf.dir.out = pdf.dir.out, latex.compile.fct = latex.compile.fct) : 
      Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
    Calls: rte.build.rdn.test -> rte.compile.latex
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 1. Error: (unknown) (@test_rte.R#24)  ───────────────────────────────────────
      Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
      1: rte.build.rdn.test(list.in = list.out, f.out = f.out, n.test = n.test, n.question = n.question, 
             latex.dir.out = latex.dir.out, pdf.dir.out = pdf.dir.out, do.randomize.questions = T, 
             latex.compile.fct = "custom", do.randomize.answers = T, do.clean.up = T) at testthat/test_rte.R:24
      2: rte.compile.latex(f.temp.tex, pdf.dir.out = pdf.dir.out, latex.compile.fct = latex.compile.fct)
      3: stop(paste("Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your", 
             "OS has a flavor of latex installed."))
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 1 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_rte.R#24) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017) (preloaded format=pdflatex)
     restricted \write18 enabled.
    entering extended mode
    Quitting from lines 121-147 (rte-vignette_creating_exams.Rmd) 
    Error: processing vignette 'rte-vignette_creating_exams.Rmd' failed with diagnostics:
    Cant find compiled pdf file. You should check for sintax errors in the tex file and whether your OS has a flavor of latex installed.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Rnightlights

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Rnits

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    trying URL 'https://ftp.ncbi.nlm.nih.gov/geo/series/GSE4nnn/GSE4158/matrix/GSE4158_series_matrix.txt.gz'
    Content type 'application/x-gzip' length 514853 bytes (502 KB)
    ==================================================
    downloaded 502 KB
    
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Rnits-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘scale_color_brewer’
    plotResults,Rnits: no visible global function definition for
      ‘facet_wrap’
    plotResults,Rnits: no visible global function definition for ‘dev.off’
    summary,Rnits: no visible global function definition for ‘hist’
    timeAlign,Rnits: no visible global function definition for ‘quantile’
    timeAlign,Rnits: no visible global function definition for ‘mvfft’
    timeAlign,Rnits: no visible global function definition for ‘abline’
    Undefined global functions or variables:
      Sample Time abline aes dev.off facet_wrap gaussian geom_point
      geom_smooth glm hat hist kmeans mad median mvfft p.adjust par predict
      quantile rnorm scale_color_brewer sd setNames setTxtProgressBar
      smooth.spline theme theme_bw txtProgressBar value ylab
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("graphics", "abline", "hist", "par")
      importFrom("stats", "gaussian", "glm", "hat", "kmeans", "mad",
                 "median", "mvfft", "p.adjust", "predict", "quantile",
                 "rnorm", "sd", "setNames", "smooth.spline")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# rnoaa

Version: 0.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RNOmni

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rnpn

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RNRCS

Version: 0.2.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rnrfa

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# robCompositions

Version: 2.0.7

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in warn_if_args_exist(list(...)) :
      Extra arguments: 'colour', 'alpha', 'shape' are being ignored.  If these are meant to be aesthetics, submit them using the 'mapping' variable within ggpairs with ggplot2::aes or ggplot2::aes_string.
    Quitting from lines 905-906 (imputation.Rnw) 
    Error: processing vignette 'imputation.Rnw' failed with diagnostics:
    variables: "x" have non standard format: "~X1".  Please rename the columns or make a new column.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘StatDA’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5701 marked UTF-8 strings
    ```

# robmed

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# robustbase

Version: 0.93-0

## Newly broken

*   checking running R code from vignettes ...
    ```
       ‘fastMcd-kmini.Rnw’ using ‘UTF-8’ ... OK
       ‘lmrob_simulation.Rnw’ using ‘UTF-8’ ... failed
       ‘psi_functions.Rnw’ using ‘UTF-8’ ... OK
     ERROR
    Errors in running code in vignettes:
    when running code in ‘lmrob_simulation.Rnw’
      ...
    
    > g.truncate.line <- geom_hline(yintercept = trunc[2], 
    +     color = theme$panel.border$colour)
    
    > g.truncate.areas <- annotate("rect", xmin = rep(-Inf, 
    +     2), xmax = rep(Inf, 2), ymin = c(0, Inf), ymax = trunc, fill = theme$panel.grid.major$c .... [TRUNCATED] 
    
      When sourcing ‘lmrob_simulation.R’:
    Error: Unequal parameter lengths: xmin (2), xmax (2), ymin (2), ymax (2), fill (0)
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘robustX’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: robustbase
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'fastMcd-kmini.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# robustHD

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# robustlmm

Version: 2.2-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: robustlmm
    Loading required package: lme4
    Loading required package: Matrix
    Loading required package: ggplot2
    Warning in compare(fmUncontam, fm, rfm, rfm2, rfm3, show.rho.functions = FALSE) :
      Comparison for objects not fitted to the same dataset
    Loading required package: reshape2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rlmer.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rODE

Version: 0.99.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘data.table’
      All declared Imports should be used.
    ```

# Roleswitch

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘pracma’ ‘reshape’ ‘plotrix’ ‘microRNA’ ‘biomaRt’ ‘Biostrings’
      ‘Biobase’ ‘DBI’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    diagnosticPlot: no visible global function definition for ‘par’
    diagnosticPlot: no visible global function definition for ‘axis’
    diagnosticPlot: no visible global function definition for ‘plot’
    getSeedMatrix: no visible global function definition for ‘data’
    getTranscriptIDwithLongest3UTR: no visible global function definition
      for ‘aggregate’
    roleswitch: no visible global function definition for ‘aggregate’
    Undefined global functions or variables:
      aggregate axis data par plot
    Consider adding
      importFrom("graphics", "axis", "par", "plot")
      importFrom("stats", "aggregate")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    3: max(p.x-p.x.prev)=0.00009
    4: max(p.x-p.x.prev)=0.00000
    
    Start roleswitch with 365 miRNA and 10945 mRNA
    1: max(p.x-p.x.prev)=0.12593
    2: max(p.x-p.x.prev)=0.00008
    3: max(p.x-p.x.prev)=0.00003
    4: max(p.x-p.x.prev)=0.00000
    Some genes or miRNA are left out in calculation
    b/c they have zero target sites or targets!
    Their probabilities are set to zero in the output matrices
    Loading required package: ggplot2
    Loading required package: hgu95av2.db
    Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
      there is no package called ‘hgu95av2.db’
    
    Error: processing vignette 'Roleswitch.Rnw' failed with diagnostics:
     chunk 10 (label = eset) 
    Error in roleswitch(eset, mirna.expr) : 
      hgu95av2.db package must be installed
    Execution halted
    ```

# rollply

Version: 0.5.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘rollply-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: build_grid_ahull_fill
    > ### Title: Regular grid in alpha-hull
    > ### Aliases: build_grid_ahull_fill
    > 
    > ### ** Examples
    > 
    > 
    > data(meadow)
    > 
    > # Plot a grid with diagnostics
    > grid <- build_grid_ahull_fill(meadow[ ,c('lon','lat')], 10000,
    +                               grid_opts = list(verbose = TRUE))
    Error in build_grid_ahull_fill(meadow[, c("lon", "lat")], 10000, grid_opts = list(verbose = TRUE)) : 
      Alpha-hull-based grids require package alphahull
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             if (logical.return) 
                 message(paste("Error:", msg), domain = NA)
             else stop(msg, call. = FALSE, domain = NA)
         })
      3: tryCatchList(expr, classes, parentenv, handlers)
      4: tryCatchOne(expr, names, parentenv, handlers[[1L]])
      5: value[[3L]](cond)
      6: stop(msg, call. = FALSE, domain = NA)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 9 SKIPPED: 0 FAILED: 1
      1. Error: inahull_cpp works (@test_inahull.R#9) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    trying URL 'http://www.nosdonnees.fr/wiki/images/b/b5/EUCircos_Regions_departements_circonscriptions_communes_gps.csv.gz'
    Content type 'application/x-gzip' length 623285 bytes (608 KB)
    ==================================================
    downloaded 608 KB
    
    Warning in eval(expr, envir, enclos) : NAs introduced by coercion
    Warning in eval(expr, envir, enclos) : NAs introduced by coercion
    Quitting from lines 168-174 (rollply.Rmd) 
    Error: processing vignette 'rollply.Rmd' failed with diagnostics:
    Alpha-hull-based grids require package alphahull
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rolypoly

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘matrixcalc’
      All declared Imports should be used.
    ```

# rorutadis

Version: 0.4.2

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rglpk’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# rotations

Version: 1.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rotations-intro.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        data   2.3Mb
    ```

# rpanel

Version: 1.1-4

## In both

*   checking whether package ‘rpanel’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/new/rpanel.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rpanel’ ...
** package ‘rpanel’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in structure(.External(.C_dotTcl, ...), class = "tclObj") : 
  [tcl] can't find package BWidget.

Error : unable to load R code in package ‘rpanel’
ERROR: lazy loading failed for package ‘rpanel’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/new/rpanel.Rcheck/rpanel’

```
### CRAN

```
* installing *source* package ‘rpanel’ ...
** package ‘rpanel’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error in structure(.External(.C_dotTcl, ...), class = "tclObj") : 
  [tcl] can't find package BWidget.

Error : unable to load R code in package ‘rpanel’
ERROR: lazy loading failed for package ‘rpanel’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rpanel/old/rpanel.Rcheck/rpanel’

```
# rpdo

Version: 0.2.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RpeakChrom

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rpf

Version: 0.59

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rplos

Version: 0.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RPPanalyzer

Version: 1.4.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'RPPanalyzer.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        extdata   4.5Mb
    ```

# rPref

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# rprev

Version: 0.2.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Rprofet

Version: 2.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rpsftm

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Rqc

Version: 1.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# rrecsys

Version: 0.9.7.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rreg

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rrepast

Version: 0.6.0

## In both

*   checking whether package ‘rrepast’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rrepast/new/rrepast.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rrepast’ ...
** package ‘rrepast’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘rrepast’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rrepast/new/rrepast.Rcheck/rrepast’

```
### CRAN

```
* installing *source* package ‘rrepast’ ...
** package ‘rrepast’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/rrepast/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘rrepast’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rrepast/old/rrepast.Rcheck/rrepast’

```
# rrpack

Version: 0.1-7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rrr

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘rrr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: residuals
    > ### Title: Reduced-Rank Regression Residuals
    > ### Aliases: residuals
    > 
    > ### ** Examples
    > 
    > data(tobacco)
    > tobacco_x <- tobacco[,4:9]
    > tobacco_y <- tobacco[,1:3]
    > tobacco_rrr <- rrr(tobacco_x, tobacco_y, rank = 1)
    > residuals(tobacco_x, tobacco_y, rank = 1, plot = FALSE)
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~index".  Please rename the columns or make a new column.
    Calls: residuals ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking whether package ‘rrr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘rrr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rrr/new/rrr.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 241-242 (rrr.Rmd) 
    Error: processing vignette 'rrr.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~index".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Rcpp’
      All declared Imports should be used.
    ```

# RSA

Version: 0.9.11

## In both

*   checking whether package ‘RSA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: loading Rplot failed
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RSA/new/RSA.Rcheck/00install.out’ for details.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RSAlgaeR

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rsample

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rSARP

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking files in ‘vignettes’ ... NOTE
    ```
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/Untitled.Rnw’
    (Is a VignetteBuilder field missing?)
    ```

# RSDA

Version: 2.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rsimsum

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rsMove

Version: 0.2.3

## Newly broken

*   checking whether package ‘rsMove’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘rsMove’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsMove/new/rsMove.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gdalUtils’
      All declared Imports should be used.
    ```

# rsoi

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘utils’
      All declared Imports should be used.
    ```

# rSQM

Version: 1.3.14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RSSL

Version: 0.6.1

## Newly broken

*   checking whether package ‘RSSL’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘RSSL’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RSSL/new/RSSL.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rstan

Version: 2.17.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 10.0Mb
      sub-directories of 1Mb or more:
        libs   7.5Mb
    ```

# rstanarm

Version: 2.17.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 19.0Mb
      sub-directories of 1Mb or more:
        R      2.0Mb
        doc    2.8Mb
        libs  13.5Mb
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘gamm4’, ‘biglm’
    ```

*   checking for GNU extensions in Makefiles ... NOTE
    ```
    GNU make is a SystemRequirements.
    ```

# rstansim

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RStoolbox

Version: 0.2.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    49 LC82240632013167LGN00 LC82240632013167LGN00
                                                                   Download.Link
    6  http://earthexplorer.usgs.gov/download/options/4923/LC82240632015157LGN00
    27 http://earthexplorer.usgs.gov/download/options/4923/LC82240632014186LGN00
    40 http://earthexplorer.usgs.gov/download/options/4923/LC82240632013327LGN00
    46 http://earthexplorer.usgs.gov/download/options/4923/LC82240632013215LGN00
    49 http://earthexplorer.usgs.gov/download/options/4923/LC82240632013167LGN00
       Browse.Link       Date Doy Year Satellite Num
    6           NA 2015-06-06 157 2015       LS8   8
    27          NA 2014-07-05 186 2014       LS8   8
    40          NA 2013-11-23 327 2013       LS8   8
    46          NA 2013-08-03 215 2013       LS8   8
    49          NA 2013-06-16 167 2013       LS8   8
    > 
    > ## Available time-series
    > ggplot(ee) + 
    +         geom_segment(aes(x = Date, xend = Date, y = 0, yend = 100 - Cloud.Cover, 
    +      col = as.factor(Year))) +
    +         scale_y_continuous(name = "Scene quality (% clear sky)")
    Error: Columns `x`, `xend` are dates/times and must be stored as POSIXct, not POSIXlt
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rsvd

Version: 0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   6.1Mb
    ```

# rsvg

Version: 1.3

## In both

*   checking whether package ‘rsvg’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/new/rsvg.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘rsvg’ ...
** package ‘rsvg’ successfully unpacked and MD5 sums checked
Homebrew 1.5.13
Homebrew/homebrew-core (git revision 23a7; last commit 2018-03-30)
Using PKG_CFLAGS=-I/usr/local/opt/librsvg/include
Using PKG_LIBS=-L/usr/local/opt/librsvg/lib -lrsvg
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because librsvg-2.0 was not found. Try installing:
 * deb: librsvg2-dev (Debian, Ubuntu, etc)
 * rpm: librsvg2-devel (Fedora, EPEL)
 * csw: librsvg_dev, sunx11_devel (Solaris)
 * brew: librsvg (OSX)
If librsvg-2.0 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a librsvg-2.0.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘rsvg’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/new/rsvg.Rcheck/rsvg’

```
### CRAN

```
* installing *source* package ‘rsvg’ ...
** package ‘rsvg’ successfully unpacked and MD5 sums checked
Homebrew 1.5.13
Homebrew/homebrew-core (git revision 23a7; last commit 2018-03-30)
Using PKG_CFLAGS=-I/usr/local/opt/librsvg/include
Using PKG_LIBS=-L/usr/local/opt/librsvg/lib -lrsvg
------------------------- ANTICONF ERROR ---------------------------
Configuration failed because librsvg-2.0 was not found. Try installing:
 * deb: librsvg2-dev (Debian, Ubuntu, etc)
 * rpm: librsvg2-devel (Fedora, EPEL)
 * csw: librsvg_dev, sunx11_devel (Solaris)
 * brew: librsvg (OSX)
If librsvg-2.0 is already installed, check that 'pkg-config' is in your
PATH and PKG_CONFIG_PATH contains a librsvg-2.0.pc file. If pkg-config
is unavailable you can set INCLUDE_DIR and LIB_DIR manually via:
R CMD INSTALL --configure-vars='INCLUDE_DIR=... LIB_DIR=...'
--------------------------------------------------------------------
ERROR: configuration failed for package ‘rsvg’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/rsvg/old/rsvg.Rcheck/rsvg’

```
# RSwissMaps

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 18627 marked UTF-8 strings
    ```

# RTCGA

Version: 1.10.0

## Newly broken

*   checking whether package ‘RTCGA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘RTCGA’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RTCGA/new/RTCGA.Rcheck/00install.out’ for details.
    ```

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘RTCGA-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: boxplotTCGA
    > ### Title: Create Boxplots for TCGA Datasets
    > ### Aliases: boxplotTCGA
    > 
    > ### ** Examples
    > 
    > library(RTCGA.rnaseq)
    Error in library(RTCGA.rnaseq) : 
      there is no package called ‘RTCGA.rnaseq’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Complete output:
      > library(testthat)
      > library(RTCGA)
      Welcome to the RTCGA (version: 1.10.0).
      Warning message:
      replacing previous import 'dplyr::vars' by 'ggplot2::vars' when loading 'RTCGA' 
      > library(RTCGA.rnaseq)
      Error in library(RTCGA.rnaseq) : 
        there is no package called 'RTCGA.rnaseq'
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘RTCGA.rnaseq’ ‘RTCGA.clinical’ ‘RTCGA.mutations’ ‘RTCGA.RPPA’
      ‘RTCGA.mRNA’ ‘RTCGA.miRNASeq’ ‘RTCGA.methylation’ ‘RTCGA.CNV’
      ‘RTCGA.PANCAN12’
    ```

*   checking R code for possible problems ... NOTE
    ```
    availableDates: no visible binding for global variable ‘.’
    downloadTCGA: no visible binding for global variable ‘.’
    ggbiplot: no visible binding for global variable ‘xvar’
    ggbiplot: no visible binding for global variable ‘yvar’
    ggbiplot: no visible global function definition for ‘muted’
    ggbiplot: no visible binding for global variable ‘varname’
    ggbiplot: no visible binding for global variable ‘angle’
    ggbiplot: no visible binding for global variable ‘hjust’
    read.mutations: no visible binding for global variable ‘.’
    read.rnaseq: no visible binding for global variable ‘.’
    survivalTCGA: no visible binding for global variable ‘times’
    whichDateToUse: no visible binding for global variable ‘.’
    Undefined global functions or variables:
      . angle hjust muted times varname xvar yvar
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘RTCGA.rnaseq’, ‘RTCGA.clinical’, ‘RTCGA.mutations’, ‘RTCGA.CNV’, ‘RTCGA.RPPA’, ‘RTCGA.mRNA’, ‘RTCGA.miRNASeq’, ‘RTCGA.methylation’
    ```

# rtematres

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rtf

Version: 0.4-11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .forest.plot.scale: no visible global function definition for ‘axTicks’
    .forest.plot.scale : <anonymous>: no visible global function definition
      for ‘lines’
    .forest.plot.scale: no visible global function definition for ‘lines’
    .forest.plot.scale: no visible global function definition for ‘text’
    .rtf.plot: no visible global function definition for ‘png’
    .rtf.plot: no visible global function definition for ‘dev.off’
    .rtf.trellis.object: no visible global function definition for ‘png’
    .rtf.trellis.object: no visible global function definition for
      ‘dev.off’
    addSessionInfo.RTF: no visible global function definition for
      ‘sessionInfo’
    Undefined global functions or variables:
      abline arrows axTicks dev.off lines par plot png points sessionInfo
      text
    Consider adding
      importFrom("grDevices", "dev.off", "png")
      importFrom("graphics", "abline", "arrows", "axTicks", "lines", "par",
                 "plot", "points", "text")
      importFrom("utils", "sessionInfo")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rtf.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# rtide

Version: 0.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rtimes

Version: 0.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rtimicropem

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘R6’
      All declared Imports should be used.
    ```

# rtip

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rTRM

Version: 1.18.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Read 8734 items
    Read 8734 items
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'rTRM_Introduction.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# RtutoR

Version: 1.1

## In both

*   checking whether package ‘RtutoR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/RtutoR/new/RtutoR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘RtutoR’ ...
** package ‘RtutoR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘RtutoR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RtutoR/new/RtutoR.Rcheck/RtutoR’

```
### CRAN

```
* installing *source* package ‘RtutoR’ ...
** package ‘RtutoR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/RtutoR/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘RtutoR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/RtutoR/old/RtutoR.Rcheck/RtutoR’

```
# rtweet

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 113868 marked UTF-8 strings
    ```

# rubias

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ggplot2’
      All declared Imports should be used.
    ```

# rusk

Version: 0.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘rusk-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: draw
    > ### Title: Beautiful graphical representation of multiplication tables
    > ### Aliases: draw
    > 
    > ### ** Examples
    > 
    > draw(table=2,modulo = 10, label=TRUE)
    Error in if (theme$panel.ontop) { : argument is of length zero
    Calls: <Anonymous> ... ggplot_gtable.ggplot_built -> <Anonymous> -> f -> lapply -> FUN
    Execution halted
    ```

*   checking whether package ‘rusk’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘rusk’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/rusk/new/rusk.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ruv

Version: 0.9.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rvertnet

Version: 0.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rvinecopulib

Version: 0.2.8.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# rWBclimate

Version: 0.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    check_ISO_code: no visible binding for global variable ‘NoAm_country’
    check_ISO_code: no visible binding for global variable ‘SoAm_country’
    check_ISO_code: no visible binding for global variable ‘Oceana_country’
    check_ISO_code: no visible binding for global variable ‘Africa_country’
    check_ISO_code: no visible binding for global variable ‘Asia_country’
    check_ISO_code: no visible binding for global variable ‘Eur_country’
    climate_map: no visible binding for global variable ‘data’
    date_correct: no visible global function definition for ‘tail’
    Undefined global functions or variables:
      Africa_country Asia_country Eur_country NoAm_country Oceana_country
      SoAm_country data tail
    Consider adding
      importFrom("utils", "data", "tail")
    to your NAMESPACE file.
    ```

# rwty

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ryouready

Version: 0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sadists

Version: 0.2.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    `stat_bin()` using `bins = 30`. Pick better value with
    `binwidth`.
    large z may result in inaccurate quantiles
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sadists.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# saeRobust

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# saeSim

Version: 0.9.0

## Newly broken

*   checking whether package ‘saeSim’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘saeSim’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/saeSim/new/saeSim.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# samExploreR

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotsamExplorer: no visible binding for global variable ‘f’
    plotsamExplorer: no visible binding for global variable ‘value’
    plotsamExplorer: no visible binding for global variable ‘group’
    Undefined global functions or variables:
      f group value
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Sub sample N: 1
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127307_chr14.bam
    Sub sample N: 2
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127308_chr14.bam
    Sub sample N: 3
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127309_chr14.bam
    Sub sample N: 4
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127302_chr14.bam
    Sub sample N: 5
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127303_chr14.bam
    Sub sample N: 6
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127304_chr14.bam
    Sub sample N: 7
    Sub sampling: X.Users.max.github.ggplot2.revdep.library.noindex.samExploreR.RNAseqData.HNRNPC.bam.chr14.extdata.ERR127305_chr14.bam
    Sub sample N: 8
    ANOVA test for label 'New, Gene' and f values 0.85, 0.9, 0.95
    ANOVA test for labels 'New, Gene', 'Old, Gene', 'New, Exon' and f value 0.9
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'Manual.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sampleClassifier

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Attaching package: ‘DelayedArray’
    
    The following objects are masked from ‘package:matrixStats’:
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following objects are masked from ‘package:base’:
    
        aperm, apply
    
    Loading required package: hgu133plus2.db
    Loading required package: org.Hs.eg.db
    
    
    Loading required package: hgu133a.db
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sampleClassifier.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SamplerCompare

Version: 1.2.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'glue.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sand

Version: 1.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘mixer’ ‘networkTomography’
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 6 marked UTF-8 strings
    ```

# santaR

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘gridExtra’
      All declared Imports should be used.
    ```

# savR

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    buildReports,savProject-character: no visible global function
      definition for ‘dev.off’
    qualityHeatmap,savProject-integer-integer-logical: no visible global
      function definition for ‘quantile’
    Undefined global functions or variables:
      dev.off quantile
    Consider adding
      importFrom("grDevices", "dev.off")
      importFrom("stats", "quantile")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Warning: `position` is deprecated
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'savR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sbpiper

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SC3

Version: 1.8.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'SC3.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.7Mb
      sub-directories of 1Mb or more:
        data   3.0Mb
        doc    3.0Mb
    ```

# scanstatistics

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scater

Version: 1.8.0

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'read10xResults' is deprecated.
      Warning: 'downsampleCounts' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'normalizeExprs' is deprecated.
      Warning: 'read10xResults' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.8Mb
      sub-directories of 1Mb or more:
        doc       5.4Mb
        extdata   2.9Mb
        libs      4.8Mb
    ```

# scatr

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘cowplot’ ‘ggplot2’ ‘ggridges’ ‘ggstance’ ‘jmvcore’
      All declared Imports should be used.
    ```

# scatterpie

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘scatterpie-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: geom_scatterpie
    > ### Title: geom_scatterpie
    > ### Aliases: geom_scatterpie
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > d <- data.frame(x=rnorm(5), y=rnorm(5))
    > d$A <- abs(rnorm(5, sd=1))
    > d$B <- abs(rnorm(5, sd=2))
    > d$C <- abs(rnorm(5, sd=3))
    > ggplot() + geom_scatterpie(aes(x=x, y=y), data=d, cols=c("A", "B", "C")) + coord_fixed()
    Error in `[.data.frame`(data, , xvar) : undefined columns selected
    Calls: geom_scatterpie -> diff -> [ -> [.data.frame
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: ggplot2
    Quitting from lines 49-51 (scatterpie.Rmd) 
    Error: processing vignette 'scatterpie.Rmd' failed with diagnostics:
    undefined columns selected
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scDD

Version: 1.4.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    
    Attaching package: 'gridExtra'
    
    The following object is masked from 'package:Biobase':
    
        combine
    
    The following object is masked from 'package:BiocGenerics':
    
        combine
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'scDD.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘grDevices’ ‘graphics’ ‘stats’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘binomial’
    validation : <anonymous>: no visible global function definition for
      ‘var’
    validation: no visible binding for global variable ‘var’
    validation: no visible global function definition for ‘par’
    validation: no visible global function definition for ‘plot’
    validation: no visible global function definition for ‘abline’
    validation: no visible global function definition for ‘points’
    Undefined global functions or variables:
      abline axis binomial density dev.off fisher.test hcl hist ks.test
      lines lm model.matrix p.adjust par pchisq pdf plot points quantile
      rbinom rect residuals rnbinom rt runif t.test var
    Consider adding
      importFrom("grDevices", "dev.off", "hcl", "pdf")
      importFrom("graphics", "abline", "axis", "hist", "lines", "par",
                 "plot", "points", "rect")
      importFrom("stats", "binomial", "density", "fisher.test", "ks.test",
                 "lm", "model.matrix", "p.adjust", "pchisq", "quantile",
                 "rbinom", "residuals", "rnbinom", "rt", "runif", "t.test",
                 "var")
    to your NAMESPACE file.
    ```

# scdhlm

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scFeatureFilter

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.4Mb
      sub-directories of 1Mb or more:
        data   3.7Mb
        doc    2.4Mb
    ```

# SCGLR

Version: 2.0.3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'scglrVignettes.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scmamp

Version: 0.2.55

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scmap

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'scmap.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘Biobase’
      All declared Imports should be used.
    ```

# scmeth

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
     $ eval      : logi TRUE
     $ fig.align : chr "center"
    
      ordinary text without R code
    
    label: meth_dist (with options) 
    List of 7
     $ echo      : logi FALSE
     $ warning   : logi FALSE
     $ message   : logi FALSE
     $ fig.height: num 7
     $ fig.width : num 10
     $ eval      : logi TRUE
     $ fig.align : chr "center"
    
    output file: qcReport.knit.md
    
    /usr/local/bin/pandoc +RTS -K512m -RTS qcReport.utf8.md --to html4 --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash --output /Users/max/qcReport.html --email-obfuscation none --self-contained --standalone --section-divs --template /Users/max/github/ggplot2/revdep/library.noindex/scmeth/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable 'theme:bootstrap' --include-in-header /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpkEF9gQ/rmarkdown-str11135268f785f.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' 
    pandoc: /Users/max/qcReport_files/figure-html/read_metrics-1.png: openBinaryFile: does not exist (No such file or directory)
    Error: pandoc document conversion failed with error 1
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SCnorm

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in parse(text = code, keep.source = FALSE) : 
      <text>:5:19: unexpected symbol
    4:                   PrintProgressPlots = TRUE                
    5:                   FilterCellNum
                         ^
    Warning in highr::hilight(x, format, prompt = options$prompt, markup = opts$markup) :
      the syntax of the source code is invalid; the fallback mode is used
    Error in parse(text = code, keep.source = FALSE) : 
      <text>:12:19: unexpected symbol
    11:                   PrintProgressPlots = TRUE                
    12:                   FilterCellNum
                          ^
    Warning in highr::hilight(x, format, prompt = options$prompt, markup = opts$markup) :
      the syntax of the source code is invalid; the fallback mode is used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SCnorm.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scone

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sconeReport : server: no visible global function definition for ‘theme’
    sconeReport : server: no visible global function definition for
      ‘element_blank’
    sconeReport : server: no visible global function definition for
      ‘ggplotly’
    sconeReport : server: no visible global function definition for
      ‘geom_violin’
    sconeReport : server: no visible global function definition for
      ‘coord_cartesian’
    sconeReport : server: no visible global function definition for
      ‘scale_fill_manual’
    sconeReport : server: no visible global function definition for
      ‘geom_point’
    sconeReport : server: no visible global function definition for
      ‘guides’
    Undefined global functions or variables:
      %>% aes coord_cartesian element_blank geom_bar geom_point geom_violin
      ggplot ggplotly guides labs plot_ly plotlyOutput renderVisNetwork
      scale_fill_manual theme visEdges visGroups visHierarchicalLayout
      visLegend visNetwork visNetworkOutput visNetworkProxy visOptions
      visSelectNodes ylim
    ```

# Sconify

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘Sconify-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: MakeHist
    > ### Title: make.hist
    > ### Aliases: MakeHist
    > 
    > ### ** Examples
    > 
    > MakeHist(wand.final, 100, "IL7.fraction.cond.2", "fraction IL7")
    Error: Column `x` must be a 1d atomic vector or a list
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 47-54 (DataQuality.Rmd) 
    Error: processing vignette 'DataQuality.Rmd' failed with diagnostics:
    Column `x` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scorecard

Version: 0.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# scorecardModelUtils

Version: 0.0.0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SCORPIUS

Version: 1.0

## Newly broken

*   checking whether package ‘SCORPIUS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SCORPIUS’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SCORPIUS/new/SCORPIUS.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rcpp’ ‘testthat’
      All declared Imports should be used.
    ```

# scPipe

Version: 1.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Title: Plot GGAlly pairs plot of QC statistics from
    > ###   'SingleCellExperiment' object
    > ### Aliases: plot_QC_pairs
    > 
    > ### ** Examples
    > 
    > data("sc_sample_data")
    > data("sc_sample_qc")
    > sce = SingleCellExperiment(assays = list(counts = as.matrix(sc_sample_data)))
    > organism(sce) = "mmusculus_gene_ensembl"
    > gene_id_type(sce) = "ensembl_gene_id"
    > QC_metrics(sce) = sc_sample_qc
    > demultiplex_info(sce) = cell_barcode_matching
    > UMI_dup_info(sce) = UMI_duplication
    > sce = detect_outlier(sce)
    > 
    > plot_QC_pairs(sce)
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~total_count_per_cell", "~outliers".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~total_count_per_cell", "~outliers".  Please rename the columns or make a new column.
    Calls: plot_QC_pairs ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 66-73 (scPipe_tutorial.Rmd) 
    Error: processing vignette 'scPipe_tutorial.Rmd' failed with diagnostics:
    there is no package called 'Rsubread'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plot_demultiplex: no visible binding for global variable ‘status’
    plot_demultiplex: no visible binding for global variable ‘count’
    plot_demultiplex: no visible binding for global variable ‘label_y’
    plot_demultiplex: no visible binding for global variable ‘label_tx’
    Undefined global functions or variables:
      count label_tx label_y status
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    '::' or ':::' import not declared from: ‘Rsubread’
    ```

# scran

Version: 1.8.1

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'exploreData' is deprecated.
      Warning: 'selectorPlot' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        libs   5.0Mb
    ```

# scsR

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘STRINGdb’ ‘BiocGenerics’ ‘Biostrings’ ‘IRanges’ ‘plyr’ ‘tcltk’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    seed_correction_pooled: no visible global function definition for
      ‘txtProgressBar’
    seed_correction_pooled: no visible global function definition for
      ‘setTxtProgressBar’
    seed_removal: no visible global function definition for
      ‘txtProgressBar’
    seed_removal: no visible global function definition for
      ‘setTxtProgressBar’
    seeds_analysis : <anonymous>: no visible global function definition for
      ‘ks.test’
    seeds_analysis: no visible global function definition for ‘phyper’
    transcribe_seqs: no visible global function definition for
      ‘txtProgressBar’
    transcribe_seqs: no visible global function definition for
      ‘setTxtProgressBar’
    Undefined global functions or variables:
      cor.test heatmap.2 ks.test lm phyper setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("stats", "cor.test", "ks.test", "lm", "phyper")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    The following object is masked from ‘package:IRanges’:
    
        desc
    
    The following object is masked from ‘package:S4Vectors’:
    
        rename
    
    Loading required package: tcltk
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    
    Attaching package: ‘scsR’
    
    The following objects are masked from ‘package:STRINGdb’:
    
        delColDf, renameColDf
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'scsR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# SDaA

Version: 0.1-3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    lahiri.design: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Loading required package: grid
    Loading required package: Matrix
    Loading required package: survival
    
    Attaching package: ‘survey’
    
    The following object is masked from ‘package:graphics’:
    
        dotchart
    
    
    Attaching package: ‘ggplot2’
    
    The following object is masked from ‘package:SDaA’:
    
        seals
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SDaA_using_survey.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sdcMicro

Version: 5.2.0

## In both

*   checking examples ... ERROR
    ```
    ...
    3 0.18    1 0.80    2    1    1    8  39.0
    4 1.90    3 9.00    3    1    5   91  17.0
    5 1.00    4 1.30    3    1    4   13 541.0
    6 1.00    4 1.40    3    1    1   14   8.0
    7 0.10    6 0.01    2    1    5    1   5.0
    8 0.15    1 0.50    2    5    1    5  92.0
    > f$fk
    [1] 2 2 2 1 1 1 1 2
    > f$Fk
    [1] 110.0  84.5  84.5  17.0 541.0   8.0   5.0 110.0
    > ## with missings:
    > x <- francdat
    > x[3,5] <- NA
    > x[4,2] <- x[4,4] <- NA
    > x[5,6]  <- NA
    > x[6,2]  <- NA
    > f2 <- freqCalc(x, keyVars=c(2,4,5,6),w=8)
    Error in `[.data.table`(dat_with_na, , `:=`(tmpid, .I)) : 
      Cannot use := to add columns to a null data.table (no columns), currently. You can use := to add (empty) columns to a 0-row data.table (1 or more empty columns), though.
    Calls: freqCalc -> [ -> [.data.table
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testhat.R’ failed.
    Last 13 lines of output:
      6 0.0 0 0 1.00 0.00
      9 0.0 0 0 0.00 1.00
      
      Changed observations:
        variable nrChanges percChanges
      1     roof         1        1.08
      ---------------------------------------------------------------------- 
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 5 SKIPPED: 5 FAILED: 2
      1. Error: data frame method gives the same as method on sdcMicro object (@test_freqCalc.R#7) 
      2. Error: method1 frequencies should be 2,2,2,2,1 (@test_freqCalc.R#110) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sdc_guidelines.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SDEFSR

Version: 0.7.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    ! Undefined control sequence.
    l.1326 \textbackslash{}providecommand\{\doi
                                               \}\{doi: 
    Here is how much of TeX's memory you used:
     12870 strings out of 494923
     188508 string characters out of 6180211
     283863 words of memory out of 5000000
     15725 multiletter control sequences out of 15000+600000
     64247 words of font info for 121 fonts, out of 8000000 for 9000
     14 hyphenation exceptions out of 8191
     32i,7n,38p,680b,470s stack positions out of 5000i,500n,10000p,200000b,80000s
    
    Error: processing vignette 'SDEFSRpackage.Rmd' failed with diagnostics:
    Failed to compile SDEFSRpackage.tex. See SDEFSRpackage.log for more info.
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘grDevices’
      All declared Imports should be used.
    ```

# SDLfilter

Version: 1.1.0

## In both

*   checking whether package ‘SDLfilter’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SDLfilter/new/SDLfilter.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘SDLfilter’ ...
** package ‘SDLfilter’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘SDLfilter’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SDLfilter/new/SDLfilter.Rcheck/SDLfilter’

```
### CRAN

```
* installing *source* package ‘SDLfilter’ ...
** package ‘SDLfilter’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘SDLfilter’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SDLfilter/old/SDLfilter.Rcheck/SDLfilter’

```
# sdmpredictors

Version: 0.2.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sdmvspecies

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Error in re-building vignettes:
      ...
    Loading required package: sp
    [1] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio1.bil" 
    [2] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio11.bil"
    [3] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio12.bil"
    [4] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio14.bil"
    [5] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio16.bil"
    [6] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio5.bil" 
    [7] "/Users/max/github/ggplot2/revdep/checks.noindex/sdmvspecies/new/sdmvspecies.Rcheck/sdmvspecies/external/env/bio7.bil" 
    
    Attaching package: ‘ggplot2’
    
    The following object is masked from ‘package:raster’:
    
        calc
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sdmvspecies.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# seaaroundus

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rgdal’
      All declared Imports should be used.
    ```

# season

Version: 0.3-5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sinusoid: no visible global function definition for ‘par’
    sinusoid: no visible global function definition for ‘plot’
    sinusoid: no visible global function definition for ‘lines’
    sinusoid: no visible global function definition for ‘axis’
    summary.monthglm: no visible global function definition for ‘qnorm’
    wtest: no visible global function definition for ‘qchisq’
    wtest: no visible global function definition for ‘pchisq’
    Undefined global functions or variables:
      acf as.formula axis box cpgram fitted gaussian glm gray hist lines
      median model.frame na.omit par pchisq plot points polygon qchisq
      qnorm quantile relevel resid residuals rgamma rnorm rug runif sd text
      time
    Consider adding
      importFrom("grDevices", "gray")
      importFrom("graphics", "axis", "box", "hist", "lines", "par", "plot",
                 "points", "polygon", "rug", "text")
      importFrom("stats", "acf", "as.formula", "cpgram", "fitted",
                 "gaussian", "glm", "median", "model.frame", "na.omit",
                 "pchisq", "qchisq", "qnorm", "quantile", "relevel", "resid",
                 "residuals", "rgamma", "rnorm", "runif", "sd", "time")
    to your NAMESPACE file.
    ```

# SEERaBomb

Version: 2018.1

## Newly broken

*   checking whether package ‘SEERaBomb’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SEERaBomb’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SEERaBomb/new/SEERaBomb.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# seewave

Version: 2.1.0

## In both

*   checking whether package ‘seewave’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/new/seewave.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rpanel’
    ```

## Installation

### Devel

```
* installing *source* package ‘seewave’ ...
** package ‘seewave’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c seewave_init.c -o seewave_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c stft.c -o stft.o
stft.c:3:11: fatal error: 'sndfile.h' file not found
# include <sndfile.h> //Library for write or read sound file
          ^~~~~~~~~~~
1 error generated.
make: *** [stft.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘seewave’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/new/seewave.Rcheck/seewave’

```
### CRAN

```
* installing *source* package ‘seewave’ ...
** package ‘seewave’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c seewave_init.c -o seewave_init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG   -I/usr/local/include   -fPIC  -Wall -g -O2  -c stft.c -o stft.o
stft.c:3:11: fatal error: 'sndfile.h' file not found
# include <sndfile.h> //Library for write or read sound file
          ^~~~~~~~~~~
1 error generated.
make: *** [stft.o] Error 1
make: *** Waiting for unfinished jobs....
ERROR: compilation failed for package ‘seewave’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/seewave/old/seewave.Rcheck/seewave’

```
# segclust2d

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# selfea

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    get_statistics_from_file: no visible global function definition for
      ‘read.csv’
    glm_anova: no visible global function definition for ‘aov’
    glm_anova: no visible global function definition for ‘glm’
    glm_anova: no visible global function definition for ‘gaussian’
    glm_anova: no visible global function definition for ‘anova’
    glm_anova: no visible global function definition for ‘quasipoisson’
    glm_anova: no visible global function definition for ‘sd’
    glm_anova: no visible global function definition for ‘p.adjust’
    ttest_cohens_d: no visible global function definition for ‘t.test’
    ttest_cohens_d: no visible global function definition for ‘sd’
    Undefined global functions or variables:
      anova aov gaussian glm p.adjust quasipoisson read.csv sd t.test
    Consider adding
      importFrom("stats", "anova", "aov", "gaussian", "glm", "p.adjust",
                 "quasipoisson", "sd", "t.test")
      importFrom("utils", "read.csv")
    to your NAMESPACE file.
    ```

# sensiPhy

Version: 0.8.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sensitivity

Version: 1.15.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SensMap

Version: 0.6

## In both

*   checking whether package ‘SensMap’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SensMap/new/SensMap.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘SensMap’ ...
** package ‘SensMap’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SensMap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SensMap/new/SensMap.Rcheck/SensMap’

```
### CRAN

```
* installing *source* package ‘SensMap’ ...
** package ‘SensMap’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SensMap/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SensMap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SensMap/old/SensMap.Rcheck/SensMap’

```
# SensMixed

Version: 2.1-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SensoMineR

Version: 1.23

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SensusR

Version: 2.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    [1] "93% done merging data for SpeedDatum (15 of 16)."
    [1] "100% done merging data for SpeedDatum (16 of 16)."
    [1] "Creating data frame for SpeedDatum."
    [1] "100% done merging data for TelephonyDatum (1 of 1)."
    [1] "Creating data frame for TelephonyDatum."
    [1] "14% done merging data for WlanDatum (1 of 7)."
    [1] "28% done merging data for WlanDatum (2 of 7)."
    [1] "42% done merging data for WlanDatum (3 of 7)."
    [1] "57% done merging data for WlanDatum (4 of 7)."
    [1] "71% done merging data for WlanDatum (5 of 7)."
    [1] "85% done merging data for WlanDatum (6 of 7)."
    [1] "100% done merging data for WlanDatum (7 of 7)."
    [1] "Creating data frame for WlanDatum."
    > plot(data$LocationDatum)
    Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=38.0676352725243,-78.9510441850485&zoom=10&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=38.0676352725243,-78.9510441850485&sensor=false
    Warning: geocode failed with status OVER_QUERY_LIMIT, location = "38.0676352725243,-78.9510441850485"
    Error in data.frame(ll.lat = ll[1], ll.lon = ll[2], ur.lat = ur[1], ur.lon = ur[2]) : 
      arguments imply differing number of rows: 0, 1
    Calls: plot ... <Anonymous> -> ggmap -> get_map -> get_googlemap -> data.frame
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SentimentAnalysis

Version: 1.3-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘SnowballC’ ‘XML’ ‘mgcv’
      All declared Imports should be used.
    ```

# sentimentr

Version: 2.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sentometrics

Version: 0.3.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5374 marked UTF-8 strings
    ```

# SEPA

Version: 1.10.0

## In both

*   R CMD check timed out
    

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘t.test’
    truetimepattern : <anonymous>: no visible global function definition
      for ‘p.adjust’
    truetimevisualize : <anonymous> : <anonymous>: no visible global
      function definition for ‘median’
    truetimevisualize: no visible binding for global variable ‘time’
    truetimevisualize: no visible binding for global variable ‘expmean’
    truetimevisualize: no visible binding for global variable ‘Gene’
    windowGOanalysis: no visible global function definition for ‘new’
    windowGOvisualize: no visible binding for global variable ‘Var1’
    windowGOvisualize: no visible binding for global variable ‘value’
    Undefined global functions or variables:
      Gene Var1 Var2 confint expmean fitted lines lm median new p.adjust
      pseudotime t.test time value xend yend
    Consider adding
      importFrom("graphics", "lines")
      importFrom("methods", "new")
      importFrom("stats", "confint", "fitted", "lm", "median", "p.adjust",
                 "t.test", "time")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# seqbias

Version: 1.28.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Versioned 'LinkingTo' value for ‘Rsamtools’ is only usable in R >= 3.0.2
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘GenomicRanges’ which was already attached by Depends.
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘Biostrings’ ‘GenomicRanges’ ‘methods’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      function definition for ‘runif’
    random.intervals: no visible global function definition for ‘GRanges’
    random.intervals: no visible global function definition for ‘IRanges’
    seqbias.fit: no visible global function definition for ‘new’
    seqbias.load: no visible global function definition for ‘new’
    seqbias.predict: no visible global function definition for ‘is’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘seqnames’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘start’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘end’
    seqbias.predict : <anonymous>: no visible global function definition
      for ‘strand’
    Undefined global functions or variables:
      GRanges IRanges end is new runif seqnames start strand
    Consider adding
      importFrom("methods", "is", "new")
      importFrom("stats", "end", "runif", "start")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking compiled code ... NOTE
    ```
    File ‘seqbias/libs/seqbias.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Objects: ‘sequencing_bias.o’, ‘twobitseq.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following object is masked from ‘package:base’:
    
        expand.grid
    
    Loading required package: IRanges
    Loading required package: GenomeInfoDb
    Loading required package: Biostrings
    Loading required package: XVector
    
    Attaching package: ‘Biostrings’
    
    The following object is masked from ‘package:base’:
    
        strsplit
    
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'overview.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# seqCAT

Version: 1.2.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘seqCAT-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot_impacts
    > ### Title: Plot SNV impact distribution
    > ### Aliases: plot_impacts
    > 
    > ### ** Examples
    > 
    > # Load test comparison data
    > data(test_comparison)
    > 
    > # Plot the impact distribution
    > impacts <- plot_impacts(test_comparison)
    Error in grouped_indices_grouped_df_impl(.data) : 
      `.data` is a corrupt grouped_df, contains 8 rows, and 3 rows in groups
    Calls: plot_impacts ... group_indices.grouped_df -> grouped_indices_grouped_df_impl
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      target is NULL, current is character
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 81 SKIPPED: 0 FAILED: 8
      1. Error: (unknown) (@test_06_plot_impacts.R#8) 
      2. Failure: samples are clustered correctly (@test_08_plot_heatmap.R#18) 
      3. Failure: samples are clustered correctly (@test_08_plot_heatmap.R#19) 
      4. Failure: samples are clustered correctly (@test_08_plot_heatmap.R#20) 
      5. Failure: correct variant/gene combinations are being plotted (@test_11_plot_variant_list.R#13) 
      6. Failure: correct variant/gene combinations are being plotted (@test_11_plot_variant_list.R#14) 
      7. Failure: correct variant/gene combinations are being plotted (@test_11_plot_variant_list.R#15) 
      8. Failure: correct variant/gene combinations are being plotted (@test_11_plot_variant_list.R#16) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Creating SNV profile ...
    Created and stored SNV profile for "HCT116" in [hct116_profile.txt].
    Reading VCF file ...
    Creating SNV profile ...
    Created and stored SNV profile for "RKO" in [rko_profile.txt].
    Reading COSMIC cell line data ...
    Reading COSMIC cell line data ...
    Reading profile for HCT116 in file hct116_profile.txt ...
    Reading profile for RKO in file rko_profile.txt ...
    Comparing HCT116 and RKO ...
    Comparing HCT116 and RKO ...
    Comparing HCT116 and COSMIC.HCT116 ...
    Reading VCF file ...
    Creating SNV profile ...
    Created and stored SNV profile for "HKE3" in [hke3_profile.txt].
    Reading profile for HKE3 in file hke3_profile.txt ...
    Comparing HCT116 and HKE3 ...
    Quitting from lines 343-345 (seqCAT.Rmd) 
    Error: processing vignette 'seqCAT.Rmd' failed with diagnostics:
    `.data` is a corrupt grouped_df, contains 8 rows, and 7 rows in groups
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# seqcombo

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SeqFeatR

Version: 0.2.6

## In both

*   checking whether package ‘SeqFeatR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/new/SeqFeatR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘SeqFeatR’ ...
** package ‘SeqFeatR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SeqFeatR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/new/SeqFeatR.Rcheck/SeqFeatR’

```
### CRAN

```
* installing *source* package ‘SeqFeatR’ ...
** package ‘SeqFeatR’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SeqFeatR/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘SeqFeatR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SeqFeatR/old/SeqFeatR.Rcheck/SeqFeatR’

```
# seqplots

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 14.8Mb
      sub-directories of 1Mb or more:
        doc        3.7Mb
        seqplots  10.1Mb
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("BWGFile_summary", ..., PACKAGE = "rtracklayer")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotHeatmap,list: no visible global function definition for ‘kmeans’
    plotHeatmap,list: no visible global function definition for ‘hclust’
    plotHeatmap,list: no visible global function definition for ‘dist’
    plotHeatmap,list: no visible global function definition for ‘cutree’
    plotHeatmap,list: no visible global function definition for
      ‘as.dendrogram’
    plotHeatmap,list: no visible global function definition for ‘title’
    Undefined global functions or variables:
      Var1 Var2 abline adjustcolor approx as.dendrogram axis box
      capture.output colorRampPalette cutree dist hclust image kmeans
      layout lines mtext par plot.new qt rainbow rect rgb text title value
    Consider adding
      importFrom("grDevices", "adjustcolor", "colorRampPalette", "rainbow",
                 "rgb")
      importFrom("graphics", "abline", "axis", "box", "image", "layout",
                 "lines", "mtext", "par", "plot.new", "rect", "text",
                 "title")
      importFrom("stats", "approx", "as.dendrogram", "cutree", "dist",
                 "hclust", "kmeans", "qt")
      importFrom("utils", "capture.output")
    to your NAMESPACE file.
    ```

# seqsetvis

Version: 1.0.1

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    The following object is masked from 'package:IRanges':
    
        shift
    
    The following objects are masked from 'package:S4Vectors':
    
        first, second
    
    
    Attaching package: 'cowplot'
    
    The following object is masked from 'package:ggplot2':
    
        ggsave
    
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `seqsetvis_overview_files/figure-html/binary heatmap-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Quitting from lines 306-314 (seqsetvis_overview.Rmd) 
    Error: processing vignette 'seqsetvis_overview.Rmd' failed with diagnostics:
    More than one expression parsed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SeqSQC

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.5Mb
      sub-directories of 1Mb or more:
        doc       1.9Mb
        extdata   3.3Mb
    ```

# Seurat

Version: 2.3.1

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      
        |                                                                            
        |                                                                      |   0%
        |                                                                            
        |======================================================================| 100%
      Running CCA
      Computing nearest neighbor graph
      Computing SNN
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 280 SKIPPED: 0 FAILED: 2
      1. Failure: tSNE plots correctly (@test_seurat_object.R#115) 
      2. Error: tSNE plots correctly (@test_seurat_object.R#116) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘loomR’
    ```

# sf

Version: 0.6-2

## In both

*   checking whether package ‘sf’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/new/sf.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
configure: CC: /usr/local/clang4/bin/clang
configure: CXX: /usr/local/clang4/bin/clang++
checking for gdal-config... no
no
configure: error: gdal-config not found or not executable.
ERROR: configuration failed for package ‘sf’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/new/sf.Rcheck/sf’

```
### CRAN

```
* installing *source* package ‘sf’ ...
** package ‘sf’ successfully unpacked and MD5 sums checked
configure: CC: /usr/local/clang4/bin/clang
configure: CXX: /usr/local/clang4/bin/clang++
checking for gdal-config... no
no
configure: error: gdal-config not found or not executable.
ERROR: configuration failed for package ‘sf’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sf/old/sf.Rcheck/sf’

```
# sglr

Version: 0.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .computePStar: no visible global function definition for ‘uniroot’
    plotBoundary: no visible global function definition for
      ‘scale_y_continuous’
    Undefined global functions or variables:
      scale_y_continuous uniroot
    Consider adding
      importFrom("stats", "uniroot")
    to your NAMESPACE file.
    ```

# sgmcmc

Version: 0.2.2

## In both

*   checking S3 generic/method consistency ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    See section ‘Generic functions and methods’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking replacement functions ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    The argument of a replacement function which corresponds to the right
    hand side must be named ‘value’.
    ```

*   checking for missing documentation entries ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    All user-level objects in a package should have documentation entries.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking foreign function calls ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    ```

*   checking Rd \usage sections ... NOTE
    ```
    /Users/max/.virtualenvs/r-tensorflow/lib/python2.7/site-packages/h5py/__init__.py:36: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
      from ._conv import register_converters as _register_converters
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# sGMRFmix

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# shadowtext

Version: 0.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# shazam

Version: 0.1.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 2 marked UTF-8 strings
    ```

# SHELF

Version: 1.3.0

## Newly broken

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 187-195 (Multivariate-normal-copula.Rmd) 
    Error: processing vignette 'Multivariate-normal-copula.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~X1".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# shiny

Version: 1.0.5

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/test-all.R’ failed.
    Last 13 lines of output:
      Component "y": 1 string mismatch
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 615 SKIPPED: 3 FAILED: 8
      1. Failure: ggplot coordmap (@test-plot-coordmap.R#34) 
      2. Failure: ggplot coordmap (@test-plot-coordmap.R#63) 
      3. Failure: ggplot coordmap (@test-plot-coordmap.R#73) 
      4. Failure: ggplot coordmap with facet_wrap (@test-plot-coordmap.R#106) 
      5. Failure: ggplot coordmap with facet_grid (@test-plot-coordmap.R#155) 
      6. Failure: ggplot coordmap with facet_grid (@test-plot-coordmap.R#192) 
      7. Failure: ggplot coordmap with 2D facet_grid (@test-plot-coordmap.R#243) 
      8. Failure: ggplot coordmap with various scales and coords (@test-plot-coordmap.R#336) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.3Mb
      sub-directories of 1Mb or more:
        R     1.0Mb
        www   6.4Mb
    ```

# ShinyItemAnalysis

Version: 1.2.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘DT’ ‘data.table’ ‘gridExtra’ ‘knitr’ ‘latticeExtra’ ‘msm’ ‘plotly’
      ‘shinyBS’ ‘xtable’
      All declared Imports should be used.
    ```

# shinyjqui

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# shinyKGode

Version: 1.0.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘KGode’ ‘XML’ ‘ggplot2’ ‘gridExtra’ ‘mvtnorm’ ‘reshape2’ ‘shinyjs’
      All declared Imports should be used.
    ```

# shinystan

Version: 2.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘brms’
    ```

# SIBER

Version: 2.1.3

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > x <- stats::rnorm(50)
    > y <- stats::rnorm(50)
    > parms <- list()
    > parms$n.iter <- 2 * 10^3
    > parms$n.burnin <- 500
    > parms$n.thin <- 2     
    > parms$n.chains <- 2    
    > priors <- list()
    > priors$R <- 1 * diag(2)
    > priors$k <- 2
    > priors$tau.mu <- 1.0E-3
    > fitEllipse(x, y, parms, priors)
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 74-93 (Centroid-Vectors.Rmd) 
    Error: processing vignette 'Centroid-Vectors.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/SIBER/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘viridis’
      All declared Imports should be used.
    ```

# sicegar

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# sidier

Version: 4.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SigFuge

Version: 1.18.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    SFfigure: no visible global function definition for ‘hcl’
    SFfigure: no visible binding for global variable ‘cluster’
    SFfigure: no visible binding for global variable ‘median’
    SFfigure: no visible binding for global variable ‘x’
    SFfigure: no visible binding for global variable ‘y1’
    SFfigure: no visible binding for global variable ‘label’
    SFfigure: no visible binding for global variable ‘value’
    SFfigure: no visible binding for global variable ‘Clusters’
    SFfigure: no visible binding for global variable ‘y2’
    SFfigure: no visible global function definition for ‘pdf’
    SFfigure: no visible global function definition for ‘dev.off’
    SFlabels: no visible global function definition for ‘kmeans’
    SFnormalize : <anonymous>: no visible global function definition for
      ‘median’
    Undefined global functions or variables:
      Clusters base cluster dev.off e hcl kmeans label median pdf rgb s
      value x y1 y2
    Consider adding
      importFrom("grDevices", "dev.off", "hcl", "pdf", "rgb")
      importFrom("stats", "kmeans", "median")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    
    Loading required package: GenomicFeatures
    'select()' returned 1:1 mapping between keys and columns
    Loading required package: Biostrings
    Loading required package: XVector
    
    Attaching package: ‘Biostrings’
    
    The following object is masked from ‘package:base’:
    
        strsplit
    
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SigFuge.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sights

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sigmoid

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SIMAT

Version: 1.12.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Name: plotEIC
    > ### Title: Plotting EIC of one peak
    > ### Aliases: plotEIC
    > 
    > ### ** Examples
    > 
    >     # load an RData file including a single run data acquired by readCDF
    >     data("Run")
    >     
    >     # load targets information
    >     data(Targets)
    >     
    >     # get all the corresponding peaks of the target list
    >     runPeaks <- getPeak(Run = Run, Targets = Targets)
    >     
    >     # plot the EIC of the first target
    >     plotEIC(runPeaks[[1]])
    Error in if (ggplot_build(mainPlot)$layout$panel_ranges[[1]]$y.range[2] >  : 
      argument is of length zero
    Calls: plotEIC
    Execution halted
    ```

## Newly fixed

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
      Warning: working directory was changed to ‘/Users/max/github/ggplot2/revdep/checks.noindex/SIMAT/old/SIMAT.Rcheck/SIMAT/extdata’, resetting
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    getRIStandard: no visible global function definition for ‘read.csv’
    plotEIC: no visible global function definition for ‘stack’
    plotEIC: no visible global function definition for ‘dnorm’
    plotEIC: no visible global function definition for ‘dev.copy’
    plotEIC: no visible binding for global variable ‘pdf’
    plotEIC: no visible global function definition for ‘dev.off’
    plotTIC: no visible binding for global variable ‘rt’
    Undefined global functions or variables:
      dev.copy dev.off dnorm pdf read.csv rt stack
    Consider adding
      importFrom("grDevices", "dev.copy", "dev.off", "pdf")
      importFrom("stats", "dnorm", "rt")
      importFrom("utils", "read.csv", "stack")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: Rcpp
    
    Error: processing vignette 'SIMAT-vignette.Rnw' failed with diagnostics:
     chunk 9 (label = plotEIC) 
    Error in if (ggplot_build(mainPlot)$layout$panel_ranges[[1]]$y.range[2] >  : 
      argument is of length zero
    Execution halted
    ```

# simcausal

Version: 0.5.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Attaching package: 'ggplot2'
    
    The following object is masked _by_ '.GlobalEnv':
    
        %+%
    
    Loading required package: lattice
    Loading required package: survival
    Loading required package: Formula
    
    Attaching package: 'Hmisc'
    
    The following objects are masked from 'package:base':
    
        format.pval, units
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'simcausal_vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SimCorrMix

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘MASS’ ‘grid’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘PoisNor’
    ```

# SimDesign

Version: 1.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘doMPI’
    ```

# simglm

Version: 0.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# simmr

Version: 0.3

## In both

*   checking whether package ‘simmr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/new/simmr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/new/simmr.Rcheck/simmr’

```
### CRAN

```
* installing *source* package ‘simmr’ ...
** package ‘simmr’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rjags’:
 .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/simmr/rjags/libs/rjags.so
  Reason: image not found
Error : package ‘rjags’ could not be loaded
ERROR: lazy loading failed for package ‘simmr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/simmr/old/simmr.Rcheck/simmr’

```
# SimMultiCorrData

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘BinNonNor’, ‘PoisNor’, ‘PoisBinOrdNor’, ‘PoisBinOrdNonNor’
    ```

# simPH

Version: 1.3.10

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Converting "begin" to numeric. Things might get wacky. Please check.
    Converting "end" to numeric. Things might get wacky. Please check.
    Converting "event" to numeric. Things might get wacky. Please check.
    
    Expanding data.
    
    Keeping only needed observations.
    
    Doing a final clean up.
    All Xl set to 0.
    All Xl set to 0.
    `geom_smooth()` using method = 'gam' and formula 'y ~ mgcv::s(x, bs = "cs")'
    Warning: Computation failed in `stat_smooth()`:
    invalid type (list) for variable 'mgcv::s(x, bs = "cs")'
    `geom_smooth()` using method = 'gam' and formula 'y ~ mgcv::s(x, bs = "cs")'
    Warning: Computation failed in `stat_smooth()`:
    invalid type (list) for variable 'mgcv::s(x, bs = "cs")'
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'simPH-overview.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SimRepeat

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BB’ ‘MASS’ ‘ggplot2’ ‘grid’ ‘triangle’
      All declared Imports should be used.
    ```

# simstudy

Version: 0.1.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# simTool

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# simulator

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sincell

Version: 1.12.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        union, unique, unsplit, which, which.max, which.min
    
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Loading required package: ggplot2
    Loading required package: VGAM
    Loading required package: stats4
    Loading required package: splines
    Loading required package: DDRTree
    Loading required package: irlba
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/EigenValues-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/plot_cellsLowDimensionalSpace_ICA-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `figure/Comparisson_graphs-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sincell-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sc_InitializingSincellObject: no visible binding for global variable
      ‘var’
    sc_StatisticalSupportByGeneSubsampling: no visible global function
      definition for ‘cor’
    sc_StatisticalSupportByReplacementWithInSilicoCellsReplicates: no
      visible global function definition for ‘cor’
    sc_clusterObj: no visible global function definition for ‘hclust’
    sc_clusterObj: no visible global function definition for ‘cutree’
    sc_distanceObj: no visible global function definition for ‘cor’
    sc_marker2color: no visible global function definition for
      ‘colorRampPalette’
    Undefined global functions or variables:
      cmdscale colorRampPalette combn cor cutree hclust plot prcomp
      quantile rnbinom rnorm runif var
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("graphics", "plot")
      importFrom("stats", "cmdscale", "cor", "cutree", "hclust", "prcomp",
                 "quantile", "rnbinom", "rnorm", "runif", "var")
      importFrom("utils", "combn")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘sincell/libs/sincell.so’:
      Found ‘_rand’, possibly from ‘rand’ (C)
        Objects: ‘pseudoreplicatesbymodel.o’, ‘pseudoreplicatesbynoise.o’,
          ‘pseudoreplicatesbynoise_cv2.o’
      Found ‘_srand’, possibly from ‘srand’ (C)
        Objects: ‘pseudoreplicatesbymodel.o’, ‘pseudoreplicatesbynoise.o’,
          ‘pseudoreplicatesbynoise_cv2.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

# Single.mTEC.Transcriptomes

Version: 1.8.0

## In both

*   checking data for ASCII and uncompressed saves ... WARNING
    ```
      Warning: package needs dependence on R (>= 2.10)
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Warning: Removed 8 rows containing non-finite values (stat_ydensity).
    Warning: Removed 8 rows containing non-finite values (stat_boxplot).
    Warning: Removed 3 rows containing non-finite values (stat_ydensity).
    Warning: Removed 3 rows containing non-finite values (stat_boxplot).
    Warning: Removed 28 rows containing missing values (geom_point).
    Need specific help about ggbio? try mailing 
     the maintainer or visit http://tengfei.github.com/ggbio/
    
    Attaching package: 'ggbio'
    
    The following objects are masked from 'package:ggplot2':
    
        geom_bar, geom_rect, geom_segment, ggsave, stat_bin,
        stat_identity, xlim
    
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Quitting from lines 2626-2642 (mTECs.Rnw) 
    Error: processing vignette 'mTECs.Rnw' failed with diagnostics:
    invalid 'type' (list) of argument
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 912.4Mb
      sub-directories of 1Mb or more:
        data  903.0Mb
        doc     9.3Mb
    ```

# singleCellTK

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        data   1.0Mb
        doc    3.7Mb
    ```

# singscore

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SISPA

Version: 1.10.0

## In both

*   checking files in ‘vignettes’ ... WARNING
    ```
    Files in the 'vignettes' directory but no files in 'inst/doc':
      ‘SISPA.Rmd’, ‘SISPA.md’, ‘SISPA_data.Rda’,
        ‘SISPA_files/figure-html/unnamed-chunk-2-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-3-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-4-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-5-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-6-1.png’,
        ‘SISPA_files/figure-html/unnamed-chunk-7-1.png’,
        ‘sispa_overview.png’
    Files named as vignettes but with no recognized vignette engine:
       ‘vignettes/SISPA.Rmd’
    (Is a VignetteBuilder field missing?)
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    cptSamples : cptsPlot: no visible global function definition for
      ‘abline’
    cptSamples : cptsPlot: no visible global function definition for ‘text’
    Undefined global functions or variables:
      abline text
    Consider adding
      importFrom("graphics", "abline", "text")
    to your NAMESPACE file.
    ```

# sitmo

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SixSigma

Version: 0.9-52

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'HelicopterInstructions.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# sjlabelled

Version: 1.0.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sjmisc

Version: 2.7.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sjPlot

Version: 2.4.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > # grouped coefficients
    > plot_model(m, group.terms = c(1, 2, 3, 3, 3, 4, 4))
    > 
    > # multiple plots, as returned from "diagnostic"-plot type,
    > # can be arranged with 'plot_grid()'
    > ## Not run: 
    > ##D p <- plot_model(m, type = "diag")
    > ##D plot_grid(p)
    > ## End(Not run)
    > 
    > # plot random effects
    > library(lme4)
    Loading required package: Matrix
    > m <- lmer(Reaction ~ Days + (Days | Subject), sleepstudy)
    > plot_model(m, type = "re")
    > 
    > # plot marginal effects
    > plot_model(m, type = "eff", terms = "Days")
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to linetype
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning: replacing previous import 'dplyr::vars' by 'ggplot2::vars' when loading 'merTools'
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 36-49 (plot_interactions.Rmd) 
    Error: processing vignette 'plot_interactions.Rmd' failed with diagnostics:
    A continuous variable can not be mapped to linetype
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘plm’
    ```

# sjstats

Version: 0.14.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘piecewiseSEM’
    ```

# skynet

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maps’
      All declared Imports should be used.
    ```

# slackr

Version: 1.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# slalom

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'vignette.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 23.9Mb
      sub-directories of 1Mb or more:
        data      4.0Mb
        extdata  18.3Mb
    ```

# slim

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SmartEDA

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘ISLR’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘InformationValue’, ‘DataExplorer’
    ```

# SmarterPoland

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.2Mb
      sub-directories of 1Mb or more:
        data   8.1Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1122 marked UTF-8 strings
    ```

# SMFI5

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls to packages already attached by Depends:
      ‘ggplot2’ ‘reshape’
      Please remove these calls from your code.
    Packages in Depends field not imported from:
      ‘corpcor’ ‘ggplot2’ ‘reshape’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    sim.cir: no visible global function definition for ‘aes’
    sim.cir: no visible global function definition for ‘geom_line’
    sim.cir: no visible global function definition for ‘ggtitle’
    sim.n.chi2: no visible global function definition for ‘rchisq’
    sim.n.chi2: no visible global function definition for ‘rnorm’
    sim.n.chi2: no visible global function definition for ‘rpois’
    sim.vasicek: no visible global function definition for ‘rnorm’
    sim.vasicek: no visible global function definition for ‘melt’
    sim.vasicek: no visible global function definition for ‘ggplot’
    sim.vasicek: no visible global function definition for ‘aes’
    sim.vasicek: no visible global function definition for ‘geom_line’
    sim.vasicek: no visible global function definition for ‘ggtitle’
    Undefined global functions or variables:
      acf aes cor cov dchisq geom_line ggplot ggtitle head melt optim plot
      pseudoinverse qnorm rchisq rnorm rpois sd title
    Consider adding
      importFrom("graphics", "plot", "title")
      importFrom("stats", "acf", "cor", "cov", "dchisq", "optim", "qnorm",
                 "rchisq", "rnorm", "rpois", "sd")
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# SMITE

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# smoof

Version: 1.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# smpic

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# snht

Version: 1.0.5

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'pairwiseSNHT.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# snpEnrichment

Version: 1.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sNPLS

Version: 0.3.31

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# soc.ca

Version: 0.7.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 833 marked UTF-8 strings
    ```

# socialmixr

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# Sofi

Version: 0.16.4.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# soGGi

Version: 1.12.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘soGGi-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plotRegion
    > ### Title: Plot regions
    > ### Aliases: plotRegion plotRegion,ChIPprofile-method
    > ###   plotRegion.ChIPprofile
    > 
    > ### ** Examples
    > 
    > data(chipExampleBig)
    > plotRegion(chipExampleBig[[2]])
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Error: `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
         } else if (length(expr) == 1) {
             name <- expr_text(expr)
             name <- gsub("\n.*$", "...", name)
             name
         } else {
             abort("`expr` must quote a symbol, scalar, or call")
         })
      11: abort("`expr` must quote a symbol, scalar, or call")
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 3 SKIPPED: 0 FAILED: 1
      1. Error: (unknown) (@test_plotting.R#4) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking whether package ‘soGGi’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::Position’ by ‘BiocGenerics::Position’ when loading ‘soGGi’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/soGGi/new/soGGi.Rcheck/00install.out’ for details.
    ```

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'c,ChIPprofile-method':
    \S4method{c}{ChIPprofile}
      Code: function(x, ...)
      Docs: function(x, ..., recursive = FALSE)
      Argument names in docs not in code:
        recursive
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        anyMissing, rowMedians
    
    Loading required package: BiocParallel
    
    Attaching package: 'DelayedArray'
    
    The following objects are masked from 'package:matrixStats':
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following objects are masked from 'package:base':
    
        aperm, apply
    
    Scale for 'x' is already present. Adding another scale for 'x', which will
    replace the existing scale.
    Quitting from lines 123-124 (soggi.Rnw) 
    Error: processing vignette 'soggi.Rnw' failed with diagnostics:
    `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking foreign function calls ... NOTE
    ```
    Foreign function call to a different package:
      .Call("rle_sum_any", ..., PACKAGE = "chipseq")
    See chapter ‘System and foreign language interfaces’ in the ‘Writing R
    Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      definition for ‘weighted.mean’
    getShifts: no visible global function definition for
      ‘readGAlignmentsFromBam’
    getSummitScore: no visible global function definition for
      ‘readGAlignmentsFromBam’
    plotRegion.ChIPprofile: no visible global function definition for
      ‘formula’
    runFindSummit: no visible global function definition for
      ‘readGAlignmentsFromBam’
    runRegionPlot : <anonymous>: no visible global function definition for
      ‘spline’
    summitPipeline: no visible global function definition for
      ‘readGAlignmentsFromBam’
    plotRegion,ChIPprofile: no visible global function definition for
      ‘formula’
    Undefined global functions or variables:
      formula read.delim readGAlignmentsFromBam spline weighted.mean
    Consider adding
      importFrom("stats", "formula", "spline", "weighted.mean")
      importFrom("utils", "read.delim")
    to your NAMESPACE file.
    ```

# soilcarbon

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 145 marked UTF-8 strings
    ```

# solarius

Version: 0.3.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SomaticCancerAlterations

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    .load_dataset: no visible global function definition for ‘data’
    .maf2gr: no visible binding for global variable ‘Chromosome’
    .maf2gr: no visible binding for global variable ‘Start_position’
    .maf2gr: no visible binding for global variable ‘End_position’
    .read_maf: no visible global function definition for ‘read.delim’
    hg2ncbi: no visible global function definition for ‘seqnameStyle<-’
    hg2ncbi: no visible global function definition for ‘genome<-’
    mutationDensity: no visible global function definition for
      ‘keepSeqlevels’
    mutationDensity: no visible global function definition for ‘as’
    ncbi2hg: no visible global function definition for ‘seqnameStyle<-’
    ncbi2hg: no visible global function definition for ‘genome<-’
    scaListDatasets: no visible global function definition for ‘data’
    Undefined global functions or variables:
      Chromosome End_position Start_position as data genome<- keepSeqlevels
      read.delim seqnameStyle<-
    Consider adding
      importFrom("methods", "as")
      importFrom("utils", "data", "read.delim")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# SomaticSignatures

Version: 2.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘NMF’
    A package should be listed in only one of these fields.
    ```

# sorvi

Version: 0.7.26

## Newly broken

*   checking whether package ‘sorvi’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘sorvi’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/sorvi/new/sorvi.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      ...
    Warning: replacing previous import 'dplyr::vars' by 'ggplot2::vars' when loading 'sorvi'
    sorvi - Tools for Finnish Open Data.
    Copyright (C) 2010-2015 Leo Lahti, Juuso Parkkinen, Juuso Haapanen, Einari Happonen, Jussi Paananen, Joona Lehtomaki ym.
    
    http://github.com/ropengov/sorvi 
    
     Hard sciences are successful because they deal with soft problems; 
     soft sciences are struggling because they deal with hard problems.
    -                        Von Foerster
    
    Computing boostrapped smoothers ...
    convert bootstrapped spaghettis to long format
    Computing density estimates for each vertical cut ...
    vertical cross-sectional density estimate
    Tile approach
    Build ggplot figure ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sorvi.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    regression_plot: no visible global function definition for
      ‘colorRampPalette’
    regression_plot: no visible global function definition for
      ‘loess.control’
    regression_plot: no visible global function definition for ‘predict’
    regression_plot : <anonymous>: no visible global function definition
      for ‘quantile’
    regression_plot : <anonymous>: no visible global function definition
      for ‘pnorm’
    regression_plot: no visible global function definition for
      ‘flush.console’
    regression_plot: no visible global function definition for ‘density’
    Undefined global functions or variables:
      colorRampPalette density flush.console loess loess.control pnorm
      predict quantile read.csv
    Consider adding
      importFrom("grDevices", "colorRampPalette")
      importFrom("stats", "density", "loess", "loess.control", "pnorm",
                 "predict", "quantile")
      importFrom("utils", "flush.console", "read.csv")
    to your NAMESPACE file.
    ```

# sotkanet

Version: 0.9.48

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SourceSet

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘r2cytoscape’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘simPATHy’
    ```

# SpaCCr

Version: 0.1.0

## Newly broken

*   checking whether package ‘SpaCCr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SpaCCr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SpaCCr/new/SpaCCr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sparklyr

Version: 0.8.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sparkTable

Version: 1.3.0

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘Rglpk’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# sparsediscrim

Version: 0.2.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SparseFactorAnalysis

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sparseMVN

Version: 0.2.1.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'sparseMVN.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sparsereg

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SparseSignatures

Version: 1.0.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SparseSignatures.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    import.counts.data: no visible global function definition for ‘.’
    patient.plot: no visible binding for global variable ‘Context’
    patient.plot: no visible binding for global variable ‘alt’
    patient.plot: no visible binding for global variable ‘N’
    signatures.plot: no visible binding for global variable ‘Context’
    signatures.plot: no visible binding for global variable ‘alt’
    signatures.plot: no visible binding for global variable ‘value’
    Undefined global functions or variables:
      . Context N alt value
    ```

# sparsevar

Version: 0.0.10

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# spartan

Version: 3.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SpatialBall

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

# spatialClust

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SpatialEpiApp

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘SpatialEpi’ ‘dplyr’ ‘dygraphs’ ‘ggplot2’
      ‘htmlwidgets’ ‘knitr’ ‘leaflet’ ‘mapproj’ ‘maptools’ ‘rgdal’ ‘rgeos’
      ‘rmarkdown’ ‘shinyjs’ ‘spdep’ ‘xts’
      All declared Imports should be used.
    ```

# spatialwarnings

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘moments’ ‘poweRlaw’
      All declared Imports should be used.
    ```

# spbabel

Version: 0.4.8

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘spbabel-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: as_tibble.sfg
    > ### Title: Individual geometries as tibbles.
    > ### Aliases: as_tibble.sfg
    > 
    > ### ** Examples
    > 
    > 
    > tibble::as_tibble(sf::st_point(c(1, 1, 1)))
    Error in loadNamespace(name) : there is no package called ‘sf’
    Calls: <Anonymous> ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             else stop(msg, call. = FALSE, domain = NA)
         })
      3: tryCatchList(expr, classes, parentenv, handlers)
      4: tryCatchOne(expr, names, parentenv, handlers[[1L]])
      5: value[[3L]](cond)
      6: stop(msg, call. = FALSE, domain = NA)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 60 SKIPPED: 1 FAILED: 3
      1. Error: (unknown) (@test-sf.R#3) 
      2. Error: (unknown) (@test-sfFromTable.R#3) 
      3. Error: (unknown) (@test-trip.R#1) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 50-55 (sfbabel.Rmd) 
    Error: processing vignette 'sfbabel.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# spcosa

Version: 0.3-8

## In both

*   checking whether package ‘spcosa’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/spcosa/new/spcosa.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘spcosa’ ...
** package ‘spcosa’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘spcosa’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/spcosa/new/spcosa.Rcheck/spcosa’

```
### CRAN

```
* installing *source* package ‘spcosa’ ...
** package ‘spcosa’ successfully unpacked and MD5 sums checked
** R
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/spcosa/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘spcosa’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/spcosa/old/spcosa.Rcheck/spcosa’

```
# speaq

Version: 2.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# spectacles

Version: 0.5-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# spef

Version: 1.0-7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SPEI

Version: 1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# spew

Version: 1.3.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: sp
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "10"
    Warning in grid.Call(C_textBounds, as.graphicsAnnot(x$label), x$x, x$y,  :
      no font could be found for family "10"
    Quitting from lines 45-64 (spew-quickstart.Rmd) 
    Error: processing vignette 'spew-quickstart.Rmd' failed with diagnostics:
    polygon edge not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘Rmpi’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘maptools’
      All declared Imports should be used.
    ```

# SpidermiR

Version: 1.10.0

## In both

*   checking whether package ‘SpidermiR’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: possible error in 'simpleNetwork(NetworkData, ': unused argument (textColour = "black") 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SpidermiR/new/SpidermiR.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .SpidermiRvisualize_gene: possible error in simpleNetwork(NetworkData,
      linkColour = "gray", textColour = "black", zoom = TRUE): unused
      argument (textColour = "black")
    Case_Study1_loading_3_network: no visible global function definition
      for ‘SpidermiRanalyze_DEnetworkTCGA’
    SpidermiRvisualize_plot_target: no visible binding for global variable
      ‘miRNAs’
    SpidermiRvisualize_plot_target: no visible binding for global variable
      ‘mRNA_target’
    Undefined global functions or variables:
      SpidermiRanalyze_DEnetworkTCGA mRNA_target miRNAs
    ```

# spikeSlabGAM

Version: 1.1-11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:mboost':
    
        %+%
    
    Warning in (function (xnew)  :
      predictions outside fitted range for lin(glucose).
    Warning in (function (xnew)  :
      predictions outside fitted range for sm(glucose).
    Warning in bsplines(mf[[i]], knots = args$knots[[i]]$knots, boundary.knots = args$knots[[i]]$boundary.knots,  :
      Some 'x' values are beyond 'boundary.knots'; Linear extrapolation used.
    Warning in (function (xnew)  :
      predictions outside fitted range for lin(glucose).
    Warning in (function (xnew)  :
      predictions outside fitted range for sm(glucose).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'UsingSpikeSlabGAM.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# spind

Version: 2.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# splatter

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SPLINTER

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    metaremove: no visible global function definition for 'is'
    metaremove: no visible global function definition for 'endoapply'
    Undefined global functions or variables:
      endoapply is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# splithalf

Version: 0.3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# spmoran

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SPONGE

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  9.8Mb
      sub-directories of 1Mb or more:
        data   8.0Mb
        doc    1.5Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      for ‘head’
    sponge_plot_simulation_results: no visible binding for global variable
      ‘mscor’
    sponge_run_benchmark: no visible binding for global variable
      ‘precomputed_cov_matrices’
    sponge_run_benchmark: no visible binding for global variable
      ‘elastic.net’
    sponge_run_benchmark: no visible binding for global variable
      ‘each.miRNA’
    sponge_subsampling: no visible binding for global variable ‘sub.n’
    sponge_subsampling: no visible binding for global variable ‘geneA’
    sponge_subsampling: no visible binding for global variable ‘geneB’
    Undefined global functions or variables:
      . .EACHI .I := J alpha attach.big.matrix chunk cor_cut cov.matrices.k
      cov.matrices.m cov.matrix df_cut dt.m each.miRNA elastic.net g_expr
      g_expr_batch gene geneA geneA_idx geneB geneB_idx gene_combis ginv
      head i k m mir mirna mscor mwhich p.adj p.val
      precomputed_cov_matrices solution sub.n
    Consider adding
      importFrom("utils", "head")
    to your NAMESPACE file.
    ```

# spongecake

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sppmix

Version: 1.0.2

## In both

*   checking whether package ‘sppmix’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/sppmix/new/sppmix.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘sppmix’ ...
** package ‘sppmix’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c ApproxFuncs_sppmix.cpp -o ApproxFuncs_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BDMCMC2d_sppmix.cpp -o BDMCMC2d_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c DAMCMC2d_sppmix.cpp -o DAMCMC2d_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c HelperFuncs_sppmix.cpp -o HelperFuncs_sppmix.o
HelperFuncs_sppmix.cpp:586:7: warning: unused variable 'i' [-Wunused-variable]
  int i,p=W.n_rows;
      ^
DAMCMC2d_sppmix.cpp:541:5: warning: unused variable 'Ry' [-Wunused-variable]
    Ry=max(data.col(1))-min(data.col(1));
    ^
DAMCMC2d_sppmix.cpp:540:10: warning: unused variable 'Rx' [-Wunused-variable]
  double Rx=max(data.col(0))-min(data.col(0)),
         ^
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c MIPPCondLoc_sppmix.cpp -o MIPPCondLoc_sppmix.o
1 warning generated.
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OpsPostGens_sppmix.cpp -o OpsPostGens_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OtherAPIs_sppmix.cpp -o OtherAPIs_sppmix.o
OpsPostGens_sppmix.cpp:18:11: warning: unused variable 'k' [-Wunused-variable]
  int i,j,k,L=allgens.size();
          ^
OpsPostGens_sppmix.cpp:20:21: warning: unused variable 'n' [-Wunused-variable]
  int m=mix1.size(),n=pp.n_rows;
                    ^
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c SimulFuncs_sppmix.cpp -o SimulFuncs_sppmix.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c registerDynamicSymbol.c -o registerDynamicSymbol.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o sppmix.so ApproxFuncs_sppmix.o BDMCMC2d_sppmix.o DAMCMC2d_sppmix.o HelperFuncs_sppmix.o MIPPCondLoc_sppmix.o OpsPostGens_sppmix.o OtherAPIs_sppmix.o RcppExports.o SimulFuncs_sppmix.o registerDynamicSymbol.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/sppmix/new/sppmix.Rcheck/sppmix/libs
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘sppmix’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sppmix/new/sppmix.Rcheck/sppmix’

```
### CRAN

```
* installing *source* package ‘sppmix’ ...
** package ‘sppmix’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c ApproxFuncs_sppmix.cpp -o ApproxFuncs_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c BDMCMC2d_sppmix.cpp -o BDMCMC2d_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c DAMCMC2d_sppmix.cpp -o DAMCMC2d_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c HelperFuncs_sppmix.cpp -o HelperFuncs_sppmix.o
HelperFuncs_sppmix.cpp:586:7: warning: unused variable 'i' [-Wunused-variable]
  int i,p=W.n_rows;
      ^
DAMCMC2d_sppmix.cpp:541:5: warning: unused variable 'Ry' [-Wunused-variable]
    Ry=max(data.col(1))-min(data.col(1));
    ^
DAMCMC2d_sppmix.cpp:540:10: warning: unused variable 'Rx' [-Wunused-variable]
  double Rx=max(data.col(0))-min(data.col(0)),
         ^
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c MIPPCondLoc_sppmix.cpp -o MIPPCondLoc_sppmix.o
1 warning generated.
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OpsPostGens_sppmix.cpp -o OpsPostGens_sppmix.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c OtherAPIs_sppmix.cpp -o OtherAPIs_sppmix.o
OpsPostGens_sppmix.cpp:18:11: warning: unused variable 'k' [-Wunused-variable]
  int i,j,k,L=allgens.size();
          ^
OpsPostGens_sppmix.cpp:20:21: warning: unused variable 'n' [-Wunused-variable]
  int m=mix1.size(),n=pp.n_rows;
                    ^
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c SimulFuncs_sppmix.cpp -o SimulFuncs_sppmix.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG -I../inst/include/ -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/RcppArmadillo/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/sppmix/mvtnorm/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c registerDynamicSymbol.c -o registerDynamicSymbol.o
2 warnings generated.
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o sppmix.so ApproxFuncs_sppmix.o BDMCMC2d_sppmix.o DAMCMC2d_sppmix.o HelperFuncs_sppmix.o MIPPCondLoc_sppmix.o OpsPostGens_sppmix.o OtherAPIs_sppmix.o RcppExports.o SimulFuncs_sppmix.o registerDynamicSymbol.o -L/Library/Frameworks/R.framework/Resources/lib -lRlapack -L/Library/Frameworks/R.framework/Resources/lib -lRblas -L/usr/local/gfortran/lib/gcc/x86_64-apple-darwin15/6.1.0 -L/usr/local/gfortran/lib -lgfortran -lquadmath -lm -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/sppmix/old/sppmix.Rcheck/sppmix/libs
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘sppmix’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/sppmix/old/sppmix.Rcheck/sppmix’

```
# sprm

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SPUTNIK

Version: 1.0.2

## In both

*   checking whether package ‘SPUTNIK’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SPUTNIK/new/SPUTNIK.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘SPUTNIK’ ...
** package ‘SPUTNIK’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘SPUTNIK’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SPUTNIK/new/SPUTNIK.Rcheck/SPUTNIK’

```
### CRAN

```
* installing *source* package ‘SPUTNIK’ ...
** package ‘SPUTNIK’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘SPUTNIK’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/SPUTNIK/old/SPUTNIK.Rcheck/SPUTNIK’

```
# squid

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# srnadiff

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# srvyr

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ss3sim

Version: 0.9.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ssviz

Version: 1.14.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Attaching package: 'Biostrings'
    
    The following object is masked from 'package:base':
    
        strsplit
    
    Loading required package: reshape
    
    Attaching package: 'reshape'
    
    The following objects are masked from 'package:S4Vectors':
    
        expand, rename
    
    Loading required package: ggplot2
    Loading required package: RColorBrewer
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ssviz.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# stability

Version: 0.2.0

## Newly broken

*   checking whether package ‘stability’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘stability’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/stability/new/stability.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# stacomiR

Version: 0.5.3.1

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgetsRGtk2’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# staRdom

Version: 1.0.5

## Newly broken

*   checking whether package ‘staRdom’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘staRdom’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/staRdom/new/staRdom.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    which will replace the existing scale.
    Scale for 'fill' is already present. Adding another scale for 'fill',
    which will replace the existing scale.
    Scale for 'fill' is already present. Adding another scale for 'fill',
    which will replace the existing scale.
    Scale for 'fill' is already present. Adding another scale for 'fill',
    which will replace the existing scale.
    Scale for 'fill' is already present. Adding another scale for 'fill',
    which will replace the existing scale.
    Scale for 'fill' is already present. Adding another scale for 'fill',
    which will replace the existing scale.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Warning: This metric uses either excitation or emission wavelengths that were not present in the data. Data has been interpolated to fit the requested wavelengths.
    Quitting from lines 409-415 (PARAFAC_analysis_of_EEM.Rmd) 
    Error: processing vignette 'PARAFAC_analysis_of_EEM.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~Comp.1".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘readr’ ‘tools’
      All declared Imports should be used.
    ```

# starma

Version: 1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# starmie

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        doc       1.1Mb
        extdata   4.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘MCMCpack’
      All declared Imports should be used.
    ```

# statar

Version: 0.6.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# statcheck

Version: 1.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# statebins

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# STATegRa

Version: 1.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.1Mb
      sub-directories of 1Mb or more:
        data   2.4Mb
        doc    2.9Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    modelSelection,list-numeric-character: no visible binding for global
      variable ‘components’
    modelSelection,list-numeric-character: no visible binding for global
      variable ‘mylabel’
    plotVAF,caClass: no visible binding for global variable ‘comp’
    plotVAF,caClass: no visible binding for global variable ‘VAF’
    plotVAF,caClass: no visible binding for global variable ‘block’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘comps’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘block’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘comp’
    selectCommonComps,list-numeric: no visible binding for global variable
      ‘ratio’
    Undefined global functions or variables:
      VAF block comp components comps mylabel ratio
    ```

# states

Version: 0.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘lubridate’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# StatRank

Version: 0.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# statsDK

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘stringr’
      All declared Imports should be used.
    ```

# statsr

Version: 0.1-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# stminsights

Version: 0.1.2

## Newly broken

*   checking whether package ‘stminsights’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘stminsights’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/stminsights/new/stminsights.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘huge’ ‘scales’ ‘shinyjs’
      All declared Imports should be used.
    ```

# stormwindmodel

Version: 0.1.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        combine
    
    Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=georgia&zoom=5&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
    Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=georgia&sensor=false
    Warning: Removed 703 rows containing missing values (geom_point).
    Warning: Removed 89 rows containing missing values (geom_point).
    Warning: Removed 1 rows containing missing values (geom_path).
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Quitting from lines 98-100 (Overview.Rmd) 
    Error: processing vignette 'Overview.Rmd' failed with diagnostics:
    package or namespace load failed for 'tigris' in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]):
     there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# stpp

Version: 2.0-3

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘rpanel’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# stranger

Version: 0.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# strataG

Version: 2.0.2

## In both

*   checking whether package ‘strataG’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/strataG/new/strataG.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘strataG’ ...
** package ‘strataG’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Hstats_C.cpp -o Hstats_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.c -o init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c misc.funcs.cpp -o misc.funcs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statChi2_C.cpp -o statChi2_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statFis_C.cpp -o statFis_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statFst_C.cpp -o statFst_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statGst_C.cpp -o statGst_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statJostD_C.cpp -o statJostD_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statPhist_C.cpp -o statPhist_C.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o strataG.so Hstats_C.o RcppExports.o init.o misc.funcs.o statChi2_C.o statFis_C.o statFst_C.o statGst_C.o statJostD_C.o statPhist_C.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/strataG/new/strataG.Rcheck/strataG/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘strataG’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/strataG/new/strataG.Rcheck/strataG’

```
### CRAN

```
* installing *source* package ‘strataG’ ...
** package ‘strataG’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c Hstats_C.cpp -o Hstats_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.c -o init.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c misc.funcs.cpp -o misc.funcs.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statChi2_C.cpp -o statChi2_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statFis_C.cpp -o statFis_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statFst_C.cpp -o statFst_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statGst_C.cpp -o statGst_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statJostD_C.cpp -o statJostD_C.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/strataG/Rcpp/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c statPhist_C.cpp -o statPhist_C.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o strataG.so Hstats_C.o RcppExports.o init.o misc.funcs.o statChi2_C.o statFis_C.o statFst_C.o statGst_C.o statJostD_C.o statPhist_C.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/strataG/old/strataG.Rcheck/strataG/libs
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘strataG’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/strataG/old/strataG.Rcheck/strataG’

```
# strider

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# StroupGLMM

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘broom’ ‘car’ ‘lmerTest’ ‘pbkrtest’
      All declared Imports should be used.
    ```

# StructFDR

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# structSSI

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    PlotHypTree: no visible global function definition for ‘browseURL’
    treePValues: no visible global function definition for ‘lm’
    treePValues: no visible global function definition for ‘pf’
    plot,GBH-ANY: no visible binding for global variable ‘sorted.hyp’
    plot,GBH-ANY: no visible binding for global variable ‘pval’
    plot,GBH-ANY: no visible binding for global variable ‘group’
    plot,GBH-ANY: no visible binding for global variable ‘type’
    Undefined global functions or variables:
      browseURL group lm pf pval sorted.hyp type
    Consider adding
      importFrom("stats", "lm", "pf")
      importFrom("utils", "browseURL")
    to your NAMESPACE file.
    ```

# strvalidator

Version: 2.0.0

## In both

*   checking package dependencies ... ERROR
    ```
    Packages required but not available: ‘gWidgets2RGtk2’ ‘RGtk2’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# SubgrpID

Version: 0.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# subSeq

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'subSeq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    summary.subsamples: no visible binding for global variable ‘o.padj’
    summary.subsamples: no visible binding for global variable
      ‘significant’
    summary.subsamples: no visible binding for global variable ‘estFDP’
    summary.subsamples: no visible binding for global variable ‘rFDP’
    summary.subsamples: no visible binding for global variable ‘metric’
    summary.subsamples: no visible binding for global variable ‘value’
    summary.subsamples: no visible binding for global variable ‘percent’
    voomLimma: no visible global function definition for ‘model.matrix’
    Undefined global functions or variables:
      . ID average.depth average.value coefficient cor count cov depth
      estFDP method metric model.matrix o.coefficient o.lfdr o.padj
      p.adjust padj percent plot proportion pvalue rFDP rbinom replication
      selectMethod significant valid value var
    Consider adding
      importFrom("graphics", "plot")
      importFrom("methods", "selectMethod")
      importFrom("stats", "cor", "cov", "model.matrix", "p.adjust", "rbinom",
                 "var")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# subspaceMOA

Version: 0.6.0

## In both

*   checking whether package ‘subspaceMOA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/subspaceMOA/new/subspaceMOA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘subspaceMOA’ ...
** package ‘subspaceMOA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘subspaceMOA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/subspaceMOA/new/subspaceMOA.Rcheck/subspaceMOA’

```
### CRAN

```
* installing *source* package ‘subspaceMOA’ ...
** package ‘subspaceMOA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘rJava’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/subspaceMOA/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘rJava’ could not be loaded
ERROR: lazy loading failed for package ‘subspaceMOA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/subspaceMOA/old/subspaceMOA.Rcheck/subspaceMOA’

```
# sugrrants

Version: 0.1.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    >    p1 <- calendar_df %>% 
    +      ggplot(aes(x = .Time, y = .Hourly_Counts, group = Date)) +
    +      geom_line()
    >    prettify(p1, size = 3, label.padding = unit(0.15, "lines"))
    >    
    >    # use in conjunction with group_by()
    >    grped_calendar <- pedestrian %>% 
    +      filter(Year == "2017", Month == "March") %>% 
    +      group_by(Sensor_Name) %>% 
    +      frame_calendar(
    +        x = Time, y = Hourly_Counts, date = Date, sunday = TRUE
    +      )
    >    
    >    p2 <- grped_calendar %>% 
    +      ggplot(aes(x = .Time, y = .Hourly_Counts, group = Date)) +
    +      geom_line() +
    +      facet_wrap(~ Sensor_Name, nrow = 2)
    Error in grouped_indices_grouped_df_impl(.data) : 
      Need at least one column for `hash()`
    Calls: %>% ... group_indices.grouped_df -> grouped_indices_grouped_df_impl
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
    Loading required package: viridisLite
    Loading required package: ggplot2
    
    Attaching package: 'ggplot2'
    
    The following object is masked from 'package:dplyr':
    
        vars
    
    
    Attaching package: 'sugrrants'
    
    The following object is masked from 'package:ggplot2':
    
        vars
    
    Quitting from lines 113-123 (frame-calendar.Rmd) 
    Error: processing vignette 'frame-calendar.Rmd' failed with diagnostics:
    Need at least one column for `hash()`
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# summariser

Version: 0.1.0

## Newly broken

*   checking whether package ‘summariser’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘summariser’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/summariser/new/summariser.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SummarizedBenchmark

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Depends: includes the non-default packages:
      ‘tidyr’ ‘SummarizedExperiment’ ‘S4Vectors’ ‘BiocGenerics’ ‘UpSetR’
      ‘rlang’ ‘stringr’ ‘BiocParallel’ ‘ggplot2’ ‘mclust’ ‘dplyr’
    Adding so many packages to the search path is excessive and importing
    selectively is preferable.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.6Mb
      sub-directories of 1Mb or more:
        data   9.0Mb
        doc    3.3Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported object imported by a ':::' call: ‘BiocGenerics:::replaceSlots’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .list2mat : <anonymous>: no visible binding for global variable
      ‘.method’
    .list2mat : <anonymous>: no visible binding for global variable ‘.val’
    .list2mat : <anonymous>: no visible binding for global variable ‘.id’
    plotROC: no visible binding for global variable ‘FDR’
    plotROC: no visible binding for global variable ‘TPR’
    plotROC: no visible binding for global variable ‘method’
    Undefined global functions or variables:
      .id .method .val FDR TPR method
    ```

# SUMMER

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘INLA’
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘RColorBrewer’ ‘classInt’ ‘grDevices’ ‘graphics’ ‘pixmap’ ‘rgeos’
      ‘sp’
      All declared Imports should be used.
    ```

# superheat

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SuperLearner

Version: 2.0-23

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      lstat       -0.019759 0.003639 -5.429 8.912e-08 ***
      
      ------------------------------------------------------------------- 
      Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1 
      ---
      Residual standard error: 0.340537 on 492 degrees of freedom;
      observations: 506;  R^2: 0.519;  adjusted R^2: 0.506;
      F-statistic: 40.86 on 13 and 492 df;  p-value: 0.
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 98 SKIPPED: 0 FAILED: 2
      1. Error: (unknown) (@test-bartMachine.R#2) 
      2. Error: (unknown) (@test-extraTrees.R#2) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SuperLearnerPresent.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sure

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘truncdist’
    ```

# surveillance

Version: 1.16.0

## In both

*   checking whether package ‘surveillance’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/surveillance/new/surveillance.Rcheck/00install.out’ for details.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking: ‘polyclip’ ‘INLA’
    ```

## Installation

### Devel

```
* installing *source* package ‘surveillance’ ...
** package ‘surveillance’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c backproj.cc -o backproj.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c determineSources.cc -o determineSources.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.c -o init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c ks.c -o ks.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stcd-assuncaocorrea.cc -o stcd-assuncaocorrea.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c surveillance.c -o surveillance.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c twins.cc -o twins.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c twinstim_siaf_polyCub_iso.c -o twinstim_siaf_polyCub_iso.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o surveillance.so RcppExports.o backproj.o determineSources.o init.o ks.o stcd-assuncaocorrea.o surveillance.o twins.o twinstim_siaf_polyCub_iso.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/surveillance/new/surveillance.Rcheck/surveillance/libs
** R
** data
** demo
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘surveillance’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/surveillance/new/surveillance.Rcheck/surveillance’

```
### CRAN

```
* installing *source* package ‘surveillance’ ...
** package ‘surveillance’ successfully unpacked and MD5 sums checked
** libs
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c RcppExports.cpp -o RcppExports.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c backproj.cc -o backproj.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c determineSources.cc -o determineSources.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c init.c -o init.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c ks.c -o ks.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c stcd-assuncaocorrea.cc -o stcd-assuncaocorrea.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c surveillance.c -o surveillance.o
/usr/local/clang4/bin/clang++ -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c twins.cc -o twins.o
/usr/local/clang4/bin/clang -I"/Library/Frameworks/R.framework/Resources/include" -DNDEBUG  -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/Rcpp/include" -I"/Users/max/github/ggplot2/revdep/library.noindex/surveillance/polyCub/include" -I/usr/local/include   -fPIC  -Wall -g -O2  -c twinstim_siaf_polyCub_iso.c -o twinstim_siaf_polyCub_iso.o
/usr/local/clang4/bin/clang++ -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup -single_module -multiply_defined suppress -L/Library/Frameworks/R.framework/Resources/lib -L/usr/local/clang4/lib -o surveillance.so RcppExports.o backproj.o determineSources.o init.o ks.o stcd-assuncaocorrea.o surveillance.o twins.o twinstim_siaf_polyCub_iso.o -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework -Wl,CoreFoundation
installing to /Users/max/github/ggplot2/revdep/checks.noindex/surveillance/old/surveillance.Rcheck/surveillance/libs
** R
** data
** demo
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘surveillance’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/surveillance/old/surveillance.Rcheck/surveillance’

```
# survivALL

Version: 0.9.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘cowplot’
      All declared Imports should be used.
    ```

# survminer

Version: 0.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.6Mb
      sub-directories of 1Mb or more:
        doc   5.1Mb
    ```

# survMisc

Version: 0.5.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'plots.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SurvRank

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# survtmle

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# survutils

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# sValues

Version: 0.1.4

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘sValues-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot.sValues
    > ### Title: Plot method for S-values
    > ### Aliases: plot.sValues
    > 
    > ### ** Examples
    > 
    > # growth regressions example
    > data(economic_growth)
    > eg_sv <- sValues(GR6096 ~ ., data = economic_growth)
    > plot(eg_sv, R2_bounds = c(0.5, 1))
    > plot(eg_sv, R2_bounds = c(0.1, 1))
    > plot(eg_sv, type = "beta_plot", variable = "OPENDEC1", error_bar = FALSE)
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 154-156 (sValues.Rnw) 
    Error: processing vignette 'sValues.Rnw' failed with diagnostics:
    `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SVAPLSseq

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:base’:
    
        aperm, apply
    
    Loading required package: limma
    
    Attaching package: ‘limma’
    
    The following object is masked from ‘package:BiocGenerics’:
    
        plotMA
    
    Warning in if (surr.select == "automatic") { :
      the condition has length > 1 and only the first element will be used
    Warning in if (surr.select == "manual") index = opt.surrs :
      the condition has length > 1 and only the first element will be used
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SVAPLSseq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# svars

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# svdvis

Version: 0.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘svdvis-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: radviz
    > ### Title: Radial Coordinates Plots
    > ### Aliases: radviz
    > 
    > ### ** Examples
    > 
    > set.seed(1234)
    > dat = matrix(rnorm(9*4), 9, 4, dimnames=list(paste(1:9),letters[1:4]))
    > radviz(dat, group=c(rep("Group 1",3),rep("Group 2",3),rep("Group 3",3)))
    Error: `data` must be uniquely named but has duplicate columns
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 50-51 (svdvis-vignette.Rmd) 
    Error: processing vignette 'svdvis-vignette.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~V1".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SVMMaj

Version: 0.2-8

## Newly broken

*   checking whether package ‘SVMMaj’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SVMMaj’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SVMMaj/new/SVMMaj.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SVMMaj’
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    Scale for 'y' is already present. Adding another scale for 'y', which will
    replace the existing scale.
    
    Attaching package: ‘kernlab’
    
    The following object is masked from ‘package:ggplot2’:
    
        alpha
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'paper.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# swa

Version: 0.8.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SWATH2stats

Version: 1.10.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages which this enhances but not available for checking:
      ‘imsbInfer’ ‘MSstats’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Percentage of peptides removed: 2.78%
    
    After filtering: 
      Number of proteins: 8
      Number of peptides: 35
    
    The library contains 6 transitions per precursor.
                      
    The data table was transformed into a table containing one row per transition.
    Performing log-transformation
    Calculating low-level statistics
    Bootstrapping samples
    Optimizing parameters
    Calculating p-values
    Calculating FDR
    Aggregating statistics
    Done
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'SWATH2stats_vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# swfdr

Version: 1.6.0

## Newly broken

*   checking whether package ‘swfdr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘swfdr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/swfdr/new/swfdr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    lm_pi0: no visible global function definition for ‘glm’
    lm_pi0: no visible binding for global variable ‘binomial’
    Undefined global functions or variables:
      binomial glm
    Consider adding
      importFrom("stats", "binomial", "glm")
    to your NAMESPACE file.
    ```

# swfscMisc

Version: 1.2

## In both

*   checking whether package ‘swfscMisc’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/swfscMisc/new/swfscMisc.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘swfscMisc’ ...
** package ‘swfscMisc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘swfscMisc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/swfscMisc/new/swfscMisc.Rcheck/swfscMisc’

```
### CRAN

```
* installing *source* package ‘swfscMisc’ ...
** package ‘swfscMisc’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘swfscMisc’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/swfscMisc/old/swfscMisc.Rcheck/swfscMisc’

```
# switchde

Version: 1.6.0

## Newly broken

*   checking whether package ‘switchde’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘switchde’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/switchde/new/switchde.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

# swmmr

Version: 0.8.1

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: 'dplyr'
    
    The following objects are masked from 'package:stats':
    
        filter, lag
    
    The following objects are masked from 'package:base':
    
        intersect, setdiff, setequal, union
    
    Quitting from lines 26-31 (How_to_convert_between_GIS_data_and_SWMM_with_swmmr.Rmd) 
    Error: processing vignette 'How_to_convert_between_GIS_data_and_SWMM_with_swmmr.Rmd' failed with diagnostics:
    there is no package called 'sf'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘sf’
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sf’
    ```

# SWMPr

Version: 2.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# SWMPrExtension

Version: 0.3.12

## Newly broken

*   checking whether package ‘SWMPrExtension’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘SWMPrExtension’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/SWMPrExtension/new/SWMPrExtension.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘rgeos’ ‘stringr’
      All declared Imports should be used.
    ```

# synergyfinder

Version: 1.6.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'synergyfinder.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    In addition: Warning message:
    In replayPlot(x) : `mgp[1:3]' are of differing sign
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .travis.yml
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    License components with restrictions not permitted:
      Mozilla Public License 2.0 + file LICENSE
    ```

*   checking R code for possible problems ... NOTE
    ```
    PlotSynergy: no visible binding for global variable ‘pred.cory’
    PlotSynergy: no visible binding for global variable ‘pred.corx’
    Undefined global functions or variables:
      pred.corx pred.cory
    ```

# synlet

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'synlet-vignette.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    siRNAPlot: no visible global function definition for ‘pdf’
    siRNAPlot: no visible global function definition for ‘dev.off’
    tTest: no visible global function definition for ‘p.adjust’
    zFactor: no visible binding for global variable ‘condition’
    zFactor: no visible binding for global variable ‘sd’
    zFactor: no visible binding for global variable ‘median’
    zFactor: no visible global function definition for ‘complete.cases’
    Undefined global functions or variables:
      COL_NAME EXPERIMENT_MODIFICATION EXPERIMENT_TYPE MASTER_PLATE PLATE
      READOUT ROW_NAME Var1 WELL_CONTENT_NAME colorRampPalette
      complete.cases condition dev.off experiments is mad median medpolish
      p.adjust pdf phyper rainbow sd siRNA t.test value write.table
    Consider adding
      importFrom("grDevices", "colorRampPalette", "dev.off", "pdf",
                 "rainbow")
      importFrom("methods", "is")
      importFrom("stats", "complete.cases", "mad", "median", "medpolish",
                 "p.adjust", "phyper", "sd", "t.test")
      importFrom("utils", "write.table")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# synthpop

Version: 1.4-3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘synthpop-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: compare.fit.synds
    > ### Title: Compare model estimates based on synthesised and observed data
    > ### Aliases: compare.fit.synds print.compare.fit.synds
    > 
    > ### ** Examples
    > 
    > ods <- SD2011[,c("sex","age","edu","smoke")]
    > s1 <- syn(ods, m = 3)
    syn  variables
    1    sex age edu smoke
    2    sex age edu smoke
    3    sex age edu smoke
    > f1 <- glm.synds(smoke ~ sex + age + edu, data = s1, family = "binomial")
    > compare(f1, ods) 
    Error: More than one expression parsed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: lattice
    Loading required package: MASS
    Loading required package: nnet
    Loading required package: ggplot2
    
    Error: processing vignette 'inference.Rnw' failed with diagnostics:
     chunk 10 
    Error : More than one expression parsed
    Execution halted
    ```

# sysid

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# systemPipeR

Version: 1.14.0

## In both

*   checking package subdirectories ... WARNING
    ```
    Invalid citation information in ‘inst/CITATION’:
      Error in library(knitcitations): there is no package called ‘knitcitations’
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'systemPipeChIPseq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      inst/extdata/.BatchJobs.R
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.9Mb
      sub-directories of 1Mb or more:
        R         1.0Mb
        doc       3.9Mb
        extdata   1.7Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘BiocGenerics’ ‘VariantAnnotation’
      All declared Imports should be used.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘close.screen’
    writeTargetsRef: no visible global function definition for ‘read.delim’
    show,INTERSECTset: no visible binding for global variable ‘vennset’
    Undefined global functions or variables:
      AllVariants Base Comparisons Counts Coverage Cycle DataFrame Feature
      Frequency IRanges Intersect_Sets Length Level Method Outliers Percent
      Quality RelDiv Rle Sample SampleMatch Strand Type aggregate alt
      altDepth<- asVCF boxplot chunk close.screen combn dev.off first
      import.bed last locateVariants low mid minQuality model.matrix
      na.omit pdf phyper plot predictCoding qwidth read.delim
      readGAlignmentPairs readGAlignments readVcf ref refDepth<- screen
      seqlengths seqlengths<- split.screen subsetByOverlaps
      summarizeOverlaps symbols test_sample text top tophatargs
      totalDepth<- vennset write.table writeVcf
    Consider adding
      importFrom("grDevices", "dev.off", "pdf")
      importFrom("graphics", "boxplot", "close.screen", "plot", "screen",
                 "split.screen", "symbols", "text")
      importFrom("stats", "aggregate", "model.matrix", "na.omit", "phyper")
      importFrom("utils", "combn", "read.delim", "write.table")
    to your NAMESPACE file.
    ```

*   checking for unstated dependencies in vignettes ... NOTE
    ```
    'library' or 'require' call not declared from: ‘GenomicAlignments’
    ```

# tableone

Version: 0.9.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# tabplot

Version: 1.3-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tadaatoolbox

Version: 0.16.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 3 marked UTF-8 strings
    ```

# taRifx

Version: 1.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Title field: should not end in a period.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘gdata’ ‘ggplot2’ ‘grid’ ‘lattice’ ‘xtable’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    xtable.summary.lme: no visible global function definition for ‘align<-’
    xtable.summary.lme: no visible global function definition for
      ‘digits<-’
    xtable.summary.lme: no visible global function definition for
      ‘display<-’
    xtablelm: no visible global function definition for ‘xtable’
    xtablelm: no visible global function definition for ‘pf’
    Undefined global functions or variables:
      aes align<- barplot caption<- convertUnit coord_cartesian digits<-
      display<- ecdf ggplot gpar grid.layout grid.lines grid.newpage
      grid.points grid.polyline grid.rect grid.segments grid.text
      interleave label<- latticeParseFormula median na.omit opts
      panel.densityplot panel.lines panel.xyplot par pf plot.new
      popViewport pushViewport quantile sd seekViewport stat_summary terms
      text theme_text time unit upViewport viewport write.csv xtable
    Consider adding
      importFrom("graphics", "barplot", "par", "plot.new", "text")
      importFrom("stats", "ecdf", "median", "na.omit", "pf", "quantile",
                 "sd", "terms", "time")
      importFrom("utils", "write.csv")
    to your NAMESPACE file.
    ```

# TarSeqQC

Version: 1.10.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotNtdPercentage,TargetExperiment: no visible global function
      definition for ‘path’
    plotRegion,TargetExperiment: no visible global function definition for
      ‘path’
    readFrequencies,TargetExperiment: no visible global function definition
      for ‘path’
    Undefined global functions or variables:
      path
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TarSeqQC-vignette.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# TCGAbiolinks

Version: 2.8.0

## In both

*   R CMD check timed out
    

*   checking installed package size ... NOTE
    ```
      installed size is 67.4Mb
      sub-directories of 1Mb or more:
        R      2.4Mb
        data   4.1Mb
        doc   60.6Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    TCGAanalyze_networkInference: no visible global function definition for
      ‘c3net’
    TCGAanalyze_networkInference: no visible global function definition for
      ‘minet’
    TCGAquery_recount2: no visible binding for global variable ‘rse_gene’
    TCGAtumor_purity: no visible binding for global variable ‘Tumor.purity’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dNetInduce’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dNetPipeline’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘dCommSignif’
    TCGAvisualize_SurvivalCoxNET: no visible global function definition for
      ‘visNet’
    TCGAvisualize_oncoprint: no visible binding for global variable ‘value’
    readTranscriptomeProfiling: no visible binding for global variable
      ‘ignore.case’
    Undefined global functions or variables:
      TabSubtypesCol_merged Tumor.purity barcode c3net clinical dCommSignif
      dNetInduce dNetPipeline ignore.case knnmi.cross limmacontrasts.fit
      limmamakeContrasts minet portions rse_gene value visNet
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Error in .requirePackage(package) : 
        unable to find required package 'maftools'
      Calls: <Anonymous> ... getClass -> getClassDef -> .classEnv -> .requirePackage
      Execution halted
    ```

# TCGAbiolinksGUI

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 17.9Mb
      sub-directories of 1Mb or more:
        app   1.1Mb
        doc  16.7Mb
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      'DT' 'IlluminaHumanMethylation27kanno.ilmn12.hg19'
      'IlluminaHumanMethylation27kmanifest'
      'IlluminaHumanMethylation450kanno.ilmn12.hg19'
      'IlluminaHumanMethylation450kmanifest'
      'IlluminaHumanMethylationEPICanno.ilm10b2.hg19'
      'IlluminaHumanMethylationEPICmanifest' 'minfi'
      All declared Imports should be used.
    Package in Depends field not imported from: 'TCGAbiolinksGUI.data'
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

# TcGSA

Version: 0.10.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tcgsaseq

Version: 1.6.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tcR

Version: 2.2.1.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.8Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    3.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘scales’
      All declared Imports should be used.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'top.fun':
      ‘slice.fun’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# TCseq

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    DBresult: no visible global function definition for ‘as’
    DBresult.cluster: no visible global function definition for ‘as’
    TCA: no visible global function definition for ‘is’
    TCA: no visible global function definition for ‘as’
    TCAFromSummarizedExperiment: no visible global function definition for
      ‘is’
    TCAFromSummarizedExperiment: no visible global function definition for
      ‘as’
    countReads: no visible global function definition for
      ‘createAnnotationFile’
    countReads: no visible global function definition for ‘featureCounts’
    timeclustplot: no visible binding for global variable ‘group’
    timecourseTable: no visible global function definition for ‘as’
    Undefined global functions or variables:
      as createAnnotationFile featureCounts group is
    Consider adding
      importFrom("methods", "as", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
    The following objects are masked from ‘package:Biobase’:
    
        anyMissing, rowMedians
    
    Loading required package: BiocParallel
    
    Attaching package: ‘DelayedArray’
    
    The following objects are masked from ‘package:matrixStats’:
    
        colMaxs, colMins, colRanges, rowMaxs, rowMins, rowRanges
    
    The following objects are masked from ‘package:base’:
    
        aperm, apply
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TCseq.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# tdr

Version: 0.13

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TeachBayes

Version: 1.0

## Newly broken

*   checking whether package ‘TeachBayes’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘TeachBayes’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TeachBayes/new/TeachBayes.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# teachingApps

Version: 1.0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘data.table’ ‘datasets’ ‘stats’
      All declared Imports should be used.
    ```

# TeachingDemos

Version: 2.10

## In both

*   R CMD check timed out
    

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘R2wd’
    ```

# teamcolors

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TELP

Version: 1.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘TELP-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: telp
    > ### Title: The Base Function of This Package The Free Evocation of Words
    > ###   Technique
    > ### Aliases: telp
    > ### Keywords: function
    > 
    > ### ** Examples
    > 
    > telp()
    Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
      [tcl] invalid command name "toplevel".
    Calls: telp ... tktoplevel -> tkwidget -> tcl -> .Tcl.objv -> structure
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# testassay

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tetraclasse

Version: 0.1.21

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# texmex

Version: 2.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# texteffect

Version: 0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TFEA.ChIP

Version: 1.0.0

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/runTests.R’ failed.
    Last 13 lines of output:
          'browseVignettes()'. To cite Bioconductor, see
          'citation("Biobase")', and for packages 'citation("pkgname")'.
      
      
      Attaching package: 'AnnotationDbi'
      
      The following object is masked from 'package:dplyr':
      
          select
      
      
      Error in library("RUnit", quietly = TRUE) : 
        there is no package called 'RUnit'
      Calls: <Anonymous> -> library
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tfestimators

Version: 1.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TFutils

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 62 marked UTF-8 strings
    ```

# themetagenomics

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# theseus

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘gridExtra’ ‘splancs’ ‘tidyverse’
      All declared Imports should be used.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Packages unavailable to check Rd xrefs: ‘DESeq2’, ‘dada2’
    ```

# thief

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# thinkr

Version: 0.11

## In both

*   checking whether package ‘thinkr’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/thinkr/new/thinkr.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘thinkr’ ...
** package ‘thinkr’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘thinkr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/thinkr/new/thinkr.Rcheck/thinkr’

```
### CRAN

```
* installing *source* package ‘thinkr’ ...
** package ‘thinkr’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/thinkr/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘thinkr’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/thinkr/old/thinkr.Rcheck/thinkr’

```
# tictactoe

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tidycensus

Version: 0.4.6

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tidyhydat

Version: 0.3.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tidyinftheo

Version: 0.2.1

## Newly broken

*   checking whether package ‘tidyinftheo’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘tidyinftheo’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/tidyinftheo/new/tidyinftheo.Rcheck/00install.out’ for details.
    ```

## In both

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      12: shannon_entropy(.)
      13: reduce_data(.data, !!enquo(X), numvars = 1, na.rm = na.rm)
      14: tidyselect::vars_select(names(reduced_tab), !!!quos(...))
      15: vars_select_eval(.vars, quos)
      16: map_lgl(quos, quo_is_helper)
      17: .f(.x[[i]], ...)
      18: extract_expr(quo)
      19: is_call(expr, paren_sym)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 37 SKIPPED: 0 FAILED: 1
      1. Error: right number of columns given (@test-errors.R#10) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘tibble’ ‘tidyverse’
      All declared Imports should be used.
    ```

# tidyLPA

Version: 0.1.3

## Newly broken

*   checking whether package ‘tidyLPA’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘tidyLPA’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/tidyLPA/new/tidyLPA.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tidytext

Version: 0.1.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tidyverse

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dbplyr’ ‘reprex’ ‘rlang’
      All declared Imports should be used.
    ```

# tidyxl

Version: 1.0.1

## In both

*   checking compiled code ... WARNING
    ```
    File ‘tidyxl/libs/tidyxl.so’:
      Found ‘_abort’, possibly from ‘abort’ (C)
        Object: ‘xlex.o’
    
    Compiled code should not call entry points which might terminate R nor
    write to stdout/stderr instead of to the console, nor use Fortran I/O
    nor system RNGs.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tigerstats

Version: 0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tigger

Version: 0.2.11

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tigris

Version: 0.7

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘sf’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# tikzDevice

Version: 0.11

## In both

*   checking examples ... ERROR
    ```
    ...
    
    	{\\tiny Hello \\LaTeX!}
    
    Run the following commands for diagnosis:
    
    	tikzTest()
    	tikzTest("{\\\\tiny Hello \\\\LaTeX!}")
    
    Common reasons for failure include:
      * The string contains a character which is special to LaTeX unless
        escaped properly, such as % or $.
      * The string makes use of LaTeX commands provided by a package and
        the tikzDevice was not told to load the package.
    
    The TeX and log files used for the calculation can help diagnose the
    problem. If these files are missing, rerun the plot and make sure to
    keep the R session open.
    TeX file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpqEhY5Q/tikzDevice1540d553d127f/tikzStringWidthCalc.tex
    Log file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//RtmpqEhY5Q/tikzDevice1540d553d127f/tikzStringWidthCalc.log
    Calls: getLatexStrWidth -> getMetricsFromLatex
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    TeX was unable to calculate metrics for:
    
    	m
    
    Run the following commands for diagnosis:
    
    	tikzTest()
    	tikzTest("m")
    
    Common reasons for failure include:
      * The string contains a character which is special to LaTeX unless
        escaped properly, such as % or $.
      * The string makes use of LaTeX commands provided by a package and
        the tikzDevice was not told to load the package.
    
    The TeX and log files used for the calculation can help diagnose the
    problem. If these files are missing, rerun the plot and make sure to
    keep the R session open.
    TeX file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp4pTWg8/tikzDevice155655b66aa5a/tikzStringWidthCalc.tex
    Log file: /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmp4pTWg8/tikzDevice155655b66aa5a/tikzStringWidthCalc.log
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# timeline

Version: 0.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘shiny’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Package in Depends field not imported from: ‘ggplot2’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    timeline: no visible global function definition for ‘ggplot’
    timeline: no visible global function definition for ‘geom_segment’
    timeline: no visible global function definition for ‘aes_string’
    timeline: no visible global function definition for ‘geom_rect’
    timeline: no visible global function definition for ‘geom_text’
    timeline: no visible global function definition for ‘theme’
    timeline: no visible global function definition for ‘element_blank’
    timeline: no visible global function definition for ‘xlab’
    timeline: no visible global function definition for ‘ylab’
    timeline: no visible global function definition for ‘xlim’
    timeline: no visible global function definition for
      ‘scale_y_continuous’
    timeline: no visible global function definition for ‘geom_point’
    timeline: no visible global function definition for ‘scale_color_grey’
    timeline: no visible global function definition for ‘geom_hline’
    Undefined global functions or variables:
      aes_string element_blank geom_hline geom_point geom_rect geom_segment
      geom_text ggplot scale_color_grey scale_y_continuous theme xlab xlim
      ylab
    ```

# timelineR

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# timelineS

Version: 0.1.1

## Newly broken

*   checking whether package ‘timelineS’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘timelineS’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/timelineS/new/timelineS.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘base’
      All declared Imports should be used.
    ```

# TimeProjection

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' calls in package code:
      ‘ggplot2’ ‘plyr’
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Packages in Depends field not imported from:
      ‘Matrix’ ‘lubridate’ ‘timeDate’
      These packages need to be imported from (in the NAMESPACE file)
      for when this namespace is loaded but not attached.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotCalendarHeatmap: no visible global function definition for ‘ddply’
    plotCalendarHeatmap: no visible global function definition for ‘.’
    plotCalendarHeatmap: no visible binding for global variable ‘year’
    plotCalendarHeatmap: no visible binding for global variable ‘month’
    plotCalendarHeatmap: no visible binding for global variable ‘week’
    plotCalendarHeatmap: no visible global function definition for ‘ggplot’
    plotCalendarHeatmap: no visible global function definition for ‘aes’
    plotCalendarHeatmap: no visible binding for global variable ‘monthweek’
    plotCalendarHeatmap: no visible binding for global variable ‘weekday’
    plotCalendarHeatmap: no visible global function definition for
      ‘geom_tile’
    plotCalendarHeatmap: no visible global function definition for
      ‘facet_grid’
    plotCalendarHeatmap: no visible global function definition for
      ‘scale_fill_gradientn’
    projectDate: no visible global function definition for ‘holidayNYSE’
    projectDate: no visible global function definition for
      ‘sparse.model.matrix’
    Undefined global functions or variables:
      . aes ddply facet_grid geom_tile ggplot holidayNYSE isWeekday month
      monthweek scale_fill_gradientn sparse.model.matrix week weekday year
    ```

# TimerQuant

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Warning in min(x, na.rm = TRUE) :
      no non-missing arguments to min; returning Inf
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'genPaperFigures.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘rainbow’
    plotPrimordiumProfile: no visible binding for global variable ‘median’
    plotPrimordiumProfile: no visible binding for global variable ‘mad’
    plotPrimordiumProfile: no visible global function definition for ‘par’
    plotPrimordiumProfile: no visible global function definition for ‘plot’
    plotPrimordiumProfile: no visible global function definition for ‘axis’
    plotPrimordiumProfile: no visible global function definition for
      ‘points’
    plotPrimordiumProfile: no visible global function definition for
      ‘polygon’
    plotPrimordiumProfile: no visible global function definition for ‘rgb’
    simulatedRatio: no visible global function definition for ‘rnorm’
    Undefined global functions or variables:
      approxfun axis mad median optimize par plot points polygon predict
      rainbow rgb rnorm
    Consider adding
      importFrom("grDevices", "rainbow", "rgb")
      importFrom("graphics", "axis", "par", "plot", "points", "polygon")
      importFrom("stats", "approxfun", "mad", "median", "optimize",
                 "predict", "rnorm")
    to your NAMESPACE file.
    ```

# tint

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.1Mb
      sub-directories of 1Mb or more:
        doc         3.7Mb
        rmarkdown   1.3Mb
    ```

# TippingPoint

Version: 1.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TissueEnrich

Version: 1.0.0

## Newly broken

*   checking whether package ‘TissueEnrich’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::vars’ by ‘dplyr::vars’ when loading ‘TissueEnrich’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TissueEnrich/new/TissueEnrich.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tmap

Version: 1.11-2

## In both

*   checking whether package ‘tmap’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/tmap/new/tmap.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘tmap’ ...
** package ‘tmap’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘tmap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/tmap/new/tmap.Rcheck/tmap’

```
### CRAN

```
* installing *source* package ‘tmap’ ...
** package ‘tmap’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘sf’
ERROR: lazy loading failed for package ‘tmap’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/tmap/old/tmap.Rcheck/tmap’

```
# Tmisc

Version: 0.1.19

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# toaster

Version: 0.5.5

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ── 1. Failure: format 'boxplot' works (@test-showData.R#12)  ───────────────────
      setdiff(p$layout$panel_ranges[[1]]$x.labels, getNumericColumns(notNullNumData)) not equal to character(0).
      target is NULL, current is character
      
      ── 2. Failure: format 'boxplot' with facets works (@test-showData.R#21)  ───────
      nrow(p$layout$panel_layout) not equal to length(cols).
      target is NULL, current is numeric
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 496 SKIPPED: 0 FAILED: 2
      1. Failure: format 'boxplot' works (@test-showData.R#12) 
      2. Failure: format 'boxplot' with facets works (@test-showData.R#21) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# toolmaRk

Version: 0.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# topdownr

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 31-36 (analysis.Rmd) 
    Error: processing vignette 'analysis.Rmd' failed with diagnostics:
    there is no package called 'topdownrdata'
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘topdownrdata’
    ```

*   checking for hidden files and directories ... NOTE
    ```
    Found the following hidden files and directories:
      .BBSoptions
    These were most likely included in error. See section ‘Package
    structure’ in the ‘Writing R Extensions’ manual.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘topdownrdata’
    ```

# TOSTER

Version: 0.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘jmvcore’
      All declared Imports should be used.
    ```

# totalcensus

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 42 marked Latin-1 strings
      Note: found 360 marked UTF-8 strings
    ```

# tourr

Version: 0.5.5

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    0.951  -0.067  
    -0.038  0.846  
    0.229  0.443  
    -0.084  -0.223  
    -0.009  -0.127  
    0.188  -0.136  
    > path_index(fl_holes, holes)
    [1] 0.8888866 0.9034753 0.9144009 0.9212867 0.9295719 0.9315650 0.9326219
    [8] 0.9341583 0.9350965
    attr(,"class")
    [1] "path_index"
    > path_index(fl_holes, cmass)
    [1] 0.11111341 0.09652468 0.08559911 0.07871328 0.07042810 0.06843499 0.06737811
    [8] 0.06584170 0.06490351
    attr(,"class")
    [1] "path_index"
    > 
    > plot(path_index(fl_holes, holes), type = "l")
    Error in geom_line() : could not find function "geom_line"
    Calls: plot -> plot.path_index -> <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tourrGui

Version: 0.4

## In both

*   checking package dependencies ... ERROR
    ```
    Package required but not available: ‘RGtk2’
    
    Package suggested but not available for checking: ‘rggobi’
    
    See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
    manual.
    ```

# toxboot

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘rmongodb’
    ```

# toxplot

Version: 0.1.1

## Newly broken

*   checking whether package ‘toxplot’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘toxplot’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/toxplot/new/toxplot.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘tidyr’
      All declared Imports should be used.
    ```

# TPD

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TPP

Version: 3.8.0

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 13551 SKIPPED: 1 FAILED: 11
      1. Failure: allOk1 (@test_create_spline_plots.R#66) 
      2. Failure: allOk2 (@test_create_spline_plots.R#87) 
      3. Failure: allOk3 (@test_create_spline_plots.R#108) 
      4. Failure: onlyAlternative (@test_create_spline_plots.R#161) 
      5. Failure: onlyNull (@test_create_spline_plots.R#185) 
      6. Failure: noHypothesisNorCondition (@test_create_spline_plots.R#209) 
      7. Failure: noData (@test_create_spline_plots.R#307) 
      8. Failure: allOk (@test_predict_and_plot_spline_models.R#27) 
      9. Failure: allOk_H0 (@test_predict_and_plot_spline_models.R#42) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking whether package ‘TPP’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘TPP’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/TPP/new/TPP.Rcheck/00install.out’ for details.
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    Fitting null models to 47 proteins (using 7 degrees of freedom)
    Fitting alternative models to 47 proteins (using 7 degrees of freedom)
    Evaluate goodness of fit of null and alternative models.
    Runtime (2 CPUs used): 16.88 secs
    
    Results table created successfully.
    
    Joining, by = "Protein_ID"
    Writing results to file: /Users/max/github/ggplot2/revdep/checks.noindex/TPP/new/TPP.Rcheck/vign_test/TPP/vignettes/NPARC_Vignette_Example/results_TPP_TR.xlsx
    File created successfully!
    
    Creating QC plots to visualize median curve fits...
    done.
    
    Creating QC plots to visualize normalization effects...
    done.
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'NPARC_analysis_of_TPP_TR_data.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 13.5Mb
      sub-directories of 1Mb or more:
        data           1.9Mb
        example_data   8.0Mb
        test_data      1.9Mb
    ```

*   checking dependencies in R code ... NOTE
    ```
    Unexported objects imported by ':::' calls:
      ‘doParallel:::.options’ ‘mefa:::rep.data.frame’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    File ‘TPP/R/TPP.R’:
      .onLoad calls:
        packageStartupMessage(msgText, "\n")
    
    See section ‘Good practice’ in '?.onAttach'.
    
    plot_fSta_distribution: no visible binding for global variable
      ‘..density..’
    plot_pVal_distribution: no visible binding for global variable
      ‘..density..’
    Undefined global functions or variables:
      ..density..
    ```

# trackeR

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    2.7Mb
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    
        append
    
    Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=57.157231,-2.104296&zoom=13&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
    Warning in sanityChecks(dat = dat, silent = silent) :
      Observations with duplicated time stamps are removed.
    Warning in sanityChecks(dat = dat, silent = silent) :
      Observations with duplicated time stamps are removed.
    Loading required package: splines
    Loading required package: Matrix
    
    Attaching package: 'fda'
    
    The following object is masked from 'package:graphics':
    
        matplot
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'trackeR.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# trackr

Version: 0.7.5

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘trackr-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: annotationText
    > ### Title: Accessor method for the annotation text (slot "annotationtext")
    > ###   of the plot object.
    > ### Aliases: annotationText annotationText,PlotFeatureSet-method
    > ###   annotationText,ggplot-method annotationText,trellis-method
    > ###   annotationText,gTree-method annotationText,PlotFeatureSet-method
    > 
    > ### ** Examples
    > 
    > library(ggplot2)
    > plt = qplot(mtcars$mpg, mtcars$cyl)
    > fs = makeFeatureSet(plt)
    Error in apply(ldf[pnames], 2, function(x) as.character(unique(x))) : 
      dim(X) must have a positive length
    Calls: makeFeatureSet ... groupInfo -> groupInfo -> .ggplotPanelLevels -> apply
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/trackr-tests.R’ failed.
    Last 13 lines of output:
      +     stopifnot(ndoc(docs) == 3)
      + } else {
      +     warning("file copy from ", rawrmdfil, " to ", rmdfil, " failed or pandoc not found. unable to perform knit_and_Record test")
      + }
      
      
      processing file: knitr_test.Rmd
        ordinary text without R code
      
      label: plot
      Quitting from lines 17-21 (knitr_test.Rmd) 
      Error in apply(ldf[pnames], 2, function(x) as.character(unique(x))) : 
        dim(X) must have a positive length
      Calls: knit_and_record ... groupInfo -> groupInfo -> .ggplotPanelLevels -> apply
      Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: histry
    Found more than one class "tbl_df" in cache; using the first, from namespace 'trackr'
    Also defined by 'tibble'
    Creating image directory at /var/folders/lb/xhxqmcrd7gv302_b1pdfykh80000gn/T//Rtmpq7HuMb/trackr_img_dir
    Quitting from lines 146-155 (Extending-trackr.Rnw) 
    Error: processing vignette 'Extending-trackr.Rnw' failed with diagnostics:
    dim(X) must have a positive length
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘RSelenium’
    ```

# TrafficBDE

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘neuralnet’
      All declared Imports should be used.
    ```

# TrajDataMining

Version: 0.1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# transcriptogramer

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.2Mb
      sub-directories of 1Mb or more:
        data   4.4Mb
        doc    1.2Mb
    ```

*   checking R code for possible problems ... NOTE
    ```
    differentiallyExpressed,Transcriptogram: no visible binding for global
      variable ‘x’
    differentiallyExpressed,Transcriptogram: no visible binding for global
      variable ‘y’
    differentiallyExpressed,Transcriptogram : <anonymous>: no visible
      binding for global variable ‘x’
    differentiallyExpressed,Transcriptogram : <anonymous>: no visible
      binding for global variable ‘y’
    Undefined global functions or variables:
      x y
    ```

# transcriptR

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# trapezoid

Version: 2.0-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    rtrapezoid: no visible global function definition for ‘runif’
    Undefined global functions or variables:
      runif
    Consider adding
      importFrom("stats", "runif")
    to your NAMESPACE file.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: trapezoid
    Loading required package: plyr
    Loading required package: ggplot2
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'trapezoid.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# TreatmentSelection

Version: 2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tree.bins

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treeclim

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘dplR’
    ```

# treecm

Version: 1.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'treecm.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# treeDA

Version: 0.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treedater

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treeio

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treemap

Version: 2.4-2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treemapify

Version: 2.5.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# treespace

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# trelliscope

Version: 0.9.8

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# trialr

Version: 0.0.1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    Iteration:  600 / 2000 [ 30%]  (Warmup)
    Iteration:  800 / 2000 [ 40%]  (Warmup)
    Iteration: 1000 / 2000 [ 50%]  (Warmup)
    Iteration: 1001 / 2000 [ 50%]  (Sampling)
    Iteration: 1200 / 2000 [ 60%]  (Sampling)
    Iteration: 1400 / 2000 [ 70%]  (Sampling)
    Iteration: 1600 / 2000 [ 80%]  (Sampling)
    Iteration: 1800 / 2000 [ 90%]  (Sampling)
    Iteration: 2000 / 2000 [100%]  (Sampling)
    
     Elapsed Time: 0.183678 seconds (Warm-up)
                   0.113529 seconds (Sampling)
                   0.297207 seconds (Total)
    
    > rstan::plot(samp, pars = 'p') # Plot the modelled response rates in the subtypes
    ci_level: 0.8 (80% intervals)
    outer_level: 0.95 (95% intervals)
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: Rcpp
    Quitting from lines 120-121 (BEBOP.Rmd) 
    Error: processing vignette 'BEBOP.Rmd' failed with diagnostics:
    missing value where TRUE/FALSE needed
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tribe

Version: 0.1.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TriMatch

Version: 0.9.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TripleR

Version: 1.5.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tRNAscanImport

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Package listed in more than one of Depends, Imports, Suggests, Enhances:
      ‘GenomicRanges’
    A package should be listed in only one of these fields.
    ```

# tRophicPosition

Version: 0.7.5

## In both

*   checking examples ... ERROR
    ```
    ...
    Running examples in ‘tRophicPosition-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: TPmodel
    > ### Title: Function to create a JAGS-based Bayesian model to calculate
    > ###   trophic position
    > ### Aliases: TPmodel
    > 
    > ### ** Examples
    > 
    > isotopeData <- generateTPData()
    > model.string <- jagsBayesianModel()
    > model <- TPmodel(data = isotopeData, model.string = model.string,
    + n.adapt = 500)
    Error: .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 108-110 (Multiple_model_calculation_of_trophic_position_in_R.Rmd) 
    Error: processing vignette 'Multiple_model_calculation_of_trophic_position_in_R.Rmd' failed with diagnostics:
    .onLoad failed in loadNamespace() for 'rjags', details:
      call: dyn.load(file, DLLpath = DLLpath, ...)
      error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so':
      dlopen(/Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
      Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/tRophicPosition/rjags/libs/rjags.so
      Reason: image not found
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘SIBER’
    ```

# tropr

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TSCAN

Version: 1.18.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'TSCAN.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
      ‘lm’
    exprmclust: no visible global function definition for ‘dist’
    plotmclust: no visible binding for global variable ‘pca_dim_1’
    plotmclust: no visible binding for global variable ‘pca_dim_2’
    plotmclust: no visible binding for global variable ‘sample_name’
    preprocess: no visible binding for global variable ‘sd’
    preprocess: no visible global function definition for ‘hclust’
    preprocess: no visible global function definition for ‘dist’
    preprocess: no visible global function definition for ‘cutree’
    preprocess: no visible global function definition for ‘aggregate’
    singlegeneplot: no visible global function definition for
      ‘fitted.values’
    singlegeneplot: no visible binding for global variable ‘predict’
    Undefined global functions or variables:
      aggregate cutree dist fitted.values hclust lm p.adjust pca_dim_1
      pca_dim_2 pchisq prcomp predict sample_name sd
    Consider adding
      importFrom("stats", "aggregate", "cutree", "dist", "fitted.values",
                 "hclust", "lm", "p.adjust", "pchisq", "prcomp", "predict",
                 "sd")
    to your NAMESPACE file.
    ```

# TSCS

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tsfknn

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tsibble

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tsiR

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TSMining

Version: 1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    Func.SAX: no visible global function definition for ‘sd’
    Func.SAX: no visible global function definition for ‘qnorm’
    Func.matrix: no visible global function definition for ‘qnorm’
    Func.motif: no visible global function definition for ‘sd’
    Func.motif: no visible global function definition for ‘na.omit’
    Func.motif: no visible global function definition for ‘combn’
    Func.motif.multivariate: no visible global function definition for
      ‘txtProgressBar’
    Func.motif.multivariate: no visible global function definition for
      ‘setTxtProgressBar’
    Undefined global functions or variables:
      combn na.omit qnorm sd setTxtProgressBar txtProgressBar
    Consider adding
      importFrom("stats", "na.omit", "qnorm", "sd")
      importFrom("utils", "combn", "setTxtProgressBar", "txtProgressBar")
    to your NAMESPACE file.
    ```

# tspmeta

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TSRchitect

Version: 1.6.0

## Newly fixed

*   checking examples ... ERROR
    ```
    ...
    > ### ** Examples
    > 
    > extdata.dir <- system.file("extdata", package="TSRchitect")
    > test.Obj <- loadTSSobj(experimentTitle="Code example", inputDir=extdata.dir,
    + isPairedBAM=TRUE, sampleNames=c("sample1-rep1", "sample1-rep2",
    + "sample2-rep1","sample2-rep2"), replicateIDs=c(1,1,2,2))
    ... loadTSSobj ...
    
    Importing paired-end reads ...
    
    
    TSS data were specified to be paired-end read alignments.
    
    Beginning import of 4 bam files ...
    
    Warning in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) :
      port 11626 cannot be opened
    Error in socketConnection(host, port, TRUE, TRUE, "a+b", timeout = timeout) : 
      cannot open the connection
    Calls: loadTSSobj ... .local -> .bpfork -> .bpforkConnect -> socketConnection
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking top-level files ... NOTE
    ```
    File
      inst/LICENSE
    will install at top-level and is not mentioned in the DESCRIPTION file.
    ```

# TSrepr

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TSS.RESTREND

Version: 0.2.13

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘gimms’
    ```

# tufterhandout

Version: 1.2.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking DESCRIPTION meta-information ... NOTE
    ```
    Malformed Description field: should contain one or more complete sentences.
    ```

# tukeytrend

Version: 0.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    
        aml
    
    
    Attaching package: 'aods3'
    
    The following object is masked from 'package:survival':
    
        rats
    
    Warning in RET$pfunction("adjusted", ...) :
      Completion with error > abseps
    Warning in RET$pfunction("adjusted", ...) :
      Completion with error > abseps
    Warning: Transformation introduced infinite values in continuous y-axis
    Warning: Transformation introduced infinite values in continuous y-axis
    Warning: Removed 12 rows containing non-finite values (stat_boxplot).
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'ExamplesTukeytrendtest.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# tvm

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# TVTB

Version: 1.6.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > # Phenotype file
    > phenoFile <- system.file("extdata", "moderate_pheno.txt", package = "TVTB")
    > phenotypes <- S4Vectors::DataFrame(read.table(phenoFile, TRUE, row.names = 1))
    > 
    > # TVTB parameters
    > tparam <- TVTBparam(Genotypes("0|0", c("0|1", "1|0"), "1|1"))
    > 
    > # Pre-process variants
    > vcf <- VariantAnnotation::readVcf(
    +     vcfFile, param = tparam, colData = phenotypes)
    > vcf <- VariantAnnotation::expand(vcf, row.names = TRUE)
    > vcf <- addFrequencies(vcf, "super_pop")
    > 
    > # Example usage ----
    > 
    > pairsInfo(vcf, "MAF", "super_pop")
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~EUR".  Please rename the columns or make a new column.
    Calls: pairsInfo ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
             TRUE       FALSE       FALSE       FALSE       FALSE       FALSE 
        rs1426654 rs150379789 rs570906312 rs538198029 rs553496066 rs574775672 
             TRUE        TRUE       FALSE       FALSE        TRUE       FALSE 
      rs140666229 rs556950130 rs575303689 rs147513140 rs187525777 rs192454382 
             TRUE       FALSE        TRUE        TRUE       FALSE       FALSE 
      rs184818838 rs566886499 rs199924625 rs555872528 rs531820822 rs201353600 
            FALSE       FALSE       FALSE       FALSE       FALSE       FALSE 
      rs550201688 rs565338261 rs201239799 rs200461129 rs146726548 
            FALSE       FALSE       FALSE       FALSE        TRUE 
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 226 SKIPPED: 2 FAILED: 1
      1. Error: all signatures work to completion (@test_pairsInfo-methods.R#28) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Overwriting INFO keys in data:
    - REF
    - HET
    - ALT
    - AAF
    - MAF
    Overwriting INFO keys in header:
    - REF
    - HET
    - ALT
    - AAF
    - MAF
    Quitting from lines 614-615 (Introduction.Rmd) 
    Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~AFR".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.2Mb
      sub-directories of 1Mb or more:
        R     2.0Mb
        doc   2.2Mb
    ```

# tweet2r

Version: 1.0

## In both

*   checking whether package ‘tweet2r’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/tweet2r/new/tweet2r.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘tweet2r’ ...
** package ‘tweet2r’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘tweet2r’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/tweet2r/new/tweet2r.Rcheck/tweet2r’

```
### CRAN

```
* installing *source* package ‘tweet2r’ ...
** package ‘tweet2r’ successfully unpacked and MD5 sums checked
** R
** byte-compile and prepare package for lazy loading
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘tweet2r’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/tweet2r/old/tweet2r.Rcheck/tweet2r’

```
# twoddpcr

Version: 1.4.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.9Mb
      sub-directories of 1Mb or more:
        data   1.2Mb
        doc    4.1Mb
    ```

# ubeR

Version: 0.1.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# UBL

Version: 0.0.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# ukbtools

Version: 0.10.1

## Newly broken

*   checking whether package ‘ukbtools’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘ukbtools’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/ukbtools/new/ukbtools.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘plyr’
      All declared Imports should be used.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 5 marked UTF-8 strings
    ```

# ukgasapi

Version: 0.13

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘ukgasapi-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: dataItemExplorer
    > ### Title: Data Item Explorer API
    > ### Aliases: dataItemExplorer
    > 
    > ### ** Examples
    > 
    > # Specify the data item(s) to enquire from API
    > dataitems <- c('Storage Injection, Actual',
    +                'Storage Withdrawal, Actual')
    > 
    > # Initialise API (requires internet connection for this step)
    > response <- dataItemExplorer(dataitems,
    +                              fromdate = '2013-10-01',
    +                              todate='2015-09-30')
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Umatrix

Version: 3.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# umx

Version: 2.6.5

## In both

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: residuals.MxModel
    > 
    > ### ** Examples
    > 
    > require(umx)
    > data(demoOneFactor)
    > latents  = c("g")
    > manifests = names(demoOneFactor)
    > m1 <- mxModel("One Factor", type = "RAM", 
    + 	manifestVars = manifests, latentVars = latents, 
    + 	mxPath(from = latents, to = manifests),
    + 	mxPath(from = manifests, arrows = 2),
    + 	mxPath(from = latents, arrows = 2, free = FALSE, values = 1.0),
    + 	mxData(cov(demoOneFactor), type = "cov", numObs = 500)
    + )
    > m1 = umxRun(m1, setLabels = TRUE, setValues = TRUE)
    Running One Factor with 10 parameters
    Error in runHelper(model, frontendStart, intervals, silent, suppressWarnings,  : 
      c++ exception (unknown reason)
    Calls: umxRun -> mxRun -> runHelper
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      6: message(e)
      7: withRestarts({
             signalCondition(cond)
             defaultHandler(cond)
         }, muffleMessage = function() NULL)
      8: withOneRestart(expr, restarts[[1L]])
      9: doWithOneRestart(return(expr), restart)
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 54 SKIPPED: 0 FAILED: 2
      1. Error: (unknown) (@test_umx_has_CI.r#41) 
      2. Error: (unknown) (@test_umx_is_RAM.r#10) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘sem’
    ```

# unvotes

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 4494 marked UTF-8 strings
    ```

# UpSetR

Version: 1.3.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  8.4Mb
      sub-directories of 1Mb or more:
        doc   7.9Mb
    ```

# useful

Version: 1.2.3

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘useful-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: plot.kmeans
    > ### Title: plot.kmeans
    > ### Aliases: plot.kmeans
    > 
    > ### ** Examples
    > 
    > 
    > k1 <- kmeans(x=iris[, 1:4], centers=3)
    > plot(k1)
    Don't know how to automatically pick scale for object of type NULL. Defaulting to continuous.
    Error: A continuous variable can not be mapped to shape
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# userfriendlyscience

Version: 0.7.1

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘userfriendlyscience-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: regrInfluential
    > ### Title: Detecting influential cases in regression analyses
    > ### Aliases: regrInfluential
    > ### Keywords: ~kwd1 ~kwd2
    > 
    > ### ** Examples
    > 
    > regrInfluential(mpg ~ hp, mtcars);
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~mpg", "~indexOfInfluentiality".  Please rename the columns or make a new column.variables: "colour" have non standard format: "~mpg", "~indexOfInfluentiality".  Please rename the columns or make a new column.
    Calls: regrInfluential ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# UsingR

Version: 2.0-6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# usmap

Version: 0.2.1

## Newly broken

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      x[1]: "~map_df[, values]"
      y[1]: "map_df[, values]"
      
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 65 SKIPPED: 0 FAILED: 7
      1. Failure: layer parameters are correct (@test-plot.R#22) 
      2. Failure: layer parameters are correct (@test-plot.R#23) 
      3. Failure: layer parameters are correct (@test-plot.R#24) 
      4. Failure: layer parameters are correct (@test-plot.R#30) 
      5. Failure: layer parameters are correct (@test-plot.R#31) 
      6. Failure: layer parameters are correct (@test-plot.R#32) 
      7. Failure: layer parameters are correct (@test-plot.R#33) 
      
      Error: testthat unit tests failed
      Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.0Mb
      sub-directories of 1Mb or more:
        extdata   6.2Mb
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1 marked UTF-8 string
    ```

# uSORT

Version: 1.6.0

## In both

*   checking whether package ‘uSORT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Note: break used in wrong context: no loop is visible 
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/uSORT/new/uSORT.Rcheck/00install.out’ for details.
    Information on the location(s) of code generating the ‘Note’s can be
    obtained by re-running with environment variable R_KEEP_PKG_SOURCE set
    to ‘yes’.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error: processing vignette 'uSORT_quick_start.Rmd' failed with diagnostics:
    there is no package called ‘BiocStyle’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  5.3Mb
      sub-directories of 1Mb or more:
        doc       2.0Mb
        extdata   3.1Mb
    ```

# valr

Version: 0.4.0

## Newly broken

*   checking whether package ‘valr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘valr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/valr/new/valr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# valuer

Version: 1.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vanddraabe

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# varian

Version: 0.2.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# variancePartition

Version: 1.10.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table, tapply,
        union, unique, unsplit, which, which.max, which.min
    
    Welcome to Bioconductor
    
        Vignettes contain introductory material; view with
        'browseVignettes()'. To cite Bioconductor, see
        'citation("Biobase")', and for packages 'citation("pkgname")'.
    
    Loading required package: iterators
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'additional_visualization.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    as.data.frame,varPartResults: no visible binding for global variable
      ‘varPart’
    sortCols,varPartResults: no visible binding for global variable
      ‘varPart’
    Undefined global functions or variables:
      varPart
    ```

# VariantAnnotation

Version: 1.26.0

## In both

*   checking examples ... ERROR
    ```
    Running examples in ‘VariantAnnotation-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: PolyPhenDb-class
    > ### Title: PolyPhenDb objects
    > ### Aliases: PolyPhen PolyPhenDb class:PolyPhenDb PolyPhenDb-class
    > ###   duplicateRSID metadata,PolyPhenDb-method columns,PolyPhenDb-method
    > ###   keys,PolyPhenDb-method select,PolyPhenDb-method
    > ### Keywords: classes methods
    > 
    > ### ** Examples
    > 
    > library(PolyPhen.Hsapiens.dbSNP131)
    Error in library(PolyPhen.Hsapiens.dbSNP131) : 
      there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/VariantAnnotation_unit_tests.R’ failed.
    Last 13 lines of output:
      4: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
        GRanges object contains 7 out-of-bound ranges located on sequence
        70477. Note that ranges located on a sequence whose length is unknown
        (NA) or on a circular sequence are not considered out-of-bound (use
        seqlengths() and isCircular() to get the lengths and circularity flags
        of the underlying sequences). You can use trim() to trim these ranges.
        See ?`trim,GenomicRanges-method` for more information.
      5: In valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
        GRanges object contains 6 out-of-bound ranges located on sequence
        70477. Note that ranges located on a sequence whose length is unknown
        (NA) or on a circular sequence are not considered out-of-bound (use
        seqlengths() and isCircular() to get the lengths and circularity flags
        of the underlying sequences). You can use trim() to trim these ranges.
        See ?`trim,GenomicRanges-method` for more information.
      Execution halted
    ```

*   checking running R code from vignettes ...
    ```
       ‘VariantAnnotation.Rnw’ ... failed
       ‘filterVcf.Rnw’ ... OK
     WARNING
    Errors in running code in vignettes:
    when running code in ‘VariantAnnotation.Rnw’
      ...
    > names(nonsyn) <- nms[idx]
    
    > rsids <- unique(names(nonsyn)[grep("rs", names(nonsyn), 
    +     fixed = TRUE)])
    
    > library(PolyPhen.Hsapiens.dbSNP131)
    
      When sourcing ‘VariantAnnotation.R’:
    Error: there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Packages suggested but not available for checking:
      ‘SIFT.Hsapiens.dbSNP132’ ‘SIFT.Hsapiens.dbSNP137’
      ‘PolyPhen.Hsapiens.dbSNP131’
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘snpStats’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    Unexported objects imported by ':::' calls:
      ‘BiocGenerics:::replaceSlots’ ‘BiocGenerics:::testPackage’
      ‘Rsamtools:::.RsamtoolsFile’ ‘Rsamtools:::.RsamtoolsFileList’
      ‘Rsamtools:::.io_check_exists’ ‘S4Vectors:::expandByColumnSet’
      ‘S4Vectors:::labeledLine’ ‘S4Vectors:::recycleVector’
      ‘S4Vectors:::selectSome’
      ‘SummarizedExperiment:::.SummarizedExperiment.charbound’
      ‘SummarizedExperiment:::.cbind.DataFrame’
      See the note in ?`:::` about the use of this operator.
    ```

*   checking R code for possible problems ... NOTE
    ```
    .predictCodingGRangesList: no visible binding for global variable
      ‘GENETIC_CODE’
    VRangesForMatching: no visible binding for global variable ‘REF’
    VRangesForMatching: no visible binding for global variable ‘ALT’
    probabilityToSnpMatrix: no visible global function definition for
      ‘post2g’
    import,VcfFile-ANY-ANY: no visible global function definition for
      ‘checkArgFormat’
    liftOver,VRanges-Chain: no visible global function definition for
      ‘update’
    Undefined global functions or variables:
      ALT GENETIC_CODE REF checkArgFormat post2g update
    Consider adding
      importFrom("stats", "update")
    to your NAMESPACE file.
    ```

*   checking compiled code ... NOTE
    ```
    File ‘VariantAnnotation/libs/VariantAnnotation.so’:
      Found non-API calls to R: ‘R_GetConnection’, ‘R_WriteConnection’
    
    Compiled code should not call non-API entry points in R.
    
    See ‘Writing portable packages’ in the ‘Writing R Extensions’ manual.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning in valid.GenomicRanges.seqinfo(x, suggest.trim = TRUE) :
      GRanges object contains 2405 out-of-bound ranges located on
      sequences 75253, 74357, 74359, 74360, 74361, 74362, 74363,
      74358, 74364, 74365, 75254, 75259, 74368, 74369, 74366, 74367,
      74370, 74372, 74373, 74374, 74375, 74378, 74377, 74380, 74381,
      75262, 75263, 75265, 75266, 75268, 75269, 75271, 75273, 75276,
      75281, 75282, 75283, 74389, 74383, 74384, 74385, 74386, 74387,
      75287, 75288, 75286, 75289, 74390, 74391, 74392, 74393, 74394,
      75291, 74395, 74396, 74397, 74398, 75302, 75304, 75305, and
      75306. Note that ranges located on a sequence whose length is
      unknown (NA) or on a circular sequence are not considered
      out-of-bound (use seqlengths() and isCircular() to get the
      lengths and circularity flags of the underlying sequences). You
      can use trim() to trim these ranges. See
      ?`trim,GenomicRanges-method` for more information.
    
    Error: processing vignette 'VariantAnnotation.Rnw' failed with diagnostics:
     chunk 32 (label = polyphen) 
    Error in library(PolyPhen.Hsapiens.dbSNP131) : 
      there is no package called ‘PolyPhen.Hsapiens.dbSNP131’
    Execution halted
    ```

# VarSelLCM

Version: 2.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vcdExtra

Version: 0.7-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘alr3’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: vcd
    Loading required package: grid
    Loading required package: gnm
    The following objects are masked _by_ .GlobalEnv:
    
        A, B, C
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vcd-tutorial.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# vcfR

Version: 1.8.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# VDAP

Version: 2.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vdg

Version: 1.2.0

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Warning in has_utility("pdfcrop") :
      pdfcrop not installed or not in PATH
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'vdg.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vdiffr

Version: 0.2.3

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > ### Aliases: expect_doppelganger
    > 
    > ### ** Examples
    > 
    > ver <- gdtools::version_freetype()
    > 
    > if (ver >= "2.6.0") {
    +   disp_hist_base <- function() hist(mtcars$disp)
    +   expect_doppelganger("disp-histogram-base", disp_hist_base)
    + }
    > 
    > if (ver >= "2.6.0" && requireNamespace("ggplot2", quietly = TRUE)) {
    +   library("ggplot2")
    +   disp_hist_ggplot <- ggplot(mtcars, aes(disp)) + geom_histogram()
    +   expect_doppelganger("disp-histogram-ggplot", disp_hist_ggplot)
    + }
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: expect_doppelganger ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vdmR

Version: 0.2.5

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘vdmR-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: vhist
    > ### Title: Generate histogram with interactive functions
    > ### Aliases: vhist
    > 
    > ### ** Examples
    > 
    > data(vsfuk2012)
    > vhist(FertilityRate, vsfuk2012, "hist1", "vsfuk2012", fill=Type)
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Error: $ operator is invalid for atomic vectors
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    `stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
    Quitting from lines 153-155 (vdmR-vignette.Rnw) 
    Error: processing vignette 'vdmR-vignette.Rnw' failed with diagnostics:
    $ operator is invalid for atomic vectors
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘Rdpack’ ‘maptools’ ‘rgeos’
      All declared Imports should be used.
    ```

# VetResearchLMM

Version: 1.0.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘nlme’
      All declared Imports should be used.
    ```

# vidger

Version: 1.0.0

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘vidger-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: vsScatterMatrix
    > ### Title: Scatter plot matrix of log10(FPKM or CPM) values
    > ### Aliases: vsScatterMatrix
    > 
    > ### ** Examples
    > 
    > # Cuffdiff example
    > data("df.cuff")
    > vsScatterMatrix(
    +  data = df.cuff, d.factor = NULL, type = 'cuffdiff', 
    +  comp = NULL, title = TRUE, grid = TRUE, 
    +  man.title = 'Example title'
    + )
    Error in make_ggmatrix_plot_obj(wrapp(sub_type, funcArgName = sub_type_name),  : 
      variables: "x" have non standard format: "~hESC".  Please rename the columns or make a new column.
    Calls: vsScatterMatrix ... do.call -> <Anonymous> -> plot_fn -> make_ggmatrix_plot_obj
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    The following objects are masked from 'package:base':
    
        aperm, apply
    
    Loading required package: edgeR
    Loading required package: limma
    
    Attaching package: 'limma'
    
    The following object is masked from 'package:DESeq2':
    
        plotMA
    
    The following object is masked from 'package:BiocGenerics':
    
        plotMA
    
    Quitting from lines 270-274 (vidger.Rmd) 
    Error: processing vignette 'vidger.Rmd' failed with diagnostics:
    variables: "x" have non standard format: "~hESC".  Please rename the columns or make a new column.
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  6.6Mb
      sub-directories of 1Mb or more:
        data   4.7Mb
        doc    1.7Mb
    ```

# vipor

Version: 0.4.5

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    Loading required package: sm
    Warning in fun(libname, pkgname) : couldn't connect to display ""
    Package 'sm', version 2.2-5.5: type help(sm) for summary information
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'methodComparison.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# viridis

Version: 0.5.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘stats’
      All declared Imports should be used.
    ```

# viridisLite

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# visdat

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# visreg

Version: 2.5-0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# visvow

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vlad

Version: 0.1.0

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    > 
    > # Plot a VLAD/CRAM chart
    > library("spcadjust")
    > data("cardiacsurgery")
    > cardiacsurgery <- dplyr::mutate(cardiacsurgery, phase=factor(ifelse(date < 2*365, "I", "II")))
    > S2 <- subset(cardiacsurgery, c(surgeon==2), c("phase", "Parsonnet", "status"))
    > S2I <- subset(S2, c(phase=="I"))
    > S2II <- subset(S2, c(phase=="II"))
    > coeff <- coef(glm(status ~ Parsonnet, data=S2I, family="binomial"))
    > EO <- sapply(1:nrow(S2), function(i) calceo(df=S2[i, c("Parsonnet", "status")], coeff=coeff))
    > 
    > df1 <- data.frame(cbind(subset(S2, select=c("phase")), "n"=1:nrow(S2), "cEO"=cumsum(EO)))
    > df2 <- tidyr::gather(df1, "variable", value, c(-n, -phase))
    > 
    > ggplot2::qplot(data=df2, n, value, colour=phase, geom=c("line", "point"),
    +                xlab="Patient number", ylab="CUSUM E-O") +
    +   ggplot2::geom_hline(yintercept=0, linetype="dashed") +
    +   ggplot2::theme_classic()
    Error in geom_line() : could not find function "geom_line"
    Calls: <Anonymous> -> do.call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vmd

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vortexR

Version: 1.1.4

## In both

*   checking whether package ‘vortexR’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/vortexR/new/vortexR.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘vortexR’ ...
** package ‘vortexR’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘vortexR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/vortexR/new/vortexR.Rcheck/vortexR’

```
### CRAN

```
* installing *source* package ‘vortexR’ ...
** package ‘vortexR’ successfully unpacked and MD5 sums checked
** R
** data
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/vortexR/rJava/libs/rJava.so
  Reason: image not found
ERROR: lazy loading failed for package ‘vortexR’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/vortexR/old/vortexR.Rcheck/vortexR’

```
# voteogram

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# voxel

Version: 1.3.4

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    
    > ### Name: plotGAMM
    > ### Title: GAMM plotting using ggplot2
    > ### Aliases: plotGAMM
    > 
    > ### ** Examples
    > 
    > 
    > set.seed(1)
    > data <- data.frame(x = (seq(.25,25, .25) +rnorm(100)), group = rep(1:2, 5), z=rnorm(100),
    +               index.rnorm = rep(rnorm(50, sd = 50), 2), index.var = rep(1:50, 2))
    > data$y <- (data$x)*data$group*10 + rnorm(100, sd = 700) + data$index.rnorm + data$z
    > data$group <- ordered(data$group)
    > gamm <- gamm4::gamm4(y ~ + s(x) + s(x, by=group) + z + group, data=data, random = ~ (1|index.var))
    > plot <- plotGAMM(gammFit <- gamm, smooth.cov <- "x", groupCovs = "group", 
    +                     plotCI <- T, rawOrFitted = "raw", grouping = "index.var")
    [1] "Working with a GAMM4 object"
    [1] "orderedAsFactor functionality temporally disabled"
    > plot
    Error: Column `x` must be a 1d atomic vector or a list
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘methods’
      All declared Imports should be used.
    ```

# vpc

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vqtl

Version: 2.0.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘iterators’ ‘knitr’ ‘purrr’ ‘testthat’
      All declared Imports should be used.
    ```

# vrcp

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# VRPM

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# VSE

Version: 0.99

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# vsn

Version: 3.48.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    ...
    Warning: replacing previous import 'AnnotationDbi::head' by 'utils::head' when loading 'hgu95av2cdf'
    
    
    Attaching package: 'limma'
    
    The following object is masked from 'package:BiocGenerics':
    
        plotMA
    
    Warning in xy.coords(x, y, xlabel, ylabel, log) :
      1 x value <= 0 omitted from logarithmic plot
    Warning in xy.coords(x, y, xlabel, ylabel, log) :
      603 y values <= 0 omitted from logarithmic plot
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/nkid-calib4-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/nkid-calib5-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/vsnQ1-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    convert: profile 'icc': 'RGB ': RGB color space not permitted on grayscale PNG `A-vsn_files/figure-html/vsnQ2-1.png' @ warning/png.c/MagickPNGWarningHandler/1744.
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'C-likelihoodcomputations.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# vtreat

Version: 1.0.4

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# VWPre

Version: 1.1.0

## Newly broken

*   checking whether package ‘VWPre’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘VWPre’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/VWPre/new/VWPre.Rcheck/00install.out’ for details.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
    These occur at time bin(s): 1150
    Subsequent Empirical Logit calculations may be influenced by the number of samples (depending on the number of observations requested).
    Sampling rate(s) present in the data are: 20 Hz.
    
    There are 50 samples per bin.
    One data point every 50 millisecond(s)
    Preparing ELogit output...
    Number of Observations equal to Number of Samples. 
     Calculation will be based on Number of Samples.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Grand average of Proportion Looks calculated using Event means.
    Quitting from lines 124-139 (SR_Plotting.Rmd) 
    Error: processing vignette 'SR_Plotting.Rmd' failed with diagnostics:
    `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# waffle

Version: 0.7.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wakefield

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# walker

Version: 0.2.1

## In both

*   checking examples ... ERROR
    ```
    ...
    
    WARNING: There aren't enough warmup iterations to fit the
             three stages of adaptation as currently configured.
             Reducing each adaptation stage to 15%/75%/10% of
             the given number of warmup iterations:
               init_buffer = 3
               adapt_window = 20
               term_buffer = 2
    
    [1] "Error in sampler$call_sampler(args_list[[i]]) : "
    [2] "  c++ exception (unknown reason)"                
    error occurred during calling the sampler; sampling not done
    >   
    > g_y <- geom_point(data = data.frame(y = Nile, x = time(Nile)), 
    +   aes(x, y, alpha = 0.5), inherit.aes = FALSE) 
    > g_rw1 <- plot_coefs(rw1_fit) + g_y
    > g_rw2 <- plot_coefs(rw2_fit) + g_y
    Stan model 'walker_lm' does not contain samples.
    Error in 1:nrow(coef_data) : argument of length 0
    Calls: plot_coefs
    Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Quitting from lines 124-128 (walker.Rmd) 
    Error: processing vignette 'walker.Rmd' failed with diagnostics:
    argument of length 0
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is  7.9Mb
      sub-directories of 1Mb or more:
        doc    2.2Mb
        libs   5.4Mb
    ```

# walkr

Version: 0.3.4

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'walkr.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# walrus

Version: 1.0.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘R6’ ‘WRS2’ ‘ggplot2’
      All declared Imports should be used.
    ```

# warbleR

Version: 1.1.12

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘pbmcapply’ ‘proxy’ ‘signal’
      All declared Imports should be used.
    ```

# waterfalls

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Wats

Version: 0.10.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking installed package size ... NOTE
    ```
      installed size is 12.3Mb
      sub-directories of 1Mb or more:
        doc  12.1Mb
    ```

# wavClusteR

Version: 2.14.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    'library' or 'require' call to ‘doMC’ in package code.
      Please use :: or requireNamespace() instead.
      See section 'Suggested packages' in the 'Writing R Extensions' manual.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    plotSubstitutions: no visible global function definition for ‘barplot’
    processChunk: no visible global function definition for ‘extractAt’
    processMD: no visible global function definition for ‘registerDoMC’
    readSortedBam: no visible global function definition for ‘scanBamFlag’
    readSortedBam : <anonymous>: no visible binding for global variable
      ‘rname’
    readSortedBam : <anonymous>: no visible binding for global variable
      ‘qwidth’
    Undefined global functions or variables:
      Compartment DNAString DNAStringSet Percentage abline axis barplot
      dbinom extractAt grid hist legend lines pairs panel.smooth par
      polygon qwidth rect registerDoMC rname scanBamFlag seqlevels
      setTxtProgressBar strwidth text txtProgressBar write.table
    Consider adding
      importFrom("graphics", "abline", "axis", "barplot", "grid", "hist",
                 "legend", "lines", "pairs", "panel.smooth", "par",
                 "polygon", "rect", "strwidth", "text")
      importFrom("stats", "dbinom")
      importFrom("utils", "setTxtProgressBar", "txtProgressBar",
                 "write.table")
    to your NAMESPACE file.
    ```

# wbstats

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking data for non-ASCII characters ... NOTE
    ```
      Note: found 1528 marked UTF-8 strings
    ```

# webr

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘moonBook’
      All declared Imports should be used.
    ```

# wesanderson

Version: 0.3.6

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# WHO

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# widyr

Version: 0.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wiggleplotr

Version: 1.4.0

## In both

*   checking for code/documentation mismatches ... WARNING
    ```
    Codoc mismatches from documentation object 'getGenotypePalette':
    getGenotypePalette
      Code: function(old = FALSE)
      Docs: function()
      Argument names in code not in docs:
        old
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    plotCoverage: no visible global function definition for ‘is’
    plotTranscripts: no visible global function definition for ‘is’
    Undefined global functions or variables:
      is
    Consider adding
      importFrom("methods", "is")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# wikipediatrend

Version: 1.1.14

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# windfarmGA

Version: 1.2.1

## In both

*   checking whether package ‘windfarmGA’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/windfarmGA/new/windfarmGA.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘windfarmGA’ ...
** package ‘windfarmGA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Creating a generic function for ‘toJSON’ from package ‘jsonlite’ in package ‘googleVis’
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘windfarmGA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/windfarmGA/new/windfarmGA.Rcheck/windfarmGA’

```
### CRAN

```
* installing *source* package ‘windfarmGA’ ...
** package ‘windfarmGA’ successfully unpacked and MD5 sums checked
** R
** inst
** byte-compile and prepare package for lazy loading
Warning in fun(libname, pkgname) : couldn't connect to display ""
Creating a generic function for ‘toJSON’ from package ‘jsonlite’ in package ‘googleVis’
Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘polyclip’
ERROR: lazy loading failed for package ‘windfarmGA’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/windfarmGA/old/windfarmGA.Rcheck/windfarmGA’

```
# wordbankr

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dbplyr’
      All declared Imports should be used.
    ```

# WordR

Version: 0.3.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wppExplorer

Version: 2.1-1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wql

Version: 0.4.9

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wrswoR

Version: 1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# wrswoR.benchmark

Version: 0.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# WRTDStidal

Version: 1.1.0

## Newly broken

*   checking whether package ‘WRTDStidal’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘WRTDStidal’
      Warning: replacing previous import ‘forecast::autolayer’ by ‘ggplot2::autolayer’ when loading ‘WRTDStidal’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/WRTDStidal/new/WRTDStidal.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# WVPlots

Version: 0.3.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# x.ent

Version: 1.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# XBSeq

Version: 1.12.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking whether the namespace can be loaded with stated dependencies ... NOTE
    ```
    Warning: no function found corresponding to methods exports from ‘XBSeq’ for: ‘conditions’, ‘conditions<-’, ‘dispTable’
    
    A namespace must be able to be loaded with just the base namespace
    loaded: otherwise if the namespace gets loaded by a saved object, the
    session will be unable to start.
    
    Probably some imports need to be declared in the NAMESPACE file.
    ```

*   checking R code for possible problems ... NOTE
    ```
    ...
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay’
    estimateRealCount,XBSeqDataSet: no visible global function definition
      for ‘assay<-’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘conditions’
    estimateSCV,XBSeqDataSet: no visible global function definition for
      ‘dispTable<-’
    Undefined global functions or variables:
      ..count.. DataFrame Gamma Group Sample SummarizedExperiment assay
      assay<- assays baseMean coefficients complete.cases conditions cor
      data ddelap dispTable dispTable<- dnbinom dpois formula glm
      log2FoldChange median optim p.adjust pbeta predict qbeta quantile
      rnbinom scvBiasCorrectionFits
    Consider adding
      importFrom("stats", "Gamma", "coefficients", "complete.cases", "cor",
                 "dnbinom", "dpois", "formula", "glm", "median", "optim",
                 "p.adjust", "pbeta", "predict", "qbeta", "quantile",
                 "rnbinom")
      importFrom("utils", "data")
    to your NAMESPACE file.
    ```

# xgboost

Version: 0.6.4.1

## In both

*   checking examples ... WARNING
    ```
    Found the following significant warnings:
    
      Warning: 'cBind' is deprecated.
    Deprecated functions may be defunct as soon as of the next release of
    R.
    See ?Deprecated.
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Loading required package: xgboost
    Loading required package: Matrix
    Loading required package: data.table
    Loading required package: vcd
    Loading required package: grid
       [[ suppressing 10 column names 'TreatmentPlacebo', 'TreatmentTreated', 'SexMale' ... ]]
    Warning in chisq.test(df$Age, output_vector) :
      Chi-squared approximation may be incorrect
    Warning in chisq.test(df$AgeDiscret, output_vector) :
      Chi-squared approximation may be incorrect
    Warning in chisq.test(df$AgeCat, output_vector) :
      Chi-squared approximation may be incorrect
    Loading required package: knitr
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'xgboost.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# XGR

Version: 1.1.3

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# xkcd

Version: 0.0.5

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘xkcd-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: xkcdaxis
    > ### Title: Plot the axis
    > ### Aliases: xkcdaxis
    > 
    > ### ** Examples
    > 
    > xrange <- range(mtcars$mpg)
    > yrange <- range(mtcars$wt)
    > p <- ggplot() +
    +      geom_point(aes(mpg, wt), data=mtcars) +
    +      xkcdaxis(xrange,yrange)
    Error in as.data.frame.default(x[[i]], optional = TRUE) : 
      cannot coerce class ‘c("quosure", "formula")’ to a data.frame
    Calls: xkcdaxis ... <Anonymous> -> as.data.frame -> as.data.frame.default
    Execution halted
    ```

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'xkcd-intro.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# XLConnect

Version: 0.2-15

## In both

*   checking whether package ‘XLConnect’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/XLConnect/new/XLConnect.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘XLConnect’ ...
** package ‘XLConnect’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘XLConnectJars’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘XLConnectJars’ could not be loaded
ERROR: lazy loading failed for package ‘XLConnect’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/XLConnect/new/XLConnect.Rcheck/XLConnect’

```
### CRAN

```
* installing *source* package ‘XLConnect’ ...
** package ‘XLConnect’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** demo
** inst
** byte-compile and prepare package for lazy loading
Error: package or namespace load failed for ‘XLConnectJars’:
 .onLoad failed in loadNamespace() for 'rJava', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so, 6): Library not loaded: /Library/Java/JavaVirtualMachines/jdk-9.jdk/Contents/Home/lib/server/libjvm.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/XLConnect/rJava/libs/rJava.so
  Reason: image not found
Error : package ‘XLConnectJars’ could not be loaded
ERROR: lazy loading failed for package ‘XLConnect’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/XLConnect/old/XLConnect.Rcheck/XLConnect’

```
# xmrr

Version: 1.0.1

## Newly broken

*   checking whether package ‘xmrr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘xmrr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/xmrr/new/xmrr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# Xplortext

Version: 1.1.1

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# xpose

Version: 0.4.2

## Newly broken

*   checking examples ... ERROR
    ```
    Running examples in ‘xpose-Ex.R’ failed
    The error most likely occurred in:
    
    > ### Name: amt_vs_idv
    > ### Title: Compartment kinetics
    > ### Aliases: amt_vs_idv
    > 
    > ### ** Examples
    > 
    > amt_vs_idv(xpdb_ex_pk)
    Using data from $prob no.1
    Tidying data by ID, SEX, MED1, MED2, AMT ... and 20 more variables
    Warning: x is not part of the available keywords. Check ?template_titles for a full list.
    Error in if (!(tag_pos == "manual" || tag_pos %in% valid_pos)) { : 
      missing value where TRUE/FALSE needed
    Calls: <Anonymous> ... print.ggplot -> ggplot_gtable -> ggplot_gtable.ggplot_built
    Execution halted
    ```

*   checking tests ...
    ```
     ERROR
    Running the tests in ‘tests/testthat.R’ failed.
    Last 13 lines of output:
      ══ testthat results  ═══════════════════════════════════════════════════════════
      OK: 380 SKIPPED: 6 FAILED: 39
      1. Error: (unknown) (@test-console_outputs.R#4) 
      2. Error: (unknown) (@test-edits.R#4) 
      3. Failure: have proper error check for distribution_function res_distrib (@test-plots.R#72) 
      4. Failure: have proper error check for distribution_function res_distrib (@test-plots.R#74) 
      5. Error: have proper error check for distribution_function res_distrib 
      6. Failure: have proper error check for distribution_function res_qq (@test-plots.R#72) 
      7. Failure: have proper error check for distribution_function res_qq (@test-plots.R#74) 
      8. Error: have proper error check for distribution_function res_qq 
      9. Failure: have proper error check for distribution_function prm_distrib (@test-plots.R#72) 
      1. ...
      
      Error: testthat unit tests failed
      Execution halted
    ```

*   checking re-building of vignette outputs ... WARNING
    ```
    Error in re-building vignettes:
      ...
    Returning data from run001.ext, $prob no.1, subprob no.0, method foce
    Returning parameter estimates from $prob no.1, subprob no.0, method foce
    Quitting from lines 13-26 (customize_plots.Rmd) 
    Error: processing vignette 'customize_plots.Rmd' failed with diagnostics:
    no applicable method for 'mutate_' applied to an object of class "uneval"
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# xray

Version: 0.2

## Newly broken

*   checking whether package ‘xray’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘xray’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/xray/new/xray.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# xsp

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# xtractomatic

Version: 3.4.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘dplyr’
      All declared Imports should be used.
    ```

# xxIRT

Version: 2.0.3

## Newly broken

*   checking whether package ‘xxIRT’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘xxIRT’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/xxIRT/new/xxIRT.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘graphics’
      All declared Imports should be used.
    ```

# YAPSA

Version: 1.6.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# yorkr

Version: 0.0.7

## Newly broken

*   checking whether package ‘yorkr’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘yorkr’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/yorkr/new/yorkr.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# yuimaGUI

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# zebu

Version: 0.1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# zenplots

Version: 0.0-3

## In both

*   checking re-building of vignette outputs ... WARNING
    ```
    ...
        parLapplyLB, parRapply, parSapply, parSapplyLB
    
    The following objects are masked from 'package:stats':
    
        IQR, mad, sd, var, xtabs
    
    The following objects are masked from 'package:base':
    
        Filter, Find, Map, Position, Reduce, anyDuplicated, append,
        as.data.frame, basename, cbind, colMeans, colSums, colnames,
        dirname, do.call, duplicated, eval, evalq, get, grep, grepl,
        intersect, is.unsorted, lapply, lengths, mapply, match, mget,
        order, paste, pmax, pmax.int, pmin, pmin.int, rank, rbind,
        rowMeans, rowSums, rownames, sapply, setdiff, sort, table,
        tapply, union, unique, unsplit, which, which.max, which.min
    
    Loading required package: grid
    Quitting from lines 219-227 (selected_features.Rmd) 
    Error: processing vignette 'selected_features.Rmd' failed with diagnostics:
    package 'rJava' could not be loaded
    Execution halted
    ```

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# zFactor

Version: 0.1.7

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespace in Imports field not imported from: ‘rootSolve’
      All declared Imports should be used.
    ```

# zFPKM

Version: 1.2.0

## Newly broken

*   checking whether package ‘zFPKM’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘dplyr::vars’ by ‘ggplot2::vars’ when loading ‘zFPKM’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/zFPKM/new/zFPKM.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking R code for possible problems ... NOTE
    ```
    PlotGaussianFitDF: no visible global function definition for ‘dnorm’
    PlotGaussianFitDF: no visible binding for global variable ‘density’
    PlotGaussianFitDF: no visible binding for global variable ‘log2fpkm’
    PlotGaussianFitDF: no visible binding for global variable ‘sample_name’
    zFPKMCalc: no visible global function definition for ‘density’
    zFPKMTransform: no visible global function definition for ‘is’
    Undefined global functions or variables:
      density dnorm is log2fpkm sample_name
    Consider adding
      importFrom("methods", "is")
      importFrom("stats", "density", "dnorm")
    to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
    contains 'methods').
    ```

# zinbwave

Version: 1.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# zonator

Version: 0.5.9

## Newly broken

*   checking whether package ‘zonator’ can be installed ... WARNING
    ```
    Found the following significant warnings:
      Warning: replacing previous import ‘ggplot2::calc’ by ‘raster::calc’ when loading ‘zonator’
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/zonator/new/zonator.Rcheck/00install.out’ for details.
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking package dependencies ... NOTE
    ```
    Package suggested but not available for checking: ‘zdat’
    ```

# zoo

Version: 1.8-1

## Newly broken

*   checking examples ... ERROR
    ```
    ...
    + ## with different line types
    + autoplot(zoo(data), facet = NULL) + geom_point() + aes(linetype = Series)
    + 
    + ## illustrate just fortify() method
    + z <- zoo(data)
    + fortify(z)
    + fortify(z, melt = TRUE)
    + fortify(z, melt = TRUE, names = c("Time", NA, "Data"))
    + fortify(z, melt = TRUE, names = c(Index = "Time"))
    + 
    + ## with/without splitting
    + z <- zoo(cbind(a.A = 1:2, a.B = 2:3, b.A = 3:4, c.B = 4:5))
    + fortify(z)
    + fortify(z, melt = TRUE, sep = ".",
    +   names = list(Series = c("Lower", "Upper")))
    + 
    + }
    Loading required package: ggplot2
    Loading required package: scales
    Error: `expr` must quote a symbol, scalar, or call
    Execution halted
    ```

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd cross-references ... NOTE
    ```
    Package unavailable to check Rd xrefs: ‘stinepack’
    ```

*   checking re-building of vignette outputs ... NOTE
    ```
    Error in re-building vignettes:
      ...
    
    Attaching package: ‘zoo’
    
    The following objects are masked from ‘package:base’:
    
        as.Date, as.Date.numeric
    
    Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
      Running 'texi2dvi' on 'zoo-design.tex' failed.
    Calls: buildVignettes -> texi2pdf -> texi2dvi
    Execution halted
    ```

# zooaRch

Version: 1.2

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

# zooaRchGUI

Version: 1.0.2

## In both

*   checking whether package ‘zooaRchGUI’ can be installed ... ERROR
    ```
    Installation failed.
    See ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/new/zooaRchGUI.Rcheck/00install.out’ for details.
    ```

## Installation

### Devel

```
* installing *source* package ‘zooaRchGUI’ ...
** package ‘zooaRchGUI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘zooaRchGUI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/new/zooaRchGUI.Rcheck/zooaRchGUI’

```
### CRAN

```
* installing *source* package ‘zooaRchGUI’ ...
** package ‘zooaRchGUI’ successfully unpacked and MD5 sums checked
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
Error : .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so':
  dlopen(/Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/max/github/ggplot2/revdep/library.noindex/zooaRchGUI/rjags/libs/rjags.so
  Reason: image not found
ERROR: lazy loading failed for package ‘zooaRchGUI’
* removing ‘/Users/max/github/ggplot2/revdep/checks.noindex/zooaRchGUI/old/zooaRchGUI.Rcheck/zooaRchGUI’

```
# zoocat

Version: 0.2.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking Rd \usage sections ... NOTE
    ```
    S3 methods shown with full name in documentation object 'merge':
      ‘cbind.zoocat’
    
    The \usage entries for S3 methods should use the \method markup and not
    their full name.
    See chapter ‘Writing R documentation files’ in the ‘Writing R
    Extensions’ manual.
    ```

# ztype

Version: 0.1.0

## In both

*   checking PDF version of manual ... WARNING
    ```
    LaTeX errors when creating PDF version.
    This typically indicates Rd problems.
    ```

*   checking dependencies in R code ... NOTE
    ```
    Namespaces in Imports field not imported from:
      ‘dplyr’ ‘ggplot2’ ‘lubridate’
      All declared Imports should be used.
    ```

