# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.1 (2016-06-21) |
|system   |x86_64, darwin13.4.0         |
|ui       |X11                          |
|language |(EN)                         |
|collate  |en_US.UTF-8                  |
|tz       |America/Chicago              |
|date     |2016-10-04                   |

## Packages

|package |*  |version    |date       |source                       |
|:-------|:--|:----------|:----------|:----------------------------|
|ggplot2 |   |2.1.0.9001 |2016-10-04 |local (hadley/ggplot2@NA)    |
|knitr   |   |1.14.9     |2016-10-04 |Github (yihui/knitr@63407ab) |

# Check results
171 packages with problems

|package                |version       | errors| warnings| notes|
|:----------------------|:-------------|------:|--------:|-----:|
|ACDm                   |1.0.4         |      1|        0|     0|
|aop                    |1.0.0         |      1|        0|     0|
|archivist              |2.1           |      1|        0|     0|
|BACA                   |1.3           |      1|        0|     0|
|BACCT                  |1.0           |      1|        0|     0|
|backShift              |0.1.3         |      1|        0|     0|
|bamdit                 |3.0.0         |      1|        0|     0|
|bayesAB                |0.6.0         |      1|        0|     0|
|BCEA                   |2.2-3         |      1|        0|     1|
|Bclim                  |3.1.0         |      0|        1|     0|
|bcrm                   |0.4.6         |      1|        0|     1|
|biogram                |1.3           |      0|        1|     0|
|BioPET                 |0.2.0         |      1|        0|     0|
|BlandAltmanLeh         |0.3.1         |      0|        1|     0|
|blscrapeR              |0.4.2         |      1|        0|     0|
|brainGraph             |0.62.0        |      1|        0|     0|
|brms                   |1.0.1         |      1|        0|     0|
|bsam                   |1.0.0         |      1|        0|     0|
|BTSPAS                 |2014.0901     |      1|        0|     1|
|cate                   |1.0.4         |      1|        0|     0|
|classify               |1.3           |      1|        0|     0|
|clifro                 |3.0-0         |      0|        1|     0|
|clusterfly             |0.4           |      1|        0|     0|
|coloc                  |2.3-1         |      0|        1|     2|
|colorplaner            |0.1.1         |      1|        0|     0|
|CopulaDTA              |0.0.4         |      0|        1|     0|
|corrr                  |0.2.0         |      1|        0|     0|
|cowplot                |0.6.3         |      1|        2|     0|
|crmPack                |0.2.0         |      1|        0|     0|
|Crossover              |0.1-16        |      1|        1|     0|
|Cyclops                |1.2.0         |      1|        0|     0|
|data.table             |1.9.6         |      1|        0|     1|
|ddpcr                  |1.5           |      0|        1|     0|
|demi                   |1.1.2         |      1|        0|     0|
|dfpk                   |1.0           |      1|        0|     0|
|dtwSat                 |0.2.1         |      0|        1|     0|
|dynr                   |0.1.7         |      1|        0|     0|
|eechidna               |0.1           |      1|        0|     0|
|eeptools               |0.9.1         |      2|        1|     1|
|emojifont              |0.3.4         |      0|        1|     1|
|EmpiricalCalibration   |1.2.0         |      1|        0|     0|
|factoextra             |1.0.3         |      1|        0|     1|
|fbroc                  |0.4.0         |      1|        0|     0|
|fheatmap               |1.0.1         |      1|        0|     1|
|fitbitScraper          |0.1.7         |      0|        1|     0|
|forecast               |7.2           |      1|        0|     0|
|fuzzyforest            |1.0.2         |      1|        0|     0|
|gapfill                |0.9.3         |      2|        0|     2|
|GenCAT                 |1.0.3         |      1|        1|     3|
|geomnet                |0.1.2.1       |      1|        0|     0|
|GGally                 |1.2.0         |      2|        0|     0|
|ggalt                  |0.1.1         |      1|        0|     0|
|ggenealogy             |0.2.0         |      0|        1|     0|
|ggExtra                |0.5           |      1|        1|     0|
|ggfortify              |0.2.0         |      1|        0|     1|
|ggnetwork              |0.5.1         |      1|        1|     0|
|ggpmisc                |0.2.10        |      0|        1|     0|
|ggpubr                 |0.1.0         |      1|        0|     0|
|ggswissmaps            |0.0.9         |      0|        1|     1|
|ggtern                 |2.1.4         |      1|        0|     0|
|gitter                 |1.1.1         |      1|        0|     0|
|glycanr                |0.3.0         |      2|        0|     1|
|gmwm                   |2.0.0         |      0|        1|     1|
|gridDebug              |0.5-0         |      1|        0|     0|
|heatmaply              |0.5.0         |      1|        1|     0|
|hyperSpec              |0.98-20150304 |      1|        1|     1|
|IAPWS95                |1.0.0         |      1|        0|     0|
|ibmdbR                 |1.47.1        |      1|        0|     0|
|ie2miscdata            |1.0.1         |      0|        1|     1|
|imageData              |0.1-21        |      1|        0|     0|
|imager                 |0.31          |      1|        1|     1|
|IntClust               |0.0.2         |      1|        0|     0|
|IntegratedJM           |1.4           |      1|        0|     0|
|JWileymisc             |0.2.1         |      0|        1|     0|
|LANDD                  |1.1.0         |      1|        0|     0|
|largeVis               |0.1.9.1       |      0|        1|     0|
|LDheatmap              |0.99-2        |      1|        0|     0|
|localgauss             |0.35          |      1|        0|     0|
|ltbayes                |0.3           |      1|        0|     0|
|magick                 |0.3           |      1|        0|     0|
|manhattanly            |0.1.0         |      0|        1|     0|
|mcprofile              |0.2-1         |      1|        1|     1|
|MendelianRandomization |0.2.0         |      1|        0|     0|
|MergeGUI               |0.2-1         |      1|        0|     0|
|MetaIntegrator         |1.0.3         |      1|        0|     0|
|metaMix                |0.2           |      1|        0|     0|
|MissingDataGUI         |0.2-5         |      1|        0|     0|
|mixOmics               |6.1.0         |      1|        0|     0|
|MixSIAR                |3.1.7         |      2|        0|     0|
|mlr                    |2.9           |      2|        0|     1|
|Momocs                 |1.0.0         |      1|        1|     0|
|morse                  |2.2.0         |      1|        0|     0|
|mosaic                 |0.14.4        |      1|        1|     2|
|mrMLM                  |1.3           |      1|        0|     0|
|mtconnectR             |1.0.1         |      1|        0|     0|
|multiDimBio            |1.0.1         |      1|        0|     0|
|mvdalab                |1.1           |      1|        0|     0|
|mwaved                 |1.1.4         |      1|        0|     0|
|myTAI                  |0.4.0         |      1|        0|     0|
|NFP                    |0.99.1        |      1|        0|     0|
|NMF                    |0.20.6        |      1|        1|     4|
|nparACT                |0.6           |      1|        0|     0|
|npregfast              |1.3.0         |      1|        0|     0|
|onlineCPD              |1.0           |      1|        0|     0|
|OriGen                 |1.4.3         |      1|        0|     0|
|OutbreakTools          |0.1-14        |      1|        1|     0|
|pcadapt                |3.0.2         |      0|        1|     1|
|pcaPA                  |2.0.2         |      1|        0|     0|
|pcrsim                 |1.0.0         |      1|        0|     0|
|PGRdup                 |0.2.2.1       |      0|        1|     2|
|photobiologyInOut      |0.4.11        |      0|        1|     0|
|photobiologyPlants     |0.4.0         |      0|        1|     0|
|PKgraph                |1.7           |      1|        0|     0|
|PlasmaMutationDetector |1.5.2         |      1|        0|     0|
|playwith               |0.9-54        |      1|        0|     1|
|plotluck               |1.0.0         |      1|        1|     0|
|plotly                 |4.5.2         |      2|        0|     0|
|prcbench               |0.5.0         |      2|        1|     0|
|pRF                    |1.2           |      1|        0|     0|
|proportion             |1.2.3         |      0|        1|     1|
|PSCBS                  |0.61.0        |      1|        0|     0|
|quanteda               |0.9.8         |      1|        0|     0|
|quickpsy               |0.1.3         |      1|        0|     0|
|radiant.model          |0.6.0         |      1|        0|     0|
|rangeMapper            |0.3-0         |      1|        0|     0|
|rnoaa                  |0.6.0         |      0|        1|     1|
|robCompositions        |2.0.2         |      0|        1|     1|
|RPPanalyzer            |1.4.3         |      1|        0|     0|
|RSA                    |0.9.10        |      0|        1|     0|
|RSSL                   |0.6           |      1|        0|     0|
|rstan                  |2.12.1        |      1|        0|     0|
|rstanarm               |2.12.1        |      1|        0|     0|
|rsvg                   |1.0           |      1|        0|     0|
|Rz                     |0.9-1         |      0|        1|     2|
|scmamp                 |0.2.5         |      1|        0|     0|
|sdmvspecies            |0.3.2         |      0|        1|     0|
|seewave                |2.0.4         |      1|        0|     0|
|SensusR                |2.0.0         |      1|        0|     0|
|SeqFeatR               |0.2.3         |      1|        0|     0|
|sgd                    |1.1           |      1|        0|     0|
|shiny                  |0.14          |      1|        0|     1|
|simmr                  |0.3           |      1|        0|     0|
|sjPlot                 |2.1.0         |      1|        0|     0|
|snpEnrichment          |1.7.0         |      1|        0|     0|
|soc.ca                 |0.7.3         |      1|        0|     0|
|SpaDES                 |1.2.0         |      1|        0|     1|
|sparkTable             |1.2.0         |      1|        0|     1|
|specmine               |1.0           |      1|        0|     0|
|ss3sim                 |0.9.2         |      1|        0|     0|
|StroupGLMM             |0.1.0         |      1|        0|     0|
|structSSI              |1.1.1         |      1|        0|     0|
|strvalidator           |1.7.0         |      1|        0|     0|
|subspaceMOA            |0.5.1         |      1|        0|     0|
|surveillance           |1.12.1        |      0|        1|     2|
|survminer              |0.2.2         |      1|        1|     1|
|survMisc               |0.5.3         |      1|        0|     0|
|sValues                |0.1.4         |      1|        1|     0|
|tabplot                |1.3           |      1|        1|     0|
|TcGSA                  |0.10.1        |      1|        0|     0|
|tcgsaseq               |1.0.1         |      1|        0|     2|
|TELP                   |1.0           |      1|        0|     0|
|toaster                |0.5.2         |      1|        0|     0|
|trackeR                |0.0.3         |      0|        1|     0|
|UBL                    |0.0.5         |      1|        0|     0|
|UpSetR                 |1.2.4         |      0|        1|     1|
|userfriendlyscience    |0.4-1         |      1|        0|     1|
|vdmR                   |0.2.2         |      2|        1|     0|
|vipor                  |0.4.3         |      0|        1|     0|
|vmsbase                |2.1.3         |      1|        0|     0|
|VSE                    |0.99          |      1|        0|     0|
|XGR                    |1.0.3         |      1|        0|     0|

## ACDm (1.0.4)
Maintainer: Markus Belfrage <markus.belfrage@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ACDm-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: acf_acd
> ### Title: Autocorrelation function plots for ACD models
> ### Aliases: acf_acd
> 
> ### ** Examples
... 43 lines ...
Number of log-likelihood function evaluations: 114 

Estimation time: 0.3428 secs 

Description: Estimated at 2016-10-02 10:19:34 by user hadley

> acf_acd(fitModel, conf_level = 0.95, max = 50, min = 1)
Error in as.character(what) : 
  cannot coerce type 'closure' to vector of type 'character'
Calls: acf_acd ... <Anonymous> -> list_to_dataframe -> rbind.fill -> <Anonymous>
Execution halted
```

## aop (1.0.0)
Maintainer: Lyle D. Burgoon <Lyle.D.Burgoon@usace.army.mil>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘graph’ ‘Rgraphviz’

Package suggested but not available for checking: ‘BiocGenerics’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## archivist (2.1)
Maintainer: Przemyslaw Biecek <przemyslaw.biecek@gmail.com>  
Bug reports: https://github.com/pbiecek/archivist/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘archivist-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: aread
> ### Title: Read Artifacts Given as md5hashes from the Repository
> ### Aliases: aread
> 
> ### ** Examples
> 
> # read the object from local directory
> setLocalRepo(system.file("graphGallery", package = "archivist"))
> pl <- aread("f05f0ed0662fe01850ec1b928830ef32")
> # plot it
> pl
Error in stopifnot(is.list(x), is.list(val)) : 
  attempt to apply non-function
Calls: <Anonymous> ... ggplot_build -> <Anonymous> -> f -> <Anonymous> -> stopifnot
Execution halted
```

## BACA (1.3)
Maintainer: Vittorio Fortino <vittorio.fortino@ttl.fi>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘RDAVIDWebService’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## BACCT (1.0)
Maintainer: Hongtao Zhang <hongtao.zhang@abbvie.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘BACCT’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/BACCT.Rcheck/00install.out’ for details.
```

## backShift (0.1.3)
Maintainer: Christina Heinze <heinze@stat.math.ethz.ch>  
Bug reports: https://github.com/christinaheinze/backShift/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘backShift’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/backShift.Rcheck/00install.out’ for details.
```

## bamdit (3.0.0)
Maintainer: Pablo Emilio Verde <pabloemilio.verde@hhu.de>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘bamdit’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/bamdit.Rcheck/00install.out’ for details.
```

## bayesAB (0.6.0)
Maintainer: Frank Portman <frank1214@gmail.com>  
Bug reports: https://github.com/FrankPortman/bayesAB/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  2. Failure: Success (@test-generics.R#32) --------------------------------------
  plot(x, rep(0.5, 4)) produced warnings.
  
  
  testthat results ================================================================
  OK: 99 SKIPPED: 0 FAILED: 2
  1. Failure: Success (@test-generics.R#31) 
  2. Failure: Success (@test-generics.R#32) 
  
  Error: testthat unit tests failed
  Execution halted
```

## BCEA (2.2-3)
Maintainer: Gianluca Baio <gianluca@stats.ucl.ac.uk>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘BCEA-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: evppi
> ### Title: Expected Value of Perfect Partial Information (EVPPI) for
> ###   selected parameters
> ### Aliases: evppi evppi.default
> ### Keywords: Health economic evaluation Expected value of partial
... 12 lines ...
> # 
> # Runs the health economic evaluation using BCEA
> m <- bcea(e,c,ref=2,interventions=treats)
> #
> # Computes the EVPPI for a bunch of parameters
> inp <- CreateInputs(vaccine)
Error in CreateInputs(vaccine) : 
  You need to install the package 'R2jags'. Please run in your R terminal:
 install.packages('R2jags')
Execution halted
** found \donttest examples: check also with --run-donttest

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’
```

## Bclim (3.1.0)
Maintainer: Andrew Parnell <Andrew.Parnell@ucd.ie>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: inline
Warning in predictExtrapUp(alpha, lambda, beta, predictPositionsRescaled[depthIndRange],  :
  Unable to find suitable chronologies for top of core - truncated to date of extraction
Warning in predictExtrapUp(alpha, lambda, beta, predictPositionsRescaled[depthIndRange],  :
  Unable to find suitable chronologies for top of core - truncated to date of extraction
Warning in sqrt(v.out) : NaNs produced
Warning in stats::rnorm(length(t.diff), mean = 0, sd = sqrt(v.out)) :
  NAs produced
Quitting from lines 83-99 (Bclim.Rmd) 
Error: processing vignette 'Bclim.Rmd' failed with diagnostics:
Some interpolated climates have been given NAs
Execution halted

```

## bcrm (0.4.6)
Maintainer: Michael Sweeting <mjs212@medschl.cam.ac.uk>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘bcrm-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: bcrm
> ### Title: Bayesian Continual Reassessment Method for Phase I
> ###   Dose-Escalation Trials
> ### Aliases: bcrm
> 
... 133 lines ...
> ## Using rjags (requires JAGS to be installed)
> TwoPLogistic.mean.bcrm<-bcrm(stop=list(nmax=18),data=data,sdose=sdose
+   ,dose=dose,ff="logit2",prior.alpha=list(4,mu,Sigma),target.tox=target.tox
+   ,constrain=FALSE,pointest="mean",method="rjags")
Error: .onLoad failed in loadNamespace() for 'rjags', details:
  call: dyn.load(file, DLLpath = DLLpath, ...)
  error: unable to load shared object '/Users/hadley/R-revdep/rjags/libs/rjags.so':
  dlopen(/Users/hadley/R-revdep/rjags/libs/rjags.so, 10): Library not loaded: /usr/local/lib/libjags.4.dylib
  Referenced from: /Users/hadley/R-revdep/rjags/libs/rjags.so
  Reason: image not found
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘BRugs’
```

## biogram (1.3)
Maintainer: Michal Burdukiewicz <michalburdukiewicz@gmail.com>  
Bug reports: https://github.com/michbur/biogram/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: slam
Quitting from lines 15-40 (overview.Rmd) 
Error: processing vignette 'overview.Rmd' failed with diagnostics:
formal argument "panel.grid.major" matched by multiple actual arguments
Execution halted

```

## BioPET (0.2.0)
Maintainer: Jeremy Roth <jhroth@uw.edu>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘BioPET-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: enrichment_analysis
> ### Title: Prognostic Enrichment with Real Data
> ### Aliases: enrichment_analysis
> 
> ### ** Examples
> 
> data(dcaData)
> 
> ## using a single biomarker in the dataset
> analysis.single.marker <- enrichment_analysis(Cancer ~ Marker1,
+ data=dcaData,
+ reduction.under.treatment=0.3,
+ cost.screening=100,
+ cost.keeping=1000)
Warning in mean.default(response) :
  argument is not numeric or logical: returning NA
Error: Unsupported use of matrix or array for column indexing
Execution halted
```

## BlandAltmanLeh (0.3.1)
Maintainer: Bernhard Lehnert <bernhard.lehnert@uni-greifswald.de>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in readLines(con) : incomplete final line found on 'Intro.Rmd'
Warning: It seems you should call rmarkdown::render() instead of knitr::knit2html() because Intro.Rmd appears to be an R Markdown v2 document.
Quitting from lines 171-174 (Intro.Rmd) 
Error: processing vignette 'Intro.Rmd' failed with diagnostics:
Don't know how to add o to a plot
Execution halted

```

## blscrapeR (0.4.2)
Maintainer: Kris Eberwein <eberwein@knights.ucf.edu>  
Bug reports: https://github.com/keberwein/blscrapeR/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
      as.Date, as.Date.numeric
  
  trying URL 'http://www.bls.gov/lau/laucntycur14.txt'
  Error in download.file("http://www.bls.gov/lau/laucntycur14.txt", temp) : 
    cannot open URL 'http://www.bls.gov/lau/laucntycur14.txt'
  Calls: test_check ... force -> source_file -> eval -> eval -> download.file
  In addition: Warning message:
  In download.file("http://www.bls.gov/lau/laucntycur14.txt", temp) :
    cannot open URL 'http://www.bls.gov/lau/laucntycur14.txt': HTTP status was '404 Not Found'
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 0
  Execution halted
```

## brainGraph (0.62.0)
Maintainer: Christopher G. Watson <cgwatson@bu.edu>  
Bug reports: 
        https://groups.google.com/forum/?hl=en#!forum/brainGraph-help

1 error  | 0 warnings | 0 notes

```
checking whether package ‘brainGraph’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/brainGraph.Rcheck/00install.out’ for details.
```

## brms (1.0.1)
Maintainer: Paul-Christian Buerkner <paul.buerkner@gmail.com>  
Bug reports: https://github.com/paul-buerkner/brms/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
     "legend.box.background", "legend.box.spacing", "panel.spacing", "panel.spacing.x", 
     "panel.spacing.y", "plot.subtitle", "plot.caption", "strip.placement")), element = "axis.title.x", 
         label = "ll", expand_x = FALSE, expand_y = TRUE)
  26: calc_element(element, theme) at /Users/hadley/Documents/ggplot/ggplot/R/theme-elements.r:144
  27: Reduce(combine_elements, parents, theme[[element]]) at /Users/hadley/Documents/ggplot/ggplot/R/theme.r:643
  28: f(init, x[[i]])
  
  testthat results ================================================================
  OK: 833 SKIPPED: 1 FAILED: 1
  1. Error: plotting functions don't throw unexpected errors (@tests.plots.R#11) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bsam (1.0.0)
Maintainer: Ian Jonsen <ian.jonsen@mq.edu.au>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘bsam’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/bsam.Rcheck/00install.out’ for details.
```

## BTSPAS (2014.0901)
Maintainer: Carl J Schwarz <cschwarz@stat.sfu.ca>

1 error  | 0 warnings | 1 note 

```
checking whether package ‘BTSPAS’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/BTSPAS.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘BRugs’
```

## cate (1.0.4)
Maintainer: Qingyuan Zhao <qyzhao@stanford.edu>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘sva’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## classify (1.3)
Maintainer: Dr Chris Wheadon <chris.wheadon@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘classify’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/classify.Rcheck/00install.out’ for details.
```

## clifro (3.0-0)
Maintainer: Blake Seers <blake.seers@gmail.com>  
Bug reports: https://github.com/ropensci/clifro/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: ggplot2
Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=Auckland&zoom=8&size=640x640&scale=2&maptype=hybrid&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=Auckland&sensor=false
Quitting from lines 136-154 (cfStation.Rmd) 
Error: processing vignette 'cfStation.Rmd' failed with diagnostics:
x must be length 1 or 2
Execution halted

```

## clusterfly (0.4)
Maintainer: Hadley Wickham <h.wickham@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘rggobi’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## coloc (2.3-1)
Maintainer: Chris Wallace <chris.wallace@cimr.cam.ac.uk>  
Bug reports: https://github.com/chr1swallace/coloc/issues

0 errors | 1 warning  | 2 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: colorspace
Loading required package: MASS
Loading required package: BMA
Loading required package: survival
Loading required package: leaps
Loading required package: robustbase

Attaching package: ‘robustbase’

The following object is masked from ‘package:survival’:

    heart

Loading required package: inline
Loading required package: rrcov
Scalable Robust Estimators with High Breakdown Point (version 1.4-3)


Error: processing vignette 'vignette.Rnw' failed with diagnostics:
 chunk 7 
Error in library(snpStats) : there is no package called ‘snpStats’
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘snpStats’

checking R code for possible problems ... NOTE
approx.bf.p: no visible global function definition for ‘qnorm’
coeff.plot: no visible global function definition for ‘polygon’
coeff.plot: no visible global function definition for ‘abline’
coeff.plot: no visible global function definition for ‘arrows’
coeff.plot: no visible global function definition for ‘points’
coeff.plot: no visible global function definition for ‘segments’
coloc.abf.snpStats: no visible global function definition for
  ‘single.snp.tests’
coloc.abf.snpStats: no visible global function definition for
... 43 lines ...
  cor glm impute.snps integrate lm object optimize pchisq pf points
  polygon prcomp qnorm segments single.snp.tests snp.imputation var
  vcov
Consider adding
  importFrom("graphics", "abline", "arrows", "axis", "box", "points",
             "polygon", "segments")
  importFrom("stats", "as.formula", "coef", "coefficients", "cor", "glm",
             "integrate", "lm", "optimize", "pchisq", "pf", "prcomp",
             "qnorm", "var", "vcov")
  importFrom("utils", "combn")
to your NAMESPACE file.
```

## colorplaner (0.1.1)
Maintainer: William Murphy <william.murphy.rd@gmail.com>  
Bug reports: https://github.com/wmurphyrd/colorplaner/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(colorplaner)
  > 
  > test_check("colorplaner")
  1. Failure: Layer-level mappings produce guide axis titles without error (@test_scale_training.R#47) 
  print(...) produced warnings.
  
  
  testthat results ================================================================
  OK: 46 SKIPPED: 0 FAILED: 1
  1. Failure: Layer-level mappings produce guide axis titles without error (@test_scale_training.R#47) 
  
  Error: testthat unit tests failed
  Execution halted
```

## CopulaDTA (0.0.4)
Maintainer: Victoria N Nyaga <victoria.nyaga@outlook.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 291-292 (CopulaDTA_Vignette.Rmd) 
Error: processing vignette 'CopulaDTA_Vignette.Rmd' failed with diagnostics:
index out of bounds ('unit' subsetting)
Execution halted

```

## corrr (0.2.0)
Maintainer: Simon Jackson <drsimonjackson@gmail.com>  
Bug reports: https://github.com/drsimonj/corrr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘corrr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: network_plot
> ### Title: Network plot of a correlation data frame
> ### Aliases: network_plot
> 
> ### ** Examples
> 
> x <- correlate(mtcars)
> network_plot(x)
Warning: Ignoring unknown aesthetics: x, y
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... transform_position -> lapply -> FUN -> rescale -> zero_range
Execution halted
```

## cowplot (0.6.3)
Maintainer: Claus O. Wilke <wilke@austin.utexas.edu>  
Bug reports: https://github.com/wilkelab/cowplot/issues

1 error  | 2 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘cowplot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: switch_axis_position
> ### Title: Switches the axis position of the x or y axis in a plot.
> ### Aliases: switch_axis_position
> 
> ### ** Examples
> 
> p <- ggplot(mtcars, aes(mpg, disp)) + geom_line(colour = "blue")
> ggdraw(switch_axis_position(p, axis = 'y'))
Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
Error in gt$grobs[[iyl]] : 
  attempt to select less than one element in get1index
Calls: ggdraw -> switch_axis_position -> switch_yaxis_position
Execution halted

checking whether package ‘cowplot’ can be installed ... WARNING
Found the following significant warnings:
  Warning: `legend.margin` must be specified using `margin()`. For the old behavior use legend.spacing
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/cowplot.Rcheck/00install.out’ for details.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: cowplot
Loading required package: ggplot2

Attaching package: 'cowplot'

The following object is masked from 'package:ggplot2':

    ggsave

Loading required package: grid
Quitting from lines 17-26 (axis_position.Rmd) 
Error: processing vignette 'axis_position.Rmd' failed with diagnostics:
attempt to select less than one element in get1index
Execution halted

```

## crmPack (0.2.0)
Maintainer: Daniel Sabanes Bove <sabanesd@roche.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘crmPack’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/crmPack.Rcheck/00install.out’ for details.
```

## Crossover (0.1-16)
Maintainer: Kornelius Rohmeyer <rohmeyer@small-projects.de>  
Bug reports: https://github.com/kornl/Crossover/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘Crossover-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot
> ### Title: Plots information about the search algorithm and its process.
> ### Aliases: plot plot,CrossoverSearchResult,missing-method
> 
> ### ** Examples
> 
> 
> ## Not run: 
> ##D x <- searchCrossOverDesign(s=9, p=5, v=4, model=4)
> ##D plot(x)
> ## End(Not run)
> 
> x <- searchCrossOverDesign(s=9, p=5, v=4, model=4, n=c(50,10), jumps=c(10, 10))
> plot(x, show.jumps=TRUE)
> plot(x, type=2)
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L, 1L, 1L,  : 
  replacement has 500 rows, data has 1
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: mvtnorm
Loading required package: survival
Loading required package: TH.data

Attaching package: 'TH.data'

The following object is masked from 'package:MASS':

    geyser

Quitting from lines 23-29 (./childs/Search.Rnw) 
Quitting from lines NA-239 (./childs/Search.Rnw) 
Error: processing vignette 'Crossover.Rnw' failed with diagnostics:
replacement has 100000 rows, data has 1
Execution halted

```

## Cyclops (1.2.0)
Maintainer: Marc A. Suchard <msuchard@ucla.edu>  
Bug reports: https://github.com/ohdsi/cyclops/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘Cyclops’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/Cyclops.Rcheck/00install.out’ for details.
```

## data.table (1.9.6)
Maintainer: Matt Dowle <mattjdowle@gmail.com>  
Bug reports: https://github.com/Rdatatable/data.table/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
Running the tests in ‘tests/tests.R’ failed.
Last 13 lines of output:
  [1] FALSE
  > y = TRUE 
  [1] TRUE
  Tests 1372.3+ not run. If required call library(GenomicRanges) first.
  Tests 1441-1444 not run. If required install the 'fr_FR.utf8' locale.
  
  Error in eval(expr, envir, enclos) : 
    3 errors out of 4390 (lastID=1557.4, endian=little, sizeof(long double)==16) in inst/tests/tests.Rraw on Sun Oct  2 10:48:46 2016. Search tests.Rraw for test numbers: 167, 167.2, 1241.2.
  Calls: test.data.table -> sys.source -> eval -> eval
  In addition: Warning message:
  In library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
    there is no package called 'GenomicRanges'
  Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘GenomicRanges’
```

## ddpcr (1.5)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/daattali/ddpcr/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 205-213 (extend.Rmd) 
Error: processing vignette 'extend.Rmd' failed with diagnostics:
vector: cannot make a vector of mode 'closure'.
Execution halted

```

## demi (1.1.2)
Maintainer: Sten Ilmjarv <sten.ilmjarv@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘affxparser’ ‘affy’ ‘oligo’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## dfpk (1.0)
Maintainer: Artemis Toumazi <artemis.toumazi@inserm.fr>  
Bug reports: http://github.com/artemis-toumazi/dfpk/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘dfpk’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/dfpk.Rcheck/00install.out’ for details.
```

## dtwSat (0.2.1)
Maintainer: Victor Maus <vwmaus1@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: zoo

Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Loading required package: raster
Loading required package: sp
Loading required package: ggplot2
Loaded dtwSat v0.2.1. See ?dtwSat for help, citation("dtwSat") for use in publication.

Read 137 items
Read 1 item
Warning: Ignoring unknown aesthetics: y
Warning: Ignoring unknown aesthetics: y
Quitting from lines 436-437 (applying_twdtw.Rmd) 
Error: processing vignette 'applying_twdtw.Rmd' failed with diagnostics:
vector: cannot make a vector of mode 'closure'.
Execution halted

```

## dynr (0.1.7)
Maintainer: Lu Ou <lzo114@psu.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘dynr’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/dynr.Rcheck/00install.out’ for details.
```

## eechidna (0.1)
Maintainer: Ben Marwick <benmarwick@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘eechidna-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: hexDat
> ### Title: Electorate hexagon data in a tidy form
> ### Aliases: hexDat
> ### Keywords: datasets
> 
... 18 lines ...

    layout

> p <- ggplot(hexDat, aes(xcent, ycent, text = Electorate)) + 
+   geom_hex(stat = "identity") + 
+   lims(x=c(-80, 8), y=c(-40, 50))
> ggplotly(p, tooltip = "text")
Error in gg2list(p, width = width, height = height, tooltip = tooltip,  : 
  attempt to apply non-function
Calls: ggplotly -> ggplotly.ggplot -> gg2list
Execution halted
```

## eeptools (0.9.1)
Maintainer: Jared E. Knowles <jknowles@gmail.com>

2 errors | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘eeptools-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: autoplot.lm
> ### Title: A function to replicate the basic plot function for linear
> ###   models in ggplot2
> ### Aliases: autoplot.lm
> 
> ### ** Examples
> 
> # Univariate
> a <- runif(1000)
> b <- 7 * a + rnorm(1)
> mymod <- lm(b~a)
> autoplot(mymod)
Error in theme(axis.title = element_text(size = rel(0.8), face = "bold"),  : 
  formal argument "legend.text" matched by multiple actual arguments
Calls: autoplot ... theme_dpi -> %+replace% -> is.theme -> inherits -> theme
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
             size = 0.5), strip.background = element_rect(fill = "grey90", colour = "grey50"), 
         strip.background = element_rect(fill = "grey90", colour = "grey50"), strip.text = element_text(size = rel(0.9), 
             face = "bold"), strip.text.x = element_text(size = rel(0.9), face = "bold"), 
         strip.text.y = element_text(size = rel(0.9), face = "bold"), legend.text = element_text(), 
         legend.title = element_text(), panel.margin = grid::unit(0.48, "cm"))
  
  testthat results ================================================================
  OK: 513 SKIPPED: 0 FAILED: 2
  1. Error: All themes result in a valid ggplot2 object (@test-plots.R#6) 
  2. Error: Autoplot works as expected for linear models (@test-plots.R#31) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: ggplot2
Quitting from lines 172-181 (intro.Rmd) 
Error: processing vignette 'intro.Rmd' failed with diagnostics:
formal argument "legend.text" matched by multiple actual arguments
Execution halted


checking R code for possible problems ... NOTE
theme_dpi: possible error in theme(axis.title = element_text(size =
  rel(0.8), face = "bold"), axis.title.y = element_text(vjust = 0.35,
  angle = 90), axis.text = element_text(size = rel(0.8)), axis.ticks =
  element_line(colour = "black", size = rel(1.5)), legend.key =
  element_rect(colour = "grey80"), legend.title = element_text(),
  legend.text = element_text(), panel.background = element_rect(fill =
  "white", colour = NA), panel.border = element_rect(fill = NA, colour
  = "grey50"), panel.grid.major = element_line(colour = "grey90", :
  formal argument "legend.text" matched by multiple actual arguments
... 69 lines ...
  = element_rect(fill = "grey90", colour = "grey50"), strip.text =
  element_text(size = rel(0.9), face = "bold"), strip.text.x =
  element_text(size = rel(0.8), face = "bold"), strip.text.y =
  element_text(size = rel(0.8), face = "bold"), legend.text =
  element_text(size = base_size * : formal argument "legend.text"
  matched by multiple actual arguments
theme_dpi_mapPNG: possible error in 0.65), panel.margin =
  grid::unit(0.48, "cm"), plot.margin = grid::unit(c(1, 1, 0.5, 0.5),
  "lines"), plot.title = element_text(family = base_family, size =
  base_size * 1.4)): formal argument "legend.text" matched by multiple
  actual arguments
```

## emojifont (0.3.4)
Maintainer: Guangchuang Yu <guangchuangyu@gmail.com>  
Bug reports: https://github.com/GuangchuangYu/emojifont/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 119-144 (emojifont.Rmd) 
Error: processing vignette 'emojifont.Rmd' failed with diagnostics:
there is no package called 'ggtree'
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘ggtree’
```

## EmpiricalCalibration (1.2.0)
Maintainer: Martijn Schuemie <schuemie@ohdsi.org>  
Bug reports: https://github.com/OHDSI/EmpiricalCalibration/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘EmpiricalCalibration-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotTrueAndObserved
> ### Title: Plot true and observed values
> ### Aliases: plotTrueAndObserved
> 
> ### ** Examples
> 
> data <- simulateControls(n = 50 * 3, mean = 0.25, sd = 0.25, trueLogRr = log(c(1, 2, 4)))
> plotTrueAndObserved(data$logRr, data$seLogRr, data$trueLogRr)
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L, 1L, 1L,  : 
  replacement has 150 rows, data has 8
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## factoextra (1.0.3)
Maintainer: Alboukadel Kassambara <alboukadel.kassambara@gmail.com>  
Bug reports: https://github.com/kassambara/factoextra/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘factoextra-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: fviz_ca
> ### Title: Visualize Correspondence Analysis
> ### Aliases: fviz_ca fviz_ca_biplot fviz_ca_col fviz_ca_row
> 
> ### ** Examples
... 44 lines ...
> 
> # Change gradient color
> # Use repel = TRUE to avoid overplotting (slow if many points)
> fviz_ca_row(res.ca, col.row = "cos2", repel = TRUE) + 
+       scale_color_gradient2(low = "white", mid = "#2E9FDF", 
+       high = "#FC4E07", midpoint = 0.5, space = "Lab")+
+ theme_minimal()
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... transform_position -> lapply -> FUN -> rescale -> zero_range
Execution halted
** found \donttest examples: check also with --run-donttest

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘NbClust’
```

## fbroc (0.4.0)
Maintainer: Erik Peter <jerikpeter@googlemail.com>  
Bug reports: http://github.com/erikpeter/fbroc/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘fbroc-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.fbroc.conf.paired
> ### Title: Plots function for object of class 'fbroc.conf.paired'
> ### Aliases: plot.fbroc.conf.paired
> 
> ### ** Examples
> 
> data(roc.examples)
> example <- boot.paired.roc(roc.examples$Cont.Pred, roc.examples$Cont.Pred.Outlier,
+                            roc.examples$True.Class, n.boot = 1000)
> # Confidence regions for the difference in TPR at specific FPR values                           
> tpr.conf <- conf(example, conf.for = "tpr", steps = 50)
> plot(tpr.conf)
Warning: Ignoring unknown aesthetics: y
Error in paste(Delta, "True Positive Rate") : object 'Delta' not found
Calls: plot ... grobName -> element_grob -> element_grob.element_text -> paste
Execution halted
```

## fheatmap (1.0.1)
Maintainer: Sivasish Sindiri<sentisci@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘fheatmap-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name:  Column Annotations 
> ### Title: Annotations of columns in "fheatmap_data"
> ### Aliases: ' annotation_col '
> ### Keywords: annotation_col
> 
> ### ** Examples
> 
> data(annotation_col)
> fheatmap(fheatmap_data,annotation_col=annotation_col)
Error in theme(panel.grid.major = element_blank(), panel.grid.minor = element_blank(),  : 
  formal argument "plot.margin" matched by multiple actual arguments
Calls: fheatmap -> draw_tree -> theme_blank -> theme
Execution halted

checking R code for possible problems ... NOTE
theme_blank: possible error in theme(panel.grid.major =
  element_blank(), panel.grid.minor = element_blank(), panel.background
  = element_blank(), axis.line = element_blank(), axis.ticks =
  element_blank(), axis.text = element_blank(), panel.grid =
  element_blank(), panel.border = element_blank(), plot.margin =
  rep(unit(0, "null"), 4), panel.margin = unit(0, "null"), plot.margin
  = rep(unit(0, "null"), 4), legend.position = "none"): formal argument
  "plot.margin" matched by multiple actual arguments
```

## fitbitScraper (0.1.7)
Maintainer: Cory Nissen <corynissen@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 29-40 (fitbitScraper-examples.Rmd) 
Error: processing vignette 'fitbitScraper-examples.Rmd' failed with diagnostics:
Value for option cookie (10022) must be length-1 string
Execution halted

```

## forecast (7.2)
Maintainer: Rob Hyndman <Rob.Hyndman@monash.edu>  
Bug reports: https://github.com/robjhyndman/forecast/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘forecast-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: gglagplot
> ### Title: Time series lag ggplots
> ### Aliases: gglagplot gglagchull
> 
> ### ** Examples
> 
> gglagplot(AirPassengers)
> gglagchull(AirPassengers)
> gglagplot(AirPassengers,seasonal=FALSE)
> 
> lungDeaths <- cbind(mdeaths, fdeaths)
> gglagplot(lungDeaths, lags=2)
Error in gList(list(x = 0.5, y = 0.5, width = 1, height = 1, just = "centre",  : 
  only 'grobs' allowed in "gList"
Calls: <Anonymous> ... makeContent.gtable -> setChildren -> do.call -> gList
Execution halted
```

## fuzzyforest (1.0.2)
Maintainer: Daniel Conn <djconn17@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘fuzzyforest-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ff
> ### Title: Fits fuzzy forest algorithm.
> ### Aliases: ff
> 
> ### ** Examples
... 9 lines ...
Attaching package: ‘fastcluster’

The following object is masked from ‘package:stats’:

    hclust

Error in loadNamespace(i, c(lib.loc, .libPaths()), versionCheck = vI[[i]]) : 
  there is no package called ‘GO.db’
Error: package or namespace load failed for ‘WGCNA’
Execution halted
** found \donttest examples: check also with --run-donttest
```

## gapfill (0.9.3)
Maintainer: Florian Gerber <florian.gerber@math.uzh.ch>

2 errors | 0 warnings | 2 notes

```
checking examples ... ERROR
Running examples in ‘gapfill-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: Image
> ### Title: Image Panels
> ### Aliases: Image
> 
> ### ** Examples
> 
> Image(ndvi)
Error in theme(line = element_blank(), strip.background = element_rect(fill = "white",  : 
  formal argument "line" matched by multiple actual arguments
Calls: Image -> theme
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/run-all.R’ failed.
Last 13 lines of output:
  4: theme(line = element_blank(), strip.background = element_rect(fill = "white", color = "white"), 
         strip.text.x = element_text(), strip.text.y = element_text(), axis.ticks = element_blank(), 
         axis.text.x = element_blank(), axis.text.y = element_blank(), axis.title.x = element_blank(), 
         axis.title.y = element_blank(), line = element_blank(), legend.position = "right", 
         panel.border = element_blank(), plot.margin = unit(c(1, 1, 1, 1), "mm"), panel.margin = unit(2, 
             "mm"))
  
  testthat results ================================================================
  OK: 859 SKIPPED: 0 FAILED: 1
  1. Error: Image (@test-Image.R#5) 
  
  Error: testthat unit tests failed
  Execution halted

checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘doMPI’

checking R code for possible problems ... NOTE
Image: possible error in theme(line = element_blank(), strip.background
  = element_rect(fill = "white", color = "white"), strip.text.x =
  element_text(), strip.text.y = element_text(), axis.ticks =
  element_blank(), axis.text.x = element_blank(), axis.text.y =
  element_blank(), axis.title.x = element_blank(), axis.title.y =
  element_blank(), line = element_blank(), legend.position = "right",
  panel.border = element_blank(), plot.margin = unit(c(1, 1, 1, 1),
  "mm"), panel.margin = unit(2, "mm")): formal argument "line" matched
  by multiple actual arguments
```

## GenCAT (1.0.3)
Maintainer: Eric Reed <reeder@bu.edu>

1 error  | 1 warning  | 3 notes

```
checking examples ... ERROR
Running examples in ‘GenCAT-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: GenCAT
> ### Title: Running GenCAT
> ### Aliases: GenCAT
> ### Keywords: ~kwd1 ~kwd2
> 
... 15 lines ...
            SNP effect_allele other_allele   testStat chr position class
17232 rs9572807             C            T -0.3014771  13 72423959 DACH1
24151 rs4389009             G            A -1.4443860  13 99193519 STK24
37178 rs7151730             T            A  0.1387317  14 33531585 NPAS3
58941 rs1076958             G            A  2.0597766  14 91131857 TTC7B
13089 rs7987481             G            A -1.0024210  13 47318950 LRCH1
58302  rs411064             G            A  0.2132412  14 90008860 FOXN3
> 
> library(snpStats)
Error in library(snpStats) : there is no package called ‘snpStats’
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Loading required package: doParallel
Loading required package: foreach
Loading required package: iterators
Loading required package: parallel
Loading required package: ggplot2
Quitting from lines 62-69 (GenCAT-vignette.Rmd) 
Error: processing vignette 'GenCAT-vignette.Rmd' failed with diagnostics:
unable to find required package 'snpStats'
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘snpStats’

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘snpStats’

checking data for non-ASCII characters ... NOTE
  Error in .requirePackage(package) : 
    unable to find required package 'snpStats'
  Calls: <Anonymous> ... .extendsForS3 -> extends -> getClassDef -> .requirePackage
  Execution halted
```

## geomnet (0.1.2.1)
Maintainer: Samantha Tyner <sctyner@iastate.edu>  
Bug reports: https://github.com/sctyner/geomnet/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘geomnet-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: football
> ### Title: College football games network (undirected)
> ### Aliases: football
> ### Keywords: datasets
> 
... 23 lines ...
+     layout = 'fruchtermanreingold'
+   ) +
+   theme_net() +
+   theme(legend.position = "bottom") +
+   scale_colour_brewer("Conference", palette = "Paired")  +
+   guides(linetype = FALSE)
Warning: Ignoring unknown aesthetics: label
Error in get(name, envir = asNamespace(pkg), inherits = FALSE) : 
  object 'panel_scales' not found
Calls: <Anonymous> ... f -> <Anonymous> -> f -> <Anonymous> -> f -> ::: -> get
Execution halted
```

## GGally (1.2.0)
Maintainer: Barret Schloerke <schloerke@gmail.com>  
Bug reports: https://github.com/ggobi/ggally/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘GGally-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: getPlot
> ### Title: getPlot
> ### Aliases: [.ggmatrix getPlot
> ### Keywords: hplot
> 
> ### ** Examples
> 
>  data(tips, package = "reshape")
>  plotMatrix2 <- ggpairs(tips[, 3:2], upper = list(combo = "denstrip"))
>  plotMatrix2[1, 2]
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
Error in element_grob.element_text(el, ...) : object 'tip' not found
Calls: <Anonymous> ... grobName -> element_grob -> element_grob.element_text
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1.  Error: diagAxis (@test-gg-plots.R#82) 
  2.  Failure: stops (@test-ggmatrix_getput.R#23) 
  3.  Failure: stops (@test-ggmatrix_getput.R#23) 
  4.  Failure: stops (@test-ggmatrix_getput.R#23) 
  5.  Failure: stops (@test-ggmatrix_getput.R#23) 
  6.  Failure: stops (@test-ggmatrix_getput.R#23) 
  7.  Failure: stops (@test-ggmatrix_getput.R#23) 
  8.  Error: axisLabels (@test-zzz_ggpairs.R#350) 
  9.  Error: strips and axis (@test-zzz_ggpairs.R#373) 
  10. Error: subtypes (@test-zzz_ggpairs.R#614) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ggalt (0.1.1)
Maintainer: Bob Rudis <bob@rudis.net>  
Bug reports: https://github.com/hrbrmstr/ggalt/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggalt-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: coord_proj
> ### Title: Similar to 'coord_map' but uses the PROJ.4 library/package for
> ###   projection transformation
> ### Aliases: coord_proj
> 
... 7 lines ...
> 
> gg <- ggplot()
> gg <- gg + geom_map(data=world, map=world,
+                     aes(x=long, y=lat, map_id=region))
Warning: Ignoring unknown aesthetics: x, y
> gg <- gg + coord_proj("+proj=wintri")
> gg
Error in gPathFromVector(names) : 
  a 'grob' path must contain at least one 'grob' name
Calls: <Anonymous> ... valid.data -> recycle.data -> gPath -> gPathFromVector
Execution halted
```

## ggenealogy (0.2.0)
Maintainer: Lindsay Rutter <lrutter@iastate.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  conversion failure on 'Barbara Choroś-Tomczyk' in 'mbcsToSbcs': dot substituted for <c5>
Warning in graphics::strwidth(pMPDF$label, "inches") :
  conversion failure on 'Barbara Choroś-Tomczyk' in 'mbcsToSbcs': dot substituted for <9b>
Warning in graphics::strwidth(pMPDF$label, "inches") :
  conversion failure on 'Ondřej Chochola' in 'mbcsToSbcs': dot substituted for <c5>
Warning in graphics::strwidth(pMPDF$label, "inches") :
  conversion failure on 'Ondřej Chochola' in 'mbcsToSbcs': dot substituted for <99>
... 8 lines ...
  conversion failure on 'Peter Čerňanský' in 'mbcsToSbcs': dot substituted for <8c>
Warning in graphics::strwidth(pMPDF$label, "inches") :
  conversion failure on 'Peter Čerňanský' in 'mbcsToSbcs': dot substituted for <c5>
Warning in graphics::strwidth(pMPDF$label, "inches") :
  conversion failure on 'Peter Čerňanský' in 'mbcsToSbcs': dot substituted for <88>

Error: processing vignette 'ggenealogy.Rnw' failed with diagnostics:
 chunk 57 (label = plotAnimate) 
Error in gg2list(p, width = width, height = height, tooltip = tooltip,  : 
  attempt to apply non-function
Execution halted
```

## ggExtra (0.5)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/daattali/ggExtra/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggExtra-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ggMarginal
> ### Title: Add marginal density/histogram to ggplot2 scatterplots
> ### Aliases: ggMarginal
> 
> ### ** Examples
> 
> 
> # basic usage
> p <- ggplot2::ggplot(mtcars, ggplot2::aes(wt, mpg)) + ggplot2::geom_point()
> ggMarginal(p)
Warning in mean.default(getLimits(pbTop, "y")) :
  argument is not numeric or logical: returning NA
Error: Don't know how to add o to a plot
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 94-95 (overview.Rmd) 
Error: processing vignette 'overview.Rmd' failed with diagnostics:
Don't know how to add o to a plot
Execution halted

```

## ggfortify (0.2.0)
Maintainer: Masaaki Horikoshi <sinhrks@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  
  
  testthat results ================================================================
  OK: 1087 SKIPPED: 4 FAILED: 6
  1. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#180) 
  2. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#181) 
  3. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#182) 
  4. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#189) 
  5. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#190) 
  6. Failure: autoplot ts works for multivariate timeseries (@test-ts.R#191) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  5.5Mb
  sub-directories of 1Mb or more:
    doc   4.9Mb
```

## ggnetwork (0.5.1)
Maintainer: Francois Briatte <f.briatte@gmail.com>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggnetwork-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: geom_edgetext_repel
> ### Title: Draw repulsive edge labels.
> ### Aliases: geom_edgelabel_repel geom_edgetext_repel
> 
> ### ** Examples
... 41 lines ...
Loading required package: sna
Loading required package: statnet.common
sna: Tools for Social Network Analysis
Version 2.4 created on 2016-07-23.
copyright (c) 2005, Carter T. Butts, University of California-Irvine
 For citation information, type citation("sna").
 Type help(package="sna") to get started.

Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... transform_position -> lapply -> FUN -> rescale -> zero_range
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
network: Classes for Relational Data
Version 1.13.0 created on 2015-08-31.
copyright (c) 2005, Carter T. Butts, University of California-Irvine
                    Mark S. Handcock, University of California -- Los Angeles
                    David R. Hunter, Penn State University
                    Martina Morris, University of Washington
                    Skye Bender-deMoll, University of Washington
 For citation information, type citation("network").
 Type help("network-package") to get started.

Loading required package: statnet.common
sna: Tools for Social Network Analysis
Version 2.4 created on 2016-07-23.
copyright (c) 2005, Carter T. Butts, University of California-Irvine
 For citation information, type citation("sna").
 Type help(package="sna") to get started.

Quitting from lines 199-205 (ggnetwork.Rmd) 
Error: processing vignette 'ggnetwork.Rmd' failed with diagnostics:
x must be length 1 or 2
Execution halted

```

## ggpmisc (0.2.10)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/ggpmisc

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 770-784 (examples.Rmd) 
Error: processing vignette 'examples.Rmd' failed with diagnostics:
stat_fit_augment requires the following missing aesthetics: y
Execution halted

```

## ggpubr (0.1.0)
Maintainer: Alboukadel Kassambara <alboukadel.kassambara@gmail.com>  
Bug reports: https://github.com/kassambara/ggpubr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggpubr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: diff_express
> ### Title: Differential gene expression analysis results
> ### Aliases: diff_express
> 
> ### ** Examples
... 6 lines ...
+    fdr = 0.05, fc = 2, size = 0.4,
+    palette = c("#B31B21", "#1465AC", "darkgray"),
+    genenames = as.vector(diff_express$name),
+    legend = "top", top = 20,
+    font.label = c("bold", 11),
+    font.legend = "bold",
+    font.main = "bold",
+    ggtheme = ggplot2::theme_minimal())
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... transform_position -> lapply -> FUN -> rescale -> zero_range
Execution halted
```

## ggswissmaps (0.0.9)
Maintainer: Sandro Petrillo Burri <gibo.gaf@gmail.com>

0 errors | 1 warning  | 1 note 

```
checking whether package ‘ggswissmaps’ can be installed ... WARNING
Found the following significant warnings:
  Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/ggswissmaps.Rcheck/00install.out’ for details.

checking installed package size ... NOTE
  installed size is  5.1Mb
  sub-directories of 1Mb or more:
    R      3.4Mb
    data   1.6Mb
```

## ggtern (2.1.4)
Maintainer: Nicholas Hamilton <nick@ggtern.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ggtern’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/ggtern.Rcheck/00install.out’ for details.
```

## gitter (1.1.1)
Maintainer: Omar Wagih <wagih@ebi.ac.uk>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘EBImage’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## glycanr (0.3.0)
Maintainer: Ivo Ugrina <ivo@iugrina.com>  
Bug reports: https://github.com/iugrina/glycanr/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘glycanr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: quantilenorm
> ### Title: Quantile Normalization of glycan data
> ### Aliases: quantilenorm
> 
> ### ** Examples
> 
> data(mpiu)
> mpiun <- quantilenorm(mpiu)
Error in quantilenorm(mpiu) : 
  Unable to proceed since package preprocessCore from
        BioConductor is not available on this system. This
        package is a prerequisite to use the quantilenorm function!
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  The following objects are masked from 'package:base':
  
      intersect, setdiff, setequal, union
  
  Error in quantilenorm(mpiu, transpose = TRUE) : 
    Unable to proceed since package preprocessCore from
          BioConductor is not available on this system. This
          package is a prerequisite to use the quantilenorm function!
  Calls: test_check ... force -> source_file -> eval -> eval -> quantilenorm
  testthat results ================================================================
  OK: 5 SKIPPED: 0 FAILED: 0
  Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘preprocessCore’
```

## gmwm (2.0.0)
Maintainer: Stephane Guerrier <stephane@illinois.edu>  
Bug reports: https://github.com/SMAC-Group/gmwm/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
(as 'lib' is unspecified)
installing the source package 'imudata'

trying URL 'http://smac-group.com/datarepo/src/contrib/imudata_2.0.0.tar.gz'
Content type 'application/octet-stream' length 55753263 bytes (53.2 MB)
==================================================
downloaded 53.2 MB


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘imudata’
```

## gridDebug (0.5-0)
Maintainer: Paul Murrell <p.murrell@auckland.ac.nz>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘graph’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## heatmaply (0.5.0)
Maintainer: Tal Galili <tal.galili@gmail.com>  
Bug reports: https://github.com/talgalili/heatmaply/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘heatmaply-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: heatmapr
> ### Title: Creates a heatmapr object
> ### Aliases: heatmapr
> 
> ### ** Examples
> 
> library(heatmaply)
> hm <- heatmapr(mtcars, scale = "column", colors = "Blues")
> heatmaply(hm)
Error in gg2list(p, width = width, height = height, tooltip = tooltip,  : 
  attempt to apply non-function
Calls: heatmaply ... heatmaply.heatmapr -> ggplotly -> ggplotly.ggplot -> gg2list
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
The following object is masked from 'package:stats':

    filter

The following object is masked from 'package:graphics':

    layout
... 8 lines ...

Suggestions and bug-reports can be submitted at: https://github.com/talgalili/heatmaply/issues
Or contact: <tal.galili@gmail.com>

	To suppress this message use:  suppressPackageStartupMessages(library(heatmaply))
---------------------

Quitting from lines 83-85 (heatmaply.Rmd) 
Error: processing vignette 'heatmaply.Rmd' failed with diagnostics:
attempt to apply non-function
Execution halted
```

## hyperSpec (0.98-20150304)
Maintainer: Claudia Beleites <chemometrie@beleites.de>

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘hyperSpec-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: qplotspc
> ### Title: Spectra plotting with ggplot2
> ### Aliases: qplotspc
> 
> ### ** Examples
> 
> qplotspc (chondro)
Warning in qplotspc(chondro) :
  Number of spectra exceeds spc.nmax. Only the first 10 are plotted.
>   qplotspc (paracetamol, c (2800 ~ max, min ~ 1800)) + scale_x_reverse (breaks = seq (0, 3200, 400))
Error in gtable_add_grob(panel_table, strips$x$top, 2, panel_pos_col$l,  : 
  Not all inputs have either length 1 or same length same as 'grobs'
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> gtable_add_grob
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in plotc(x, ...) : Intensity at first wavelengh only is used.
Warning in (function (x, y, z, subscripts, at = pretty(z), ..., col.regions = regions$col,  :
  'x' values are not equispaced; output may be wrong
Warning in hyperSpec:::.levelplot(x, data, ...) :
  Only first wavelength is used for plotting
Loading required package: latticeExtra
Loading required package: RColorBrewer
... 8 lines ...
Warning in (function (x, y, z, subscripts, at = pretty(z), ..., col.regions = regions$col,  :
  'x' values are not equispaced; output may be wrong
Warning in (function (x, y, z, subscripts, at = pretty(z), ..., col.regions = regions$col,  :
  'y' values are not equispaced; output may be wrong
Warning: Ignoring unknown aesthetics: y

Error: processing vignette 'plotting.Rnw' failed with diagnostics:
 chunk 77 (label = ggplot2-do) 
Error in gtable_add_grob(panel_table, strips$x$top, 2, panel_pos_col$l,  : 
  Not all inputs have either length 1 or same length same as 'grobs'
Execution halted

checking R code for possible problems ... NOTE
Warning: local assignments to syntactic functions: ~
Warning: local assignments to syntactic functions: ~
.cut.ticks: no visible global function definition for ‘head’
.jdx.TABULAR.PAC: no visible global function definition for ‘head’
.jdx.hdr.concentrations: no visible global function definition for
  ‘modifyList’
.jdx.readhdr: no visible global function definition for ‘maintainer’
.labels: no visible global function definition for ‘modifyList’
.levelplot: no visible global function definition for ‘modifyList’
... 56 lines ...
  ‘modifyList’
plot,hyperSpec-character: no visible global function definition for
  ‘modifyList’
Undefined global functions or variables:
  chull col2rgb colorRampPalette head maintainer modifyList read.table
  relist rgb str tail unstack write.table
Consider adding
  importFrom("grDevices", "chull", "col2rgb", "colorRampPalette", "rgb")
  importFrom("utils", "head", "maintainer", "modifyList", "read.table",
             "relist", "str", "tail", "unstack", "write.table")
to your NAMESPACE file.
```

## IAPWS95 (1.0.0)
Maintainer: Benedito Baptista <bene46@msn.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘IAPWS95’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/IAPWS95.Rcheck/00install.out’ for details.
```

## ibmdbR (1.47.1)
Maintainer: Michael Wurst <inza@de.ibm.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘RODBC’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## ie2miscdata (1.0.1)
Maintainer: Irucka Embry <iembry@usgs.gov>  
Bug reports: https://github.com/iembry-USGS/ie2miscdata/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in (function (package, help, pos = 2, lib.loc = NULL, character.only = FALSE,  : 
  there is no package called 'states2k'
Quitting from lines 24-109 (US-Engineering-Weather-Map.Rmd) 
Error: processing vignette 'US-Engineering-Weather-Map.Rmd' failed with diagnostics:
object 'states2k_map' not found
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘states2k’
```

## imageData (0.1-21)
Maintainer: Chris Brien <Chris.Brien@unisa.edu.au>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘imageData-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: corrPlot
> ### Title: Calculates and plots correlation matrices for a set of responses
> ### Aliases: corrPlot
> ### Keywords: hplot
> 
> ### ** Examples
> 
> data(exampleData)
> responses <- c("Area","Area.SV","Area.TV", "Image.Biomass", "Max.Height","Centre.Mass",
+                "Density", "Compactness.TV", "Compactness.SV")
> corrPlot(responses, longi.dat, pairs.sets=list(c(1:4),c(5:7)))
Error in `$<-.data.frame`(`*tmp*`, "hjust", value = 0.5) : 
  replacement has 1 row, data has 0
Calls: corrPlot ... <Anonymous> -> get_x_axis_labels -> $<- -> $<-.data.frame
Execution halted
```

## imager (0.31)
Maintainer: Simon Barthelme <simon.barthelme@gipsa-lab.fr>  
Bug reports: https://github.com/dahtah/imager/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Error: cpp_plugin (@test_cpp_api.R#14) --------------------------------------
  Error 1 occurred building shared library.
  1: cppFunction(foo.inline, depends = "imager") at testthat/test_cpp_api.R:14
  2: sourceCpp(code = code, env = env, rebuild = rebuild, cacheDir = cacheDir, showOutput = showOutput, 
         verbose = verbose)
  3: stop("Error ", status, " occurred building shared library.")
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Error: cpp_plugin (@test_cpp_api.R#14) 
  
  Error: testthat unit tests failed
  Execution halted

checking whether package ‘imager’ can be installed ... WARNING
Found the following significant warnings:
  ../inst/include/CImg.h:13559:29: warning: multiple unsequenced modifications to 'ptrs' [-Wunsequenced]
  ../inst/include/CImg.h:13738:19: warning: multiple unsequenced modifications to 'ptrs' [-Wunsequenced]
  ../inst/include/CImg.h:22316:23: warning: unsequenced modification and access to 'right' [-Wunsequenced]
  ../inst/include/CImg.h:29395:1563: warning: unsequenced modification and access to '_n2x' [-Wunsequenced]
  ../inst/include/CImg.h:29395:1547: warning: multiple unsequenced modifications to '_n3x' [-Wunsequenced]
  ../inst/include/CImg.h:29395:1591: warning: unsequenced modification and access to '_n1x' [-Wunsequenced]
  ../inst/include/CImg.h:29415:1568: warning: unsequenced modification and access to '_n2x' [-Wunsequenced]
  ../inst/include/CImg.h:29415:1552: warning: multiple unsequenced modifications to '_n3x' [-Wunsequenced]
... 9 lines ...
  ../inst/include/CImg.h:29478:920: warning: unsequenced modification and access to '_n1x' [-Wunsequenced]
  ../inst/include/CImg.h:30531:19: warning: unsequenced modification and access to 'right' [-Wunsequenced]
  ../inst/include/CImg.h:31964:1060: warning: multiple unsequenced modifications to '_n2x' [-Wunsequenced]
  ../inst/include/CImg.h:31964:1081: warning: unsequenced modification and access to '_n1x' [-Wunsequenced]
  ../inst/include/CImg.h:44923:18: warning: unsequenced modification and access to 'err' [-Wunsequenced]
  ../inst/include/CImg.h:44931:18: warning: unsequenced modification and access to 'err' [-Wunsequenced]
  ../inst/include/CImg.h:44941:22: warning: unsequenced modification and access to 'err' [-Wunsequenced]
  ../inst/include/CImg.h:45125:18: warning: unsequenced modification and access to 'err' [-Wunsequenced]
  ../inst/include/CImg.h:45133:18: warning: unsequenced modification and access to 'err' [-Wunsequenced]
  ../inst/include/CImg.h:45142:20: warning: unsequenced modification and access to 'err' [-Wunsequenced]
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/imager.Rcheck/00install.out’ for details.

checking installed package size ... NOTE
  installed size is 11.1Mb
  sub-directories of 1Mb or more:
    data      1.4Mb
    doc       3.4Mb
    include   2.6Mb
    libs      2.5Mb
```

## IntClust (0.0.2)
Maintainer: Marijke Van Moerbeke <marijke.vanmoerbeke@uhasselt.be>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘a4Core’ ‘Biobase’ ‘limma’

Packages suggested but not available for checking:
  ‘MLP’ ‘biomaRt’ ‘org.Hs.eg.db’ ‘a4Base’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## IntegratedJM (1.4)
Maintainer: Rudradev Sengupta <rudradev.sengupta@uhasselt.be>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘Biobase’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## JWileymisc (0.2.1)
Maintainer: Joshua F. Wiley <jwiley.psych@gmail.com>  
Bug reports: https://github.com/JWiley/JWileymisc/issues

0 errors | 1 warning  | 0 notes

```
checking whether package ‘JWileymisc’ can be installed ... WARNING
Found the following significant warnings:
  Warning: `legend.margin` must be specified using `margin()`. For the old behavior use legend.spacing
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/JWileymisc.Rcheck/00install.out’ for details.
```

## LANDD (1.1.0)
Maintainer: Shangzhao Qiu <qsz1328@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘GOstats’ ‘GOSemSim’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## largeVis (0.1.9.1)
Maintainer: Amos Elberg <amos.elberg@gmail.com>  
Bug reports: https://github.com/elbamos/largeVis/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 14-55 (benchmarks.Rmd) 
Error: processing vignette 'benchmarks.Rmd' failed with diagnostics:
formal argument "legend.key.size" matched by multiple actual arguments
Execution halted

```

## LDheatmap (0.99-2)
Maintainer: Brad McNeney <mcneney@sfu.ca>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘chopsticks’

Packages suggested but not available for checking:
  ‘rtracklayer’ ‘GenomicRanges’ ‘GenomeInfoDb’ ‘IRanges’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## localgauss (0.35)
Maintainer: Tore Selland Kleppe <tore.kleppe@uis.no>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘localgauss’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/localgauss.Rcheck/00install.out’ for details.
```

## ltbayes (0.3)
Maintainer: Timothy R. Johnson <trjohns@uidaho.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ltbayes’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/ltbayes.Rcheck/00install.out’ for details.
```

## magick (0.3)
Maintainer: Jeroen Ooms <jeroen.ooms@stat.ucla.edu>  
Bug reports: https://github.com/ropensci/magick/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘magick’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/magick.Rcheck/00install.out’ for details.
```

## manhattanly (0.1.0)
Maintainer: Sahir Bhatnagar <sahir.bhatnagar@gmail.com>  
Bug reports: https://github.com/sahirbhatnagar/manhattanly/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in arrange_impl(.data, dots) :
  '.Random.seed' is not an integer vector but of type 'NULL', so ignored
Warning: 'scatter' objects don't have these attributes: 'evaluate'
Valid attributes include:
'type', 'visible', 'showlegend', 'legendgroup', 'opacity', 'name', 'uid', 'hoverinfo', 'stream', 'x', 'x0', 'dx', 'y', 'y0', 'dy', 'text', 'mode', 'hoveron', 'line', 'connectgaps', 'fill', 'fillcolor', 'marker', 'textposition', 'textfont', 'r', 't', 'error_y', 'error_x', 'xaxis', 'yaxis', 'xsrc', 'ysrc', 'textsrc', 'textpositionsrc', 'rsrc', 'tsrc', 'key'

Warning: 'scatter' objects don't have these attributes: 'evaluate'
... 6 lines ...
'type', 'visible', 'showlegend', 'legendgroup', 'opacity', 'name', 'uid', 'hoverinfo', 'stream', 'x', 'x0', 'dx', 'y', 'y0', 'dy', 'text', 'mode', 'hoveron', 'line', 'connectgaps', 'fill', 'fillcolor', 'marker', 'textposition', 'textfont', 'r', 't', 'error_y', 'error_x', 'xaxis', 'yaxis', 'xsrc', 'ysrc', 'textsrc', 'textpositionsrc', 'rsrc', 'tsrc', 'key'

Warning: 'scatter' objects don't have these attributes: 'evaluate'
Valid attributes include:
'type', 'visible', 'showlegend', 'legendgroup', 'opacity', 'name', 'uid', 'hoverinfo', 'stream', 'x', 'x0', 'dx', 'y', 'y0', 'dy', 'text', 'mode', 'hoveron', 'line', 'connectgaps', 'fill', 'fillcolor', 'marker', 'textposition', 'textfont', 'r', 't', 'error_y', 'error_x', 'xaxis', 'yaxis', 'xsrc', 'ysrc', 'textsrc', 'textpositionsrc', 'rsrc', 'tsrc', 'key'

Warning: Ignoring unknown aesthetics: text
Quitting from lines 104-105 (manhattanly.Rmd) 
Error: processing vignette 'manhattanly.Rmd' failed with diagnostics:
attempt to apply non-function
Execution halted
```

## mcprofile (0.2-1)
Maintainer: Daniel Gerhard <00gerhard@gmail.com>

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘mcprofile-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: hoa
> ### Title: Higher order asymptotics using the modified likelihood root
> ### Aliases: hoa
> ### Keywords: misc
> 
... 52 lines ...
3 - 0       2.813   0.594
10 - 0      2.979   0.592

> # computing profiles for the modified likelihood root
> hp <- hoa(dmcp)
> 
> plot(hp)
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L, 1L, 1L,  : 
  replacement has 137 rows, data has 1
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: ggplot2
Waiting for profiling to be done...
Loading required package: mvtnorm
Loading required package: survival
Loading required package: TH.data
Loading required package: MASS

Attaching package: 'TH.data'

The following object is masked from 'package:MASS':

    geyser

Quitting from lines 173-174 (mcprofile.Rmd) 
Error: processing vignette 'mcprofile.Rmd' failed with diagnostics:
replacement has 147 rows, data has 1
Execution halted


checking R code for possible problems ... NOTE
confint.mcprofile : <anonymous>: no visible global function definition
  for ‘na.omit’
confint.mcprofile: no visible global function definition for ‘qnorm’
confint.mcprofile: no visible global function definition for ‘qt’
confint.mcprofile: no visible global function definition for ‘vcov’
confint.mcprofile : <anonymous> : pfun: no visible global function
  definition for ‘predict’
confint.mcprofile : <anonymous>: no visible global function definition
  for ‘uniroot’
... 53 lines ...
  coefficients df.residual family gaussian lm lm.fit model.frame
  model.matrix model.offset model.response model.weights na.omit
  p.adjust pnorm predict printCoefmat pt qchisq qf qnorm qt uniroot
  vcov
Consider adding
  importFrom("stats", "coefficients", "df.residual", "family",
             "gaussian", "lm", "lm.fit", "model.frame", "model.matrix",
             "model.offset", "model.response", "model.weights",
             "na.omit", "p.adjust", "pnorm", "predict", "printCoefmat",
             "pt", "qchisq", "qf", "qnorm", "qt", "uniroot", "vcov")
to your NAMESPACE file.
```

## MendelianRandomization (0.2.0)
Maintainer: Stephen Burgess <sb452@medschl.cam.ac.uk>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘MendelianRandomization-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mr_plot
> ### Title: Draw a scatter plot of the genetic associations and/or causal
> ###   estimates
> ### Aliases: mr_plot mr_plot,MRAll-method mr_plot,MRInput-method
> 
> ### ** Examples
> 
> mr_plot(mr_input(bx = ldlc, bxse = ldlcse, by = chdlodds, byse = chdloddsse),
+   line="ivw")
Error in gg2list(p, width = width, height = height, tooltip = tooltip,  : 
  attempt to apply non-function
Calls: mr_plot ... plotly_build.gg -> ggplotly -> ggplotly.ggplot -> gg2list
Execution halted
```

## MergeGUI (0.2-1)
Maintainer: Xiaoyue Cheng <xycheng@iastate.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘MergeGUI’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/MergeGUI.Rcheck/00install.out’ for details.
```

## MetaIntegrator (1.0.3)
Maintainer: Winston A. Haynes <hayneswa@stanford.edu>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available:
  ‘multtest’ ‘Biobase’ ‘preprocessCore’ ‘GEOquery’ ‘GEOmetadb’

Packages suggested but not available for checking:
  ‘BiocStyle’ ‘BiocGenerics’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## metaMix (0.2)
Maintainer: Sofia Morfopoulou <sofia.morfopoulou.10@ucl.ac.uk>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘Rmpi’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## MissingDataGUI (0.2-5)
Maintainer: Xiaoyue Cheng <xycheng@unomaha.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘MissingDataGUI’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/MissingDataGUI.Rcheck/00install.out’ for details.
```

## mixOmics (6.1.0)
Maintainer: Kim-Anh Le Cao <k.lecao@uq.edu.au>  
Bug reports: mixomics@math.univ-toulouse.fr or
        https://bitbucket.org/klecao/package-mixomics/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘mixOmics-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plsda
> ### Title: Partial Least Squares Discriminant Analysis (PLS-DA).
> ### Aliases: plsda
> ### Keywords: regression multivariate
> 
> ### ** Examples
> 
> ## First example
> data(breast.tumors)
> X <- breast.tumors$gene.exp
> Y <- breast.tumors$sample$treatment
> 
> plsda.breast <- plsda(X, Y, ncomp = 2)
> plotIndiv(plsda.breast, ind.names = TRUE, ellipse = TRUE, legend = TRUE)
Warning: Ignoring unknown aesthetics: label
Warning: Ignoring unknown aesthetics: label
Error in is.finite(x) : default method not implemented for type 'list'
Calls: plotIndiv ... <Anonymous> -> f -> train_cartesian -> <Anonymous> -> f
Execution halted
```

## MixSIAR (3.1.7)
Maintainer: Brian Stock <b1stock@ucsd.edu>  
Bug reports: https://github.com/brianstock/MixSIAR/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘MixSIAR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mixsiar_gui
> ### Title: Run the GUI version of MixSIAR
> ### Aliases: mixsiar_gui
> 
> ### ** Examples
... 6 lines ...
Error in mixsiar_gui() : 
  *** gWidgetsRGtk2 package not able to be loaded. ***
        If 'library('gWidgetsRGtk2')' does not work, MixSIAR GUI will not run.
        On Windows/Linux, try 'install.packages('gWidgetsRGtk2')'.
        On Mac, close R, download and install GTK+ from:
        http://r.research.att.com/#other. Then install latest X11 application
        (xQuartz) from http://xquartz.macosforge.org/landing/.

        If installing GTK+ continues to be problematic, consider using the
        script version of MixSIAR. See manual for help and examples.
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  OK: 26 SKIPPED: 0 FAILED: 9
  1. Error: Wolves ex works (@test_ex1_wolves.R#33) 
  2. Error: Geese ex works (@test_ex2_geese.R#27) 
  3. Error: Lake ex works (@test_ex3_lake.R#27) 
  4. Error: Palmyra ex works (@test_ex4_palmyra.R#27) 
  5. Error: Killer whale ex works (@test_ex5_killerwhale.R#28) 
  6. Error: Snail ex works (@test_ex6_snail.R#27) 
  7. Error: Storm-petrel ex works (@test_ex7_stormpetrel.R#27) 
  8. Error: Cladocera ex works (@test_ex8_cladocera.R#31) 
  9. Error: Isopod ex works (@test_ex9_isopod.R#27) 
  
  Error: testthat unit tests failed
  Execution halted
```

## mlr (2.9)
Maintainer: Bernd Bischl <bernd_bischl@gmx.net>  
Bug reports: https://github.com/mlr-org/mlr/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘mlr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotFilterValues
> ### Title: Plot filter values using ggplot2.
> ### Aliases: plotFilterValues
> 
> ### ** Examples
> 
> fv = generateFilterValuesData(iris.task, method = "chi.squared")
Error in .jnew("weka/core/Attribute", attname[i], .jcast(levels, "java/util/List")) : 
  java.lang.UnsupportedClassVersionError: weka/core/Attribute : Unsupported major.minor version 51.0
Calls: generateFilterValuesData ... read_model_frame_into_Weka -> read_data_into_Weka -> .jnew -> .External
Execution halted
** found \donttest examples: check also with --run-donttest

checking tests ... ERROR
Running the tests in ‘tests/run-base.R’ failed.
Last 13 lines of output:
  > test_check("mlr", filter = "base")
  Loading required package: mlr
  Loading required package: BBmisc
  Loading required package: ggplot2
  Loading required package: ParamHelpers
  Loading required package: stringi
  Timing stopped at: 0.018 0.002 0.022 
  Error in ._jobjRef_dollar(x[["jobj"]], name) : 
    no field, method or inner class called 'handled' 
  Calls: test_check ... <Anonymous> -> isTRUE -> $ -> $.Throwable -> ._jobjRef_dollar
  testthat results ================================================================
  OK: 95 SKIPPED: 0 FAILED: 0
  Execution halted

checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘C50’ ‘GPfit’ ‘RRF’
```

## Momocs (1.0.0)
Maintainer: Vincent Bonhomme <bonhomme.vincent@gmail.com>  
Bug reports: https://github.com/vbonhomme/Momocs/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘Momocs-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot2
> ### Title: Plots Principal Component Analysis ala ggplot2
> ### Aliases: plot2 plot2.PCA
> 
> ### ** Examples
> 
> data(bot)
> bp <- PCA(efourier(bot, 8))
> plot2(bp)
Error in abs(.x.range.gg(gg0)) : 
  non-numeric argument to mathematical function
Calls: plot2 -> plot2.PCA
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following object is masked from 'package:stats':

    filter

The following object is masked from 'package:base':

... 8 lines ...
'nb.pts' missing and set to 91
'degree' missing and set to 5
'nb.pts' missing and set to 91
'degree' missing and set to 5
'nb.pts' missing and set to 91
'degree' missing and set to 5
'nb.h' not provided and set to 10 (99% harmonic power)
Quitting from lines 396-397 (Momocs_speed_dating.Rmd) 
Error: processing vignette 'Momocs_speed_dating.Rmd' failed with diagnostics:
non-numeric argument to mathematical function
Execution halted
```

## morse (2.2.0)
Maintainer: Philippe Veber <philippe.veber@univ-lyon1.fr>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘morse’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/morse.Rcheck/00install.out’ for details.
```

## mosaic (0.14.4)
Maintainer: Randall Pruim <rpruim@calvin.edu>  
Bug reports: https://github.com/ProjectMOSAIC/mosaic/issues

1 error  | 1 warning  | 2 notes

```
checking examples ... ERROR
Running examples in ‘mosaic-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mplot
> ### Title: Generic plotting
> ### Aliases: mplot mplot.TukeyHSD mplot.data.frame mplot.default mplot.lm
> ###   mplot.summary.lm
> 
... 17 lines ...
> ##D mplot( HELPrct, "histogram" )
> ## End(Not run)
> mplot(summary(lm(width ~ length * sex, data=KidsFeet)), system="ggplot")
> mplot(summary(lm(width ~ length * sex, data=KidsFeet)), rows=c("sex", "length"))
> mplot(summary(lm(width ~ length * sex, data=KidsFeet)), rows=TRUE)
> mplot(TukeyHSD( lm(age ~ substance, data=HELPrct) ) )
> mplot(TukeyHSD( lm(age ~ substance, data=HELPrct) ), system="ggplot2" )
Error in as.character(what) : 
  cannot coerce type 'closure' to vector of type 'character'
Calls: <Anonymous> ... <Anonymous> -> list_to_dataframe -> rbind.fill -> <Anonymous>
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 87-90 (GraphicsWithMosaic.Rmd) 
Error: processing vignette 'GraphicsWithMosaic.Rmd' failed with diagnostics:
cannot coerce type 'closure' to vector of type 'character'
Execution halted


checking installed package size ... NOTE
  installed size is  9.4Mb
  sub-directories of 1Mb or more:
    R     2.0Mb
    doc   6.8Mb

checking R code for possible problems ... NOTE
mMap: no visible binding for global variable ‘facet’
mScatter: no visible binding for global variable ‘facet’
mUniplot: no visible binding for global variable ‘facet’
Undefined global functions or variables:
  facet
```

## mrMLM (1.3)
Maintainer: Yuanming Zhang<soyzhang@mail.hzau.edu.cn>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘mrMLM’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/mrMLM.Rcheck/00install.out’ for details.
```

## mtconnectR (1.0.1)
Maintainer: Subramanyam Ravishankar <subramanyam@systeminsights.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Error: `mapped_plot` not equal to lazyLoadDBfetch(c(36596L, 320L), datafile, compressed, envhook).
  Component "layers": Component 1: Component 3: Component 2: Component 6: Component 5: target, current do not match when deparsed
  Component "layers": Component 1: Component 7: Component 2: Component 1: target, current do not match when deparsed
  Component "layers": Component 1: Component 9: Component 2: Names: 11 string mismatches
  Component "layers": Component 1: Component 9: Component 2: Length mismatch: comparison on first 11 components
  Component "layers": Component 1: Component 9: Component 2: Component 1: target, current do not match when deparsed
  Component "layers": Component 1: Component 9: Component 2: Component 2: target, current do not match when deparsed
  Component "layers": Component 1: Component 9: Component 2: Component 3: target, current do not match when deparsed
  Component "layers": Component 1: Component 9: Component 2: Component 4: Modes of target, current: function, list
  Co
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 0
  Execution halted
```

## multiDimBio (1.0.1)
Maintainer: Samuel V. Scarpino <scarpino@utexas.edu>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘pcaMethods’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## mvdalab (1.1)
Maintainer: Nelson Lee Afanador <nelson.afanador@mvdalab.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘mvdalab-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ScoreContrib
> ### Title: Generates a score contribution plot
> ### Aliases: ScoreContrib
> 
> ### ** Examples
> 
> data(Penta)
> ## Number of bootstraps set to 500 to demonstrate flexibility
> ## Use a minimum of 1000 (default) for results that support bootstraping
> mod1 <- plsFit(log.RAI ~., scale = TRUE, data = Penta[, -1], 
+                ncomp = 2, validation = "oob", boots = 500)
> Score.Contributions1 <- ScoreContrib(mod1, obs1 = 1, obs2 = 3)
> plot(Score.Contributions1)
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L, 1L, 1L,  : 
  replacement has 15 rows, data has 1
Calls: plot ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## mwaved (1.1.4)
Maintainer: Justin Rory Wishart <justin.wishart@mq.edu.au>  
Bug reports: https://github.com/jrwishart/mwaved/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘mwaved’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/mwaved.Rcheck/00install.out’ for details.
```

## myTAI (0.4.0)
Maintainer: Hajk-Georg Drost <hgd23@cam.ac.uk>  
Bug reports: https://github.com/HajkD/myTAI/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘edgeR’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## NFP (0.99.1)
Maintainer: Yang Cao <yiluheihei@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘graph’ ‘KEGGgraph’

Packages suggested but not available for checking:
  ‘BiocStyle’ ‘graphite’ ‘NFPdata’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## NMF (0.20.6)
Maintainer: Renaud Gaujoux <renaud@tx.technion.ac.il>  
Bug reports: http://github.com/renozao/NMF/issues

1 error  | 1 warning  | 4 notes

```
checking examples ... ERROR
Running examples in ‘NMF-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: nmfModel
> ### Title: Factory Methods NMF Models
> ### Aliases: nmfModel nmfModel,data.frame,data.frame-method
> ###   nmfModel,formula,ANY-method nmfModel,matrix,ANY-method
> ###   nmfModel,matrix,matrix-method nmfModel-methods
... 222 lines ...
features: 20 
basis/rank: 6 
samples: 10 
fixed coef [3]:
  gr = <1, 2>
  b = 0.0101301828399301, 0.21454192395322, ..., 0.767450851621106
> 
> # 3-rank model that fits a given ExpressionSet (with fixed coef terms)
> e <- ExpressionSet(x)
Error: could not find function "ExpressionSet"
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 385-398 (NMF-vignette.Rnw) 
Error: processing vignette 'NMF-vignette.Rnw' failed with diagnostics:
unable to find required package 'Biobase'
Execution halted


checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘RcppOctave’ ‘doMPI’ ‘Biobase’

checking R code for possible problems ... NOTE
.wrapResult: no visible global function definition for ‘exprs’
algorithm,NMFStrategyOctave: no visible global function definition for
  ‘fstop’
evar,ANY: no visible binding for global variable ‘Biobase’
nmf,matrix-numeric-NMFStrategy : run.all: no visible binding for global
  variable ‘n’
nmf,matrix-numeric-NMFStrategy : run.all: no visible binding for global
  variable ‘RNGobj’
nmfModel,formula-ANY : merge_pdata: no visible global function
  definition for ‘pData’
nmfModel,formula-ANY: no visible global function definition for ‘exprs’
rss,matrix: no visible binding for global variable ‘Biobase’
Undefined global functions or variables:
  Biobase RNGobj exprs fstop n pData

checking Rd cross-references ... NOTE
Packages unavailable to check Rd xrefs: ‘RcppOctave’, ‘Biobase’

checking data for non-ASCII characters ... NOTE
  Error in .requirePackage(package) : 
    unable to find required package 'Biobase'
  Calls: <Anonymous> ... .extendsForS3 -> extends -> getClassDef -> .requirePackage
  Execution halted
```

## nparACT (0.6)
Maintainer: Christine Blume <christine.blume@sbg.ac.at>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘nparACT-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: nparACT-package
> ### Title: Non-Parametric Measures of Actigraphy Data
> ### Aliases: nparACT-package nparACT
> ### Keywords: package
> 
> ### ** Examples
> 
> # example function nparACT_base
> data(sleepstudy)
> r <- nparACT_base("sleepstudy", SR = 4/60) 
Warning: Ignoring unknown aesthetics: x
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L, 1L, 1L,  : 
  replacement has 96 rows, data has 1
Calls: nparACT_base ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## npregfast (1.3.0)
Maintainer: Marta Sestelo <sestelo@uvigo.es>  
Bug reports: http://github.com/sestelo/npregfast/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘npregfast’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/npregfast.Rcheck/00install.out’ for details.
```

## onlineCPD (1.0)
Maintainer: Zachary Zanussi <zachary9506@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘onlineCPD-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: findCP
> ### Title: Wisely Pick Changepoints
> ### Aliases: findCP
> 
> ### ** Examples
> 
> ## Two changepoints within \code{buffer} points:
> set.seed(5)
> y <- c(rnorm(15,0.5,0.01),0.505,0.51,0.015,0.52,rnorm(15,0.6,0.05))
> res <- offlineCPD(y)
> plot(res,buffer=10)     ## Will replace the two changepoints
Error: Aesthetics must be either length 1 or the same as the data (1): x, y, xend, yend, alpha
Execution halted
```

## OriGen (1.4.3)
Maintainer: John Michael O. Ranola <ranolaj@uw.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘OriGen’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/OriGen.Rcheck/00install.out’ for details.
```

## OutbreakTools (0.1-14)
Maintainer: Thibaut Jombart <thibautjombart@gmail.com>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘OutbreakTools-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ToyOutbreakRaw
> ### Title: Raw simulated outbreak dataset
> ### Aliases: ToyOutbreakRaw
> ### Keywords: dataset
> 
... 572 lines ...
> plot(get.contacts(x, from=2, to=2.1), main="Contact network - day 2",
+      displaylabels=TRUE)
> plot(get.contacts(x, from=3, to=3.1), main="Contact network - day 3",
+      displaylabels=TRUE)
> 
> ## Mapping the outbreak (by sex) ##
> plot(x,'geo',location=c('lon','lat'),zoom=13,colorBy='Sex')
Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=13&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> rescale -> zero_range
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in .local(x, ...) :
  The following sequence IDs are not in the dataset: 311, 222
Warning in .local(x, ...) :
  The following sequence IDs are not in the dataset: 311, 222
Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
  replaced special characters in figure filename "figs/OutbreakTools-out.width==".8\\textwidth"" -> "figs/OutbreakTools-out.width___.8\\textwidth_"
Warning in (if (out_format(c("latex", "sweave", "listings"))) sanitize_fn else paste0)(path,  :
  dots in figure paths replaced with _ ("figs/OutbreakTools-out_width____8//textwidth_")
Warning: The shape palette can deal with a maximum of 6 discrete values because
more than 6 becomes difficult to discriminate; you have 11. Consider
specifying shapes manually if you must have them.
Warning: Removed 58 rows containing missing values (geom_point).
Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
Warning: `show_guide` has been deprecated. Please use `show.legend` instead.
Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=51.516439,-0.171146&zoom=14&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Quitting from lines 1022-1023 (OutbreakTools.Rnw) 
Error: processing vignette 'OutbreakTools.Rnw' failed with diagnostics:
x must be length 1 or 2
Execution halted

```

## pcadapt (3.0.2)
Maintainer: Keurcien Luu <keurcien.luu@imag.fr>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 220-221 (pcadapt.Rmd) 
Error: processing vignette 'pcadapt.Rmd' failed with diagnostics:
there is no package called 'qvalue'
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘qvalue’
```

## pcaPA (2.0.2)
Maintainer: Carlos A. Arias <caariasr22@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘pcaPA’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/pcaPA.Rcheck/00install.out’ for details.
```

## pcrsim (1.0.0)
Maintainer: Oskar Hansson <oskar.hansson@fhi.no>  
Bug reports: https://github.com/OskarHansson/pcrsim/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘pcrsim’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/pcrsim.Rcheck/00install.out’ for details.
```

## PGRdup (0.2.2.1)
Maintainer: J. Aravind <j.aravind@icar.gov.in>

0 errors | 1 warning  | 2 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
ERROR: Duplicated records found in prim.key field
ERROR: NULL records found in prim.key field
OK: No duplicated records found in prim.key field
OK: No NULL records found in prim.key field
Fuzzy matching
Phonetic matching
Semantic matching
Fuzzy matching
Phonetic matching
Semantic matching
Quitting from lines 995-1061 (Introduction.Rmd) 
Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
formal argument "axis.ticks.y" matched by multiple actual arguments
Execution halted


checking R code for possible problems ... NOTE
ViewProbDup: possible error in theme(panel.background =
  element_rect(fill = "transparent"), axis.title.x =
  element_text(colour = "black", margin = margin(20, 0, 20, 0)),
  axis.ticks.x = element_blank(), axis.text.x = element_blank(),
  axis.ticks.y = element_blank(), axis.line = element_line(colour =
  NA), axis.line.y = element_line(colour = "black"), axis.ticks.y =
  element_line(colour = "black"), axis.text.y = element_text(colour =
  "black"), axis.title.y = element_blank(), plot.margin = unit(c(0,
  0.5, 0.5, 0.5), "cm"), panel.grid = element_blank()): formal argument
  "axis.ticks.y" matched by multiple actual arguments

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘RecordLinkage’
```

## photobiologyInOut (0.4.11)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/photobiologyinout/

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Read 3 items
Read 16 items
Quitting from lines 179-180 (user-guide.Rnw) 
Error: processing vignette 'user-guide.Rnw' failed with diagnostics:
attempt to apply non-function
Execution halted

```

## photobiologyPlants (0.4.0)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/photobiologyplants

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 79-81 (dataplotsPlants.Rnw) 
Error: processing vignette 'dataplotsPlants.Rnw' failed with diagnostics:
attempt to apply non-function
Execution halted

```

## PKgraph (1.7)
Maintainer: Xiaoyong Sun <johnsunx1@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘rggobi’

Depends: includes the non-default packages:
  ‘RGtk2’ ‘gWidgetsRGtk2’ ‘cairoDevice’ ‘lattice’ ‘rggobi’ ‘ggplot2’
  ‘proto’
Adding so many packages to the search path is excessive and importing
selectively is preferable.

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## PlasmaMutationDetector (1.5.2)
Maintainer: Yves Rozenholc <yves.rozenholc@parisdescartes.fr>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available:
  ‘GenomicRanges’ ‘VariantAnnotation’ ‘S4Vectors’ ‘Rsamtools’
  ‘rtracklayer’ ‘SummarizedExperiment’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## playwith (0.9-54)
Maintainer: Felix Andrews <felix@nfrac.org>

1 error  | 0 warnings | 1 note 

```
checking whether package ‘playwith’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/playwith.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘latticist’
```

## plotluck (1.0.0)
Maintainer: Stefan Schroedl <stefan.schroedl@gmx.de>  
Bug reports: https://github.com/stefan-schroedl/plotluck/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Factor variable price has too many levels (11602), truncating to 30
  *** 2D box/violin plot ***
  Factor variable rating has too many levels (91), truncating to 30
  Error in match.arg(opts$geom, choices.geom) : 
    'arg' should be one of "spine", "heat", "auto"
  Calls: test_check ... print -> plotluck -> determine.plot.type.1 -> match.arg
  In addition: Warning message:
  'switch' is deprecated.
  Use 'strip.position' instead.
  See help("Deprecated") 
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 0
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 83-85 (plotluck.Rmd) 
Error: processing vignette 'plotluck.Rmd' failed with diagnostics:
'sum' not defined for ordered factors
Execution halted

```

## plotly (4.5.2)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: https://github.com/ropensci/plotly/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘plotly-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: style
> ### Title: Modify trace(s)
> ### Aliases: style
> 
> ### ** Examples
> 
> 
> p <- qplot(data = mtcars, wt, mpg, geom = c("point", "smooth"))
> # keep the hover info for points, but remove it for the line/ribbon
> style(p, hoverinfo = "none", traces = c(2, 3))
Error in gg2list(p, width = width, height = height, tooltip = tooltip,  : 
  attempt to apply non-function
Calls: style ... plotly_build.gg -> ggplotly -> ggplotly.ggplot -> gg2list
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Error: factor levels determine tick order (@test-cookbook-axes.R#28) 
  2. Error: ylim hides points (@test-cookbook-axes.R#36) 
  3. Error: scale_y(limits) hides points (@test-cookbook-axes.R#41) 
  4. Error: Using coord_cartesian zooms into the area (@test-cookbook-axes.R#49) 
  5. Error: A scatterplot with regular (linear) axis scaling (@test-cookbook-axes.R#64) 
  6. Error: log2 scaling of the y axis (with visually-equal spacing) (@test-cookbook-axes.R#71) 
  7. Error: log2 coordinate transformation with visually-diminishing spacing (@test-cookbook-axes.R#77) 
  8. Error: log2 transform with labels (@test-cookbook-axes.R#86) 
  9. Error: scale_y_log10 (@test-cookbook-axes.R#92) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## prcbench (0.5.0)
Maintainer: Takaya Saito <takaya.saito@outlook.com>  
Bug reports: https://github.com/takayasaito/prcbench/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘prcbench-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: autoplot.evalcurve
> ### Title: Plot the result of Precision-Recall curve evaluation
> ### Aliases: autoplot.evalcurve
> 
> ### ** Examples
> 
> library(ggplot2)
> 
> ## Plot evaluation results on test datasets r1, r2, and r3
> testset <- create_testset("curve", c("c1", "c2", "c3"))
> toolset <- create_toolset(set_names = "crv5")
> eres1 <- run_evalcurve(testset, toolset)
Error: PerfMeas needed for this function to work. Please install it.
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Error: run_benchmark (@test_main_benchmark.R#32) 
  2. Failure: run_benchmark: testset (@test_main_benchmark.R#58) 
  3. Failure: run_benchmark: testset (@test_main_benchmark.R#61) 
  4. Failure: run_benchmark: toolset (@test_main_benchmark.R#84) 
  5. Failure: run_benchmark: toolset (@test_main_benchmark.R#87) 
  6. Error: run_benchmark: toolset & testset (@test_main_benchmark.R#106) 
  7. Error: run_benchmark: times (@test_main_benchmark.R#131) 
  8. Failure: run_benchmark: unit (@test_main_benchmark.R#166) 
  9. Failure: run_benchmark: unit (@test_main_benchmark.R#166) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 119-124 (introduction.Rmd) 
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
PerfMeas needed for this function to work. Please install it.
Execution halted

```

## pRF (1.2)
Maintainer: Ankur Chakravarthy <ankur.chakravarthy.10@ucl.ac.uk>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘multtest’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## proportion (1.2.3)
Maintainer: Rajeswaran Viswanathan <v.rajeswaran@gmail.com>  
Bug reports: https://github.com/RajeswaranV/proportion/issues

0 errors | 1 warning  | 1 note 

```
checking sizes of PDF files under ‘inst/doc’ ... WARNING
  ‘gs+qpdf’ made some significant size reductions:
     compacted ‘proportion.pdf’ from 689Kb to 387Kb
  consider running tools::compactPDF(gs_quality = "ebook") on these files

checking Rd cross-references ... NOTE
Packages unavailable to check Rd xrefs: ‘PropCIs’, ‘BlakerCI’, ‘prevalence’
```

## PSCBS (0.61.0)
Maintainer: Henrik Bengtsson <henrikb@braju.com>  
Bug reports: https://github.com/HenrikBengtsson/PSCBS/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘DNAcopy’

Package suggested but not available for checking: ‘aroma.light’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## quanteda (0.9.8)
Maintainer: Kenneth Benoit <kbenoit@lse.ac.uk>  
Bug reports: https://github.com/kbenoit/quanteda/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
      as.dist, dist
  
  The following object is masked from 'package:base':
  
      as.matrix
  
  testthat results ================================================================
  OK: 238 SKIPPED: 1 FAILED: 2
  1. Failure: test plot.kwic facet order parameter (@testPlots.R#70) 
  2. Failure: test plot.kwic keeps order of keywords passed (@testPlots.R#81) 
  
  Error: testthat unit tests failed
  Execution halted
```

## quickpsy (0.1.3)
Maintainer: Linares Daniel <danilinares@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘quickpsy-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotcurves
> ### Title: Plot the curves
> ### Aliases: plotcurves
> 
> ### ** Examples
> 
> library(MPDiR) # contains the Vernier data
> fit <- quickpsy(Vernier, Phaseshift, NumUpward, N,
+                 grouping = .(Direction, WaveForm, TempFreq), B = 5)
> plotcurves(fit)
Error: Aesthetics must be either length 1 or the same as the data (8): ymin, ymax, colour, x
Execution halted
```

## radiant.model (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.model/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.model-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.model.predict
> ### Title: Plot method for model.predict functions
> ### Aliases: plot.model.predict
> 
> ### ** Examples
> 
> regress("diamonds", "price", c("carat","clarity")) %>%
+   predict(pred_cmd = "carat = 1:10") %>%
+   plot(xvar = "carat")
> logistic("titanic", "survived", c("pclass","sex","age"), lev = "Yes") %>%
+   predict(pred_cmd="pclass=levels(pclass), sex=levels(sex), age=seq(0,100,20)") %>%
+   plot(xvar = "age", color = "sex", facet_col = "pclass")
Error in rep(yes, length.out = length(ans)) : 
  attempt to replicate an object of type 'closure'
Calls: %>% ... <Anonymous> -> plot -> plot.model.predict -> ifelse
Execution halted
```

## rangeMapper (0.3-0)
Maintainer: Mihai Valcu <valcu@orn.mpg.de>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘rangeMapper-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: rangeTraits
> ### Title: A container of functions to apply on a 'SpatialPolygons' object
> ### Aliases: rangeTraits
> 
> ### ** Examples
... 19 lines ...
 Geospatial Data Abstraction Library extensions to R successfully loaded
 Loaded GDAL runtime: GDAL 1.11.4, released 2016/01/25
 Path to GDAL shared files: /Users/hadley/R-revdep/rgdal/gdal
 Loaded PROJ.4 runtime: Rel. 4.9.1, 04 March 2015, [PJ_VERSION: 491]
 Path to PROJ.4 shared files: /Users/hadley/R-revdep/rgdal/proj
 Linking to sp version: 1.2-3 
> r = readOGR(troaed$dsn, troaed$layer)
Error in ogrInfo(dsn = dsn, layer = layer, encoding = encoding, use_iconv = use_iconv,  : 
  Layer not found
Calls: readOGR ... suppressMessages -> withCallingHandlers -> ogrInfo -> .Call
Execution halted
```

## rnoaa (0.6.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: http://www.github.com/ropensci/rnoaa/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Loading required package: ggplot2
Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=Delhi&zoom=11&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=Delhi&sensor=false
Quitting from lines 103-110 (rnoaa_openaq.Rmd) 
Error: processing vignette 'rnoaa_openaq.Rmd' failed with diagnostics:
x must be length 1 or 2
Execution halted


checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    vign   1.1Mb
```

## robCompositions (2.0.2)
Maintainer: Matthias Templ <templ@tuwien.ac.at>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in warn_if_args_exist(list(...)) :
  Extra arguments: 'colour', 'alpha', 'shape' are being ignored.  If these are meant to be aesthetics, submit them using the 'mapping' variable within ggpairs with ggplot2::aes or ggplot2::aes_string.
Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
Quitting from lines 905-906 (imputation.Rnw) 
Error: processing vignette 'imputation.Rnw' failed with diagnostics:
only defined on a data frame with all numeric variables
Execution halted


checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘StatDA’
```

## RPPanalyzer (1.4.3)
Maintainer: Astrid Wachter <astrid.wachter@med.uni-goettingen.de>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘limma’ ‘Biobase’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## RSA (0.9.10)
Maintainer: Felix Schönbrodt <felix@nicebread.de>

0 errors | 1 warning  | 0 notes

```
checking whether package ‘RSA’ can be installed ... WARNING
Found the following significant warnings:
  Warning: loading Rplot failed
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/RSA.Rcheck/00install.out’ for details.
```

## RSSL (0.6)
Maintainer: Jesse Krijthe <jkrijthe@gmail.com>  
Bug reports: http://www.github.com/jkrijthe/RSSL

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘RSSL-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: EMLeastSquaresClassifier
> ### Title: An Expectation Maximization like approach to Semi-Supervised
> ###   Least Squares Classification
> ### Aliases: EMLeastSquaresClassifier
> 
... 27 lines ...
+ )
> 
> df %>% 
+  ggplot(aes(x=X1,y=X2,color=Class)) +
+  geom_point() +
+  coord_equal() +
+  scale_y_continuous(limits=c(-2,2)) +
+  stat_classifier(aes(linetype=..classifier..),
+                  classifiers=classifiers)
Error: stat_classifier requires the following missing aesthetics: z
Execution halted
```

## rstan (2.12.1)
Maintainer: Ben Goodrich <benjamin.goodrich@columbia.edu>  
Bug reports: https://github.com/stan-dev/rstan/issues/

1 error  | 0 warnings | 0 notes

```
checking whether package ‘rstan’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/rstan.Rcheck/00install.out’ for details.
```

## rstanarm (2.12.1)
Maintainer: Ben Goodrich <benjamin.goodrich@columbia.edu>  
Bug reports: https://github.com/stan-dev/rstanarm/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘rstanarm’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/rstanarm.Rcheck/00install.out’ for details.
```

## rsvg (1.0)
Maintainer: Jeroen Ooms <jeroen.ooms@stat.ucla.edu>  
Bug reports: https://github.com/jeroenooms/rsvg/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘rsvg’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/rsvg.Rcheck/00install.out’ for details.
```

## Rz (0.9-1)
Maintainer: Masahiro Hayashi <rinm884@gmail.com>

0 errors | 1 warning  | 2 notes

```
checking dependencies in R code ... WARNING
'library' or 'require' call to ‘vcd’ in package code.
  Please use :: or requireNamespace() instead.
  See section 'Suggested packages' in the 'Writing R Extensions' manual.
Packages in Depends field not imported from:
  ‘foreign’ ‘ggplot2’ ‘grid’ ‘psych’
  These packages need to be imported from (in the NAMESPACE file)
  for when this namespace is loaded but not attached.
Unexported object imported by a ':::' call: ‘foreign:::adQuote’
  See the note in ?`:::` about the use of this operator.
  Including base/recommended package(s):
  ‘foreign’

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  summary.CrossTable
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking R code for possible problems ... NOTE
calc.hist.breaks: no visible global function definition for
  ‘nclass.Sturges’
calc.hist.breaks: no visible global function definition for ‘nclass.FD’
calc.hist.breaks: no visible global function definition for
  ‘nclass.scott’
check.class: no visible global function definition for ‘as.formula’
gtkColorSelectionForPlotGetColor: no visible global function definition
  for ‘rgb’
gtkColorSelectionWidgetNew: no visible global function definition for
... 9 lines ...
write.spss: no visible global function definition for ‘write.table’
write.stata: no visible global function definition for ‘write.table’
Undefined global functions or variables:
  addmargins as.formula assocstats colors localeToCharset nclass.FD
  nclass.Sturges nclass.scott rgb theme_grey write.table
Consider adding
  importFrom("grDevices", "colors", "nclass.FD", "nclass.Sturges",
             "nclass.scott", "rgb")
  importFrom("stats", "addmargins", "as.formula")
  importFrom("utils", "localeToCharset", "write.table")
to your NAMESPACE file.
```

## scmamp (0.2.5)
Maintainer: Borja Calvo <borja.calvo@ehu.eus>  
Bug reports: https://github.com/b0rxa/scmamp/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘graph’ ‘Rgraphviz’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## sdmvspecies (0.3.2)
Maintainer: Xiaoquan Kong <u1mail2me@gmail.com>  
Bug reports: https://github.com/howl-anderson/sdmvspecies/issues

0 errors | 1 warning  | 0 notes

```
checking for executable files ... WARNING
Found the following executable file:
  inst/external/env/bio14.bil
Source packages should not contain undeclared executable files.
See section ‘Package structure’ in the ‘Writing R Extensions’ manual.
```

## seewave (2.0.4)
Maintainer: Jerome Sueur <sueur@mnhn.fr>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘seewave’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/seewave.Rcheck/00install.out’ for details.
```

## SensusR (2.0.0)
Maintainer: Matthew S. Gerber <gerber.matthew@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘SensusR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.LocationDatum
> ### Title: Plot location data.
> ### Aliases: plot.LocationDatum
> 
> ### ** Examples
... 189 lines ...
[1] "71% done merging data for WlanDatum (5 of 7)."
[1] "85% done merging data for WlanDatum (6 of 7)."
[1] "100% done merging data for WlanDatum (7 of 7)."
[1] "Creating data frame for WlanDatum."
> plot(data$LocationDatum)
Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=38.0676352725243,-78.9510441850485&zoom=10&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=38.0676352725243,-78.9510441850485&sensor=false
Warning: `panel.margin` is deprecated. Please use `panel.spacing` property instead
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> rescale -> zero_range
Execution halted
```

## SeqFeatR (0.2.3)
Maintainer: Bettina Budeus <bettina.budeus@stud.uni-due.de>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘Biostrings’ ‘qvalue’ ‘widgetTools’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## sgd (1.1)
Maintainer: Dustin Tran <dustin@cs.columbia.edu>  
Bug reports: https://github.com/airoldilab/sgd/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘sgd’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/sgd.Rcheck/00install.out’ for details.
```

## shiny (0.14)
Maintainer: Winston Chang <winston@rstudio.com>  
Bug reports: https://github.com/rstudio/shiny/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  
  testthat results ================================================================
  OK: 429 SKIPPED: 0 FAILED: 7
  1. Error: ggplot coordmap (@test-plot-coordmap.R#34) 
  2. Failure: ggplot coordmap with facet_wrap (@test-plot-coordmap.R#80) 
  3. Error: ggplot coordmap with facet_wrap (@test-plot-coordmap.R#81) 
  4. Failure: ggplot coordmap with facet_grid (@test-plot-coordmap.R#129) 
  5. Error: ggplot coordmap with facet_grid (@test-plot-coordmap.R#130) 
  6. Failure: ggplot coordmap with 2D facet_grid (@test-plot-coordmap.R#214) 
  7. Error: ggplot coordmap with 2D facet_grid (@test-plot-coordmap.R#215) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  6.8Mb
  sub-directories of 1Mb or more:
    www   5.7Mb
```

## simmr (0.3)
Maintainer: Andrew Parnell <andrew.parnell@ucd.ie>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘simmr’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/simmr.Rcheck/00install.out’ for details.
```

## sjPlot (2.1.0)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/devel/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjPlot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: adjust_plot_range
> ### Title: Adjust y range of ggplot-objects
> ### Aliases: adjust_plot_range
> 
> ### ** Examples
... 7 lines ...
> gp <- sjp.frq(efc$e15relat, prnt.plot = FALSE)
> # show current plot
> plot(gp$plot)
> # show adjusted plot
> adjust_plot_range(gp$plot)
Scale for 'y' is already present. Adding another scale for 'y', which will
replace the existing scale.
Error in abs(gy$panel$ranges[[1]]$y.range[1]) : 
  non-numeric argument to mathematical function
Calls: adjust_plot_range
Execution halted
```

## snpEnrichment (1.7.0)
Maintainer: Mickael Canouil <mickael.canouil@cnrs.fr>  
Bug reports: https://github.com/mcanouil/snpEnrichment/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘snpStats’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## soc.ca (0.7.3)
Maintainer: Anton Grau Larsen <agl.dbp@cbs.dk>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘soc.ca-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: add.to.label
> ### Title: Add values to label
> ### Aliases: add.to.label
> 
> ### ** Examples
... 76 lines ...
[13] "Film:\n Horror"        "Film:\n Musical"       "Film:\n Romance"      
[16] "Film:\n SciFi"         "Art:\n Impressionism"  "Art:\n Landscape"     
[19] "Art:\n ModernArt"      "Art:\n PerformanceArt" "Art:\n Portrait"      
[22] "Art:\n RenaissanceArt" "Art:\n StillLife"      "Eat:\n Fish&Chips"    
[25] "Eat:\n FrenchRest"     "Eat:\n IndianRest"     "Eat:\n ItalianRest"   
[28] "Eat:\n Pub"            "Eat:\n SteakHouse"    
> map.ctr(result.label)
Warning: Ignoring unknown aesthetics: fill
Error in zero_range(from) : x must be length 1 or 2
Calls: <Anonymous> ... transform_position -> lapply -> FUN -> rescale -> zero_range
Execution halted
```

## SpaDES (1.2.0)
Maintainer: Alex M Chubaty <alexander.chubaty@canada.ca>  
Bug reports: https://github.com/PredictiveEcology/SpaDES/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  ...
  
  
  testthat results ================================================================
  OK: 891 SKIPPED: 16 FAILED: 3
  1. Failure: simulation runs with simInit and spades (@test-simulation.R#82) 
  2. Failure: simulation runs with simInit and spades (@test-simulation.R#83) 
  3. Failure: simulation runs with simInit and spades (@test-simulation.R#84) 
  
  Error: testthat unit tests failed
  In addition: Warning message:
  In fun(libname, pkgname) : couldn't connect to display ""
  Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘fastshp’
```

## sparkTable (1.2.0)
Maintainer: Alexander Kowarik <alexander.kowarik@statistik.gv.at>  
Bug reports: https://github.com/alexkowa/sparkTable/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sparkTable-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot-methods
> ### Title: Plot objects of class 'sparkline', 'sparkbar', 'sparkhist' or
> ###   'sparkbox'
> ### Aliases: plot plot-methods plot,sparkbar-method plot,sparkbox-method
> ###   plot,sparkline-method plot,sparkhist-method
> 
> ### ** Examples
> 
>   data(pop)
>   x <- pop[pop[,2]=="Insgesamt",3]
>   a <- newSparkLine(values=x, pointWidth=8)
>   plot(a)
Error in theme(legend.position = "none", line = element_blank(), text = element_blank(),  : 
  formal argument "axis.ticks" matched by multiple actual arguments
Calls: plot -> plot -> theme
Execution halted

checking R code for possible problems ... NOTE
plot,sparkbar: possible error in theme(legend.position = "none", line =
  element_blank(), text = element_blank(), title = element_blank(),
  axis.line = element_blank(), axis.ticks = element_blank(),
  legend.background = element_rect(fill = "white", colour = NA),
  plot.background = element_blank(), strip.background =
  element_rect(fill = "white", colour = "white"), panel.margin =
  unit(0, "null"), plot.margin = rep(unit(0, "null"), 4), panel.grid =
  element_blank(), axis.ticks = element_blank(), axis.ticks.length =
  unit(0, "null"), axis.text.x = element_blank(), : formal argument
... 33 lines ...
  axis.line = element_blank(), axis.ticks = element_blank(),
  legend.background = element_rect(fill = "white", colour = NA),
  plot.background = element_blank(), strip.background =
  element_rect(fill = "white", colour = "white"), panel.margin =
  unit(0, "null"), plot.margin = rep(unit(0, "null"), 4), panel.grid =
  element_blank(), axis.ticks = element_blank(), axis.ticks.length =
  unit(0, "null"), axis.text.x = element_blank(), : formal argument
  "axis.ticks" matched by multiple actual arguments
plot,sparkline: possible error in axis.text.y = element_blank(),
  panel.background = element_rect(fill = allColors(x)[4])): formal
  argument "axis.ticks" matched by multiple actual arguments
```

## specmine (1.0)
Maintainer: Christopher Costa <chrisbcl@hotmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘xcms’ ‘MAIT’ ‘genefilter’ ‘impute’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## ss3sim (0.9.2)
Maintainer: Sean Anderson <sean@seananderson.ca>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ss3sim-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot_scalar_boxplot
> ### Title: Print scalar values as boxplots.
> ### Aliases: plot_scalar_boxplot
> 
> ### ** Examples
> 
> scalar_dat$depletion <- with(scalar_dat,
+   (depletion_om - depletion_em) / depletion_om)
> plot_scalar_boxplot(scalar_dat, x = "E", y = "depletion", horiz = "D",
+   relative.error = TRUE)
Error in as.character(what) : 
  cannot coerce type 'closure' to vector of type 'character'
Calls: plot_scalar_boxplot ... <Anonymous> -> list_to_dataframe -> rbind.fill -> <Anonymous>
Execution halted
```

## StroupGLMM (0.1.0)
Maintainer: Muhammad Yaseen <myaseen208@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘StroupGLMM’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/StroupGLMM.Rcheck/00install.out’ for details.
```

## structSSI (1.1.1)
Maintainer: Kris Sankaran <kriss1@stanford.edu>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘multtest’

Package suggested but not available for checking: ‘phyloseq’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## strvalidator (1.7.0)
Maintainer: Oskar Hansson <oskar.hansson@fhi.no>  
Bug reports: https://github.com/OskarHansson/strvalidator/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘strvalidator’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/strvalidator.Rcheck/00install.out’ for details.
```

## subspaceMOA (0.5.1)
Maintainer: Marwan Hassani <rsubspace@cs.rwth-aachen.de>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘subspaceMOA-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: DSC_HDDStream
> ### Title: Density-based Projected Clustering over High-Dimensional Data
> ### Aliases: DSC_HDDStream
> 
> ### ** Examples
> 
> dsc <- DSC_HDDStream()
Error in rJava::.jcall("SubspaceClustererBuilder", returnSig = "Lmoa/clusterers/hddstream/HDDStream;",  : 
  RcallMethod: cannot determine object class
Calls: DSC_HDDStream -> <Anonymous> -> .External
Execution halted
```

## surveillance (1.12.1)
Maintainer: Sebastian Meyer <Sebastian.Meyer@ifspm.uzh.ch>  
Bug reports: https://r-forge.r-project.org/tracker/?group_id=45

0 errors | 1 warning  | 2 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
glrnb: Fitting Poisson model because alpha == 0
... 8 lines ...
Loading required package: parallel
 **********   GAMLSS Version 4.4-0  ********** 
For more on GAMLSS look at http://www.gamlss.org/
Type gamlssNews() to see new features/changes/bug fixes.


Error: processing vignette 'monitoringCounts.Rnw' failed with diagnostics:
 chunk 17 (label = fPlot1) 
Error in theme(axis.line = element_blank(), panel.grid.major = element_blank(),  : 
  formal argument "axis.line" matched by multiple actual arguments
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’

checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    doc   2.2Mb
```

## survminer (0.2.2)
Maintainer: Alboukadel Kassambara <alboukadel.kassambara@gmail.com>  
Bug reports: https://github.com/kassambara/survminer/issues

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘survminer-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ggsurvplot
> ### Title: Drawing Survival Curves Using ggplot2
> ### Aliases: ggsurvplot print.ggsurvplot
> 
> ### ** Examples
... 56 lines ...
> # Use brewer color palette "Dark2"
> # Add risk table
> ggsurvplot(fit, linetype = "strata",
+           conf.int = TRUE, pval = TRUE,
+           palette = "Dark2", risk.table = TRUE)
Warning: Ignoring unknown aesthetics: x, y
Warning in is.na(x) :
  is.na() applied to non-(list or vector) of type 'NULL'
Error in -x : invalid argument to unary operator
Calls: ggsurvplot ... summary.survfit -> findrow -> find2 -> findInterval
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: ggplot2
Quitting from lines 55-64 (Informative_Survival_Plots.Rmd) 
Error: processing vignette 'Informative_Survival_Plots.Rmd' failed with diagnostics:
there is no package called 'RTCGA.clinical'
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘RTCGA.clinical’
```

## survMisc (0.5.3)
Maintainer: Chris Dardis <christopherdardis@gmail.com>  
Bug reports: https://github.com/dardisco/survMisc/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘survMisc-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: autoplotTableAndPlot
> ### Title: Arrange a survival plot with corresponding table and legend.
> ### Aliases: autoplot.tableAndPlot autoplotTableAndPlot
> ### Keywords: graphics
> 
> ### ** Examples
> 
> data("kidney", package="KMsurv")
> autoplot(survfit(Surv(time, delta) ~ type, data=kidney), type="fill")
Warning: Ignoring unknown aesthetics: x, y
Warning: Ignoring unknown aesthetics: x, y
Error in if (is.nan(times1[1])) times1 <- times : 
  argument is of length zero
Calls: autoplot -> autoplot.survfit
Execution halted
```

## sValues (0.1.4)
Maintainer: Carlos Cinelli <carloscinelli@hotmail.com>  
Bug reports: https://github.com/carloscinelli/sValues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘sValues-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.sValues
> ### Title: Plot method for S-values
> ### Aliases: plot.sValues
> 
> ### ** Examples
> 
> # growth regressions example
> data(economic_growth)
> eg_sv <- sValues(GR6096 ~ ., data = economic_growth)
> plot(eg_sv, R2_bounds = c(0.5, 1))
> plot(eg_sv, R2_bounds = c(0.1, 1))
> plot(eg_sv, type = "beta_plot", variable = "OPENDEC1", error_bar = FALSE)
> plot(eg_sv, type = "beta_plot", variable = "OPENDEC1", error_bar = TRUE)
Error in as.character(what) : 
  cannot coerce type 'closure' to vector of type 'character'
Calls: <Anonymous> ... <Anonymous> -> list_to_dataframe -> rbind.fill -> <Anonymous>
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 154-156 (sValues.Rnw) 
Error: processing vignette 'sValues.Rnw' failed with diagnostics:
cannot coerce type 'closure' to vector of type 'character'
Execution halted

```

## tabplot (1.3)
Maintainer: Martijn Tennekes <mtennekes@gmail.com>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tabplot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.tabplot
> ### Title: Plot a tabplot-object
> ### Aliases: plot.tabplot plot.tabplot_compare
> 
> ### ** Examples
> 
> # load diamonds dataset from ggplot2
> require(ggplot2)
Loading required package: ggplot2
> data(diamonds)
> 
> tab <- tableplot(diamonds)
Error in if (dfile == getOption("fftempdir")) finalizer <- "delete" else finalizer <- "close" : 
  argument is of length zero
Calls: tableplot ... tablePrepare -> lapply -> FUN -> clone -> clone.ff -> assign -> ff
Error in setwd(cwd) : character argument expected
Calls: tableplot ... lapply -> FUN -> clone -> clone.ff -> assign -> ff -> setwd
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Quitting from lines 27-39 (tabplot-timings.Rmd) 
Error: processing vignette 'tabplot-timings.Rmd' failed with diagnostics:
(list) object cannot be coerced to type 'double'
Execution halted

```

## TcGSA (0.10.1)
Maintainer: Boris P. Hejblum <bhejblum@hsph.harvard.edu>  
Bug reports: https://github.com/borishejblum/TcGSA/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘multtest’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## tcgsaseq (1.0.1)
Maintainer: Boris P. Hejblum <bhejblum@hsph.harvard.edu>  
Bug reports: https://github.com/denisagniel/tcgsaseq/issues

1 error  | 0 warnings | 2 notes

```
checking examples ... ERROR
Running examples in ‘tcgsaseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: voom_weights
> ### Title: Precision weights accounting for heteroscedasticity in RNA-seq
> ###   count data
> ### Aliases: voom_weights
> 
... 7 lines ...
> n <- 12
> p <- 2
> y <- t(sapply(1:G, FUN=function(x){rnbinom(n=n, size=0.07, mu=200)}))
> 
> x <- sapply(1:p, FUN=function(x){rnorm(n=n, mean=n, sd=1)})
> 
> my_w <-  voom_weights(y, x, doPlot=TRUE)
> w_voom <- limma::voom(counts=y, design=x, plot=TRUE) #slightly faster than us. Same results
Error in loadNamespace(name) : there is no package called ‘limma’
Calls: :: ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘limma’

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘limma’
```

## TELP (1.0)
Maintainer: Gabriel Henrique Oliveira Assuncao <gabrielhenryoliver@hotmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘TELP-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: telp
> ### Title: The Base Function of This Package The Free Evocation of Words
> ###   Technique
> ### Aliases: telp
> ### Keywords: function
> 
> ### ** Examples
> 
> telp()
Error in structure(.External(.C_dotTclObjv, objv), class = "tclObj") : 
  [tcl] invalid command name "toplevel".
Calls: telp ... tktoplevel -> tkwidget -> tcl -> .Tcl.objv -> structure
Execution halted
```

## toaster (0.5.2)
Maintainer: Gregory Kanevsky <gregory.kanevsky@teradata.com>  
Bug reports: https://github.com/teradata-aster-field/toaster/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘RODBC’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## trackeR (0.0.3)
Maintainer: Hannah Frick <h.frick@ucl.ac.uk>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: zoo

Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Loading required package: ggplot2

Attaching package: 'trackeR'

The following object is masked from 'package:base':

    append

Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=57.157231,-2.104296&zoom=13&size=640x640&scale=2&maptype=terrain&sensor=false
Quitting from lines 90-91 (TourDetrackeR.Rmd) 
Error: processing vignette 'TourDetrackeR.Rmd' failed with diagnostics:
x must be length 1 or 2
Execution halted

```

## UBL (0.0.5)
Maintainer: Paula Branco <paobranco@gmail.com>  
Bug reports: https://github.com/paobranco/UBL/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘UBL’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/UBL.Rcheck/00install.out’ for details.
```

## UpSetR (1.2.4)
Maintainer: Jake Conway <jake_conway@student.uml.edu>  
Bug reports: http://github.com/hms-dbmi/UpSetR/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in readLines(input, encoding = "UTF-8") :
  incomplete final line found on '_output.yaml'
Warning in readLines(input, encoding = "UTF-8") :
  incomplete final line found on '_output.yaml'
Warning in readLines(input, encoding = "UTF-8") :
  incomplete final line found on '_output.yaml'
Warning in readLines(input, encoding = "UTF-8") :
  incomplete final line found on '_output.yaml'
Quitting from lines 77-78 (set.metadata.plots.Rmd) 
Error: processing vignette 'set.metadata.plots.Rmd' failed with diagnostics:
formal argument "axis.title.x" matched by multiple actual arguments
Execution halted


checking R code for possible problems ... NOTE
metadataHeat: possible error in theme(panel.background =
  element_rect("white"), plot.title = element_text(margin = margin(b =
  titleAdjustment), hjust = 0, size = 9.25, angle = 90), plot.margin =
  unit(c(0, 0, 0, 0), "lines"), axis.title.x = element_text(size =
  8.3), legend.position = "none", axis.line = element_blank(),
  axis.line = element_line(colour = "gray0"), axis.line.y =
  element_blank(), axis.line.x = element_line(colour = "gray0", size =
  0.3), axis.text.y = element_blank(), axis.ticks.y = element_blank(),
  axis.title.y = element_blank(), : formal argument "axis.title.x"
... 7 lines ...
  titleAdjustment), hjust = 0, size = 8.3, angle = 90), plot.margin =
  unit(c(0, 0, 0, 0), "lines"), axis.title.x = element_text(size = 7),
  legend.position = "none", axis.line = element_blank(), axis.line =
  element_line(colour = "gray0"), axis.line.y = element_blank(),
  axis.line.x = element_line(colour = "gray0", size = 0.3), axis.text.y
  = element_blank(), axis.ticks.y = element_blank(), axis.title.y =
  element_blank(), : formal argument "axis.title.x" matched by multiple
  actual arguments
metadataText: possible error in axis.text.x = element_blank(),
  axis.ticks.x = element_blank(), axis.title.x = element_blank()):
  formal argument "axis.title.x" matched by multiple actual arguments
```

## userfriendlyscience (0.4-1)
Maintainer: Gjalt-Jorn Peters <gjalt-jorn@userfriendlyscience.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘userfriendlyscience-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: asymmetricalScatterMatrix
> ### Title: asymmetricalScatterMatrix
> ### Aliases: asymmetricalScatterMatrix
> ### Keywords: misc
> 
> ### ** Examples
> 
> asymmetricalScatterMatrix(infert, cols=c("parity", "age"),
+                           rows=c("induced", "case", "spontaneous"),
+                           showCorrelations="top-right");
Error in theme(axis.text = element_text(colour = "#000000", size = rel(0.8)),  : 
  formal argument "strip.background" matched by multiple actual arguments
Calls: asymmetricalScatterMatrix ... dlvTheme -> %+replace% -> is.theme -> inherits -> theme
Execution halted

checking R code for possible problems ... NOTE
didacticPlotTheme: possible error in theme(axis.text =
  element_text(colour = "#000000", size = rel(0.8)), axis.ticks =
  element_line(colour = "black"), legend.text = element_text(size =
  rel(0.6)), legend.key = element_rect(colour = "grey80"),
  legend.position = "top", legend.direction = "horizontal",
  legend.key.size = unit(6, "mm"), panel.background = element_rect(fill
  = "white", colour = NA), panel.border = element_rect(fill = NA,
  colour = "grey50"), panel.grid.major = element_line(colour =
  "grey90", size = 0.2), panel.grid.minor = element_line(colour =
... 13 lines ...
  legend.key.size = unit(6, "mm"), panel.background = element_rect(fill
  = "white", colour = NA), panel.border = element_rect(fill = NA,
  colour = "grey50"), panel.grid.major = element_line(colour =
  "grey90", size = 0.2), : formal argument "strip.background" matched
  by multiple actual arguments
dlvTheme: possible error in panel.grid.minor = element_line(colour =
  "grey98", size = 0.5), strip.background = element_rect(fill =
  "grey80", colour = "grey50"), strip.background = element_rect(fill =
  "grey80", colour = "grey50"), panel.margin = unit(c(0.5), "cm"),
  ...): formal argument "strip.background" matched by multiple actual
  arguments
```

## vdmR (0.2.2)
Maintainer: Tomokazu Fujino <fujino@fwu.ac.jp>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘vdmR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: vhist
> ### Title: Generate histogram with interactive functions
> ### Aliases: vhist
> 
> ### ** Examples
> 
> data(vsfuk2012)
> vhist(FertilityRate, vsfuk2012, "hist1", "vsfuk2012", fill=Type)
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
Error in grid.Call.graphics(L_downviewport, name$name, strict) : 
  Viewport 'panel.3-4-3-4' was not found
Calls: vhist ... downViewport -> downViewport.vpPath -> grid.Call.graphics
Execution halted

checking tests ... ERROR
Running the tests in ‘tests/run-all.R’ failed.
Last 13 lines of output:
  Viewport 'panel.3-4-3-4' was not found
  1: vhist(Sepal.Length, iris, "hist01", "iris", fill = Species) at /Users/hadley/Documents/ggplot/ggplot/revdep/checks/vdmR.Rcheck/vdmR/tests/test-vdmR.R:7
  2: grid::downViewport("panel.3-4-3-4")
  3: downViewport.default("panel.3-4-3-4")
  4: downViewport(vpPath(name), strict, recording = recording)
  5: downViewport.vpPath(vpPath(name), strict, recording = recording)
  6: grid.Call.graphics(L_downviewport, name$name, strict)
  
  DONE ===========================================================================
  Error: Test failures
  In addition: Warning message:
  Placing tests in `inst/tests/` is deprecated. Please use `tests/testthat/` instead 
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
`stat_bin()` using `bins = 30`. Pick better value with `binwidth`.
Quitting from lines 40-42 (vdmR-vignette.Rnw) 
Error: processing vignette 'vdmR-vignette.Rnw' failed with diagnostics:
Viewport 'panel.3-4-3-4' was not found
Execution halted

```

## vipor (0.4.3)
Maintainer: Scott Sherrill-Mix <shescott@upenn.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: sm
Package 'sm', version 2.2-5.4: type help(sm) for summary information

Error: processing vignette 'methodComparison.Rnw' failed with diagnostics:
 chunk 18 (label = showDiamond) 
Error in sort.list(y) : 'x' must be atomic for 'sort.list'
Have you called 'sort' on a list?
Execution halted

```

## vmsbase (2.1.3)
Maintainer: Lorenzo D'Andrea <support@vmsbase.org>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘vmsbase’ can be installed ... ERROR
Installation failed.
See ‘/Users/hadley/Documents/ggplot/ggplot/revdep/checks/vmsbase.Rcheck/00install.out’ for details.
```

## VSE (0.99)
Maintainer: Hansen He <hansenhe@uhnresearch.ca>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘GenomicRanges’ ‘IRanges’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## XGR (1.0.3)
Maintainer: Hai Fang <hfang@well.ox.ac.uk>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available:
  ‘GenomicRanges’ ‘IRanges’ ‘S4Vectors’ ‘supraHex’ ‘rtracklayer’
  ‘BiocGenerics’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

